var textForPages =["Towards\u0009Automated\u0009Structure\u0009Analysis\u0009\u000d\u000aof\u0009Nanoparticles\u000d\u000a\u0009\u0009\u000d\u000aUNIVERSIT Y\u0009OF\u0009COPENHAGEN\u000d\u000aFACULTY\u0009OF\u0009 SCIENCE\u000d\u000aDEPARTMENT\u0009OF\u0009CHEMISTRY\u0009&\u0009NANO - SCIENCE\u0009CENTER\u000d\u000aAndy\u0009Sode\u0009Anker\u000d\u000aPhD\u0009Dissertation","","Towards\u0009Automated\u0009Structure\u0009Analysis\u0009of\u0009\u000d\u000aNanoparticles\u000d\u000aPhD\u0009Dissertation\u0009Submitted\u0009by\u0009Andy\u0009Sode\u0009Anker\u000d\u000aUnder\u0009Supervision\u0009of Kirsten\u0009Marie\u0009\u00d8rnsbjerg\u0009Jensen\u000d\u000aDepartment\u0009of\u0009Chemistry\u000d\u000aFaculty\u0009of\u0009Science\u000d\u000aUniversity\u0009of\u0009Copenhagen\u000d\u000aAugust\u00092023",""," \u000d\u000ai\u000d\u000aContents\u000d\u000aPreface.................................................................................................................................................................................................\u0009iii\u000d\u000aAcknowledgements..........................................................................................................................................................................\u0009v\u000d\u000aList\u0009of\u0009Publications........................................................................................................................................................................vii\u000d\u000aAbstract................................................................................................................................................................................................xi\u000d\u000aDansk\u0009Resum\u00e9................................................................................................................................................................................xiii\u000d\u000a1 Scattering\u0009for\u0009Nanostructure\u0009Characterisation..........................................................................................................\u00091\u000d\u000a1.1 Scattering:\u0009from\u0009Electron\u0009to\u0009Material........................................................................................................................................\u00091\u000d\u000a1.1.1 Scattering\u0009from an\u0009Electron ................................................................................................................................................\u00091\u000d\u000a1.1.2 Scattering\u0009from\u0009an\u0009Atom.......................................................................................................................................................\u00092\u000d\u000a1.1.3 Scattering\u0009from\u0009a\u0009Material...................................................................................................................................................\u00093\u000d\u000a1.1.4 Paper\u0009I...........................................................................................................................................................................................\u00093\u000d\u000a1.2 Total\u0009Scattering\u0009with\u0009Pair\u0009Distribution\u0009Function\u0009Analysis............................................................................................ 10\u000d\u000a1.3 Combined\u0009Modelling\u0009of\u0009Pair\u0009Distribution\u0009Function\u0009and\u0009Small-Angle\u0009X-ray\u0009Scattering\u0009Data ......................... 12\u000d\u000a2 Conventional\u0009Scattering\u0009Data\u0009Modelling ...................................................................................................................15\u000d\u000a2.1.1 Paper\u0009II...................................................................................................................................................................................... 15\u000d\u000a2.2 Pushing\u0009the\u0009Boundaries\u0009of\u0009Conventional\u0009Scattering\u0009Data\u0009Analysis .......................................................................... 45\u000d\u000a2.2.1 Paper\u0009III .................................................................................................................................................................................... 46\u000d\u000a3 Supervised\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis ................................................59\u000d\u000a3.1 Structure\u0009Identification\u0009from\u0009Scattering\u0009Data .................................................................................................................... 61\u000d\u000a3.1.1 Paper\u0009IV..................................................................................................................................................................................... 63\u000d\u000a3.2 Extraction\u0009of\u0009Structural\u0009Motifs\u0009from\u0009Scattering\u0009Data...................................................................................................... 87\u000d\u000a3.2.1 Paper\u0009V...................................................................................................................................................................................... 89\u000d\u000a3.2.2 Paper\u0009VI...................................................................................................................................................................................101\u000d\u000a4 Generative\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis..............................................125\u000d\u000a4.1 A\u0009Step\u0009Towards\u0009a\u0009General\u0009Approach\u0009for\u0009Structure\u0009Determination\u0009of\u0009Nanomaterials ....................................127\u000d\u000a4.1.1 Paper\u0009VII.................................................................................................................................................................................129\u000d\u000a4.1.2 Paper\u0009VIII ...............................................................................................................................................................................138\u000d\u000a4.2 Matching\u0009Experimental\u0009and\u0009Simulated\u0009Data.....................................................................................................................152\u000d\u000a4.2.1 Paper\u0009IX...................................................................................................................................................................................153\u000d\u000a5 Concluding\u0009Remarks\u0009and\u0009Outlook .............................................................................................................................169\u000d\u000a6 A\u0009Word\u0009About\u0009Automation ...........................................................................................................................................171\u000d\u000aReferences......................................................................................................................................................................................173\u000d\u000aAPPENDICES .................................................................................................................................................................................181\u000d\u000aAppendix I\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00091 ......................................................................................................................181\u000d\u000aAppendix II\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00092.....................................................................................................................181\u000d\u000aAppendix III\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00093...................................................................................................................182\u000d\u000aAppendix IV\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00094 ...................................................................................................................183\u000d\u000aReferences.........................................................................................................................................................................................................183","|\u0009Contents\u000d\u000aii\u000d\u000aAPPENDICES\u0009\u2013\u0009PAPERS............................................................................................................................................................185\u000d\u000aPaper\u0009I\u0009\u0009...............................................................................................................................................................................................................185\u000d\u000aPaper\u0009II...............................................................................................................................................................................................................185\u000d\u000aPaper\u0009III.............................................................................................................................................................................................................185\u000d\u000aPaper\u0009IV .............................................................................................................................................................................................................185\u000d\u000aPaper\u0009V\u0009...............................................................................................................................................................................................................185\u000d\u000aPaper\u0009VI.............................................................................................................................................................................................................185\u000d\u000aPaper\u0009VII............................................................................................................................................................................................................197\u000d\u000aPaper\u0009VIII..........................................................................................................................................................................................................197\u000d\u000aPaper\u0009IX .............................................................................................................................................................................................................197\u000d\u000aCIF-Finder\u0009paper............................................................................................................................................................................................197\u000d\u000aIntergrowth paper ........................................................................................................................................................................................221\u000d\u000aTransition\u0009metal\u0009tungstate\u0009paper...........................................................................................................................................................235\u000d\u000a\u0009\u0009"," \u000d\u000aiii\u000d\u000aPreface\u000d\u000aThis\u0009dissertation\u0009has\u0009been\u0009submitted\u0009to\u0009the\u0009Faculty\u0009of\u0009Science\u0009at\u0009the\u0009University\u0009of\u0009Copenhagen\u0009as\u0009part\u0009\u000d\u000aof\u0009the\u0009requirements\u0009to\u0009obtain\u0009a\u0009PhD\u0009degree\u0009in\u0009Chemistry.\u0009The\u0009research\u0009presented\u0009in\u0009this\u0009dissertation\u0009has\u0009\u000d\u000abeen\u0009 conducted\u0009 under\u0009 the\u0009 supervision\u0009 of\u0009 Assoc.\u0009 Prof. Kirsten\u0009 M. \u00d8. Jensen\u0009 at\u0009 the\u0009 Department\u0009 of\u0009\u000d\u000aChemistry\u0009and\u0009Nano-Science\u0009Center,\u0009University\u0009of\u0009Copenhagen\u0009as\u0009part\u0009of\u0009the\u0009integrated\u00093+5\u0009MSc\u0009and\u000d\u000aPhD\u0009 programme\u0009 from\u0009 September\u0009 2018\u0009 to\u0009 August\u0009 2023. The\u0009 overall\u0009 topic\u0009 for\u0009the dissertation\u0009 is\u0009 to\u0009\u000d\u000aautomate\u0009scattering\u0009data analysis\u0009 from\u0009nanoparticles with\u0009a\u0009 focus\u0009on\u0009 total\u0009X-ray\u0009scattering\u0009with\u0009pair\u0009\u000d\u000adistribution\u0009function\u0009analysis.\u0009The\u0009dissertation\u0009has\u0009been\u0009divided\u0009into\u0009six chapters:\u0009\u0009\u000d\u000aChapter\u00091\u0009presents\u0009the\u0009Debye\u0009scattering\u0009equation,\u0009total\u0009scattering\u0009with\u0009pair\u0009distribution\u0009function\u0009\u000d\u000aanalysis,\u0009and\u0009small-angle\u0009X-ray\u0009scattering.\u000d\u000aChapter\u00092\u0009introduces\u0009conventional\u0009scattering\u0009data modelling for nanostructure\u0009characterisation\u000d\u000afollowed\u0009 by\u0009 a\u0009 study (Chapter\u0009 2.1) pushing\u0009 the\u0009 boundaries\u0009 of\u0009 conventional\u0009 scattering\u0009 data\u0009\u000d\u000amodelling.\u0009\u000d\u000aChapter\u00093 serves\u0009as\u0009a\u0009motivation\u0009 for\u0009using\u0009supervised\u0009machine\u0009learning\u0009 to\u0009analyse\u0009scattering\u0009\u000d\u000adata\u0009 in\u0009 materials\u0009 chemistry followed\u0009 by\u0009 two\u0009 studies\u0009 (Chapter\u0009 3.1-2)\u0009 of\u0009 supervised\u0009 machine\u0009\u000d\u000alearning\u0009for\u0009nanostructure\u0009identification\u0009from\u0009scattering\u0009data.\u000d\u000aChapter\u0009 4 stresses three challenges\u0009 of\u0009 applying\u0009 supervised\u0009 machine\u0009 learning\u0009 to\u0009 analyse\u0009\u000d\u000aexperimental\u0009structure\u0009characterisation\u0009data while\u0009introducing\u0009unsupervised\u0009machine\u0009learning.\u0009\u000d\u000aThis is\u0009followed by two\u0009studies\u0009(Chapter\u00094.1-2)\u0009using\u0009generative machine\u0009learning\u0009to\u0009overcome\u0009\u000d\u000achallenges\u0009that\u0009supervised\u0009machine\u0009learning\u0009cannot\u0009overcome.\u000d\u000aDuring\u0009the\u0009PhD\u0009period,\u0009I\u0009spent\u00097\u0009months\u0009as\u0009a\u0009visiting\u0009researcher\u0009in\u0009Dr.\u0009Jeyan\u0009Thiyagalingam\u2019s\u0009research\u0009\u000d\u000agroup\u0009at\u0009Scientific\u0009Computing\u0009Department,\u0009Rutherford\u0009Appleton\u0009Laboratory\u0009in\u0009England\u0009where\u0009I\u0009worked\u0009\u000d\u000aclosely\u0009with\u0009Sr. Lecturer\u0009Keith\u0009T. Butler. Here,\u0009I\u0009witnessed\u0009how\u0009machine\u0009learning\u0009can\u0009be\u0009used\u0009to\u0009analyse\u0009\u000d\u000avarious scientific\u0009 data\u0009 from\u0009 e.g.\u0009 scattering,\u0009 spectroscopy\u0009 and\u0009 imaging experiments.\u0009 Chapter\u0009 4.2\u0009\u000d\u000aexemplifies\u0009how spectroscopy\u0009data\u0009can\u0009be\u0009analysed\u0009with\u0009machine\u0009learning\u0009similar to\u0009what I\u0009have\u0009shown\u0009\u000d\u000apossible\u0009for\u0009scattering\u0009data.\u000d\u000aGeneral\u0009conclusions\u0009and\u0009an\u0009outlook\u0009for\u0009further\u0009work\u0009are\u0009given\u0009in\u0009Chapter\u00095.\u0009\u000d\u000aFinally,\u0009I\u0009will\u0009finish the\u0009dissertation\u0009in\u0009Chapter\u00096 with\u0009A\u0009Word\u0009About\u0009Automation.\u0009",""," \u000d\u000av\u000d\u000aAcknowledgements\u000d\u000aI\u0009feel\u0009extremely privileged.\u0009Like\u0009all\u0009PhD\u0009students,\u0009I\u0009have\u0009faced\u0009numerous\u0009challenges\u0009during\u0009my\u00095-year\u0009\u000d\u000aPhD\u0009journey\u0009\u2013\u0009personal,\u0009scientific,\u0009social,\u0009and\u0009more.\u0009These\u0009hurdles\u0009are\u0009part\u0009of\u0009any\u0009years-long\u0009project.\u000d\u000aHowever,\u0009I\u0009must\u0009say\u0009that\u0009these\u0009challenges\u0009have\u0009been\u0009minimal\u0009in\u0009my\u0009case.\u0009I\u0009have\u0009been\u0009fortunate\u0009to\u0009enjoy\u0009\u000d\u000agreat\u0009scientific\u0009freedom,\u0009sufficient\u0009resources\u0009to\u0009explore\u0009most\u0009of\u0009my\u0009ideas,\u0009and\u0009the\u0009opportunity\u0009to\u0009work\u0009\u000d\u000awith\u0009exceptionally\u0009kind\u0009and\u0009intelligent\u0009people.\u0009I\u0009am\u0009extremely\u0009privileged,\u0009and\u0009for\u0009that,\u0009I\u0009want\u0009to\u0009deeply\u0009\u000d\u000athank\u0009a\u0009series\u0009of\u0009people who\u0009have\u0009made\u0009this\u0009journey\u0009possible!\u000d\u000aI\u0009started\u0009my\u0009academic\u0009journey\u0009in\u0009the\u0009spring\u00092016,\u0009during my\u0009first year\u0009of\u0009my\u0009Bachelor\u0009degree At\u0009the\u0009\u000d\u000atime,\u0009I\u0009volunteered\u0009to\u0009do\u0009research\u0009in\u0009the\u0009lab\u0009of\u0009newly\u0009appointed\u0009tenure-track\u0009assist.\u0009Prof.\u0009Kirsten\u0009M.\u0009\u00d8.\u0009\u000d\u000aJensen.\u0009 In\u0009my\u0009 third\u0009 year,\u0009 I\u0009 undertook\u0009 a\u0009 7.5\u0009 ECTS\u0009 research\u0009 project\u0009 and\u0009 completed\u0009my\u0009 BSc\u0009 project\u0009in\u0009\u000d\u000aKirsten's\u0009group.\u0009Afterwards,\u0009she\u0009recruited\u0009me\u0009 for\u0009 the\u0009integrated\u00093+5\u0009MSc\u0009and PhD\u0009programme. I\u0009am\u0009\u000d\u000aimmensely\u0009grateful\u0009to\u0009Kirsten\u0009for\u0009welcoming\u0009me into\u0009her\u0009lab\u0009at\u0009such\u0009an\u0009early\u0009stage.\u0009Her\u0009decision\u0009to\u0009send\u0009\u000d\u000ame\u0009to\u0009conferences\u0009in\u0009my\u0009second\u0009year\u0009and\u0009summer\u0009schools\u0009in\u0009my\u0009third\u0009provided\u0009me opportunities\u0009that\u0009\u000d\u000aare extremely\u0009rare.\u0009These\u0009experiences\u0009were\u0009instrumental\u0009in\u0009shaping\u0009my\u0009research\u0009into\u0009what\u0009it\u0009is\u0009today.\u0009\u000d\u000aAdditionally,\u0009 I\u0009 appreciate\u0009 the\u0009 support\u0009 and\u0009 guidance\u0009 she\u0009 provided\u0009 during\u0009 my\u0009 PhD,\u0009 along\u0009 with\u0009 the\u0009\u000d\u000asignificant\u0009academic\u0009freedom\u0009to\u0009pursue\u0009my\u0009own\u0009(crazy)\u0009ideas\u0009and\u0009ambitions.\u0009Above\u0009all,\u0009Kirsten\u0009taught\u0009\u000d\u000ame\u0009the\u0009nuances\u0009of\u0009scientific\u0009writing,\u0009a\u0009skill\u0009I\u0009now\u0009consider\u0009invaluable, and\u0009I\u0009have\u0009begun\u0009to\u0009enjoy.\u000d\u000aI\u0009would\u0009like\u0009to\u0009extend\u0009the\u0009gratitude\u0009to\u0009Assist.\u0009Prof.\u0009Raghavendra\u0009Selvan\u0009for\u0009teaching\u0009and\u0009guiding\u0009me\u0009in\u0009\u000d\u000athe\u0009world\u0009of\u0009big\u0009data\u0009science and\u0009to\u0009Sr.\u0009Lecturer\u0009Keith\u0009T.\u0009Butler\u0009for\u0009guiding me\u0009in\u0009the\u0009world\u0009of\u0009big\u0009data\u0009\u000d\u000ain\u0009chemistry. With\u0009all three of\u0009you,\u0009I\u0009had\u0009an\u0009incredible\u0009amount\u0009of\u0009fun,\u0009many\u0009good\u0009scientific\u0009and\u0009personal\u0009\u000d\u000adiscussions\u0009and\u0009I\u0009consider\u0009your\u0009mentoring\u0009for\u0009being\u0009crucial\u0009for\u0009where\u0009I\u0009am\u0009today.\u0009\u000d\u000aMy\u0009research\u0009has\u0009been\u0009generously\u0009supported\u0009by\u0009several\u0009funding\u0009bodies\u0009including\u0009Danscatt,\u0009the\u0009European\u0009\u000d\u000aResearch\u0009 Council,\u0009 the\u0009 Siemens\u0009 Foundation,\u0009 Knud\u0009 H\u00f8jgaard\u0009 Foundation,\u0009 Viet\u0009 Jacobsen\u0009 Foundation,\u0009\u000d\u000aThomas\u0009 B.\u0009 Thriges\u0009 Foundation,\u0009 Henry\u0009 and\u0009 Mary\u0009 Skovs\u0009 Foundation,\u0009 Haynmann\u0009 Foundation,\u0009\u000d\u000aManufacturer\u0009 Vilhelm\u0009 Pedersen\u0009 and\u0009 Wife\u0009 Foundation,\u0009 Augustinus\u0009 Foundation,\u0009 and\u0009 the\u0009 Danish\u0009 Data\u0009\u000d\u000aScience\u0009Academy.\u0009Their\u0009financial\u0009support played\u0009a\u0009pivotal\u0009role\u0009in\u0009the\u0009realisation\u0009of\u0009this\u0009research. I\u0009would\u0009\u000d\u000alike\u0009to\u0009thank\u0009beamline\u0009P02.1\u0009and\u0009P07\u0009at\u0009Petra\u0009III,\u0009the\u0009beamline\u0009SWING\u0009at\u0009Soleil,\u0009the\u0009beamline\u0009ID02\u0009at\u0009\u000d\u000aEuropean\u0009 Synchrotron\u0009 Radiation\u0009 Facility,\u0009 the\u0009 DanMAX\u0009 beamline\u0009 at\u0009 the\u0009 MAX\u0009 IV\u0009 Laboratory\u0009 for\u0009\u000d\u000aconducting\u0009experiments\u0009at\u0009their\u0009instruments.\u0009\u0009\u000d\u000aI\u0009feel\u0009fortunate\u0009to\u0009have\u0009been\u0009part\u0009of\u0009the\u0009Nanostructure\u0009group.\u0009Their\u0009support\u0009and\u0009companionship\u0009made\u0009\u000d\u000athe\u0009research\u0009an\u0009enjoyable\u0009experience.\u0009A\u0009special\u0009note\u0009of\u0009thanks\u0009to\u0009Emil\u0009\u2013\u0009the\u0009scientific\u0009collaboration and\u0009\u000d\u000afriendship\u0009made\u0009a\u0009significant\u0009impact\u0009on\u0009both\u0009my\u0009research\u0009and\u0009personal\u0009journey. I\u0009am\u0009deeply\u0009thankful\u0009to\u0009\u000d\u000aFrederik L.\u0009 Johansen,\u0009 Ulrik Friis-Jensen,\u0009 Olivia Aalling-Frederiksen,\u0009 and\u0009 Assist.\u0009 Prof.\u0009 Rebecca K.\u0009\u000d\u000aPittkowski,\u0009who\u0009has\u0009proofread the\u0009dissertation.\u0009My\u0009gratitude\u0009also\u0009extends\u0009to\u0009the\u0009former\u0009members\u0009of\u0009the\u0009\u000d\u000aNanostructure\u0009group,\u0009individuals\u0009I have met\u0009at\u0009conferences,\u0009summer\u0009schools,\u0009during\u0009my\u0009external\u0009stay,\u0009\u000d\u000abeamtimes,\u0009and\u0009from\u0009the\u0009CHEAC\u0009center.\u0009Our\u0009discussions,\u0009scientific\u0009and\u0009otherwise,\u0009often\u0009held\u0009over\u0009coffee\u0009\u000d\u000aor\u0009beer,\u0009have\u0009evolved\u0009many\u0009of\u0009you\u0009from\u0009colleagues\u0009to\u0009friends.\u000d\u000aFinally,\u0009I\u0009wish\u0009to\u0009express\u0009my\u0009heartfelt\u0009appreciation\u0009to\u0009my\u0009family,\u0009friends,\u0009and\u0009my\u0009wonderful\u0009wife.\u0009Their\u0009\u000d\u000asupport\u0009and\u0009pride\u0009in\u0009my\u0009achievements\u0009carried\u0009me\u0009through\u0009the\u0009challenging\u0009times\u0009of\u0009my\u0009PhD\u0009journey.\u000d\u000aHere's\u0009to\u0009all\u0009those\u0009who\u0009have\u0009made\u0009this\u0009journey\u0009memorable.\u0009Thank\u0009you!\u000d\u000aCopenhagen,\u0009August 2023\u0009\u000d\u000aAndy\u0009Sode\u0009Anker",""," \u000d\u000avii\u000d\u000aList\u0009of\u0009Publications\u000d\u000aThe\u0009list\u0009of\u0009publications\u0009is\u0009arranged in\u0009chronological\u0009order,\u0009with\u0009the\u0009most\u0009recent\u0009publications listed first.\u0009\u000d\u000aSelected manuscripts\u0009are\u0009included in\u0009APPENDICES\u0009\u2013\u0009PAPERS.\u0009An\u0009asterisk\u0009(*)\u0009marks that\u0009the\u0009work\u0009is\u0009\u000d\u000apresented in\u0009the\u0009dissertation.\u0009\u000d\u000a(*)\u0009 Machine\u0009 Learning\u0009 for\u0009 Analysis\u0009 of\u0009 Experimental\u0009 Scattering\u0009 and\u0009 Spectroscopy\u0009 Data\u0009 in\u0009 Materials\u0009\u000d\u000aChemistry,\u0009A.\u0009S.\u0009Anker,\u0009K.\u0009T.\u0009Butler,\u0009R.\u0009Selvan,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009to\u0009be submitted\u0009to\u0009Chemical\u0009Science,\u00092023\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2023-27t53\u000d\u000a(*) POMFinder:\u0009Identifying\u0009polyoxometalate\u0009cluster\u0009structures\u0009from\u0009pair\u0009distribution\u0009function\u0009data\u0009using\u0009\u000d\u000aexplainable\u0009machine\u0009learning,\u0009A.\u0009S.\u0009Anker,\u0009E. T.\u0009S.\u0009Kj\u00e6r,\u0009M. Juelsholt\u0009and\u0009K. M.\u0009\u00d8.\u0009Jensen,\u0009submitted\u0009to\u0009\u000d\u000aIUCrJ,\u00092023\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2023-91xz7\u000d\u000aFormation\u0009of\u0009intermetallic\u0009PdIn\u0009nanoparticles:\u0009Influence\u0009of\u0009surfactants\u0009on\u0009nanoparticle\u0009structure,\u0009B. Wang,\u0009\u000d\u000aJ. K.\u0009Mathiesen,\u0009N. Schlegel,\u0009A.\u0009S.\u0009Anker,\u0009F. L.\u0009Johansen,\u0009A. Kirsch,\u0009E. T. S.\u0009Kj\u00e6r,\u0009O. Aalling-Frederiksen,\u0009T.\u000d\u000aM.\u0009Nielsen,\u0009M. S.\u0009Thomsen,\u0009R. K.\u0009Jakobsen,\u0009M. Arenz,\u0009K. M. \u00d8.\u0009Jensen,\u0009submitted\u0009to\u0009NanoScale\u0009Advances,\u0009\u000d\u000a2023\u000d\u000aIron\u0009Oxide\u0009Nanoparticles\u0009as\u0009a\u0009Contrast\u0009Agent\u0009for\u0009Synchrotron\u0009Imaging\u0009of\u0009Sperm,\u0009M.\u0009B.\u0009Lindh\u00f8j,\u0009S.\u0009R.\u0009Cooper,\u0009\u000d\u000aA.\u0009S.\u0009Anker,\u0009A.\u0009Bonnin,\u0009M.\u0009Kristensen,\u0009K.\u0009Qvortrup,\u0009K.\u0009Almstrup,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009T.\u0009B.\u0009Dyrby,\u0009J.\u0009Sporring,\u0009\u000d\u000asubmitted\u0009to\u0009Heliyon,\u00092023\u000d\u000aThe\u0009human\u0009factor:\u0009results\u0009of\u0009a\u0009small-angle\u0009scattering\u0009data\u0009analysis\u0009Round\u0009Robin,\u0009B.\u0009R.\u0009Pauw,\u0009G.\u0009J.\u0009Smales,\u0009\u000d\u000aA.\u0009S.\u0009Anker,\u0009and\u000946\u0009other\u0009researchers,\u0009submitted\u0009to\u0009Journal\u0009of\u0009Applied\u0009Crystallography,\u00092023\u000d\u000aPreprint\u0009DOI:\u000910.48550/arXiv.2303.03772\u000d\u000aAtomic\u0009 structural\u0009 changes\u0009in\u0009 the\u0009formation\u0009 of\u0009 transition\u0009metal\u0009 tungstates:\u0009 the\u0009 role\u0009 of\u0009 polyoxometalate\u0009\u000d\u000astructures\u0009in\u0009material\u0009crystallization,\u0009S.\u0009L.\u0009Skj\u00e6rv\u00f8, A.\u0009S\u0009Anker,\u0009M.\u0009C.\u0009Wied,\u0009E.\u0009T.\u0009S.\u0009Kj\u00e6r,\u0009M.\u0009Juelsholt,\u0009T.\u0009L.\u0009\u000d\u000aChristiansen,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009Chemical\u0009Science, 14\u0009(18),\u00094806-4816, 2023\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2022-456nv\u000d\u000a(*)\u0009DeepStruc:\u0009 Towards\u0009 structure\u0009 solution\u0009 from\u0009 pair\u0009 distribution\u0009 function\u0009 data\u0009 using\u0009 deep\u0009 generative\u0009\u000d\u000amodels,\u0009E.\u0009T.\u0009S.\u0009Kj\u00e6r\u2021\u0009&\u0009A.\u0009S.\u0009Anker\u2021,\u0009M.\u0009N.\u0009Weng,\u0009S.\u0009J.\u0009L.\u0009Billinge,\u0009R.\u0009Selvan,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009RSC\u0009Digital\u0009\u000d\u000aDiscovery,\u00092,\u000969-80,\u00092023 (Front\u0009cover)\u000d\u000a\u2021 Both\u0009authors\u0009contributed\u0009equally\u0009to\u0009this\u0009work.\u000d\u000aThe\u0009paper was\u0009accepted\u0009at\u0009NeurIPS\u00092022\u0009for\u0009the\u0009Artificial\u0009Intelligence\u0009for\u0009Accelerated\u0009Materials\u0009Design\u0009\u000d\u000aworkshop\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2022-0zrdl-v2\u0009\u0009\u000d\u000a(*)\u0009 Using\u0009 generative\u0009 adversarial\u0009 networks\u0009 to\u0009 match\u0009 experimental\u0009 and\u0009 simulated\u0009 inelastic\u0009 neutron\u0009\u000d\u000ascattering\u0009data,\u0009A.\u0009S.\u0009Anker\u0009(corresponding\u0009author),\u0009K.\u0009T.\u0009Butler\u0009(corresponding\u0009author),\u0009M.\u0009D.\u0009Le,\u0009T.\u0009\u000d\u000aG.\u0009Perring,\u0009J.\u0009Thiyagalingam,\u0009Digital\u0009Discovery,\u00092,\u0009578-590,\u00092023 (Front\u0009cover)\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2022-mlrds\u000d\u000a(*)\u0009Extracting\u0009Structural\u0009Motifs\u0009from\u0009Pair\u0009Distribution\u0009Function\u0009Data\u0009of\u0009Nanostructures\u0009using\u0009Explainable\u0009\u000d\u000aMachine\u0009 Learning,\u0009A.\u0009 S.\u0009 Anker,\u0009 E.\u0009 T.\u0009 S.\u0009 Kj\u00e6r,\u0009 M.\u0009 Juelsholt,\u0009 T.\u0009 L.\u0009 Christiansen,\u0009 S.\u0009 L.\u0009 Skj\u00e6rv\u00f8,\u0009 M.\u0009 R.\u0009 V.\u0009\u000d\u000aJ\u00f8rgensen,\u0009 I.\u0009 Kantor,\u0009 D.\u0009 R.\u0009 S\u00f8rensen,\u0009 S.\u0009 J.\u0009 L.\u0009 Billinge,\u0009 R.\u0009 Selvan,\u0009 K.\u0009 M.\u0009 \u00d8.\u0009 Jensen,\u0009 npj\u0009 Computational\u0009\u000d\u000aMaterials,\u00098\u0009(1),\u0009213,\u00092022","|\u0009List\u0009of\u0009Publications\u000d\u000aviii\u000d\u000aThe\u0009paper was\u0009accepted\u0009at\u0009NeurIPS\u00092022\u0009for\u0009the\u0009Artificial\u0009Intelligence\u0009for\u0009Accelerated\u0009Materials\u0009Design\u0009\u000d\u000aworkshop.\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2022-v1vfx\u000d\u000aExploring\u0009 the\u0009 Composition\u0009 Space\u0009 of\u0009 High-Entropy\u0009 Alloy\u0009 Nanoparticles\u0009 for\u0009 the\u0009 Electrocatalytic\u0009 H2/CO\u0009\u000d\u000aOxidation\u0009with\u0009Bayesian\u0009Optimization,\u0009V.\u0009A.\u0009Mints,\u0009J.\u0009K.\u0009Pedersen,\u0009A.\u0009Bagger,\u0009J.\u0009Quinson, A.\u0009S.\u0009Anker,\u0009K.\u0009M.\u0009\u000d\u000a\u00d8.\u0009Jensen,\u0009J.\u0009Rossmeisl,\u0009M.\u0009Arenz,\u0009ACS\u0009Catalysis,\u000912\u0009(18),\u000911263\u201311271, 2022\u000d\u000aBreaking\u0009with\u0009the\u0009Principles\u0009of\u0009Coreduction\u0009to\u0009Form\u0009Stoichiometric\u0009Intermetallic\u0009PdCu\u0009Nanoparticles,\u0009J.\u0009K.\u0009\u000d\u000aMathiesen,\u0009E.\u0009D.\u0009B\u00f8jesen,\u0009J.\u0009K.\u0009Pedersen,\u0009E.\u0009T.\u0009S.\u0009Kj\u00e6r,\u0009M.\u0009Juelsholt,\u0009S.\u0009Cooper,\u0009J.\u0009Quinson,\u0009A.\u0009S.\u0009Anker,\u0009G.\u0009\u000d\u000aCutts,\u0009D.\u0009S.\u0009Keeble,\u0009M.\u0009S.\u0009Thomsen,\u0009J.\u0009Rossmeisl,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009Small\u0009Methods,\u00096,\u00096,\u00092200420,\u00092022\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2021-zpbqb-v3\u000d\u000aSynthesis\u0009of\u0009fluorescent\u0009polystyrene\u0009nanoparticles:\u0009a\u0009reproducible\u0009and\u0009scalable\u0009method,\u0009D.\u0009Barto\u0161,\u0009L.\u0009Wang,\u0009\u000d\u000aA.\u0009S.\u0009Anker,\u0009M.\u0009Rewers,\u0009O.\u0009Aalling-Frederiksen,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009T.\u0009J.\u0009S\u00f8rensen,\u0009PeerJ\u0009Materials\u0009Science,\u00094,\u0009\u000d\u000a2ee,\u00092022\u000d\u000aCharacterisation\u0009of\u0009intergrowth\u0009in\u0009metal\u0009oxide\u0009materials\u0009using\u0009structure-mining:\u0009the\u0009case\u0009of\u0009\u03b3-MnO2,\u0009N.\u0009P.\u0009\u000d\u000aL.\u0009Magnard, A.\u0009S.\u0009Anker,\u0009O.\u0009Aalling-Frederiksen,\u0009A.\u0009Kirsch,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009Dalton\u0009Transactions,\u000951\u0009(45),\u0009\u000d\u000a17150\u201317161,\u00092022 (Front\u0009cover)\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2022-rk7tp\u000d\u000aWe\u0009 are\u0009 never\u0009 ever\u0009 getting\u0009 (back\u0009 to)\u0009 ideal\u0009 symmetry:\u0009 structure\u0009 and\u0009 luminescence\u0009 in\u0009 a\u0009 ten-coordinated\u0009\u000d\u000aeuropium\u0009 (iii)\u0009 sulfate\u0009 crystal,\u0009 M.\u0009 S.\u0009 Thomsen,\u0009 A.\u0009 S.\u0009 Anker,\u0009 L.\u0009 Kacenauskaite,\u0009 T.\u0009 J.\u0009 S\u00f8rensen,\u0009 Dalton\u0009\u000d\u000aTransactions,\u000952,\u000923,\u00098960\u20138963,\u00092022\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2022-0cbms-v2\u000d\u000aA\u0009simple\u0009setup\u0009miniaturisation\u0009with\u0009multiple\u0009benefits\u0009for\u0009Green\u0009Chemistry\u0009in\u0009nanoparticle\u0009synthesis,\u0009J.\u0009K.\u0009\u000d\u000aMathiesen,\u0009S.\u0009R.\u0009Cooper, A.\u0009S.\u0009Anker,\u0009T.\u0009L.\u0009Kinnibrugh,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009J.\u0009Quinson,\u0009ACS\u0009Omega,\u00097,\u00095,\u00094714\u2013\u000d\u000a4721,\u00092022\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2021-csr9s-v3\u0009\u000d\u000a(*)\u0009Structural\u0009Changes\u0009during\u0009the\u0009Growth\u0009of\u0009Atomically\u0009Precise\u0009Metal\u0009Oxido\u0009Nanoclusters\u0009from\u0009Combined\u0009\u000d\u000aPair\u0009Distribution\u0009Function\u0009and\u0009Small-Angle\u0009X-ray\u0009Scattering\u0009Analysis,\u0009A.\u0009S.\u0009Anker,\u0009T.\u0009L.\u0009Christiansen,\u0009M.\u0009\u000d\u000aWeber,\u0009M.\u0009Schmiele,\u0009E.\u0009Brok,\u0009E.\u0009T.\u0009S.\u0009Kj\u00e6r,\u0009P.\u0009Juh\u00e1s,\u0009R.\u0009Thomas,\u0009M.\u0009Mehring,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009Angewandte\u0009\u000d\u000aChemie,\u000960,\u000920407\u201320416,\u00092021 (Back\u0009cover)\u000d\u000aSize-induced\u0009 amorphous\u0009 structure\u0009 in\u0009 tungsten\u0009 oxide\u0009 nanoparticles,\u0009 M.\u0009 Juelsholt, A.\u0009 S.\u0009 Anker,\u0009 T.\u0009 L.\u0009\u000d\u000aChristiansen,\u0009M.\u0009R.\u0009V.\u0009J\u00f8rgensen,\u0009I.\u0009Kantor,\u0009D.\u0009R.\u0009S\u00f8rensen,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009RSC\u0009Nanoscale,\u000913\u0009(47),\u000920144\u2013\u000d\u000a20156,\u00092021\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv-2021-4xtg9\u000d\u000aFormation\u0009and\u0009growth\u0009mechanism\u0009for\u0009niobium\u0009oxide\u0009nanoparticles:\u0009atomistic\u0009insight\u0009from\u0009in\u0009situ\u0009X-ray\u0009\u000d\u000atotal\u0009scattering,\u0009O.\u0009Aalling-Frederiksen,\u0009M.\u0009Juelsholt,\u0009A.\u0009S.\u0009Anker,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009RSC\u0009Nanoscale,\u000913\u0009(17),\u0009\u000d\u000a8087\u20138097,\u00092021\u000d\u000a(*)\u0009Characterising\u0009the\u0009Atomic\u0009Structure\u0009of\u0009Mono-Metallic\u0009Nanoparticles\u0009from\u0009X-Ray\u0009Scattering\u0009Data\u0009Using\u0009\u000d\u000aConditional\u0009Generative\u0009Models,\u0009A.\u0009S.\u0009Anker\u2021 &\u0009E.\u0009T.\u0009S.\u0009Kj\u00e6r\u2021,\u0009E.\u0009B.\u0009Dam,\u0009S.\u0009J.\u0009L.\u0009Billinge,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen,\u0009R.\u0009\u000d\u000aSelvan,\u0009SIGKDD,\u000916th\u0009International\u0009Workshop\u0009on\u0009Mining\u0009and\u0009Learning\u0009with\u0009Graphs,\u00092020\u000d\u000a\u2021 Both\u0009authors\u0009contributed\u0009equally\u0009to\u0009this\u0009work.\u000d\u000aPreprint\u0009DOI:\u000910.26434/chemrxiv.12662222.v1"," \u000d\u000aix\u000d\u000aManuscripts\u0009in\u0009preparation\u0009with\u0009major\u0009contributions:\u000d\u000a(*) A\u0009 GPU-Accelerated\u0009 Open-Source\u0009 Python\u0009 Package\u0009 for\u0009 Rapid\u0009 Calculation\u0009 of\u0009 the\u0009 Debye\u0009 Scattering\u0009\u000d\u000aEquation:\u0009 Applications\u0009 in\u0009 Small-Angle\u0009 Scattering,\u0009 Powder\u0009 diffraction,\u0009 and\u0009 Total\u0009 Scattering\u0009 with\u0009 Pair\u0009\u000d\u000aDistribution\u0009Function\u0009Analysis,\u0009F. L.\u0009Johansen,\u2021 A.\u0009S.\u0009Anker,\u000d\u000a\u2021 U. Friis-Jensen,\u0009E. B.\u0009Dam,\u0009K. M.\u0009\u00d8.\u0009Jensen,\u0009R.\u000d\u000aSelvan\u0009(in\u0009preparation)\u000d\u000a\u2021 Both\u0009authors\u0009contributed\u0009equally\u0009to\u0009this\u0009work.\u000d\u000a(*) ClusterFinder:\u0009 A\u0009 non-machine\u0009 learning\u0009 approach\u0009 to\u0009 find\u0009 cluster\u0009 structures\u0009 from\u0009 pair\u0009 distribution\u0009\u000d\u000afunction\u0009data,\u0009A.\u0009S.\u0009Anker,\u0009F.\u0009L.\u0009Johansen,\u0009U.\u0009Friis-Jensen, S.\u0009J.\u0009L.\u0009Billinge,\u0009K.\u0009M.\u0009\u00d8.\u0009Jensen\u0009(in\u0009preparation)\u000d\u000aCIF-Finder:\u0009A\u0009machine\u0009learning\u0009tool\u0009for\u0009structure\u0009characterization\u0009of\u0009X-ray\u0009total\u0009scattering\u0009data,\u0009E.\u0009T.\u0009S.\u0009\u000d\u000aKj\u00e6r,\u0009 A.\u0009 S.\u0009 Anker,\u0009 A.\u0009 Kirsch,\u0009 J.\u0009 Lajer,\u0009 O.\u0009 Aalling-Frederiksen,\u0009 S.\u0009 J.\u0009 L.\u0009 Billinge,\u0009 K.\u0009 M.\u0009 \u00d8.\u0009 Jensen\u0009 (In\u0009\u000d\u000apreparation)\u0009",""," \u000d\u000axi\u000d\u000aAbstract\u000d\u000aCurrent\u0009crystallographic\u0009methodologies,\u0009despite\u0009being\u0009critical\u0009for\u0009structure\u0009characterisation\u0009in\u0009materials\u0009\u000d\u000achemistry,\u0009 struggle\u0009 with\u0009 nanomaterials\u0009 due\u0009 to\u0009 their\u0009 limited\u0009 long-range\u0009 order.\u0009 In\u0009 this\u0009 dissertation,\u0009 I\u0009\u000d\u000atherefore\u0009explore\u0009alternative\u0009methodologies,\u0009namely\u0009total\u0009scattering\u0009(TS)\u0009with\u0009pair\u0009distribution\u0009function\u0009\u000d\u000a(PDF)\u0009analysis,\u0009and\u0009 small-angle\u0009X-ray\u0009 scattering\u0009 (SAXS),\u0009leveraging\u0009 the\u0009Debye\u0009 scattering\u0009equation\u0009 to\u0009\u000d\u000acalculate\u0009 the\u0009 scattering\u0009 pattern\u0009 from\u0009 a\u0009 structural\u0009 model\u0009 (Chapter\u0009 1+2).\u0009 It\u0009 is\u0009 demonstrated\u0009 how\u0009\u000d\u000ascattering\u0009data\u0009conventionally\u0009is\u0009modelled\u0009by\u0009performing\u0009a\u0009least-square\u0009refinement\u0009of\u0009the\u0009simulated\u0009and\u0009\u000d\u000aexperimental\u0009data.\u0009This\u0009process\u0009is\u0009widely\u0009known\u0009as\u0009structure\u0009refinement.\u0009While\u0009structure\u0009refinement\u0009is\u0009\u000d\u000aeffective\u0009in\u0009providing\u0009structural\u0009information\u0009from experimental\u0009scattering\u0009data,\u0009this\u0009conventional\u0009data\u0009\u000d\u000amodelling\u0009approach\u0009rely\u0009on\u0009a\u0009structural\u0009model\u0009as\u0009input.\u0009It\u0009therefore requires\u0009time-consuming\u0009literature\u0009\u000d\u000areviews,\u0009followed\u0009by\u0009identification\u0009of the\u0009structural\u0009model\u0009in\u0009structural\u0009databases. Modelling\u0009methods\u0009\u000d\u000ahave\u0009been\u0009developed\u0009that\u0009automate\u0009structural\u0009model\u0009extraction\u0009from\u0009scattering\u0009data\u0009by\u0009refinement\u0009of\u0009\u000d\u000ahundreds\u0009 of\u0009 thousands\u0009 of\u0009 structures.\u0009 However,\u0009 these\u0009 methods\u0009 become\u0009 computationally\u0009 demanding\u0009\u000d\u000a(Chapter\u00092.1).\u000d\u000aInstead,\u0009supervised\u0009machine\u0009learning\u0009 (ML)\u0009can\u0009potentially\u0009accelerate\u0009data\u0009analysis\u0009by\u0009predicting\u0009 the\u0009\u000d\u000arequired\u0009 structural\u0009 model\u0009 for\u0009 conventional\u0009 structure\u0009 refinements\u0009 (Chapter\u0009 3).\u0009 Examples\u0009 are\u0009 shown\u0009\u000d\u000awhere\u0009supervised\u0009ML\u0009instantaneously (<1\u0009sec)\u0009identifies\u0009a\u0009polyoxometalate\u0009cluster\u0009 from\u0009a\u0009structural\u0009\u000d\u000adatabase\u0009(Chapter\u00093.1)\u0009and\u0009where\u0009explainable\u0009supervised\u0009ML\u0009is\u0009used\u0009to\u0009accelerate\u0009the\u0009motif\u0009extraction\u0009\u000d\u000awith\u0009many\u0009orders\u0009of\u0009magnitude\u0009(Chapter\u00093.2);\u0009in\u0009both\u0009examples\u0009from\u0009scattering\u0009data.\u0009\u000d\u000aHowever,\u0009supervised\u0009ML\u0009faces\u0009three\u0009main\u0009challenges:\u00091)\u0009handling\u0009data\u0009with\u0009contributions\u0009from\u0009multiple\u0009\u000d\u000achemical\u0009 components.\u0009 2)\u0009 handling\u0009 data\u0009 from\u0009 structures\u0009 not\u0009 present\u0009in\u0009 the\u0009 training\u0009 database\u0009 and\u0009 3)\u0009\u000d\u000aaccounting\u0009for\u0009experimental\u0009data\u0009that\u0009contains\u0009signals\u0009not\u0009included\u0009in\u0009the\u0009simulated\u0009data\u0009(Chapter\u00094).\u0009\u000d\u000aChallenge\u00091\u0009is\u0009the\u0009most well-studied\u0009of\u0009these\u0009three\u0009challenges\u0009for\u0009scattering\u0009data\u0009and\u0009it\u0009is\u0009therefore\u0009only\u0009\u000d\u000abriefly\u0009 covered\u0009 in\u0009 this\u0009 dissertation.\u0009 Instead, challenges\u0009 2\u0009 and\u0009 3,\u0009 are\u0009 addressed\u0009 using generative\u0009 ML\u0009\u000d\u000amodels which\u0009is a\u0009subfield\u0009of\u0009unsupervised\u0009ML.\u0009 Initial\u0009results\u0009 for\u0009a\u0009subset\u0009of\u0009chemical\u0009systems\u0009show\u0009\u000d\u000apromise\u0009 for\u0009 a\u0009 broader\u0009 use\u0009 for\u0009 scattering\u0009 data\u0009 analysis.\u0009 Specifically,\u0009 issue\u0009 2\u0009 is\u0009 addressed\u0009 by\u0009 using\u0009\u000d\u000agenerative\u0009ML\u0009models\u0009to\u0009solve\u0009mono-metallic\u0009nanoparticles\u0009with\u0009less\u0009than\u0009200\u0009atoms\u0009from\u0009PDF\u0009data.\u0009\u000d\u000aHere,\u0009the\u0009generative\u0009ML\u0009model\u0009demonstrates\u0009considerable\u0009potential\u0009in\u0009interpolating\u0009between\u0009structures\u000d\u000a(Chapter\u00094.1).\u0009Issue\u00093\u0009is\u0009addressed\u0009by\u0009using\u0009generative\u0009ML\u0009models\u0009to\u0009learn\u0009the\u0009distribution\u0009of\u0009simulated\u0009\u000d\u000aand\u0009experimental\u0009data\u0009and\u0009translate\u0009between\u0009these\u0009two\u0009(Chapter\u00094.2).\u0009\u0009\u000d\u000aAlthough\u0009the\u0009current\u0009examples\u0009mainly\u0009analyse\u0009scattering\u0009data,\u0009the\u0009developed\u0009modelling\u0009methodology\u0009is\u0009\u000d\u000aintended\u0009to\u0009be\u0009general\u0009for\u0009scattering\u0009and\u0009spectroscopy\u0009data\u0009from\u0009techniques\u0009where\u0009a\u0009structural\u0009model\u0009\u000d\u000acan\u0009describe\u0009the\u0009experimental\u0009data\u0009through\u0009structure\u0009refinement.\u000d\u000aThe\u0009dissertation\u0009 concludes with\u0009 a\u0009 key\u0009message:\u0009 It\u0009is\u0009 essential\u0009 to\u0009make\u0009 these\u0009 tools\u0009 accessible\u0009 to\u0009 the\u0009\u000d\u000amaterials\u0009chemist!\u0009\u000d\u000aMaking\u0009 the\u0009 ML\u0009 tools\u0009 more\u0009 accessible\u0009 to\u0009 the\u0009 materials\u0009 chemist\u0009 requires\u0009 greater\u0009 transparency\u0009 in\u0009 the\u0009\u000d\u000apublication\u0009of\u0009research\u0009data,\u0009code,\u0009and\u0009software\u0009which\u0009can facilitate\u0009other researchers\u2019\u0009application\u0009of\u000d\u000atrained\u0009ML\u0009models\u0009to\u0009their\u0009own\u0009experimental\u0009data. Equally\u0009important\u0009is\u0009the\u0009ability\u0009of\u0009these ML\u0009models\u0009\u000d\u000ato indicate\u0009the\u0009potential risk\u0009of\u0009failure\u0009when\u0009applied to a\u0009specific\u0009dataset. A\u0009clear\u0009understanding\u0009of\u0009the\u0009\u000d\u000amodels'\u0009limitations helps\u0009to\u0009prevent\u0009misuse,\u0009misinterpretations,\u0009or\u0009inaccurate\u0009results. Moreover,\u0009there\u0009\u000d\u000ais\u0009 a\u0009 need\u0009 for\u0009 ML\u0009 models\u0009 that\u0009 are\u0009 interpretable\u0009 and\u0009 explainable,\u0009 enhancing\u0009 the\u0009 understanding\u0009 of\u0009\u000d\u000aprediction\u0009mechanisms\u0009and\u0009promoting\u0009scientific\u0009insight\u0009(Chapter\u00095).\u0009",""," \u000d\u000axiii\u000d\u000aDansk\u0009Resum\u00e9\u000d\u000aDe\u0009krystallografiske\u0009metoder\u0009som\u0009typisk\u0009bruges\u0009inden\u0009for\u0009materialekemi, er\u0009v\u00e6rdifulde,\u0009men\u0009de\u0009fejler,\u0009\u000d\u000an\u00e5r\u0009vi\u0009unders\u00f8ger\u0009nanomaterialer\u0009uden\u0009langtr\u00e6kkende\u0009orden.\u0009I\u0009denne\u0009afhandling\u0009unders\u00f8ger\u0009jeg\u0009derfor\u0009\u000d\u000aalternative\u0009 metoder,\u0009 nemlig\u0009 total\u0009 spredning\u0009 (TS)\u0009 med\u0009 parfordelingsfunktionsanalyse\u0009 (PDF)\u0009 og\u0009\u000d\u000asm\u00e5vinkel\u0009 r\u00f8ntgenspredning\u0009 (SAXS).\u0009 Her\u0009 bruger\u0009 jeg\u0009 Debye\u0009 spredningsligningen\u0009 til\u0009 at\u0009 beregne\u0009\u000d\u000aspredningsm\u00f8nstret fra\u0009 en\u0009 strukturmodel\u0009 (Kapitel\u0009 1+2). Jeg\u0009 demonstrerer,\u0009 hvordan\u0009 spredningsdata\u0009\u000d\u000akonventionelt\u0009modelleres\u0009 ved\u0009at udf\u00f8re\u0009en\u0009mindste kvadraters\u0009forfining\u0009af\u0009de\u0009simulerede\u0009overfor\u0009de\u000d\u000aeksperimentelle\u0009data.\u0009Denne\u0009proces\u0009er\u0009bredt\u0009kendt\u0009som\u0009strukturforfining.\u0009Selvom\u0009strukturforfining\u0009er\u0009\u000d\u000aeffektiv\u0009til\u0009at\u0009give\u0009strukturel\u0009information\u0009fra\u0009eksperimentelle\u0009spredningsdata,\u0009er\u0009denne\u0009konventionelle\u0009\u000d\u000adatamodelleringsmetode\u0009 afh\u00e6ngig\u0009 af\u0009 en\u0009 strukturmodel\u0009 som\u0009 input.\u0009 Det\u0009 kr\u00e6ver\u0009 omhyggelige\u000d\u000alitteraturunders\u00f8gelser,\u0009 efterfulgt\u0009 af\u0009 identifikation\u0009 af\u0009 strukturmodellen\u0009 i\u0009 en\u0009 strukturdatabaser.\u0009 Som\u0009\u000d\u000aalternativ\u0009til\u0009denne\u0009tidskr\u00e6vende\u0009metode\u0009er modelleringsmetoder blevet\u0009udviklet,\u0009som automatiserer\u0009\u000d\u000abestemmelsen\u0009af\u0009strukturmodellerne fra\u0009spredningsdata\u0009ved\u0009at\u0009forfine hundredtusinder\u0009af\u0009strukturer\u000d\u000atil\u0009at\u0009passe\u0009med\u0009dataene.\u0009Disse\u0009metoder\u0009er\u0009dog beregningstunge\u0009(Kapitel\u00092.1). I\u0009stedet\u0009har superviseret\u0009\u000d\u000amaskinl\u00e6ring\u0009 (ML)\u0009 potentiale\u0009 til\u0009 at accelerere\u0009 dataanalyse\u0009 ved\u0009 at\u0009 identificere den\u0009 n\u00f8dvendige\u0009\u000d\u000astrukturmodel\u0009til\u0009brug\u0009i\u0009konventionelle\u0009strukturforfininger\u0009(Kapitel\u00093).\u0009I\u0009denne\u0009afhandling freml\u00e6gges\u0009\u000d\u000aeksempler,\u0009 hvor\u0009 superviseret\u0009 ML\u0009 p\u00e5\u0009 under\u0009 et\u0009 sekund identificerer\u0009 en\u0009 polyoxometalat-klynge fra\u0009 en\u0009\u000d\u000astrukturdatabase\u0009(Kapitel\u00093.1)\u0009og\u0009hvor\u0009forklarlig\u0009superviseret\u0009ML\u0009bruges\u0009til\u0009at\u0009accelerere\u0009udvindingen\u0009\u000d\u000aaf\u0009strukturelle\u0009motiver\u0009signifikant\u0009(Kapitel\u00093.2);\u0009i\u0009begge\u0009eksempler\u0009fra\u0009spredningsdata.\u000d\u000aSuperviseret\u0009ML\u0009st\u00e5r\u0009i\u0009\u00f8jeblikket\u0009over\u0009for\u0009tre\u0009hovedudfordringer:\u00091)\u0009h\u00e5ndtering\u0009af\u0009data\u0009med\u0009bidrag\u0009fra\u0009\u000d\u000aflere\u0009 kemiske\u0009 komponenter.\u0009 2)\u0009 h\u00e5ndtering\u0009 af\u0009 data\u0009 fra\u0009 strukturer,\u0009 der\u0009 ikke\u0009 er\u0009 repr\u00e6senteret i\u0009\u000d\u000atr\u00e6ningsdatabasen,\u0009og\u00093)\u0009behandling\u0009af\u0009eksperimentelle\u0009data,\u0009der\u0009indeholder\u0009signaler,\u0009som\u0009ikke\u0009findes\u0009i\u0009\u000d\u000ade\u0009 simulerede\u0009 (Kapitel\u0009 4).\u0009 Udfordring\u0009 1\u0009 er\u0009 den\u0009 mest\u0009 velstuderede\u0009 af\u0009 disse\u0009 tre\u0009 udfordringer\u0009 for\u0009\u000d\u000aspredningsdata,\u0009og\u0009den\u0009er\u0009derfor\u0009kun\u0009kort\u0009beskrevet i\u0009denne\u0009afhandling.\u0009Udfordring\u00092\u0009og\u00093\u0009adresseres\u000d\u000aved\u0009 hj\u00e6lp\u0009 af\u0009 generative\u0009 ML-modeller,\u0009 hvilket\u0009 er\u0009 en\u0009 underkategori\u0009 af\u0009 usuperviseret\u0009 ML.\u0009 Indledende\u0009\u000d\u000aresultater\u0009 for\u0009 et\u0009 dels\u00e6t\u0009 af\u0009 kemiske\u0009 systemer\u0009 viser\u0009lovende\u0009 resultater for en\u0009 bredere\u0009 anvendelse\u0009 for\u0009\u000d\u000aspredningsdataanalyse.\u0009Problem\u00092\u0009adresseres ved\u0009hj\u00e6lp\u0009af\u0009generative\u0009ML-modeller\u0009til\u0009at\u0009l\u00f8se\u0009strukturen\u0009\u000d\u000aaf\u0009mono-metalliske\u0009nanopartikler\u0009med\u0009mindre\u0009end\u0009200\u0009atomer\u0009fra\u0009PDF-data.\u0009I\u0009dette\u0009eksempel viser den\u0009\u000d\u000agenerative\u0009ML-model\u0009betydeligt\u0009potentiale i\u0009at\u0009interpolere\u0009mellem\u0009strukturer\u0009(Kapitel\u00094.1).\u0009Udfordring\u0009\u000d\u000a3\u0009behandles\u0009ved\u0009hj\u00e6lp\u0009af\u0009generative\u0009ML-modeller\u0009til\u0009at\u0009l\u00e6re\u0009at\u0009modellere sandsynlighedsfordelingen af\u0009\u000d\u000asimulerede\u0009og\u0009eksperimentelle\u0009data\u0009og\u0009overs\u00e6tte\u0009mellem\u0009disse\u0009to\u0009(Kapitel\u00094.2).\u000d\u000aSelvom\u0009 eksemplerne\u0009 pr\u00e6senteret\u0009 her, hovedsageligt\u0009 analyserer\u0009 r\u00f8ntspredningsdata,\u0009 er\u0009 der\u0009 et\u0009 stort\u0009\u000d\u000apotentiale\u0009for\u0009udvidelse\u0009og\u0009brug\u0009til\u0009analyse\u0009af\u0009sprednings-\u0009og\u0009spektroskopidata\u0009fra\u0009andre\u0009teknikker,\u0009hvor\u0009\u000d\u000aen\u0009strukturmodel\u0009kan\u0009beskrive\u0009de\u0009eksperimentelle\u0009data\u0009gennem\u0009strukturforfining.\u000d\u000aAfhandlingen\u0009slutter\u0009med\u0009et\u0009n\u00f8gle-budskab:\u0009Det\u0009er\u0009afg\u00f8rende\u0009at\u0009g\u00f8re\u0009disse\u0009v\u00e6rkt\u00f8jer\u0009tilg\u00e6ngelige\u0009for\u0009\u000d\u000aden\u0009generelle\u0009materialekemiker!\u000d\u000aAt\u0009 g\u00f8re\u0009 ML-v\u00e6rkt\u00f8jerne\u0009 mere\u0009 tilg\u00e6ngelige\u0009 for\u0009 materialekemikeren\u0009 kr\u00e6ver\u0009 st\u00f8rre\u0009 transparens\u0009 i\u0009\u000d\u000aoffentligg\u00f8relsen\u0009 af\u0009 forskningsdata,\u0009 kode\u0009 og\u0009 software,\u0009 som\u0009 kan\u0009 lette\u0009 andre\u0009 forskeres\u0009 anvendelse\u0009 af\u0009\u000d\u000atr\u00e6nede\u0009ML-modeller\u0009p\u00e5 deres\u0009egne\u0009eksperimentelle\u0009data.\u0009Lige\u0009s\u00e5\u0009vigtig\u0009er\u0009evnen\u0009hos\u0009disse\u0009ML-modeller\u0009\u000d\u000atil\u0009at\u0009indikere\u0009den\u0009potentielle\u0009risiko\u0009for\u0009fejl,\u0009n\u00e5r\u0009de\u0009anvendes\u0009p\u00e5\u0009et\u0009specifikt\u0009datas\u00e6t.\u0009En\u0009klar\u0009forst\u00e5else\u0009af\u0009\u000d\u000amodellernes\u0009 begr\u00e6nsninger\u0009 hj\u00e6lper\u0009 med\u0009 at\u0009 forhindre\u0009 misbrug,\u0009 misfortolkninger\u0009 eller\u0009 un\u00f8jagtige\u0009\u000d\u000aresultater.\u0009Desuden\u0009er\u0009der\u0009et\u0009behov\u0009for\u0009ML-modeller,\u0009der\u0009kan\u0009fortolkes\u0009og\u0009er\u0009forklarlige,\u0009hvilket\u0009fremmer\u0009\u000d\u000avidenskabelig\u0009indsigt\u0009(Kapitel\u00095).",""," \u000d\u000a1\u000d\u000a1 Scattering\u0009for\u0009Nanostructure\u0009Characterisation*\u000d\u000aThe\u0009development\u0009of\u0009advanced,\u0009 functional\u0009materials\u0009relies on\u0009understanding\u0009the\u0009intricate\u0009relationship\u0009\u000d\u000abetween\u0009 material\u0009 structure\u0009 and\u0009 properties.\u0009 Over\u0009 the\u0009 past\u0009 century,\u0009 crystallographic\u0009 methods\u0009 using\u0009\u000d\u000ascattering\u0009and\u0009diffraction\u0009have\u0009been\u0009essential\u0009for\u0009materials\u0009chemistry research.\u0009Crystallography\u0009enables\u000d\u000aab\u0009 initio determination\u0009 of\u0009 crystal\u0009 structures\u0009 from\u0009 diffraction\u0009 data\u0009 and\u0009 has\u0009 provided\u0009 a\u0009 wealth\u0009 of\u000d\u000aknowledge\u0009 about crystal\u0009 chemistry,\u0009 which is\u0009 now\u0009 employed in\u0009 the\u0009 design\u0009 of\u0009 functional\u0009 materials.2,3\u000d\u000aHowever,\u0009 in\u0009 the\u0009 case\u0009 of\u0009 nanomaterials,\u0009with\u0009 limited\u0009 long-range\u0009 order,\u0009traditional\u0009 crystallography\u0009 is\u000d\u000achallenged,\u0009 and\u0009 ab\u0009 initio structure\u0009 determination is\u0009 not\u0009 currently\u0009 possible.4 Instead,\u0009 TS with\u0009 PDF\u0009\u000d\u000aanalysis\u0009has\u0009become\u0009a vital\u0009tool\u0009for\u0009characterising nanomaterial\u0009structures.5,6 The\u0009PDF is\u0009a\u0009real-space\u0009\u000d\u000afunction\u0009 obtained from the\u0009 Fourier\u0009 transform\u0009 of\u0009 the\u0009 normalised\u0009 and\u0009 corrected\u0009 X-ray,\u0009 neutron,\u0009 or\u0009\u000d\u000aelectron\u0009reciprocal-space\u0009scattering\u0009signal.\u0009 It represents\u0009a\u0009histogram\u0009of\u0009inter-atomic\u0009distances\u0009in\u0009 the\u0009\u000d\u000asample.\u0009 Compared\u0009 to\u0009 crystallographic\u0009 methods\u0009 relying\u0009 on\u0009 long-range\u0009 order,\u0009 PDF\u0009 analysis\u0009 provides\u000d\u000aatomic\u0009 structure\u0009 information\u0009 for\u0009 nanomaterials,4,7,8 disordered,5,9,10 or\u0009 amorphous\u0009 materials.7,8,11\u000d\u000aAlthough PDF\u0009is\u0009less\u0009sensitive\u0009to\u0009size\u0009and\u0009shape,\u0009complementing it\u0009with\u0009SAXS\u0009can\u0009offer\u0009insights\u0009into\u0009the\u0009\u000d\u000asize\u0009distribution\u0009and\u0009morphology\u0009of\u0009the\u0009nanoparticles.4,12-14\u000d\u000aBoth\u0009PDF\u0009and\u0009SAXS can\u0009be\u0009modelled\u0009by\u0009employing\u0009the\u0009Debye scattering\u0009equation\u0009for\u0009the\u0009calculation\u0009of\u0009\u000d\u000ascattering\u0009intensities,\u0009considering the\u0009position\u0009of\u0009each\u0009atom\u0009in\u0009the\u0009structure:15,16\u000d\u000aEquation\u00091.1\u0009 !(#) = &&'!(#)'\"(#)\u000d\u000asin (#,!\")\u000d\u000a#,!\"\u000d\u000a#\u000d\u000a\"$%\u000d\u000a#\u000d\u000a!$%\u000d\u000aIn\u0009this\u0009equation,\u0009# is\u0009the\u0009scattering\u0009vector,\u0009,!\" is the\u0009distance\u0009between\u0009atom-pair, - and\u0009.,\u0009and\u0009' is\u0009the\u0009\u000d\u000aatomic\u0009scattering\u0009factor. Despite\u0009being\u0009computationally\u0009expensive,\u0009the\u0009Debye\u0009scattering\u0009equation\u0009can\u0009be\u0009\u000d\u000aused to\u0009compute\u0009the\u0009scattering\u0009pattern of\u0009any\u0009structure.\u000d\u000aThe\u0009following\u0009sections will\u0009outline\u0009the scattering\u0009theory\u0009behind\u0009the\u0009Debye scattering equation\u0009and\u0009TS\u000d\u000awith\u0009 PDF\u0009 analysis.\u0009 Finally,\u0009 an\u0009 example\u0009 demonstrating\u0009 the\u0009 value\u0009 of\u0009 combining\u0009 PDF\u0009 modelling with\u0009\u000d\u000aadditional\u0009data\u0009from\u0009SAXS\u0009will\u0009be\u0009presented.\u000d\u000a1.1 Scattering:\u0009from\u0009Electron\u0009to\u0009Material\u000d\u000aBoth\u0009 in-house\u0009 X-ray\u0009 scattering\u0009 equipment\u0009 and\u0009 synchrotron-based\u0009 systems\u0009 rely\u0009 on\u0009 the\u0009 interaction\u0009\u000d\u000abetween\u0009X-rays\u0009and\u0009the electron\u0009cloud of\u0009the atoms in\u0009the\u0009material.\u0009In\u0009this\u0009section,\u0009we\u0009will\u0009investigate\u000d\u000athe\u0009 scattering process\u0009 that\u0009occurs when\u0009an\u0009X-ray\u0009 beam\u0009interacts\u0009with\u0009a single\u0009 unbound electron,\u0009an\u0009\u000d\u000aatom,\u0009 and\u0009 finally,\u0009 a\u0009 material.\u0009 While\u0009 the\u0009 principles\u0009 of\u0009 electron\u0009 and\u0009 neutron\u0009 scattering\u0009 share\u0009 many\u0009\u000d\u000asimilarities with\u0009X-ray\u0009scattering,\u0009these\u0009will\u0009not be\u0009covered\u0009in\u0009detail,\u0009as\u0009X-ray\u0009techniques\u0009have\u0009mainly\u0009\u000d\u000abeen\u0009used\u0009for my\u0009research.\u0009\u000d\u000a1.1.1 Scattering\u0009from\u0009an\u0009Electron\u000d\u000aWe\u0009will\u0009begin\u0009by\u0009examining\u0009the\u0009scattering\u0009of\u0009X-rays\u0009from\u0009a\u0009single\u0009unbound\u0009electron,\u0009as\u0009shown\u0009in\u0009Figure\u0009\u000d\u000a1.1.\u0009When\u0009an\u0009oscillating\u0009electromagnetic\u0009wave,\u0009such\u0009as\u0009X-rays,\u0009interacts\u0009with\u0009an\u0009electron,\u0009it\u0009causes\u0009the\u0009\u000d\u000aelectron\u0009 to\u0009 oscillate,\u0009 thus\u0009 leading\u0009 to\u0009 acceleration.\u0009 As\u0009 per\u0009 the\u0009 fundamental\u0009 principles\u0009 of\u0009\u000d\u000aelectromagnetism,\u0009 any\u0009 accelerated\u0009 charged\u0009 particle\u0009 emits\u0009 electromagnetic\u0009 waves.\u0009 Here,\u0009 several\u0009\u000d\u000aprocesses\u0009 may\u0009 occur\u0009 which\u0009 can\u0009 either\u0009 be\u0009 coherent\u0009 or\u0009 incoherent\u0009 and\u0009 elastic\u0009 or\u0009 inelastic.\u0009 Thomson\u0009\u000d\u000ascattering,\u0009shown\u0009in\u0009Figure\u00091.1,\u0009is\u0009the\u0009process\u0009of\u0009elastic\u0009and\u0009coherent\u0009scattering\u0009i.e.\u0009the\u0009energy\u0009of\u0009the\u0009Xray\u0009beam\u0009is\u0009unchanged\u0009and\u0009the\u0009phase\u0009is\u0009shifted\u0009by\u0009a\u0009constant. Another\u0009common\u0009scattering\u0009process\u0009is\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009adapted\u0009from\u0009the\u0009theory\u0009section\u0009of\u0009my\u0009Master\u2019s Thesis.\u000d\u000a1","|\u0009Scattering\u0009for\u0009Nanostructure\u0009Characterisation\u000d\u000a2\u000d\u000aCompton\u0009 scattering,\u0009 which\u0009 is\u0009 incoherent\u0009 and\u0009 inelastic and\u0009 appears\u0009 as\u0009 a\u0009 diffuse\u0009 background\u0009 in\u0009 the\u0009\u000d\u000ascattering\u0009 pattern. In\u0009 this\u0009 work,\u0009 we\u0009 will\u0009 focus\u0009 on\u0009 Thomson\u0009 scattering,\u0009 as\u0009 it\u0009 contains\u0009 the\u0009 structural\u0009\u000d\u000ainformation\u0009 of\u0009 the\u0009materials. The\u0009 outgoing\u0009X-ray\u0009wave\u0009 from\u0009 the\u0009electron\u0009propagates\u0009 spherically\u0009and\u0009\u000d\u000aexperiences\u0009a\u0009phase\u0009shift\u0009of\u0009/ compared\u0009to\u0009the\u0009incoming\u0009X-ray\u0009wave.17-19\u000d\u000aFigure\u00091.1:\u0009The\u0009incoming\u0009wave, with\u0009wavevector !!\"!#,\u0009is\u0009scattered\u0009on the\u0009unbound\u0009electron\u0009resulting\u0009in\u0009an outgoing\u0009\u000d\u000awave\u0009with\u0009wavevector\u0009!$!\"%&.\u0009The\u0009change\u0009in\u0009wavevectors,\u0009!!\"!#\u0009\u2013\u0009!$!\"%&,\u0009is\u0009referred\u0009to\u0009as\u0009the\u0009scattering\u0009vector,\u0009\".\u0009#\u000d\u000ais\u0009half\u0009of\u0009the\u0009scattering\u0009angle\u0009between\u0009!!\"!# and\u0009!$!\"%&.\u0009Figure and Figure\u0009caption are\u0009adapted\u0009 from\u0009my\u0009Master\u2019s\u000d\u000aThesis.1\u000d\u000aAs\u0009illustrated in\u0009Figure\u00091.1,\u0009the\u0009scattering\u0009process\u0009can\u0009be\u0009described\u0009by\u0009the\u0009scattering\u0009vector,\u00090,\u0009which\u0009is\u0009\u000d\u000adefined\u0009as the\u0009difference\u0009between the\u0009wavevectors\u0009of\u0009the\u0009incident\u0009X-ray\u0009beam,\u00091&'&(, and\u0009the\u0009scattered\u0009Xray\u0009beam, 1)&'*+.\u0009To\u0009fulfil\u0009the\u0009conditions\u0009for\u0009elastic\u0009scattering\u0009we\u0009have\u00092 = |1!,!-| = 41.!,/04.\u0009\u000d\u000aFor\u0009 Thomson\u0009 scattering,\u0009 the\u0009magnitude\u0009 of\u0009 the\u0009 wavevectors\u0009is\u0009 12\u000d\u000a3 ,\u0009 where\u0009 5 is\u0009 the\u0009 wavelength\u0009 of\u0009 the\u0009\u000d\u000aincoming\u0009X-ray\u0009beam.\u0009From\u0009geometrical\u0009considerations in\u0009Figure\u00091.1, the magnitude\u0009of\u0009the\u0009scattering\u0009\u000d\u000avector can\u0009be\u0009derived.\u000d\u000aEquation\u00091.2\u0009 |0| = Q = 228-9(:) = 4\u03c0\u000d\u000a5 sin(:)\u000d\u000a1.1.2 Scattering\u0009from\u0009an\u0009Atom\u000d\u000aSo\u0009 far,\u0009we\u0009have\u0009only\u0009considered\u0009point-scattering\u0009 from\u0009a\u0009single\u0009electron.\u0009However,\u0009 the\u0009electron\u0009cloud\u0009\u000d\u000aaround\u0009an\u0009atom\u0009usually\u0009consists\u0009of\u0009more\u0009than\u0009one\u0009electron.\u0009When\u0009we\u0009consider\u0009scattering\u0009from\u0009multiple\u0009\u000d\u000aelectrons\u0009 in\u0009 the\u0009 same\u0009 atom,\u0009 we\u0009 must\u0009 include\u0009 the\u0009 effect\u0009 of\u0009 constructive\u0009 or\u0009 destructive\u0009 interference\u000d\u000abetween\u0009waves\u0009scattered\u0009from different\u0009points\u0009within\u0009the\u0009electron\u0009cloud.\u0009The\u0009interference\u0009effects\u0009will\u0009\u000d\u000abe\u0009dependent\u0009on\u0009the\u0009morphology\u0009of\u0009the\u0009electron\u0009cloud\u0009and\u0009the\u0009wavelength of\u0009the\u0009X-rays.\u0009Therefore,\u0009the\u0009\u000d\u000ascattering\u0009amplitude,\u0009!!\"#$(#),\u0009will\u0009be\u00090-dependent\u0009and\u0009is\u0009defined\u0009as\u0009the\u0009atomic\u0009form\u0009factor,\u0009'(0).\u0009\u000d\u000aEffectively\u0009the\u0009atomic\u0009form\u0009factor\u0009is\u0009a\u0009Fourier\u0009transform\u0009of\u0009all\u0009electron\u0009positions,\u0009=,\u0009in\u0009the\u0009electron\u0009cloud\u0009\u000d\u000adistribution,\u0009>.\u000d\u000a20,21\u0009\u0009\u000d\u000aEquation\u00091.3\u0009 !!\"#$(#) = '(0) = % >(=)?&4\u22196@,\u000d\u000aFrom\u0009this\u0009equation,\u0009we\u0009can\u0009derive\u0009that\u0009when\u0009# \u2192 0 \u00c57%,\u0009then\u0009' \u2192 D, where Z is the number of electrons \u000d\u000ain the atom.\u0009For\u0009X-rays,\u0009the\u0009atomic\u0009form\u0009factor\u0009rapidly\u0009decreases\u0009with\u0009increasing\u0009scattering\u0009vector due\u0009to\u0009\u000d\u000athe\u0009spatial\u0009extension\u0009of\u0009the\u0009electron\u0009cloud.\u0009This\u0009is\u0009also\u0009the\u0009case\u0009for\u0009electron\u0009scattering\u0009which also\u0009scatters\u000d\u000afrom\u0009the\u0009electron\u0009cloud.\u0009However,\u0009this\u0009is\u0009not\u0009the\u0009case\u0009for\u0009neutrons\u0009that\u0009scatter\u0009from\u0009the\u0009nuclei\u0009since\u0009the\u0009"," \u000d\u000a3\u000d\u000anuclei\u0009can\u0009practically\u0009be\u0009seen\u0009as\u0009a\u0009point\u0009in\u0009space\u0009without\u0009extension.\u0009Consequently,\u0009 the\u0009neutron\u0009 form\u0009\u000d\u000afactor\u0009is\u0009a\u0009Fourier\u0009transform\u0009of\u0009a\u0009point\u0009scatterer\u0009(the\u0009nuclei)\u0009which\u0009results\u0009in\u0009a\u0009constant.\u000d\u000a1.1.3 Scattering\u0009from\u0009a\u0009Material\u000d\u000aWe\u0009will\u0009now\u0009extend\u0009the\u0009theory\u0009of\u0009scattering\u0009from\u0009a\u0009single\u0009atom\u0009to\u0009materials\u0009which\u0009consist\u0009of\u0009multiple\u0009\u000d\u000aatoms.\u0009By\u0009ignoring\u0009multiple\u0009scattering\u0009processes,\u0009we can sum\u0009the\u0009scattering\u0009contribution\u0009of\u0009each\u0009atom,\u0009\u000d\u000a.,\u0009 in\u0009 the\u0009 material\u0009 with\u0009 position,\u0009 =.\u0009 The\u0009 scattering\u0009 amplitude,\u0009 !$!\"%&'!((#), is\u0009 thereby\u0009 the\u0009 Fourier\u0009\u000d\u000atransform\u0009of\u0009all\u0009atomic\u0009positions in\u0009the\u0009material.\u000d\u000aEquation\u00091.4\u0009 !$!\"%&'!((#) = '()(#))'*\u2219,'\u000d\u000a-\u000d\u000a)./\u000d\u000aBy\u0009measuring\u0009the\u0009scattering\u0009amplitude,\u0009it\u0009would\u0009be possible\u0009to\u0009extract\u0009the\u0009atomic\u0009position\u0009of\u0009each\u0009atom\u0009\u000d\u000ain\u0009the\u0009material.\u0009In\u0009practice,\u0009it\u0009is\u0009not\u0009the\u0009amplitude\u0009that\u0009is\u0009measured\u0009during\u0009a\u0009scattering\u0009experiment\u0009but\u0009\u000d\u000athe\u0009scattering\u0009intensity,\u0009!,\u0009which\u0009is\u0009proportional\u0009to\u0009the\u0009magnitude\u0009of\u0009the\u0009squared\u0009scattering\u0009amplitude.21\u000d\u000aEquation\u00091.5\u0009 *(#) \u221d |!$!\"%&'!((#)|0\u000d\u000aIn\u00091915,\u0009Debye\u0009derived\u0009that\u0009for\u0009materials\u0009with\u0009isotropic\u0009scattering,\u0009the intensity is\u0009independent of the\u000d\u000ascattering vector direction\u0009and can\u0009be\u0009rewritten\u0009to Equation\u00091.1\u0009(the\u0009Debye\u0009scattering\u0009equation) which\u0009\u000d\u000ais\u0009repeated\u0009here:15,22\u0009\u0009\u000d\u000aEquation\u00091.6\u0009 $(&) = ))*!(&)*((&)\u000d\u000asin (&/!()\u000d\u000a&/!(\u000d\u000a)\u000d\u000a(*+\u000d\u000a)\u000d\u000a!*+\u000d\u000aBy\u0009examining\u0009the\u0009intensity\u0009of\u0009the\u0009scattered\u0009X-rays\u0009from\u0009a\u0009material\u0009as\u0009a\u0009function\u0009of\u0009#,\u0009we\u0009can\u0009retrieve\u0009\u000d\u000ainformation\u0009about\u0009the\u0009structure\u0009of\u0009the\u0009material.\u0009The\u0009Debye\u0009scattering\u0009equation\u0009is\u0009extremely\u0009versatile\u000d\u000aand\u0009can\u0009be\u0009used\u0009to\u0009calculate\u0009the\u0009scattering\u0009of\u0009any isotropic\u0009scattering\u0009material.\u0009However, its\u0009applicability\u0009\u000d\u000ahas\u0009been\u0009limited\u0009to\u0009smaller\u0009atomic\u0009structures\u0009because\u0009the\u0009computation\u0009of\u0009the\u0009double\u0009sum\u0009scales\u0009as\u0009O(n2)\u0009\u000d\u000awith\u0009the\u0009number\u0009of\u0009atoms.\u000d\u000a1.1.4 Paper\u0009I\u000d\u000aPaper\u0009 I\u0009 introduces\u0009 DebyeCalculator,\u0009 a\u0009 GPU-accelerated\u0009 open-source\u0009 Python\u0009 Package\u0009 for\u0009 rapid\u0009\u000d\u000acalculation\u0009of\u0009the\u0009Debye\u0009scattering\u0009equation.\u0009DebyeCalculator\u0009is\u0009orders\u0009of\u0009magnitude\u0009faster\u0009than DiffPyCMI23 because\u0009 the\u0009 double\u0009 sum\u0009 calculations\u0009 are\u0009 distributed\u0009 across\u0009 multiple\u0009 GPU\u0009 cores.\u0009 The\u0009 Python\u0009\u000d\u000apackage\u0009is freely\u0009available licensed\u0009under\u0009the\u0009Apache\u0009License\u00092.0 while\u0009being\u0009easily\u0009assessable\u0009on\u0009Google\u0009\u000d\u000aColab\u0009without\u0009the\u0009need\u0009of\u0009local\u0009software\u0009installations and\u0009with\u0009a\u0009graphical\u0009user\u0009interface.\u0009\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009Paper\u0009I which\u0009is\u0009in\u0009preparation.\u0009","DRAFT\u000d\u000a1 A GPU-Accelerated Open-Source Python Package for\u000d\u000a2 Rapid Calculation of the Debye Scattering Equation:\u000d\u000a3 Applications in Small-Angle Scattering, Powder\u000d\u000a4 Scattering, and Total Scattering with Pair Distribution\u000d\u000a5 Function Analysis\u000d\u000aFrederik L. Johansen 1,2*\u00b6, Andy S. Anker 1*\u00b6, Ulrik Friis-Jensen 1,2 6 ,\u000d\u000aErik B. Dam 2, Kirsten M. \u00d8. Jensen 1\u00b6, and Raghavendra Selvan 2,3\u00b6 7\u000d\u000a8 1 Department of Chemistry & Nano-Science Center, University of Copenhagen, Denmark 2 Department\u000d\u000a9 of Computer Science, University of Copenhagen, Denmark 3 Department of Neuroscience, University of\u000d\u000a10 Copenhagen, Denmark w Corresponding author * These authors contributed equally.\u000d\u000aDOI: 10.xxxxxx/draft\u000d\u000aSoftware\u000d\u000a\u2022 Review\u000d\u000a\u2022 Repository\u000d\u000a\u2022 Archive\u000d\u000aEditor: Open Journals\u000d\u000aReviewers:\u000d\u000a\u2022 @openjournals\u000d\u000aSubmitted: 01 January 1970\u000d\u000aPublished: unpublished\u000d\u000aLicense\u000d\u000aAuthors of papers retain copyright\u000d\u000aand release the work under a\u000d\u000aCreative Commons Attribution 4.0\u000d\u000aInternational License (CC BY 4.0).\u000d\u000a11 Summary\u000d\u000a12 The Debye scattering equation, derived in 1915 by Peter Debye, is used to calculate the\u000d\u000a13 scattering intensities considering the position of each atom in the structure:(Debye, 1915;\u000d\u000a14 Scardi et al., 2016)\u000d\u000a\u0a70\u0009\u0a78\u000a =\u000d\u000a\ud835\u000d\u000a\u1731\u000d\u000a\ud835=1\u000d\u000a\ud835\u000d\u000a\u1731\u000d\u000a\ud835=1\u000d\u000a\u0a83\ud835\u0a83\ud835\u000d\u000asin\u0009\u0a78\u0a92\ud835\ud835\u000a\u000d\u000a\u0a78\u0a92\ud835\ud835\u000d\u000a(1)\u000d\u000a15 In this equation Q is the momentum transfer of the scattered radiation, N is the number\u000d\u000aof atoms and r\u03bd\u03bc 16 is the distance between atoms \u03bd and \u03bc. For X-ray radiation, the atomic\u000d\u000a17 scattering factor, b, depends strongly on Q and is usually denoted as f(Q), but for neutrons b is\u000d\u000a18 independent of Q. The Debye scattering equation can be used to compute the scattering pattern\u000d\u000a19 of any atomic structure and is commonly used to study both crystalline and non-crystalline\u000d\u000a20 materials with a range of scattering techniques like powder difraction (PD), total scattering\u000d\u000a21 (TS) with pair distribution function (PDF) analysis and small-angle scattering (SAS).(Scardi\u000d\u000a22 et al., 2016) Although the Debye scattering equation is extremely versatile, the computation\u000d\u000aof the double sum, which scales \u0a76\u0009\u0a75223 \u000a, has limited the practical use of the equation.\u000d\u000a24 With the advancement in computer hardware,(Schaller, 1997) larger structures are now feasible\u000d\u000a25 to analyse using the Debye scattering equation. Modern central processing Units (CPUs),\u000d\u000a26 ranging from tens to hundreds of cores ofer an opportunity to parallelise computations,\u000d\u000a27 signifcantly enhancing compute efciency. The same goes for Graphics processing units\u000d\u000a28 (GPUs), which are designed with many cores acting as individual processing units that can work\u000d\u000a29 on diferent tasks simultaneously. In contrast, CPUs usually have fewer cores optimised for more\u000d\u000a30 general-purpose computing. This means that a GPU can execute multiple simple instructions in\u000d\u000a31 parallel, while a CPU might handle fewer parallel tasks.(Garland et al., 2008) Therefore, GPUs\u000d\u000a32 are suited for calculations such as the Debye scattering equation, where many computations\u000d\u000a33 can be performed simultaneously. Taking advantage of GPU acceleration yields computational\u000d\u000a34 speeds that surpass those of even the most advanced multi-core CPUs; by orders of magnitude.\u000d\u000a35 We introduce a GPU-accelerated open-source Python package, named DebyeCalculator, for\u000d\u000a36 rapid calculation of the Debye scattering equation from xyz-fles. The xyz-format is commonly\u000d\u000aJohansen et al. (2023). A GPU-Accelerated Open-Source Python Package for Rapid Calculation of the Debye Scattering Equation: Applications\u000d\u000ain Small-Angle Scattering, Powder Scattering, and Total Scattering with Pair Distribution Function Analysis. Journal of Open Source Software,\u000d\u000a0(0), \u00bfPAGE? https://doi.org/10.xxxxxx/draft.\u000d\u000a1","DRAFT\u000d\u000a37 used in materials chemistry for the description of discrete particles and simply consists of a list\u000d\u000a38 of atomic identities and their respective Cartesian coordinates (x, y and z). We further calculate\u000d\u000a39 the PDF as described in Underneath the Bragg Peaks.(Egami & Billinge, 2003) We show\u000d\u000a40 that our software can simulate the PD, TS, SAS and PDF data orders of magnitudes faster\u000d\u000a41 than DifPy-CMI.(Juh\u00e1s et al., 2015) DebyeCalculator is an open-source project that is readily\u000d\u000a42 available through GitHub: https://github.com/FrederikLizakJohansen/DebyeCalculator.\u000d\u000a43 The DebyeCalculator, illustrated in the following pseudocode, begins with an initialisation\u000d\u000a44 function that sets user-defned parameters or sets them to default. They include aspects of the\u000d\u000a45 computational environment (such as Q-range, Q-step, r-range, r-step, batch size, and device),\u000d\u000a46 atomic vibrations, radiation type, and instrumental parameters. During this initialisation phase,\u000d\u000a47 the calculation of the atomic form factors is prepared based on the radiation type. Once\u000d\u000a48 initialised, the DebyeCalculator can compute various quantities: the scattering intensity I(Q)\u000d\u000a49 through the Debye scattering equation, the Total Scattering Structure Function S(Q), the\u000d\u000a50 Reduced Total Scattering Function F(Q), and the Reduced Atomic Pair Distribution Function\u000d\u000a51 G(r). In this section, we specifcally illustrate the pseudocode for the G(r) calculation. This is\u000d\u000a52 because the process for calculating G(r) inherently involves the calculations for I(Q), S(Q), and\u000d\u000a53 F(Q). When calling the gr function, the DebyeCalculator loads the structure and computes\u000d\u000a54 the atomic form factors.(Waasmaier & Kirfel, 1995) Following this, it calculates the scattering\u000d\u000a55 intensity I(Q) using the Debye scattering equation and subsequently determines the structure\u000d\u000a56 factor S(Q). The function F(Q) is derived using Q-values and S(Q). Necessary modifcations,\u000d\u000a57 such as dampening and Lorch modifcations, are applied before computing the G(r). The\u000d\u000a58 DebyeCalculator outputs the calculated functions to the CPU by default to allow for immediate\u000d\u000a59 analysis of the results, but users have the fexibility to retain the output on the GPU. It is\u000d\u000a60 worth noting that the majority of the compute time is spent on the double sum calculation\u000d\u000a61 in the Debye scattering equation. This is where a GPU is powerful as it has more cores to\u000d\u000a62 distribute the calculations to than a CPU. The double sum calculation is then multiplied by\u000d\u000a63 the momentum transfers and form factors. This approach is computationally inexpensive but\u000d\u000a64 demands signifcant memory, restricting the ability to calculate larger structures. Therefore, the\u000d\u000a65 DebyeCalculator provides a batching schema that enables users with restricted access to GPU\u000d\u000a66 memory to compute I(Q), S(Q), F(Q) and G(r) for larger structures, trading of only a small\u000d\u000a67 increase in computation time. Users with more substantial GPU memory can accommodate\u000d\u000a68 large structures while maintaining high computation speeds.\u000d\u000aCLASS DebyeCalculator:\u000d\u000aFUNCTION Initialise(parameters...):\u000d\u000a- Set class parameters based on given input or defaults\u000d\u000a- Setup computational environment (e.g., Q-values, r-values)\u000d\u000a- Load atomic formfactor coefficients\u000d\u000a- Setup form factor calculation based on radiation type\u000d\u000aFUNCTION gr(structure_path, keep_on_device=False):\u000d\u000a- Load atomic structure from given structure_path\u000d\u000a- Calculate atomic formfactors\u000d\u000a- Calculate scattering intensity I(Q) (Debye scattering equation)\u000d\u000a- Compute structure factor S(Q) based on I(Q)\u000d\u000a- Calculate F(Q) based on Q-values and S(Q)\u000d\u000a- Apply modifications if necessary (like dampening and Lorch)\u000d\u000a- Calculate pair distribution function G(r) based on F(Q)\u000d\u000a- Return G(r) either on GPU or CPU\u000d\u000a69 In order to benchmark our implementation, we compare simulated scattering patterns from\u000d\u000a70 DebyeCalculator against DifPy-CMI,(Juh\u00e1s et al., 2015) which is a widely recognised software\u000d\u000a71 for scattering pattern computations. Here, our implementation obtains the same scattering\u000d\u000a72 patterns as DifPy-CMI (Supporting Information), while being faster on CPU for structures\u000d\u000a73 up to ~72 \u00c5 (Figure 1). Both calculations are run on a x86-64 CPU with 64GB of memory\u000d\u000aJohansen et al. (2023). A GPU-Accelerated Open-Source Python Package for Rapid Calculation of the Debye Scattering Equation: Applications\u000d\u000ain Small-Angle Scattering, Powder Scattering, and Total Scattering with Pair Distribution Function Analysis. Journal of Open Source Software,\u000d\u000a0(0), \u00bfPAGE? https://doi.org/10.xxxxxx/draft.\u000d\u000a2","DRAFT\u000d\u000a74 and a batch size of 10,000. Running the calculations on the GPU provides another notable\u000d\u000a75 boost in speed (Figure 1). This improvement primarily stems from the distribution of the\u000d\u000a76 double sum calculations across a more extensive set of cores than is feasible on the CPU. With\u000d\u000a77 smaller atomic structures, an overhead associated with initiating GPU calculations results in\u000d\u000a78 the NVIDIA RTX A3000 Laptop GPU computations being slower than DifPy-CMI and our\u000d\u000a79 CPU implementation. Once the atomic structure size exceeds ~14 \u00c5 in diameter, we observe a\u000d\u000a80 ~5 times speed-up using an NVIDIA RTX A3000 Laptop GPU with 6GB of memory and a\u000d\u000a81 batch size of 10,000. The choice of GPU hardware has a substantial infuence on this speed\u000d\u000a82 advantage. As demonstrated in Figure 1, using an NVIDIA Titan RTX GPU, which ofers\u000d\u000a83 24GB of memory, the speed benefts become even more evident. The NVIDIA Titan RTX GPU\u000d\u000a84 delivers a performance that is ~10 times faster, seemingly across all structure sizes, underlining\u000d\u000a85 the signifcant role of the hardware. With the advancements of GPUs like NVIDIA\u2019s Grace\u000d\u000a86 Hopper Superchip,(NVIDIA, 2023) which boasts 576GB of fast-access to memory, there is\u000d\u000a87 potential for DebyeCalculator to achieve even greater speeds in the future.\u000d\u000aFigure 1: Computation-time comparison of the G(r) calculation using our CPU- and GPU-implementation\u000d\u000aagainst DifPy-CMI.(Juh\u00e1s et al., 2015) For the CPU-implementation, a batch size of 10,000 was chosen\u000d\u000a(x86-64 CPU with 6GB). Conversely, the GPU implementations were run with a batch size of 10,000\u000d\u000a(NVIDIA RTX A3000 Laptop GPU with 6GB of memory and NVIDIA Titan RTX GPU with 24GB of\u000d\u000amemory).\u000d\u000a88 Statement of need\u000d\u000a89 Several software packages already exist for simulating the Debye scattering equation, including\u000d\u000a90 DifPy-CMI,(Juh\u00e1s et al., 2015) debyer,(Wojdyr, 2023) Debussy,(Cervellino et al., 2010, 2015)\u000d\u000a91 TOPAS,(Coelho, 2018) and BCL::SAXS.(Putnam et al., 2015) Our software distinguishes\u000d\u000a92 itself in several ways. Firstly, it is freely available and open-source licensed under the Apache\u000d\u000a93 License 2.0. Moreover, it is conveniently implemented as a \u2018pip\u2019 install package and has\u000d\u000a94 been integrated with Google Colab[https://github.com/FrederikLizakJohansen/DebyeCalcula95 torGPU/blob/main/quickstart/QuickStart.ipynb], allowing users to rapidly calculate the Debye\u000d\u000a96 scattering equation without the need of local software installations. DebyeCalculator can be\u000d\u000a97 run on both CPU, GPU and through an interactive interface (see Figure 2), where users can\u000d\u000a98 calculate I(Q), S(Q), F(Q) and G(r) from structural models with one click.\u000d\u000aJohansen et al. (2023). A GPU-Accelerated Open-Source Python Package for Rapid Calculation of the Debye Scattering Equation: Applications\u000d\u000ain Small-Angle Scattering, Powder Scattering, and Total Scattering with Pair Distribution Function Analysis. Journal of Open Source Software,\u000d\u000a0(0), \u00bfPAGE? https://doi.org/10.xxxxxx/draft.\u000d\u000a3","DRAFT\u000d\u000aFigure 2: The interact mode of DebyeCalculator provides a one-click interface, where user can update\u000d\u000aparameters and visualise I(Q), S(Q), F(Q) and G(r). Additionally, the I(Q), S(Q), F(Q), G(r) and xyz-fle\u000d\u000acan be downloaded, including metadata.\u000d\u000aJohansen et al. (2023). A GPU-Accelerated Open-Source Python Package for Rapid Calculation of the Debye Scattering Equation: Applications\u000d\u000ain Small-Angle Scattering, Powder Scattering, and Total Scattering with Pair Distribution Function Analysis. Journal of Open Source Software,\u000d\u000a0(0), \u00bfPAGE? https://doi.org/10.xxxxxx/draft.\u000d\u000a4","DRAFT\u000d\u000a99 Acknowledgements\u000d\u000a100 This work is part of a project that has received funding from the European Research Council\u000d\u000a101 (ERC) under the European Union\u2019s Horizon 2020 Research and Innovation Programme (grant\u000d\u000a102 agreement No. 804066).\u000d\u000a103 Supporting Information\u000d\u000aFigure 3: Comparison of the calculated I(Q), SAXS, F(Q) and G(r) of DebyeCalculator and DifPyCMI(Juh\u00e1s et al., 2015) on a synthetic crystallographic information fle describing a CoO2 antifuorite\u000d\u000acrystal structure.(Friis-Jensen, 2023) Note that the scattering pattern simulated with DifPy-CMI is\u000d\u000ahidden underneath the scattering pattern simulated with DebyeCalculator.\u000d\u000a104 References\u000d\u000a105 Cervellino, A., Frison, R., Bertolotti, F., & Guagliardi, A. (2015). DEBUSSY 2.0: The new\u000d\u000a106 release of a debye user system for nanocrystalline and/or disordered materials. J. Appl.\u000d\u000a107 Crystallogr., 48(6), 2026\u20132032.\u000d\u000a108 Cervellino, A., Giannini, C., & Guagliardi, A. (2010). DEBUSSY: A debye user system for\u000d\u000a109 nanocrystalline materials. J. Appl. Crystallogr., 43(6), 1543\u20131547.\u000d\u000a110 Coelho, A. A. (2018). TOPAS and TOPAS-academic: An optimization program integrating\u000d\u000a111 computer algebra and crystallographic objects written in c++. J. Appl. Crystallogr., 51(1),\u000d\u000a112 210\u2013218.\u000d\u000a113 Debye, P. (1915). Zerstreuung von r\u00f6ntgenstrahlen. Annalen Der Physik, 351(6), 809\u2013823.\u000d\u000a114 Egami, T., & Billinge, S. J. (2003). Underneath the bragg peaks: Structural analysis of\u000d\u000a115 complex materials. Elsevier.\u000d\u000a116 Friis-Jensen, U. (2023). In GitHub. https://github.com/FrederikLizakJohansen/\u000d\u000a117 DebyeCalculatorGPU/blob/main/example_data/AntiFluorite_Co2O.cif\u000d\u000aJohansen et al. (2023). A GPU-Accelerated Open-Source Python Package for Rapid Calculation of the Debye Scattering Equation: Applications\u000d\u000ain Small-Angle Scattering, Powder Scattering, and Total Scattering with Pair Distribution Function Analysis. Journal of Open Source Software,\u000d\u000a0(0), \u00bfPAGE? https://doi.org/10.xxxxxx/draft.\u000d\u000a5","DRAFT\u000d\u000a118 Garland, M., Le Grand, S., Nickolls, J., Anderson, J., Hardwick, J., Morton, S., Phillips, E.,\u000d\u000a119 Zhang, Y., & Volkov, V. (2008). Parallel computing experiences with CUDA. IEEE Micro,\u000d\u000a120 28(4), 13\u201327.\u000d\u000a121 Juh\u00e1s, P., Farrow, C., Yang, X., Knox, K., & Billinge, S. (2015). Complex modeling: A\u000d\u000a122 strategy and software program for combining multiple information sources to solve ill posed\u000d\u000a123 structure and nanostructure inverse problems. Acta Crystallogr. A, 71(6), 562\u2013568.\u000d\u000a124 NVIDIA. (2023). In NVIDIA. https://resources.nvidia.com/en-us-grace-cpu/grace-hopper-superchip\u000d\u000a125 Putnam, D. K., Weiner, B. E., Woetzel, N., Lowe Jr, E. W., & Meiler, J. (2015). BCL:: SAXS:\u000d\u000a126 GPU accelerated d ebye method for computation of small angle x-ray scattering profles.\u000d\u000a127 Proteins: Struct., Funct., Genet., 83(8), 1500\u20131512.\u000d\u000a128 Scardi, P., Billinge, S. J., Neder, R., & Cervellino, A. (2016). Celebrating 100 years of the debye\u000d\u000a129 scattering equation. In Acta Crystallogr. A (No. 6; Vol. 72, pp. 589\u2013590). International\u000d\u000a130 Union of Crystallography.\u000d\u000a131 Schaller, R. R. (1997). Moore\u2019s law: Past, present and future. IEEE Spectrum, 34(6), 52\u201359.\u000d\u000a132 Waasmaier, D., & Kirfel, A. (1995). New analytical scattering-factor functions for free atoms\u000d\u000a133 and ions. Acta Crystallographica Section A, 51(3), 416\u2013431. https://doi.org/10.1107/\u000d\u000a134 S0108767394013292\u000d\u000a135 Wojdyr. (2023). Wojdyr/debyer: Debye\u2019s scattering equation &amp; other analysis of atomistic\u000d\u000a136 models. In GitHub. https://github.com/wojdyr/debyer\u000d\u000aJohansen et al. (2023). A GPU-Accelerated Open-Source Python Package for Rapid Calculation of the Debye Scattering Equation: Applications\u000d\u000ain Small-Angle Scattering, Powder Scattering, and Total Scattering with Pair Distribution Function Analysis. Journal of Open Source Software,\u000d\u000a0(0), \u00bfPAGE? https://doi.org/10.xxxxxx/draft.\u000d\u000a6"," \u000d\u000a10\u000d\u000a1.2 Total\u0009Scattering\u0009with\u0009Pair\u0009Distribution\u0009Function\u0009Analysis\u000d\u000aData\u0009from\u0009scattering\u0009experiments\u0009such\u0009as\u0009Single\u0009Crystal\u0009Diffraction\u0009(SCD)\u0009and\u0009Powder\u0009Diffraction\u0009(PD)\u000d\u000ais\u0009 typically\u0009 both\u0009 measured\u0009 and\u0009 modelled\u0009 in\u0009 reciprocal-space.\u0009 However,\u0009 scattering\u0009 data\u0009 can\u0009 also\u0009 be\u0009\u000d\u000arepresented\u0009and\u0009modelled\u0009in\u0009 real-space using\u0009a\u0009Fourier\u0009 transformation.\u0009Fourier\u0009 transformations\u0009are\u0009\u000d\u000acommonly used\u0009to\u0009convert\u0009dataset\u0009variables\u0009between\u0009different\u0009domains\u0009such\u0009as\u0009time\u0009and\u0009frequency,\u0009or\u0009\u000d\u000aas\u0009demonstrated in\u0009the\u0009previous\u0009section,\u0009between\u0009electron\u0009cloud\u0009distribution\u0009and an\u0009atomic\u0009form factor.\u0009\u000d\u000aEssentially,\u0009a Fourier\u0009transform\u0009is\u0009a\u0009mathematical\u0009tool\u0009that\u0009allows\u0009to\u0009\u2018change\u0009the\u0009space\u2019\u0009of\u0009the\u0009data.\u0009Figure\u000d\u000a1.2A\u0009illustrates\u0009ideal\u0009simulated\u0009scattering\u0009data\u0009of\u0009a\u0009[Bi38O45]\u0009cluster\u0009in\u0009reciprocal-space\u0009and\u0009the\u0009Fourier\u0009\u000d\u000atransform\u0009of\u0009the\u0009data.\u0009Simulation\u0009details\u0009are\u0009given\u0009in\u0009Appendix\u0009I.\u000d\u000aAlthough\u0009the\u0009reciprocal-space\u0009and\u0009the\u0009real-space\u0009data\u0009contain\u0009the\u0009same\u0009information,\u0009the\u0009representation\u000d\u000adiffers\u0009and\u0009highlights different structural information.\u0009High-intensity peaks\u0009in\u0009the\u0009scattering\u0009pattern\u0009in\u0009\u000d\u000areciprocal-space,\u0009known\u0009as\u0009Bragg\u0009peaks,\u0009arise\u0009from\u0009a\u0009periodic\u0009arrangement\u0009of\u0009atoms\u0009in\u0009the\u0009material.\u0009The\u0009\u000d\u000aabsence\u0009of\u0009Bragg\u0009peaks\u0009and\u0009the\u0009presence\u0009of\u0009broad\u0009features\u0009in\u0009Figure 1.2A\u0009indicate that\u0009there\u0009is\u0009a\u0009limited\u000d\u000aperiodic\u0009arrangement\u0009of\u0009atoms\u0009in\u0009the\u0009[Bi38O45]\u0009cluster.\u0009The\u0009reciprocal-space\u0009and\u0009real-space\u0009are\u0009inversely\u0009\u000d\u000arelated,\u0009meaning\u0009that\u0009intensity\u0009in\u0009the\u0009low\u0009Q-regime\u0009originates\u0009from larger\u0009distances\u0009in\u0009real-space.\u0009Here,\u0009\u000d\u000athe\u0009first\u0009peak\u0009in the\u0009scattering\u0009pattern\u0009originates\u0009from\u0009the\u0009entire\u0009nanocluster and\u0009provides information\u0009\u000d\u000aabout\u0009the\u0009shape\u0009and\u0009size\u0009of\u0009the\u0009[Bi38O45]\u0009cluster.\u000d\u000aIn\u0009real-space,\u0009it\u0009is\u0009easier\u0009to\u0009extract\u0009information\u0009about\u0009the\u0009local\u0009order\u0009of\u0009the\u0009material.\u0009For\u0009example,\u0009one\u0009\u000d\u000acan\u0009easily\u0009identify\u0009peaks\u0009at\u0009r\u0009=\u00093.8\u0009\u00c5,\u00096.6\u0009\u00c5,\u00098.7\u0009\u00c5\u0009and\u000910.2\u0009\u00c5\u0009in\u0009Figure 1.2A,\u0009which\u0009correspond\u0009to\u0009common\u000d\u000adistances\u0009in\u0009the\u0009material.\u0009Based\u0009on\u0009the\u0009intensity,\u0009G(r), of\u0009these\u0009peaks,\u0009one\u0009can\u0009index\u0009the\u0009scattering\u0009atomatom\u0009pair.\u0009High-intensity\u0009peaks\u0009come from\u0009Bi\u00b7\u00b7\u00b7Bi\u0009distances\u0009since\u0009bismuth\u0009('8!\u000d\u000a97:/;E# = 0 \u00c57%F = 83)\u0009\u000d\u000ascatters\u0009 significantly\u0009more\u0009 than\u0009 oxygen\u0009 ('<\u000d\u000a97:/;E# = 0 \u00c57%F = 8).\u0009Medium-intensity peaks\u0009 are\u0009 from\u0009\u000d\u000aBi\u00b7\u00b7\u00b7O\u0009distances\u0009and\u0009low-intensity peaks\u0009are\u0009from\u0009O\u00b7\u00b7\u00b7O\u0009distances in\u0009the\u0009material.\u0009Furthermore,\u0009we can\u0009\u000d\u000aestimate the\u0009material size\u0009by\u0009identifying\u0009the\u0009peak\u0009with\u0009the\u0009highest\u0009r-value\u0009(about\u0009r\u0009=\u000912.2\u0009\u00c5).\u0009Reciprocalspace\u0009 and\u0009 real-space\u0009 are\u0009 thereby\u0009 complementary\u0009 data\u0009 representations\u0009 that\u0009 highlight\u0009 different\u0009\u000d\u000ainformation\u0009about\u0009the\u0009structure\u0009of\u0009a\u0009material.\u0009"," \u000d\u000a11\u000d\u000aFigure\u00091.2:\u0009A)\u0009Simulated\u0009and\u0009B)\u0009experimental\u0009TS\u0009data\u0009of\u0009the\u0009[Bi38O45]\u0009cluster\u0009in reciprocal-space\u0009and\u0009the\u0009data\u0009after\u0009\u000d\u000aFourier\u0009transformation\u0009to\u0009real-space\u0009(the\u0009PDF).\u0009Figure\u0009and\u0009Figure\u0009caption\u0009are\u0009adapted\u0009from\u0009my\u0009Master\u2019s\u0009Thesis.1\u000d\u000aHowever, obtaining useful\u0009real-space\u0009data\u0009from\u0009an\u0009experimental reciprocal-space\u0009scattering\u0009pattern\u0009can\u0009\u000d\u000abe challenging.\u0009Figure 1.2B\u0009presents an\u0009example\u0009of experimental reciprocal-space\u0009scattering\u0009data\u0009for the\u0009\u000d\u000a[Bi38O45]\u0009 cluster\u0009 and\u0009 the\u0009 corresponding\u0009 real-space\u0009 data.\u0009 Details\u0009 about\u0009 the\u0009 experiment\u0009 and\u0009 the\u000d\u000atransformation to\u0009real-space\u0009can\u0009be\u0009found in\u0009reference24 and\u0009Appendix\u0009I.\u0009\u000d\u000aThe\u0009 simulated reciprocal-space\u0009 data\u0009 of\u0009 the\u0009 [Bi38O45]\u0009 cluster\u0009 resembles the\u0009 experimental data,\u0009 but\u000d\u000awithout the background\u0009signal\u0009from\u0009incoherent\u0009scattering.\u0009As the\u0009structural\u0009information\u0009resides in\u0009the\u0009\u000d\u000acoherent\u0009 scattering,\u0009 we\u0009 separate\u0009 the\u0009 coherent\u0009 from\u0009 the\u0009 incoherent\u0009 scattering by\u0009 subtraction\u0009 of\u0009 the\u0009\u000d\u000asample\u0009environment\u0009and\u0009correction\u0009of\u0009incoherent\u0009effects.\u0009In\u0009these studies,\u0009PDFgetX325 has\u0009been\u0009used\u0009to\u0009\u000d\u000agenerate\u0009the\u0009PDFs,\u0009which\u0009uses\u0009an\u0009ad\u0009hoc approach to\u0009determine the\u0009coherent\u0009intensity,\u0009!=>?.\u0009PDFgetX3\u0009\u000d\u000aexploits\u0009that\u0009the\u0009Q-frequencies\u0009of\u0009the\u0009coherent\u0009scattering\u0009are\u0009larger\u0009than\u0009~ 1@\u000d\u000a:!\"\u000d\u000a,\u0009where\u0009,,A is\u0009the\u0009nearestneighbour\u0009distance,\u0009and that\u0009Q-frequencies\u0009of\u0009incoherent\u0009scattering effects such\u0009as\u0009Compton\u0009scattering\u0009\u000d\u000aand\u0009 multiple\u0009 scattering\u0009 have\u0009 a\u0009 lower\u0009 Q-frequency.\u0009 By\u0009 fitting\u0009 an\u0009 arbitrary\u0009 function\u0009 to\u0009 the\u0009 measured\u0009\u000d\u000aintensity\u0009with\u0009no\u0009 frequencies\u0009higher\u0009 than\u0009 1@\u000d\u000a:!\"\u000d\u000a,\u0009the\u0009incoherent\u0009scattering\u0009is\u0009determined\u0009and\u0009separated\u0009\u000d\u000afrom\u0009the\u0009coherent\u0009scattering.\u0009\u0009\u000d\u000aWe\u0009 introduce\u0009 the\u0009 Total\u0009 Scattering\u0009 Structure\u0009 Function,\u0009 S(Q),\u0009 which\u0009 focuses\u0009 solely\u0009 on\u0009 the\u0009 coherent\u0009\u000d\u000ascattering:20\u000d\u000aEquation\u00091.7\u0009 J(#) = !=>?(#) + \u2329'(#)\u232a1 \u2212 \u2329'(#)1\u232a\u000d\u000aO\u2329'(#)\u232a1\u000d\u000aS(Q)\u0009 normalises\u0009 the\u0009 coherent\u0009 elastic\u0009 scattering\u0009 intensity,\u0009 !=>?,\u0009 using the\u0009 average\u0009 scattering\u0009 power,\u0009\u000d\u000a\u2329'(#)\u232a.\u0009For\u0009X-ray\u0009scattering\u0009data\u0009this\u0009results\u0009in\u0009an\u0009amplified\u0009signal\u0009in\u0009high\u0009# which\u0009contains\u0009the\u0009local\u0009\u000d\u000astructural\u0009information.\u0009The\u0009scattering\u0009intensity\u0009in\u0009the\u0009high\u0009# regime\u0009is\u0009low\u0009due\u0009to\u0009the\u0009Q-dependency\u0009of\u0009\u000d\u000athe\u0009 atomic\u0009 scattering\u0009 factors.\u0009 The\u0009 \u2329'(#)\u232a1 \u2212 \u2329'(#)1\u232a term\u0009 arises\u0009 from\u0009 an\u0009 imperfect\u0009 cancellation\u0009 in\u0009\u000d\u000adestructive\u0009and\u0009positive\u0009interference\u0009between\u0009scattering\u0009by\u0009different\u0009elements.20\u0009\u0009","|\u0009Scattering\u0009for\u0009Nanostructure\u0009Characterisation\u000d\u000a12\u000d\u000aThe\u0009next\u0009mathematical\u0009treatment\u0009of\u0009the\u0009data\u0009is\u0009to calculate\u0009the\u0009Reduced\u0009Total\u0009Scattering\u0009Function,\u0009P(#),\u0009\u000d\u000awhere\u0009the\u0009high\u0009# regime\u0009is\u0009further\u0009enhanced:20\u000d\u000aEquation\u00091.8\u0009 P(#) = #(J(#) \u2212 1)\u000d\u000aWe\u0009use\u0009the\u0009Fourier\u0009transform\u0009on\u0009P(#) to obtain\u0009the\u0009Reduced\u0009Atomic\u0009Pair\u0009Distribution\u0009Function,\u0009R(,),\u0009\u000d\u000aalso\u0009referred\u0009to\u0009as\u0009the\u0009PDF:20\u000d\u000aEquation\u00091.9\u0009 R(,) = 2\u000d\u000a/ S P(#) sin(#,) @#\u000d\u000aB\u000d\u000aC\u000d\u000aHowever,\u0009we\u0009face another\u0009issue\u0009in real\u0009experiments,\u0009as\u0009they\u0009do not\u0009contain\u0009data\u0009from\u0009# = 0 \u00c57% to\u0009# =\u000d\u000a\u221e \u00c57%,\u0009 but\u0009 rather\u0009 a\u0009 limited\u0009#:/,DE from\u0009#A!, to\u0009#A/F.\u0009 Consequently,\u0009the PDF\u0009 obtained\u0009 from\u0009 actual\u0009\u000d\u000aexperiments includes\u0009mathematical artefacts\u0009that\u0009arise from\u0009the limited\u0009#:/,DE:20\u000d\u000aEquation\u00091.10 R(,) = 2\u000d\u000a/ S P(#) sin(#,) @#\u000d\u000aG\"#$\u000d\u000aG\"%!\u000d\u000aThe\u0009 limited\u0009#:/,DE leads\u0009 to\u0009 a\u0009 mathematical\u0009 artefact\u0009 known\u0009 as\u0009 \u2018termination\u0009 ripples\u2019,\u0009 as illustrated\u0009 in\u0009\u000d\u000aFigure 1.2B.\u0009To reduce the\u0009impact of\u0009termination\u0009ripples,\u0009we\u0009perform\u0009TS\u0009experiments\u0009with\u0009the\u0009largest\u0009\u000d\u000apossible\u0009#:/,DE.\u0009 We\u0009 achieve this\u0009 by\u0009 using the\u0009 rapid-acquisition\u0009 PDF\u0009 (RA\u2013PDF)26 setup\u0009 where\u0009 a\u0009 2D\u0009\u000d\u000adetector\u0009is\u0009positioned as\u0009close\u0009 to\u0009 the\u0009sample\u0009as\u0009possible.\u0009Combining RA\u2013PDF\u0009with\u0009a\u0009low\u0009wavelength\u000d\u000aallows\u0009us\u0009to obtain\u0009high\u0009Q-values\u0009(Equation\u00091.2), often up\u0009to\u000920\u0009\u2013\u000930\u0009\u00c5-1.\u0009While\u0009this could potentially\u0009be\u0009\u000d\u000aachieved using a\u0009 point\u0009 detector,\u0009 a\u0009 2D\u0009 detector\u0009 offers\u0009 the\u0009 advantage\u0009 of\u0009 higher\u0009 intensity,\u0009 enabling\u000d\u000ameasurements\u0009to\u0009be\u0009performed\u0009in seconds.\u0009\u000d\u000aAlthough a\u0009 short sample-to-detector\u0009 distance\u0009provides a\u0009larger\u0009#:/,DE,\u0009it\u0009 sacrifices\u0009 some\u0009 reciprocalspace\u0009resolution. In\u0009fact,\u0009the\u0009data\u0009in\u0009Figure\u00091.2B\u0009lacks data points\u0009below\u0009Q\u0009=\u00090.19\u0009\u00c5-1 and\u0009shows visible\u0009\u000d\u000apoor\u0009reciprocal-space\u0009resolution\u0009in\u0009the\u0009low\u0009Q-regime.\u0009This limited\u0009#A!, leads\u0009to a\u0009mathematical\u0009artefact\u0009\u000d\u000ain\u0009the\u0009PDF,\u0009illustrated\u0009with\u0009a dotted\u0009line\u0009(baseline)\u0009in\u0009Figure\u00091.2B.\u0009Furthermore,\u0009it\u0009makes it\u0009challenging\u0009\u000d\u000ato extract\u0009information\u0009about\u0009the\u0009shape\u0009and\u0009size\u0009distribution\u0009of\u0009the\u0009nanoparticles\u0009from\u0009the\u0009PDF,\u0009as\u0009this\u0009\u000d\u000ainformation\u0009is\u0009found\u0009in the\u0009low\u0009Q-regime.\u0009\u000d\u000a1.3 Combined\u0009 Modelling\u0009 of Pair\u0009 Distribution\u0009 Function and\u0009 Small-Angle\u0009 X-ray\u0009\u000d\u000aScattering\u0009Data\u000d\u000aWhile\u0009X-ray\u0009TS\u0009 data\u0009with\u0009 PDF\u0009analysis\u0009is\u0009a\u0009 powerful\u0009approach to\u0009 characterise\u0009 the\u0009 structure\u0009 of\u0009most\u0009\u000d\u000amaterials,\u0009 the\u0009 information\u0009 in\u0009 a\u0009 TS\u0009 experiment\u0009 is\u0009 not\u0009 always\u0009 sufficient\u0009 to\u0009 characterise\u0009 the\u0009 structure.\u0009\u000d\u000aSometimes,\u0009 we\u0009 need\u0009 data\u0009 from\u0009 multiple\u0009 experiments\u0009 to\u0009 determine\u0009 the\u0009 atomic\u0009 structure.\u0009 Figure\u0009 1.3\u000d\u000ademonstrates\u0009such\u0009an\u0009example by\u0009comparing simulated\u0009X-ray\u0009TS,\u0009PDF,\u0009and\u0009SAXS\u0009data\u0009of\u0009the\u0009[Bi18O36]\u0009\u000d\u000acluster\u0009and\u0009 the\u0009 [Bi22O38]\u0009cluster.\u0009SAXS\u0009is\u0009 the\u0009measurement\u0009 of\u0009 the\u0009 scattering\u0009intensities\u0009at\u0009 the\u0009low\u0009Qregime\u0009often\u0009performed\u0009by\u0009having\u0009a\u0009long\u0009sample\u0009to\u0009detector\u0009distance. The\u0009simulation\u0009details\u0009are\u0009given\u0009\u000d\u000ain\u0009Appendix\u0009I.\u000d\u000aThe\u0009two\u0009clusters\u0009have\u0009no long-range\u0009order,\u0009and\u0009they\u0009possess a\u0009very\u0009similar\u0009local\u0009structure\u0009built\u0009up\u0009by\u0009\u000d\u000a[Bi6O8]\u0009octahedra,\u0009why\u0009it\u0009is\u0009difficult\u0009to\u0009distinguish\u0009them\u0009in\u0009both\u0009the\u0009TS data\u0009and\u0009the\u0009PDF\u0009data, Figure\u0009\u000d\u000a1.3A\u2013B.\u0009The\u0009most\u0009prominent\u0009difference\u0009in the\u0009scattering\u0009from\u0009the\u0009two\u0009clusters\u0009in Figure\u00091.3A\u2013B\u0009is\u0009the\u0009\u000d\u000ascattering\u0009in\u0009the\u0009peak\u0009around Q\u0009=\u00090.45\u0009\u00c5-1,\u0009which\u0009is\u0009also\u0009part\u0009of\u0009the\u0009signal\u0009giving rise\u0009to\u0009the\u0009SAXS\u0009pattern\u0009\u000d\u000aillustrated\u0009 in\u0009 Figure\u0009 1.3C.\u0009 The\u0009 two\u0009 clusters\u0009 have\u0009 distinct\u0009 shapes,\u0009 which\u0009 makes\u0009 them\u0009 easily\u0009\u000d\u000adistinguishable\u0009 in\u0009 the\u0009 SAXS\u0009 data.\u0009 However,\u0009 as\u0009 we\u0009 demonstrated\u0009 in\u0009 section\u0009 1.2,\u0009 a\u0009 TS\u0009 experiment\u0009 is\u0009"," \u000d\u000a13\u000d\u000atypically\u0009instrumentally\u0009challenged\u0009in\u0009this\u0009part\u0009of\u0009the\u0009Q-regime.\u0009Therefore,\u0009TS\u0009and\u0009SAXS\u0009measurements\u0009\u000d\u000aare\u0009typically performed in\u0009separate\u0009experiments\u0009or\u0009at\u0009separate\u0009detectors.\u0009\u000d\u000aFigure\u00091.3:\u0009A)\u0009Comparison\u0009of\u0009the\u0009simulated\u0009TS,\u0009PDF\u0009and\u0009SAXS\u0009data\u0009of\u0009B)\u0009the\u0009[Bi18O36]\u0009cluster\u0009and\u0009C)\u0009the\u0009[Bi22O38]\u0009\u000d\u000acluster.27,28 Figure\u0009and\u0009Figure\u0009caption\u0009are\u0009adapted\u0009from\u0009my\u0009Master\u2019s\u0009Thesis.\u000d\u000a1\u000d\u000aOppositely the\u0009SAXS\u0009data\u0009does\u0009not describe\u0009the\u0009local\u0009order\u0009of\u0009the\u0009structure,\u0009which\u0009can\u0009be\u0009described from\u000d\u000aPDF\u0009 data.\u0009 A\u0009 combination\u0009 of\u0009 PDF\u0009 and\u0009 SAXS\u0009 data\u0009 is\u0009 therefore\u0009 ideal\u0009 to\u0009 fully\u0009 describe\u0009 both\u0009 the\u0009 atomic\u0009\u000d\u000astructure\u0009and\u0009the\u0009morphology\u0009of\u0009the\u0009clusters.\u0009Even\u0009when\u0009combining\u0009the\u0009X-ray\u0009PDF\u0009and\u0009SAXS,\u0009it\u0009can\u0009be\u0009\u000d\u000adifficult\u0009 to\u0009 describe\u0009 the\u0009 atomic\u0009 positions\u0009 and\u0009 vibrations\u0009 of\u0009 the\u0009 oxygen\u0009 atoms\u0009 since\u0009 oxygen\u0009\u000d\u000a('<\u000d\u000a97:/;E# = 0 \u00c57%F = 8) scatter\u0009significantly\u0009less\u0009than\u0009bismuth\u0009('8!\u000d\u000a97:/;E# = 0 \u00c57%F = 83).\u0009Instead,\u0009\u000d\u000aone\u0009could\u0009perform\u0009a\u0009neutron\u0009scattering\u0009experiment,\u0009where\u0009 the\u0009bound\u0009coherent\u0009scattering\u0009length\u0009are\u0009\u000d\u000acomparable\u0009for\u0009oxygen\u0009and\u0009bismuth\u0009('<\u000d\u000a#EH-:>,E# = 0 \u00c57%F = 5.803 fm) and\u0009('8!\u000d\u000a#EH-:>,E# = 0 \u00c57%F =\u000d\u000a8.532 fm).\u000d\u000a19 Conclusively,\u0009for\u0009a\u0009comprehensive\u0009description\u0009of\u0009the\u0009atomic\u0009structure\u0009of\u0009these\u0009clusters,\u0009one\u0009\u000d\u000awould\u0009need\u0009a\u0009range\u0009of\u0009structure\u0009characterisation\u0009techniques.\u0009For\u0009this\u0009chemical\u0009system\u0009that\u0009could\u0009be\u0009both\u0009\u000d\u000aTS\u0009with\u0009PDF\u0009and\u0009small-angle\u0009scattering measured\u0009with both\u0009X-rays\u0009and\u0009neutrons.\u0009",""," \u000d\u000a15\u000d\u000a2 Conventional\u0009Scattering\u0009Data\u0009Modelling*\u000d\u000aIn\u0009recent\u0009decades,\u0009materials\u0009chemistry research\u0009has\u0009witnessed\u0009remarkable\u0009progress\u0009owing\u0009to\u0009the\u0009rapid\u0009\u000d\u000adevelopment\u0009 of\u0009 synchrotron\u0009 and\u0009 neutron\u0009 sources,29,30 and\u0009 the\u0009 advancement\u0009 of laboratory\u0009\u000d\u000ainstruments.31-33 Terabytes\u0009of\u0009data\u0009are\u0009commonly\u0009acquired\u0009from\u0009in\u0009situ experiments\u0009to\u0009follow\u0009material\u0009\u000d\u000aformation\u0009 reactions\u0009 or\u0009 operando experiments\u0009 to\u0009 study\u0009 the\u0009 atomic\u0009 arrangement\u0009 of\u0009 batteries\u0009 during\u0009\u000d\u000acharging\u0009and\u0009recharging.34-36 Conventional\u0009scattering\u0009data\u0009modelling\u0009approaches,\u0009such\u0009as\u0009least-squares\u0009\u000d\u000afitting\u0009algorithms,\u0009excel\u0009at\u0009minimizing\u0009discrepancies\u0009(Often\u0009using\u0009the\u0009Rwp value\u0009that\u0009is\u0009further\u0009explained\u0009\u000d\u000ain\u0009Appendix\u0009II)\u0009between\u0009experimental\u0009and\u0009simulated\u0009data\u0009derived\u0009from\u0009a\u0009structural\u0009model,\u0009for\u0009example,\u0009\u000d\u000ausing\u0009 the\u0009 Debye\u0009 scattering\u0009 equation.\u0009 This\u0009 provides invaluable\u0009 structural\u0009 information\u0009 like\u0009 atomic\u0009\u000d\u000avibrations,\u0009 particle\u0009 shape,\u0009 and\u0009 particle\u0009 size\u0009 (Figure\u0009 2.1).\u0009 Nevertheless,\u0009 these\u0009 algorithms\u0009 rely\u0009 on a\u0009\u000d\u000astructural\u0009model\u0009as\u0009input,\u0009which\u0009can\u0009require\u0009time-consuming\u0009literature\u0009reviews\u0009to\u0009identify,\u0009followed\u0009by\u0009\u000d\u000alocating\u0009the\u0009structural\u0009model\u0009in\u0009databases\u0009such\u0009as\u0009the\u0009Inorganic\u0009Crystal\u0009Structure\u0009Database\u0009(ICSD)\u0009or\u0009\u000d\u000athe\u0009Crystallography\u0009Open\u0009Database\u0009(COD).37,38 Once\u0009the\u0009structural\u0009model\u0009is\u0009found,\u0009it must\u0009be\u0009fitted\u0009to\u0009\u000d\u000athe\u0009dataset\u0009using\u0009specialised software.\u0009This\u0009process,\u0009known\u0009as\u0009structure\u0009refinement, can\u0009be\u0009complicated\u000d\u000aand\u0009often\u0009demands\u0009expertise\u0009in\u0009the\u0009field. Similar challenges\u0009are\u0009faced\u0009in\u0009the\u0009modelling\u0009of\u0009some\u0009types\u0009of\u0009\u000d\u000aspectroscopy\u0009data\u0009like\u0009inelastic\u0009neutron\u0009scattering\u0009(INS)\u0009data\u0009and\u0009X-ray\u0009absorption\u0009spectroscopy\u0009(XAS)\u0009\u000d\u000adata,\u0009as\u0009shown\u0009in\u0009Figure\u00092.1.\u0009\u0009\u000d\u000aIn\u0009 my\u0009 PhD,\u0009 I\u0009 have\u0009 contributed,\u0009 as\u0009 a\u0009 co-author,\u0009to multiple studies\u0009that\u0009 use conventional\u0009 modelling\u0009\u000d\u000astrategies for PDF,\u0009SAXS,\u0009and\u0009PD data but\u0009are\u0009not\u0009part\u0009of\u0009this\u0009PhD\u0009dissertation.\u0009These\u0009approaches have\u0009\u000d\u000abeen\u0009employed to\u0009characterise local\u0009structural\u0009motifs\u0009in\u0009amorphous\u0009materials,8 metallic\u0009nanoclusters\u000d\u000aand\u0009 nanoparticles,\u000d\u000a39-41 metal\u0009 oxide\u0009 nanoparticles,42 carbon-based nanoparticles,41,43\u0009 to\u0009 understand\u000d\u000aformation\u0009and\u0009growth mechanisms\u0009for\u0009metal\u0009oxide nanoparticles,\u000d\u000a44 and\u0009growth\u0009mechanisms\u0009of\u0009ionic\u0009\u000d\u000ametal\u0009oxido\u0009nanoclusters\u0009in\u0009solution.24,45\u000d\u000aFigure\u00092.1:\u0009Conventional modelling\u0009workflow\u0009for\u0009scattering\u0009(powder\u0009diffraction:\u0009PD,\u0009pair\u0009distribution\u0009function:\u0009PDF\u0009\u000d\u000aor\u0009 small-angle\u0009 scattering:\u0009 SAS) and\u0009 spectroscopy\u0009 (inelastic\u0009 neutron\u0009 scattering:\u0009 INS\u0009 or\u0009 X-ray\u0009 absorption\u0009\u000d\u000aspectroscopy:\u0009XAS) typically\u0009involve\u0009an\u0009expert\u0009scientist\u0009manually\u0009building a\u0009structural\u0009model.\u0009This\u0009model\u0009is\u0009then\u0009\u000d\u000arefined\u0009using\u0009specialised software\u0009 to\u0009extract\u0009structural\u0009information\u0009such\u0009as\u0009size,\u0009shape,\u0009and\u0009vibrations\u0009 from\u0009 the\u0009\u000d\u000adataset.\u0009This\u0009procedure is\u0009repeated\u0009for\u0009each\u0009new\u0009dataset\u0009measured. Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u000d\u000aPaper\u0009II.\u000d\u000a46\u000d\u000a2.1.1 Paper\u0009II\u0009\u000d\u000aOn\u0009 the\u0009 following\u0009 pages,\u0009 you will find\u0009 Paper\u0009 II\u0009 which\u0009 is\u0009 uploaded\u0009to\u0009 ChemRxiv. The\u0009 supplementary\u000d\u000ainformation\u0009for\u0009Paper\u0009II\u0009can\u0009be\u0009found\u0009in APPENDICES\u0009\u2013\u0009PAPERS.\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009II.","Machine Learning for Analysis of Experimental Scattering and Spectroscopy Data in \u000d\u000aMaterials Chemistry \u000d\u000aAndy S. Anker1\u000d\u000a, Keith T. Butler2\u000d\u000a, Raghavendra Selvan3,4 and Kirsten M. \u00d8. Jensen1*\u000d\u000a*Correspondence to kirsten@chem.ku.dk (KM\u00d8J) \u000d\u000a1: Department of Chemistry and Nano-Science Center, University of Copenhagen, 2100 \u000d\u000aCopenhagen \u00d8, Denmark \u000d\u000a2: Department of Chemistry, 20 Gordon St, London WC1H 0AJ, UK \u000d\u000a3: Department of Computer Science, University of Copenhagen, 2100 Copenhagen \u00d8, Denmark \u000d\u000a4: Department of Neuroscience, University of Copenhagen, 2200 Copenhagen N, Denmark \u000d\u000aAbstract \u000d\u000aThe rapid growth of materials chemistry data, driven by advancements in large-scale radiation \u000d\u000afacilities as well as laboratory instruments, has outpaced conventional data analysis and modelling \u000d\u000amethods, which can require enormous manual effort. To address this bottleneck, we investigate the \u000d\u000aapplication of supervised and unsupervised machine learning (ML) techniques for scattering and \u000d\u000aspectroscopy data analysis in materials chemistry research. Our perspective focuses on ML \u000d\u000aapplications in powder diffraction (PD), pair distribution function (PDF), small-angle scattering \u000d\u000a(SAS), inelastic neutron scattering (INS), and X-ray absorption spectroscopy (XAS) data, but the \u000d\u000alessons that we learn are generally applicable across materials chemistry. We review the ability of \u000d\u000aML to efficiently and accurately identify physical and structural models and extract information from \u000d\u000aexperimental data. Furthermore, we discuss the challenges associated with supervised ML and \u000d\u000ahighlight how unsupervised ML can mitigate these limitations, thus enhancing experimental materials \u000d\u000achemistry data analysis. Our perspective emphasises the transformative potential of ML in materials \u000d\u000achemistry characterisation and identifies promising directions for future applications. The perspective \u000d\u000aaims to guide newcomers to ML-based experimental data analysis, alerting them to the potential \u000d\u000apitfalls and offering guidance for success. ","Introduction \u000d\u000aDuring recent decades, materials science research has been accelerated by the rapid development of \u000d\u000alarge-scale radiation facilities1, 2 and the advancement of laboratory instruments.3-5 It is now common \u000d\u000ato acquire large amounts of data from e.g. in situ or operando experiments.6-8 The combination of \u000d\u000ascattering and spectroscopy with computed tomography, allowing detailed position-resolved studies \u000d\u000aof e.g. batteries and catalysts also results in larger and larger amounts of data with the continued \u000d\u000adevelopment of synchrotron and neutron sources.9-12 As illustrated in Figure 1, analysis of scatteringand spectroscopy data can provide invaluable structural information on e.g., lattice parameters, \u000d\u000aoxidation states, atomic positions, and crystallite or particle size and shape. In conventional data \u000d\u000amodelling approaches, data are often analysed using minimisation techniques such as least-squares \u000d\u000afitting algorithms, where the difference between experimental data and simulated data is minimised\u000d\u000aby refining parameters in a physical model, e.g., representing the atomic structure. This process is \u000d\u000aknown as structure refinement. Identification of the structure model to use in structure refinement can \u000d\u000abe a limitation in data analysis: to identify the structural model, extensive database and literature \u000d\u000asearches are often needed. Automated screening of large numbers of structure models can be \u000d\u000acombined with structure refinement methods, and have been used for e.g., identifying a cluster13 or \u000d\u000acrystal structure14 from, e.g., PDF data. However, least-squares fitting algorithms are computationally \u000d\u000aexpensive, which limits their use for structure model identification. Consequently, data analysis is \u000d\u000aoften a major bottleneck for materials chemistry research.15, 16 With the continuing advancement of \u000d\u000amodern radiation facilities,17 the need for tools that can aid scientists in structural analysis is in \u000d\u000aincreasing demand. ML has emerged as a powerful tool for automating several aspects of scatteringand spectroscopy data analysis.18-27 In this perspective, we describe the application of supervised and \u000d\u000aunsupervised ML to experimental scattering (PD, PDF, and SAS) and spectroscopy (INS and XAS) \u000d\u000adata. For a short introduction to supervised and unsupervised ML and the most popular ML \u000d\u000aalgorithms, we refer to Machine Learning Algorithms - A Review, Batta Mahesh.28","Fig. 1 | Traditional scattering and spectroscopy modelling workflows involve an expert scientist \u000d\u000amanually creating a structural model using input data from a database or the literature. This model is \u000d\u000arefined using dedicated software to extract structural information from the dataset such as atomic \u000d\u000apositions, crystallite size, crystallite shape, and atomic vibrations This process is repeated for each \u000d\u000anew dataset measured. \u000d\u000aMost applications of ML in materials\u2019 chemistry apply supervised ML methods. Supervised ML is \u000d\u000abroadly the task of predicting a label based on a given set of input features. As will be exemplified \u000d\u000athroughout the perspective, we observe three main applications of supervised ML for the analysis of \u000d\u000ascattering and spectroscopy data: 1) Identifying a physical model from a scattering or spectroscopy \u000d\u000adataset (Figure 2A). Here, experimental datasets are the input features, and the model is supervised \u000d\u000ato relate the datasets to the physical models, which are the labels. 2) Predicting scattering or \u000d\u000aspectroscopy data from a physical model. This can be achieved by using the data as labels and the \u000d\u000aphysical model as input features (Figure 2B). 3) Bypassing the model refinement step to directly \u000d\u000aobtain structural information (Figure 2C). This is done by training the supervised ML model on data \u000d\u000awith varying structural parameters. \u000d\u000aTo train an ML model using supervised methods, one needs a dataset consisting of many pairs of \u000d\u000alabels and input features. This dataset, consisting of e.g., structure models and simulated data, is \u000d\u000agenerally split into a training, validation, and test set, often in a 3:1:1 ratio. The model is trained on \u000d\u000athe training set, while being continuously evaluated on the validation set, using a user-defined \u000d\u000aobjective function, called the loss function. Depending on the chosen class of models, the training \u000d\u000awill often improve on the training set until it can fit any trends in the data, including noise \u000d\u000a(overtraining). The validation set is used to ensure that the model training is stopped before it is \u000d\u000aovertrained. Once training is complete, the test set, which has not been used during training or \u000d\u000avalidation, is employed to estimate the accuracy of the model on future unseen data (generalisation). \u000d\u000aThe quality and size of the training set thus plays a crucial role in the model\u2019s efficiency and accuracy, \u000d\u000awith larger, higher-quality datasets typically yielding better results. A model\u2019s ability to interpolate \u000d\u000aand extrapolate, or make predictions within the range of the training data and beyond it, is generally \u000d\u000ainfluenced by the ML algorithm and the range and diversity of the training set. Many factors therefore \u000d\u000aneed consideration when selecting and training an ML model. These include the choice of ML ","algorithm (tree-based methods, neural networks (NNs), genetic algorithms, etc.),28, 29 the number of \u000d\u000aparameters in the ML model, and both the quality and quantity of the training set. The model\u2019s ease \u000d\u000aof training and deployment can be influenced by the choice of ML algorithm. Interpretability of the \u000d\u000amodel depends strongly on the algorithm used, for example an individual decision tree is easily \u000d\u000ainterpretable, whereas a deep neural network with millions of parameters is not, and requires post hoc\u000d\u000amethods to understand its operation.30 When it comes to scalability, NNs have many more trainable \u000d\u000aparameters compared to tree-based methods. This makes tree-based method efficient learners in small \u000d\u000adata regime, however, NNs often prove more effective at handling larger datasets. NNs are today \u000d\u000acommonly trained on large datasets, as used in for example, chatGPT31 and AlphaFold.32 This \u000d\u000asuperiority in scalability might explain why NNs have become the predominant ML algorithm for \u000d\u000astructural analysis as large databases of training data have become increasingly available. \u000d\u000aWhile training an ML model can be computationally expensive, this is a one-time cost. Subsequent \u000d\u000apredictions using the ML model can be computationally inexpensive and integrated into web-based \u000d\u000asolutions, or can be done at synchrotron or neutron facilities for real-time structure prediction while \u000d\u000athe experiments are going on. ","Fig. 2 | A) Use of ML for identifying a structural model. During training, a supervised ML model is \u000d\u000atrained on pairs of structural models and scattering or spectroscopy simulations, here shown for PDF \u000d\u000adata. Afterwards, the ML model can quickly and computationally inexpensively identify the structural \u000d\u000amodel from (experimental) scattering or spectroscopy data. B) Use of ML for predicting scattering \u000d\u000aor spectroscopy data from a structure model. During training, a supervised ML model is trained on \u000d\u000apairs of structural models and scattering or spectroscopy simulations, here shown for PDF data. After \u000d\u000atraining, the ML model quickly and computationally inexpensively predicts scattering or \u000d\u000aspectroscopy data from a structural model. C) Use of ML to predict structural parameters. During \u000d\u000atraining, a supervised ML model is trained on pairs of scattering or spectroscopy simulations with \u000d\u000avarying structural parameters, here shown for SAXS data. After training, the ML model quickly and \u000d\u000acomputationally inexpensively identifies the structural information from the (experimental) scattering \u000d\u000aor spectroscopy data. ","However, supervised ML is limited by its reliance on paired input data and labels for training, which \u000d\u000acan be challenging to obtain for experimental data analysis. As will be discussed and exemplified \u000d\u000abelow, we observe three common issues encountered when analysing experimental scattering and \u000d\u000aspectroscopy data with supervised ML. These are illustrated in Figure 3: 1) Handling data with \u000d\u000acontributions from multiple chemical components. 2) Handling data arising from structures not \u000d\u000apresent in the training database and 3) accounting for experimental data that contain signals not \u000d\u000aincluded in the simulated data. In all three scenarios, the labelled data are inadequate for solving the \u000d\u000aproblem at hand, making unsupervised ML methods a more suitable alternative, or complementary \u000d\u000atool. Unsupervised ML models work without paired labels and input features, using only input \u000d\u000afeatures or intermediate input-derived labels, such as in autoencoders.33 Unsupervised ML is often \u000d\u000aused to present complex data in a low-dimensional space (dimensionality reduction), enabling the \u000d\u000aanalysis of high-dimensional dataset similarities, clustering, and the extraction of underlying data \u000d\u000atrends that are difficult to comprehend from the input representation space.28 Unsupervised methods \u000d\u000acan also be applied to \u2019demix\u2019 data, i.e., separating the signal from each component in a multi-phase \u000d\u000ascattering or spectroscopy dataset. \u000d\u000aIn the following sections, we use selected examples to provide an overview of how supervised ML \u000d\u000ahas been used to identify structural models and structural information from experimental PD, PDF, \u000d\u000aSAXS, INS and XAS data or predict the dataset from the structure. We also outline and exemplify \u000d\u000ahow unsupervised ML has been applied to address the three issues presented in Figure 3, and we \u000d\u000adiscuss the potential future impact of ML in the analysis of experimental materials chemistry data. ","Fig. 3 | Three common issues when analysing experimental scattering and spectroscopy data. \u000d\u000aSupervised ML models are trained on pairs of structural models from a structural database and \u000d\u000ascattering or spectroscopy simulations. However, supervised ML methods are challenged by; Issue \u000d\u000a1: The experimental data are obtained from a system containing multiple chemical species, which is \u000d\u000anot taken into account in the ML model. Issue 2: The required structural model is not included in the \u000d\u000astructure database used for training the ML model. Issue 3: The experimental data contains \u000d\u000abackground noise, instrumental effects or other phenomena not encountered by the simulated data. \u000d\u000aX-ray Absorption Spectroscopy: Large XAS Databases Accelerate Supervised ML Structure \u000d\u000aIdentification ","XAS is a powerful experimental technique for investigating the electronic and atomic structure of \u000d\u000amaterials. In XAS, a sample is exposed to a monochromatic X-ray beam, whose energy is varied in \u000d\u000aa range of ca. 10\u2013100 eV around the K-edge or L-edge of the elements in question, i.e. the energy \u000d\u000aneeded to eject electrons from the 1s or 2s orbital. This causes the sample to absorb some of the Xrays. By measuring and analysing the absorbed X-rays as a function of energy, it is possible to obtain \u000d\u000ainformation about the local electronic structure and chemical environment of the atoms in the sample. \u000d\u000aInformation about oxidation state and coordination environment can be obtained through X-ray \u000d\u000aAbsorption Near Edge Spectroscopy (XANES), while Extended X-ray Absorption Fine Structure \u000d\u000a(EXAFS) can provide knowledge of local atomic structure. \u000d\u000aConventional analysis of XAS data requires expertise in the complex data analysis as well as manual \u000d\u000awork. To address this, Zheng et al. created a large XANES database, XASdb, with more than 800,000 \u000d\u000acomputed reference XANES entries from over 40,000 materials from the open-science Materials \u000d\u000aProject database.\u000d\u000a34 Their supervised ML model, illustrated in Figure 4, was used for the analysis of \u000d\u000aXANES data. Given a XANES spectrum as input, it outputs a list of the chemical compounds whose \u000d\u000aspectra are most similar to the target spectrum (The task is illustrated in Figure 2A). From these \u000d\u000acompounds, chemical information such as oxidation state and coordination environment can be \u000d\u000aextracted. It predicts the chemical compound with 69.2 % top-5 accuracy on a test set of 13 simulated \u000d\u000aXANES spectra. However, the correct oxidation state is within top-5 with 84.6 % accuracy and the \u000d\u000acoordination environment with 76.9 % accuracy. On six experimental XANES spectra, it predicts \u000d\u000aoxidation state with 83.3 % accuracy, coordination environment with 83.3 % accuracy and the \u000d\u000achemical compound with 33.3 % top-5 accuracy.35 These results highlight the impact of large \u000d\u000adatabases like the open-science Materials Project34 and JARVIS.36 As these databases grow, they will \u000d\u000alikely catalyse supervised ML analysis of scattering and spectroscopy data in materials chemistry. ","Fig. 4 | Workflow schema of the Ensemble-Learned Spectra IdEntification (ELSIE) algorithm. The \u000d\u000aELSIE algorithm consists of two steps. In the first step, the absorbing species is identified and used \u000d\u000ato narrow down the candidate computed reference spectra. In the second step, the algorithm yields a \u000d\u000arank-ordered list of computational spectra according to similarity with respect to the target spectrum. \u000d\u000aThe figure is adapted from Zheng et al.35 (Under Creative Commons Attribution 4.0 International \u000d\u000aLicense https://creativecommons.org/licenses/by/4.0/). \u000d\u000aThe above example shows that large XANES databases, like XASdb, can be used to address the \u000d\u000aspectrum-to-structure problem, as illustrated in Figure 2A. However, it can also be used to address \u000d\u000athe inverse problem: structure-to-spectrum, as illustrated in Figure 2B. Calculating a XANES\u000d\u000aspectrum from a structure can be computationally demanding but by using a supervised ML model \u000d\u000ato predict the spectrum, this process can be done in milliseconds to seconds.37-39 Supervised ML can \u000d\u000aalso be used to directly predict chemical information such as average size, shape, morphology and ","oxidation states of first metallic nanoparticles40, 41 and metallic oxides,42 Bader charge,43 mean nearest \u000d\u000aneighbour distances,43 and local chemical environment,44 from an XANES spectrum or predict the \u000d\u000aradial distribution function from the experimental EXAFS data.45-49\u000d\u000aAnalysing XAS data from samples containing more than one chemical species remains a challenge, \u000d\u000aas supervised ML models trained on data simulated from a single chemical species are constrained to \u000d\u000abe used on experimental data from individual chemical species, and attempting to account for all \u000d\u000apossible chemistries, e.g. by training on simulated data from mixed samples, leads to a combinatorial \u000d\u000aexplosion. Instead, linear unsupervised ML techniques like principal component analysis (PCA) and \u000d\u000anon-negative matrix factorisation (NMF) have been used to discover trends in large XAS datasets \u000d\u000aand separate them into signals from their respective chemical components.50-55 For example, \u000d\u000aTanimoto et al. used NMF to identify and map spatial domains from absorption spectra in 2D-XAS \u000d\u000aimages of lithium ion batteries. Using NMF for this was successful despite the small differences in \u000d\u000athe spectra between the various compounds present in the sample.53 The authors recognised that NMF \u000d\u000acan be challenged by background effects as these can be predominant in some of the NMF-extracted \u000d\u000acomponents. Therefore, they subtracted a reference X-ray absorption spectrum obtained on \u000d\u000aLi0.5CoO2, which also includes that background signal. This trick enables the NMF method to \u000d\u000adistinguish small differences in the spectra. \u000d\u000aSmall-angle scattering: Supervised ML for Dataset and Parameter Prediction \u000d\u000aSAS is a strong technique to obtain information about the morphology, orientation and size \u000d\u000adistribution of e.g., nanoparticles in solution and solids.56 In a SAS experiment, X-ray or neutron \u000d\u000ascattering data are measured at small scattering vectors, e.g., the Q-range from ca. 0.001 \u00c5-1 to 1 \u00c5-1. \u000d\u000aThis region of the scattering signal contains structural information about the species in the sample on \u000d\u000athe nanometer scale. \u000d\u000aTraditional SAS data fitting is done by refining a model against the data. The model must describe \u000d\u000athe particle shape, size, and size distribution as well as possible agglomerations of e.g. nanoparticles \u000d\u000aor large molecules in the sample, and much work is often needed in deciding on a suitable model. \u000d\u000aThis step can be time-consuming and prone to errors.57 Here, ML can assist by providing a more \u000d\u000aefficient approach to fitting experimental data to theoretical models. The Computational ReverseEngineering Analysis for Scattering Experiments Genetic Algorithm (CREASE-GA) tool, developed \u000d\u000aby members of Prof. A. Jayaraman\u2019s research group, can reconstruct 3D structures from SAS patterns ","using a genetic algorithm.58-65 CREASE-GA compares the goodness-of-fit between the experimental \u000d\u000aSAXS pattern and a population of 3D structures. A genetic algorithm29 is then used to update the 3D \u000d\u000astructure population to better describe the experimental SAS pattern. This process continues until \u000d\u000aconvergence, determining the 3D structure of the sample in question. Originally, the SAS patterns \u000d\u000afrom the 3D structure population were calculated using the Debye scattering equation. This posed a \u000d\u000acomputational bottleneck for CREASE-GA.\u000d\u000a58 However, the authors have recently managed to \u000d\u000aaccelerate CREASE by over 95 % by employing NNs to estimate the SAS patterns.60, 63\u000d\u000aSupervised ML has also proved to be an efficient tool for direct parameter extraction from SAS data, \u000d\u000awhich might be difficult or time-consuming for humans to detect, such as orientation,66 shape,67-69 or \u000d\u000athe model for SAS form factor fitting.70-72 Thereby, the ML model can bypass the cumbersome step \u000d\u000aof identifying the structural model for structure refinement. For example, the Scattering Ai aNalysis \u000d\u000a(SCAN) tool can predict the model for SAS form factor fitting from a SAXS pattern obtained from a \u000d\u000ananoparticle. With the SCAN tool, the user can choose from a range of ML algorithms including treebased algorithms and NNs. These algorithms individually achieve accuracies between 27.4 % and \u000d\u000a95.9 % based on a test set of simulated SAXS data. However, when combined, they achieve an \u000d\u000aaccuracy of 97.3 %. We are grateful to the authors for making SCAN open source, which has made \u000d\u000ait possible to implement it as a Hugging Face app.73 This makes it easily useable, also for users \u000d\u000awithout programming experience, as illustrated in Figure 6. \u000d\u000aFigure 6 | A) The SCAN72 tool directly predicts structural information such as particle shape from a \u000d\u000aSAXS pattern. B) Overview of the SCAN72 tool's ease of use for predicting structural information \u000d\u000afrom a SAS pattern. Simply click \u201cBrowse files\u201d, wait for the model to predict the structural \u000d\u000ainformation, and, if needed, download the detailed information in an Excel sheet. All of this is \u000d\u000apossible through the Hugging Face app.73","Powder Diffraction: Structure Identification \u000d\u000aPD is a fundamental technique in materials chemistry that is used to analyse the crystal structure of a \u000d\u000apowdered sample. In PD, a powder of crystalline particles is exposed to an X-ray or neutron beam, \u000d\u000acausing Bragg diffraction due to the periodic atomic arrangement in the sample. By analysing the \u000d\u000aBragg peak position, intensity and shape, information about the crystalline structure of the material \u000d\u000acan be obtained. \u000d\u000aRecent advances in ML techniques offer promising new opportunities in PD data analysis. For \u000d\u000aexample, it has been demonstrated that a sample\u2019s crystal system and space group can be predicted \u000d\u000afrom X-ray PD data using NNs26, 74-76 and tree-based techniques.77 Suzuki et al. demonstrated that an \u000d\u000aadvantage of the tree-based ML approaches is that they are interpretable.77 Interpretability enables us \u000d\u000ato understand the ML model\u2019s prediction mechanism and thus analyse when it predicts differently \u000d\u000afrom a human expert. This can either show when the ML model is wrong and need to be corrected or \u000d\u000areveal when it uncovers unexpected correlations that may lead to scientific insights. In contrast, Park \u000d\u000aet al. used an NN which is easier to train on possible larger future datasets containing millions or \u000d\u000abillions of PD patterns.75 \u000d\u000aHowever, these methods can only be used to determine the crystal system or the space group from \u000d\u000aPD data. To identify the full structural model for e.g., structure refinement, the unit cell, and unit cell \u000d\u000acontent is also needed. Garcia-Cardona et al.\u000d\u000a78 made progress towards this for neutron diffraction \u000d\u000adata, where the crystal system (cubic, tetragonal, trigonal, monoclinic, and triclinic) could be \u000d\u000apredicted with an accuracy of 92.65 % (Figure 2A) using convolutional NNs, which are a type of \u000d\u000aNNs that capture the relationship between neighbouring data points e.g. neighbouring intensities in \u000d\u000athe diffraction pattern. Subsequently, another supervised tree-based ML model was used to predict \u000d\u000aunit cell parameters (unit cell length and angles) from the data.78 The authors note that the ML models \u000d\u000apossess good performance on simulated data but more sophisticated models are required before it is \u000d\u000aapplicable on experimental data. Furthermore, an ML model that is capable of precisely predicting a \u000d\u000afull structural model, including unit cell content, as required for e.g., Rietveld refinement of \u000d\u000aexperimental PD data, is yet to be developed. \u000d\u000aIf working in a more restricted chemical space with well-defined components, it is possible to use \u000d\u000asupervised ML models for direct prediction of structural parameters for the phases included in the \u000d\u000aspace. Dong et al. demonstrated that it is possible to directly predict structural information such as \u000d\u000ascale factor, lattice parameter and crystallite size (Figure 2C) from PD patterns from a system of 5 ","different metal oxides using a convolutional NN that they call Parameter Quantification Network \u000d\u000a(PQ-Net).\u000d\u000a2\u000d\u000a They obtained an experimental X-ray diffraction computed tomography dataset of a multiphase Ni-Pd/CeO2-ZrO2/Al2O3 containing about 20,000 diffraction patterns, each pixel containing a \u000d\u000amultiphase pattern. Treating such a large quantity of Rietveld refinements takes significant computer \u000d\u000atime. To overcome this limitation, PQ-Net was trained on simulated PD data with varying scale \u000d\u000afactors, lattice parameters and crystallite size for NiO, PdO, CeO2, ZrO2 and theta-Al2O. A 2nd degree \u000d\u000aChebyshev polynomial background and Poisson noise were also added to the training data. After \u000d\u000atraining, PQ-Net can identify the crystalline phase, scale factor, lattice parameter and crystallite size \u000d\u000afor each experimental PD pattern in the dataset, orders of magnitudes faster than done using \u000d\u000aconventional Rietveld refinement. As seen in Figure 7, the results of using PQ-Net are comparable to \u000d\u000athose determined through Rietveld methods on experimental data. \u000d\u000aFigure 7 | Crystallite size (colourbar axis corresponding to nm) and lattice parameter a (colourbar axis \u000d\u000acorresponding to \u00c5) maps for CeO2 and ZrO2 obtained with the Rietveld method, results obtained ","with the PQ-Net, their absolute difference for the experimental multi-phase NiO-PdO-CeO2-ZrO2-\u000d\u000aAl2O3 system and the uncertainty maps of the deep ensemble PQ-Net.2\u000d\u000a (Under Creative Commons \u000d\u000aAttribution 4.0 International License https://creativecommons.org/licenses/by/4.0/). \u000d\u000aDong et al. required 100,000 datasets for training to achieve good results predicting structural \u000d\u000ainformation on experimental data from a chemical system with five components. For larger and \u000d\u000acomplex systems with more possible components, the supervised ML model must be trained on even \u000d\u000amore data of both individual structural models and combinations of these. An example of this \u000d\u000aapproach is the work by Lee et al., who used a supervised ML model for phase identification in a \u000d\u000aquaternary chemical system consisting of Sr, Al, Li, and O. This system thus spans simple to ternary \u000d\u000aoxides and multiple different polymorphs, and in total 170 inorganic compounds appear in the \u000d\u000achemical space.79 Here, the ML model was trained on 1,785,405 synthetic combinatorically mixed \u000d\u000aPD patterns. After training, the model was able to phase identify and give rough estimates of phase \u000d\u000afractions of multicomponent systems from XRD data. \u000d\u000aInstead of training the supervised ML model on large databases of combinations of phases, \u000d\u000aunsupervised ML methods, such as PCA and NMF, can demix multiphase PD patterns into individual \u000d\u000aphase patterns, as also addressed for XAS data above.80, 81 Here, a set of experimental diffraction \u000d\u000apatterns are given as input to the unsupervised ML algorithm, which decomposes it into its constituent \u000d\u000aparts. However, the PCA and NMF algorithms may encounter difficulties if the PD pattern of a \u000d\u000achemical phase changes during the reaction, for example, through peak shifting from a unit cell \u000d\u000achange, variations in peak intensity from a change in thermal vibrations, or a change in the crystalline \u000d\u000asize, leading to different peak widths. Stanev et al. addressed the peak shifting problem by combining \u000d\u000aNMF with cross-correlation analysis of the demixed PD patterns. They were hence able to cluster the \u000d\u000ademixed PD patterns originating from the same chemical phase.82\u000d\u000aChen et al. employed deep reasoning networks (an unsupervised method) to map the crystal-structure \u000d\u000aphase diagram of Bi\u2013Cu\u2013V oxide using experimental PD data.83 Based on PD data from Bi\u2013Cu\u2013V \u000d\u000aoxides prepared in various compositions, the ML model was trained to demix the phases in the PD \u000d\u000apatterns, and subsequently map the crystal-structure phase diagram of Bi\u2013Cu\u2013V oxide.83 Once \u000d\u000atrained, the deep reasoning network can take a PD pattern from a sample in the composition space as \u000d\u000ainput, and demix signals from multiple phases into their constituent components. Using a linear \u000d\u000acombination of the components, the PD pattern can be reconstructed, and the phase diagram can be ","constructed with phase concentrations. The authors demonstrated this process on X-ray PD patterns \u000d\u000afrom a phase diagram of Bi\u2013Cu\u2013V containing 19 chemical phases. \u000d\u000aTotal Scattering with Pair Distribution Function: Nanoparticle structure determination \u000d\u000aTotal scattering experiments are similar to PD, as the scattering of X-rays or neutrons from a sample \u000d\u000ais measured. However, for total scattering, it is not only the Bragg diffraction peaks that are analysed \u000d\u000abut also the diffuse scattering arising from local structural order. This enables structural analysis of \u000d\u000aboth crystalline and non-crystalline samples. Total scattering data are often analysed in real space \u000d\u000athrough PDF analysis. A PDF is generated by normalising, correcting, and Fourier-transforming the \u000d\u000atotal scattering signal, and the PDF represents a histogram of interatomic distances. Like PD, PDF \u000d\u000acan be used to characterise crystalline materials, but has especially emerged as a powerful technique \u000d\u000ato characterise the atomic arrangement in non- or poorly crystalline materials such as clusters in \u000d\u000asolution and disordered, amorphous, and nanomaterials.84, 85 As for XAS, SAS and PD discussed \u000d\u000aabove, ML models can help accelerate the PDF modelling process. For example, supervised ML \u000d\u000amodels have been developed to predict space groups from PDF data of crystalline materials.86 We \u000d\u000ahave furthermore shown that supervised ML can be used to identify the structure of polyoxometalate \u000d\u000aclusters.87, 88 \u000d\u000aIn a slightly different application of supervised methods, we have recently demonstrated how \u000d\u000aexplainable supervised ML can be used to extract information on the local atomic arrangement \u000d\u000apresent in a sample.89 The aim of PDF analysis of e.g. nanostructured materials is often to identify \u000d\u000amodels for the main structural motifs in a material. Our algorithm, ML-MotEx provides this \u000d\u000ainformation by using SHAP (SHapley Additive exPlanation)90, 91 values to identify which atoms in a \u000d\u000agiven starting model are important for the fit quality. The starting model should be chosen to contain \u000d\u000athe main atomic arrangements expected to be found in the sample. If analysing the structure of e.g. \u000d\u000aan amorphous materials, the starting model may be a related crystalline structure. Based on the model, \u000d\u000athousands of structure fragments are generated by iteratively removing atoms from the starting model, \u000d\u000aand a PDF fit is done for each of the fragments. A supervised ML model is then trained on the \u000d\u000athousands of PDF fits, and ultimately, each atom can be assigned an atom contribution value which \u000d\u000adescribes how much it contributes to the goodness-of-fit. By analysing the SHAP values, it is thus \u000d\u000apossible to identify which motifs in the starting model are important in the material to describe the \u000d\u000adata. ML-MotEx has so far been used to identify the structure of ionic clusters in solution,89 extract ","structural motifs in amorphous metal oxides,92 and identify stacking faulted domains on MnO2 from \u000d\u000aboth X-ray PD and PDF data.93 \u000d\u000aThe supervised ML methods used for structure identification for both PD and PDF data discussed so \u000d\u000afar are limited to identifying structural models that are part of the structural database on which they \u000d\u000ahave been trained. Ultimately, the aim of a PDF experiment may be to solve the structure of new \u000d\u000ananomaterials. To explore structural models beyond any existing structural database (issue 2 in Figure \u000d\u000a3), some classes of unsupervised ML could be useful. We have recently used a graph-based \u000d\u000aconditional variational autoencoder,33 DeepStruc (Figure 8A) to determine the atomic structure of \u000d\u000ametallic nanoparticles up to 200 atoms in size from PDF data.94, 95 Given a PDF, DeepStruc can output \u000d\u000aa particle structure, and we obtained mean absolute errors of 0.093 \u00b1\u00090.058 \u00c5 on the atomic positions \u000d\u000ain metallic nanoparticles from simulated PDFs. Figure 8B shows the results of applying DeepStruc \u000d\u000ato experimental PDFs obtained from three chemical systems, consisting of two magic-sized clusters \u000d\u000aI) Au144(p-MBA)6096 and II) Au144(PET)60,\u000d\u000a96 and III) a 1.8 nm Pt nanoparticle.97 All three structures \u000d\u000amatch the structures found in the literature and provide good data fits. Although DeepStruc is \u000d\u000asupervised in the sense that it is trained on structure and PDF pairs, it also has abilities from \u000d\u000aunsupervised ML as it learns to probabilistically map cluster structures and PDFs into a twodimensional chemically meaningful space, which we refer to as the latent space. By inspecting the \u000d\u000alatent space, it is possible to find relations between different types of cluster models. DeepStruc places \u000d\u000adecahedral (orange) structures in the latent space between face-centered cubic (fcc) (light blue) and \u000d\u000ahexagonal closed packed (hcp) (pink) structures. This spatial arrangement can be explained by \u000d\u000aconsidering that decahedral structures are constructed from five tetrahedrally shaped fcc crystals, \u000d\u000aseparated by {111} twin boundaries.13, 98, 99 The twin boundaries, resembling stacking faulted regions \u000d\u000aof fcc, justify their location in the latent space between fcc and hcp.48, 95, 96 The capability of \u000d\u000aDeepStruc to interpolate between cluster structures arises from each structure in the latent space being \u000d\u000aprobabilistically rather than deterministically predicted. This has been demonstrated in Anker & Kj\u00e6r \u000d\u000aet al.,94 where we show that generative models28 are necessary to go beyond the structural database \u000d\u000aused for training the ML model. Specifically, we showed that a generative model, like DeepStruc, \u000d\u000acan interpolate between structural models, as shown in Figure 8C, while still yielding sensible results. \u000d\u000aMore traditional deterministic models, which are not probabilistic, could not interpolate between \u000d\u000astructures and thereby not go beyond the structural database when predicting a structural model from \u000d\u000aa PDF. ","DeepStruc is integrated with the Hugging Face platform, enabling users to rapidly determine the \u000d\u000astructure of small metallic nanoparticles from PDFs using a simple two-click process.100 The Hugging \u000d\u000aFace integration provides a user-friendly experience, without requiring data storage or complex \u000d\u000asoftware installations. \u000d\u000aFigure 8 | DeepStruc is a Conditional Variational Autoencoder that can solve the structure of a small \u000d\u000amono-metallic nanoparticle from a PDF. A) DeepStruc predicts the xyz-coordinates of the mono-","metallic nanoparticle structure with a PDF provided as the conditional input. The encoder uses the \u000d\u000astructure and PDF as input, while the prior only takes the PDF as input. A latent space embedding is \u000d\u000agiven as input to the decoder to obtain the structural output, which produces the corresponding monometallic nanoparticle xyz-coordinates. During the training of DeepStruc, both the blue and green \u000d\u000aregions are used, while only the green region is used for structure prediction during the inference \u000d\u000aprocess. B) PDF fit of the reconstructed structure from three different nanoparticle systems: I) \u000d\u000aAu144(p-MBA)60 PDF,96 the II) Au144(PET)60 PDF96 using a reconstructed structure icosahedral \u000d\u000astructure and III) a 1.8 nm Pt nanoparticle PDF from Quinson et al.97 Figure 8A and B are adapted \u000d\u000afrom Kj\u00e6r & Anker et al.95 (Under Creative Commons Attribution 3.0 Unreported License \u000d\u000ahttps://creativecommons.org/licenses/by/3.0/). C) Structures generated by decoding different extents \u000d\u000aof interpolation of the latent variables obtained for PDF-A and PDF-B. The generated structures start \u000d\u000afrom Structure-A and progressively evolve towards Structure-B. This work uses a Conditional \u000d\u000aVariational Autoencoder similar to DeepStruc and we compare it with a Deterministic Autoencoder. \u000d\u000aFigure 8C is from Anker & Kj\u00e6r et al.94 (Under Creative Commons Attribution-NonCommercialNoDerivatives 4.0 International License https://creativecommons.org/licenses/by-nc-nd/4.0/).\u000d\u000aUnsupervised ML algorithms have also been employed to either uncover trends in PDFs obtained \u000d\u000afrom multiple samples or to separate the signal from different phases in a PDF.101-103 NMF has proven \u000d\u000ato be especially useful, and has been used to analyse PDFs obtained from various materials and \u000d\u000aconditions, including battery materials, amorphous solid dispersions, or data collected under highpressure. It has also been used to extract the interface PDF between a Fe and a Fe3O4 phase.104-108\u000d\u000aRecently, efforts have also been made to develop an efficient and accurate NMF algorithm that can \u000d\u000abe used during data measurement.109, 110 This NMF algorithm is available at PDF-in-the-cloud.111, 112\u000d\u000aInelastic Neutron Scattering: Extraction of a Materials Hamiltonians \u000d\u000aINS is an inelastic experimental technique for investigating the vibrational and magnetic properties \u000d\u000aof atoms in materials. During an INS experiment, a neutron beam interacts with the atomic nuclei and \u000d\u000athe magnetic moment of the electrons in the material. By measuring the initial and final neutron \u000d\u000aenergy, one can determine the energy of the interaction or excitation, which allows to study of both \u000d\u000athe atomic and magnetic structure. Analysing the inelastically scattered neutrons thus provides \u000d\u000ainformation about the Hamiltonian, which governs atomic and magnetic interactions. However, \u000d\u000ainterpreting experimental INS data or extracting the Hamiltonian can be challenging due to a large ","amount of measured data and the complexity of simulating accurate INS data that resembles the \u000d\u000ascattering process. \u000d\u000aFor example, determining the appropriate spin wave model of the half-doped bilayer manganite, \u000d\u000aPr(Ca, Sr)2Mn2O7 (abbreviated as PCSMO) has been debated, with a Goodenough spin wave model \u000d\u000a(Figure 9A),113, 114 or a Dimer spin wave model (Figure 9A) being considered.115 Due to the subtle \u000d\u000adifferences between the two models, determining which model the INS spectra correspond to has \u000d\u000abeen challenging, as it requires a meticulous manual fitting process. After extensive experimentation \u000d\u000aand careful data treatment, it was ultimately determined that the Goodenough spin wave model best \u000d\u000adescribes the experimental data (Figure 9B).116\u000d\u000aTo ease this task, a supervised ML model has been developed to assist in analysing INS data. By \u000d\u000atraining supervised ML models on simulated INS spectra calculated using physics-driven equations, \u000d\u000aHamiltonians can be predicted from INS data. Specifically, Butler et al. demonstrated that NNs can \u000d\u000apredict magnetic Hamiltonians or classify the spin wave model from simulated INS data of PCSMO, \u000d\u000asaving significant time compared to manual data analysis.27 They first used a logistic regression117\u000d\u000amodel, illustrated in Figure 9C, which makes a simple binary classification, either Goodenough or \u000d\u000aDimer, but gives no indication of the reliability of the prediction. It was thereby challenging to judge \u000d\u000awhen to trust the model. To resolve this problem, they used a deterministic uncertainty quantification \u000d\u000a(DUQ) classifier (Figure 9C),27, 116 to perform uncertainty classification instead. The DUQ classifier \u000d\u000aoutputs a weight vector associated with the input that is correlated to the class predictions. If all the \u000d\u000aweights in the weight vector are close to a class, the prediction has a large certainty, while the \u000d\u000acertainty is larger with a larger spread of weight vectors. \u000d\u000aTo reliably predict the spin wave model from experimental INS data (Figure 9D), the DUQ classifier \u000d\u000awas trained on computationally expensive resolution convoluted INS spectra. Physics-driven \u000d\u000asimulations may not always capture the experimental noise, instrumental effects or other artefacts \u000d\u000afrom phenomena not described by the underlying theory (issue 3 in Figure 3). In this example, the \u000d\u000acomputationally inexpensive resolution unconvolved INS simulations did not capture any \u000d\u000ainstrumental effects. To address this issue, we introduced an unsupervised image-to-image algorithm, \u000d\u000aExp2SimGAN, which is a generative adversarial network118 (GAN) capable of learning the simulated \u000d\u000aand experimental data distributions and transforming between them, e.g. transforming a simulated \u000d\u000adataset into one that resembles an experimental dataset, or vice versa.119 By using Exp2SimGAN to \u000d\u000aconvert experimental INS spectra into simulated-like data, the DUQ classifier, trained on \u000d\u000acomputationally inexpensive resolution unconvolved INS spectra, can be applied to the experimental ","INS data (Figure 9E).119 This approach helps bridge the gap between simulations and experimental \u000d\u000adata, allowing for more accurate and efficient analysis. \u000d\u000aFig. 9 | Determining the spin wave model from experimental INS data using ML. A-B) Two magnetic \u000d\u000aexchange models in a single sheet of Mn ions in a half-doped manganite. (Left) Goodenough model \u000d\u000a(Right) Dimer model.27 C) Schematic representation of the DUQ method. The input initially passes \u000d\u000athrough a series of convolutional NNs (orange block) to extract features. In standard logistic \u000d\u000aregression, the outputs from the convolutional NNs are classified by summing the weights connecting \u000d\u000aeach filter, fi, to the class C of interest. This is a simple binary classification. The DUQ classifier \u000d\u000ainstead outputs a weight vector associated with the input that is correlated to the class predictions. If \u000d\u000aall the weights in the weight vector are close to a class (based on distances, Kc, from the weight vector \u000d\u000ato the centre, ec, of clusters of training examples), the prediction has a large certainty, while the \u000d\u000acertainty is larger with a larger spread of weight vectors. Figure 9C is from Butler et al.27 (Under ","Creative Commons Attribution 4.0 International License \u000d\u000ahttps://creativecommons.org/licenses/by/4.0/). D) 2D representation of experimental data of PCSMO \u000d\u000ameasured at 4 K using the MAPS spectrometer.116 The INS spectra are arranged in terms of incident \u000d\u000aneutron energy (Ei) and bins of energy transfer \u03c9 = 0.10\u20130.16Ei, etc. 119 E) The DUQ classifier cannot \u000d\u000aidentify the spin wave model of an experimental dataset with high certainty. However, Exp2SimGAN \u000d\u000amatches the experimental dataset to the simulated training set of the DUQ classifier enabling the \u000d\u000aclassification of the spin wave model with high certainty. Figure 9A+B+D are from Anker et al.119\u000d\u000a(Under Creative Commons Attribution 3.0 Unreported License\u000d\u000ahttps://creativecommons.org/licenses/by/3.0/).\u000d\u000aSamarakoon et al. demonstrated an alternative approach for the analysis of INS data using \u000d\u000aautoencoders.120-122 They showed that autoencoders can eliminate background signals and artefacts \u000d\u000afrom the experimental INS spectrum by compressing them into a latent space. Once in the latent \u000d\u000aspace, the magnetic behaviour can be categorised, and the autoencoder can solve the inverse problem \u000d\u000aby extracting the Hamiltonians from the experimental INS spectrum. This is achieved by decoding \u000d\u000athe INS spectrum from the latent space positions. As a result, the autoencoder works as a fast \u000d\u000asurrogate model for INS simulations accelerating the fitting procedure of the experimental INS \u000d\u000aspectrum. Later work integrates ML modelling approaches into the INS experiments enabling realtime analysis of INS data.123 \u000d\u000aRemaining challenges and future outlook \u000d\u000aIn this perspective, we have shown how analysis of scattering and spectroscopy data is becoming \u000d\u000aincreasingly feasible using supervised and unsupervised ML approaches. Especially, the progress of \u000d\u000alarge open-source databases has catalysed the use of supervised ML. \u000d\u000aSupervised ML is now widely used to identify structural models (Figure 2A) from data, to predict \u000d\u000adata from structural models (Figure 2B), or to directly provide structural information from data \u000d\u000a(Figure 2C). However, we have highlighted three issues that supervised ML faces in automating the \u000d\u000aanalysis of scattering and spectroscopy data (Figure 3). Issue 1: handling datasets originating from a \u000d\u000amix of chemical phases. Here, unsupervised ML, especially NMF, has successfully been used to \u000d\u000ademix datasets into constituent components. We anticipate the emergence of combination methods, \u000d\u000awhere unsupervised ML firstly demixes complex datasets whereafter they are independently\u000d\u000aanalysised using supervised ML. Issue 2: handling data from a structural model that is not part of a ","database. Here, generative modelling appears promising for interpolating between structural models \u000d\u000ain a database. Issue 3: handling experimental data. For the ML models to significantly impact the data \u000d\u000aanalysis of scattering and spectroscopy data, they must perform well on experimental data and not \u000d\u000aonly on simulated data. Often in materials chemistry, supervised ML models are trained on physicsdriven simulations which do not include instrumental artefacts, noise or other phenomena not directly \u000d\u000adescribed by the underlying physics. Here, new methods are needed to make simulated data resemble \u000d\u000aexperimental data. Unsupervised image-to-image algorithms could potentially address this \u000d\u000achallenge.119\u000d\u000aHowever, using ML to resolve more complicated challenges in materials chemistry is still challenged \u000d\u000aby limited sizes of datasets connecting structure and spectroscopy/scattering signal. One way to \u000d\u000ahandle limited data is to constrain the ML model with chemical knowledge. Here, physics-informed \u000d\u000aNNs serve as an inspiration, as they embed partial differential equations as constraints into the NN \u000d\u000aoptimisation problem, for example, when using an NN as a surrogate model for the Schr\u00f6dinger \u000d\u000aequation.124 As a result, the range of potential solutions is limited to a manageable size for ML to\u000d\u000ahandle with the available information. However, not all chemistry can be expressed as differentiable \u000d\u000aequations, necessitating the development of similar approaches that can incorporate chemical \u000d\u000aknowledge into the ML architecture as \u2019chemistry-driven ML\u2019. Equivariant graph-based NNs show \u000d\u000apromise, as they leverage group representation theory to design architectures that are equivariant to \u000d\u000aspecified symmetry groups, making them well-suited for analysing chemical systems with underlying \u000d\u000asymmetries.125 We expect another impact to come from interpretable and explainable ML which \u000d\u000aenables researchers to understand the underlying mechanisms behind predictions, build trust in ML \u000d\u000amodel outcomes, and uncover unexpected correlations that may lead to scientific insights. For those \u000d\u000ainterested, we refer to a recent review paper by Oviedo et al.30 for more about interpretable and \u000d\u000aexplainable ML in materials chemistry. \u000d\u000aCurrently, it is not mandatory to publish data, code, and software requirements alongside research \u000d\u000apapers, making it difficult for other researchers to apply trained ML models to their own experimental \u000d\u000adata. A paradigm shift from publishing papers with code to publishing code with papers may thus be \u000d\u000aneeded. For the ML developer, we refer to N. Artrith et al.\u000d\u000a126 for best practices in ML for chemistry. \u000d\u000aWe suggest that publishing code with papers would greatly benefit the field, allowing materials \u000d\u000achemists to analyse data easily or automatically without domain expertise. \u000d\u000aIf we unceasingly share ML models, expand open-source databases, and bridge the gap between \u000d\u000asimulated and experimental data, the next decade holds promise to integrate analysis of scattering ","and spectroscopy data with ML into self-driving laboratories. Self-driving laboratories are currently \u000d\u000areceiving much attention for e.g. identifying new, improved photocatalysts for hydrogen production \u000d\u000afrom water,131 synthesising pharmaceutical compounds,132 and optimising nanostructure synthesis \u000d\u000abased on their optical properties.133, 134 As illustrated in Figure 10, the self-driving laboratory will \u000d\u000asynthesise a material, perform a scattering or spectroscopy experiment, and the data can be \u000d\u000aautomatically analysed with ML. The findings will then be fed into an active learning framework that \u000d\u000asuggests the next experiment based on structural insight. \u000d\u000aFig. 10 | The proposed self-driving laboratory to integrate analysis of scattering and spectroscopy \u000d\u000adata with ML into self-driving laboratories. The automated experiment platform synthesises a \u000d\u000amaterial and performs one or multiple structure characterisation experiments. The data from these \u000d\u000aexperiments will be analysed using ML. The analysed data will be automatically fed into an active \u000d\u000alearning framework that will suggest the next experiment enabling a directed synthesis of functional \u000d\u000amaterials via insight at the atomic level. \u000d\u000aAUTHOR CONTRIBUTIONS ","All authors were involved in the preparation of the manuscript and approved the final manuscript. \u000d\u000aCOMPETING INTERESTS \u000d\u000aThe authors declare no competing interests. \u000d\u000aAcknowledgements \u000d\u000aThis work is part of a project that has received funding from the European Research Council (ERC) \u000d\u000aunder the European Union\u2019s Horizon 2020 Research and Innovation Programme (grant agreement \u000d\u000aNo. 804066). Funding from the Danish Ministry of Higher Education and Science through the \u000d\u000aSMART Lighthouse is gratefully acknowledged. We acknowledge support from the Danish National \u000d\u000aResearch Foundation Center for High Entropy Alloy Catalysis (DNRF 149). \u000d\u000aReferences \u000d\u000a1. C. Wang, U. Steiner and A. Sepe, Small, 2018, 14, e1802291. \u000d\u000a2. H. Dong, K. T. Butler, D. Matras, S. W. T. Price, Y. Odarchenko, R. Khatry, A. Thompson, \u000d\u000aV. Middelkoop, S. D. M. Jacques, A. M. Beale and A. Vamvakeros, npj Comput. Mater., \u000d\u000a2021, 7, 74. \u000d\u000a3. M. Xia, T. Liu, N. Peng, R. Zheng, X. Cheng, H. Zhu, H. Yu, M. Shui and J. Shu, Small \u000d\u000aMethods, 2019, 3, 1900119. \u000d\u000a4. T. Ebensperger, P. Stahlhut, F. Nachtrab, S. Zabler and R. Hanke, J. Instrum., 2012, 7, \u000d\u000aC10008. \u000d\u000a5. B. Hornberger, J. Kasahara, M. Gifford, R. Ruth and R. Loewen, presented in part at the \u000d\u000aSPIE Optical Engineering + Applications, 2019. \u000d\u000a6. J. Nelson, S. Misra, Y. Yang, A. Jackson, Y. Liu, H. Wang, H. Dai, J. C. Andrews, Y. Cui \u000d\u000aand M. F. Toney, J. Am. Chem. Soc., 2012, 134, 6337-6343. \u000d\u000a7. G. A. Elia, G. Greco, P. H. Kamm, F. Garc\u00eda-Moreno, S. Raoux and R. Hahn, Adv. Funct. \u000d\u000aMater., 2020, 30, 2003913. \u000d\u000a8. E. T. S. Kj\u00e6r, O. Aalling-Frederiksen, L. Yang, N. K. Thomas, M. Juelsholt, S. J. L. \u000d\u000aBillinge and K. M. \u00d8. Jensen, Chem. Methods, 2022, 2, e202200034. \u000d\u000a9. H. Matsui, N. Ishiguro, T. Uruga, O. Sekizawa, K. Higashi, N. Maejima and M. Tada, \u000d\u000aAngew. Chem. Int. Ed., 2017, 56, 9371-9375. \u000d\u000a10. A. Vamvakeros, D. Matras, T. E. Ashton, A. A. Coelho, H. Dong, D. Bauer, Y. Odarchenko, \u000d\u000aS. W. T. Price, K. T. Butler, O. Gutowski, A.-C. Dippel, M. v. Zimmerman, J. A. Darr, S. D. \u000d\u000aM. Jacques and A. M. Beale, Small Methods, 2021, 5, 2100512. \u000d\u000a11. K. M. \u00d8. Jensen, X. Yang, J. V. Laveda, W. G. Zeier, K. A. See, M. D. Michiel, B. C. \u000d\u000aMelot, S. A. Corr and S. J. L. Billinge, J. Electrochem. Soc., 2015, 162, A1310-A1314. \u000d\u000a12. J. Becher, D. F. Sanchez, D. E. Doronkin, D. Zengel, D. M. Meira, S. Pascarelli, J.-D. \u000d\u000aGrunwaldt and T. L. Sheppard, Nat. Cat., 2021, 4, 46-53. \u000d\u000a13. S. Banerjee, C.-H. Liu, K. M. \u00d8. Jensen, P. Juhas, J. D. Lee, M. Tofanelli, C. J. Ackerson, \u000d\u000aC. B. Murray and S. J. L. Billinge, Acta Crystallogr. A, 2020, 76, 24-31. ","14. L. Yang, P. Juh\u00e1s, M. W. Terban, M. G. Tucker and S. J. L. Billinge, Acta Crystallogr. A, \u000d\u000a2020, 76, 395-409. \u000d\u000a15. A. Agrawal and A. Choudhary, APL Mater., 2016, 4, 053208. \u000d\u000a16. K. T. Butler, D. W. Davies, H. Cartwright, O. Isayev and A. Walsh, Nature, 2018, 559, 547-\u000d\u000a555. \u000d\u000a17. G. Pacchioni, Nat. Rev. Phys., 2019, 1, 100-101. \u000d\u000a18. B. Sullivan, R. Archibald, J. Azadmanesh, V. G. Vandavasi, P. S. Langan, L. Coates, V. \u000d\u000aLynch and P. Langan, J. Appl. Cryst., 2019, 52, 854-863. \u000d\u000a19. T. W. Ke, A. S. Brewster, S. X. Yu, D. Ushizima, C. Yang and N. K. Sauter, J Synchrotron \u000d\u000aRadiat, 2018, 25, 655-670. \u000d\u000a20. M. Doucet, A. M. Samarakoon, C. Do, W. T. Heller, R. Archibald, D. Alan Tennant, T. \u000d\u000aProffen and G. E. Granroth, Mach. learn.: sci. technol., 2021, 2, 023001. \u000d\u000a21. J. A. Aguiar, M. L. Gong, R. R. Unocic, T. Tasdizen and B. D. Miller, Sci. Adv., 2019, 5, \u000d\u000aeaaw1949. \u000d\u000a22. S. Muto and M. Shiga, Microscopy, 2019, 69, 110-122. \u000d\u000a23. H. Wang, Y. Xie, D. Li, H. Deng, Y. Zhao, M. Xin and J. Lin, J. Chem. Inf. Model, 2020, \u000d\u000a60, 2004-2011. \u000d\u000a24. M. Tatlier, Neural. Comput. Appl., 2011, 20, 365-371. \u000d\u000a25. J. K. Bunn, J. Hu and J. R. Hattrick-Simpers, JOM, 2016, 68, 2116-2125. \u000d\u000a26. F. Oviedo, Z. Ren, S. Sun, C. Settens, Z. Liu, N. T. P. Hartono, S. Ramasamy, B. L. DeCost, \u000d\u000aS. I. P. Tian, G. Romano, A. Gilad Kusne and T. Buonassisi, npj Comput. Mater., 2019, 5, \u000d\u000a60. \u000d\u000a27. K. T. Butler, M. D. Le, J. Thiyagalingam and T. G. Perring, J. Phys.: Condens. Matter, \u000d\u000a2021, 33, 194006. \u000d\u000a28. B. Mahesh, Int. J. Sci. Res., 2020, 9, 381-386. \u000d\u000a29. S. Katoch, S. S. Chauhan and V. Kumar, Multimed. Tools Appl., 2021, 80, 8091-8126. \u000d\u000a30. F. Oviedo, J. L. Ferres, T. Buonassisi and K. T. Butler, Acc. Mater. Res., 2022, 3, 597-607. \u000d\u000a31. OpenAi, arXiv preprint arXiv:2303.08774, 2023, DOI: 10.48550/arXiv.2303.08774. \u000d\u000a32. J. Jumper, R. Evans, A. Pritzel, T. Green, M. Figurnov, O. Ronneberger, K. \u000d\u000aTunyasuvunakool, R. Bates, A. \u017d\u00eddek and A. Potapenko, Nature, 2021, 596, 583-589. \u000d\u000a33. D. P. Kingma and M. Welling, arXiv preprint arXiv:1312.6114, 2013, DOI: \u000d\u000a10.48550/arXiv.1312.6114. \u000d\u000a34. A. Jain, S. P. Ong, G. Hautier, W. Chen, W. D. Richards, S. Dacek, S. Cholia, D. Gunter, D. \u000d\u000aSkinner, G. Ceder and K. A. Persson, APL Mater., 2013, 1, 011002. \u000d\u000a35. C. Zheng, K. Mathew, C. Chen, Y. Chen, H. Tang, A. Dozier, J. J. Kas, F. D. Vila, J. J. \u000d\u000aRehr, L. F. J. Piper, K. A. Persson and S. P. Ong, npj Comput. Mater., 2018, 4, 12. \u000d\u000a36. K. Choudhary, K. F. Garrity, A. C. E. Reid, B. DeCost, A. J. Biacchi, A. R. Hight Walker, \u000d\u000aZ. Trautt, J. Hattrick-Simpers, A. G. Kusne, A. Centrone, A. Davydov, J. Jiang, R. Pachter, \u000d\u000aG. Cheon, E. Reed, A. Agrawal, X. Qian, V. Sharma, H. Zhuang, S. V. Kalinin, B. G. \u000d\u000aSumpter, G. Pilania, P. Acar, S. Mandal, K. Haule, D. Vanderbilt, K. Rabe and F. Tavazza, \u000d\u000anpj Comput. Mater., 2020, 6, 173. \u000d\u000a37. M. R. Carbone, M. Topsakal, D. Lu and S. Yoo, Phys. Rev. Lett., 2020, 124, 156401. \u000d\u000a38. C. D. Rankine, M. M. M. Madkhali and T. J. Penfold, J. Phys. Chem. A, 2020, 124, 4263-\u000d\u000a4270. \u000d\u000a39. C. Rankine and T. Penfold, J. Chem. Phys., 2022, 156, 164102. \u000d\u000a40. J. Timoshenko, D. Lu, Y. Lin and A. I. Frenkel, J. Phys. Chem., 2017, 8, 5091-5098. \u000d\u000a41. J. Timoshenko, S. Roese, H. H\u00f6vel and A. I. Frenkel, Radiat. Phys. Chem., 2020, 175, \u000d\u000a108049. ","42. Y. Liu, N. Marcella, J. Timoshenko, A. Halder, B. Yang, L. Kolipaka, M. J. Pellin, S. \u000d\u000aSeifert, S. Vajda and P. Liu, J. Chem. Phys., 2019, 151, 164201. \u000d\u000a43. S. B. Torrisi, M. R. Carbone, B. A. Rohr, J. H. Montoya, Y. Ha, J. Yano, S. K. Suram and \u000d\u000aL. Hung, npj Comput. Mater., 2020, 6, 109. \u000d\u000a44. M. R. Carbone, S. Yoo, M. Topsakal and D. Lu, Phys. Rev. Mater., 2019, 3, 033604. \u000d\u000a45. M. Ahmadi, J. Timoshenko, F. Behafarid and B. Roldan Cuenya, J. Phys. Chem. C, 2019, \u000d\u000a123, 10666-10676. \u000d\u000a46. J. Timoshenko, H. S. Jeon, I. Sinev, F. T. Haase, A. Herzog and B. R. Cuenya, Chem. Sci., \u000d\u000a2020, 11, 3727-3736. \u000d\u000a47. J. Timoshenko, M. Ahmadi and B. Roldan Cuenya, J. Phys. Chem. C, 2019, 123, 20594-\u000d\u000a20604. \u000d\u000a48. J. Timoshenko, A. Anspoks, A. Cintins, A. Kuzmin, J. Purans and A. I. Frenkel, Phys. Rev. \u000d\u000aLett., 2018, 120, 225502. \u000d\u000a49. J. Timoshenko, C. J. Wrasman, M. Luneau, T. Shirman, M. Cargnello, S. R. Bare, J. \u000d\u000aAizenberg, C. M. Friend and A. I. Frenkel, Nano Lett., 2019, 19, 520-529. \u000d\u000a50. A. C. Scheinost, R. Kretzschmar, S. Pfister and D. R. Roberts, Environ. Sci. Technol., 2002, \u000d\u000a36, 5021-5028. \u000d\u000a51. T. Ressler, J. Wong, J. Roos and I. L. Smith, Environ. Sci. Technol., 2000, 34, 950-958. \u000d\u000a52. S. Wasserman, J. Phys. IV, 1997, 7, C2-203-C202-205. \u000d\u000a53. H. Tanimoto, X. Hongkun, M. Mizumaki, Y. Seno, J. Uchiwada, R. Yamagami, H. \u000d\u000aKumazoe, K. Iwamitsu, Y. Kimura and K. Amezawa, J. Phys. Commun., 2021, 5, 115005. \u000d\u000a54. A. A. Gambardella, M. Cotte, W. de Nolf, K. Schnetz, R. Erdmann, R. van Elsas, V. \u000d\u000aGonzalez, A. Wallert, P. D. Iedema and M. Eveno, Sci. Adv., 2020, 6, eaay8782. \u000d\u000a55. S. Tetef, N. Govind and G. T. Seidler, Phys. Chem. Chem. Phys., 2021, 23, 23586-23601. \u000d\u000a56. T. Li, A. J. Senesi and B. Lee, Chem Rev, 2016, 116, 11128-11180. \u000d\u000a57. B. R. Pauw, G. J. Smales, A. S. Anker, D. M. Balazs, F. L. Beyer, R. Bienert, W. G. \u000d\u000aBouwman, I. Bre\u00dfler, J. Breternitz and E. S. Brok, arXiv preprint arXiv:2303.03772, 2023, \u000d\u000aDOI: 10.48550/arXiv.2303.03772. \u000d\u000a58. D. J. Beltran-Villegas, M. G. Wessels, J. Y. Lee, Y. Song, K. L. Wooley, D. J. Pochan and \u000d\u000aA. Jayaraman, J. Am. Chem. Soc., 2019, 141, 14916-14930. \u000d\u000a59. M. G. Wessels and A. Jayaraman, Macromolecules, 2021, 54, 783-796. \u000d\u000a60. M. G. Wessels and A. Jayaraman, ACS Polym. Au, 2021, 1, 153-164. \u000d\u000a61. C. M. Heil and A. Jayaraman, ACS Mater. Au, 2021, 1, 140-156. \u000d\u000a62. Z. Ye, Z. Wu and A. Jayaraman, JACS Au, 2021, 1, 1925-1936. \u000d\u000a63. C. M. Heil, A. Patil, A. Dhinojwala and A. Jayaraman, ACS Cent. Sci., 2022, 8, 996-1007. \u000d\u000a64. Z. Wu and A. Jayaraman, Macromolecules, 2022, 55, 11076-11091. \u000d\u000a65. C. M. Heil, Y. Ma, B. Bharti and A. Jayaraman, JACS Au, 2023, 3, 889-904. \u000d\u000a66. S. Liu, C. N. Melton, S. Venkatakrishnan, R. J. Pandolfi, G. Freychet, D. Kumar, H. Tang, \u000d\u000aA. Hexemer and D. M. Ushizima, MRS Commun., 2019, 9, 586-592. \u000d\u000a67. C. Do, W.-R. Chen and S. Lee, MRS Adv., 2020, 5, 1577-1584. \u000d\u000a68. H. Ikemoto, K. Yamamoto, H. Touyama, D. Yamashita, M. Nakamura and H. Okuda, J. \u000d\u000aSyncrotron Radiat., 2020, 27, 1069-1073. \u000d\u000a69. G. Song, L. Porcar, M. Boehm, F. Cecillon, C. Dewhurst, Y. Le Goc, J. Locatelli, P. Mutti \u000d\u000aand T. Weber, EPJ Web Conf., 2020, 225, 01004. \u000d\u000a70. R. K. Archibald, M. Doucet, T. Johnston, S. R. Young, E. Yang and W. T. Heller, J. Appl. \u000d\u000aCryst., 2020, 53, 326-334. \u000d\u000a71. http://www.sasview.org/). ","72. P. Tomaszewski, S. Yu, M. Borg and J. R\u00f6nnols, 2021 Swedish Workshop on Data Science \u000d\u000a(SweDS), IEEE, 2021, 1-6. \u000d\u000a73. https://huggingface.co/spaces/AndySAnker/SCattering_Ai_aNalysis \u000d\u000a74. A. Ziletti, D. Kumar, M. Scheffler and L. M. Ghiringhelli, Nat. Commun., 2018, 9, 2775. \u000d\u000a75. W. B. Park, J. Chung, J. Jung, K. Sohn, S. P. Singh, M. Pyo, N. Shin and K.-S. Sohn, \u000d\u000aIUCrJ, 2017, 4, 486-494. \u000d\u000a76. B. D. Lee, J.-W. Lee, W. B. Park, J. Park, M.-Y. Cho, S. Pal Singh, M. Pyo and K.-S. Sohn, \u000d\u000aAdv. Intell. Syst., 2022, 4, 2200042. \u000d\u000a77. Y. Suzuki, H. Hino, T. Hawai, K. Saito, M. Kotsugi and K. Ono, Sci. Rep., 2020, 10, 21790. \u000d\u000a78. C. Garcia-Cardona, R. Kannan, T. Johnston, T. Proffen, K. Page and S. K. Seal, 2019 IEEE \u000d\u000aInternational Conference on Big Data (Big Data), IEEE, 2019. \u000d\u000a79. J.-W. Lee, W. B. Park, J. H. Lee, S. P. Singh and K.-S. Sohn, Nat. Commun., 2020, 11, 86. \u000d\u000a80. C. J. Long, D. Bunker, X. Li, V. L. Karen and I. Takeuchi, Rev. Sci. Instrum., 2009, 80, \u000d\u000a103902. \u000d\u000a81. D. Chernyshov, I. Dovgaliuk, V. Dyadkin and W. van Beek, Crystals, 2020, 10, 581. \u000d\u000a82. V. Stanev, V. V. Vesselinov, A. G. Kusne, G. Antoszewski, I. Takeuchi and B. S. \u000d\u000aAlexandrov, npj Comput. Mater., 2018, 4, 43. \u000d\u000a83. D. Chen, Y. Bai, S. Ament, W. Zhao, D. Guevarra, L. Zhou, B. Selman, R. B. van Dover, J. \u000d\u000aM. Gregoire and C. P. Gomes, Nat. Mach. Intell., 2021, 3, 812-822. \u000d\u000a84. T. L. Christiansen, S. R. Cooper and K. M. \u00d8. Jensen, Nanoscale Adv., 2020, 2, 2234-2254. \u000d\u000a85. S. J. L. Billinge and I. Levin, Science, 2007, 316, 561-565. \u000d\u000a86. C.-H. Liu, Y. Tao, D. Hsu, Q. Du and S. J. L. Billinge, Acta Crystallogr. A, 2019, 75, 633-\u000d\u000a643. \u000d\u000a87. https://huggingface.co/spaces/AndySAnker/POMFinder. \u000d\u000a88. A. S. Anker, E. T. Kj\u00e6r, M. Juelsholt and K. M. \u00d8. Jensen, 2023, DOI: 10.26434/chemrxiv2023-91xz7. \u000d\u000a89. A. S. Anker, E. T. S. Kj\u00e6r, M. Juelsholt, T. L. Christiansen, S. L. Skj\u00e6rv\u00f8, M. R. V. \u000d\u000aJ\u00f8rgensen, I. Kantor, D. R. S\u00f8rensen, S. J. L. Billinge, R. Selvan and K. M. \u00d8. Jensen, npj \u000d\u000aComput. Mater., 2022, 8, 213. \u000d\u000a90. S. M. Lundberg, G. Erion, H. Chen, A. DeGrave, J. M. Prutkin, B. Nair, R. Katz, J. \u000d\u000aHimmelfarb, N. Bansal and S.-I. Lee, Nat. Mach. Intell., 2020, 2, 56-67. \u000d\u000a91. S. M. Lundberg and S.-I. Lee, Proceedings of the 31st International Conference on Neural \u000d\u000aInformation Processing Systems, 2017, DOI: 10.48550/arXiv.1705.07874, 4765-4774. \u000d\u000a92. S. L. Skj\u00e6rv\u00f8, A. S. Anker, M. C. Wied, E. Kj\u00e6r, M. Juelsholt, T. L. Christiansen and K. M. \u000d\u000a\u00d8. Jensen, Chem. Sci., 2023, 14, 4806-4816. \u000d\u000a93. N. P. L. Magnard, A. S. Anker, O. Aalling-Frederiksen, A. Kirsch and K. M. \u00d8. Jensen, \u000d\u000aDalton Trans., 2022, 51, 17150-17161. \u000d\u000a94. A. S. Anker, E. T. S. Kj\u00e6r, E. B. Dam, S. J. L. Billinge, K. M. \u00d8. Jensen and R. Selvan, \u000d\u000aProceedings of the 16th International Workshop on Mining and Learning with Graphs \u000d\u000a(MLG), 2020. \u000d\u000a95. E. T. S. Kj\u00e6r, A. S. Anker, M. N. Weng, S. J. L. Billinge, R. Selvan and K. M. \u00d8. Jensen, \u000d\u000aDigital Discovery, 2023, 2, 69-80. \u000d\u000a96. K. M. \u00d8. Jensen, P. Juhas, M. A. Tofanelli, C. L. Heinecke, G. Vaughan, C. J. Ackerson and \u000d\u000aS. J. L. Billinge, Nat. Commun., 2016, 7, 11859. \u000d\u000a97. J. Quinson, L. Kacenauskaite, T. L. Christiansen, T. Vosch, M. Arenz and K. M. \u00d8. Jensen, \u000d\u000aACS Omega, 2018, 3, 10351-10356. \u000d\u000a98. L. D. Marks, Philos. Mag. A, 1984, 49, 81-93. ","99. S. Banerjee, C.-H. Liu, J. D. Lee, A. Kovyakh, V. Grasmik, O. Prymak, C. Koenigsmann, \u000d\u000aH. Liu, L. Wang, A. M. M. Abeykoon, S. S. Wong, M. Epple, C. B. Murray and S. J. L. \u000d\u000aBillinge, J. Phys. Chem. C, 2018, 122, 29498-29506. \u000d\u000a100. https://huggingface.co/spaces/AndySAnker/DeepStruc. \u000d\u000a101. S. Chen, A. Y. Sheikh and R. Ho, J. Pharm. Sci., 2014, 103, 3879-3890. \u000d\u000a102. N. Chieng, H. Trnka, J. Boetker, M. Pikal, J. Rantanen and H. Grohganz, Int. J. Pharm., \u000d\u000a2013, 454, 167-173. \u000d\u000a103. K. W. Chapman, S. H. Lapidus and P. J. Chupas, J. Appl. Cryst., 2015, 48, 1619-1626. \u000d\u000a104. X. Hua, A. S. Eggeman, E. Castillo-Mart\u00ednez, R. Robert, H. S. Geddes, Z. Lu, C. J. Pickard, \u000d\u000aW. Meng, K. M. Wiaderek, N. Pereira, G. G. Amatucci, P. A. Midgley, K. W. Chapman, U. \u000d\u000aSteiner, A. L. Goodwin and C. P. Grey, Nat. Mater., 2021, 20, 841-850. \u000d\u000a105. H. S. Geddes, H. D. Hutchinson, A. R. Ha, N. P. Funnell and A. L. Goodwin, Nanoscale, \u000d\u000a2021, 13, 13220-13224. \u000d\u000a106. X. Hua, P. K. Allan, H. S. Geddes, E. Castillo-Mart\u00ednez, P. A. Chater, T. S. Dean, A. \u000d\u000aMinelli, P. G. Bruce and A. L. Goodwin, Cell Rep. Phys. Sci., 2021, 2, 100543. \u000d\u000a107. A. Herlihy, H. S. Geddes, G. C. Sosso, C. L. Bull, C. J. Ridley, A. L. Goodwin, M. S. Senn \u000d\u000aand N. P. Funnell, J. Appl. Cryst., 2021, 54, 1546-1554. \u000d\u000a108. H. S. Geddes, H. Blade, J. F. McCabe, L. P. Hughes and A. L. Goodwin, Chem. Commun., \u000d\u000a2019, 55, 13346-13349. \u000d\u000a109. R. Gu, S. J. Billinge and Q. Du, Acta Crystallogr. A, 2023, 79. \u000d\u000a110. C.-H. Liu, C. J. Wright, R. Gu, S. Bandi, A. Wustrow, P. K. Todd, D. O'Nolan, M. L. \u000d\u000aBeauvais, J. R. Neilson, P. J. Chupas, K. W. Chapman and S. J. L. Billinge, J. Appl. Cryst., \u000d\u000a2021, 54, 768-775. \u000d\u000a111. Z. Thatcher, C.-H. Liu, L. Yang, B. C. McBride, G. Thinh Tran, A. Wustrow, M. A. \u000d\u000aKarlsen, J. R. Neilson, D. B. Ravnsbaek and S. J. L. Billinge, Acta Crystallogr. A, 2022, 78, \u000d\u000a242-248. \u000d\u000a112. https://PDFitc.org. \u000d\u000a113. J. B. Goodenough, Phys. Rev., 1955, 100, 564-573. \u000d\u000a114. J. Kanamori, J. Phys. Chem. Solids, 1959, 10, 87-98. \u000d\u000a115. A. Daoud-Aladine, J. Rodr\u00edguez-Carvajal, L. Pinsard-Gaudart, M. T. Fern\u00e1ndez-D\u00edaz and A. \u000d\u000aRevcolevschi, Phys. Rev. Lett., 2002, 89, 097205. \u000d\u000a116. G. E. Johnstone, T. G. Perring, O. Sikora, D. Prabhakaran and A. T. Boothroyd, Phys. Rev. \u000d\u000aLett., 2012, 109, 237202. \u000d\u000a117. D. W. Hosmer Jr, S. Lemeshow and R. X. Sturdivant, Applied logistic regression, John \u000d\u000aWiley & Sons, 2013. \u000d\u000a118. I. Goodfellow, J. Pouget-Abadie, M. Mirza, B. Xu, D. Warde-Farley, S. Ozair, A. Courville \u000d\u000aand Y. Bengio, Advances in neural information processing systems, 2014, 27. \u000d\u000a119. A. S. Anker, K. T. Butler, M. D. Le, T. G. Perring and J. Thiyagalingam, Digital Discovery, \u000d\u000a2023, 2, 578-590. \u000d\u000a120. A. M. Samarakoon, K. Barros, Y. W. Li, M. Eisenbach, Q. Zhang, F. Ye, V. Sharma, Z. L. \u000d\u000aDun, H. Zhou, S. A. Grigera, C. D. Batista and D. A. Tennant, Nat. Commun., 2020, 11, \u000d\u000a892. \u000d\u000a121. A. M. Samarakoon, P. Laurell, C. Balz, A. Banerjee, P. Lampen-Kelley, D. Mandrus, S. E. \u000d\u000aNagler, S. Okamoto and D. A. Tennant, Phys. Rev. Res., 2022, 4, L022061. \u000d\u000a122. A. M. Samarakoon and D. A. Tennant, J. Phys.: Condens. Matter, 2021, 34, 044002. \u000d\u000a123. A. Samarakoon, D. A. Tennant, F. Ye, Q. Zhang and S. A. Grigera, Commun. Mater., 2022, \u000d\u000a3, 84. \u000d\u000a124. M. Raissi, P. Perdikaris and G. E. Karniadakis, J. Comput. Phys., 2019, 378, 686-707. ","125. S.-O. Kaba and S. Ravanbakhsh, Advances in Neural Information Processing Systems, \u000d\u000a2022, DOI: 10.48550/arXiv.2211.15420. \u000d\u000a126. N. Artrith, K. T. Butler, F.-X. Coudert, S. Han, O. Isayev, A. Jain and A. Walsh, Nat. \u000d\u000aChem., 2021, 13, 505-508. \u000d\u000a127. M. W. Terban and S. J. L. Billinge, Chem. Rev., 2022, 122, 1208-1272. \u000d\u000a128. A. S. Anker, T. L. Christiansen, M. Weber, M. Schmiele, E. Brok, E. T. S. Kj\u00e6r, P. Juh\u00e1s, \u000d\u000aR. Thomas, M. Mehring and K. M. \u00d8. Jensen, Angew. Chem. Int. Ed., 2021, 60, 2-12. \u000d\u000a129. A. Mannodi-Kanakkithodi, G. Pilania, R. Ramprasad, T. Lookman and J. E. Gubernatis, \u000d\u000aComput. Mater. Sci., 2016, 125, 92-99. \u000d\u000a130. B. P. MacLeod, F. G. Parlane, C. C. Rupnow, K. E. Dettelbach, M. S. Elliott, T. D. \u000d\u000aMorrissey, T. H. Haley, O. Proskurin, M. B. Rooney and N. Taherimakhsousi, Nat. \u000d\u000aCommun., 2022, 13, 995. \u000d\u000a131. B. Burger, P. M. Maffettone, V. V. Gusev, C. M. Aitchison, Y. Bai, X. Wang, X. Li, B. M. \u000d\u000aAlston, B. Li, R. Clowes, N. Rankin, B. Harris, R. S. Sprick and A. I. Cooper, Nature, 2020, \u000d\u000a583, 237-241. \u000d\u000a132. S. Steiner, J. Wolf, S. Glatzel, A. Andreou, J. M. Granda, G. Keenan, T. Hinkley, G. \u000d\u000aAragon-Camarasa, P. J. Kitson, D. Angelone and L. Cronin, Science, 2019, 363, eaav2211. \u000d\u000a133. Y. Jiang, D. Salley, A. Sharma, G. Keenan, M. Mullin and L. Cronin, Sci. Adv., 2022, 8, \u000d\u000aeabo2626. \u000d\u000a134. H. Tao, T. Wu, S. Kheiri, M. Aldeghi, A. Aspuru-Guzik and E. Kumacheva, Adv. Funct. \u000d\u000aMater., 2021, 31, 2106725. "," \u000d\u000a45\u000d\u000a2.2 Pushing\u0009the\u0009Boundaries\u0009of\u0009Conventional\u0009Scattering\u0009Data\u0009Analysis*\u000d\u000aAn\u0009alternative method\u0009for\u0009identifying\u0009structural\u0009models\u0009involves automating the\u0009screening\u0009process\u0009of\u0009a\u0009\u000d\u000avast\u0009number\u0009of\u0009structures. For\u0009PDF\u0009data,\u0009researchers\u0009have\u0009developed\u0009automated\u0009brute-force\u0009methods\u0009\u000d\u000athat automatically\u0009compare\u0009experimental\u0009data\u0009with simulated\u0009data\u0009from\u0009all\u0009structures\u0009in databases\u0009like\u000d\u000aICSD\u0009and\u0009COD.\u0009One\u0009way\u0009to\u0009compare\u0009the\u0009data\u0009is\u0009by\u0009calculating\u0009the\u0009Pearson\u0009Correlation\u0009Coefficient\u0009(PCC)\u000d\u000adirectly\u0009 on\u0009 the\u0009 experimental\u0009 and\u0009 simulated\u0009 data.\u000d\u000a36 The\u0009 PCC\u0009 is\u0009 relatively\u0009 fast\u0009 (minutes for\u0009 1000\u000d\u000astructures)\u0009but\u0009not\u0009robust to a\u0009change\u0009in simulation\u0009parameters\u0009such\u0009as\u0009vibrations,\u0009lattice\u0009parameters,\u0009\u000d\u000aand\u0009 atomic\u0009 positions.\u0009 Alternatively,\u0009 \u2018structure-mining\u201947 and\u0009 \u2018cluster-mining\u201948 methods\u0009 have\u0009 been\u0009\u000d\u000aintroduced\u0009 to automatically perform\u0009least-squares\u0009 refinement of\u0009 the\u0009experimental\u0009data\u0009 to\u0009 simulated\u0009\u000d\u000adata\u0009from\u0009all\u0009structures\u0009in\u0009the databases. Structure-mining\u0009assumes\u0009periodic\u0009boundary\u0009conditions\u0009in\u0009the\u0009\u000d\u000astructure\u0009refinement\u0009process\u0009and\u0009is\u0009therefore\u0009optimised\u0009for\u0009crystalline\u0009materials\u0009while\u0009cluster-mining\u0009\u000d\u000auses\u0009the\u0009Debye\u0009scattering\u0009equation\u0009in\u0009the\u0009structure\u0009refinement\u0009process\u0009and\u0009is\u0009therefore\u0009better\u0009suited\u0009for\u0009\u000d\u000asmaller\u0009non-periodic\u0009structures\u0009such\u0009as\u0009clusters.\u0009While\u0009least-squares\u0009refinements are slower\u0009(hours\u0009for\u000d\u000a1000\u0009structures) than\u0009using\u0009the\u0009PCC,\u0009it is\u0009more\u0009robust to\u0009changes\u0009in\u0009structural\u0009parameters as\u0009they\u0009are\u000d\u000aconcurrently determined.\u0009\u000d\u000aNonetheless,\u0009these\u0009brute-force\u0009methods are\u0009restrained\u0009by\u0009the\u0009availability of\u0009an\u0009appropriate database\u0009of\u0009\u000d\u000astructures\u0009 or\u0009 a structure-generating algorithm.\u0009 A\u0009 study\u0009 on\u0009 molybdenum\u0009 oxide\u0009 nanomaterials\u000d\u000aintroduced\u0009 another\u0009 automated\u0009 approach,\u0009 here\u0009 called\u0009 a\u0009 \u2018motif-extractor\u2019\u0009 approach.\u000d\u000a49 This\u0009 approach\u0009\u000d\u000aidentified structural\u0009motifs\u0009by\u0009fitting\u0009automatically\u0009generated\u0009fragments\u0009of\u0009candidate\u0009crystal\u0009structures\u0009\u000d\u000ato\u0009 PDFs.\u0009 These\u0009 candidate\u0009 crystal\u0009 structures\u0009 were\u0009 identified\u0009 from\u0009 a\u0009 crystallographic\u0009 database using\u0009\u000d\u000achemical\u0009knowledge. While\u0009this\u0009method\u0009is\u0009highly\u0009computationally expensive\u0009(~1\u0009week\u0009for\u0009a\u0009structure\u0009\u000d\u000awith\u000920\u0009atoms,\u0009which doubles\u0009per\u0009extra\u0009atom), it\u0009covers all\u0009structural\u0009motifs\u0009contained\u0009in\u0009the\u0009candidate\u0009\u000d\u000acrystal\u0009structures.\u0009\u0009\u000d\u000aIn\u0009Paper\u0009III,\u000d\u000a24 we\u0009performed\u0009in\u0009situ\u0009X-ray\u0009TS\u0009and\u0009SAXS\u0009experiments\u0009at various temperatures\u0009(30\u201380 \u00b0C)\u0009\u000d\u000aon\u0009[Bi6O5(OH)3(NO3)5]\u22c53H2O50 in\u0009dimethyl\u0009sulfoxide\u0009(DMSO),\u0009with\u0009a\u0009time\u0009resolution\u0009of\u00094\u0009seconds.\u0009This\u0009\u000d\u000aenabled us\u0009to\u0009investigate the\u0009formation\u0009of\u0009the\u0009[Bi38O45]\u0009cluster\u0009starting\u0009from\u0009[Bi6O5(OH)3(NO3)5]\u22c53H2O\u0009\u000d\u000ain\u0009 DMSO using\u0009 PDF\u0009 and\u0009 SAXS.\u0009 We\u0009 obtained detailed\u0009 information\u0009 on\u0009 the\u0009 reaction\u0009 pathway,\u0009 from\u0009\u000d\u000aoctahedral\u0009[Bi6O8]\u0009units\u0009 to\u0009[Bi38O45]\u0009nanoclusters as\u0009illustrated in\u0009Figure\u00092.2.\u0009To\u0009analyse\u0009 the\u0009data,\u0009we\u0009\u000d\u000ainitially applied conventional\u0009PDF\u0009modelling,\u0009with\u0009 the\u0009Debye\u0009scattering\u0009equation,\u0009 to\u0009characterise\u0009 the\u0009\u000d\u000astructure\u0009of\u0009the\u0009[Bi6O5(OH)3(NO3)5]\u22c53H2O\u0009crystal\u0009and\u0009the\u0009[Bi38O45]\u0009cluster.\u0009To\u0009identify\u0009intermediates\u0009and\u0009\u000d\u000athe\u0009reaction\u0009pathway,\u0009we\u0009developed\u0009an\u0009extension\u0009to\u0009the\u0009previously\u0009mentioned motif-extractor.\u0009In\u0009our\u000d\u000aautomated\u0009 approach,\u0009 all\u0009 relevant\u0009 smaller\u0009 cluster\u0009 structures,\u0009 based\u0009 on\u0009 the\u0009[Bi38O45]\u0009 framework,\u0009 are\u000d\u000atested\u0009 against\u0009 the\u0009 experimental\u0009 PDF,\u0009 enabling us\u0009 to\u0009 efficiently identify\u0009 size\u0009 and\u0009 structure\u0009 evolution\u0009\u000d\u000aduring\u0009 the\u0009 reaction.\u0009 Our\u0009 findings revealed [Bi22Oy]\u0009 as\u0009 a\u0009 metastable\u0009 intermediate\u0009 structure\u0009 in\u0009 the\u0009\u000d\u000areaction.\u0009However,\u0009our\u0009results\u0009were\u0009ambiguous\u0009as\u0009the\u0009PDF\u0009data\u0009could\u0009also\u0009be\u0009described\u0009by\u0009the\u0009[Bi18O36]\u0009\u000d\u000acluster\u0009as\u0009shown\u0009in Figure\u00091.3.\u0009The PDF\u0009data alone was thereby\u0009insufficient\u0009for a\u0009definite\u0009determination\u0009\u000d\u000aof the\u0009structure.\u0009Thus,\u0009we\u0009did\u0009a\u0009similar\u0009SAXS\u0009experiment\u0009and\u0009combined\u0009PDF\u0009and\u0009SAXS\u0009modelling\u0009(also\u000d\u000acalled\u0009\u2018complex\u0009modelling\u2019),\u0009which excluded\u0009the\u0009[Bi18O36]\u0009cluster.\u0009In\u0009the\u0009process\u0009of\u0009complex\u0009modeling,\u0009\u000d\u000awe\u0009 encountered\u0009 a\u0009 common\u0009 problem:\u0009 balancing\u0009 dataset\u0009 contributions\u0009 during\u0009 structure\u0009\u000d\u000arefinement.\u000d\u000a13,24,51,52 We\u0009discovered that\u0009 a\u0009 possible\u0009 solution\u0009 can\u0009 be\u0009 to\u0009 fine-tune\u0009 the\u0009 weighting\u0009during\u0009\u000d\u000aleast-square\u0009refinement\u0009such\u0009that\u0009they\u0009satisfy both\u0009the\u0009PDF\u0009and\u0009SAXS\u0009data.\u0009\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009III.\u0009","|\u0009Conventional\u0009Scattering\u0009Data\u0009Modelling\u000d\u000a46\u000d\u000aIn\u0009conclusion,\u0009the\u0009combination\u0009of\u0009PDF\u0009and\u0009SAXS\u0009thus\u0009provided\u0009an\u0009insight\u0009into\u0009the\u0009cluster\u0009chemistry\u0009in\u0009\u000d\u000asolution\u0009that\u0009the\u0009techniques\u0009could\u0009not\u0009provide\u0009alone.\u0009The\u0009methods\u0009and\u0009modelling\u0009tools\u0009developed\u0009in\u0009this\u0009\u000d\u000astudy,\u0009 using\u0009 automated\u0009 and\u0009 combined\u0009 SAXS\u0009 and\u0009 PDF\u0009 analysis,\u0009 thus\u0009 pushed\u0009 the\u0009 boundaries\u0009 of\u0009\u000d\u000aconventional\u0009scattering\u0009modelling.\u0009\u000d\u000aFigure\u0009 2.2:\u0009 The\u0009 development\u0009 of\u0009 tools\u0009 for\u0009 combined,\u0009 automated\u0009 analysis\u0009 of\u0009 PDF\u0009 and\u0009 SAXS\u0009 data\u0009 enabled\u0009 the\u0009\u000d\u000aidentification\u0009 of\u0009 a\u0009 [Bi22O26]\u0009 cluster\u0009 as\u0009 an\u0009 intermediate\u0009 cluster in\u0009 the\u0009 formation\u0009 of\u0009 atomically\u0009 precise\u0009 [Bi38O45]\u0009\u000d\u000ananoclusters\u0009from\u0009 [Bi6O5(OH)3(NO3)5]\u22c53H2O\u0009 crystals\u0009 dissolved\u0009in\u0009DMSO. Figure\u0009 and\u0009 Figure\u0009 caption\u0009 are adapted\u0009\u000d\u000afrom\u0009Paper\u0009III.24\u000d\u000a2.2.1 Paper\u0009III\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009Paper\u0009III\u0009which\u0009is\u0009published\u0009in\u0009Angewandte\u0009Chemie\u0009International\u0009\u000d\u000aEdition.\u0009The\u0009supplementary\u0009information\u0009for\u0009Paper\u0009III\u0009can\u0009be\u0009found\u0009in\u0009APPENDICES\u0009\u2013\u0009PAPERS.","Chemie\u000d\u000awww.angewandte.org\u000d\u000a2021\u201360/37\u000d\u000aInternational Edition\u000d\u000aA Journal of the German Chemical Society Angewandte\u000d\u000a.\u000d\u000aBy combining X-ray \u2026\u000d\u000a\u2026 total scattering and small-angle X-ray scattering, it is possible to obtain unique insight\u000d\u000ainto the formation of atomically precise nanoclusters. In a study of bismuth oxide\u000d\u000aclusters on page 20407, Michael Mehring, Kirsten M. \u00d8. Jensen et al. investigate the\u000d\u000aformation of [Bi38O45] clusters from [Bi6O8], and show that an intermediate structure of\u000d\u000a[Bi22O26] plays an important role in the process. This intermediate structure was\u000d\u000aidentified through a new automated modelling approach.","Cluster Compounds Very Important Paper\u000d\u000aStructural Changes during the Growth of Atomically Precise Metal\u000d\u000aOxido Nanoclusters from Combined Pair Distribution Function and\u000d\u000aSmall-Angle X-ray Scattering Analysis\u000d\u000aAndy S. Anker, Troels Lindahl Christiansen, Marcus Weber, Martin Schmiele, Erik Brok,\u000d\u000aEmil T. S. Kj\u00e6r, Pavol Juh\u00b5s, Rico Thomas, Michael Mehring,* and Kirsten M. \u00d8. Jensen*\u000d\u000aAbstract: The combination of in situ pair distribution function\u000d\u000a(PDF) analysis and small-angle X-ray scattering (SAXS)\u000d\u000aenables analysis of the formation mechanism of metal oxido\u000d\u000ananoclusters and cluster\u2013solvent interactions as they take place.\u000d\u000aHerein, we demonstrate the method for the formation of\u000d\u000aclusters with a [Bi38O45] core. Upon dissolution of crystalline\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O in DMSO, an intermediate rapidly\u000d\u000aforms, which slowly grows to stable [Bi38O45] clusters. To\u000d\u000aidentify the intermediate, we developed an automated modeling method, where smaller [BixOy] structures based on the\u000d\u000a[Bi38O45] framework are tested against the data. [Bi22O26] was\u000d\u000aidentified as the main intermediate species, illustrating how\u000d\u000acombined PDF and SAXS analysis is a powerful tool to gain\u000d\u000ainsight into nucleation on an atomic scale. PDF also provides\u000d\u000ainformation on the interaction between nanoclusters and\u000d\u000asolvent, which is shown to depend on the nature of the ligands\u000d\u000aon the cluster surface.\u000d\u000aIntroduction\u000d\u000aThe design of new functional materials relies on understanding the fundamental chemical reactions that govern\u000d\u000amaterial formation and growth. In inorganic and materials\u000d\u000achemistry, we are still challenged in describing these processes\u000d\u000aon an atomic level,[1] as studies of nucleation and growth\u000d\u000aphenomena are challenging. In recent years, atomically\u000d\u000aprecise metal oxido nanoclusters of titanium,[2] cerium[3] and\u000d\u000abismuth have been intensively studied due to their wide range\u000d\u000aof applications, for example, for photocatalysis,[2] oxygen\u000d\u000astorage,[3] in medicine,[4] as radiopaque materials[4b, 5] or as\u000d\u000abuilding blocks for advanced catalysts.[6] Apart from their\u000d\u000atechnological relevance understanding the solution chemistry\u000d\u000aof such metal oxido clusters on an atomic and molecular scale\u000d\u000acan open new opportunities for synthesizing nanoscale metal\u000d\u000aoxides in a controlled manner.[6, 7]\u000d\u000aWe have in recent years focused on the chemistry of\u000d\u000abismuth oxido clusters and have synthesized a range of\u000d\u000adifferently sized molecules and nanoclusters.[4a, 8] These nanoclusters are generally composed of a bismuth oxido core\u000d\u000a[BixOy]\u000d\u000az+ of varying nuclearity and charge, and anionic ligands\u000d\u000a(e.g. nitrate, carboxylates, aryl sulfonates, silanolates) for\u000d\u000acharge compensation. In the following, we will refer to the\u000d\u000astructures of the nanoclusters as [BixOy] neglecting charge\u000d\u000aand ligand shell. The nanoclusters can be considered built\u000d\u000afrom [Bi6O8] units which share edges (as defined from the\u000d\u000aoctahedra formed from Bi atoms, Figure 1A,B) to form larger\u000d\u000aclusters. These clusters represent cut-outs of the fluorite-type\u000d\u000astructure which constitutes the basis of d- and b-Bi2O3.\u000d\u000a[6, 9] For\u000d\u000aexample, the [Bi22O26] cluster[8e] (Figure 1 C) is composed of 6\u000d\u000aof these basic motifs and the [Bi38O45] cluster[4a] (Figure 1D)\u000d\u000aof 13. Similar structures are seen in metal oxido clusters of\u000d\u000aother large cations such as [Ce22O24],[10] [Ce38O54],[3a]\u000d\u000a[U38O56],[11] and [Pu38O56].[12] The [M6Ox] building block,\u000d\u000awhere M is a metal, is therefore anticipated to play an\u000d\u000aimportant role in the cluster growth, however, an understanding of the mechanisms controlling this process in\u000d\u000asolution is still missing. In situ investigations have shed light\u000d\u000aon cluster reactions in solution, and the cluster growth from\u000d\u000a[Bi6O8] to [Bi38O45] has previously been investigated with\u000d\u000aelectrospray ionization mass spectrometry (ESI-MS).[13] A\u000d\u000avariety of bismuth oxido clusters were observed during the\u000d\u000areaction, but [Bi6O8], [Bi22O27] and [Bi38O45] clusters were\u000d\u000a[*] A. S. Anker, Dr. T. L. Christiansen, E. T. S. Kj\u00e6r, Dr. K. M. \u00d8. Jensen\u000d\u000aDepartment of Chemistry and Nano-Science Center\u000d\u000aUniversity of Copenhagen\u000d\u000aUniversitetsparken 5, 2100 Copenhagen E (Denmark)\u000d\u000aE-mail: kirsten@chem.ku.dk\u000d\u000aDr. M. Weber, R. Thomas, Prof. Dr. M. Mehring\u000d\u000aFakult\u2030t f\u00b8r Naturwissenschaften, Institut f\u00b8r Chemie\u000d\u000aProfessur Koordinationschemie, Technische Universit\u2030t Chemnitz\u000d\u000aStrasse der Nationen 62, 09111 Chemnitz (Germany)\u000d\u000aand\u000d\u000aCenter for Materials, Architectures and Integration of\u000d\u000aNanomembranes (MAIN)\u000d\u000aRosenbergstrasse 6, 09126 Chemnitz (Germany)\u000d\u000aE-mail: michael.mehring@chemie.tu-chemnitz.de\u000d\u000aDr. M. Schmiele, Dr. E. Brok\u000d\u000aNiels Bohr Institute and Nano-Science Center\u000d\u000aUniversity of Copenhagen\u000d\u000aUniversitetsparken 5, 2100 Copenhagen E (Denmark)\u000d\u000aDr. P. Juh\u00b5s\u000d\u000aComputational Science Initiative, Brookhaven National Laboratory\u000d\u000a98 Rochester Street, Upton, NY 11973 (USA)\u000d\u000aSupporting information and the ORCID identification number(s) for\u000d\u000athe author(s) of this article can be found under:\u000d\u000ahttps://doi.org/10.1002/anie.202103641.\u000d\u000a\u232b 2021 The Authors. Angewandte Chemie International Edition\u000d\u000apublished by Wiley-VCH GmbH. This is an open access article under\u000d\u000athe terms of the Creative Commons Attribution Non-Commercial\u000d\u000aLicense, which permits use, distribution and reproduction in any\u000d\u000amedium, provided the original work is properly cited and is not used\u000d\u000afor commercial purposes.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000aHow to cite: Angew. Chem. Int. Ed. 2021, 60, 20407\u2013 20416\u000d\u000aInternational Edition: doi.org/10.1002/anie.202103641\u000d\u000aGerman Edition: doi.org/10.1002/ange.202103641\u000d\u000aAngew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH 20407","identified as particularly stable, which agrees well with other\u000d\u000acluster studies.[3a, 9a, 11, 12, 14] From these experiments, Sattler\u000d\u000aet al.[13] proposed that cluster growth happens through di- or\u000d\u000aoligomerization reactions and rearrangement processes, with\u000d\u000acluster growth starting from hexanuclear metal oxido clusters.\u000d\u000aBased on the ESI-MS findings, as well as on knowledge on\u000d\u000acrystal structures of polynuclear clusters (containing 6\u201310 Bi\u000d\u000aatoms), the reaction is expected to proceed through intermediate species formed by addition of bismuth containing\u000d\u000afragments of dissociated [Bi6O8] units to the cluster core.[15]\u000d\u000aHowever, while mass spectrometry gives information about\u000d\u000athe atomic composition of the species present during growth,\u000d\u000athe evolution of the atomic structure in solution is yet to be\u000d\u000aunraveled.\u000d\u000aThe molecular structures of many bismuth oxido nanoclusters are well known, as these can be characterized with\u000d\u000asingle crystal diffraction. The structures of the clusters shown\u000d\u000ain Figure 1 have all been determined from single crystal X-ray\u000d\u000adiffraction studies. However, characterizing cluster structures\u000d\u000adirectly in solution, as is needed for in situ studies, is more\u000d\u000achallenging. Extended X-ray absorption fine structure\u000d\u000a(EXAFS) has been used to characterize the atomic structure\u000d\u000aof some small clusters[14a, 16] and limited structural information\u000d\u000acan be obtained from Nuclear Magnetic Resonance (NMR)\u000d\u000aspectroscopy.[16b] However, these methods only provide\u000d\u000ainformation on the very local atomic range. Here, we use in\u000d\u000asitu X-ray total scattering (TS) and pair distribution function\u000d\u000aanalysis[17] (PDF) to follow the formation of [Bi38O45] from\u000d\u000a[Bi6O8] as a model reaction for nanocluster formation and\u000d\u000agrowth. As the Fourier transform of the TS signal, the PDF\u000d\u000arepresents a histogram of all interatomic distances in a sample,\u000d\u000aand thus provide atomic structural information from the local\u000d\u000ato the global range. PDF is an excellent technique for in situ\u000d\u000astudies of materials formation in solution,[18] and has been\u000d\u000aapplied to study a few large cluster systems previously[12a 16b] in\u000d\u000aorder to characterize their solution structure, but not in situ to\u000d\u000afollow cluster reactions. While PDF analysis offers information on the atomic structure of matter, it is less sensitive to for\u000d\u000aexample, cluster or particle size and shape. A way to\u000d\u000acomplement the structural information available in the PDF\u000d\u000ais to combine the technique with small-angle X-ray scattering\u000d\u000a(SAXS), which provides information on the size, morphology\u000d\u000aand size dispersion of clusters or particles.[19]\u000d\u000aWe here use PDF and SAXS to study the formation of the\u000d\u000a[Bi38O45] cluster starting from [Bi6O5(OH)3(NO3)5]\u00b73H2O[8a]\u000d\u000ain dimethyl sulfoxide (DMSO) and obtain detailed information on the reaction pathway from octahedral [Bi6O8] units to\u000d\u000a[Bi38O45] nanoclusters. To analyze the data, we have developed an automated modeling approach[20] to identify intermediates and the reaction pathway. This automated approach,\u000d\u000awhere all relevant smaller cluster structures, based on the\u000d\u000a[Bi38O45] framework, is tested against the experimental PDF,\u000d\u000aallows us to readily identify the size and structure evolution\u000d\u000aduring the reaction. Our results clearly identify [Bi22Oy] as\u000d\u000aa metastable intermediate structure in the reaction, and we do\u000d\u000anot observe [Bi6O8] units as a reaction intermediate on the\u000d\u000atime scale of our experiments. We also demonstrate how PDF\u000d\u000acan be used to characterize restructuring effects of the solvent\u000d\u000aat the surface of nanoclusters and show that the cluster\u2013\u000d\u000asolvent interaction affects the [Bi38O45] clusters themselves, as\u000d\u000ait introduces disorder in their structure. This effect is highly\u000d\u000adependent on the presence and type of the ligand covering the\u000d\u000aclusters. The methods and modeling tools developed here,\u000d\u000ausing automated and combined SAXS and PDF analysis thus\u000d\u000aprovides insights into cluster chemistry in solution.\u000d\u000aResults and Discussion\u000d\u000aFigure 2A shows in situ X-ray total scattering data\u000d\u000aobtained during the formation of [Bi38O45] from\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O dissolved in DMSO. The experiment was done at T= 80 8C, and data sets obtained from\u000d\u000asimilar experiments performed at lower temperatures (T=\u000d\u000a308C\u2013608C) can be found in Figure S1. At the beginning of\u000d\u000athe reaction, Bragg peaks from the crystalline\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O phase (Figure 2D) is seen, as\u000d\u000acrystals at this point are still in suspension in DMSO. After\u000d\u000aapproximately 7 min, the crystals have fully dissolved, and\u000d\u000aonly diffuse scattering is seen. We therefore turn to PDF for\u000d\u000afurther structural analysis, and the corresponding PDFs are\u000d\u000aplotted in Figure 2 B. The PDFs are the Fourier transformation of the TS data, where the signal from DMSO has been\u000d\u000asubtracted, as illustrated in Figure S2. In the beginning of the\u000d\u000aexperiment (Figure 2 E), the PDF shows the presence of longrange order from the crystalline starting material, and as\u000d\u000aexpected, the data can be fitted with the\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O structure. Towards the end of\u000d\u000athe experiments the long-range order disappears,\u000d\u000aand the PDFs now show peaks up to only about 12 \u00e4.\u000d\u000aFigure 1. A) Structure of crystalline [Bi6O5(OH)3(NO3)5]\u00b73H2O.[8a] Hydrogen atoms have been omitted for clarity. B) An octahedral [Bi6O8] unit.\u000d\u000aC) The [Bi22O26]\u000d\u000a[8e] cluster structure. D) The [Bi38O45]\u000d\u000a[4a] cluster structure. In all structures, bismuth is shown in purple, oxygen in red, and nitrogen\u000d\u000ain blue.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000a20408 www.angewandte.org \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH Angew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","In Figure 2 F, the PDF obtained at the end of the\u000d\u000aexperiment is fitted with the [Bi38O45] model, confirming\u000d\u000athe product of the reaction. The cluster model (Figure 1D)\u000d\u000awas obtained from the structure of crystalline\u000d\u000a[Bi38O45(NO3)20(DMSO)28](NO3)4\u00b74DMSO,[4a] where an isolated [Bi38O45] cluster was cut out of the crystal structure. The\u000d\u000amodeling (described in detail in Table S1) was set up so that\u000d\u000athe basic atomic structure was kept fixed during the refinement. The excellent fit illustrates that the structure of the\u000d\u000a[Bi38O45] cluster in solution is similar (if not identical) to that\u000d\u000aseen in the crystalline phase, as also confirmed from DLS\u000d\u000ameasurements (see Figure S3).\u000d\u000aThe presence of the [Bi38O45] cluster at the end of the\u000d\u000aexperiment is further supported by SAXS experiments.\u000d\u000aIn situ SAXS data from a similar experiment are shown\u000d\u000ain Figure 2 C. Bragg peaks from the crystalline\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O phase can again be identified at\u000d\u000athe beginning of the reaction (Figure 2 C) but disappear after\u000d\u000aapproximately 10 min. Figure 2G shows the SAXS data\u000d\u000aobtained at the end of the experiment fitted with the\u000d\u000a[Bi38O45] cluster using the Debye equation as described in\u000d\u000adetail in Table S2. Again, the fits show excellent agreement\u000d\u000abetween data and model.\u000d\u000aHaving established that we can use PDF and SAXS to\u000d\u000aconfirm the starting point and end point of the reaction, we\u000d\u000anow move to characterize the structures seen during the\u000d\u000acluster growth and identify intermediate species. Figure 3A\u000d\u000ashows selected PDFs from the process, clearly illustrating that\u000d\u000aa smaller cluster than [Bi38O45] is present at the beginning of\u000d\u000athe reaction after dissolution. Initially, we had expected to see\u000d\u000athe presence of isolated [Bi6O8] clusters in solution, since it is\u000d\u000adescribed as particularly stable.[9a, 13] The calculated PDF from\u000d\u000aa [Bi6O8] cluster is shown in Figure 3A showing only a few\u000d\u000apeaks from the small structure. Surprisingly, we cannot\u000d\u000aFigure 2. A) Background subtracted in situ X-ray TS data obtained during the experiment performed at T=808C plotted as a function of time.\u000d\u000aWhile Bragg peaks are seen at the beginning of the reaction, only diffuse features are seen after dissolution. B) Corresponding PDFs plotted as\u000d\u000aa function of time. C) Background subtracted SAXS data obtained during a similar experiment plotted as a function of time. The intensity is\u000d\u000ashown on a logarithmic scale. D) Simulated X-ray scattering data of the [Bi6O5(OH)3(NO3)5]\u00b73H2O crystal structure, compared to background\u000d\u000asubtracted data from t=0 min and t=70 min, T=80 8C. E) Fit of the [Bi6O5(OH)3(NO3)5]\u00b73H2O crystal structure to the PDF obtained from the\u000d\u000afirst frame in the experiment at t=0 min. Rwp=29.0%. F) Fit of the [Bi38O45] cluster structure to the PDF obtained at the end of the experiment at\u000d\u000at=70 min; Rwp=16.6%. G) Fit of the [Bi38O45] cluster structure to the SAXS data obtained at the end of the experiment at t=37 min; Rwp=3.1%.\u000d\u000aThe SAXS data are plotted in a log\u2013log plot.\u000d\u000aFigure 3. A, B) Comparison of experimental PDFs (A) and SAXS data (log\u2013log plot) (B) with calculated patterns from an isolated [Bi6O8] unit and\u000d\u000aa [Bi10O14] \u201cdimer\u201d. C) Fit of the [Bi38O45] cluster structure to the PDF obtained after the dissolution of the crystalline starting material; T=808C,\u000d\u000at=8 min. D) Asymmetric peak at 3.8 \u00e4 in the PDF. E) A log\u2013log plot of the fit of the [Bi38O45] cluster structure to the SAXS data obtained after the\u000d\u000adissolution of the crystalline precursor; T=808C, t=16 min.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000aAngew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH www.angewandte.org 20409 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","identify a PDF similar to this at any point in our in situ data\u000d\u000afrom any reaction temperature, as peaks to higher r values are\u000d\u000aseen immediately after dissolution has occurred. This indicates that the small [Bi6O8] clusters are not stable in DMSO\u000d\u000aon the time scales and experimental conditions of the in situ\u000d\u000aX-ray scattering experiments. This observation is confirmed\u000d\u000aby SAXS (Figure 3 B), where isolated, monodisperse [Bi6O8]\u000d\u000aclusters are not seen at any point in our data. We also do not\u000d\u000aobserve the presence of for example, [Bi10O14] structures (two\u000d\u000aedge-sharing [Bi6O8] units) or any smaller fragments. While\u000d\u000awe cannot rule out that such structures are present in very\u000d\u000asmall concentrations in the samples, they are not main species.\u000d\u000a[Bi6O8] was observed as a very dominant species in in situ\u000d\u000aESI-MS experiments.[13] However, the conditions used for the\u000d\u000aESI-MS measurements were different from the current\u000d\u000ascattering studies, as the hydrolysis reactions in the ESI-MS\u000d\u000aexperiments were followed at room temperature, at much\u000d\u000alower Bi concentration (0.05 mM\u20131 mM for ESI-MS vs.\u000d\u000a230 mM for PDF/SAXS), and in a different solvent, namely\u000d\u000aa mixture of DMSO and dichloromethane, CH2Cl2. These\u000d\u000ahighly different conditions may explain the differences in the\u000d\u000aspecies observed. However, ESI-MS is also more sensitive to\u000d\u000aminor species than both PDF and SAXS, although they will\u000d\u000aonly be detected if they can easily be charged. The two\u000d\u000aexperiments are thus not in contradiction but provide\u000d\u000ainformation for different conditions and different time scales:\u000d\u000aWhile the ESI-MS studies show that [Bi6O8] species form\u000d\u000arapidly after dissolution, their absence from our in situ X-ray\u000d\u000ascattering data indicate very low solubility and thus low\u000d\u000aconcentration of these species and fast condensation after\u000d\u000adissolution in DMSO. Both experiments show the formation\u000d\u000aof [Bi22O26] clusters as quite stable intermediate and [Bi38O45]\u000d\u000aas final products.\u000d\u000aWe can now analyze the structure of the intermediate\u000d\u000aspecies. From the similar PDF peak positions between the\u000d\u000aintermediate structure (t = 8 min) and the final cluster (t =\u000d\u000a70 min) seen in Figure 3A, it is evident that the intermediate\u000d\u000ais structurally related to the [Bi38O45] cluster, as many of the\u000d\u000amain peaks show up in both PDFs. However, the peaks\u000d\u000arepresenting the largest interatomic distances are not present\u000d\u000ain the PDF from the intermediate cluster, which also appears\u000d\u000ato be significantly more disordered, as seen from the\u000d\u000aasymmetric broadening of the first Bi\u00b7\u00b7\u00b7Bi peak at r= 3.8 \u00e4\u000d\u000ain Figure 3D. Figure 3 C shows a fit of the [Bi38O45] structure\u000d\u000ato the PDF from the intermediate cluster. In Figure 3 E, the\u000d\u000aSAXS fit of the [Bi38O45] cluster to data representing the\u000d\u000aintermediate confirm that the cluster is smaller than the final\u000d\u000a[Bi38O45].\u000d\u000aTo characterize the cluster size evolution during the\u000d\u000areaction, we first analyze the in situ SAXS data with a form\u000d\u000afactor analysis. Details on the modeling and examples of fits\u000d\u000aare given in Table S3 and Figure S4. The results (Figures S5\u000d\u000aand S6) show that the average cluster size increases during the\u000d\u000areaction, and the clusters present are of a higher size\u000d\u000adispersity at the beginning of the reaction than at the end,\u000d\u000awhere the presence of stable [Bi38O45] clusters results in a very\u000d\u000alow size dispersity.\u000d\u000aThe observation that the intermediate cluster is smaller\u000d\u000abut structurally close to the [Bi38O45] structure agrees well\u000d\u000awith the chemistry of bismuth oxido clusters discussed above,\u000d\u000aas several different sizes of bismuth oxido clusters, all built up\u000d\u000afrom [Bi6O8] units can be synthesized. We therefore attempted to fit other known cluster structures, with sizes between\u000d\u000a[Bi6O8] to [Bi38O45], to the intermediate PDF as shown in\u000d\u000aFigure S7. The best cluster candidates were the [Bi18O36] and\u000d\u000a[Bi22O38] structures, agreeing well with observations from in\u000d\u000asitu ESI-MS analysis, molecular dynamics simulations and\u000d\u000aquantum chemical calculations.[13, 15a,b] However, to look\u000d\u000afurther into the time-dependent structural changes, and to\u000d\u000aprobe a larger structural space, we developed an automated\u000d\u000amethod, where all smaller cluster structures based on the\u000d\u000a[Bi38O45] framework were tested against the experimental\u000d\u000aPDF. Automated modeling methods for PDF analysis have\u000d\u000arecently been proposed by Banerjee et al.[21] for analysis of\u000d\u000ametal nanoparticles. In a recent study, we developed a method, where structural motifs in molybdenum oxide structures\u000d\u000awere identified by fitting automatically generated fragments\u000d\u000aof know polyoxometalate structures to PDFs.[20] We here\u000d\u000aapply an extension of this latter method to in situ data in order\u000d\u000ato identify intermediates and reaction pathways.\u000d\u000aThe principle is illustrated in Figure 4. First, the [Bi38O45]\u000d\u000amodel is fit to the PDF obtained from the last frame in the\u000d\u000aexperimental series. It was thereafter tested whether a smaller\u000d\u000aversion of the structure, i.e., with some Bi and O atoms\u000d\u000aremoved would result in a better fit to the data, as described\u000d\u000ain more detail in the Supporting Information, section I.\u000d\u000aThis method can give us an overview of the best fitting\u000d\u000astructure (built up from edge-sharing [Bi6O8] units) for every\u000d\u000aframe obtained during the reaction. Figure 5A\u2013C shows the\u000d\u000aresults from the automated modeling described above, where\u000d\u000athe refined number of Bi atoms is plotted as a function of time\u000d\u000afor the in situ experiments done at T= 808C, 60 8C, and 30 8C.\u000d\u000aResults from experiments done at other temperatures are\u000d\u000agiven in Figure S8. Figure 5A\u2013C firstly show that larger\u000d\u000astructures appear to be stabilized at higher reaction temperatures, and that the clusters grow with time. In Figure 5A (T=\u000d\u000a808C) plateaus at approximately 32 Bi atoms and 28 Bi atoms\u000d\u000aare identified, labelled as I and II. The same plateau at 28 Bi\u000d\u000aatoms is seen for the data obtained at T= 608C (Figure 5 B),\u000d\u000awhereas data from T= 30 8C (Figure 5 C) show plateaus at 25\u000d\u000aand 20 Bi atoms (III and IV). The occurrence of clusters of\u000d\u000aa given size is illustrated in Figure 5D, which shows a histogram of the cluster structures identified in all three datasets.\u000d\u000aThis shows that a large proportion of the identified clusters\u000d\u000ahave between 20 and 25 Bi atoms. In Figure 6 E\u2013H, we show\u000d\u000athe structures I\u2013IV along with fits to selected data. While\u000d\u000athese structures fit the data well, they appear unphysical, as\u000d\u000a\u201cdangling\u201d Bi atoms are seen. However, when looking closer,\u000d\u000awe identify that they share a stable backbone of 22 Bi atoms.\u000d\u000aThis is particularly clear for cluster III and IV, as seen in\u000d\u000aFigure 6A, where the Bi positions of these are compared to\u000d\u000athe [Bi22O26] structure, and fits of the [Bi22O26] cluster to the\u000d\u000asame frames (Figure 6 B,C) are reasonable.\u000d\u000aThe results from the automated modeling thus indicate\u000d\u000athat [Bi22O26] is an important intermediate in the reaction. We\u000d\u000aalso saw other frequent sizes, such as Bi28 and Bi32. However,\u000d\u000athese structures do not appear as closed-shell, physical\u000d\u000astructures, and are unlikely to be stable. The SAXS form\u000d\u000aAngewandte Research Articles Chemie\u000d\u000a20410 www.angewandte.org \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH Angew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","factor analysis described above (Figures S5 and S6) furthermore showed that before the reaction finished, a larger size\u000d\u000adispersion is seen, thus indicating that multiple cluster sizes\u000d\u000acan be present during the reaction. We therefore tried fitting\u000d\u000aboth the PDF and SAXS data with a two-structure model with\u000d\u000athe [Bi22O26] and [Bi38O45] clusters. The refined parameters\u000d\u000aare given in Tables S4\u20137 and the fits to both SAXS and PDF\u000d\u000adata are shown in Figure S9, where they are compared to the\u000d\u000aFigure 4. Sketch of the automated modeling process. The last frame of the in situ dataset is modeled with the [Bi38O45] cluster structure and all\u000d\u000astructures in which up to 8 Bi atoms from the outer shell have been removed together with nonbonding oxygen atoms. The best fitting structure\u000d\u000a(with three Bi atoms added) is used as a starting point for the fitting of the second-last frame. This process is repeated for all frames in the\u000d\u000areaction.\u000d\u000aFigure 5. A\u2013C) The number of bismuth atoms in the best fitting structure for each frame is plotted versus reaction time for the experiments done\u000d\u000aat T=80, 60, and 308C. D) Histogram representing the number of Bi atoms in the best fitting structure through the reaction. Frequency is the\u000d\u000anumber of times the specific cluster size is present during the reaction. E\u2013H) Clusters and fits that have been obtained in the automated\u000d\u000amodeling process.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000aAngew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH www.angewandte.org 20411 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","results from the automated modeling. In section K in the\u000d\u000aSupporting Information, we compare simulated SAXS data of\u000d\u000athe [Bi22O26] and [Bi38O45] clusters to data at two different\u000d\u000atime points in the reaction and compare two-phase SAXS\u000d\u000arefinements to fits using individual [Bi22O26] and [Bi38O45]\u000d\u000aclusters. Generally, the two-structure models provide similar\u000d\u000afit qualities or better.\u000d\u000aHaving established that the main species present throughout the reaction are [Bi22O26] and [Bi38O45], we can sequentially analyze the data to extract phase fractions of the two\u000d\u000aclusters as a function of time. The results from this analysis are\u000d\u000ashown in Figure 7 (see Figure S12 for results from other\u000d\u000areaction temperatures). Results from SAXS is shown in blue,\u000d\u000aand PDF in red, illustrating how the [Bi38O45] fraction\u000d\u000aincreases with function of time and temperature. Analysis of\u000d\u000aSAXS and PDF data independently show similar phase\u000d\u000afractions of the two clusters in the high temperature cases, and\u000d\u000athe amount of the [Bi38O45] cluster steadily increases. We\u000d\u000atherefore attempted a combined analysis method, that is,\u000d\u000ausing the so-called complex modeling approach.[19b\u2013d, 22] Complex modeling using SAXS and PDF analysis has been\u000d\u000aexemplified for CdS nanoparticles, where Farrow et al.[19b]\u000d\u000ashowed that by combining the two techniques, a much more\u000d\u000arobust description of particle size and shape could be\u000d\u000aobtained. Here, we have in a similar way used the program\u000d\u000aDiffpy-CMI for complex modeling of PDF and SAXS data.\u000d\u000aHowever, we have applied the Debye equation to calculate\u000d\u000athe scattering pattern in both the SAXS regime and PDF\u000d\u000aregime, as described in detail in the Supporting Information,\u000d\u000asections L and M. The results of the combined modeling are\u000d\u000ashown in green in Figure 7 and generally agree well with the\u000d\u000aindividual SAXS and PDF analysis, but with more certainty\u000d\u000aon the refined parameters as seen from the less scattered data\u000d\u000apoints.\u000d\u000aWhile the results from the experiments done at higher\u000d\u000atemperatures (T= 508C\u201380 8C) generally show clear trends\u000d\u000aand a good agreement between the SAXS and PDF results,\u000d\u000alarger discrepancies are observed for the data seen from\u000d\u000aexperiments done at lower temperatures, illustrated for T=\u000d\u000a308C in Figure 7 C. Here, the PDF results are somewhat\u000d\u000ascattered, while the SAXS refinements appear to give\u000d\u000aa clearer trend. The dispersity analysis illustrated in Figures\u000d\u000aS5 and S6 furthermore showed a larger dispersion of sizes for\u000d\u000alow temperature experiments, and it may be that small\u000d\u000afractions of other cluster structures or fragments are present\u000d\u000afor longer times during these slower reactions.\u000d\u000aThe data indicate that in none of the experiments, a full\u000d\u000areaction to [Bi38O45] takes place, as some [Bi22O26] is present\u000d\u000aat the end of the experiment. The refined fraction of the final\u000d\u000a[Bi38O45] cluster is approximately 60% at the end of the in situ\u000d\u000aexperiment at T= 808C, and for the reactions done at lower\u000d\u000atemperatures, the [Bi22O26] cluster remains the majority\u000d\u000aspecies in the solution. However, PDF and SAXS data\u000d\u000acollected from samples kept up to 275 days at room temperature show excellent fits with a single phase [Bi38O45] model\u000d\u000a(Figures S16 and S18).\u000d\u000aCombined, the results from the automated PDF analysis\u000d\u000aand the complex modeling of SAXS and PDF data allow us to\u000d\u000aestablish the overall reaction route. When\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O is dissolved in DMSO, larger\u000d\u000aclusters quickly form, and we never observe free [Bi6O8] (or\u000d\u000asmaller structure fragments) in our data at a time resolution\u000d\u000aFigure 6. A) Comparison of the Bi positions in the [Bi22O26] cluster with those of structures III and IV, found from the automated modeling\u000d\u000amethod. B,C) Fit to the in situ PDF (T=308C) obtained at t=152 min (B) and t=103 min (C) using the [Bi22O26] cluster as the model.\u000d\u000aFigure 7. Phase fractions of the [Bi22O26] and [Bi38O45] clusters, plotted as a function of time for experiments at A) T=80 8C, B) 60 8C, and C) 308C.\u000d\u000aResults from PDF are plotted in red, from SAXS in blue, and from complex modeling in green.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000a20412 www.angewandte.org \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH Angew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","of 4 s. This process most likely takes place through condensation reactions and rearrangement processes, as also suggested\u000d\u000aby Sattler et al.[13] The main intermediate species we observe\u000d\u000ais [Bi22O26], as concluded from both SAXS and PDF data.\u000d\u000aThis cluster is quite stable and co-exists with [Bi38O45] in the\u000d\u000atime range investigated with in situ experiments. The local\u000d\u000astructure changes taking place during dissolution are discussed further in the Supporting Information, section O.\u000d\u000aThe experiments discussed so far were all done from\u000d\u000a[Bi6O5(OH)3(NO3)5]\u00b73H2O. Nearly identical results were\u000d\u000aobtained when the starting crystalline material was changed\u000d\u000ato [Bi6O4(OH)4(NO3)6(H2O)2]\u00b7(H2O) (see Figure S18). This\u000d\u000aindicates that the results presented are general for compounds\u000d\u000abased on a hexanuclear bismuth oxido core [Bi6O8] dissolved\u000d\u000ain DMSO.\u000d\u000aUsing SAXS and PDF, we have identified the main\u000d\u000aintermediate species in the process. Nevertheless, some\u000d\u000aquestions regarding the cluster growth remain unanswered\u000d\u000afrom the current study. As discussed above, it has previously\u000d\u000abeen suggested that cluster growth takes place through\u000d\u000aoligomerization reactions involving fragments of [Bi6O8] units\u000d\u000aas key species. Our data and time resolution do not allow us to\u000d\u000aidentify these short-lived species or the specific mechanism.\u000d\u000aSeveral processes may take place in the reaction: for example,\u000d\u000aelusive, larger intermediates assembled from [Bi6O8] units\u000d\u000amay form, which could collapse or partly dissociate to result\u000d\u000ain the [Bi22O26] structure. Dissociation of [Bi6O8] fragments\u000d\u000acould also take place before oligomerization to lead stepwise\u000d\u000ato [Bi22O26] and finally to [Bi38O45]. This hypothesis is\u000d\u000asupported by ESI-MS studies of the hydrolysis of bismuth\u000d\u000acarboxylates, which revealed intermediates between Bi3- and\u000d\u000aBi13-oxido clusters. Independent analytic techniques to support their formation in solution are missing, but several single\u000d\u000acrystal X-ray structures of other polynuclear bismuth oxido\u000d\u000aclusters including at least one [Bi6O8] unit with various ligands\u000d\u000ahave been reported.[9a, 15c] The time resolution for the scattering experiments currently possible limits our insight into the\u000d\u000afast processes in solution; for example, molecular dynamic\u000d\u000asimulations have suggested that the growth from [Bi6O8] units\u000d\u000atake place on the nanosecond to microsecond time scale.[15a]\u000d\u000aIn the future, new possibilities for ultrafast, high flux X-ray\u000d\u000ascattering studies may open for studies of such processes.[23] If\u000d\u000aone could chemically stabilize a series of species formed on\u000d\u000athe way to the [Bi22O26] clusters, new insight could also be\u000d\u000aobtained.\u000d\u000aLigand Exchange: From [Bi38O45(NO3)20(DMSO)d] to\u000d\u000a[Bi38O45(OMc)24(DMSO)d]\u000d\u000aPDF analysis also allows us to study cluster\u2013solvent\u000d\u000ainteractions and any structural changes that may take place\u000d\u000aduring exchange of ligands in the clusters. Figure 8A shows\u000d\u000aPDFs obtained when heating a solution of\u000d\u000a[Bi38O45(NO3)20(DMSO)28](NO3)4\u00b74DMSO in DMSO with\u000d\u000asodium methacrylate. In this process, nitrate ligands are\u000d\u000asubstituted with methacrylate[5, 9a] making the clusters soluble\u000d\u000ain organic solvents. After 7 min of the reaction, significant\u000d\u000achanges appear. Figure 8 B shows selected PDFs obtained\u000d\u000aover the course of the reaction. At the beginning, both the\u000d\u000apeaks at r\u21e1 3.8 \u00e4 and r\u21e1 6.7 \u00e4 have shoulders which\u000d\u000adisappear through the ligand exchange, and all of the peaks\u000d\u000abecome narrower and more symmetric as the reaction\u000d\u000aprogresses.\u000d\u000aWe start by modeling the first frame of the reaction with\u000d\u000athe [Bi38O45] cluster structure, Figure 9A. While this model\u000d\u000afits all main PDF peaks, significant misfits are seen, and the\u000d\u000adifference curve is similar to an exponentially dampened\u000d\u000asinusoidal function. This behavior has previously been\u000d\u000adescribed by Zobel et al., who related it to solvent restructuring effects on the surface of nanoparticles.[24] Figure 9 B shows\u000d\u000athe fit if a dampened sinusoidal function is added to the\u000d\u000a[Bi38O45] model. This significantly improves the fit to the data,\u000d\u000asuggesting a strong interaction between the cluster (or its\u000d\u000aligands) and the DMSO solvent. A PDF obtained from pure\u000d\u000aDMSO is shown in Figure S19 confirming that the oscillation\u000d\u000adoes not arise from the solvent itself.\u000d\u000aAt the end of the reaction, the data (Figure 9D) is very\u000d\u000awell described by the [Bi38O45] cluster, giving a Rwp value of\u000d\u000a12.2%. Here, the exponential dampened sinusoidal does not\u000d\u000acontribute to the fit as seen in Figure 9 E. Figure 9 C (bottom)\u000d\u000ashows the Rwp values of the fits during the reaction. This\u000d\u000abehavior indicates that the structuring of the solvent dominates at the beginning of the reaction but when the ligand is\u000d\u000aexchanged to methacrylate, the effect vanishes. This is also\u000d\u000aclear when plotting the contribution of the dampened sinusoidal oscillation through the reaction, Figure 9 F.\u000d\u000aThe ligand exchange, and the solvent interactions also\u000d\u000aappear to affect the atomic structure of the bismuth oxido\u000d\u000aclusters. Figure 9 C (top) shows the refined ADPs of the Bi\u000d\u000aatoms through the reaction. This parameter relates to the\u000d\u000awidth of the PDF peaks, and decreases through the reaction,\u000d\u000aFigure 8. A) PDFs obtained from in situ X-ray TS data collected during the ligand-exchange experiment performed at T=808C, plotted as\u000d\u000aa function of time. B) Selected PDFs from the experiment.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000aAngew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH www.angewandte.org 20413 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","showing that the ligand-exchanged clusters with methacrylate\u000d\u000aare more ordered. The PDFs obtained from the beginning of\u000d\u000athe reaction furthermore show highly asymmetric peaks, as\u000d\u000aseen most clearly from the shoulder peak at about 4.2 \u00e4. This\u000d\u000aasymmetry is likely to be an effect of the strong interaction\u000d\u000abetween the cluster and the DMSO solvent. Modeling details\u000d\u000aare given in Table S8,9.\u000d\u000aThe solvent restructuring in the beginning of the reaction\u000d\u000ais likely due to partial dissociation of nitrate ligands, which\u000d\u000aresults in a cationic cluster, making the coordination to\u000d\u000aDMSO strong. This can result in some distortion of the\u000d\u000a[Bi38O45] core as seen from PDF peaks asymmetry. When the\u000d\u000anitrate ligand is substituted with methacrylate, the strong\u000d\u000abonding between methacrylate and the [Bi38O45] clusters\u000d\u000ahamper the coordination of DMSO to the nanoclusters, and\u000d\u000athe effect in the PDF vanishes. This is also\u000d\u000areflected when considering the crystal structures of\u000d\u000a[Bi38O45(NO3)20(DMSO)28](NO3)4\u00b74DMSO[4a] and\u000d\u000a[Bi38O45(OMc)24(DMSO)9]\u00b72DMSO\u00b77H2O:[5] For the nitratecoordinated cluster, 28 DMSO molecules coordinate to the\u000d\u000abismuth oxido core, while this is only 9 for the crystal\u000d\u000astructure containing methacrylate.\u000d\u000aA sketch of how the solvent restructures at the surface of\u000d\u000athe [Bi38O45] clusters is illustrated in Figure 10. The DMSO\u000d\u000amolecules coordinate to the nanocluster through strong\u000d\u000abonding to the surface of the bismuth oxido cluster with\u000d\u000adirections perpendicular to the surface of the [Bi38O45]\u000d\u000aclusters. This results in regions with alternating low and high\u000d\u000aelectron density due to packing of solvent molecules.\u000d\u000aCollectively, this gives rise to a sinusoidal oscillation contributing to the PDF. Since the solvent effect is strongest close to\u000d\u000athe surface of the [Bi38O45] clusters and it slowly transforms\u000d\u000ainto a bulk solvent structure of DMSO, the sinusoidal\u000d\u000aoscillation exponentially decays. However, we observe solvent effects of up to r= 30 \u00e4.\u000d\u000aConclusion\u000d\u000aIn situ PDF and SAXS have been used to follow the\u000d\u000areaction from a suspension of [Bi6O5(OH)3(NO3)5]\u00b73H2O in\u000d\u000aDMSO to [Bi38O45] clusters in solution (Figure 11). The\u000d\u000astructural modeling was done using the Debye equation, and\u000d\u000awe applied combined modeling of the PDF and SAXS data.\u000d\u000aTo follow the cluster growth taking place during the reaction,\u000d\u000awe introduced an automated modeling method, where\u000d\u000aintermediate species were identified by iteratively removing\u000d\u000aatoms from the final [Bi38O45] structure. This analysis showed\u000d\u000athat starting from [Bi6O8] a [Bi22O26] intermediate very\u000d\u000aquickly forms and slowly transforms to [Bi38O45]. The\u000d\u000aformation of the [Bi22O26] structure unexpectedly proceeded\u000d\u000aimmediately after dissolution of the crystalline starting\u000d\u000amaterial, meaning that the time resolution of the X-ray\u000d\u000ascattering experiments does not allow us to identify any shortlived building blocks that may form before the larger clusters\u000d\u000aare seen. Questions regarding e.g., oligomerization reactions\u000d\u000aand cluster fragmentation thus remain open, but the combination of SAXS and PDF provided new insight into the\u000d\u000acluster growth process of bismuth oxido species including\u000d\u000ainformation on the time scale at which processes occur.\u000d\u000aWe furthermore demonstrate that due to nitrate ligand\u000d\u000adissociation, a strong interaction between DMSO and the\u000d\u000abismuth oxido clusters induce restructuring of solvent molecules around the bismuth oxido clusters. This interaction leads\u000d\u000ato disorder of the atomic structure in the bismuth oxido\u000d\u000aclusters themselves, as seen from the width and asymmetry of\u000d\u000aPDF peaks. When methacrylate is introduced as a ligand in\u000d\u000aFigure 9. Representative fit of a PDF obtained during ligand exchange, obtained t=0 min into the ligand-exchange reaction A) without including\u000d\u000asolvent restructuring effects (wave), B) including solvent restructuring effects (wave). C) Rwp values and ADP values of Bi obtained when modeling\u000d\u000athe time-resolved PDFs, obtained during ligand exchange, with the [Bi38O45] model. D) Representative fit of a PDF obtained during ligand\u000d\u000aexchange, obtained t=15 min into the reaction without including solvent restructuring effects and E) when including solvent restructuring effects\u000d\u000aat the surface of the nanocluster. F) The contribution of the solvent restructuring effects during the ligand-exchange reaction.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000a20414 www.angewandte.org \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH Angew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","the [Bi38O45] cluster, it screens the cluster from strong solvent\u000d\u000ainteractions, consequently leaving the [Bi38O45] core more\u000d\u000aordered and closer to that observed in the crystalline\u000d\u000astructure. However, the [Bi38O45] core structure stays intact\u000d\u000aduring the ligand substitution reaction, which demonstrates\u000d\u000athat such atomically precise metal oxido nanoclusters can\u000d\u000aserve as model structures to study ligand effects at the surface\u000d\u000aof metal oxide nanoparticles.\u000d\u000aThe chemical knowledge built up in this study is likely\u000d\u000atransferable to the growth of other metal oxido clusters, at\u000d\u000aleast to metal oxides of the fluorite type such as uranium,\u000d\u000acerium and plutonium oxido nanoclusters, which also show\u000d\u000aa stable nuclearity of 38 metal atoms in their cluster core. The\u000d\u000ainsights that can be obtained from combined in situ X-ray\u000d\u000ascattering studies furthermore open for understanding material formation in solution. For example, SAXS and PDF\u000d\u000astudies may allow the identification of prenucleation clusters,\u000d\u000aand provide a much deeper understanding of the fundamental\u000d\u000aprocesses involved in nucleation. Studies of atomically\u000d\u000amonodisperse metal oxido cluster structures, as done here,\u000d\u000acan furthermore provide knowledge of metal oxide chemistry,\u000d\u000aand it is likely that the metastable [Bi22O26] cluster observed\u000d\u000ahere may be an important species in bismuth oxide formation.\u000d\u000aUnderstanding the solution chemistry of metal oxido clusters\u000d\u000aon an atomic and molecular scale can thus open new\u000d\u000aopportunities for synthesizing nanoscale metal oxides in\u000d\u000aa controlled manner.\u000d\u000aAcknowledgements\u000d\u000aThis work is part of a project that has received funding from\u000d\u000athe European Research Council (ERC) under the European\u000d\u000aUnion s Horizon 2020 Research and Innovation Programme\u000d\u000a(grant agreement No. 804066). We furthermore thank DANSCATT (supported by the Danish Agency for Science and\u000d\u000aFigure 10. Illustration of how the surface of the [Bi38O45] cluster, with nitrate as the ligand, induces a solvent restructuring effect up to r=30 \u00e4.\u000d\u000aInteractions between the bismuth oxido clusters and the DMSO molecules induce an ordering of the solvent molecules near the [Bi38O45] cluster\u000d\u000asurface, which gives rise to an exponentially dampened sinusoidal oscillation in the PDF. Carbon atoms are shown in black, sulfur in yellow, and\u000d\u000ahydrogen in beige.\u000d\u000aFigure 11. The [Bi6O5(OH)3(NO3)5]\u00b73H2O crystals dissolve in DMSO to form a [Bi22O26] cluster as an intermediate before forming atomically\u000d\u000aprecise [Bi38O45] clusters.\u000d\u000aAngewandte Research Articles Chemie\u000d\u000aAngew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH www.angewandte.org 20415 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License","Higher Education) for support. A.S.A. acknowledges the\u000d\u000aSiemens Foundation for support for his thesis project. We\u000d\u000aacknowledge DESY (Hamburg, Germany), a member of the\u000d\u000aHelmholtz Association HGF, for the provision of experimental facilities. Parts of this research were carried out at\u000d\u000abeamline P02.1 and P07 at Petra III, and we thank Martin\u000d\u000aEtter, Jozef Bednarcik and Ann-Christin Dippel for assistance in using the beamline. We acknowledge SOLEIL for\u000d\u000aprovision of synchrotron radiation facilities, and we thank\u000d\u000aThomas Bizien for assistance in using beamline SWING. We\u000d\u000aacknowledge the European Synchrotron Radiation Facility\u000d\u000afor provision of synchrotron radiation facilities, and we thank\u000d\u000aPeter Boesecke for assistance in using beamline ID02. We\u000d\u000athank the Deutsche Forschungsgemeinschaft (DFG SPP1415:\u000d\u000aCrystalline Nonequilibrium Phases\u2014Preparation, Characterization and in situ Studies of Formation Mechanisms; ME\u000d\u000a2284/3-2) for support. M.S. and E.B. acknowledge the\u000d\u000aInnovation Fund Denmark (IFD) as part of project Linking\u000d\u000aIndustry to Neutrons and X-rays (LINX) under File No. 5152-\u000d\u000a00005B for partly funding this project. Open access funding\u000d\u000aenabled and organized by Projekt DEAL.\u000d\u000aConflict of Interest\u000d\u000aThe authors declare no conflict of interest.\u000d\u000aKeywords: bismuth \u00b7 in situ studies \u00b7 nanostructures \u00b7\u000d\u000apair distribution function \u00b7 small-angle X-ray scattering\u000d\u000a[1] E. D. B\u00f8jesen, B. B. Iversen, CrystEngComm 2016, 18, 8332 \u2013\u000d\u000a8353.\u000d\u000a[2] a) P. D. Matthews, T. C. King, D. S. Wright, Chem. Commun.\u000d\u000a2014, 50, 12815 \u2013 12823; b) P. Coppens, Y. Chen, E. Trzop, Chem.\u000d\u000aRev. 2014, 114, 9645 \u2013 9661; c) W.-H. Fang, L. Zhang, J. Zhang, J.\u000d\u000aAm. Chem. Soc. 2016, 138, 7480 \u2013 7483; d) S. Yang, H.-C. Su, J.-L.\u000d\u000aHou, W. Luo, D.-H. Zou, Q.-Y. Zhu, J. Dai, Dalton Trans. 2017,\u000d\u000a46, 9639 \u2013 9645; e) P. D. Matthews, N. Li, H.-K. Luo, D. S.\u000d\u000aWright, Chem. Eur. J. 2016, 22, 4632 \u2013 4633.\u000d\u000a[3] a) K. J. Mitchell, K. A. Abboud, G. Christou, Nat. Commun.\u000d\u000a2017, 8, 1445; b) B. Russell-Webster, K. A. Abboud, G. Christou,\u000d\u000aChem. Commun. 2020, 56, 5382 \u2013 5385.\u000d\u000a[4] a) L. Miersch, T. R\u00b8ffer, M. Schlesinger, H. Lang, M. Mehring,\u000d\u000aInorg. Chem. 2012, 51, 9376 \u2013 9384; b) F. Y. Du, J. M. Lou, R.\u000d\u000aJiang, Z. Z. Fang, X. F. Zhao, Y. Y. Niu, S. Q. Zou, M. M. Zhang,\u000d\u000aA. H. Gong, C. Y. Wu, Int. J. Nanomed. 2017, 12, 5973 \u2013 5992.\u000d\u000a[5] L. Miersch, T. R\u00b8ffer, M. Mehring, Chem. Commun. 2011, 47,\u000d\u000a6353 \u2013 6355.\u000d\u000a[6] M. Schlesinger, S. Schulze, M. Hietschold, M. Mehring, Dalton\u000d\u000aTrans. 2013, 42, 1047 \u2013 1056.\u000d\u000a[7] M. Weber, M. Schlesinger, M. Mehring, Cryst. Growth Des. 2016,\u000d\u000a16, 5678 \u2013 5688.\u000d\u000a[8] a) F. Lazarini, Acta Crystallogr. 1978, 34, 3169 \u2013 3173; b) D. L.\u000d\u000aRogow, H. Fei, D. P. Brennan, M. Ikehata, P. Y. Zavalij, A. G.\u000d\u000aOliver, S. R. J. Oliver, Inorg. Chem. 2010, 49, 5619 \u2013 5624; c) M.\u000d\u000aMehring, S. Paalasmaa, M. Sch\u00b8rmann, Eur. J. Inorg. Chem.\u000d\u000a2005, 4891 \u2013 4901; d) M. Mehring, D. Mansfeld, S. Paalasmaa, M.\u000d\u000aSch\u00b8rmann, Chem. Eur. J. 2006, 12, 1767 \u2013 1781; e) D. Mansfeld,\u000d\u000aM. Mehring, M. Sch\u00b8rmann, Angew. Chem. Int. Ed. 2005, 44,\u000d\u000a245 \u2013 249; Angew. Chem. 2005, 117, 250 \u2013 254; f) X. Kou, X.\u000d\u000aWang, D. Mendoza-Espinosa, L. N. Zakharov, A. L. Rheingold,\u000d\u000aW. H. Watson, K. A. Brien, L. K. Jayarathna, T. A. Hanna,\u000d\u000aInorg. Chem. 2009, 48, 11002 \u2013 11016; g) M. Weber, T. R\u00b8ffer, F.\u000d\u000aSpeck, F. G\u00e7hler, D. P. Weimann, C. A. Schalley, T. Seyller, H.\u000d\u000aLang, M. Mehring, Inorg. Chem. 2020, 59, 3353 \u2013 3366.\u000d\u000a[9] a) M. Mehring in Clusters\u2014Contemporary Insight in Structure\u000d\u000aand Bonding (Ed.: S. Dehnen), Springer International Publishing, Cham, 2017, pp. 201 \u2013 268; b) M. Mehring, Coord. Chem.\u000d\u000aRev. 2007, 251, 974 \u2013 1006.\u000d\u000a[10] I. L. Malaestean, A. Ellern, S. Baca, P. Kogerler, Chem.\u000d\u000aCommun. 2012, 48, 1499 \u2013 1501.\u000d\u000a[11] C. Falaise, C. Volkringer, J. F. Vigier, A. Beaurain, P. Roussel, P.\u000d\u000aRabu, T. Loiseau, J. Am. Chem. Soc. 2013, 135, 15678 \u2013 15681.\u000d\u000a[12] a) L. Soderholm, P. M. Almond, S. Skanthakumar, R. E. Wilson,\u000d\u000aP. C. Burns, Angew. Chem. Int. Ed. 2008, 47, 298 \u2013 302; Angew.\u000d\u000aChem. 2008, 120, 304 \u2013 308; b) R. E. Wilson, S. Skanthakumar, L.\u000d\u000aSoderholm, Angew. Chem. Int. Ed. 2011, 50, 11234 \u2013 11237;\u000d\u000aAngew. Chem. 2011, 123, 11430 \u2013 11433.\u000d\u000a[13] D. Sattler, M. Schlesinger, M. Mehring, C. A. Schalley, ChemPlusChem 2013, 78, 1005 \u2013 1014.\u000d\u000a[14] a) K. Takao, S. Takao, A. C. Scheinost, G. Bernhard, C. Hennig,\u000d\u000aInorg. Chem. 2012, 51, 1336 \u2013 1344; b) S. Takao, K. Takao, W.\u000d\u000aKraus, F. Emmerling, A. C. Scheinost, G. Bernhard, C. Hennig,\u000d\u000aEur. J. Inorg. Chem. 2009, 4771 \u2013 4775; c) S. L. Estes, M. R.\u000d\u000aAntonio, L. Soderholm, J. Phys. Chem. C 2016, 120, 5810 \u2013 5818.\u000d\u000a[15] a) M. Walther, D. Zahn, Eur. J. Inorg. Chem. 2015, 1178 \u2013 1181;\u000d\u000ab) M. Walther, D. Zahn, Chem. Phys. Lett. 2018, 691, 87 \u2013 90;\u000d\u000ac) M. Schlesinger, A. Pathak, S. Richter, D. Sattler, A. Seifert, T.\u000d\u000aR\u00b8ffer, P. C. Andrews, C. A. Schalley, H. Lang, M. Mehring, Eur.\u000d\u000aJ. Inorg. Chem. 2014, 4218 \u2013 4227.\u000d\u000a[16] a) C. Hennig, A. Ikeda-Ohno, W. Kraus, S. Weiss, P. Pattison, H.\u000d\u000aEmerich, P. M. Abdala, A. C. Scheinost, Inorg. Chem. 2013, 52,\u000d\u000a11734 \u2013 11743; b) Y.-J. Hu, K. E. Knope, S. Skanthakumar, L.\u000d\u000aSoderholm, Eur. J. Inorg. Chem. 2013, 4159 \u2013 4163; c) C. Hennig,\u000d\u000aS. Takao, K. Takao, S. Weiss, W. Kraus, F. Emmerling, A. C.\u000d\u000aScheinost, Dalton Trans. 2012, 41, 12818 \u2013 12823.\u000d\u000a[17] S. J. L. Billinge, M. G. Kanatzidis, Chem. Commun. 2004, 749 \u2013\u000d\u000a760.\u000d\u000a[18] K. M. \u00d8. Jensen, C. Tyrsted, M. Bremholm, B. B. Iversen,\u000d\u000aChemSusChem 2014, 7, 1594 \u2013 1611.\u000d\u000a[19] a) S. J. L. Billinge, I. Levin, Science 2007, 316, 561 \u2013 565; b) C.\u000d\u000aFarrow, C. Shi, P. Juh\u00b5s, X. Peng, S. J. L. Billinge, J. Appl.\u000d\u000aCrystallogr. 2014, 47, 561 \u2013 565; c) P. Juh\u00b5s, C. L. Farrow, X.\u000d\u000aYang, K. R. Knox, S. J. L. Billinge, Acta Crystallogr. Sect. A 2015,\u000d\u000a71, 562 \u2013 568; d) C. L. Farrow, S. J. L. Billinge, Acta Crystallogr.\u000d\u000aSect. A 2009, 65, 232 \u2013 239.\u000d\u000a[20] T. L. Christiansen, E. T. S. Kj\u00e6r, A. Kovyakh, M. L. R\u00e7deren, M.\u000d\u000aH\u00f8j, T. Vosch, K. M. \u00d8. Jensen, J. Appl. Crystallogr. 2020, 53,\u000d\u000a148 \u2013 158.\u000d\u000a[21] S. Banerjee, C.-H. Liu, J. D. Lee, A. Kovyakh, V. Grasmik, O.\u000d\u000aPrymak, C. Koenigsmann, H. Liu, L. Wang, A. M. M. Abeykoon,\u000d\u000aS. S. Wong, M. Epple, C. B. Murray, S. J. L. Billinge, J. Phys.\u000d\u000aChem. C 2018, 122, 29498 \u2013 29506.\u000d\u000a[22] a) M. G. Tucker, D. A. Keen, M. T. Dove, A. L. Goodwin, Q.\u000d\u000aHui, J. Phys. Condens. Matter 2007, 19, 335218; b) V. Krayzman,\u000d\u000aI. Levin, M. G. Tucker, J. Appl. Crystallogr. 2008, 41, 705 \u2013 714.\u000d\u000a[23] P. Fromme, Nat. Chem. Biol. 2015, 11, 895 \u2013 899.\u000d\u000a[24] a) M. Zobel, R. B. Neder, S. A. J. Kimber, Science 2015, 347,\u000d\u000a292 \u2013 294; b) S. L. J. Thom\u2030, S. W. Krauss, M. Eckardt, P. Chater,\u000d\u000aM. Zobel, Nat. Commun. 2019, 10, 995.\u000d\u000aManuscript received: March 14, 2021\u000d\u000aAccepted manuscript online: May 30, 2021\u000d\u000aVersion of record online: July 1, 2021\u000d\u000aAngewandte Research Articles Chemie\u000d\u000a20416 www.angewandte.org \u232b 2021 The Authors. Angewandte Chemie International Edition published by Wiley-VCH GmbH Angew. Chem. Int. Ed. 2021, 60, 20407 \u2013 20416 15213773, 2021, 37, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/anie.202103641 by Royal Danish Library, Wiley Online Library on [02/05/2023]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License",""," \u000d\u000a59\u000d\u000a3 Supervised\u0009 Machine\u0009 Learning for Experimental\u0009 Scattering\u0009 Data\u0009\u000d\u000aAnalysis*\u000d\u000aAs\u0009discussed in\u0009Section\u00090,\u0009conventional\u0009modelling\u0009techniques\u0009effectively provide\u0009structural\u0009information\u000d\u000afrom\u0009 experimental\u0009 data\u0009 when a\u0009 structural\u0009 model is\u0009 given\u0009 as\u0009 input.\u0009 However,\u0009 these\u0009 methods are\u0009\u000d\u000acomputationally\u0009expensive,\u0009making them\u0009less\u0009suitable\u0009for\u0009predicting the\u0009structural\u0009model by\u0009iteratively\u0009\u000d\u000afitting\u0009each\u0009structure\u0009in\u0009a\u0009database to\u0009the\u0009experimental\u0009data.\u0009Consequently,\u0009data\u0009analysis\u0009is\u0009currently\u000d\u000aconsidered\u0009 a\u0009 bottleneck\u0009 in\u0009 materials\u0009 chemistry\u0009 research.53,54 With\u0009 the\u0009 continuing\u0009 advancement\u0009 of\u0009\u000d\u000amodern\u0009radiation\u0009facilities,55,56 there\u0009is\u0009an\u0009increasing\u0009need\u0009for\u0009tools\u0009that\u0009can\u0009provide\u0009scientists\u0009with\u0009a\u0009\u000d\u000astructural\u0009model\u0009to\u0009fit\u0009the\u0009data. In\u0009light\u0009of\u0009these\u0009challenges,\u0009there\u0009is\u0009a\u0009growing\u0009demand\u0009for\u0009faster\u0009and\u0009\u000d\u000amore\u0009efficient data\u0009analysis\u0009approaches than\u0009conventional\u0009minimisation\u0009methods to\u0009enable automated\u0009\u000d\u000aanalysis\u0009that\u0009can\u0009be\u0009performed\u0009during\u0009measurements.\u0009Here,\u0009ML\u0009in\u0009particular\u0009has\u0009emerged\u0009as\u0009a\u0009powerful\u0009\u000d\u000atool\u0009for automating several\u0009aspects\u0009of\u0009scattering-\u0009and\u0009spectroscopy\u0009data\u0009analysis.57-66\u000d\u000aML\u0009is\u0009a\u0009field of\u0009artificial\u0009intelligence\u0009that\u0009focuses\u0009on\u0009developing\u0009models\u0009capable\u0009of\u0009automatically\u0009learning\u0009\u000d\u000afrom\u0009data\u0009and\u0009making\u0009predictions\u0009without\u0009explicitly programming\u0009the\u0009rules.\u0009Supervised\u0009ML\u0009is\u0009a\u0009subfield\u0009\u000d\u000aof\u0009ML where\u0009models\u0009are\u0009trained\u0009using\u0009labelled data,\u0009with\u0009the\u0009goal\u0009of\u0009learning\u0009a\u0009mapping\u0009between\u0009input\u0009\u000d\u000afeatures\u0009and\u0009corresponding\u0009labels\u0009to\u0009classify\u0009new and unseen\u0009data.\u0009For\u0009example,\u0009predicting\u0009a\u0009structural\u0009\u000d\u000amodel\u0009from\u0009a\u0009scattering\u0009dataset,\u0009as\u0009illustrated\u0009in\u0009Figure\u00093.1A,\u0009is\u0009a\u0009supervised\u0009ML\u0009task.\u0009Here,\u0009the\u0009datasets\u000d\u000aserve\u0009as input\u0009 features,\u0009and\u0009 the\u0009model\u0009is\u0009supervised\u0009 to\u0009identify the\u0009structural\u0009models,\u0009which\u0009are\u0009 the\u0009\u000d\u000alabels.\u0009 Conversely,\u0009 predicting\u0009 scattering\u0009 data\u0009 from\u0009 a\u0009 structural\u0009 model\u0009 can\u0009 be\u0009 achieved\u0009 by\u0009 using\u0009 the\u0009\u000d\u000ascattering\u0009datasets as\u0009labels\u0009and\u0009 the\u0009structural\u0009model\u0009as\u0009input\u0009 features,\u0009as\u0009illustrated\u0009in\u0009Figure\u00093.1B.\u0009\u000d\u000aAnother\u0009 common\u0009 application\u0009 involves\u0009 directly\u0009 predicting structural\u0009 information\u0009 (Figure\u0009 3.1C),\u0009\u000d\u000abypassing\u0009the\u0009structure\u0009refinement\u0009step.\u0009This\u0009is\u0009done\u0009by\u0009training\u0009the\u0009supervised\u0009ML\u0009model\u0009on\u0009data\u0009with\u0009\u000d\u000avarying\u0009structural\u0009parameters.\u000d\u000aWhile\u0009the\u0009underlying\u0009principles\u0009of\u0009the\u0009three\u0009methodologies\u0009(Figure\u00093.1)\u0009are\u0009the\u0009same,\u0009my PhD\u0009research\u0009\u000d\u000ahas\u0009mainly\u0009focused\u0009on\u0009accelerating\u0009data\u0009analysis\u0009by\u0009predicting\u0009the\u0009structure\u0009as\u0009illustrated\u0009in\u0009Figure\u00093.1A\u0009\u000d\u000aand\u0009using\u0009conventional\u0009modelling\u0009strategies\u0009with\u0009this\u0009structural\u0009model\u0009as\u0009input. I\u0009have\u0009mainly\u0009worked\u0009\u000d\u000awith\u0009scattering\u0009data,\u0009however,\u0009 the\u0009modelling\u0009methodology\u0009is\u0009applicable\u0009 to\u0009data\u0009 from\u0009other\u0009structure\u0009\u000d\u000acharacterisation\u0009 techniques,\u0009 such\u0009 as\u0009 INS\u0009 and\u0009 XAS,\u0009 where\u0009 a\u0009 structural\u0009 model\u0009 can\u0009 also\u0009 describe\u0009 the\u0009\u000d\u000aexperimental\u0009data through\u0009structure\u0009refinement or\u0009simulation\u0009of\u0009data.\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009II."," \u000d\u000a89\u000d\u000aFigure\u00093.1:\u0009A)\u0009During\u0009training,\u0009a\u0009supervised\u0009ML\u0009model\u0009is\u0009trained\u0009using pairs\u0009of\u0009structural\u0009models\u0009and\u0009scattering\u0009\u000d\u000asimulations,\u0009here\u0009shown\u0009for\u0009PDF\u0009data.\u0009Although\u0009the\u0009ML\u0009training\u0009procedure\u0009is\u0009computationally\u0009expensive,\u0009it\u0009is\u0009a\u0009onetime\u0009cost.\u0009Afterwards,\u0009the\u0009ML\u0009model\u0009quickly\u0009and\u0009computationally\u0009inexpensively\u0009predicts\u0009the\u0009structural\u0009model\u0009from\u0009\u000d\u000athe\u0009 scattering\u0009 data.\u0009B)\u0009 During\u0009 training,\u0009 a\u0009 supervised\u0009 ML\u0009 model\u0009 is\u0009 trained\u0009 using pairs\u0009 of\u0009 structural\u0009 models\u0009 and\u0009\u000d\u000ascattering\u0009 simulations,\u0009 here\u0009 shown\u0009 for\u0009 PDF\u0009 data.\u0009 After\u0009 training,\u0009 the\u0009 ML\u0009 model\u0009 quickly\u0009 and\u0009 computationally\u0009\u000d\u000ainexpensively\u0009predicts\u0009the\u0009scattering\u0009data\u0009from\u0009a\u0009structural\u0009model.\u0009C)\u0009During\u0009training,\u0009a\u0009supervised\u0009ML\u0009model\u0009is\u0009\u000d\u000atrained\u0009using pairs\u0009of\u0009scattering\u0009simulations\u0009with\u0009varying\u0009structural\u0009parameters,\u0009here\u0009shown\u0009for\u0009SAXS\u0009data.\u0009After\u0009\u000d\u000atraining,\u0009 the\u0009ML\u0009model\u0009 quickly\u0009and\u0009 computationally\u0009inexpensively\u0009 predicts\u0009 the\u0009 structural\u0009information\u0009 from\u0009 the\u0009\u000d\u000ascattering\u0009data. Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u0009Paper\u0009II.46\u000d\u000aML\u0009 models\u0009 must\u0009 be validated\u0009 on\u0009 data\u0009 that\u0009 has\u0009 not\u0009 been\u0009 used during\u0009 training.\u0009 Before\u0009 initiating\u0009 the\u0009\u000d\u000atraining\u0009process,\u0009the\u0009dataset\u0009is\u0009split\u0009into\u0009a\u0009training,\u0009validation,\u0009and\u0009test\u0009set.\u0009Most\u0009typical\u0009are\u0009splits\u0009of\u000960\u0009\u000d\u000a%\u0009training\u0009data,\u000920\u0009%\u0009validation\u0009data\u0009and\u000920\u0009% test\u0009data.\u0009The\u0009ML\u0009model\u0009learns\u0009from\u0009the\u0009training\u0009set\u0009by\u0009\u000d\u000aminimising\u0009a\u0009user-defined\u0009objective\u0009function,\u0009known\u0009as\u0009a\u0009loss\u0009function.\u0009One\u0009training\u0009episode\u0009consists\u0009of\u0009\u000d\u000apredicting\u0009with\u0009 the\u0009ML\u0009model\u0009on\u0009 the\u0009 training\u0009set,\u0009calculating\u0009 the\u0009loss\u0009 function,\u0009and\u0009updating\u0009the\u0009ML\u0009\u000d\u000amodel.\u0009The\u0009ML\u0009model\u0009is\u0009continuously\u0009evaluated\u0009on\u0009unseen\u0009data\u0009by\u0009calculation\u0009of\u0009the loss\u0009function from\u0009\u000d\u000apredictions\u0009on\u0009the\u0009validation\u0009set.\u0009After training\u0009is\u0009complete,\u0009the\u0009test\u0009set,\u0009which\u0009has\u0009not\u0009been\u0009used\u0009during\u0009\u000d\u000atraining\u0009or\u0009validation,\u0009is\u0009employed\u0009to\u0009estimate\u0009the\u0009accuracy\u0009of\u0009the\u0009model.\u0009\u000d\u000aLet\u0009us\u0009consider\u0009an\u0009example\u0009of training\u0009an\u0009ML\u0009model to\u0009predict\u0009a\u00093rd-degree\u0009polynomial (green\u0009curve)\u000d\u000aincluding\u0009 noise\u0009 (blue points),\u0009as\u0009 shown\u0009in Figure\u0009 3.2.\u0009Here,\u0009we\u0009 use\u0009a\u0009 polynomial\u0009with\u0009an\u0009increasing\u0009\u000d\u000anumber\u0009of\u0009parameters,\u0009M,\u0009to\u0009mimic\u0009the\u0009process\u0009of\u0009training\u0009an\u0009ML\u0009model.\u0009ML\u0009models\u0009are\u0009(nearly)\u0009always\u000d\u000aover-parameterised,\u0009meaning\u0009they\u0009use\u0009more\u0009parameters\u0009than\u0009necessary\u0009to\u0009model\u0009the data.\u0009The\u0009higher\u0009\u000d\u000athe\u0009number\u0009of parameters\u0009in\u0009an ML\u0009model,the\u0009better its capability\u0009to\u0009describe\u0009complex\u0009data.\u0009The\u0009process\u0009\u000d\u000aof\u0009training adjusts\u0009these\u0009parameters to\u0009best\u0009fit\u0009the\u0009training\u0009data.\u0009Therefore,\u0009a\u0009longer\u0009training\u0009duration\u0009\u000d\u000acan lead\u0009 to\u0009 an\u0009 improved\u0009 description\u0009 of\u0009 the\u0009 training\u0009 data.\u0009 In\u0009 our example,\u0009we\u0009 use\u0009 the\u0009 variable\u0009 M\u0009to\u0009\u000d\u000asymbolise the\u0009number\u0009of\u0009iterations used\u0009to\u0009train the\u0009supervised\u0009ML\u0009model.\u0009Initially\u0009(At\u0009M\u0009=\u00090),\u0009the\u0009ML\u0009\u000d\u000amodel will\u0009not\u0009describe\u0009the\u0009data\u0009very\u0009well,\u0009presented\u0009with an\u0009average value\u0009in Figure\u00093.2A\u0009(red\u0009curve).\u0009\u000d\u000aAfter\u0009some\u0009training\u0009(M\u0009=\u00091,\u0009Figure\u00093.2B),\u0009the\u0009ML\u0009model makes\u0009a\u0009prediction\u0009that\u0009is\u0009better resembling\u0009the\u0009\u000d\u000atraining\u0009data,\u0009represented\u0009by\u0009a\u0009line\u0009that\u0009approximates\u0009the\u0009simple\u0009increasing\u0009trend\u0009in\u0009the\u0009data.\u0009This\u0009stage,\u0009\u000d\u000awhere\u0009training\u0009is\u0009incomplete,\u0009is\u0009called\u0009underfitting.\u0009Further training\u0009(M\u0009=\u00093) enables the\u0009ML\u0009model to\u0009","\u0009 |\u0009Supervised\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a90\u000d\u000acapture the\u0009 trend\u0009 of\u0009 a\u0009 3rd-degree\u0009 polynomial,\u0009 representing optimal\u0009 training.\u0009 However, if\u0009 training\u0009\u000d\u000acontinues beyond\u0009this\u0009point,\u0009the\u0009ML\u0009model will\u0009keep\u0009minimising\u0009the\u0009loss function,\u0009fitting all\u0009the\u0009training\u0009\u000d\u000adata\u0009points\u0009(Figure\u00093.2D).\u0009Although\u0009this\u0009behaviour\u0009allows\u0009the\u0009model\u0009to fit\u0009the\u0009training\u0009data\u0009well (blue\u0009\u000d\u000adots),\u0009it\u0009hinders\u0009its\u0009ability\u0009to generalise\u0009well\u0009to\u0009other\u0009points\u0009in\u0009a\u00093rd-degree\u0009polynomial (green\u0009curve).\u0009\u000d\u000aThis\u0009behaviour\u0009is\u0009called\u0009overfitting.\u0009The\u0009validation\u0009set\u0009can\u0009be\u0009used\u0009to\u0009monitor\u0009model\u0009performance\u0009on\u0009\u000d\u000aunseen\u0009data\u0009points,\u0009and\u0009thereby,\u0009it can\u0009be\u0009used\u0009to\u0009stop\u0009training\u0009before\u0009overfitting\u0009occurs.\u0009By\u0009checking\u0009\u000d\u000athe\u0009model\u2019s\u0009performance\u0009on the\u0009validation\u0009set\u0009for\u0009each\u0009iteration,\u0009the\u0009optimal\u0009training\u0009iterations\u0009can\u0009be\u0009\u000d\u000adetermined\u0009 (in\u0009 this\u0009 case,\u0009 M\u0009 =\u0009 3 in\u0009 Figure\u0009 3.2C).\u0009 Another\u0009 way\u0009to avoid\u0009 overfitting\u0009is\u0009 to\u0009increase\u0009 the\u0009\u000d\u000atraining\u0009set\u0009size,\u0009although\u0009this\u0009will extend\u0009the training\u0009time.\u0009\u0009\u000d\u000aFigure\u00093.2:\u0009Modelling\u0009of\u0009a\u00093rd-degree\u0009polynomial\u0009with\u0009noise\u0009using\u0009A)\u0009A\u0009constant,\u0009B)\u0009A\u0009line,\u0009C)\u0009A\u00093rd-degree\u0009polynomial\u0009\u000d\u000aand\u0009D)\u0009A\u000911th-degree\u0009polynomial. Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u0009my\u0009Master\u2019s\u0009Thesis.1\u000d\u000aWhile\u0009training\u0009an\u0009ML\u0009model\u0009can\u0009be\u0009computationally\u0009expensive,\u0009it is\u0009a\u0009one-time\u0009cost.\u0009Once\u0009trained,\u0009the\u0009\u000d\u000aML\u0009 model\u2019s predictions\u0009 are\u0009 computationally\u0009 inexpensive,\u0009 allowing\u0009 integration into\u0009 web-based\u0009\u000d\u000aapplications,\u0009 synchrotrons,\u0009 and\u0009 neutron\u0009 sources\u0009 for\u0009 real-time\u0009 structure\u0009 prediction.\u0009 As\u0009 a\u0009 result,\u0009 both\u0009\u000d\u000astructure-\u0009and\u0009structural\u0009parameter\u0009prediction\u0009tasks\u0009have the\u0009potential\u0009to\u0009accelerate\u0009the\u0009analysis\u0009of\u0009both\u0009\u000d\u000ascattering\u0009and\u0009spectroscopy\u0009data\u0009in\u0009materials\u0009chemistry.\u000d\u000aWhile\u0009 the\u0009described process\u0009is\u0009general\u0009 for\u0009all\u0009supervised\u0009ML,\u0009other\u0009 factors\u0009need\u0009consideration\u0009when\u0009\u000d\u000atraining\u0009 an\u0009 ML\u0009 model.\u0009 These\u0009 include the\u0009 choice\u0009 of\u0009 ML\u0009 algorithm\u0009 (tree-based\u0009 methods,\u0009 deep\u0009 neural\u0009\u000d\u000anetworks,\u0009support\u0009vector\u0009machine,\u0009etc.),\u0009the\u0009number\u0009of\u0009parameters\u0009in\u0009the\u0009ML\u0009model,\u0009and\u0009the\u0009quality\u0009and\u0009\u000d\u000asize\u0009of\u0009the\u0009training\u0009set.\u0009For\u0009instance,\u0009the\u0009model's\u0009ease\u0009of\u0009training\u0009and\u0009deployment\u0009can\u0009be\u0009influenced\u0009by\u0009\u000d\u000athe\u0009 choice\u0009 of\u0009ML\u0009 algorithm.\u0009 Interpretability often\u0009 varies\u0009with\u0009 the\u0009 type\u0009 of\u0009 algorithm\u0009 used\u0009\u2014\u0009 simpler\u0009\u000d\u000amodels\u0009like\u0009tree-based\u0009ML\u0009models are\u0009easier\u0009to\u0009interpret\u0009than\u0009deep\u0009neural\u0009networks.\u0009When\u0009it\u0009comes\u0009to\u0009\u000d\u000ascalability,\u0009 neural\u0009 networks\u0009 often\u0009 prove\u0009 more\u0009 effective\u0009 as\u0009 they\u0009 can\u0009 handle\u0009 larger\u0009 datasets\u0009 more\u0009\u000d\u000aefficiently\u0009 than\u0009 tree-based\u0009methods.\u0009The\u0009quality\u0009and\u0009size\u0009of\u0009 the\u0009 training\u0009set\u0009play\u0009a\u0009crucial role\u0009in\u0009 the\u0009\u000d\u000amodel's\u0009efficiency\u0009and\u0009accuracy,\u0009with\u0009larger,\u0009 higher-quality\u0009 datasets\u0009 typically\u0009 yielding\u0009 better\u0009 results.\u0009\u000d\u000aFinally,\u0009a\u0009model's\u0009ability\u0009 to\u0009interpolate\u0009and\u0009extrapolate,\u0009 or\u0009make\u0009predictions\u0009within\u0009 the\u0009 range\u0009 of\u0009 the\u0009\u000d\u000atraining\u0009data\u0009and\u0009beyond\u0009it,\u0009is\u0009generally\u0009influenced\u0009by\u0009the\u0009ML\u0009algorithm\u0009and\u0009the\u0009range\u0009and\u0009diversity\u0009of\u0009\u000d\u000athe\u0009training\u0009set.\u000d\u000aIn\u0009the\u0009following\u0009two\u0009chapters,\u0009I\u0009will\u0009present\u0009two\u0009studies\u0009using\u0009supervised\u0009ML\u0009to\u0009analyse\u0009PDF,\u0009SAXS\u0009and\u0009\u000d\u000aPD data.\u0009However,\u0009 the\u0009 developed\u0009methodologies are intended to\u0009 be general\u0009 for\u0009both\u0009 scattering\u0009 and\u0009\u000d\u000aspectroscopy\u0009data\u0009analysis.\u0009\u0009\u000d\u000a3.1 Structure\u0009Identification from\u0009Scattering\u0009Data*\u000d\u000aOne\u0009viable\u0009strategy to\u0009overcome\u0009the\u0009challenge\u0009associated\u0009with structural\u0009model\u0009identification\u0009is\u0009to\u0009use\u0009\u000d\u000asupervised classification\u0009ML\u0009models.\u0009In\u0009this\u0009context,\u0009an ML\u0009model\u0009is\u0009trained\u0009on\u0009a\u0009database\u0009comprising\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009IV.\u000d\u000ax\u000d\u000ay\u000d\u000ax\u000d\u000ay\u000d\u000ax\u000d\u000ay\u000d\u000ax\u000d\u000ay\u000d\u000aA M = 0 B M = 1 C M = 3 D M = 11","\u0009 |\u0009Supervised\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a62\u000d\u000astructural\u0009models and\u0009 simulated\u0009 scattering\u0009data with\u0009 the\u0009 objective of identifying the\u0009 corresponding\u0009\u000d\u000astructure\u0009from\u0009a\u0009given\u0009dataset.\u0009While\u0009the\u0009computational\u0009cost\u0009of\u0009the\u0009training\u0009process\u0009can\u0009be\u0009substantial,\u0009\u000d\u000ait\u0009is\u0009a\u0009one-time cost. After\u0009 training,\u0009 the\u0009ML\u0009model\u0009can\u0009be\u0009integrated\u0009into\u0009large-scale\u0009radiation\u0009 facility\u0009\u000d\u000asoftware\u0009or\u0009as\u0009web applications allowing\u0009users\u0009to\u0009instantaneously\u0009(<1\u0009second)\u0009identify\u0009 the\u0009structure\u0009\u000d\u000afrom\u0009a\u0009dataset.\u0009\u0009\u000d\u000aWe\u0009 introduce\u0009 a\u0009 high-throughput\u0009 database\u0009 screening method\u0009 (POMFinder) designed\u0009 for\u0009 the\u0009\u000d\u000aidentification\u0009 of\u0009 polyoxometalate\u0009 (POM)\u0009 clusters\u0009 from\u0009 PDFs while\u0009 they\u0009 are\u0009 being\u0009 measured.\u000d\u000a67 The\u0009\u000d\u000amethod\u0009uses\u0009a\u0009tree-based\u0009ML\u0009classifier,\u0009that\u0009has\u0009been\u0009trained\u0009on\u0009simulated\u0009PDFs\u0009derived\u0009from\u0009443\u0009POM\u0009\u000d\u000aclusters.\u0009 Here,\u0009 a tree-based\u0009 ML classifier\u0009 makes\u0009 a\u0009 series\u0009 of\u0009 \"yes/no\"\u0009 decisions,\u0009 based\u0009 on\u0009 the\u0009 PDF\u0009\u000d\u000aintensities, to\u0009identify the\u0009POM\u0009structure.\u0009These\u0009POMs\u0009have\u0009been\u0009extracted\u0009from\u0009crystal\u0009structures from\u0009\u000d\u000aCOD\u0009and\u0009 ICSD.\u000d\u000a37,38 POM\u0009clusters\u0009are\u0009a\u0009 family\u0009of\u0009large\u0009polyanion\u0009clusters\u0009mostly\u0009constructed\u0009of\u0009 [MO6]\u0009\u000d\u000aoctahedra,\u0009where\u0009M\u0009often\u0009is\u0009either Mo,\u0009W,\u0009V\u0009or\u0009Nb.68-71 POMs\u0009have\u0009been\u0009investigated extensively due\u0009to\u0009\u000d\u000atheir\u0009diverse chemistry\u0009and\u0009their significant\u0009role\u0009in the\u0009formation\u0009mechanism of\u0009metal\u0009oxide\u0009crystals.10,72\u000d\u000aThey have\u0009 predominantly\u0009 been\u0009 examined\u0009 in\u0009 their\u0009 crystalline\u0009 form,\u0009 however,\u0009 the\u0009 application\u0009 of\u0009 PDF\u0009\u000d\u000aanalysis\u0009enables\u0009studies\u0009of POMs in\u0009solution as\u0009well,\u0009which\u0009paves the\u0009way\u0009for\u0009novel\u0009insights\u0009into\u0009their\u0009\u000d\u000achemistry.\u0009\u0009\u000d\u000aPOMFinder exhibits\u0009 an\u0009 accuracy\u0009 of\u0009 94.0\u0009 %\u0009 on\u0009 simulated\u0009 PDFs and maintains good\u0009 results on\u000d\u000aexperimental\u0009PDF\u0009data,\u0009including\u0009data\u0009collected\u0009with\u0009fast\u0009acquisition\u0009time\u0009during\u0009an\u0009in\u0009situ experiment.\u000d\u000aPOMFinder\u0009has\u0009comparable\u0009performance\u0009when\u0009trained\u0009and\u0009employed on\u0009SAXS\u0009data.\u0009The\u0009code\u0009is\u0009opensource,\u0009available\u0009on\u0009GitHub\u0009and\u0009can\u0009be\u0009installed\u0009via\u0009a\u0009pip\u0009installation\u0009of\u0009a\u0009Python\u0009library.\u0009The\u0009code\u0009is\u0009also\u0009\u000d\u000aintegrated\u0009into\u0009a\u0009Hugging\u0009Face\u0009web\u0009application,\u0009enabling\u0009users\u0009with\u0009no\u0009prior\u0009ML\u0009experience to\u0009rapidly\u0009\u000d\u000aemploy\u0009POMFinder\u0009in a\u0009simple\u0009two-click\u0009process\u0009for\u0009the\u0009identification of\u0009POM\u0009clusters from\u0009PDF\u0009data.\u0009\u000d\u000aThe\u0009Hugging\u0009Face\u0009integration\u0009provides\u0009a\u0009user-friendly\u0009experience,\u0009without\u0009requiring\u0009data\u0009storage\u0009or\u0009\u000d\u000acomplicated software\u0009installations. As\u0009seen\u0009in\u0009Figure\u00093.3,\u0009the\u0009user\u0009simply\u0009clicks \u201cBrowse\u0009files\u201d,\u0009uploads\u000d\u000aa\u0009PDF,\u0009and\u0009POMFinder\u0009will\u0009output\u0009the\u0009top-5\u0009structural\u0009models\u0009and\u0009the\u0009model\u0009estimated probability\u0009of\u0009\u000d\u000abeing\u0009that\u0009given\u0009POM. The\u0009probability\u0009serves\u0009as\u0009a\u0009proxy\u0009of\u0009how\u0009confident\u0009POMFinder is\u0009in\u0009its\u0009predictions.\u0009\u000d\u000aIn\u0009contrast to\u0009the\u0009conventional\u0009modelling\u0009approach,\u0009where\u0009PDFs\u0009from\u0009all\u0009POM\u0009clusters\u0009in\u0009the\u0009database\u0009\u000d\u000aare fitted\u0009to\u0009the\u0009data\u0009using a\u0009least-squares\u0009refinement,\u0009POMFinder\u0009quickly\u0009reduces\u0009candidate\u0009structures\u0009\u000d\u000ato\u0009a\u0009manageable\u0009number. The\u0009entire\u0009process\u0009of\u0009uploading\u0009a\u0009PDF\u0009to\u0009POMFinder\u0009(Figure\u00093.3A),\u0009achieving\u0009\u000d\u000afive\u0009candidate\u0009structures\u0009(Figure\u00093.3B),\u0009and\u0009fitting\u0009them\u0009(Figure\u00093.3C)\u0009is\u0009visualised\u0009in\u0009Figure\u00093.3.\u0009Note\u000d\u000athat\u0009 the\u0009 web\u0009 application\u0009 can\u0009 only\u0009 predict\u0009 the\u0009 structural\u0009 model\u0009 (Figure\u0009 3.3A+B),\u0009 while\u0009 subsequent\u0009\u000d\u000aspecialised\u0009software\u0009is\u0009still\u0009required\u0009for\u0009performing\u0009the\u0009fitting\u0009process.\u000d\u000aThis\u0009research\u0009demonstrates\u0009the\u0009potential\u0009of\u0009using\u0009an\u0009ML-driven\u0009approach\u0009to\u0009rapidly\u0009screen\u0009a\u0009database\u0009\u000d\u000aof\u0009 known\u0009 structures\u0009 for\u0009 the\u0009 analysis\u0009 of\u0009 scattering\u0009 data. Our\u0009 findings\u0009 also\u0009 demonstrate that\u0009 ML\u0009 can\u0009\u000d\u000aidentify\u0009 the\u0009 structural\u0009 model\u0009 for\u0009 structure\u0009 refinement\u0009 from\u0009 multiple\u0009 dataset\u0009 without\u0009 the\u0009 need\u0009 of\u0009\u000d\u000aweighting\u0009of\u0009data\u0009contributions which\u0009is\u0009a\u0009problem\u0009for\u0009conventional\u0009complex\u0009modelling\u0009methods.\u000d\u000a13,24,51\u000d\u000aSpecifically,\u0009POMFinder\u0009was\u0009trained\u0009on\u0009both\u0009PDF\u0009and\u0009SAXS\u0009data without\u0009weighing\u0009the\u0009datasets.\u0009Here,\u0009\u000d\u000aPOMFinder\u0009consistently\u0009achieved\u0009higher\u0009accuracy\u0009than\u0009if\u0009it was\u0009trained\u0009on\u0009PDF\u0009or\u0009SAXS\u0009data\u0009alone. The\u0009\u000d\u000asupervised\u0009 ML strategy can also be\u0009 adapted\u0009 to\u0009 analyse\u0009 data\u0009 from\u0009 various\u0009 other\u0009 scattering\u0009 and\u0009\u000d\u000aspectroscopy\u0009techniques, or\u0009it\u0009can\u0009be extended\u0009to\u0009include a\u0009broader variety\u0009of\u0009chemical\u0009systems\u0009during\u0009\u000d\u000athe\u0009 training\u0009 process.\u0009An\u0009 example\u0009 of\u0009 a\u0009 supervised\u0009 ML\u0009 model\u0009 that\u0009 was\u0009 trained\u0009 on\u0009 a\u0009 larger\u0009 variety\u0009 of\u0009\u000d\u000achemical\u0009structures\u0009is\u0009CIF-Finder.\u0009The\u0009CIF-Finder\u0009paper, where\u0009I\u0009am\u0009the\u00092nd author,\u0009is\u0009in\u0009APPENDICES\u0009\u2013\u0009\u000d\u000aPAPERS.\u0009The\u0009ML\u0009model is\u0009a\u0009 tree-based\u0009model\u0009trained\u0009on\u0009simulated\u0009PDFs\u0009derived\u0009 from 10,900\u0009metal\u0009\u000d\u000aoxides.\u0009After\u0009training,\u0009CIF-Finder identifies metal\u0009oxides\u0009from\u0009PDF\u0009data with\u0009a\u0009top-3\u0009accuracy\u0009of\u000990\u0009%\u0009\u000d\u000aon\u0009 simulated\u0009 PDFs\u0009 and\u0009 it\u0009 works\u0009 well\u0009 on\u0009 experimental\u0009 PDFs\u0009 measured on\u0009 metal\u0009 oxides,\u0009 ultra-small\u0009\u000d\u000ananoparticles\u0009and\u0009fast\u0009acquisition\u0009data\u0009measured\u0009during\u0009an\u0009in\u0009situ experiment.\u0009\u0009","\u0009 |\u0009Supervised\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a92\u000d\u000aBoth\u0009POMFinder\u0009and\u0009CIF-Finder\u0009are\u0009tree-based\u0009classification\u0009models.\u0009We\u0009chose to\u0009use\u0009tree-based\u0009ML\u0009\u000d\u000amodels because\u0009they have\u0009proven\u0009to\u0009provide\u0009state-of-the-art\u0009results\u0009on\u0009classification\u0009benchmarks,\u000d\u000a73 they\u0009\u000d\u000aare\u0009interpretable,\u0009and\u0009they\u0009are\u0009relatively\u0009easy\u0009to\u0009train\u0009and\u0009deploy.\u0009However,\u0009for\u0009large\u0009amounts\u0009of\u0009data,\u0009\u000d\u000atree-based\u0009methods\u0009do\u0009not\u0009scale\u0009well\u0009as\u0009 the\u0009complexity\u0009of\u0009 the\u0009 trees\u0009increases which\u0009consumes\u0009large\u0009\u000d\u000aamounts\u0009 of\u0009computer\u0009memory.\u0009Therefore,\u0009a\u0009neural\u0009network\u0009will\u0009be\u0009beneficial\u0009 for\u0009 future\u0009 revisions\u0009 of\u0009\u000d\u000athese\u0009supervised\u0009ML\u0009models since\u0009they\u0009can be\u0009scaled\u0009to\u0009train on\u0009a\u0009larger\u0009variety\u0009of\u0009structures.\u000d\u000aFigure\u00093.3:\u0009A)\u0009Overview\u0009of\u0009the\u0009POMFinder tool's\u0009ease\u0009of\u0009use\u0009for\u0009structural\u0009model predictions\u0009from\u0009PDF data.\u0009Simply\u0009\u000d\u000aclick\u0009\u201cBrowse\u0009files\u201d,\u0009upload\u0009a\u0009PDF\u0009and\u0009wait\u0009for\u0009the\u0009model\u0009to\u0009predict\u0009the\u0009structural\u0009model.\u0009Finally,\u0009download\u0009the\u0009top5\u0009 structural\u0009 models.\u0009 All\u0009 of\u0009 this\u0009 is\u0009 possible\u0009 through\u0009 the\u0009 Hugging\u0009 Face\u0009 app.\u0009\u000d\u000ahttps://huggingface.co/spaces/AndySAnker/POMFinder\u0009 B)\u0009 The top-5\u0009 predictions\u0009 are\u0009 1)\u0009 a\u0009W11O35\u0009Keggin-based\u0009\u000d\u000afragment\u0009from\u0009the\u0009dimeric\u0009K5.5Na7Nd(SiW11O39 (H2O))2(CH3COO)2(H2O)10 complex74,\u00092)\u0009a\u0009W12O36 fragment\u0009from\u0009the\u0009\u000d\u000aK5H(CoW12O40)\u0009 (H2O)15 crystal75,\u0009 3)\u0009 a\u0009 W12O40\u0009 fragment\u0009 from\u0009 an\u0009 ionic\u0009 crystal\u0009 structure76 of\u0009\u000d\u000a(Al13O4(OH)24(H2O)12)(H2W12O40)(OH)(H2O)23.12,\u0009 4)\u0009 a\u0009 W12O36\u0009 fragment\u0009 from\u0009 the\u0009 porous\u0009 inorganic\u0009 of\u0009 the\u0009 form\u0009\u000d\u000aK2NaH2(BW12O40)(H2O)12,\u000d\u000a77 and\u0009 5)\u0009 a\u0009 W12Rb4BO43\u0009 fragment\u0009 from\u0009 another\u0009 ionic\u0009 crystal78\u000d\u000aRb4(Cr3O(OOCH)6(H2O)3(BW12O40)(H2O)16.\u0009 W\u0009 is\u0009 shown\u0009 in\u0009 brown,\u0009 O\u0009 in\u0009 red,\u0009 and\u0009 Rb\u0009 in\u0009 pink.\u0009 C)\u0009 Afterwards,\u0009 the\u0009\u000d\u000astructural\u0009 models\u0009 can\u0009 be\u0009 fitted\u0009 to\u0009 the\u0009 experimental\u0009 PDF.\u0009 Here,\u0009 fits\u0009 of the\u0009 top-5\u0009 predictions.\u0009 The\u0009 refinement\u0009\u000d\u000aparameters\u0009are\u0009reported\u0009in\u0009Appendix\u0009III.\u0009Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u0009Paper\u0009IV.67\u000d\u000a3.1.1 Paper\u0009IV\u000d\u000aOn\u0009 the\u0009 following\u0009 pages,\u0009 you\u0009 will\u0009 find\u0009 Paper\u0009 IV\u0009which\u0009 is\u0009 uploaded\u0009 to\u0009 ChemRxiv.\u0009 The\u0009 supplementary\u0009\u000d\u000ainformation\u0009for\u0009Paper\u0009IV\u0009can\u0009be\u0009found\u0009in\u0009APPENDICES\u0009\u2013\u0009PAPERS.","POMFinder: Identifying polyoxometalate cluster structures from pair distribution function \u000d\u000adata using explainable machine learning \u000d\u000aAndy S. Anker1\u000d\u000a, Emil T. S. Kj\u00e6r1\u000d\u000a, Mikkel Juelsholt2\u000d\u000a and Kirsten M. \u00d8. Jensen1\u000d\u000a* \u000d\u000a*Correspondence to kirsten@chem.ku.dk (KM\u00d8J) \u000d\u000a1: Department of Chemistry & Nano-Science Center, University of Copenhagen, 2100 Copenhagen \u000d\u000a\u00d8, Denmark \u000d\u000a2: Department of Materials, University of Oxford, Parks Road, Oxford, OX1 3PH, UK \u000d\u000aAbstract \u000d\u000aCharacterisation of material structure with Pair Distribution Function (PDF) analysis typically \u000d\u000ainvolves refining a structure model against an experimental dataset. However, finding or constructing \u000d\u000aa suitable atomic model for PDF modelling can be an extremely labour-intensive task, requiring \u000d\u000acarefully browsing through large numbers of possible models. We present POMFinder, a machine \u000d\u000alearning (ML) classifier that rapidly screens a database of structures, here polyoxometalate (POM) \u000d\u000aclusters, to identify candidate structures for PDF data modelling. The approach is demonstrated to \u000d\u000aidentify suitable POMs on experimental data, including in situ data collected with fast acquisition \u000d\u000atime. This automated approach shows significant potential for identifying suitable structure models \u000d\u000afor structure refinements to extract quantitative, structural parameters in materials chemistry research. \u000d\u000aThe code is open source and user-friendly, making it accessible to those without prior ML knowledge. \u000d\u000aWe also demonstrate that POMFinder offers a promising modelling framework for combined \u000d\u000amodelling of multiple scattering techniques compared to conventional refinement methods. \u000d\u000aIntroduction \u000d\u000aThe continued development of increasingly bright synchrotron and neutron facilities means that \u000d\u000ascattering and spectroscopy data can now be measured at impressive speeds.1-3 Hundreds of gigabytes \u000d\u000aor even terabytes of data are now commonly collected in each experiment, each containing thousands \u000d\u000aor millions of individual measurements. With this amount of data, it is an enormous challenge to \u000d\u000awork through each dataset manually, and the development of automated methods for data analysis is \u000d\u000athus becoming more and more necessary.2,4,5\u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","For many X-ray and neutron based scattering techniques such as small-angle scattering, powder \u000d\u000adiffraction, and total scattering with pair distribution function (PDF) analysis, data analysis is often \u000d\u000adone through least-square optimisation.6-8 Here, structure models found in, e.g., structure databases, \u000d\u000aare used to simulate data, which are then refined against experimental data. This allows to extract \u000d\u000aquantitative, structural parameters. This approach can, in principle, be automated by, e.g. testing \u000d\u000aentire databases of structures against experimental datasets.9-12 However, least-squares fitting \u000d\u000aalgorithms are computationally expensive, which makes them unsuited for automatically identifying \u000d\u000aand refining structures for experiments with many datasets.1\u000d\u000a Consequently, identifying structural \u000d\u000amodels is today a bottleneck for modelling large quantities of scattering data. \u000d\u000aIn this study, we present a tree-based machine learning (ML) classifier that instantaneously identifies \u000d\u000aa chemical structure from a PDF, enabling high-throughput database screening. The PDF here refers \u000d\u000ato the reduced Pair Distribution Function, G(r), which represents a histogram of real-space \u000d\u000ainteratomic distances and can be used to identify atomic arrangements in materials. G(r) is obtained \u000d\u000aby Fourier transforming the total scattering structure function, S(Q), which is the corrected and \u000d\u000anormalised total scattering data:13\u000d\u000a!(#) \u0009 = \u0009\u00092/)\u0009\u0009 \u222b \u0009+[-(+) \u0009\u2212 \u00091]\u0009123(+\u0009 \u00b7 \u0009#)5+ !!\"#\u000d\u000a!!$% Eq. 1 \u000d\u000aThe Q-range used for modern total scattering experiments ranges from 0.1\u20131 \u00c5-1 to 15\u201330 \u00c5-1. PDF \u000d\u000ahas in recent years shown to be a powerful technique for characterisation of disordered materials,12,14-\u000d\u000a16 amorphous materials,17-19 clusters in solution,20-22 and nanomaterials,23,24 where conventional \u000d\u000acrystallographic approaches are challenged.15,16 PDFs are usually analysed by fitting a reasonable \u000d\u000astarting model to the experimental PDF using dedicated software such as DiffPy-CMI,25 DISCUS,26,27\u000d\u000aor TOPAS.\u000d\u000a28 In some cases, for example, for well-characterised, crystalline materials, identifying a \u000d\u000astarting model for structural refinements is easily done. In other cases, finding or constructing a good \u000d\u000ainitial atomic model for modelling the PDF can be an extremely labour-intensive task, requiring \u000d\u000acarefully browsing through large numbers of possible starting models. However, we and others have \u000d\u000ashown that ML methods such as neural networks and tree-based ML have much potential to improve \u000d\u000athe speed of PDF analysis.29-35 ML has for example been used to identify crystallographic space \u000d\u000agroups from PDFs,30 extract structural motifs31,34,35 or to determine the structure of small, metallic \u000d\u000ananoparticles.29,32 \u000d\u000aWe here use a tree-based ML classifier to identify the structure of polyoxometalate (POM) clusters \u000d\u000ain solution based on a PDF. POM clusters are a family of large polyanion clusters mostly constructed \u000d\u000aof [MO6] octahedra, where M is often Mo, W, V or Nb.36-39 POMs have been extensively studied \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","both due to their rich chemistry and their many applications in e.g. molecular magnets, as catalysts \u000d\u000afor water splitting, as conductors, or in medicine.36-39 Furthermore, it has been shown that the \u000d\u000aformation of metal oxide crystals can be dependent on the structure of the POM cluster, which has a \u000d\u000ahuge impact on the formation mechanism.12,40 While POMs have so far been mainly studied in the \u000d\u000acrystalline form, PDF analysis allows POM structure studies in solution, which open for a new \u000d\u000aunderstanding of their chemistry. \u000d\u000aThe ML model, which we refer to as POMFinder, has been trained on simulated PDFs from 443 POM \u000d\u000aclusters, cut out of crystal structures containing POMs obtained from the Crystallography Open \u000d\u000aDatabase (COD)41 and the Inorganic Crystal Structure Database (ICSD).42 POMFinder allows \u000d\u000aidentifying POM structures from PDFs and has an accuracy of 94.0 % on simulated data within the \u000d\u000a1st prediction. It also shows good performance on experimental PDF data. We use SHapley Additive \u000d\u000aexPlanations (SHAP)43,44 analysis to understand the predictions of POMFinder. With SHAP analysis, \u000d\u000awe can calculate the contribution of each input feature in the ML model to its predictions. Using \u000d\u000aSHAP analysis on simulated PDFs from X-rays (xPDF), neutrons (nPDF) and electrons (ePDF), we \u000d\u000ashow that POMFinder learns trends corresponding to the scattering power of the different elements \u000d\u000ain POMs and uses this information in its predictions. Finally, we show that the method can be \u000d\u000aextended to jointly use data from multiple scattering techniques instead of analysing the data \u000d\u000aseparately: comparable to the \u2018complex modelling\u2019 approach.23 We use simulated xPDF, small-angle \u000d\u000ax-ray scattering (SAXS) data, nPDF and ePDF, as well as combinations of the above datasets. A \u000d\u000acommon problem of complex modelling is to weigh the datasets;20,25,45 however, this is not necessary \u000d\u000awhen using ML to identify the structural model. \u000d\u000aConstruction of the POM database and training of the POMFinder model \u000d\u000aWe aim to create an ML model that can quickly and efficiently match an atomic POM structure to \u000d\u000aexperimental data. We have chosen to focus on X-ray PDF data as the structural characterisation \u000d\u000atechnique, and POMs as the structures of interest. However, in principle, the dataset can be any \u000d\u000ainformation that can be modelled using an atomistic model and represented in a tabular data format. \u000d\u000aThe goal is not to have an algorithm that can output the perfect model to all experimental data every \u000d\u000atime with no user input. Instead, the successful ML model can filter out all bad models and give the \u000d\u000auser a handful of models which can be used for further analysis. \u000d\u000aA pseudo-code of how to create the POM database and train POMFinder can be seen in Figure 1. The \u000d\u000astructural database of POM clusters is built from CIF files obtained from both the COD and ICSD \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","using chemical restraints appropriate for POM clusters, as discussed below. Afterwards, a number \u000d\u000a(N) of PDFs are simulated for each POM structure with varying simulation parameters (Qmin, Qmax, \u000d\u000aan instrumental dampening parameter Qdamp and an isotropic atomic displacement parameter, ADP). \u000d\u000aThe Q-range used in the PDF (eq. 1) affects the r-resolution of the PDF, and the limited Q-range (Qmin\u000d\u000a\u2013 Qmax) creates termination ripples in the PDF.13 Therefore, automated PDF data analysis must be \u000d\u000aapplicable across various Q-ranges. The parameters are varied using Latin hypercube sampling,46 and \u000d\u000aa gradient boosting decision trees (GBDT) model47 is trained to classify which POM structure best \u000d\u000amatches the input data. In the following sections, we will elaborate on this process. The database of \u000d\u000aPOM clusters and the code used to train POMFinder is available at \u000d\u000ahttps://github.com/AndySAnker/POMFinder/. \u000d\u000aFigure 1: Pseudo-code describing how to create POM clusters from a CIF database and how to \u000d\u000atrain POMFinder. A POM database is built from ICSD and COD by cutting out clusters from all \u000d\u000acrystal structures with chemical compositions similar to POM clusters in solution (step 1).37 A \u000d\u000anumber of PDFs are then simulated with various parameters for each POM cluster (step 2), which are \u000d\u000aused to train a GBDT model to classify which structure the dataset matches with (step 3). \u000d\u000aBuilding a database of polyoxometalate clusters \u000d\u000aThe COD & ICSD databases contain hundreds of thousands of crystallographic information files \u000d\u000a(CIFs). When building our database, we first screened for CIFs with the same metal-oxygen ratios \u000d\u000adescribed in a comprehensive review about POM clusters in solution by N. I. Gumerova and A. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","Rompel.37 This restrained the database to 56 different metal \u2013 oxygen ratios, yielding 1281 CIFs. \u000d\u000aClusters were then cut out of the CIFs by creating a 2x2x2 unit cell of the crystal and extracting all \u000d\u000aclusters of atoms not bonded to other atoms in the structure. Some clusters span more than a single \u000d\u000aunit cell, so to capture the complete POM cluster, a 2x2x2 unit cell was needed. Next, all isolated \u000d\u000aclusters that did not fulfil the chemical restraints (the 56 different metal \u2013 oxygen ratios) were \u000d\u000aremoved. Figure 2 illustrates an example of a cluster that was cut out of a crystal built from Keggin \u000d\u000apolyoxoanions, K2NaH2[BW12O40]\u221912H2O.48\u000d\u000aFigure 2: A POM cluster cutout from a crystal structure. A) The crystal structure of \u000d\u000aK2NaH2[BW12O40]\u221912H2O48 and B) the corresponding POM cluster. W is shown in brown, O in red, \u000d\u000aNa in yellow, B in blue and K in grey. H is omitted for clarity. \u000d\u000aThis procedure yielded 969 potential polyoxometalate clusters. The simulated PDFs\u2019 Pearson \u000d\u000acorrelation coefficient (PCC) was used to remove similar structures from the database.49 The PDFs \u000d\u000awere compared iteratively by simulating a PDF of the 1st and 2nd cluster with the parameters given in \u000d\u000asection A in the Supplementary Information (SI) and comparing their absolute PCC. The PCC is a \u000d\u000ameasure from -1 to 1 of how linearly correlated two continuous datasets are, where -1 represents \u000d\u000ainverse datasets, and 1 represents identical datasets. If the absolute PCC was higher than 0.99, the 2nd\u000d\u000acluster was not included in the database. The 3rd cluster was then compared to the 1st and 2nd cluster \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","by the same procedure, etc. The value of 0.99 was defined by manually inspecting structures, their \u000d\u000acorresponding PDFs and the PCC. Examples of 3 structures, their corresponding simulated PDFs and \u000d\u000athe PDFs Pearson value correlation can be seen in section A in the SI. This process was done with all \u000d\u000a926 structures, yielding 443 unique structures. We here note that it is not guaranteed that the clusters \u000d\u000aare perfectly cut out of the crystal structure, which makes it important for the user to inspect the \u000d\u000aresults of POMFinder and establish if they make chemical sense. \u000d\u000aSimulation of PDFs from the POM structures and training process of POMFinder \u000d\u000aFor each structure, a number (N) of PDFs were simulated with a broad range of instrumental \u000d\u000aparameters sampled using Latin hypercube sampling.46 The simulations were done using DiffPyCMI.25 The parameters are Qmin, Qmax, Qdamp, and the ADPs. Section B in the SI gives the range of \u000d\u000asimulation parameters of the PDF data. The PDFs are normalised to have G(r)max = 1, and all \u000d\u000aintensities up to r = 1 \u00c5 are set to 0 since the POM clusters are unlikely to have atomic distances that \u000d\u000acontribute to the signal in this range of the PDF. An example of an experimental PDF before and after \u000d\u000anormalisation is shown in Section B in the SI. \u000d\u000aThe simulated datasets and their corresponding instrumental parameters (Qmin, Qmax, Qdamp, and ADP) \u000d\u000aare input in a GBDT model. The GBDT algorithm used is XGBoost with default parameters except \u000d\u000afor the learning rate, which was set to 0.3 and the early stop criterion of 5 rounds without \u000d\u000aimprovement.47 The problem is a 443 class classification problem with an input of 443\u00d7N simulated \u000d\u000aPDFs. For each structure, two of the 100 simulated PDFs are randomly chosen and set aside during \u000d\u000athe training of the model and later used as validation- and test sets. The validation set is used to \u000d\u000avalidate when the GBDT model has converged. \u000d\u000aThe loss curve (multiclass log loss) is plotted in section C in the SI, which shows that the model can\u000d\u000apredict the training data with 100 % accuracy, while the validation data is predicted with a small loss. \u000d\u000aThe concluding accuracy of the model can be determined on the test set, which is data that the model \u000d\u000ahas not been trained or validated on, i.e. comparable to how POMFinder can be used for experimental \u000d\u000adata. When POMFinder is trained on 100 PDFs for each structure, the accuracy on the test set is 94.0 \u000d\u000a%. \u000d\u000aUse of POMFinder\u000d\u000aPOMFinder is a simple tool to use since everything is fully automated. As seen in Figure 3, one \u000d\u000asimply provides a dataset as input to POMFinder, and it will return a list of likely structures as output. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","The input here is a PDF, but it can, in principle, be any data that can be modelled using an atomistic \u000d\u000amodel and represented in a tabular data format. The output will be given in the XYZ format providing \u000d\u000athe elements and coordinates of all the atoms in the structure. \u000d\u000aFigure 3: Pseudo-code describing how to use POMFinder. The dataset is simply given to \u000d\u000aPOMFinder as input, which outputs an ordered list of suitable POM clusters from which a few can \u000d\u000abe fitted to the dataset.\u000d\u000aResults & Discussion \u000d\u000aIdentification of POM structures from experimental PDFs \u000d\u000aWe start by demonstrating the power of POMFinder on an experimental PDF from a 0.05 M aqueous \u000d\u000asolution of ammonium metatungstate hydrate, (NH4)6[H2W12O40]\u2219xH2O, which is known to yield \u000d\u000a[H2W12O40]\u000d\u000a6- ions with the 8-Keggin structure.40 The data was collected at the DanMAX beamline \u000d\u000a(MAX IV, Lund) using a wavelength of \u03bb = 0.3542 \u00c5, achieving a Qmax of 20 \u00c5\u22121. The acquisition \u000d\u000atime for the total scattering dataset was 15 minutes. Keggin structures (Figure 2B) have the chemical \u000d\u000acomposition [XM12O40]\u000d\u000an-, where X is a tetrahedrally coordinated cationic central atom in the middle \u000d\u000aof the cluster or 1\u20133 H+ ions, M is the metal atom of the cluster, and n is the negative charge of the \u000d\u000acluster. Keggin clusters are divided into 5 rotational isomers, with increasing degree of edge-sharing: \u000d\u000a8, :, ;, < and =,\u000d\u000a37,50,51 however, the < isomer is not present in our POM database. \u000d\u000aWhen giving the experimental PDF as input to POMFinder, the output is an ordered list of how \u000d\u000aprobable the PDF originates from each of the 443 POM structures in the POM database. The first five \u000d\u000aentries of the list are given in section D in the SI along with the probabilities assigned by POMFinder. \u000d\u000aThe list clearly shows a dominance of structures with W11-12O35-43 composition which corresponds to \u000d\u000aKeggin fragments. The five structures with the highest probability assigned by POMFinder are shown \u000d\u000ain Figure 4, along with the fitted structure to the experimental PDF. The first four candidate structures \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","fit the PDF reasonably well. The best candidate, Figure 4B, with a Rwp value of 29.6 %, is the 8-\u000d\u000aKeggin structure. All the other structures are also 8-Keggin structures or fragments. \u000d\u000aFigure 4: POMFinder top 5 predictions on experimental high quality X-ray PDF data. \u000d\u000aComparison of the PDF obtained from the 0.05 M ammonium metatungstate solution and the fitted \u000d\u000aPDF of A) a W11O35 Keggin-based fragment from the dimeric K5.5Na7Nd(SiW11O39\u000d\u000a(H2O))2(CH3COO)2(H2O)10 complex52, B) a W12O36 fragment from the K5H(CoW12O40) (H2O)15\u000d\u000acrystal53, C) a W12O40 fragment from an ionic crystal structure54 of \u000d\u000a(Al13O4(OH)24(H2O)12)(H2W12O40)(OH)(H2O)23.12, D) a W12O36 fragment from the porous inorganic \u000d\u000aof the form K2NaH2(BW12O40)(H2O)12,\u000d\u000a55 and E) a W12Rb4BO43 fragment from another ionic crystal56\u000d\u000aRb4(Cr3O(OOCH)6(H2O)3(BW12O40)(H2O)16. W is shown in brown, O in red, and Rb in pink. \u000d\u000aRefinement parameters are reported in section D in the SI.\u000d\u000aUsing POMFinder on fast acquisition datasets with a lower Qmax \u000d\u000aHaving established that POMFinder can identify a POM structure from a high-quality experimental \u000d\u000aPDF, we are interested in examining the use of POMFinder for data acquired with a fast time \u000d\u000aresolution, as is the case for in situ data. X-ray TS with PDF analysis is a powerful technique to study \u000d\u000athe formation of, e.g., oxides, and it has previously been shown that POM structures can play a large \u000d\u000arole in their formation.34,40,57,58 Therefore, we test POMFinder on fast acquisition experimental PDFs \u000d\u000awith 2 s time resolution from the 0.05 M solution of ammonium metatungstate. The data quality for \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","this dataset only allows a Qmax of 16 \u00c5-1. The data are the same as reported by Juelsholt et al.40 on the \u000d\u000aformation of tungsten oxide. The experimental PDF of the ammonium metatungstate solution (Figure \u000d\u000a5) shows a small structure with PDF peaks up to about 7 \u00c5. When inputting the PDF to POMFinder, \u000d\u000awe again obtain an ordered list of possible structures, with the best five listed in section D in the SI. \u000d\u000aFigure 5 shows the fit of the five best predictions on the experimental PDF. The best fitting POM \u000d\u000afragments, Figure 6A and 5D, are lacunary 8-Keggin structures with 3 out of 4 triads. The second \u000d\u000astructure, Figure 5B, also reasonably fits the experimental PDF with a 8-Keggin structure. In contrast, \u000d\u000athe structures in both Figure 5C and Figure 5E are too large to describe the experimental PDF well. \u000d\u000aNevertheless, using POMFinder, we can identify main motifs and thus determine a good model from \u000d\u000afast acquisition PDFs. \u000d\u000aFigure 5: POMFinder top 5 predictions on experimental, fast acquisition X-ray PDF data. \u000d\u000aComparison of the PDF from a 0.05 M solution of ammonium metatungstate in oleylamine with A) \u000d\u000aa W9SiO34 fragment from a Keggin-based Na2[C(NH2)3]2[{(CH3)2Sn(H2O)}3(A->-SiW9O34)]\u00b710H2O \u000d\u000acrystal59, B) a W12O36 fragment from the crystal structure of a porous framework based on Keggin \u000d\u000apolyoxoanions, K2NaH2[BW12O40]\u221912H2O.48, C) a W20O64 fragment from a pseudo-Keggin based \u000d\u000acrystal60 with chemical composition H(2-x)Bi2W20O70(HWO3), D) a SbW9O30 fragment from a \u000d\u000aK11[Sb3(SiW9O34)2]\u00b731H2O crystal structure61 and E) a V15O42 fragment from the bicapped Keggin \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","structure62 (TMA)3H6VV15042\u00b72.5H20 (TMA = tetramethylammonium). W is shown in brown, Sb in \u000d\u000agrey, O in red, Si in blue, and V in pink. Refinement parameters are reported in section D in the SI. \u000d\u000aAs discussed by Juelsholt et al.,\u000d\u000a40 another cluster appears when heating the 0.05 M solution of \u000d\u000aammonium metatungstate in oleylamine to 200 \u00b0C. The experimental PDF after ca 4 minutes of \u000d\u000aheating is shown in Figure 6. When inputting the PDF to POMFinder, we again obtain an ordered list \u000d\u000aof possible structures, with the best five listed in section D in the SI. Figure 6A-E shows the 5 first \u000d\u000astructures suggested by POMFinder and their fits to the PDF. The second prediction, Figure 6B, is \u000d\u000athe only POM fragment that reasonably fits the experimental PDF. This is a paratungstate POM, \u000d\u000awhich also was concluded by Juelsholt et al.40 POMFinder is thus very well suited for analysis of in \u000d\u000asitu data where small structural changes in the cluster structure are observed. \u000d\u000aFigure 6: POMFinder top 5 predictions on experimental fast acquisition X-ray PDF data.\u000d\u000aComparison of the PDF from 0.05 M solution of ammonium metatungstate in oleylamine heated to \u000d\u000a200 \u00b0C for 4 minutes and the calculated PDF of A) a W48O152 fragment from the polyanion \u000d\u000aK26.5Li9.5[H4As8W48O184]\u00b790H2O63, B) a W12O42 fragment from the acidic sodium polytungstates64\u000d\u000aNa5[H7W12O42]\u00b720H2O, C) a W11K3O38 fragment from the crystal structure65 K6H4W11O38\u00b7H2O, D) a \u000d\u000aW2O7 fragment from the crystal structure66 of Bi2W2O9, and E) a Re2O8 fragment from the crystal \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","structure Bi28Re2O49.\u000d\u000a67 W is shown in brown, K in grey, O in red, Si in blue, Re in green. Refinement \u000d\u000aparameters are reported in section D in the SI. \u000d\u000aRationalising POMFinder\u2019s predictions using SHAP values \u000d\u000aThe above results have established that POMFinder can identify the POM structure present in \u000d\u000asolutions from experimental PDFs. Yet, it is not clear what POMFinder bases its predictions on. To\u000d\u000aget this understanding, we use SHAP analysis. SHAP is a feature importance measure, which yields \u000d\u000ainformation about how the ML model exploits the individual features in the input data to make its \u000d\u000apredictions. Here, features are Qmin, Qmax, Qdamp and G(r) values for r-values between r = 0.0 \u00c5 and r \u000d\u000a= 10.0 \u00c5 with a step size of 0.1 \u00c5. A SHAP value is calculated for each feature for each PDF in the \u000d\u000atraining set. The amplitude of the calculated SHAP value for a given feature provides information \u000d\u000aabout how important the feature is, while the sign of the SHAP value tells if the feature is confirming \u000d\u000aor disqualifying the specific structure as a match to the dataset. Figure 7A and C shows a SHAP \u000d\u000aanalysis of the two fast acquisition PDFs discussed above, predicting the 8-Keggin and the \u000d\u000aparatungstate cluster, respectively. The top of the plot shows the SHAP values of the most important \u000d\u000afeatures, i.e. those that give the highest amplitude of SHAP values. The value of the features, in this \u000d\u000acase, the G(r) intensity, is indicated by colour: High PDF intensities (G(r) values) in the PDFs in the \u000d\u000atraining set are represented in red, while low G(r) values are coloured blue. For the 8-Keggin cluster, \u000d\u000athe SHAP analysis shows that the two most important features are the G(r) values at r = 6.0 \u00c5 and r \u000d\u000a= 3.6 \u00c5. When inspecting the PDF and POM structures, these r-values correspond to two W \u2013 W \u000d\u000adistances, as indicated in the structure drawing in Figure 7B. This means that POMFinder bases its \u000d\u000apredictions highly on PDF peaks arising from W \u2013 W distances. W has a high X-ray scattering power \u000d\u000acompared to oxygen (number of electrons: W: 74, O: 8), and W\u2013W peaks are thus much more \u000d\u000aprominent in X-ray PDFs compared to W \u2013 O or O \u2013 O peaks.68 For paratungstate, we observe the \u000d\u000asame trend, Figure 7C and D. We conclude that POMFinder predominantly bases its predictions on \u000d\u000athe intensities of the PDF peaks describing the 1st and 3rd metal \u2013 metal shell for these two structures. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","Figure 7: SHAP analysis of POMFinder on experimental PDFs. A) For every PDF in the test set,\u000d\u000aSHAP values are calculated for all PDF intensities (G(r) values) indicated with red for a peak and \u000d\u000ablue as a low intensity. The r-values of the PDF intensity are shown as labels. Here, only the impact \u000d\u000aof predicting the 8-Keggin cluster is shown, while C) shows the impact of predicting the paratungstate \u000d\u000acluster. B) Histogram of absolute SHAP values for each PDF intensity plotted versus the r-values on \u000d\u000atop of the PDFs of the B) 8-Keggin cluster and D) the paratungstate cluster. W is shown in brown, \u000d\u000aand O in red. \u000d\u000aInstead of using SHAP to explain how POMFinder makes its predictions on individual PDFs, it is \u000d\u000apossible to get a global explanation by calculating an average of all the absolute SHAP values from \u000d\u000athe 443 POM structures in the POM structure database (shown in section E in the SI). This analysis \u000d\u000ashows that the average absolute SHAP value for the Qmin, Qmax and Qdamp values is insignificant, \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","meaning that POMFinder is not sensitive to the provided user input of Qmin, Qmax and Qdamp in the \u000d\u000aranges used for training POMFinder. The average absolute SHAP value for G(r) values in the r = 0 \u2013 \u000d\u000a1 \u00c5 range is 0 since they are fixed to G(r<1 \u00c5) = 0. However, the rest of the G(r) values all have some \u000d\u000acontribution to the prediction of POMFinder. Especially the G(r) values corresponding to PDF peaks \u000d\u000afor M \u2013 O distances (~2.0 \u00c5), the 1st and 3rd metal \u2013 metal distances (~3.3 \u00c5 and ~6.2 \u00c5) are \u000d\u000aimportant in the prediction. \u000d\u000aTo confirm that the predictions from POMFinder relate to the scattering power of the elements, we \u000d\u000aconducted the same SHAP analysis on simulated nPDF and ePDF data. From the same POM \u000d\u000adatabase, we first simulated 100 nPDFs and ePDFs from each POM structure in our database with \u000d\u000adifferent Qmin, Qmax, Qdamp, and ADP parameters, trained a GBDT model using a 98:1:1 \u000d\u000atraining:validation:test set split and then applied SHAP analysis to investigate the results. Figure 8 \u000d\u000ashows the SHAP values for each feature in POMFinder when trained on the simulated xPDF, nPDF\u000d\u000aand ePDF compared with their relative simulated PDFs. \u000d\u000aWhen POMFinder is trained on nPDFs, the SHAP value is high for features corresponding to O \u2013 O \u000d\u000apeaks and W \u2013 O peaks. The neutron scattering lengths of W and O are 4.9 fm and 5.8 fm, \u000d\u000arespectively.68 This means that POMFinder bases its predictions on the O \u2013 O and the W \u2013 O peaks \u000d\u000awhen trained on nPDFs, rather than the W \u2013 W peaks as seen for xPDFs as discussed above. This is \u000d\u000alikely due to tungsten\u2019s and oxygen's comparable neutron scattering length in contrast to the scattering \u000d\u000acontrast between tungsten and oxygen in xPDF and ePDF experiments. As expected, POMFinder \u000d\u000aprimarily bases its predictions on the W \u2013 W distances when trained on ePDFs (electron scattering \u000d\u000afactors: W: 12.5 \u00c5, O: 2.0 \u00c5).68 However, it gives higher weights to the oxygens than for xPDF. We \u000d\u000athus see a clear trend between the scattering power of the element and the reasoning of POMFinder. \u000d\u000aA similar global analysis of all structures in our POM database is shown in section E in the SI, which \u000d\u000aprovides comparable results. We, therefore, hypothesise that POMFinder learns about the scattering \u000d\u000acontrast of different elements when predicting which POM fragment a PDF matches. \u000d\u000aFigure 9 shows the performance of POMFinder on the test set when POMFinder is trained using splits \u000d\u000aof 2:1:1, 3:1:1, 5:1:1, 8:1:1 and 98:1:1 PDFs per POM structure. Unsurprisingly, the performance \u000d\u000a(defined as the accuracy of the model on the test set) of POMFinder increases when trained on more \u000d\u000adata. Generally, POMFinder performs comparably when trained on xPDF, nPDF and ePDF data, as \u000d\u000aseen in Figure 9. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","Figure 8: Analysis of the influence of scattering probe on POMFinder\u2019s prediction. The X-ray \u000d\u000a(top), neutron (middle) and electron (bottom) PDF are plotted on top of a measure (SHAP values) \u000d\u000aof how important each datapoint in the PDF is for POMFinder to make its prediction on A) the 8-\u000d\u000aKeggin cluster and the B) paratungstate cluster.\u000d\u000aCombination with data from other techniques\u000d\u000aIt has previously been shown that combined modelling of data from multiple scattering techniques \u000d\u000acan provide more robust results than separately modelling data from the individual scattering \u000d\u000atechniques.20,25,69-72 However, it is a cumbersome process to do combined modelling of data from \u000d\u000amultiple scattering techniques using a least-square approach, and it can be challenging to weigh the \u000d\u000acontribution from each dataset.20,25,72,73 We hypothesise that this problem can be overcome with ML \u000d\u000amethods, and we here take the first steps to extend POMFinder to combined datasets. Specifically, \u000d\u000awe train POMFinder on a combination of xPDF/SAXS and a combination of xPDF/SAXS/nPDF data. \u000d\u000aWe do not weigh the datasets. The SAXS simulations provide information on the size and shape of \u000d\u000athe POM clusters, and are thus highly complementary to the PDF data discussed above. Details on \u000d\u000athe SAXS simulations are given in Section B in the SI. \u000d\u000aThe results on performance are given in Figure 9, where we observe that when combining information \u000d\u000afrom PDF and SAXS experiments, the performance increases; especially when using small training \u000d\u000asets, where POMFinder is challenged when using data from only one technique. This example \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","demonstrates that POMFinder can easily be extended to identify a structure from combined datasets \u000d\u000aand that combining information from various datasets provides a higher performance on a test set. \u000d\u000a \u000d\u000aFigure 9: The performance of the model trained with various simulated datasets and different \u000d\u000aamounts of datasets per structure. Section F in the SI lists the mean and standard deviation based \u000d\u000aon five iterations where the model was trained on different simulated PDFs and predictions were \u000d\u000amade on the same test set. \u000d\u000aConclusion \u000d\u000aWe have demonstrated how our tree-based ML classifier, POMFinder, can screen a POM structure \u000d\u000adatabase to identify structural candidates for the modelling of PDF data. Instead of using the \u000d\u000atraditional approach in scattering data analysis, where PDFs from all POM clusters in the database \u000d\u000aare fitted to the data through a least-squares refinement, we show that POMFinder can first be used \u000d\u000ato narrow down the field of candidate structures very quickly to 5 POM clusters, which can then be \u000d\u000afurther analysed. The POM database was made by cutting out clusters from the COD and ICSD \u000d\u000adatabases following appropriate chemical restraints for POM structures. A GBDT model, XGBoost, \u000d\u000awas trained on simulated X-ray PDF data to classify the POM cluster with an accuracy of 94.0 % on \u000d\u000asimulated PDFs. POMFinder also performs well on experimental data, including in situ data collected \u000d\u000awith fast acquisition time. This ultrafast method allows, e.g., visualising the structural model in 3D \u000d\u000awhile collecting data. Using SHAP analysis, we showed that POMFinder bases its predictions on \u000d\u000atrends comparable to the scattering contrast of the elements in the clusters. Finally, we show that ML \u000d\u000aoffers a promising and more flexible modelling framework for structure identification from combined \u000d\u000adatasets in contrast to conventional complex modelling refinement methods as it is not necessary to \u000d\u000aweigh the data contributions.20,25,72 The code is open source, and the method can be directly applied \u000d\u000aby users without prior ML knowledge to characterise POM clusters.\u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","POMFinder can be extended to include more types of chemical systems by extending the structural \u000d\u000adatabase used to generate the training data. In this project, we have focused on screening a database \u000d\u000aof POM fragments. However, the ultimate goal is to include any cluster fragment from the entire \u000d\u000adatabases of known crystal structures, such as COD & ICSD with more than 600k entries jointly. The \u000d\u000aapproach used for POMFinder can also be extended to analyse data from other scattering and \u000d\u000aspectroscopy techniques. We thus see POMFinder as a proof-of-concept, showing how a database of \u000d\u000aknown structures can quickly be screened for analysis of, e.g., scattering data using simple \u000d\u000aexplainable ML methods. \u000d\u000aAcknowledgements \u000d\u000aThis work is part of a project that has received funding from the European Research Council (ERC) \u000d\u000aunder the European Union\u2019s Horizon 2020 Research and Innovation Programme (grant agreement \u000d\u000aNo. 804066). We are grateful to the Villum Foundation for financial support through a Villum Young \u000d\u000aInvestigator grant (VKR00015416). Funding from the Danish Ministry of Higher Education and \u000d\u000aScience through the SMART Lighthouse is gratefully acknowledged. We furthermore thank \u000d\u000aDANSCATT (supported by the Danish Agency for Science and Higher Education) for support. S. A \u000d\u000aand M. J. acknowledge the Siemens Foundation for support for their thesis projects. We acknowledge \u000d\u000aDESY (Hamburg, Germany), a member of the Helmholtz Association HGF, for the provision of \u000d\u000aexperimental facilities. Parts of this research were carried out at beamline P02.1 at PETRAIII, and \u000d\u000awe would like to thank Martin Etter for his assistance in using the beamline. We acknowledge MAX \u000d\u000aIV Laboratory for time on Beamline DanMAX under Proposal 20200731. Research conducted at \u000d\u000aMAX IV is supported by the Swedish Research council under contract 2018-07152, the Swedish \u000d\u000aGovernmental Agency for Innovation Systems under contract 2018-04969, and Formas under \u000d\u000acontract 2019-02496. DanMAX is funded by the NUFI grant no. 4059-00009B. \u000d\u000aReferences \u000d\u000a1. Wang, C.; Steiner, U.; Sepe, A., Synchrotron Big Data Science. Small 2018, 14 (46), \u000d\u000ae1802291. \u000d\u000a2. Dong, H.; Butler, K. T.; Matras, D.; Price, S. W. T.; Odarchenko, Y.; Khatry, R.; \u000d\u000aThompson, A.; Middelkoop, V.; Jacques, S. D. M.; Beale, A. M.; Vamvakeros, A., A deep \u000d\u000aconvolutional neural network for real-time full profile analysis of big powder diffraction data. npj \u000d\u000aComput. Mater. 2021, 7 (1), 74. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","3. Pacchioni, G., An upgrade to a bright future. Nat. Rev. Phys. 2019, 1 (2), 100-101. \u000d\u000a4. Chen, Z.; Andrejevic, N.; Drucker, N. C.; Nguyen, T.; Xian, R. P.; Smidt, T.; \u000d\u000aWang, Y.; Ernstorfer, R.; Tennant, D. A.; Chan, M.; Li, M., Machine learning on neutron and xray scattering and spectroscopies. Chem. Phys. Rev. 2021, 2 (3), 031301. \u000d\u000a5. Choudhary, K.; DeCost, B.; Chen, C.; Jain, A.; Tavazza, F.; Cohn, R.; Park, C. \u000d\u000aW.; Choudhary, A.; Agrawal, A.; Billinge, S. J., Recent advances and applications of deep \u000d\u000alearning methods in materials science. npj Comput. Mater. 2022, 8 (1), 1-26. \u000d\u000a6. Pedersen, J. S., Analysis of small-angle scattering data from colloids and polymer \u000d\u000asolutions: modeling and least-squares fitting. Adv. Colloid Interface Sci. 1997, 70, 171-210. \u000d\u000a7. Rietveld, H., A profile refinement method for nuclear and magnetic structures. J. \u000d\u000aAppl. Crystallogr. 1969, 2 (2), 65-71. \u000d\u000a8. Chepkemboi, C.; Jorgensen, K.; Sato, J.; Laurita, G., Strategies and Considerations \u000d\u000afor Least-Squares Analysis of Total Scattering Data. ACS Omega 2022, 7 (17), 14402-14411. \u000d\u000a9. Banerjee, S.; Liu, C.-H.; Jensen, K. M. O.; Juhas, P.; Lee, J. D.; Tofanelli, M.; \u000d\u000aAckerson, C. J.; Murray, C. B.; Billinge, S. J. L., Cluster-mining: an approach for determining core \u000d\u000astructures of metallic nanoparticles from atomic pair distribution function data. Acta Crystallogr. A \u000d\u000a2020, 76 (1), 24-31. \u000d\u000a10. Yang, L.; Juh\u00e1s, P.; Terban, M. W.; Tucker, M. G.; Billinge, S. J. L., Structuremining: screening structure models by automated fitting to the atomic pair distribution function \u000d\u000aover large numbers of models. Acta Crystallogr. A 2020, 76 (3), 395-409. \u000d\u000a11. Aimi, A.; Fujimoto, K., Development of an Automatic, High-Throughput Structural \u000d\u000aRefinement Method Using Rietveld Analysis. ACS Combinatorial Science 2020, 22 (1), 35-41. \u000d\u000a12. Christiansen, T. L.; Kj\u00e6r, E. T. S.; Kovyakh, A.; R\u00f6deren, M. L.; H\u00f8j, M.; Vosch, \u000d\u000aT.; Jensen, K. M. \u00d8., Structure analysis of supported disordered molybdenum oxides using pair \u000d\u000adistribution function analysis and automated cluster modelling. J. Appl. Crystallogr. 2020, 53 (1), \u000d\u000a148-158. \u000d\u000a13. Egami, T.; Billinge, S. J., Underneath the Bragg peaks: Structural analysis of \u000d\u000acomplex materials. 2nd ed.; Elsevier: Oxford, 2012. \u000d\u000a14. Yang, X.; Masadeh, A. S.; McBride, J. R.; Bo\u017ein, E. S.; Rosenthal, S. J.; Billinge, \u000d\u000aS. J. L., Confirmation of disordered structure of ultrasmall CdSe nanoparticles from X-ray atomic \u000d\u000apair distribution function analysis. Phys. Chem. Chem. Phys. 2013, 15 (22), 8480-8486. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","15. Billinge, S. J. L.; Kanatzidis, M. G., Beyond crystallography: the study of disorder, \u000d\u000ananocrystallinity and crystallographically challenged materials with pair distribution functions. \u000d\u000aChem. Commun. 2004, 7, 749-760. \u000d\u000a16. Keen, D. A.; Goodwin, A. L., The crystallography of correlated disorder. Nature \u000d\u000a2015, 521 (7552), 303-309. \u000d\u000a17. Christiansen, T. L.; Cooper, S. R.; Jensen, K. M. \u00d8., There's no place like real-space: \u000d\u000aelucidating size-dependent atomic structure of nanomaterials using pair distribution function \u000d\u000aanalysis. Nanoscale Adv. 2020, 2 (6), 2234-2254. \u000d\u000a18. Juelsholt, M.; Anker, A. S.; Christiansen, T. L.; J\u00f8rgensen, M. R. V.; Kantor, I.; \u000d\u000aS\u00f8rensen, D. R.; Jensen, K. M. \u00d8., Size-induced amorphous structure in tungsten oxide \u000d\u000ananoparticles. Nanoscale 2021, 13 (47), 20144-20156. \u000d\u000a19. Bennett, T. D.; Cheetham, A. K., Amorphous Metal\u2013Organic Frameworks. Acc. \u000d\u000aChem. Res. 2014, 47 (5), 1555-1562. \u000d\u000a20. Anker, A. S.; Christiansen, T. L.; Weber, M.; Schmiele, M.; Brok, E.; Kj\u00e6r, E. T. \u000d\u000aS.; Juh\u00e1s, P.; Thomas, R.; Mehring, M.; Jensen, K. M. \u00d8., Structural Changes during the Growth \u000d\u000aof Atomically Precise Metal Oxido Nanoclusters from Combined Pair Distribution Function and \u000d\u000aSmall-Angle X-ray Scattering Analysis. Angew. Chem. Int. Ed. 2021, 60, 2-12. \u000d\u000a21. Jensen, K. M. \u00d8.; Juhas, P.; Tofanelli, M. A.; Heinecke, C. L.; Vaughan, G.; \u000d\u000aAckerson, C. J.; Billinge, S. J. L., Polymorphism in magic-sized Au144(SR)60 clusters. Nat. \u000d\u000aCommun. 2016, 7, 11859. \u000d\u000a22. Szczerba, D.; Tan, D.; Do, J.-L.; Titi, H. M.; Mouhtadi, S.; Chaumont, D.; del \u000d\u000aCarmen Marco de Lucas, M.; Geoffroy, N.; Meyer, M.; Rousselin, Y.; Hudspeth, J. M.; \u000d\u000aSchwanen, V.; Spoerk-Erdely, P.; Dippel, A.-C.; Ivashko, O.; Gutowski, O.; Glaevecke, P.; \u000d\u000aBazhenov, V.; Arhangelskis, M.; Halasz, I.; Fri\u0161\u010di\u0107, T.; Kimber, S. A. J., Real-Time Observation \u000d\u000aof \u201cSoft\u201d Magic-Size Clusters during Hydrolysis of the Model Metallodrug Bismuth Disalicylate. J. \u000d\u000aAm. Chem. Soc. 2021, 143 (40), 16332-16336. \u000d\u000a23. Billinge, S. J. L.; Levin, I., The Problem with Determining Atomic Structure at the \u000d\u000aNanoscale. Science 2007, 316 (5824), 561-565. \u000d\u000a24. Cooper, S. R.; Candler, R. O.; Cosby, A. G.; Johnson, D. W.; Jensen, K. M. \u00d8.; \u000d\u000aHutchison, J. E., Evolution of Atomic-Level Structure in Sub-10 Nanometer Iron Oxide \u000d\u000aNanocrystals: Influence on Cation Occupancy and Growth Rates. ACS Nano 2020, 14 (5), 5480-\u000d\u000a5490. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","25. Juh\u00e1s, P.; Farrow, C. L.; Yang, X.; Knox, K. R.; Billinge, S. J. L., Complex \u000d\u000amodeling: a strategy and software program for combining multiple information sources to solve ill \u000d\u000aposed structure and nanostructure inverse problems. Acta Crystallogr. A 2015, 71 (Pt 6), 562-568. \u000d\u000a26. Proffen, T.; Neder, R., DISCUS: A program for diffuse scattering and defect-structure \u000d\u000asimulation. J. Appl. Crystallogr. 1997, 30 (2), 171-175. \u000d\u000a27. Proffen, T.; Neder, R. B., DISCUS, a program for diffuse scattering and defect \u000d\u000astructure simulations \u2013 update. J. Appl. Crystallogr. 1999, 32 (4), 838-839. \u000d\u000a28. Coelho, A. A., TOPAS and TOPAS-Academic: an optimization program integrating \u000d\u000acomputer algebra and crystallographic objects written in C++. J. Appl. Crystallogr. 2018, 51 (1), \u000d\u000a210-218. \u000d\u000a29. Anker, A. S.; Kj\u00e6r, E. T. S.; Dam, E. B.; Billinge, S. J. L.; Jensen, K. M. \u00d8.; \u000d\u000aSelvan, R. In Characterising the Atomic Structure of Mono-Metallic Nanoparticles from X-Ray \u000d\u000aScattering Data Using Conditional Generative Models, 16th International Workshop on Mining and \u000d\u000aLearning with Graphs (MLG), San Diego, 2020. \u000d\u000a30. Liu, C.-H.; Tao, Y.; Hsu, D.; Du, Q.; Billinge, S. J. L., Using a machine learning \u000d\u000aapproach to determine the space group of a structure from the atomic pair distribution function. \u000d\u000aActa Crystallogr. A 2019, 75 (4), 633-643. \u000d\u000a31. Anker, A. S.; Kj\u00e6r, E. T. S.; Juelsholt, M.; Christiansen, T. L.; Skj\u00e6rv\u00f8, S. L.; \u000d\u000aJ\u00f8rgensen, M. R. V.; Kantor, I.; S\u00f8rensen, D. R.; Billinge, S. J. L.; Selvan, R.; Jensen, K. M. \u00d8., \u000d\u000aExtracting structural motifs from pair distribution function data of nanostructures using explainable \u000d\u000amachine learning. npj Comput. Mater. 2022, 8 (1), 213. \u000d\u000a32. Kj\u00e6r, E. T. S.; Anker, A. S.; Weng, M. N.; Billinge, S. J. L.; Selvan, R.; Jensen, K. \u000d\u000aM. \u00d8., DeepStruc: towards structure solution from pair distribution function data using deep \u000d\u000agenerative models. Digital Discovery 2023, 2 (1), 69-80. \u000d\u000a33. Kl\u00f8ve, M.; Sommer, S.; Iversen, B. B.; Hammer, B.; Dononelli, W., A Machine\u2010\u000d\u000aLearning\u2010Based Approach for Solving Atomic Structures of Nanomaterials Combining Pair \u000d\u000aDistribution Functions with Density Functional Theory. Adv. Mater. 2023, 35 (13), 2208220. \u000d\u000a34. Skj\u00e6rv\u00f8, S. L.; Anker, A. S.; Wied, M. C.; Kj\u00e6r, E.; Juelsholt, M.; Christiansen, T. \u000d\u000aL.; Jensen, K. M. \u00d8., Atomic structural changes in the formation of transition metal tungstates: The \u000d\u000arole of polyoxometalate structures in material crystallization. Chem. Sci. 2023, 14, 4806-4816. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","35. Magnard, N. P. L.; Anker, A. S.; Aalling-Frederiksen, O.; Kirsch, A.; Jensen, K. M. \u000d\u000a\u00d8., Characterisation of intergrowth in metal oxide materials using structure-mining: the case of \u03b3MnO2. Dalton Trans. 2022, 51 (45), 17150-17161. \u000d\u000a36. Gumerova, N. I.; Rompel, A., Synthesis, structures and applications of electron-rich \u000d\u000apolyoxometalates. Nat. Rev. Chem. 2018, 2 (2), 0112. \u000d\u000a37. Gumerova, N. I.; Rompel, A., Polyoxometalates in solution: speciation under \u000d\u000aspotlight. Chem. Soc. Rev. 2020, 49 (21), 7568-7601. \u000d\u000a38. Miras, H. N.; Yan, J.; Long, D.-L.; Cronin, L., Engineering polyoxometalates with \u000d\u000aemergent properties. Chem. Soc. Rev. 2012, 41 (22), 7403-7430. \u000d\u000a39. Long, D.-L.; Tsunashima, R.; Cronin, L., Polyoxometalates: Building Blocks for \u000d\u000aFunctional Nanoscale Systems. Angew. Chem. Int. Ed. 2010, 49 (10), 1736-1758. \u000d\u000a40. Juelsholt, M.; Lindahl Christiansen, T.; Jensen, K. M. \u00d8., Mechanisms for Tungsten \u000d\u000aOxide Nanoparticle Formation in Solvothermal Synthesis: From Polyoxometalates to Crystalline \u000d\u000aMaterials. J. Phys. Chem. C 2019, 123 (8), 5110-5119. \u000d\u000a41. Gra\u017eulis, S.; Merkys, A.; Vaitkus, A., Crystallography Open Database (COD). In \u000d\u000aHandbook of Materials Modeling : Methods: Theory and Modeling, Andreoni, W.; Yip, S., Eds. \u000d\u000aSpringer International Publishing: Cham, 2018; pp 1-19. \u000d\u000a42. Allen, F. H.; Bergerhoff, G.; Sievers, R., Crystallographic databases. International \u000d\u000aUnion of Crystallography: Chester, 1987. \u000d\u000a43. Lundberg, S. M.; Erion, G.; Chen, H.; DeGrave, A.; Prutkin, J. M.; Nair, B.; Katz, \u000d\u000aR.; Himmelfarb, J.; Bansal, N.; Lee, S.-I., From local explanations to global understanding with \u000d\u000aexplainable AI for trees. Nat. Mach. Intell. 2020, 2 (1), 56-67. \u000d\u000a44. Lundberg, S. M.; Lee, S.-I. In A Unified Approach to Interpreting Model Predictions, \u000d\u000aProceedings of the 31st International Conference on Neural Information Processing Systems, Long \u000d\u000aBeach, California, Long Beach, California, 2017; pp 4765-4774. \u000d\u000a45. Krayzman, V.; Levin, I.; Tucker, M. G., Simultaneous reverse Monte Carlo \u000d\u000arefinements of local structures in perovskite solid solutions using EXAFS and the total scattering \u000d\u000apair-distribution function. J. Appl. Crystallogr. 2008, 41 (4), 705-714. \u000d\u000a46. Bouhlel, M. A.; Hwang, J. T.; Bartoli, N.; Lafage, R.; Morlier, J.; Martins, J. R. R. \u000d\u000aA., A Python surrogate modeling framework with derivatives. Advances in Engineering Software \u000d\u000a2019, 135, 102662. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","47. Chen, T.; Guestrin, C., XGBoost: A Scalable Tree Boosting System. In Proceedings \u000d\u000aof the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining, \u000d\u000aAssociation for Computing Machinery: San Francisco, California, USA, 2016; pp 785\u2013794. \u000d\u000a48. Xin-Bao, H.; Zhi-Ming, Z.; Zhi-Shu, W.; Huan, Z.; Hui, D.; En-Bo, W., A 3D \u000d\u000apurely inorganic porous framework based on Keggin polyoxoanions. Inorg. Chem. Commun. 2012,\u000d\u000a18, 47-49. \u000d\u000a49. Kj\u00e6r, E. T. S.; Aalling-Frederiksen, O.; Yang, L.; Thomas, N. K.; Juelsholt, M.; \u000d\u000aBillinge, S. J. L.; Jensen, K. M. \u00d8., In Situ Studies of the Formation of Tungsten and Niobium \u000d\u000aOxide Nanoparticles: Towards Automated Analysis of Reaction Pathways from PDF Analysis using \u000d\u000athe Pearson Correlation Coefficient. Chemistry\u2013Methods 2022, 2 (9), e202200034. \u000d\u000a50. Jeannin, Y. P., The nomenclature of polyoxometalates: how to connect a name and a \u000d\u000astructure. Chem. Rev. 1998, 98 (1), 51-76. \u000d\u000a51. Sartzi, H.; Miras, H. N.; Vil\u00e0-Nadal, L.; Long, D.-L.; Cronin, L., Trapping the \u03b4 \u000d\u000aIsomer of the Polyoxometalate-Based Keggin Cluster with a Tripodal Ligand. Angew. Chem. Int. \u000d\u000aEd. 2015, 54 (51), 15488-15492. \u000d\u000a52. Saini, M. K.; Gupta, R.; Parbhakar, S.; Kumar Mishra, A.; Mathur, R.; Hussain, F., \u000d\u000aDimeric complexes of rare-earth substituted Keggin-type silicotungstates: syntheses, crystal \u000d\u000astructure and solid state properties. RSC Adv. 2014, 4 (48), 25357-25364. \u000d\u000a53. Glass, E. N.; Fielden, J.; Kaledin, A. L.; Musaev, D. G.; Lian, T.; Hill, C. L., \u000d\u000aExtending Metal-to-Polyoxometalate Charge Transfer Lifetimes: The Effect of Heterometal \u000d\u000aLocation. Chem. Eur. J. 2014, 20 (15), 4297-4307. \u000d\u000a54. Son, J. H.; Kwon, Y.-U.; Han, O. H., New Ionic Crystals of Oppositely Charged \u000d\u000aCluster Ions and Their Characterization. Inorg. Chem. 2003, 42 (13), 4153-4159. \u000d\u000a55. Han, X.-B.; Zhang, Z.-M.; Wang, Z.-S.; Zhang, H.; Duan, H.; Wang, E.-B., A 3D \u000d\u000apurely inorganic porous framework based on Keggin polyoxoanions. Inorg. Chem. Commun. 2012,\u000d\u000a18, 47-49. \u000d\u000a56. Uchida, S.; Kawamoto, R.; Mizuno, N., Recognition of Small Polar Molecules with \u000d\u000aan Ionic Crystal of \u03b1-Keggin-Type Polyoxometalate with a Macrocation. Inorg. Chem. 2006, 45\u000d\u000a(13), 5136-5144. \u000d\u000a57. B\u00f8jesen, E. D.; Jensen, K. M. \u00d8.; Tyrsted, C.; Mamakhel, A. H.; Andersen, H. L.; \u000d\u000aReardon, H.; Chevalier, J.; Dippel, A.-C.; Iversen, B. B., The chemistry of ZnWO4 nanoparticle \u000d\u000aformation. Chem. Sci. 2016, 7 (10), 6394-6406. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","58. Saha, D.; Jensen, K. M.; Tyrsted, C.; B\u00f8jesen, E. D.; Mamakhel, A. H.; Dippel, A. \u000d\u000aC.; Christensen, M.; Iversen, B. B., In Situ Total X\u2010Ray Scattering Study of WO3 Nanoparticle \u000d\u000aFormation under Hydrothermal Conditions. Angew. Chem. 2014, 126 (14), 3741-3744. \u000d\u000a59. Piedra-Garza, L. F.; Reinoso, S.; Dickman, M. H.; Sanguineti, M. M.; Kortz, U., \u000d\u000aThe first 3-dimensional assemblies of organotin-functionalized polyanions. Dalton Trans. 2009, \u000d\u000a(31), 6231-6234. \u000d\u000a60. Patrut, A.; B\u00f6gge, H.; Forizs, E.; Rusu, D.; Lowy, D. A.; Margineany, D.; \u000d\u000aNaumesco, A., Spectroscopic and crystal structure investigation of a new bismuth(iii) containing \u000d\u000apolyoxometalate cluster. Rev. Roum. Chim 2010, 55 (11-12), 865-870. \u000d\u000a61. Assran, A. S.; Izarova, N. V.; Banerjee, A.; Rabie, U. M.; Abou-El-Wafa, M. H. \u000d\u000aM.; Kortz, U., The antimony(iii)-bridged heteropolyanion sandwich dimers [SbIII3(A-\u03b1XW9O34)2]\u000d\u000a11\u2212 (X = SiIV, GeIV) and C-shaped double-sandwich [SbIII6O2(PW6O26)(A-\u03b1PW9O34)2]\u000d\u000a15\u2212. Dalton Trans. 2012, 41 (33), 9914-9921. \u000d\u000a62. Hou, D.; Hagen, K. S.; Hill, C. L., Pentadecavanadate, V15O429\u2013, a new highly \u000d\u000acondensed fully oxidized isopolyvanadate with kinetic stability in water. J. Chem. Soc., Chem. \u000d\u000aCommun. 1993, (4), 426-428. \u000d\u000a63. Mbomekall\u00e9, I.-M.; Bassil, B. S.; Suchopar, A.; Keita, B.; Nadjo, L.; Ammam, M.; \u000d\u000aHaouas, M.; Taulelle, F.; Kortz, U., Improved Synthesis, Structure, and Solution Characterization \u000d\u000aof the Cyclic 48-Tungsto-8-Arsenate(V), [H4As8W48O184]\u000d\u000a36\u2212. J. Cluster Sci. 2014, 25 (1), 277-285. \u000d\u000a64. Redrup, K. V.; Weller, M. T., Hydrothermal routes to new sodium hydrogen \u000d\u000apolytungstates. Dalton Trans. 2009, (23), 4468-4472. \u000d\u000a65. Lehmann, T.; Fuchs, J., Struktur und Schwingungsspektrum des \u000d\u000aKaliumundecawolframats Kg6H4W11O38\u00b7 11H2O / Structure and Vibrational Spectrum of the \u000d\u000aPotassium Undecatungstate K6H4WnO38 -11 H2O. Z. Naturforsch. B 1988, 43 (1), 89-93. \u000d\u000a66. Champarnaud-Mesjard, J.-C.; Frit, B.; Watanabe, A., Crystal structure of Bi2W2O9, \u000d\u000athe n=2 member of the homologous series (Bi2O2)BVInO3n+1 of cation-deficient Aurivillius phases. \u000d\u000aJ. Mater. Chem. 1999, 9 (6), 1319-1322. \u000d\u000a67. Crumpton, T. E.; Mosselmans, J. F. W.; Greaves, C., Structure and oxide ion \u000d\u000aconductivity in Bi28Re2O49, a new bismuth rhenium oxide containing tetrahedral and octahedral \u000d\u000aRe(vii). J. Mater. Chem. 2005, 15 (1), 164-167. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005","68. Prince, E., International tables for crystallography: Volume C, Mathematical, \u000d\u000aphysical and chemical tables. Kluwer Academic Publishers for International Union of \u000d\u000aCrystallography: Dordrecht; Boston; London, 2004. \u000d\u000a69. Farrow, C.; Shi, C.; Juh\u00e1s, P.; Peng, X.; Billinge, S. J. L., Robust structure and \u000d\u000amorphology parameters for CdS nanoparticles by combining small-angle X-ray scattering and \u000d\u000aatomic pair distribution function data in a complex modeling framework. J. Appl. Crystallogr. \u000d\u000a2014, 47 (2), 561-565. \u000d\u000a70. Farrow, C. L.; Billinge, S. J. L., Relationship between the atomic pair distribution \u000d\u000afunction and small-angle scattering: implications for modeling of nanoparticles. Acta Crystallogr. A \u000d\u000a2009, 65 (3), 232-239. \u000d\u000a71. Tucker, M. G.; Keen, D. A.; Dove, M. T.; Goodwin, A. L.; Hui, Q., RMCProfile: \u000d\u000areverse Monte Carlo for polycrystalline materials. J. Phys.: Condens. Matter 2007, 19 (33), 335218. \u000d\u000a72. Krayzman, V.; Levin, I.; Tucker, M. G., Simultaneous reverse Monte Carlo \u000d\u000arefinements of local structures in perovskite solid solutions using EXAFS and the total scattering \u000d\u000apair-distribution function. J. Appl. Cryst. 2008. \u000d\u000a73. Terban, M. W.; Billinge, S. J. L., Structural Analysis of Molecular Materials Using \u000d\u000athe Pair Distribution Function. Chem. Rev. 2022, 122 (1), 1208-1272. \u000d\u000a7\"\"fi!\u000d\u0004\u00044fffl ff 6\u0004\u0006\u0005 \u0007\u000a\u0009\u0008\u0009\u000437\u00189 Bfl#\u0007\u0005\u0007\u0008\u000c\u0006BD\u000b \u0006\u0007\u0004\u0003\u0001 7\"\"fi!\u000d\u0004\u0004ff 3fl4 ff 6\u0004\u0005\u0005\u0005\u0005\u0005\u0005\u0005\u0008\u0005\u0007\u000c\u0006\u0007\u0006\u000b0\u0001\u000fff:\"\u0018:\"\u0001:ff\"\u0001fi\u0018\u0018  \u0018#fl\u0018$\u00184\u00012C\u0001\u000f7\u00189/Bfl# \u0001\u0005\u000a\u0008\u0009\u000b\u000c\u0009\u0001 \u000f\u000f\u0001\u000e1.\u000f.-\u0001\u0009 \u0005"," \u000d\u000a87\u000d\u000a3.2 Extraction of\u0009Structural\u0009Motifs\u0009from\u0009Scattering\u0009Data*\u000d\u000aClassification\u0009 ML\u0009 models have\u0009 emerged\u0009 as\u0009 a\u0009 prominent\u0009 approach\u0009 in\u0009 the\u0009 screening\u0009 of\u0009 extensive\u0009\u000d\u000adatabases.46 However,\u0009the\u0009efficacy\u0009of\u0009these\u0009methods\u0009is\u0009fundamentally\u0009tied\u0009to\u0009the\u0009availability\u0009of\u0009a\u0009suitable\u000d\u000astructure database\u0009or\u0009an\u0009algorithmic\u0009structure\u0009generator.\u0009This\u0009dependency\u0009raises\u0009an\u0009inevitable\u0009question:\u000d\u000ahow\u0009can\u0009we\u0009move\u0009beyond\u0009these\u0009limitations?\u000d\u000aAb\u0009initio structure\u0009solution,\u0009the\u0009ability to\u0009solve\u0009a\u0009structure\u0009without\u0009prior\u0009knowledge\u0009about a\u0009dataset,\u0009\u000d\u000aeliminates the\u0009reliance\u0009on\u0009pre-existing\u0009structural\u0009databases.\u0009Yet,\u0009the\u0009current\u0009use of\u0009ab\u0009initio structure\u0009\u000d\u000asolution\u0009from PDF\u0009data\u0009has\u0009seen\u0009limited success.\u0009This\u0009approach\u0009has\u0009been\u0009successful only\u0009in\u0009a\u0009handful\u0009of\u0009\u000d\u000aexamples,\u0009and\u0009these\u0009structures\u0009have\u0009typically\u0009been\u0009simple\u0009and\u0009symmetrical,\u0009akin\u0009to\u0009the\u0009C60 buckyball.79-\u000d\u000a83 When\u0009it\u0009comes\u0009to\u0009less\u0009symmetrical,\u0009more\u0009complex\u0009structures,\u0009information\u0009loss\u0009becomes\u0009a\u0009significant\u0009\u000d\u000ahurdle.\u000d\u000aGiven\u0009 these\u0009 challenges,\u0009we\u0009 can\u0009use motif-extractor\u0009algorithms\u0009 that\u0009integrate prior\u0009information\u0009 from\u0009\u000d\u000acrystal\u0009 structures\u0009 identified\u0009 from\u0009 a\u0009 crystallographic\u0009 database with chemical\u0009 knowledge.49 Paper\u0009 V\u0009\u000d\u000aintroduces a\u0009 motif-extractor\u0009 method named\u0009 Machine\u0009 Learning\u0009 based\u0009 Motif\u0009 Extractor\u0009 (ML-MotEx),\u0009\u000d\u000aillustrated\u0009 in\u0009 Figure\u0009 3.4.\u000d\u000a84 ML-MotEx employs\u0009 a\u0009 unique\u0009 approach,\u0009 using\u0009 explainable\u0009 ML\u0009 to\u0009 extract\u000d\u000astructural\u0009motifs for\u0009structure\u0009refinement\u0009of\u0009PDF\u0009data.\u0009Inspired\u0009by\u0009previous\u0009motif-extractor\u0009algorithms,\u000d\u000a23,\u000949 ML-MotEx\u0009first\u0009generates\u0009a\u0009catalogue,\u0009containing thousands\u0009of\u0009potential\u0009structural\u0009motifs.\u0009Each\u0009of\u0009\u000d\u000athese\u0009structural\u0009motifs is derived\u0009from\u0009a\u0009supercell\u0009created\u0009from\u0009a\u0009chosen\u0009bulk\u0009structure (step\u00091).\u0009While\u000d\u000aprevious\u0009algorithms\u0009had\u0009to\u0009generate\u00092N-1\u0009potential\u0009structural\u0009motifs,\u0009for\u0009each\u0009supercell\u0009with\u0009N\u0009atoms,\u0009\u000d\u000aML-MotEx\u0009 only\u0009 requires\u0009 a\u0009 sub-sample\u0009 (~10,000).\u0009 Following\u0009 this,\u0009 the\u0009 PDF\u0009 is\u0009 computed\u0009 from\u0009 each\u0009\u000d\u000astructural\u0009motif and\u0009fitted to\u0009 the\u0009 target\u0009dataset\u0009(step\u00092). The\u0009extracted\u0009structures\u0009and\u0009associated\u0009Rwp\u000d\u000avalues\u0009 are\u0009 then\u0009 fed\u0009 to\u0009 a\u0009 tree-based ML\u0009 algorithm.85 This\u0009 algorithm\u0009trains\u0009 a\u0009 regression ML\u0009model to\u0009\u000d\u000apredict\u0009 Rwp values\u0009 for\u0009 new\u0009 fits based\u0009 on\u0009 the\u0009 atomic\u0009 structure\u0009 model (step\u0009 3).\u0009 Finally,\u0009 ML-MotEx\u0009\u000d\u000aquantifies\u0009the\u0009importance\u0009of\u0009each\u0009atom\u0009in\u0009the\u0009starting\u0009structure\u0009to achieve\u0009a\u0009low\u0009Rwp value\u0009with\u0009the\u0009given\u0009\u000d\u000afitting\u0009 algorithm\u0009 (step\u0009 4).\u0009 This\u0009 quantification\u0009 is\u0009 done\u0009 using\u0009 SHAP\u0009 (Shapley\u0009 Additive\u0009\u000d\u000aexPlanation)86,87 values,\u0009 a\u0009 known\u0009 method\u0009 for\u0009 explaining\u0009 tree-based\u0009 ML\u0009 models.\u0009 The\u0009 SHAP\u0009 value\u2019s\u0009\u000d\u000amagnitude reflects\u0009how important\u0009an\u0009atom is\u0009for\u0009the\u0009fit\u0009quality,\u0009while\u0009its sign\u0009reveals whether\u0009the\u0009feature\u0009\u000d\u000ainfluences the Rwp value\u0009of\u0009the\u0009fit\u0009towards\u00091\u0009(poor\u0009fit)\u0009or\u00090\u0009(perfect\u0009fit),\u0009i.e. why each\u0009feature is\u0009important.\u0009\u000d\u000aML-MotEx\u0009model's\u0009efficacy\u0009is\u0009demonstrated\u0009through\u0009four\u0009distinct\u0009examples.\u0009The\u0009first\u0009example\u0009is\u0009a\u0009simple\u0009\u000d\u000aproof-of-concept based\u0009on\u0009simulated\u0009PDF\u0009data\u0009from\u0009a\u0009C60 buckyball.\u0009In\u0009the\u0009second\u0009and\u0009third\u0009examples,\u0009\u000d\u000awe apply ML-MotEx\u0009to\u0009experimental\u0009X-ray\u0009PDF\u0009data\u0009obtained\u0009from\u0009amorphous,\u0009disordered\u0009molybdenum\u0009\u000d\u000aoxides49 and\u0009tungstate\u0009\u03b1-Keggin\u0009clusters\u0009in\u0009solution.72 These\u0009examples\u0009demonstrate ML-MotEx\u2019s ability\u0009\u000d\u000ato\u0009 identify\u0009 main structural\u0009 motifs\u0009 from\u0009 experimental\u0009 PDF\u0009 data.\u0009 Importantly,\u0009 ML-MotEx\u0009 produces\u000d\u000aequivalent\u0009results\u0009using\u0009different\u0009starting\u0009models. In\u0009the\u0009last\u0009example,\u0009ML-MotEx\u0009identifies\u0009a\u0009bismuth\u0009\u000d\u000aoxido\u0009cluster from\u0009a\u0009starting\u0009model\u0009of\u0009a\u0009cut-out\u0009of\u0009the\u0009\u03b2-Bi2O3 structure.\u0009\u000d\u000aML-MotEx\u0009and\u0009other\u0009motif-extractor methods\u0009make\u0009it\u0009possible to\u0009build\u0009structural models\u0009that\u0009capture\u0009\u000d\u000alocal\u0009 bonding\u0009 topologies\u0009 suggested\u0009 by\u0009 known\u0009 chemistries.\u0009 These can\u0009 be\u0009 derived\u0009 from\u0009 database\u0009\u000d\u000astructures.\u0009By\u0009embedding established\u0009chemical\u0009knowledge\u0009into\u0009the model\u0009construction,\u0009motif-extractor\u0009\u000d\u000aapproaches include more\u0009information\u0009than\u0009ab\u0009initio structure\u0009solution\u0009methods but reduce reliance on\u0009\u000d\u000aextensive\u0009 structural\u0009 databases compared\u0009 to\u0009 brute-force\u0009methods.\u0009While the\u0009ML-MotEx\u0009model\u0009is\u0009 still\u0009\u000d\u000aslower\u0009(minutes\u0009for\u0009a\u0009structure\u0009with\u0009tenths\u0009of atoms,\u0009hours\u0009for\u0009one with\u0009hundreds of atoms)\u0009than\u0009the\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009V+VI.","\u0009 |\u0009Supervised\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a94\u000d\u000athe\u0009previous state-of-the-art motif-extractor\u0009method49\u0009(~1\u0009week\u0009for\u0009a\u0009structure\u0009with\u000920\u0009atoms,\u0009which\u0009\u000d\u000adoubles\u0009per\u0009extra\u0009atom).\u0009\u0009\u000d\u000aML-MotEx\u0009has\u0009been\u0009used in\u0009 two\u0009other\u0009published\u0009studies,\u0009of which I\u0009am the\u00092nd author\u0009(appended\u0009 to\u0009\u000d\u000aAPPENDICES\u0009 \u2013\u0009 PAPERS).\u0009 The\u0009 first\u0009 of\u0009 these\u0009 studies uses ML-MotEx to\u0009 determine\u0009 stacking\u0009 fault\u0009 size\u0009\u000d\u000adomain\u0009distributions\u0009 from\u0009experimental\u0009PD\u0009and\u0009PDF\u0009data\u0009measured\u0009 on\u0009 \u03b3-MnO2 nanoparticles.\u000d\u000a88 The\u0009\u000d\u000asecond\u0009study\u0009uses\u0009ML-MotEx\u0009on\u0009an\u0009in\u0009situ PDF\u0009dataset to study\u0009the\u0009formation\u0009mechanisms\u0009of\u0009CoWO4 and\u0009\u000d\u000arelated\u0009compounds.89 Here,\u0009ML-MotEx\u0009traces the\u0009structural\u0009changes\u0009that\u0009occur during\u0009the\u0009crystallisation\u0009\u000d\u000aprocess,\u0009where\u0009different\u0009motifs become\u0009dominant\u0009during\u0009the\u0009material\u0009formation.\u0009In\u0009this\u0009study,\u0009the\u0009speed\u0009\u000d\u000aof\u0009ML-MotEx\u0009was\u0009vital\u0009in\u0009order\u0009to\u0009use\u0009it\u0009on\u0009time\u0009resolved\u0009PDF\u0009data.\u0009\u000d\u000aFigure\u00093.4:\u0009Firstly,\u0009a\u0009starting\u0009model\u0009is\u0009provided.\u0009Using\u0009this\u0009starting\u0009model,\u0009a\u0009structure\u0009catalogue\u0009is\u0009generated,\u0009and\u0009\u000d\u000athe\u0009structures\u0009in\u0009the\u0009catalogue\u0009are\u0009fitted\u0009to\u0009the\u0009experimental\u0009data\u0009in\u0009question.\u0009An\u0009ML\u0009algorithm\u0009is\u0009then trained\u0009to\u0009\u000d\u000apredict\u0009Rwp values\u0009and\u0009finally\u0009calculate quantified\u0009values\u0009of\u0009feature\u0009importance\u0009for\u0009the\u0009fit\u0009quality. Figure\u0009and\u0009Figure\u0009\u000d\u000acaption\u0009are adapted\u0009from\u0009Paper\u0009V.84\u000d\u000aA\u0009limitation of ML-MotEx is\u0009its\u0009need\u0009for\u0009an\u0009initial supercell structure that\u0009contains\u0009the\u0009target\u0009motif. This\u0009\u000d\u000acan\u0009restrict\u0009its\u0009applicability\u0009in\u0009cases\u0009where\u0009such\u0009an\u0009initial\u0009structure\u0009is\u0009not\u0009readily\u0009available\u0009or\u0009identifiable.\u000d\u000aHere, ML-MotEx\u0009must further\u0009iterate\u0009over\u0009supercells\u0009from\u0009all\u0009possible\u0009candidate\u0009crystal\u0009structures.\u0009The\u0009\u000d\u000aML-MotEx\u0009algorithm\u0009is\u0009too\u0009slow\u0009to\u0009do\u0009this\u0009over\u0009a\u0009large\u0009number\u0009of\u0009candidate structures,\u0009and\u0009the\u0009success\u0009\u000d\u000aof\u0009the\u0009approach\u0009relies\u0009on\u0009chemical\u0009intuition\u0009to\u0009suggest\u0009a\u0009small\u0009number\u0009of\u0009candidate\u0009structures. In\u0009Paper\u0009\u000d\u000aVI,\u0009we\u0009address\u0009this\u0009issue\u0009with\u0009a\u0009newly\u0009developed motif-extractor\u0009algorithm\u0009called\u0009ClusterFinder.\u0009\u000d\u000aAt\u0009the\u0009heart\u0009of\u0009the\u0009algorithm\u0009is\u0009the\u0009calculation\u0009to\u0009generate\u0009a\u0009ranked list\u0009of\u0009atoms\u0009based\u0009on\u0009their\u0009likelihood\u0009\u000d\u000aof\u0009being\u0009in the\u0009target\u0009cluster.\u0009Here,\u0009we\u0009take\u0009inspiration\u0009from\u0009the\u0009Liga\u0009algorithm,79,80 which\u0009scores\u0009atoms\u000d\u000afrom\u0009their\u0009contribution\u0009to\u0009the\u0009data\u0009disagreement. Inspired\u0009by\u0009this,\u0009we\u0009use a\u0009simple\u0009scoring\u0009approach for\u000d\u000aeach\u0009atom\u0009site:\u0009removing\u0009the\u0009atom,\u0009computing\u0009Rwp,\u0009and\u0009comparing\u0009it\u0009with\u0009the\u0009full\u0009supercell\u2019s\u0009Rwp.\u0009Each\u0009\u000d\u000atime\u0009 the Rwp value\u0009 is\u0009 determined\u0009 through\u0009 structure\u0009 refinement. This\u0009 procedure\u0009 is\u0009 illustrated\u0009\u000d\u000aschematically\u0009for\u0009a\u0009trivial\u0009example\u0009of\u0009a\u0009binary\u0009molecule\u0009in\u0009Figure\u00093.5.\u0009It\u0009is\u0009important\u0009to\u0009note\u0009that\u0009both\u0009\u000d\u000aML-MotEx\u0009 and\u0009 ClusterFinder\u0009 rank\u0009 the\u0009 atoms\u0009 in\u0009 the\u0009 supercell,\u0009 and\u0009 human input\u0009 is\u0009 still\u0009 required to\u0009\u000d\u000adetermine\u0009which\u0009atoms\u0009to\u0009remove.\u0009In\u0009some\u0009cases,\u0009such\u0009as\u0009in\u0009Figure\u00093.5,\u0009it\u0009is\u0009trivial\u0009to\u0009remove\u0009certain\u0009\u000d\u000aatoms\u0009(3\u0009and\u00094) from\u0009the\u0009ClusterFinder\u0009output,\u0009but\u0009this\u0009task\u0009might\u0009not\u0009always\u0009be\u0009trivial. In\u0009comparison\u000d\u000ato\u0009ML-MotEx,\u0009ClusterFinder\u0009is\u0009extremely\u0009rapid (seconds\u0009for\u0009a\u0009structure\u0009with hundreds\u0009of\u0009atoms)\u0009while\u0009\u000d\u000aproviding comparable\u0009results.\u0009We\u0009provide\u0009five\u0009examples\u0009where\u0009we\u0009screen\u0009a\u0009relevant\u0009subset\u0009of\u0009the\u0009ICSD37\u000d\u000a(ICSD\u0009contains\u0009188,631\u0009structure\u0009entries)\u0009for\u0009a\u0009suitable\u0009starting\u0009model.\u0009A\u0009process\u0009that\u0009takes\u0009anywhere\u0009"," \u000d\u000a95\u000d\u000afrom\u0009 3\u0009 to\u0009 42\u0009 minutes.\u0009Conclusively, ClusterFinder\u0009 can rapidly\u0009 discover\u0009 the\u0009 target\u0009 cluster\u0009 if\u0009 it\u0009 exists\u0009\u000d\u000aanywhere\u0009within\u0009a\u0009structure in\u0009a\u0009given\u0009structural database.\u0009\u000d\u000aFigure\u00093.5:\u0009 Illustration\u0009of\u0009the\u0009ClusterFinder\u0009process.\u0009A\u0009starting\u0009model\u0009is\u0009provided\u0009as\u0009input.\u0009Atoms\u0009are\u0009iteratively\u0009\u000d\u000aremoved\u0009 from\u0009 the\u0009 starting\u0009 model\u0009 and\u0009 fitted\u0009 to\u0009 the\u0009 experimental\u0009 PDF.\u0009 Atoms\u0009 are\u0009 colour-coded\u0009 based\u0009 on\u0009 their\u0009\u000d\u000ainfluence\u0009on\u0009fit\u0009quality\u0009\u2013\u0009yellow\u0009indicates\u0009a\u0009decrease\u0009in\u0009Rwp value\u0009(improved\u0009fit)\u0009while\u0009blue\u0009signifies\u0009an\u0009increase\u0009in\u0009\u000d\u000aRwp value\u0009(worsened\u0009fit). Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from Paper\u0009VI.\u0009\u000d\u000a3.2.1 Paper\u0009V\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009Paper\u0009V\u0009which\u0009is\u0009published\u0009in\u0009npj\u0009Computational\u0009Materials.\u0009The\u0009\u000d\u000asupplementary\u0009information\u0009for\u0009Paper\u0009V\u0009can\u0009be\u0009found\u0009in\u0009APPENDICES\u0009\u2013\u0009PAPERS.","ARTICLE OPEN\u000d\u000aExtracting structural motifs from pair distribution function\u000d\u000adata of nanostructures using explainable machine learning\u000d\u000aAndy S. Anker 1\u000d\u000a, Emil T. S. Kj\u00e6r 1\u000d\u000a, Mikkel Juelsholt2\u000d\u000a, Troels Lindahl Christiansen1\u000d\u000a, Susanne Linn Skj\u00e6rv\u00f81\u000d\u000a,\u000d\u000aMads Ry Vogel J\u00f8rgensen 3,4, Innokenty Kantor4,5, Daniel Risskov S\u00f8rensen 3,4, Simon J. L. Billinge 6,7, Raghavendra Selvan8,9 and\u000d\u000aKirsten M. \u00d8. Jensen1\u2709\u000d\u000aCharacterization of material structure with X-ray or neutron scattering using e.g. Pair Distribution Function (PDF) analysis most often\u000d\u000arely on refining a structure model against an experimental dataset. However, identifying a suitable model is often a bottleneck.\u000d\u000aRecently, automated approaches have made it possible to test thousands of models for each dataset, but these methods are\u000d\u000acomputationally expensive and analysing the output, i.e. extracting structural information from the resulting fits in a meaningful\u000d\u000away, is challenging. Our Machine Learning based Motif Extractor (ML-MotEx) trains an ML algorithm on thousands of fits, and uses\u000d\u000aSHAP (SHapley Additive exPlanation) values to identify which model features are important for the fit quality. We use the method\u000d\u000afor 4 different chemical systems, including disordered nanomaterials and clusters. ML-MotEx opens for a type of modelling where\u000d\u000aeach feature in a model is assigned an importance value for the fit quality based on explainable ML.\u000d\u000anpj Computational Materials           (2022) 8:213  ; https://doi.org/10.1038/s41524-022-00896-3\u000d\u000aINTRODUCTION\u000d\u000aThe development of advanced, functional materials builds on an\u000d\u000aunderstanding of the intricate relationship between material\u000d\u000astructure and properties, and over the past century, crystallographic methods using scattering and diffraction have thus\u000d\u000abeen essential for materials science. Crystallography allows ab\u000d\u000ainitio determination of crystal structures from diffraction data and\u000d\u000ahas provided us with the vast knowledge of crystal chemistry that\u000d\u000ais now used in the design of functional materials. However, in the\u000d\u000acase of nanomaterials with limited long-range order, crystallographic methods are challenged, and ab initio structure\u000d\u000adetermination, or structure solution, is not currently possible.\u000d\u000aOver the past decades, total scattering with Pair Distribution\u000d\u000aFunction (PDF) analysis has become an essential tool for the\u000d\u000acharacterization of nanomaterial structure1,2\u000d\u000a. The PDF is the\u000d\u000aFourier transform of normalized and corrected X-ray, neutron, or\u000d\u000aelectron scattering intensities, and is a function in real space\u000d\u000arepresenting a histogram of inter-atomic distances in the sample.\u000d\u000aCompared to crystallographic methods relying on long-range\u000d\u000aorder, PDF analysis can be applied for nanomaterials3\u20135\u000d\u000a, disordered1,6,7\u000d\u000a, or amorphous materials3,5,8\u000d\u000a. However, structure\u000d\u000asolution from the PDF is not possible except in a very few simple\u000d\u000acases9\u000d\u000a, using either the Reverse Monte Carlo method10 or the LIGA\u000d\u000aalgorithm11,12. In the absence of broadly applicable ab initio\u000d\u000ananostructure determination methods, it is, therefore, necessary\u000d\u000ato propose reasonable starting models and to then \u2018refine\u2019 the\u000d\u000amodel parameters against the data using local minimization\u000d\u000amethods. The step of finding a starting model can be a major\u000d\u000achallenge and is thus a bottleneck in complex material\u000d\u000acharacterization. In the case of PDF analysis of nanomaterials,\u000d\u000asuch models are often guessed at by considering related bulk\u000d\u000amaterials; however, these are often not good starting models for\u000d\u000avery small clusters and nanoparticles, where significant structural\u000d\u000achanges may take place3,5,13,14. A way of building plausible\u000d\u000astarting models is thus needed, where structure models can be\u000d\u000abuilt capturing local bonding topologies suggested by known\u000d\u000achemistries.\u000d\u000aRecently, automated methods such as \u2018structure mining\u2019 and\u000d\u000a\u2018cluster mining\u2019 have appeared in the literature to help overcome\u000d\u000athis challenge15\u201317. In a study of the structure of metallic\u000d\u000ananoparticles, Banerjee et al. automatically generated thousands\u000d\u000aof discrete metal nanocluster structures and fitted PDFs from each\u000d\u000aof them to experimental data to identify the best model in an\u000d\u000aautomated manner17. In a recent study of molybdenum oxide\u000d\u000ananomaterials, we introduced another approach, where we\u000d\u000aautomatically generated a large number of MoOx cluster structure\u000d\u000amodels and compared their PDFs to experimental data in order to\u000d\u000aidentify dominating structural motifs in the sample, i.e. arrangements of atoms that dominate the material structure on the local\u000d\u000ascale7\u000d\u000a. We hypothesized that the structural motifs present in\u000d\u000aamorphous molybdenum oxides can also be found in crystalline\u000d\u000astructures, and therefore used crystal structures of molybdenum\u000d\u000aoxides as starting models. From these models, we cut out\u000d\u000athousands of different cluster structure models of different sizes\u000d\u000ato build a \u2018catalogue\u2019 of structure candidates. These models were\u000d\u000aall tested against our data to identify the best fitting structural\u000d\u000amotif. We recently used a similar approach for the identification of\u000d\u000aa bismuth oxido cluster intermediate structure in a study of cluster\u000d\u000agrowth18.\u000d\u000aWhile these approaches can extend the structural space\u000d\u000asearched when identifying models for structure refinement, new\u000d\u000achallenges arise. Firstly, the refinement processes can be\u000d\u000acomputationally heavy, which can limit the number of catalogue\u000d\u000astructures that are tested. For example, our brute-force approach\u000d\u000a1\u000d\u000aDepartment of Chemistry and Nano-Science Center, University of Copenhagen, 2100 Copenhagen, Denmark. 2\u000d\u000aDepartment of Materials, University of Oxford, Parks Road, Oxford,\u000d\u000aUK. 3\u000d\u000aDepartment of Chemistry & iNANO, Aarhus University, 8000 Aarhus, Denmark. 4\u000d\u000aMAX IV Laboratory, Lund University, 224 84 Lund, Sweden. 5\u000d\u000aDepartment of Physics, Technical\u000d\u000aUniversity of Denmark, 2880 Lyngby, Denmark. 6\u000d\u000aDepartment of Applied Physics and Applied Mathematics, Columbia University, New York, NY 10027, USA. 7\u000d\u000aCondensed Matter\u000d\u000aPhysics and Materials Science Department, Brookhaven National Laboratory, Upton, NY 11973, USA. 8\u000d\u000aDepartment of Computer Science, University of Copenhagen, 2100\u000d\u000aCopenhagen, Denmark. 9\u000d\u000aDepartment of Neuroscience, University of Copenhagen, 2200 Copenhagen, Denmark. \u2709email: kirsten@chem.ku.dk\u000d\u000awww.nature.com/npjcompumats\u000d\u000aPublished in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences\u000d\u000a1234567890():,;","for cluster identification above generates 2N \u2212 1 structures for\u000d\u000astarting model sizes with N atoms. Each structure must have its\u000d\u000aPDF computed and then refined against the target measured PDF,\u000d\u000aso that its fit quality can be evaluated. This process is\u000d\u000acomputationally costly and does not scale well with the number\u000d\u000aof structure candidates. Furthermore, for disordered, amorphous,\u000d\u000aand nanostructured systems, many hundred models may provide\u000d\u000asimilar fit qualities, and if only reporting a few of them, it is difficult\u000d\u000ato assess which structural features of these models are important.\u000d\u000aWe, therefore, need effective and unbiased methods to compare\u000d\u000amany fits to extract structural information.\u000d\u000aHere, we introduce a method that uses an explainable Machine\u000d\u000aLearning (ML) model that, after training, will predict the\u000d\u000aagreement factor for a test cluster with a given dataset.\u000d\u000aFurthermore, the use of explainable ML informs which features\u000d\u000ain the model are important for the agreement factor19\u201324. Our\u000d\u000aMachine Learning based Motif Extractor (ML-MotEx) model is\u000d\u000aillustrated in Fig. 1. Firstly, it builds a large catalogue of thousands\u000d\u000aof candidate structural motifs, which are \u2018cut outs\u2019 from a chosen\u000d\u000abulk structure7,18 (step 1). The PDF is then computed from each\u000d\u000aone, and each model is fit to the target dataset (step 2). The\u000d\u000astructures and Rwp values from each fit are handed to an ML\u000d\u000aalgorithm applying gradient boosting decision trees (GBDTs)25,\u000d\u000awhich learns to predict Rwp values for new fits based on an atomic\u000d\u000astructure model (step 3). The ML-MotEx algorithm then outputs\u000d\u000aquantified values of how important each atom or feature in the\u000d\u000astarting structure is for the fit to yield a low Rwp value with the\u000d\u000agiven fitting algorithm (step 4). This is done by using SHAP\u000d\u000a(Shapley Additive exPlanation)26,27 values, which is a known\u000d\u000amethod for explaining tree-based ML models. The amplitude of\u000d\u000athe SHAP value reflects how important a structural feature is for\u000d\u000athe fit quality, while the sign of the SHAP value reflects whether\u000d\u000athe feature affects the Rwp value of the fit towards 1 (poor fit) or 0\u000d\u000a(perfect fit), in other words why it is important.\u000d\u000aCompared to the automated, brute-force methods previously\u000d\u000aintroduced for PDF analysis7,15\u201317, we can much faster screen a\u000d\u000alarger number of structures. Our method only needs to screen a\u000d\u000asub-sample (~10,000) of the much larger number of motifs that\u000d\u000acan be generated from bulk material to learn how to predict\u000d\u000awhich structures provide a good agreement with the data. The\u000d\u000aanalysis done for the examples presented below would take\u000d\u000a~24 days for starting models with 24 atoms, ~3 \u00d7 106 years for\u000d\u000astarting models with 48 atoms, and ~6 \u00d7 1013 years for starting\u000d\u000amodels with 72 atoms using a brute-force approach (Supplementary Notes 1), while ML-MotEx analysis is done in minutes or hours.\u000d\u000aFurthermore, the use of explainable ML provides a way to better\u000d\u000aanalyse the output of the screening: instead of just identifying the\u000d\u000amodel that provides the lowest Rwp value, we are able to output a\u000d\u000ameasure of how important each atom or feature (e.g. size or\u000d\u000ashape) in the starting model is for the fit to yield a low Rwp value\u000d\u000a(step 4). This procedure is automated and can be done in quasireal experimental time and without human bias.\u000d\u000aWe illustrate the use of ML-MotEx using four different examples.\u000d\u000aWe first show the principles of the method using a simple model\u000d\u000asystem based on simulated X-ray PDF data from a C60 buckyball.\u000d\u000aWe further demonstrate the use of ML-MotEx on experimental\u000d\u000aX-ray PDF data from amorphous, disordered molybdenum oxides7\u000d\u000aand tungstate \u03b1-Keggin clusters in solution28, where it allows\u000d\u000aidentifying the main structural motifs present in the samples using\u000d\u000adifferent starting models. Lastly, we extend the method to use a\u000d\u000a\u2018cookie-cutter\u2019 strategy to generate structures for the catalogue of\u000d\u000acandidate motifs. Here, the algorithm is used to identify a bismuth\u000d\u000aoxido cluster by using a cut-out of the \u03b2-Bi2O3 structure as starting\u000d\u000amodel. The examples illustrate that it is possible to obtain\u000d\u000aknowledge of dominating structural motifs from PDF in an\u000d\u000aautomated manner using ML.\u000d\u000aRESULTS\u000d\u000aML-MotEx algorithm\u000d\u000aML-MotEx consists of four steps. These four steps are shown in Fig.\u000d\u000a1 and the simplified pseudo-code of the algorithm in Fig. 2. In the\u000d\u000afirst step, a starting structure model is used to generate a\u000d\u000acatalogue of candidate structure motifs. As detailed in the\u000d\u000aMethods section, the structures are generated by removing\u000d\u000adifferent numbers of atoms from the original starting structure,\u000d\u000aFig. 1 Illustration of the ML-MotEx process. Firstly, a starting model is provided. Using this starting model, a structure catalogue is\u000d\u000agenerated, and the structures in the catalogue are fitted to the experimental data in question. An ML algorithm is then trained to predict Rwp\u000d\u000avalues and finally calculating quantified values of feature importance for the fit quality.\u000d\u000aA.S. Anker et al.\u000d\u000a2\u000d\u000anpj Computational Materials (2022)  213  Published in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences\u000d\u000a1234567890():,;","which results in thousands of smaller, candidate structure motifs.\u000d\u000aIn the second step, a fitting script is used to fit the generated\u000d\u000acandidate structures to the dataset. In the third step, the fitting\u000d\u000aresults are handed to the explainable ML algorithm, which is\u000d\u000aoptimized and trained. By using this information, SHAP values of\u000d\u000athe atoms or structural features in the starting model are\u000d\u000acalculated in the fourth step. The output of the algorithm is thus\u000d\u000athe starting model along with SHAP values, indicating the\u000d\u000aimportance of each individual atom in the structure for the fit\u000d\u000aquality, or in other words; how much each individual atom or\u000d\u000afeature affects the Rwp value either positively or negatively. We\u000d\u000arefer to this value as the \u201catom contribution value\u201d. We\u000d\u000afurthermore define the ratio between the atom contribution\u000d\u000avalue and its uncertainty as the \u201cconfidence factor\u201d. Further\u000d\u000adefinitions and descriptions of the individual steps of the\u000d\u000aalgorithm are given in the Methods section.\u000d\u000aExample 1: Proof-of-concept: identification of the C60\u000d\u000abuckyball\u000d\u000aWe first show the use of ML-MotEx with a simple, proof-of-concept\u000d\u000aexample, using a calculated PDF from an ideal C60 buckyball (Fig.\u000d\u000a3a). The aim is to identify the structural motif, the C60 buckyball,\u000d\u000afrom the data.\u000d\u000aWe first need a starting structure that contains the motifs we\u000d\u000aare looking for. In this simplified example, we use a single unit cell\u000d\u000aof the crystal structure of C6029. However, we discarded all\u000d\u000asymmetry and generated a discrete structure model corresponding to the 132 atoms in one unit cell. This model is shown in Fig.\u000d\u000a3b, where one whole C60 structure (Fig. 3a) is seen along with\u000d\u000afragments of the neighbouring C60 buckyballs. The simulated PDF\u000d\u000aof the C60 buckyball and the starting model is shown in Fig. 3c.\u000d\u000aWe can now use this starting model to generate a catalogue of\u000d\u000astructures, which are all fitted to the data. The structures are\u000d\u000acreated by removing different numbers of atoms from the original\u000d\u000astarting structure, which results in thousands of smaller, candidate\u000d\u000astructure motifs. This model generation and fitting steps are\u000d\u000aidentical to our previously reported brute-force approach, where\u000d\u000awe simply compare the Rwp values of all the fits to identify the\u000d\u000abest structure motif. We first consider this simple approach. One\u000d\u000aof the limitations of the brute-force method is that the possible\u000d\u000acandidate structures are exponential in N, the number of atoms in\u000d\u000athe model. Since each atom in the starting model can be present\u000d\u000aor absent, the number of possible subclusters is equal to 2N \u2212 1.\u000d\u000aFor large models such as the C60 starting model containing 132\u000d\u000aatoms, this is ~1040, a gigantic number, making it impossible to\u000d\u000ainvestigate all candidate structures. For this example, we used\u000d\u000a384,260 structures to train ML-MotEx, which is only a very small\u000d\u000afraction of the 2132 \u2212 1 possible candidate structures. Note that\u000d\u000athe model with a single C60 buckyball was not in the generated\u000d\u000astructure catalogue.\u000d\u000aAll these 384,260 structures were fitted to the PDF calculated\u000d\u000afrom the C60 cluster. Only a scale factor, an isotropic expansion/\u000d\u000acontraction factor, and isotropic Atomic Displacement Parameters\u000d\u000a(ADPs) were refined, as detailed in Supplementary Table 2. We\u000d\u000anote that refinement of the atom positions can be added to the\u000d\u000afitting procedure to expand the chemical space that is investigated. However, this would be computationally expensive, and it\u000d\u000awould allow deviations from the chemical topologies set up in the\u000d\u000astarting model.\u000d\u000aTo get an overview of the results from these fits, we plot the Rwp\u000d\u000avalue versus the number of atoms in the structure. To further\u000d\u000ainvestigate the results, one must visually inspect the fits of the\u000d\u000acatalogue of candidate structure motifs and their Rwp value. Some\u000d\u000aof the candidate structure motifs are shown as inserts in Fig. 3e,\u000d\u000awhere transparent grey atoms represent atoms deleted from the\u000d\u000amodels. The fits of these structures to the dataset are presented in\u000d\u000aFig. 3e, along with the Rwp values. The Rwp value appears to drop\u000d\u000awhen the \u2018outer\u2019 atoms are removed, while it increases when the\u000d\u000aatoms that are part of the centre C60 buckyball are removed. From\u000d\u000ainvestigating these few, but manually selected, structures and\u000d\u000atheir corresponding fitted Rwp value, one can hypothesize that the\u000d\u000astructure giving the best fit should be the C60 buckyball. However,\u000d\u000athis method can be biased by human interaction, and it is timeconsuming and difficult to go through the many fits to extract\u000d\u000astructural information.\u000d\u000aWe, therefore, move on to the ML-MotEx method. Using the\u000d\u000acatalogue of candidate structure motifs and the corresponding\u000d\u000aRwp values obtained above, we train a GBDT model on the training\u000d\u000aset to predict the Rwp value of the candidate structure motifs.\u000d\u000aFigure 4 shows the predicted Rwp values of the ML algorithm\u000d\u000aversus the Rwp value of the structures when they are fitted to the\u000d\u000asimulated C60 dataset in DiffPy-CMI30. For the structures used in\u000d\u000athe test set, the GBDT model predicts the Rwp value with a mean\u000d\u000aabsolute error of 2.0%.\u000d\u000aWe now use explainable ML to explain Rwp values with the use\u000d\u000aof the feature importance tool SHAP values27. As described in\u000d\u000adetail in the Methods section, a SHAP value is calculated for each\u000d\u000astructural feature (here, each atom and the cluster size) for each\u000d\u000acandidate structure motif that is fitted to the PDF during the\u000d\u000atraining process. The amplitude of the SHAP value reflects how\u000d\u000aimportant a structural feature is for the fit quality, while the sign of\u000d\u000athe SHAP value reflects whether the feature affects the Rwp value\u000d\u000aof the fit towards 100% (poor fit) or 0% (perfect fit), in other words\u000d\u000awhy it is important.\u000d\u000aFigure 5a shows the most important results from the SHAP\u000d\u000avalue analysis. The first feature we consider is the number of\u000d\u000aatoms, with SHAP values shown in the top part of Fig. 5a. The plot\u000d\u000arepresents SHAP values for the cluster size feature with the size\u000d\u000ashown on a colour scale, going from small (blue) to large clusters\u000d\u000a(red). From the large amplitude of some of the SHAP values\u000d\u000aobserved from this feature, we see that the number of atoms in\u000d\u000athe structure motif is the most important feature for the Rwp value.\u000d\u000aAll small clusters (0\u201334 atoms, plotted in blue colours) show a\u000d\u000alarge positive SHAP value, which implies that the Rwp value of the\u000d\u000afit to the PDF data is high, i.e. the fit quality is low. All small\u000d\u000aclusters can thereby be discarded as structural models for\u000d\u000asatisfyingly describing the data.\u000d\u000aFig. 2 Pseudo-code describing the four steps of ML-MotEx. A starting model, fitting script, and dataset are given as input. Firstly, a catalogue\u000d\u000aof candidate structure motifs is generated (step 1), which are fitted to the dataset (step 2). The output from step 1 and 2 is then given to an ML\u000d\u000aalgorithm, which learns to predict goodness-of-fit (Rwp) values based on the structure motif (step 3). Lastly, SHAP values are calculated for each\u000d\u000afeature (step 4) which can be converted to atom contribution values.\u000d\u000aA.S. Anker et al.\u000d\u000a3\u000d\u000aPublished in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences npj Computational Materials (2022)  213 ","Next, we can investigate the SHAP values obtained for the\u000d\u000aindividual atoms in the structure. We first consider atom 13, as\u000d\u000alabelled in the structure drawing in Fig. 5b. The SHAP values\u000d\u000aobtained from this atom for each of the fits in the training set are\u000d\u000aall plotted on the SHAP axis. For the models where the atom is not\u000d\u000apresent in the model, the SHAP value is shown in blue, while it is\u000d\u000ashown in red for the atoms where it is present in the model. If first\u000d\u000aconsidering the cases where the atom is kept in the model, the\u000d\u000aatom 13 SHAP values are generally negative, which means that\u000d\u000athe presence of this atom pushes the Rwp value towards 0%. We\u000d\u000ainterpret this as ML-MotEx wanting to keep the atom in the model.\u000d\u000aThe SHAP values obtained for the fits without the atom present\u000d\u000aare positive, which confirms that if removing the atom, the fit\u000d\u000aquality becomes worse. Based on the SHAP values obtained for\u000d\u000athe atom in each fit, we calculate an atom contribution value. The\u000d\u000aatom contribution value is defined in the Methods section and is\u000d\u000acalculated as the difference between the average SHAP values\u000d\u000aobtained for the atom when kept in the model, and when\u000d\u000aremoved from the model. A negative atom contribution value\u000d\u000ameans that the atom pushes the Rwp value down if kept in the\u000d\u000astructure. The atom contribution value obtained for atom 13 is\u000d\u000anegative, and we, therefore, colour it yellow in the structural\u000d\u000arepresentation in Fig. 5b to indicate that it should be kept in the\u000d\u000amodel. We use this strategy to automatically go through all the\u000d\u000aatoms in the starting model and colour them yellow/black based\u000d\u000aon their impact on the Rwp value. The result can be seen in Fig. 5b\u000d\u000awhere the 60 atoms with the lowest atom contribution values are\u000d\u000acoloured yellow. The results are also shown in Supplementary Fig.\u000d\u000a1, where the atom contribution values are plotted using a\u000d\u000acontinuous colour bar. The results show that ML-MotEx mainly\u000d\u000afavours the atoms comprising the central buckyball. While the\u000d\u000aaverage confidence factor (as defined in the Methods section) is\u000d\u000a1.26 for all of the atoms in the starting model, we observe that the\u000d\u000aaverage confidence factor of the mislabelled atoms is 0.37,\u000d\u000ameaning that ML-MotEx is less confident about the atom\u000d\u000acontribution values of those.\u000d\u000aThe ML-MotEx algorithm thus provides an unbiased method to\u000d\u000aextract important motifs from PDF data, without any inputs other\u000d\u000athan a starting model and a fitting script. We emphasize that the\u000d\u000astructural motifs extracted with ML-MotEx are based on the Rwp\u000d\u000avalue of the fits and are thereby not necessarily physically\u000d\u000areasonable. It is therefore important to still critically consider the\u000d\u000aFig. 3 Analysis of a simulated PDF from a C60 buckyball. a C60 buckyball, b single C60 unit cell29, treated as a discrete structure with 132\u000d\u000aatoms and c their simulated PDFs. The simulation parameters mimic typical values of a PDF dataset and are presented in Supplementary Table\u000d\u000a3. d Rwp values obtained in the fits using the C60 structure catalogue, plotted as a function of number of atoms in the structure motifs. Note\u000d\u000athat the model with a single C60 buckyball is not included in the set of 384,260 structures tested. This would result in a perfect fit with\u000d\u000aRwp = 0%. e Examples of candidate structure motifs with their corresponding fits to the simulated C60 buckyball data. Grey, semitransparent\u000d\u000aatoms are removed from the starting model.\u000d\u000aFig. 4 Predicted Rwp values versus true Rwp values. Rwp values\u000d\u000afrom the fits of the catalogue structures to the simulated C60\u000d\u000adataset, plotted versus the predicted Rwp values from the GBDT\u000d\u000amodel from the same structures. The mean squared error (MSE) and\u000d\u000athe mean absolute error (MAE) are based on all 76,852 predictions in\u000d\u000athe test set, which are structures the model has not been trained on.\u000d\u000aA.S. Anker et al.\u000d\u000a4\u000d\u000anpj Computational Materials (2022)  213  Published in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences","extracted motif with chemical knowledge, in the same manner as\u000d\u000afor conventional PDF refinements. In this process, one could refine\u000d\u000aadditional parameters such as atom positions. Consequently, in\u000d\u000aFig. 5b, the user should identify the full C60 buckyball as the\u000d\u000astructural motif rather than just choosing the motif of the yellow\u000d\u000aatoms. Another approach to avoid unphysically results from MLMotEx would be to include e.g. density function theory (DFT)\u000d\u000acalculations in the goodness-of-fit value.\u000d\u000aExample 2: Identification of structural motifs in disordered\u000d\u000amolybdenum oxides\u000d\u000aAs discussed above, we have recently used the brute-force\u000d\u000aautomated modelling method to identify structural motifs in\u000d\u000adisordered molybdenum oxides from PDF analysis7\u000d\u000a. Here we show\u000d\u000athat by reassessing the data with ML-MotEx, we can reproduce the\u000d\u000aresults from Christiansen et al.7 but in an automated way that\u000d\u000aallows analysis of the resulting structure model using SHAP values.\u000d\u000aFigure 6a shows the difference-PDF (d-PDF) obtained from\u000d\u000aamorphous molybdenum oxide supported on \u03b3-Al2O3 nanoparticles (15 w% Mo), where the signal from the \u03b3-Al2O3 nanoparticles\u000d\u000ahas been subtracted. The d-PDF thus only reflects the structure of\u000d\u000athe supporting material. The aim is to develop a structural model\u000d\u000afor the amorphous MoOx. In our previous paper, different starting\u000d\u000amodels were tested, which were all based on structures of\u000d\u000amolybdenum-based polyoxometalates (POMs) built from [MoO4]\u000d\u000atetrahedra and [MoO6] octahedra. The analysis showed that the\u000d\u000abest fitting models did not contain tetrahedral motifs. Instead, the\u000d\u000abrute-force automated modelling approach hinted at a unit of\u000d\u000athree edge-sharing [MoO6] octahedra, or a \u2018triad\u2019, to be present in\u000d\u000athe structure. However, the use of the computationally expensive\u000d\u000abrute-force method limited the number of atoms that could be\u000d\u000aincluded in the starting model. This meant that a range of\u000d\u000adifferent smaller starting models were used to test different\u000d\u000astructure hypotheses. With ML-MotEx, we can instead test much\u000d\u000alarger systems and thereby include several different structural\u000d\u000amotifs at the same time in one starting model, as well as\u000d\u000aquantitatively analyse the results using SHAP values. We, therefore, use a larger POM as starting model, namely the entire\u000d\u000aMo36O128 cluster cut-out of the K8(Mo36O112(H2O)16)\u00b7(H2O)37\u000d\u000acrystal structure31, which contains a range of different chemical\u000d\u000atopologies. Figure 6a shows the simulated data from the Mo36O128\u000d\u000acluster, which has some similarities to the experimental PDF, and\u000d\u000aFig. 6b shows the structure of the Mo36O128 cluster.\u000d\u000aWe apply ML-MotEx to the molybdenum oxide system in the\u000d\u000asame manner as we did to the C60 buckyball. First, we used the\u000d\u000astarting model to make a catalogue of candidate structure motifs,\u000d\u000aas described in detail in the Methods section. These are all fit to\u000d\u000athe experimental PDF, and the results are used to train the GBDT\u000d\u000amodel. The fits are made with the same fitting algorithm as used\u000d\u000ain the paper from Christiansen et al.7 Figure 6c illustrates the Rwp\u000d\u000avalues of the fits, plotted as a function of the number of\u000d\u000amolybdenum atoms present in the structural motif. The best\u000d\u000afitting models contain 5\u20137 molybdenum atoms. The model that\u000d\u000afits the data with the lowest Rwp value (45%) can be identified as a\u000d\u000aMo5O24 structure, as shown in Supplementary Fig. 2. However, it is\u000d\u000adifficult to justify that this structural model is unique in\u000d\u000arepresenting the structure in the sample, purely based on the\u000d\u000aRwp value.\u000d\u000aWe, therefore, use steps 3 and 4 of ML-MotEx to analyse the\u000d\u000aresults of the ensemble of fits. The resulting SHAP values are\u000d\u000ashown in Fig. 7a. The plot should be interpreted in the same way\u000d\u000aas for the C60 example: Each atom is assigned a SHAP value in\u000d\u000aeach of the fits in the training set. For the models where the atom\u000d\u000ais not present in the model, the SHAP value is shown in blue, while\u000d\u000ait is shown in red for the atoms where it is present in the model.\u000d\u000aWhen considering the amplitudes of the SHAP values, we see that\u000d\u000athe atoms labelled with 14, 15, 19, and 20 are marked as very\u000d\u000aimportant by ML-MotEx. When these atoms are present in the\u000d\u000astructure (red), they all have large negative SHAP values,\u000d\u000aindicating that their presence in the model pushes the Rwp down.\u000d\u000aWhen they are not present in the structure (blue), they all have\u000d\u000alarge positive SHAP values, also indicating that they should be\u000d\u000apresent in the structure to obtain a good fit. Atoms 22 and 23 are\u000d\u000aexamples of atoms that ML-MotEx do not suggest keeping in the\u000d\u000astructure. As seen from the SHAP values, their presence pushes up\u000d\u000athe Rwp value.\u000d\u000aFig. 5 Summary of the ML-MotEx analysis of C60 PDF. a Plot of the SHAP values obtained in the C60 analysis, showing if atoms in the starting\u000d\u000amodel are favourable for the fit quality. For the models where the atom is not present in the model, the SHAP value is shown in blue, while it is\u000d\u000ashown in red for the atoms where it is present in the model. The SHAP values are plotted as a violin plot. An enlarged summary SHAP plot of\u000d\u000apanel a is shown in Supplementary Fig. 14. b Structural visualization of kept and removed atoms. The atoms with the 60 lowest atoms\u000d\u000acontribution values have been coloured yellow, while the rest are coloured black. Supplementary Figure 1 shows a similar representation but\u000d\u000awhere the atom contribution values are directly shown from a continuous colour bar.\u000d\u000aA.S. Anker et al.\u000d\u000a5\u000d\u000aPublished in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences npj Computational Materials (2022)  213 ","Based on the SHAP analysis, atom contribution values were\u000d\u000acalculated. The results are visually illustrated in Fig. 7b, where the\u000d\u000amolybdenum atoms in the structure are coloured yellow if the\u000d\u000aatom contributed to a better fit quality, otherwise it is coloured\u000d\u000ablack. Figure 7b clearly shows a specific motif that ML-MotEx\u000d\u000awants to keep in the model. The yellow molybdenum atoms are all\u000d\u000apart of a \u2018triad\u2019 structure, where three [MoO6] octahedra share\u000d\u000aedges and all oxygen atoms that bond to 3 or 4 Mo atoms is\u000d\u000aconnected to yellow molybdenum atoms. This is further illustrated\u000d\u000ain Supplementary Fig. 3. Specifically, the resulting structural unit\u000d\u000athat ML-MotEx wants to keep is similar to heptamolybdate\u000d\u000a[Mo7O24]\u000d\u000a6\u2212, which can be described as several triads connected\u000d\u000aFig. 6 Analysis of experimental PDF from disordered molybdenum oxide. a Comparison of experimental PDF from a disordered\u000d\u000amolybenum oxide7\u000d\u000a, and simulated data from Mo36O128 cluster, used as starting model. The simulation parameters mimic typical values of a\u000d\u000aPDF dataset and can be seen in Supplementary Table 3. b Structure of the Mo36O128 cluster. c Rwp values obtained in the fits using the\u000d\u000aMo36O128 structure catalogue, plotted as a function of number of atoms in the structure motifs.\u000d\u000aFig. 7 Summary of the ML-MotEx analysis of experimental PDF from disordered molybdenum oxide. a Plot of the SHAP values obtained in\u000d\u000athe molybdenum oxide analysis, showing if atoms in the starting model Mo36O128 are favourable for the fit quality. For the models where the\u000d\u000aatom is not present in the model, the SHAP value is shown in blue, while it is shown in red for the atoms where it is present in the model. The\u000d\u000aSHAP values are plotted as a violin plot. An enlarged summary SHAP plot of panel a is shown in Supplementary Fig. 15. b Structural\u000d\u000avisualization of kept (yellow) and removed (black) atoms. Supplementary Fig. 3 shows a similar representation but where the atom\u000d\u000acontribution values are directly shown from a continuous colour bar.\u000d\u000aA.S. Anker et al.\u000d\u000a6\u000d\u000anpj Computational Materials (2022)  213  Published in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences","through edge-sharing. These results indicate that a motif of\u000d\u000aconnected edge-sharing triads, as shown in Fig. 7b, is important in\u000d\u000aorder to describe the data of the disordered molybdenum oxides,\u000d\u000awhich was also found by Christiansen et al.7 We note here that\u000d\u000awhen fitting this model to the PDF itself, we cannot describe the\u000d\u000amedium-range order present in the PDF. The ML-MotEx rather\u000d\u000aallows identifying the main local motifs in the data.\u000d\u000aExample 3: Identification of the ionic cluster structure from\u000d\u000aPDFs\u000d\u000aTo investigate the reproducibility of the ML-MotEx method, we\u000d\u000ainvestigate if similar results are achieved with different starting\u000d\u000amodels, all containing the correct structure motif. We here model\u000d\u000aa PDF obtained from a solution of 0.05 M ammonium metatungstate hydrate, (NH4)6[H2W12O40]\u00b7H2O in water, which dissolves to\u000d\u000aform monodisperse \u03b1-Keggin clusters28. Experimental details are\u000d\u000aprovided in Supplementary Methods.\u000d\u000aTo test the ML-MotEx method, we use four different starting\u000d\u000amodels of tungstate oxide crystals, all including the \u03b1-Keggin\u000d\u000acluster motif with varying complexity. Unit cells from the four\u000d\u000afollowing crystal structures were used as starting models:\u000d\u000a[Hpy]4H2[H2W12O40] (py = pyridine) [1]32, (CH3)4N)4SiW12O40 [2]33,\u000d\u000a(((CH3)2NH2)6 (Cu(HCON(CH3)2)4)(GeW12O40)2)(HCON(CH3)2)2\u000d\u000a34 [3],\u000d\u000aand ((CH3)2NH2)3(PW12O40) [4]35. Again, we discarded all symmetry\u000d\u000aand generated a discrete structure model corresponding to the\u000d\u000aatoms in one single unit cell. All other atoms than tungsten and\u000d\u000aoxygen were furthermore removed from the structures before\u000d\u000acatalogue structures were created. Figure 8a shows the experimental dataset with simulated PDFs from the four different starting\u000d\u000amodels. Figure 8b illustrates a W12O40 \u03b1-Keggin structure.\u000d\u000aAgain, we first build structure catalogues based on the starting\u000d\u000amodels (step 1) and fit them to the experimental PDF (step 2). In\u000d\u000athis case, we extract 104 structures from each starting model,\u000d\u000awhich is just a small fraction of all possible structures that can be\u000d\u000amade from the starting models that have 24 ([2]), 48 ([1] and [3]),\u000d\u000aand 72 ([4]) atoms that are permuted. Again, a GBDT model was\u000d\u000atrained to predict the Rwp values of the structures (step 3), and\u000d\u000aSHAP values were obtained to calculate atom contribution values\u000d\u000a(step 4). The resulting SHAP value plots can be seen in\u000d\u000aSupplementary Figs. 4\u20135. While ML-MotEx takes about 100 s on\u000d\u000aan AMD Ryzen Threadripper 3990X with 64-core 2.9/4.3 GHz using\u000d\u000a104 fits on a structure with 48 atoms, it would take about ~3 \u00d7 106\u000d\u000ayears (Supplementary Notes 1) to make fits of all the 248 \u2212 1\u000d\u000apossible structures using the brute-force approach. Supplementary Table 5 in the Supplementary Information shows the exact\u000d\u000acomputer time of the fits on a MacBook Pro and a Threadripper,\u000d\u000awhich clearly demonstrates the scalability of ML-MotEx.\u000d\u000aFigure 9 shows the results of applying ML-MotEx to the 4\u000d\u000adifferent starting models. For structures [1], [3], and [4], the 24\u000d\u000aatoms most preferred by ML-MotEx were coloured yellow, while\u000d\u000athe rest were coloured black. For structure [2], 12 atoms were\u000d\u000acoloured yellow. In all 4 examples, the yellow atoms have a motif\u000d\u000aof a \u03b1-Keggin cluster, however, in Fig. 9c, d, we see a few\u000d\u000amislabelled atoms (2 of 24 atoms in the worst case). The\u000d\u000amislabelled atoms are found in the starting models containing\u000d\u000amost atoms, i.e. with the highest permutation value N. To achieve\u000d\u000aa better prediction, we could have built larger catalogues of\u000d\u000acandidate structure motifs and thus performed more fits. We,\u000d\u000atherefore, conclude that the ML-MotEx method is not completely\u000d\u000ainsensitive to the starting model, but that it yields very similar\u000d\u000aresults for all the tested starting models if it contains similar\u000d\u000amotifs. Furthermore, the example shows that ML-MotEx can be\u000d\u000aused to investigate PDF data from clusters in solution, whose\u000d\u000astructure also is part of known crystal structures. As seen from the\u000d\u000aresults in Supplementary Figs. 7\u201310, we performed an identical\u000d\u000aanalysis of a different dataset also obtained from the second\u000d\u000asolution of 0.05 M ammonium metatungstate hydrate. This\u000d\u000aanalysis provided highly comparable results, as discussed in the\u000d\u000aSupplementary Information. This illustrates the reproducibility of\u000d\u000athe method. In Supplementary Discussion 1, we discuss what\u000d\u000ahappens if a poor staring model is used, and how one can identify\u000d\u000aif the starting model does contain the right motif using the\u000d\u000aconfidence factor.\u000d\u000aWe have also used the ML-MotEx method for a larger ionic\u000d\u000acluster, namely [Bi38O45]. Here, we use the \u03b2-Bi2O3 structure as\u000d\u000astarting model and used a \u2018cookie-cutter\u2019 strategy to generate\u000d\u000astructures for the motif catalogue. This example is described in\u000d\u000aSupplementary Discussion 2, and the \u2018cookie-cutter\u2019 approach is\u000d\u000ashown in Supplementary Fig. 11.\u000d\u000aDISCUSSION\u000d\u000aIn the four examples presented above, we have shown how\u000d\u000aexplainable ML can aid in identifying structural motifs in\u000d\u000ananostructured materials and presented an approach to structure\u000d\u000acharacterization. Traditional PDF analysis investigates how an\u000d\u000aentire structure model agrees with an experimental PDF, rather\u000d\u000athan identifying how different features in the model affect the fit\u000d\u000aquality. Instead, ML-MotEx provides a quantitative measure of how\u000d\u000aeach atom or feature contributes to the fit. The use of ML\u000d\u000afurthermore allows the screening of a large number of models in\u000d\u000aan automated and fast manner. In the examples described here,\u000d\u000aML-MotEx has been used with various starting models with up to\u000d\u000a256 metal atoms; however, the algorithm can handle larger\u000d\u000asystems, as it is highly scalable. In comparison, a full brute-force\u000d\u000aapproach is computationally restricted to systems with up to\u000d\u000a15\u201330 atoms. For the type of systems described here, it is possible\u000d\u000ato use the method in quasi-experimental time, which could, for\u000d\u000aexample, be useful for analysis of time-resolved scattering data,\u000d\u000awhere the structural motifs present might change with time,\u000d\u000awhich would be revealed by changing SHAP values.\u000d\u000aFig. 8 Experimental PDF from Keggin clusters in solution.\u000d\u000aa Comparison of experimental data from a 0.05 M ammonium\u000d\u000ametatungstate hydrate solution, and simulated PDFs from the four\u000d\u000adifferent starting models [1]\u2013[4]. The simulation parameters mimic\u000d\u000atypical values of a PDF dataset and can be seen in Supplementary\u000d\u000aTable 3. b The W12O40 \u03b1-Keggin structure.\u000d\u000aA.S. Anker et al.\u000d\u000a7\u000d\u000aPublished in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences npj Computational Materials (2022)  213 ","ML-MotEx shares some similarities with the cluster build-up\u000d\u000aalgorithm LIGA11,12, which automatically builds clusters of\u000d\u000adifferent sizes based on information that is contained in interatomic distance lists extracted from the PDF. LIGA has shown to\u000d\u000abe successful at automatically reconstructing clusters (up to 150\u000d\u000aatoms) with no user input except the inter-atomic distance list,\u000d\u000aextracted from an experimental PDF, and at a low computational\u000d\u000acost. However, its use has not caught on because extracting the\u000d\u000adistance list from the data presents significant practical difficulties,\u000d\u000aand is not unique. As with ML-MotEx it uses the error each atom in\u000d\u000aa cluster contributes to the fit to weigh the decision about which\u000d\u000aatom to include in the model. Presumably, part of the success of\u000d\u000aLIGA and ML-MotEx is its use of this atom contribution for rapidly\u000d\u000afinding good candidate motifs. Unlike LIGA, ML-MotEx requires a\u000d\u000astarting model that contains the target structural motif, and it\u000d\u000aleverages ML to rapidly compute the atom contributions. It can\u000d\u000atherefore be positioned between traditional refinement (where\u000d\u000athe complete starting model is needed) and LIGA (which is ab\u000d\u000ainitio) as it finds structural motifs from within a larger model as a\u000d\u000astarting model for subsequent refinement. However, it has a\u000d\u000asignificant advantage over LIGA that it works directly on the\u000d\u000ameasured PDF and does not require the inter-atomic distance list\u000d\u000ato be extracted from the PDF data, and we expect it to be of great\u000d\u000apractical value. With this in mind, we plan to deploy ML-MotEx as\u000d\u000aan application on the PDFitc.org web server36.\u000d\u000aIt may be considered a significant drawback that ML-MotEx\u000d\u000arequires as an input a structure fragment that contains the target\u000d\u000amotif within it in order to work. We provide a confidence factor for\u000d\u000athe starting model, but ML-MotEx still requires significant\u000d\u000achemical/structural knowledge and intuition to be of use. We first\u000d\u000anote that such intuition is widespread in the chemistry community\u000d\u000aand is unlikely to be a significant drawback in practice. For\u000d\u000aexample, we have recently used the method to identify the\u000d\u000astructure of intermediates in the formation of transition metal\u000d\u000atungstates from polyoxometalate ions using in situ PDF data37,\u000d\u000aand for identifying stacking fault domain sizes in manganese\u000d\u000aoxides from PDF and PXRD38. We also note that the method is\u000d\u000asufficiently fast that it would be possible to combine it with\u000d\u000astructural screening applications such as structureMining@PDFitc15,36. Given chemical information about elements that are\u000d\u000apresent, structureMining searches structural databases for candidate structures. These are then refined to a target dataset, and a\u000d\u000arank-ordered list is returned to the user. If the PDF represents a\u000d\u000asignal from a short-range ordered structural motif, we could insert\u000d\u000aML-MotEx between the database mining and refinement steps to\u000d\u000asearch over sets of plausible structures to look for structural submotifs. It may be possible to first use structure mining to identify\u000d\u000astarting models, which could then be used for ML-MotEx analysis.\u000d\u000aThe models could then be further evaluated using both the\u000d\u000aresulting Rwp values and confidence factor.\u000d\u000aThe ML-MotEx method is currently limited to PDF analysis in the\u000d\u000afitting procedure of the algorithm (step 2), however, the rest of\u000d\u000aML-MotEx (step 1 + 3 + 4) is ready to use with data from other\u000d\u000atechniques. We are confident that a similar approach, taking\u000d\u000aadvantage of explainable ML and SHAP values, can be broadly\u000d\u000auseful for enhancing and developing how models for data analysis\u000d\u000aare identified and constructed.\u000d\u000aMETHODS\u000d\u000aStep 1: Creation of a catalogue of candidate structure motifs\u000d\u000aThe first step in ML-MotEx is to use a starting structure model to\u000d\u000agenerate a catalogue of candidate structure motifs, which are all\u000d\u000afitted to the data. The structures are generated by removing\u000d\u000adifferent numbers of atoms from the original starting structure\u000d\u000aresulting in thousands of smaller, candidate structure motifs.\u000d\u000aThis process, which we refer to as \u2018structure permutation\u2019, is\u000d\u000aillustrated in Fig. 10. Here, the starting model contains four metal\u000d\u000aatoms, which are each bonded to six oxygen atoms. Before\u000d\u000acandidate structure motifs are generated, we select which atom\u000d\u000atype should be included in the permutation process. For the\u000d\u000aproject discussed here, this selection is based on the X-ray\u000d\u000ascattering power of the atoms (i.e. heavier atoms scatter X-rays\u000d\u000astrongly, while lighter ones do not), and we, therefore, choose to\u000d\u000apermute over the four metal atoms in the structure rather than\u000d\u000aoxygen atoms. The total number of atoms that are selected for\u000d\u000apermutation (here 4) is referred to as the permutation number, N.\u000d\u000aNote that we do not take symmetry into account in this process.\u000d\u000aThe selected atoms are removed or kept in the model by\u000d\u000arandomly associating them with zeros and ones, where 0 means\u000d\u000athat we remove the atom and 1 means we keep it. This is repeated\u000d\u000amultiple times to generate a large catalogue of candidate\u000d\u000astructure motifs. The total number of possible motifs from the\u000d\u000apermutations is equal to 2N \u2212 1, but only a small fraction of these\u000d\u000aneeds to be produced for ML-MotEx to provide satisfactory results.\u000d\u000aIn Supplementary Discussion 3, we discuss how large a catalogue\u000d\u000aof candidate structure motifs ML-MotEx needs as training data to\u000d\u000aoutput reasonable results. This is likely to be highly system\u000d\u000adependent and especially dependent on N and structural\u000d\u000asymmetry. For the examples presented in the paper, we use\u000d\u000a~140\u20133000 structure motifs per N.\u000d\u000aThe atoms which were not chosen for permutation, in this case\u000d\u000aoxygen, are removed if they are not within a distance threshold\u000d\u000afrom any other atom. The threshold is user-defined and can be set\u000d\u000aFig. 9 Summary of the ML-MotEx analysis of experimental PDFs from Keggin clusters in solution. Results from the ML-MotEx method on a\u000d\u000aPDF from a solution of ammonium metatungstate hydrate, using four different starting models: a [Hpy]4H2[H2W12O40] (py = pyridine)32,\u000d\u000ab (CH3)4N)4SiW12O4033, c (((CH3)2NH2)6 (Cu(HCON(CH3)2)4)(GeW12O40)2)(HCON(CH3)2)2\u000d\u000a34, d ((CH3)2NH2)3(PW12O40)\u000d\u000a35. Atoms kept by ML-MotEx\u000d\u000aare shown in yellow while removed atoms are shown in black. The kept atoms were chosen as the 24 atoms (model A), 12 atoms (model B), 24\u000d\u000aatoms (model C), and 24 atoms (model D) with the lowest atom contribution values. In Supplementary Fig. 6 a similar representation is shown,\u000d\u000abut where the atom contribution values are directly shown using a continuous colour bar.\u000d\u000aA.S. Anker et al.\u000d\u000a8\u000d\u000anpj Computational Materials (2022)  213  Published in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences","according to PDF peaks and/or chemically valid distances (i.e.\u000d\u000abond lengths) for the expected compounds.\u000d\u000aStep 2: Fitting the catalogue of candidate structure motifs to\u000d\u000athe data\u000d\u000aIn the next step, we fit each of the candidate structures in the\u000d\u000acatalogue to the experimental PDF. We here use the Python-based\u000d\u000aprogram DiffPy-CMI30 for PDF fitting39\u201341 and apply the Debye\u000d\u000aequation for the calculation of scattering intensities and PDFs\u000d\u000afrom the structures. The fitting strategies and parameters for each\u000d\u000aof the examples presented below are listed in Supplementary\u000d\u000aTable 2. The output of the fit is a Rwp value reflecting the quality of\u000d\u000athe fit:\u000d\u000aRwp \u00bc\u000d\u000aPffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffiffi n\u000d\u000ai\u00bc1 Gobs\u00f0 \u00deri $ Gcalc ri \u00bd & \u00f0 \u00de ; P 2\u000d\u000aPn\u000d\u000ai\u00bc1 Gobs\u00f0 \u00deri\u000d\u000a2\u000d\u000as\u000d\u000a' 100 % (1)\u000d\u000aHere, Gobs and Gcalc are the observed and calculated PDFs, and P is\u000d\u000athe refinement parameters in the model.\u000d\u000aStep 3: Predicting Rwp values using Gradient Boosting\u000d\u000aDecision Trees\u000d\u000aGradient Boosting Decision Trees (GBDTs)25 are a tool that can do\u000d\u000aclassification or regression using decision trees. In this work, we\u000d\u000aare using XGBoost25 as the GBDT algorithm to do the regression\u000d\u000atask of predicting the fit quality (step 2) based on the structural\u000d\u000ainput given as zeros or ones (step 1) and the number of atoms in\u000d\u000athe structure. Figure 10b shows the input to the GBDT model.\u000d\u000aThe optimization is done by making trees of \u2018yes\u2019 and \u2018no\u2019\u000d\u000aquestions on whether to keep an atom in the structure or not,\u000d\u000abased on the resulting Rwp value. A hypothetical example of a\u000d\u000asimple tree can be seen in Fig. 1, step 3. When atom 4 is present in\u000d\u000athe structure, the GBDT model will predict a Rwp value which is 5%\u000d\u000alower than if atom 4 is not present in the structure. In the same\u000d\u000away, it will predict an Rwp value which is 12% lower if atom 1 is\u000d\u000apresent in the structure. In the decision tree, the algorithm will\u000d\u000atherefore say \u2018yes\u2019 to keep both atoms 1 and 4 in the structure. In\u000d\u000athis project, the GBDT model predicts the Rwp value using a\u000d\u000aweighted average of 100 trees.\u000d\u000aThe GBDT model performance is improved with a large amount\u000d\u000aof training data, which in this tool is provided by creating a larger\u000d\u000acatalogue of candidate structure motifs and fitting them to\u000d\u000athe data.\u000d\u000aThe GBDT model is trained on 80% of the data, which is referred\u000d\u000ato as the training set. XGBoost25 were used with default\u000d\u000aparameters except for learning rate and max depth, which were\u000d\u000aoptimized with the use of Bayesian optimization using 50\u000d\u000aiterations and cross-validation split on 342,43. While this procedure\u000d\u000aautomates the hyperparameter tuning, we demonstrate in\u000d\u000aSupplementary Fig. 12 that similar results are achieved across\u000d\u000avarious hyperparameters, and in Supplementary Fig. 13 we show\u000d\u000athat similar results are achieved across various seeds. The last 20%\u000d\u000aof the data is used to evaluate the performance of the algorithm\u000d\u000aand is referred to as the test set.\u000d\u000aStep 4: Quantifying the effect of structural features using\u000d\u000aSHAP values, assigning atomic contribution values\u000d\u000aSHAP values are used to analyse the Rwp values resulting from the\u000d\u000aprocess described above. For each fit (step 2), each atom in the\u000d\u000astarting model is assigned a SHAP value. The amplitude of the\u000d\u000aSHAP value reflects how important a structural feature is for the fit\u000d\u000aquality, while the sign of the SHAP value reflects whether the\u000d\u000afeature affects the Rwp value of the fit towards 1 (poor fit) or 0\u000d\u000a(perfect fit), in other words why it is important. Each atom in the\u000d\u000astarting model will thus get F number of SHAP values, where F\u000d\u000acorresponds to the number of fits made in step 2 of the algorithm.\u000d\u000aWe divide the F number of SHAP values into two categories; firstly,\u000d\u000athe ones where the atom was kept in the structure motif (kept\u000d\u000aatom SHAP value list) and second, the ones where the atom was\u000d\u000aremoved to create the structure motif (removed atom SHAP value\u000d\u000alist). From each of the two lists, an average SHAP value for the\u000d\u000aatoms can be calculated, defined as SHAPaverage-kept and\u000d\u000aSHAPaverage-removed. We then define an atom contribution value,\u000d\u000awhich is calculated as the difference between two average SHAP\u000d\u000avalues, i.e.:\u000d\u000aAtom contribution value \u00bc SHAPaverage-kept $ SHAPaverage-removed\u000d\u000a(2)\u000d\u000aFig. 10 Example of how structure motifs can be extracted from a starting model with 4 metal atoms coordinated to oxygen and used as\u000d\u000ainput to the GBDT model. a The metal atoms are permuted randomly by creating an array of zeros and ones, where 0 refers to a deleted atom\u000d\u000aand 1 refers to an atom that is kept in the structure. Oxygen atoms are removed if they do not bond to any metal atoms within a distance\u000d\u000athreshold that is set by the user. Note that the metal atoms (blue) are slightly distorted from the centre of the octahedra. b Example of how\u000d\u000athe four structures from panel a and Fig. 1 are given as input to the GBDT model which predicts the Rwp value.\u000d\u000aA.S. Anker et al.\u000d\u000a9\u000d\u000aPublished in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences npj Computational Materials (2022)  213 ","We also define the uncertainty of this value as described in Eq. 3:\u000d\u000aAtom contribution value RMS \u00bc \u00f0SHAPRMS\u000d\u000aaverage$kept\u000d\u000a2 $ SHAPRMS\u000d\u000aaverage$removed\u000d\u000a2\u000d\u000a\u00de\u000d\u000a0:5\u000d\u000a(3)\u000d\u000aWe define a confidence factor for each atom that describes how\u000d\u000aconfident we can be about including/excluding that atom in a\u000d\u000astructural motif;\u000d\u000aConfidence factor \u00bc atom contribution value=atom contribution value RMS\u000d\u000a(4)\u000d\u000aThe results of ML-MotEx can be visually inspected as the atoms in\u000d\u000athe starting model are coloured according to their atom contribution value, using yellow for low atom contribution value (tendency\u000d\u000ato keep atom, pushing Rwp down) and black for high atom\u000d\u000acontribution value (tendency to remove atom, pushing Rwp up). MLMotEx outputs a VESTA44 and CrystalMaker45 file where all the\u000d\u000aatoms are coloured with regard to their atom contribution value.\u000d\u000aDATA AVAILABILITY\u000d\u000aThe authors declare that the data supporting this study are available within the\u000d\u000apaper, its Supplementary Information files and the associated Github to the paper:\u000d\u000ahttps://github.com/AndySAnker/ML-MotEx. Additional data that support the findings\u000d\u000aof this study are available from the corresponding author upon request.\u000d\u000aCODE AVAILABILITY\u000d\u000aThe authors declare that the codes supporting this study are available on the\u000d\u000aassociated Github to the paper: https://github.com/AndySAnker/ML-MotEx. Additional codes that support the findings of this study are available from the\u000d\u000acorresponding author upon request.\u000d\u000aReceived: 25 April 2022; Accepted: 14 September 2022;\u000d\u000aREFERENCES\u000d\u000a1. Billinge, S. J. L. & Kanatzidis, M. G. Beyond crystallography: the study of disorder,\u000d\u000ananocrystallinity and crystallographically challenged materials with pair distribution functions. Chem. Commun. 7, 749\u2013760 (2004).\u000d\u000a2. Keen, D. A. & Goodwin, A. L. The crystallography of correlated disorder. Nature\u000d\u000a521, 303\u2013309 (2015).\u000d\u000a3. Christiansen, T. L., Cooper, S. R. & Jensen, K. M. \u00d8. There\u2019s no place like real-space:\u000d\u000aelucidating size-dependent atomic structure of nanomaterials using pair distribution function analysis. Nanoscale Adv. 2, 2234\u20132254 (2020).\u000d\u000a4. Billinge, S. J. L. & Levin, I. The problem with determining atomic structure at the\u000d\u000ananoscale. Science 316, 561\u2013565 (2007).\u000d\u000a5. Juelsholt, M. et al. Size-induced amorphous structure in tungsten oxide nanoparticles. Nanoscale 13, 20144\u201320156 (2021).\u000d\u000a6. Yang, X. et al. Confirmation of disordered structure of ultrasmall CdSe nanoparticles from X-ray atomic pair distribution function analysis. Phys. Chem. Chem.\u000d\u000aPhys. 15, 8480\u20138486 (2013).\u000d\u000a7. Christiansen, T. L. et al. Structure analysis of supported disordered molybdenum\u000d\u000aoxides using pair distribution function analysis and automated cluster modelling.\u000d\u000aJ. Appl. Crystallogr. 53, 148\u2013158 (2020).\u000d\u000a8. Bennett, T. D. & Cheetham, A. K. Amorphous metal\u2013organic frameworks. Acc.\u000d\u000aChem. Res. 47, 1555\u20131562 (2014).\u000d\u000a9. Kj\u00e6r, E. T. S. et al. DeepStruc: Towards structure solution from pair distribution\u000d\u000afunction data using deep generative models. Preprint at https://chemrxiv.org/\u000d\u000aengage/chemrxiv/article-details/62fa600cd0c5cb353465329f (2022).\u000d\u000a10. Cliffe, M. J., Dove, M. T., Drabold, D. & Goodwin, A. L. Structure determination of\u000d\u000adisordered materials from diffraction data. Phys. Rev. Lett. 104, 125501 (2010).\u000d\u000a11. Juh\u00e1s, P., Cherba, D. M., Duxbury, P. M., Punch, W. F. & Billinge, S. J. L. Ab initio\u000d\u000adetermination of solid-state nanostructure. Nature 440, 655\u2013658 (2006).\u000d\u000a12. Juh\u00e1s, P., Granlund, L., Duxbury, P. M., Punch, W. F. & Billinge, S. J. L. The Liga\u000d\u000aalgorithm for ab initio determination of nanostructure. Acta Crystallogr A 64,\u000d\u000a631\u2013640 (2008).\u000d\u000a13. Christiansen, T. L., B\u00f8jesen, E. D., Juelsholt, M., Etheridge, J. & Jensen, K. M. \u00d8. Size\u000d\u000ainduced structural changes in molybdenum oxide nanoparticles. ACS Nano 13,\u000d\u000a8725\u20138735 (2019).\u000d\u000a14. Aalling-Frederiksen, O., Juelsholt, M., Anker, A. S. & Jensen, K. M. \u00d8. Formation and\u000d\u000agrowth mechanism for niobium oxide nanoparticles: atomistic insight from\u000d\u000ain situ X-ray total scattering. Nanoscale 13, 8087\u20138097 (2021).\u000d\u000a15. Yang, L., Juh\u00e1s, P., Terban, M. W., Tucker, M. G. & Billinge, S. J. L. Structuremining: screening structure models by automated fitting to the atomic pair\u000d\u000adistribution function over large numbers of models. Acta Crystallogr. A 76,\u000d\u000a395\u2013409 (2020).\u000d\u000a16. Anker, A. S. et al. Characterising the Atomic Structure of Mono-Metallic Nanoparticles from X-Ray Scattering Data Using Conditional Generative Models. In\u000d\u000aProc. 16th International Workshop on Mining and Learning with Graphs (MLG).\u000d\u000a(Association for Computing Machinery, New York, NY, 2020) https://\u000d\u000awww.mlgworkshop.org/2020/.\u000d\u000a17. Banerjee, S. et al. Cluster-mining: an approach for determining core structures of\u000d\u000ametallic nanoparticles from atomic pair distribution function data. Acta Crystallogr. A 76, 24\u201331 (2020).\u000d\u000a18. Anker, A. S. et al. Structural changes during the growth of atomically precise\u000d\u000ametal oxido nanoclusters from combined pair distribution function and smallangle X-ray scattering analysis. Angew. Chem. Int. Ed. 60, 2\u201312 (2021).\u000d\u000a19. Butler, K. T., Le, M. D., Thiyagalingam, J. & Perring, T. G. Interpretable, calibrated\u000d\u000aneural networks for analysis and understanding of inelastic neutron scattering\u000d\u000adata. J. Phys. Condens. Matter 33, 194006 (2021).\u000d\u000a20. Suzuki, Y. et al. Symmetry prediction and knowledge discovery from X-ray diffraction patterns using an interpretable machine learning approach. Sci. Rep. 10,\u000d\u000a21790 (2020).\u000d\u000a21. Torrisi, S. B. et al. Random forest machine learning models for interpretable X-ray\u000d\u000aabsorption near-edge structure spectrum-property relationships. npj Comput.\u000d\u000aMater. 6, 109 (2020).\u000d\u000a22. Oviedo, F., Ferres, J. L., Buonassisi, T. & Butler, K. T. Interpretable and Explainable\u000d\u000aMachine Learning for Materials Science and Chemistry. Acc. Mater. Res., 3, 6,\u000d\u000a597\u2013607 (2022)\u000d\u000a23. Schmidt, J., Marques, M. R. G., Botti, S. & Marques, M. A. L. Recent advances and\u000d\u000aapplications of machine learning in solid-state materials science. npj Comput.\u000d\u000aMater. 5, 83 (2019).\u000d\u000a24. Lee, K., Ayyasamy, M. V., Delsa, P., Hartnett, T. Q. & Balachandran, P. V. Phase\u000d\u000aclassification of multi-principal element alloys via interpretable machine learning.\u000d\u000anpj Comput. Mater. 8, 25 (2022).\u000d\u000a25. Chen, T. & Guestrin, C. XGBoost: A Scalable Tree Boosting System in Proc. 22nd\u000d\u000aACM SIGKDD International Conference on Knowledge Discovery and Data Mining\u000d\u000a785\u2013794 (Association for Computing Machinery, San Francisco, 2016).\u000d\u000a26. Lundberg, S. M. et al. From local explanations to global understanding with\u000d\u000aexplainable AI for trees. Nat. Mach. Intell. 2, 56\u201367 (2020).\u000d\u000a27. Lundberg, S. M. & Lee, S.-I. A Unified Approach to Interpreting Model Predictions.\u000d\u000ain Proc. 31st International Conference on Neural Information Processing Systems,\u000d\u000a4765\u20134774 (Curran Associates, Inc., 2017).\u000d\u000a28. Juelsholt, M., Lindahl Christiansen, T. & Jensen, K. M. \u00d8. Mechanisms for tungsten\u000d\u000aoxide nanoparticle formation in solvothermal synthesis: from polyoxometalates\u000d\u000ato crystalline materials. J. Phys. Chem. C. 123, 5110\u20135119 (2019).\u000d\u000a29. Chen, X. & Yamanaka, S. Single-crystal X-ray structural refinement of the \u2018tetragonal\u2019 C60 polymer. Chem. Phys. Lett. 360, 501\u2013508 (2002).\u000d\u000a30. Juhas, P., Farrow, C. L., Yang, X., Knox, K. R. & Billinge, S. J. L. Complex modeling: a\u000d\u000astrategy and software program for combining multiple information sources to\u000d\u000asolve ill posed structure and nanostructure inverse problems. Acta Crystallogr. A\u000d\u000a71, 562\u2013568 (2015).\u000d\u000a31. Krebs, B. & Paulat-B\u00f6schen, I. The structure of the potassium isopolymolybdate\u000d\u000aK8[Mo36O12(H2O)16]\u00b7nH2O (n = 36\u22ef40). Acta Crystallogr. B 38, 1710\u20131718 (1982).\u000d\u000a32. Niu, J., Zhao, J., Wang, J. & Bo, Y. Syntheses, spectroscopic characterization,\u000d\u000athermal behavior, electrochemistry and crystal structures of two novel pyridine\u000d\u000ametatungstates. J. Coord. Chem. 57, 935\u2013946 (2004).\u000d\u000a33. Joachim, F., Axel, T. & Rosemarie, P. Strukturen und Schwingungsspektren des\u000d\u000aTetramethylammonium-\u03b1-dodekawolframatosilikats und des Tetrabutylammonium-\u03b2-dodekawolframatosilikats: Structures and Vibrational Spectra\u000d\u000aof Tetramethylammonium \u03b1-Dodecatungstosilicate and Tetrabutylammonium\u000d\u000a\u03b2-Dodecatungstosilicate. Z. Naturforsch. B 36, 161\u2013171 (1981).\u000d\u000a34. Niu, J.-Y., Han, Q.-X. & Wang, J.-P. A novel Keggin units-supported complex: synthesis,\u000d\u000acharacterization and crystal structure of [(CH3)2NH2]6[Cu(DMF)4(GeW12O40)2]\u00b72DMF. J.\u000d\u000aCoord. Chem. 56, 523\u2013530 (2003).\u000d\u000a35. Busbongthong, S. & Ozeki, T. Structural relationships among methyl-, dimethyl-,\u000d\u000aand trimethylammonium phosphododecatungstates. Bull. Chem. Soc. Jpn. 82,\u000d\u000a1393\u20131397 (2009).\u000d\u000a36. Yang, L. et al. A cloud platform for atomic pair distribution function analysis:\u000d\u000aPDFitc. Acta Crystallogr. A 77, 2\u20136 (2021).\u000d\u000a37. Skj\u00e6rv\u00f8, S. L. et al. Atomic structural changes in the formation of transition metal\u000d\u000atungstates: the role of polyoxometalate structures in material crystallization.\u000d\u000aPreprint at https://chemrxiv.org/engage/chemrxiv/article-details/62ebefdcd1\u000d\u000a31b71fc70c4ef2 (2022).\u000d\u000aA.S. Anker et al.\u000d\u000a10\u000d\u000anpj Computational Materials (2022)  213  Published in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences","38. Magnard, N. P. L., Anker, A. S., Aalling-Frederiksen, O., Kirsch, A. & Jensen, K. M. \u00d8.\u000d\u000aCharacterisation of intergrowth in metal oxide materials using structure-mining:\u000d\u000athe case of \u03b3-MnO2. Dalton Trans., https://doi.org/10.1039/D2DT02153F (2022).\u000d\u000a39. Proffen, T. & Neder, R. B. DISCUS, a program for diffuse scattering and defect\u000d\u000astructure simulations \u2013 update. J. Appl. Crystallogr. 32, 838\u2013839 (1999).\u000d\u000a40. Proffen, T. & Neder, R. B. DISCUS: a program for diffuse scattering and defectstructure simulation. J. Appl. Cryst. 30, 171\u2013175 (1997).\u000d\u000a41. Coelho, A. A. TOPAS and TOPAS-Academic: an optimization program integrating\u000d\u000acomputer algebra and crystallographic objects written in C++. J. Appl. Crystallogr. 51, 210\u2013218 (2018).\u000d\u000a42. Nogueira, F. Bayesian Optimization: Open source constrained global optimization\u000d\u000atool for Python. https://github.com/fmfn/BayesianOptimization (2014).\u000d\u000a43. Putatunda, S. & Rama, K. A Comparative Analysis of Hyperopt as Against Other\u000d\u000aApproaches for Hyper-Parameter Optimization of XGBoost. In Proc. 2018 International Conference on Signal Processing and Machine Learning, 6\u201310 (Association\u000d\u000afor Computing Machinery, New York, 2018).\u000d\u000a44. Momma, K. & Izumi, F. VESTA 3 for three-dimensional visualization of crystal,\u000d\u000avolumetric and morphology data. J. Appl. Crystallogr. 44, 1272\u20131276 (2011).\u000d\u000a45. Palmer, D. C. Visualization and analysis of crystal structures using CrystalMaker\u000d\u000asoftware. Z. Kristallogr. Cryst. Mater. 230, 559\u2013572 (2015).\u000d\u000aACKNOWLEDGEMENTS\u000d\u000aThis work is part of a project that has received funding from the European Research\u000d\u000aCouncil (ERC) under the European Union\u2019s Horizon 2020 Research and Innovation\u000d\u000aProgramme (grant agreement No. 804066). We are grateful to the Villum Foundation for\u000d\u000afinancial support through a Villum Young Investigator grant (VKR00015416). Funding from\u000d\u000athe Danish Ministry of Higher Education and Science through the SMART Lighthouse is\u000d\u000agratefully acknowledged. We furthermore thank DANSCATT (supported by the Danish\u000d\u000aAgency for Science and Higher Education) for support. We acknowledge DESY (Hamburg,\u000d\u000aGermany), a member of the Helmholtz Association HGF, for the provision of experimental\u000d\u000afacilities. Parts of this research were carried out at beamline P02.1 and P07 at Petra III and\u000d\u000awe would like to thank Martin Etter, Jozef Bednarcik, and Ann-Christin Dippel for\u000d\u000aassistance in using the beamlines. This research used resources of the Advanced Photon\u000d\u000aSource, a US Department of Energy (DOE) Office of Science User Facility operated for the\u000d\u000aDOE Office of Science by Argonne National Laboratory under contract No. DE-AC02-\u000d\u000a06CH11357. We acknowledge MAX IV Laboratory for time on Beamline DanMAX under\u000d\u000aProposal 20200731. Research conducted at MAX IV is supported by the Swedish Research\u000d\u000acouncil under contract 2018-07152, the Swedish Governmental Agency for Innovation\u000d\u000aSystems under contract 2018-04969, and Formas under contract 2019-02496. DanMAX is\u000d\u000afunded by the NUFI grant no. 4059-00009B. S.J.L.B. was supported by the U.S. National\u000d\u000aScience Foundation through grant DMREF-1922234.\u000d\u000aAUTHOR CONTRIBUTIONS\u000d\u000aA.S.A., E.T.S.K., and K.M.\u00d8.J. conceptualized the project. A.S.A., E.T.S.K., M.J., T.L.C.,\u000d\u000aS.L.S., S.J.L.B., and R.S. designed the methodology and A.S.A. and E.T.S.K. wrote\u000d\u000athe code. A.S.A., M.J., T.L.C., M.R.V.J., I.K., and D.R.S. collected the data. K.M.\u00d8.J.\u000d\u000aprocured funding and supervised the project. All authors were involved with the\u000d\u000awriting of the paper.\u000d\u000aCOMPETING INTERESTS\u000d\u000aThe authors declare no competing interests.\u000d\u000aADDITIONAL INFORMATION\u000d\u000aSupplementary information The online version contains supplementary material\u000d\u000aavailable at https://doi.org/10.1038/s41524-022-00896-3.\u000d\u000aCorrespondence and requests for materials should be addressed to Kirsten M. \u00d8.\u000d\u000aJensen.\u000d\u000aReprints and permission information is available at http://www.nature.com/\u000d\u000areprints\u000d\u000aPublisher\u2019s note Springer Nature remains neutral with regard to jurisdictional claims\u000d\u000ain published maps and institutional affiliations.\u000d\u000aOpen Access This article is licensed under a Creative Commons\u000d\u000aAttribution 4.0 International License, which permits use, sharing,\u000d\u000aadaptation, distribution and reproduction in any medium or format, as long as you give\u000d\u000aappropriate credit to the original author(s) and the source, provide a link to the Creative\u000d\u000aCommons license, and indicate if changes were made. The images or other third party\u000d\u000amaterial in this article are included in the article\u2019s Creative Commons license, unless\u000d\u000aindicated otherwise in a credit line to the material. If material is not included in the\u000d\u000aarticle\u2019s Creative Commons license and your intended use is not permitted by statutory\u000d\u000aregulation or exceeds the permitted use, you will need to obtain permission directly\u000d\u000afrom the copyright holder. To view a copy of this license, visit http://\u000d\u000acreativecommons.org/licenses/by/4.0/.\u000d\u000a\u00a9 The Author(s) 2022\u000d\u000aA.S. Anker et al.\u000d\u000a11\u000d\u000aPublished in partnership with the Shanghai Institute of Ceramics of the Chinese Academy of Sciences npj Computational Materials (2022)  213 "," \u000d\u000a101\u000d\u000a3.2.2 Paper\u0009VI\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009Paper\u0009VI\u0009which\u0009is\u0009in\u0009preparation.\u0009The\u0009supplementary\u0009information\u0009\u000d\u000afor\u0009Paper\u0009VI\u0009can\u0009be\u0009found\u0009in\u0009APPENDICES\u0009\u2013\u0009PAPERS.",""," \u000d\u000a125\u000d\u000a4 Generative Machine\u0009 Learning\u0009 for\u0009 Experimental\u0009 Scattering\u0009 Data\u0009\u000d\u000aAnalysis*\u000d\u000aSupervised\u0009 ML\u0009 is\u0009 limited\u0009 by\u0009 its\u0009 reliance\u0009 on\u0009 paired\u0009 input\u0009 data\u0009 and\u0009 labels\u0009 for\u0009 training,\u0009 which\u0009 can\u0009 be\u0009\u000d\u000achallenging\u0009 to\u0009 obtain\u0009 for\u0009experimental\u0009 data\u0009analysis.\u0009I\u0009observe\u0009 three\u0009primary issues\u0009when\u0009analysing\u0009\u000d\u000aexperimental scattering\u0009data with\u0009supervised\u0009ML.\u0009These\u0009three\u0009issues\u0009are illustrated\u0009in Figure\u00094.1.\u0009Each\u0009\u000d\u000aof them\u0009also\u0009applies to\u0009other\u0009structure\u0009characterisation\u0009datasets where\u0009structure\u0009refinement\u0009is\u0009feasible.\u0009\u000d\u000aThe\u0009first\u0009issue\u0009arises\u0009when\u0009datasets\u0009are\u0009obtained\u0009from a\u0009sample\u0009containing\u0009multiple chemical\u0009structures,\u0009\u000d\u000acausing\u0009 supervised\u0009ML\u0009models\u0009 trained\u0009 on\u0009 single-phase\u0009 systems\u0009 to\u0009 fail.\u0009Training\u0009 on\u0009data\u0009that\u0009 feature\u000d\u000amultiple\u0009chemical\u0009structures\u0009requires considerable computational\u0009resources.\u0009This issue\u0009is\u0009addressed\u0009in\u0009\u000d\u000athe\u0009CIF-Finder paper in APPENDICES\u0009\u2013\u0009PAPERS but will\u0009not\u0009be further discussed here.\u0009Issue\u0009two\u0009occurs\u0009\u000d\u000awhen\u0009the\u0009supervised\u0009ML\u0009model\u0009is\u0009trained\u0009on\u0009a\u0009structure\u0009database\u0009which\u0009does\u0009not\u0009include\u0009the\u0009chemical\u0009\u000d\u000asystem\u0009from which\u0009 the\u0009 data\u0009 is\u0009 obtained. Here,\u0009 the\u0009 ML\u0009 model\u0009tends to\u0009 either\u0009 predict\u0009 a\u0009 non-sensical\u000d\u000astructure\u0009or\u0009to\u0009predict\u0009the\u0009structurally\u0009closest entry from\u0009the\u0009database to\u0009the\u0009dataset.\u0009This\u0009structure\u0009is\u0009\u000d\u000anot\u0009 the\u0009correct\u0009 structure\u0009but\u0009 rather\u0009 the\u0009least\u0009wrong\u0009in\u0009the\u0009context\u0009 of\u0009 the\u0009 training\u0009data.\u0009This\u0009issue\u0009is\u0009\u000d\u000aaddressed\u0009in\u0009section\u00094.1.\u0009The\u0009last\u0009issue\u0009relates\u0009 to\u0009experimental\u0009data\u0009only.\u0009 In\u0009materials\u0009chemistry,\u0009we\u0009\u000d\u000aoften\u0009train\u0009ML\u0009models\u0009on\u0009large\u0009datasets\u0009of\u0009physics-based\u0009simulations, and\u0009estimate\u0009their\u0009accuracy\u0009using\u0009\u000d\u000aa\u0009test\u0009set\u0009of\u0009simulated\u0009data\u0009that\u0009was\u0009not\u0009used\u0009during\u0009the\u0009training\u0009procedure.\u0009While\u0009a\u0009test\u0009set\u0009of\u0009simulated\u0009\u000d\u000adata\u0009can\u0009be\u0009valuable\u0009to\u0009estimate the\u0009model's\u0009accuracy,\u0009it may\u0009fail\u0009to\u0009capture\u0009the\u0009complexity\u0009of\u0009real-world\u0009\u000d\u000aexperimental\u0009data\u0009accurately.\u0009Experimental\u0009data\u0009can\u0009contain\u0009background\u0009noise,\u0009instrumental\u0009effects,\u0009or\u0009\u000d\u000aother\u0009physical\u0009phenomena\u0009absent in\u0009the\u0009physics-based\u0009simulations.\u0009When\u0009we\u0009estimate\u0009the\u0009supervised\u0009\u000d\u000aML\u0009model's accuracy\u0009from\u0009simulated\u0009data that\u0009does\u0009not\u0009take\u0009these\u0009effects\u0009into\u0009account,\u0009we\u0009cannot\u0009always\u0009\u000d\u000atrust\u0009its\u0009ability\u0009to\u0009perform\u0009as\u0009well\u0009on\u0009experimental\u0009data. This\u0009issue\u0009is\u0009addressed\u0009in\u0009section\u00094.2.\u0009\u000d\u000aAll\u0009 three\u0009issues\u0009are\u0009 related\u0009 to\u0009 unsatisfactory or\u0009missing\u0009labels\u0009 of the data,\u0009making\u0009 unsupervised\u0009ML\u0009\u000d\u000amethods\u0009a\u0009more\u0009suitable\u0009alternative.\u0009Unsupervised\u0009ML\u0009models\u0009operate without\u0009relying\u0009on\u0009paired\u0009labels\u0009\u000d\u000aand\u0009 input\u0009 features,\u0009 instead\u0009 working\u0009 with\u0009 only\u0009 input\u0009 features\u0009 or\u0009 intermediate\u0009 input-derived\u0009 labels.\u000d\u000aUnsupervised\u0009ML\u0009excels at representing complex\u0009data\u0009in\u0009a\u0009low-dimensional\u0009space, facilitating\u0009tasks\u0009such\u0009\u000d\u000aas\u0009 dimensionality\u0009 reduction.\u0009 This enables the\u0009 analysis\u0009 of\u0009 high-dimensional\u0009 dataset\u0009 similarities,\u0009\u000d\u000aclustering analysis,\u0009data\u0009demixing to\u0009identify\u0009constituent\u0009parts,\u0009and\u0009extraction\u0009of\u0009underlying\u0009data\u0009trends\u0009\u000d\u000athat\u0009are\u0009difficult\u0009to\u0009comprehend\u0009in the original input\u0009representation\u0009space.\u0009However,\u0009the\u0009lack\u0009of\u0009labelled\u0009\u000d\u000adata\u0009often\u0009makes\u0009unsupervised\u0009ML more\u0009challenging\u0009to\u0009use\u0009than\u0009supervised\u0009ML,\u0009as\u0009it\u0009typically\u0009requires\u0009\u000d\u000amore\u0009 data,\u0009 or\u0009 yields\u0009 lower\u0009 accuracy. This\u0009 is\u0009 were\u0009 deep\u0009 generative\u0009 models,\u0009 such\u0009 as\u0009 Conditional\u0009\u000d\u000aVariational\u0009 Autoencoders\u0009 (CVAEs)\u0009 and\u0009 Generative\u0009 Adversarial\u0009 Networks\u0009 (GANs),\u0009 can\u0009 offer\u0009 a\u0009 hybrid\u0009\u000d\u000aapproach.\u0009 Deep\u0009 generative\u0009 models\u0009 are\u0009 classes of\u0009 ML\u0009 models\u0009 that\u0009 can\u0009 estimate\u0009 the\u0009 underlying\u0009 data\u0009\u000d\u000adistribution\u0009 from\u0009a\u0009reasonably\u0009small\u0009set\u0009of\u0009 training\u0009examples.\u0009Generative\u0009models\u0009are\u0009probabilistic\u0009in\u0009\u000d\u000acontrast\u0009to\u0009a\u0009deterministic ML\u0009model,\u0009meaning\u0009that they can\u0009potentially\u0009interpolate\u0009between\u0009structures\u0009\u000d\u000ain\u0009a\u0009given\u0009database\u0009and\u0009to\u0009a\u0009certain\u0009extent\u0009extrapolate\u0009beyond\u0009them.90,91\u000d\u000aIn\u0009 section\u00094.1,\u0009we\u0009will\u0009explore\u0009 how\u0009generative ML\u0009 can\u0009interpolate\u0009and\u0009extrapolate\u0009from\u0009a\u0009 structural\u0009\u000d\u000adatabase.\u0009 Then,\u0009 in\u0009 section\u0009 4.2 we\u0009 will\u0009 examine\u0009 how\u0009 it\u0009 can\u0009 be\u0009 used\u0009 to\u0009 learn\u0009 the\u0009 data\u0009 distribution\u0009 of\u0009\u000d\u000asimulated\u0009and\u0009experimental\u0009data\u0009and\u0009translate\u0009between\u0009them.\u0009While\u0009the\u0009methods\u0009address these\u0009issues\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009II.","\u0009 |\u0009Generative\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a126\u000d\u000afor\u0009PDF\u0009(section\u00094.1)\u0009and\u0009INS\u0009(section\u00094.2)\u0009data,\u0009the\u0009developed\u0009methodologies are intended to\u0009be general\u0009\u000d\u000afor\u0009scattering\u0009and\u0009spectroscopy\u0009data\u0009analysis.\u0009\u000d\u000aFigure\u00094.1:\u0009Three\u0009common\u0009issues\u0009when\u0009analysing\u0009experimental\u0009scattering\u0009data.\u0009Supervised\u0009ML\u0009models\u0009are\u0009trained\u0009\u000d\u000aon\u0009 pairs\u0009 of\u0009 structural\u0009 models\u0009 from\u0009 a\u0009 structural\u0009 database\u0009 and\u0009 physics-based\u0009 scattering\u0009 simulations.\u0009 However,\u0009\u000d\u000asupervised\u0009ML\u0009are\u0009challenged\u0009by;\u0009 Issue\u00091:\u0009The\u0009experimental\u0009data\u0009is\u0009obtained\u0009 from\u0009a\u0009system\u0009containing\u0009multiple\u0009\u000d\u000achemical\u0009species.\u0009Issue\u00092:\u0009The\u0009structural\u0009model\u0009is\u0009not\u0009included\u0009in\u0009the\u0009structure\u0009database\u0009used\u0009for\u0009training\u0009the\u0009ML\u0009\u000d\u000amodel.\u0009 Issue\u00093:\u0009The\u0009experimental\u0009data\u0009contains\u0009background\u0009noise,\u0009instrumental\u0009effects\u0009or\u0009other\u0009phenomena\u0009not\u0009\u000d\u000aencountered\u0009by\u0009the\u0009simulated\u0009data. Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u0009Paper\u0009II.46"," \u000d\u000a127\u000d\u000a4.1 A\u0009 Step\u0009 Towards a\u0009 General\u0009 Approach\u0009 for\u0009 Structure\u0009 Determination\u0009 of\u0009\u000d\u000aNanomaterials*\u0009\u0009\u000d\u000aSupervised\u0009classification\u0009ML\u0009is\u0009fundamentally\u0009tied\u0009to\u0009the\u0009availability\u0009of\u0009a\u0009suitable\u0009structure\u0009database\u0009or\u0009\u000d\u000aan\u0009algorithmic\u0009structure\u0009generator. Motif-extractor\u0009algorithms\u0009can stretch beyond\u0009this\u0009limitation\u0009by\u0009not\u0009\u000d\u000aonly\u0009investigating the\u0009database\u0009entries but\u0009also\u0009structural\u0009motifs\u0009within\u0009them.\u0009Still,\u0009they\u0009remain confined\u000d\u000ato\u0009chemical\u0009topologies\u0009present\u0009in\u0009the\u0009structural\u0009database.\u0009If\u0009the\u0009supervised\u0009ML\u0009models are\u0009provided\u0009data\u000d\u000afrom\u0009a\u0009structure\u0009absent\u0009 from the\u0009structural\u0009database,\u0009 they tend\u0009 to\u0009predict\u0009 the\u0009structure\u0009with\u0009closest\u0009\u000d\u000aproximity\u0009in\u0009the\u0009database\u0009or\u0009a\u0009non-sensical output\u0009as\u0009illustrated\u0009in\u0009Figure\u00094.1.\u0009So\u0009the\u0009question\u0009emerges:\u0009\u000d\u000ahow\u0009can\u0009we\u0009go\u0009beyond these\u0009constraints\u0009and\u0009analyse\u0009data\u0009from\u0009chemical\u0009structures\u0009not\u0009present\u0009in\u0009the\u0009\u000d\u000astructural\u0009database?\u0009\u0009\u000d\u000aIn\u0009 Papers\u0009 VII\u0009 and\u0009 VIII,\u0009 we\u0009 propose\u0009 to\u0009 use\u0009 a\u0009 graph-based\u0009 CVAE framework,\u0009 named\u0009 DeepStruc, for\u0009\u000d\u000astructure\u0009 solution from\u0009 PDF\u0009 data.92,93 While\u0009 traditional\u0009 representations\u0009 of\u0009 chemical\u0009 structures\u0009 often\u0009\u000d\u000ainvolve\u0009static,\u0009two-dimensional\u0009diagrams,\u0009graph-based\u0009approaches model these\u0009structures\u0009as\u0009dynamic,\u0009\u000d\u000ainterconnected\u0009networks\u0009of\u0009nodes\u0009(often\u0009atoms)\u0009and\u0009edges\u0009(often\u0009bonds),\u0009thereby\u0009capturing\u0009spatial\u0009and\u0009\u000d\u000arelational information about\u0009 the\u0009 structures.\u0009Graph\u0009 representations, which\u0009 remain unchanged\u0009 under\u0009\u000d\u000astructural\u0009 transformations\u0009 like\u0009 solid\u0009 translations\u0009 and\u0009 rotations (and\u0009 if\u0009 needed,\u0009 reflections),\u0009 have\u0009\u000d\u000atherefore\u0009seen\u0009increasing\u0009success\u0009in\u0009ML\u0009applications\u0009related\u0009to\u0009materials\u0009chemistry.94-96\u000d\u000aPaper\u0009VII\u0009introduces\u0009DeepStruc\u00090,92 which\u0009can\u0009solve\u0009a\u0009mono-metallic\u0009nanoparticle\u0009(MMNP) with\u0009facecentered\u0009cubic\u0009(fcc)\u0009structure from\u0009PDF\u0009data\u0009in\u0009just\u0009about\u000910\u0009seconds. DeepStruc\u00090\u0009is\u0009a\u0009combination\u0009of\u0009a\u0009\u000d\u000agraph-based\u0009CVAE\u0009and\u0009a\u0009trilateration\u0009algorithm and\u0009it\u0009is\u0009trained\u0009on MMNPs with\u0009fcc structures, each\u000d\u000acontaining\u0009up\u0009to\u0009100\u0009atoms,\u0009and\u0009their\u0009corresponding\u0009simulated\u0009X-ray\u0009PDF.\u0009It offers\u0009a\u0009unique\u0009structure\u0009\u000d\u000arepresentation,\u0009by mapping them\u0009as\u0009graphs\u0009within\u0009a\u0009unit\u0009sphere,\u0009ensuring\u0009consistent\u0009relative\u0009distance\u0009\u000d\u000acalculations\u0009 between\u0009 atoms across\u0009 varying\u0009 structures.\u0009 Drawing\u0009 inspiration\u0009 from\u0009 trilateration\u000d\u000atechniques\u0009used\u0009in\u0009satellite-based\u0009geolocation,97 DeepStruc\u00090 places multiple\u0009satellites\u0009onto\u0009the\u0009surface\u0009\u000d\u000aof\u0009the\u0009unit\u0009sphere by\u0009predicting the\u0009distance\u0009of\u0009each\u0009atom\u0009to\u0009each\u0009satellite,\u0009whereafter\u0009trilateration\u0009is\u0009\u000d\u000aused\u0009 to\u0009 calculate\u0009 the\u0009 xyz-coordinates\u0009 of\u0009 the\u0009 atoms.\u0009 In\u0009 Figure\u0009 4.2,\u0009 we\u0009 demonstrate\u0009 that\u0009 while\u0009\u000d\u000aDeterministic\u0009 Autoencoder\u0009 (DAE)\u0009 models\u0009 can\u0009 also\u0009 produce\u0009 valid\u0009 structures\u0009 derived\u0009 from\u0009 PDFs,\u0009 the\u0009\u000d\u000aCVAE\u0009model\u0009exhibits\u0009superior\u0009interpolation\u0009abilities.\u0009This\u0009provides opportunities\u0009for\u0009generating\u0009distinct\u0009\u000d\u000astructural\u0009motifs\u0009from\u0009the\u0009structure\u0009database.\u0009The\u0009interpolated\u0009structures\u0009from\u0009the\u0009DAE\u0009models\u0009are\u0009nonsensical,\u0009while\u0009those\u0009derived\u0009from\u0009the\u0009CVAE\u0009more\u0009closely\u0009approximate\u0009real\u0009structures.\u0009\u000d\u000aFigure\u00094.2:\u0009Structures\u0009generated\u0009 by\u0009 decoding\u0009 different\u0009 extents\u0009 of\u0009interpolation\u0009between PDF-A\u0009 and\u0009 PDF-B.\u0009 The\u0009\u000d\u000agenerated\u0009 structures\u0009 start\u0009 from\u0009Structure-A\u0009 and\u0009 progressively\u0009 evolve\u0009 towards\u0009Structure-B.\u0009Here we\u0009 compare\u0009 a\u0009\u000d\u000aCVAE with\u0009a\u0009DAE. Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u0009Paper\u0009VII.92\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009VII+VIII.","\u0009 |\u0009Generative\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a128\u000d\u000aIn\u0009Paper\u0009VIII,\u0009we\u0009introduced several improvements\u0009to\u0009the\u0009DeepStruc\u00090\u0009model.\u000d\u000a93 The\u0009most\u0009substantial\u0009\u000d\u000achange\u0009was\u0009the\u0009direct\u0009prediction\u0009of\u0009xyz-coordinates,\u0009bypassing\u0009the\u0009time-consuming\u0009trilateration\u0009step,\u0009\u000d\u000aand\u0009allowing\u0009for\u0009instantaneous\u0009predictions\u0009(<1\u0009second).\u0009Additionally,\u0009we\u0009expanded\u0009the\u0009training\u0009set\u0009to\u0009\u000d\u000ainclude\u0009seven different\u0009MMNP\u0009structure\u0009types,\u0009containing\u0009up\u0009to\u0009200\u0009atoms. DeepStruc\u00091\u0009is\u0009illustrated\u0009in\u0009\u000d\u000aFigure\u00094.3.\u0009During\u0009the\u0009training\u0009phase\u0009(blue and green\u0009region),\u0009the encoder\u0009takes\u0009both the\u0009structure\u0009and\u0009\u000d\u000aits\u0009PDF\u0009as\u0009input,\u0009while\u0009 the\u0009 prior\u0009 only\u0009 takes\u0009 the\u0009PDF as\u0009input.\u0009In\u0009 that\u0009 sense\u0009DeepStruc\u0009is\u0009 supervised.\u0009\u000d\u000aHowever,\u0009like\u0009unsupervised\u0009ML\u0009models,\u0009both\u0009the\u0009encoder\u0009and\u0009prior\u0009maps the\u0009input\u0009into\u0009a\u00092-dimensional\u0009\u000d\u000arepresentation which\u0009I\u0009refer\u0009to\u0009as\u0009the latent\u0009space. To\u0009generate the\u0009structural\u0009output,\u0009a\u0009distribution\u0009from\u0009\u000d\u000athe\u0009latent\u0009space\u0009serves as\u0009input\u0009to\u0009the\u0009decoder\u0009which\u0009predicts the\u0009associated MMNP xyz-coordinates by\u0009\u000d\u000amapping\u0009 the\u0009 2-dimensional\u0009 latent\u0009 representation\u0009 back\u0009 into\u0009 coordinate\u0009 space.\u0009 After\u0009 training (green\u0009\u000d\u000aregion),\u0009DeepStruc\u00091\u0009solves the\u0009structures\u0009from\u0009PDFs\u0009alone.\u0009Using simulated\u0009PDFs,\u0009it\u0009does\u0009so\u0009with\u0009a mean\u0009\u000d\u000aabsolute\u0009error\u0009(MAE)\u0009of\u0009the\u0009atom\u0009positions\u0009of 0.128\u0009\u00b1\u00090.073\u0009\u00c5 on\u0009the\u0009simulated\u0009test\u0009set.\u0009By refining\u0009the\u0009\u000d\u000astructure\u0009 to\u0009 the\u0009 PDF,\u000d\u000a13 through\u0009 the fitting\u0009 of\u0009 a\u0009 contraction/expansion\u0009 factor,\u0009 a\u0009 scale\u0009 factor,\u0009 and\u0009 an\u0009\u000d\u000aisotropic\u0009atomic\u0009displacement\u0009parameter,\u0009as\u0009described\u0009in\u0009Appendix\u0009IV,\u0009the\u0009MAE\u0009of\u0009the\u0009atom\u0009positions\u0009is\u0009\u000d\u000areduced\u0009to\u00090.093\u0009\u00b1 0.058\u0009\u00c5 on\u0009the\u0009same\u0009test\u0009set.\u0009DeepStruc\u00091\u0009was\u0009also\u0009used\u0009to\u0009determine\u0009the\u0009structure\u0009\u000d\u000aof\u0009 three\u0009 experimental\u0009 PDFs,\u0009 which\u0009 have previously been well-characterised.98,99 Here\u0009 DeepStruc\u0009 1\u0009\u000d\u000aobtained\u0009similar\u0009results\u0009to\u0009the\u0009literature. Our\u0009findings\u0009demonstrate that\u0009DeepStruc\u00091\u2019s\u0009constructed\u0009latent\u0009\u000d\u000aspace,\u0009shown\u0009in\u0009Figure\u00094.3,\u0009significantly\u0009incorporates\u0009chemical\u0009knowledge.\u0009It\u0009arranges\u0009structures\u0009based\u0009\u000d\u000aon both\u0009size (illustrated\u0009with\u0009the\u0009size\u0009of\u0009the\u0009points\u0009in\u0009Figure\u00094.3)\u0009and\u0009symmetry.\u0009Here,\u0009the\u0009colouring\u0009is\u0009\u000d\u000aas\u0009 follows:\u0009fcc (light\u0009blue),\u0009octahedral\u0009(dark\u0009grey),\u0009decahedral\u0009(orange),\u0009body-centered\u0009cubic\u0009(green),\u0009\u000d\u000aicosahedral\u0009 (dark\u0009 blue),\u0009 hexagonal closed\u0009 packed\u0009 (hcp)\u0009 (pink),\u0009 and\u0009 simple\u0009 cubic\u0009 (red).\u0009 Cubic\u0009\u000d\u000asymmetrical\u0009structures are\u0009closely\u0009positioned in\u0009the\u0009latent\u0009space,\u0009and\u0009similarly,\u0009octahedral\u0009structures\u0009\u000d\u000aare\u0009 found\u0009 adjacent to\u0009 fcc structures.\u0009 Moreover,\u0009 it\u0009 places decahedral\u0009 structures in the latent\u0009 space\u000d\u000abetween\u0009 fcc and\u0009 hcp structures.\u0009 This\u0009 spatial\u0009 arrangement\u0009 can\u0009 be\u0009 explained by\u0009 considering\u0009 that\u0009\u000d\u000adecahedral\u0009structures\u0009are\u0009constructed\u0009from\u0009five\u0009tetrahedrally\u0009shaped\u0009fcc crystals,\u0009separated\u0009by\u0009{111}\u0009\u000d\u000atwin\u0009boundaries.48,100,101 The\u0009twin\u0009boundaries,\u0009resembling stacking\u0009 faulted\u0009regions\u0009of\u0009fcc,\u0009justify their\u0009\u000d\u000alocation in\u0009the\u0009latent\u0009space\u0009between\u0009fcc and\u0009hcp.\u000d\u000a48,\u000995,\u000996\u0009The\u0009capability\u0009of\u0009DeepStruc\u00091\u0009to\u0009interpolate arises\u0009\u000d\u000afrom\u0009each\u0009structure\u0009in\u0009the\u0009latent\u0009space\u0009being\u0009probabilistically\u0009rather\u0009than\u0009deterministically\u0009predicted,\u0009\u000d\u000awhich\u0009was\u0009demonstrated\u0009in\u0009Paper\u0009VII.\u000d\u000aIn\u0009our\u0009exploration\u0009of\u0009DeepStruc\u00091's\u0009extrapolative\u0009abilities,\u0009we\u0009applied\u0009DeepStruc\u00091\u0009(trained\u0009exclusively\u0009\u000d\u000aon\u0009PDFs\u0009from\u0009MMNPs\u0009with\u0009up\u0009to\u0009200\u0009atoms)\u0009on\u0009simulated\u0009PDFs\u0009of\u0009MMNPs\u0009containing\u0009up\u0009to\u00091000\u0009atoms.\u0009\u000d\u000aBoth\u0009size\u0009and\u0009symmetry trends\u0009continue\u0009beyond\u0009the\u0009training\u0009data to\u0009structures\u0009containing\u0009about\u0009400\u0009\u000d\u000aatoms.\u0009Beyond\u0009 400\u0009atoms,\u0009all\u0009 structure\u0009 types\u0009 collapse\u0009 onto\u0009a\u0009line in\u0009 the\u0009 2-dimensional\u0009latent\u0009 space.\u0009\u000d\u000aHowever,\u0009 DeepStruc\u00091\u0009 continues\u0009 to provide\u0009 a\u0009 size\u0009 estimate\u0009 of\u0009 the\u0009 structure beyond\u0009 this\u0009 point.\u0009 This\u0009\u000d\u000aexperiment\u0009highlights\u0009 the\u0009capability\u0009of\u0009DeepStruc\u00091\u0009 to\u0009perform\u0009extrapolations\u0009in\u0009 the\u0009latent\u0009space.\u0009As\u0009\u000d\u000asuch,\u0009it\u0009can\u0009provide\u0009valuable\u0009insights\u0009about PDFs\u0009from\u0009structures\u0009that\u0009are\u0009not\u0009present\u0009in\u0009the\u0009training\u0009set.\u0009\u000d\u000aIn\u0009contrast,\u0009a\u0009supervised\u0009classification ML model,\u0009bound by a\u0009predefined\u0009structural\u0009database,\u0009cannot\u0009\u000d\u000aextrapolate (neither\u0009can\u0009it\u0009interpolate).\u0009\u000d\u000aDeepStruc\u0009 is\u0009 a\u0009 step towards\u0009 a\u0009 general\u0009 approach\u0009 for\u0009 structure\u0009 solution\u0009 of\u0009 nanomaterials.\u0009 Yet, its\u0009\u000d\u000aapplication\u0009remains\u0009confined to\u0009data\u0009from MMNPs, containing\u0009up\u0009to\u0009200\u0009atoms and\u0009it\u0009is\u0009assumed\u0009that\u0009\u000d\u000athe\u0009chemical\u0009composition\u0009is\u0009known.\u0009For\u0009a\u0009truly\u0009general\u0009approach,\u0009DeepStruc must predict\u0009atom\u0009labels\u000d\u000aand\u0009the\u0009chemical\u0009space of\u0009the\u0009training\u0009data\u0009must\u0009be\u0009massively\u0009expanded by\u0009including more\u0009structure\u0009\u000d\u000atypes\u0009and\u0009accommodate\u0009even larger\u0009structures.\u0009"," \u000d\u000a129\u000d\u000aFigure\u00094.3:\u0009We\u0009present\u0009a\u0009deep\u0009learning\u0009algorithm,\u0009DeepStruc,\u0009that\u0009can\u0009solve\u0009a\u0009simple\u0009nanoparticle\u0009structure\u0009directly\u0009\u000d\u000afrom\u0009a\u0009PDF\u0009 obtained\u0009 from\u0009 total\u0009 scattering\u0009 data\u0009 by\u0009 using\u0009a\u0009CVAE.\u0009DeepStruc\u0009 predicts\u0009 the\u0009xyz-coordinates\u0009 of\u0009 the\u0009\u000d\u000aMMNP\u0009structure\u0009with\u0009conditional\u0009input\u0009provided\u0009in\u0009the\u0009form\u0009of\u0009a\u0009PDF.\u0009The\u0009encoder\u0009uses\u0009the\u0009structure\u0009and\u0009its\u0009PDF\u0009\u000d\u000aas\u0009input\u0009while\u0009the\u0009prior\u0009only\u0009takes\u0009the\u0009PDF\u0009as\u0009input.\u0009To\u0009obtain\u0009the\u0009structural\u0009output\u0009a\u0009latent\u0009space\u0009embedding\u0009is\u0009\u000d\u000agiven\u0009 as\u0009 input\u0009 to\u0009 the\u0009 decoder\u0009 which\u0009 produces\u0009 the\u0009 corresponding\u0009 MMNP xyz-coordinates.\u0009 During\u0009 training\u0009 of\u0009\u000d\u000aDeepStruc\u0009both\u0009the\u0009blue\u0009and\u0009green\u0009regions\u0009are\u0009used,\u0009while\u0009only\u0009the\u0009green\u0009region\u0009is\u0009used\u0009for\u0009structure\u0009prediction\u0009\u000d\u000aduring\u0009the\u0009inference\u0009process. Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009from\u0009Paper\u0009VIII.93\u000d\u000a4.1.1 Paper\u0009VII\u000d\u000aOn\u0009 the\u0009 following\u0009 pages,\u0009 you\u0009 will\u0009 find\u0009 Paper\u0009 VII\u0009 which\u0009 is\u0009 published\u0009 in\u0009 SIGKDD,\u0009 16th\u0009 International\u0009\u000d\u000aWorkshop\u0009on\u0009Mining\u0009and\u0009Learning\u0009with\u0009Graphs.\u0009","Characterising the atomic structure of mono-metallic\u000d\u000ananoparticles from x-ray sca!ering data using conditional\u000d\u000agenerative models\u000d\u000aAndy S. Anker\u2217\u000d\u000aandy@chem.ku.dk\u000d\u000aDepartment of Chemistry,\u000d\u000aUniversity of Copenhagen\u000d\u000aCopenhagen, Denmark\u000d\u000aEmil T. S. Kj\u00e6r\u2217\u000d\u000aetsk@chem.ku.dk\u000d\u000aDepartment of Chemistry,\u000d\u000aUniversity of Copenhagen\u000d\u000aCopenhagen, Denmark\u000d\u000aErik B. Dam\u000d\u000aerikdam@di.ku.dk\u000d\u000aDepartment of Computer Science,\u000d\u000aUniversity of Copenhagen\u000d\u000aCopenhagen, Denmark\u000d\u000aSimon J. L. Billinge\u000d\u000aDepartment of Applied Physics and\u000d\u000aApplied Mathematics, Columbia\u000d\u000aUniversity\u000d\u000aCondensed Matter Physics and\u000d\u000aMaterials Science Department,\u000d\u000aBrookhaven National Laboratory\u000d\u000aNew York, NY, USA\u000d\u000aKirsten M. \u00d8. Jensen\u000d\u000akirsten@chem.ku.dk\u000d\u000aDepartment of Chemistry,\u000d\u000aUniversity of Copenhagen\u000d\u000aCopenhagen, Denmark\u000d\u000aRaghavendra Selvan\u000d\u000araghav@di.ku.dk\u000d\u000aDepartment of Computer Science,\u000d\u000aUniversity of Copenhagen\u000d\u000aCopenhagen, Denmark\u000d\u000aFigure 1: Structures generated by decoding di!erent extents of interpolation of the latent variables obtained for PDF-A and\u000d\u000aPDF-B. The generated structures start from Structure-A and progressively evolve towards Structure-B. This work uses a Conditional Variational Autoencoder (CVAE) and we compare it with a Deterministic Autoencoder (DAE).\u000d\u000aABSTRACT\u000d\u000aThe development of new nanomaterials for energy technologies is\u000d\u000adependent on understanding the intricate relation between material\u000d\u000aproperties and atomic structure. It is, therefore, crucial to be able to\u000d\u000aroutinely characterise the atomic structure in nanomaterials, and a\u000d\u000apromising method for this task is Pair Distribution Function (PDF)\u000d\u000aanalysis. The PDF can be obtained through Fourier transformation of x-ray total scattering data, and represents a histogram of\u000d\u000aall interatomic distances in the sample. Going from the distance\u000d\u000ainformation in the PDF to a chemical structure is an unassigned\u000d\u000adistance geometry problem (uDGP), and solving this is often the\u000d\u000a\u2217\u000d\u000aBoth authors contributed equally to this research.\u000d\u000aPresented in 16th International Workshop on Mining and Learning with Graphs, Aug\u000d\u000a24, 2020, San Diego,CA\u000d\u000abottleneck in nanostructure analysis. In this work, we propose to\u000d\u000ause a Conditional Variational Autoencoder (CVAE) to automatically\u000d\u000asolve the uDGP to obtain valid chemical structures from PDFs. We\u000d\u000ause a simple model system of hypothetical mono-metallic nanoparticles containing up to 100 atoms in the face centered cubic (FCC)\u000d\u000astructure as a proof of concept. The model is trained to predict the\u000d\u000aassigned distance matrix (aDM) from a simulated PDF of the structure as the conditional input. We introduce a novel representation\u000d\u000aof structures by projecting them inside a unit sphere and adding\u000d\u000aadditional anchor points or satellites to help in the reconstruction\u000d\u000aof the chemical structure. The performance of the CVAE model is\u000d\u000acompared to a Deterministic Autoencoder (DAE) showing that both\u000d\u000amodels are able to solve the uDGP reasonably well. We further show\u000d\u000athat the CVAE learns a structured and meaningful latent embedding\u000d\u000aspace which can be used to predict new chemical structures.","MLG 2020, Aug 24, 2020, San Deigo, CA Anker and Kj\u00e6r, et al.\u000d\u000aCCS CONCEPTS\u000d\u000a\u2022 Computing methodologies \u2192 Learning latent representations; \u2022 Applied computing \u2192 Chemistry.\u000d\u000aKEYWORDS\u000d\u000agenerative modeling, mono-metallic nanoparticles, CVAE, Pair Distribution Function\u000d\u000a1 INTRODUCTION\u000d\u000aThe development of nanoscience over the last decades has given\u000d\u000acompletely new possibilities for material engineering and development. [30] Compared to their bulk counterparts, nanomaterials\u000d\u000acan, for example, show improved properties in energy technologies\u000d\u000asuch as catalysis, solar cells, and batteries. [38] In particular, many\u000d\u000anew properties arise in \u2019ultrasmall\u2019 nanoparticles, where the dimensions are smaller than 5 nm. The very large surface-to-volume\u000d\u000aratio in such materials can lead to drastic changes in the material\u000d\u000aperformance in e.g. catalysis, and a fundamental change in atomic\u000d\u000astructure may also take place when going to the nanoscale. [23, 27]\u000d\u000aIn order to engineer nanomaterials with targeted properties,\u000d\u000athe link between atomic structures and properties must be understood, and it is crucial to be able to routinely characterise the\u000d\u000aatomic structure in nanomaterials. [12, 39] However, such materials\u000d\u000achallenge many of the conventional methods for structure characterisation. Traditionally, material structure is characterised through\u000d\u000ax-ray and neutron di!raction techniques, applying crystallographic\u000d\u000amethods. [11] These di!raction techniques rely on the presence of\u000d\u000along-range, periodic atomic order in the samples. Nanomaterials\u000d\u000ado not possess this long range-atomic order, and crystallographic\u000d\u000amethods cannot directly be applied for characterisation of many\u000d\u000ananomaterials. [5, 6]\u000d\u000aA way of overcoming this \u2019nanostructure problem\u2019 is through the\u000d\u000ause of the Pair Distribution Function (PDF). [18] The PDF of a sample is obtained through Fourier transformation of total scattering\u000d\u000adata, which can be collected using e.g. large scale synchrotron facilities. The PDF represents a histogram of all interatomic distances\u000d\u000ain the sample, and is a plot of intensity versus r, i.e. interatomic\u000d\u000adistance. Peak positions correspond to atom-atom distances while\u000d\u000athe intensity of a peak depends on the number of pairs having that\u000d\u000ainteratomic distance and the type of atoms in the pair. A simulated\u000d\u000aPDF from a gold nanoparticle with the face centered cubic (FCC)\u000d\u000aatomic structure can be seen in Figure 2. The 9 black cubes mark\u000d\u000athe smallest repeated units, i.e. the FCC unit cells. Each peak in the\u000d\u000aPDF corresponds to an interatomic distance in the particle.\u000d\u000aThe PDF thus contains information on the atoms present in a\u000d\u000astructure and their relative interatomic distances, as is also the case\u000d\u000afor methods like solid-state Nuclear Magnetic Resonance (NMR)\u000d\u000aand Extended X-ray Absorption Fine Spectroscopy (EXAFS). In a\u000d\u000aPDF, these relative interatomic distances are not linked to their corresponding atomic identity. Going from this distance information\u000d\u000ato a structure can be described as an unassigned distance geometry\u000d\u000aproblem (uDGP), compared to the assigned distance geometry problem (aDGP), where the assignment of atoms is known. [3, 4, 17]\u000d\u000aObtaining physical structures from the unassigned distance matrix\u000d\u000a(uDM) is a combinatorial problem whereas it is straightforward to\u000d\u000areconstruct the structure once the atoms are assigned to obtain the\u000d\u000aassigned distance matrix (aDM). This makes it challenging to go\u000d\u000aFigure 2: Illustration of an FCC gold particle and its corresponding PDF. The 9 black cubes show the smallest repeated\u000d\u000aunits, i.e. the FCC unit cells.\u000d\u000aFigure 3: Illustration of the structure of a mono-metallic\u000d\u000ananoparticle (green) represented using the unit sphere\u000d\u000aalong with eleven satellites (blue).\u000d\u000adirectly from PDF to structure, and analysing the PDF to extract a\u000d\u000astructure model is often a bottleneck in material characterisation.\u000d\u000aIn this paper, we propose to use a deep latent variable model for\u000d\u000astructure characterisation from simulated PDF data. We address\u000d\u000athe task of going from the uDGP to the aDGP using a Conditional\u000d\u000aVariational Autoencoder (CVAE). [29, 42] As a proof of concept we\u000d\u000ause a CVAE based approach to predict atomic structures from PDFs\u000d\u000aof mono-metallic nanoparticles of sizes up to 100 atoms. We focus\u000d\u000aon a simple, hypothetical system, where the nanoparticles all are\u000d\u000amade to take the FCC structure (Figure 2), but are made of di!erent\u000d\u000aelements, and are of di!erent sizes. We demonstrate that a CVAE\u000d\u000amodel followed by an trilateration algorithm can be used to go from\u000d\u000aa PDF to the particle structure in a matter of seconds (Figure 5).\u000d\u000aWe use a novel graph representation of structures by projecting\u000d\u000athem inside a unit sphere such that one of the atoms is at the origin of the sphere, and the farthest atom is on the sphere surface.\u000d\u000aThis allows us to capture relative distances between atoms as node\u000d\u000afeatures across di!erent structures in a consistent manner. Furthermore, to improve the reconstruction of the structures, several\u000d\u000asatellites or anchor points are introduced on the surface of the unit\u000d\u000asphere inspired by trilateration techniques used in satellite based\u000d\u000ageo-localisation. [40] A sample structure in this representation is\u000d\u000ashown in Figure 3.\u000d\u000aThe CVAE receives the input structure (aDM) and the PDF (uDM)\u000d\u000aas input. We treat the PDF as the conditional input to the CVAE\u000d\u000awhich is trained to predict both the aDM and the distances between","Characterising the atomic structure of mono-metallic nanoparticles from x-ray sca!ering data using conditional generative models MLG 2020, Aug 24, 2020, San Deigo, CA\u000d\u000aeach atom and the satellites. The \"nal location of atoms are determined by trilateration. We also study the in#uence of the number of\u000d\u000asatellites on the quality of reconstructed structures. We compare the\u000d\u000aability of the CVAE with a Deterministic Autoencoder (DAE) which\u000d\u000aalso receives the PDF as the conditional input and predicts the aDM.\u000d\u000aAdditionally, we explore the latent embedding space learnt by the\u000d\u000atwo models by interpolating between latent vector points to predict\u000d\u000anovel structural motifs as depicted in Figure 1.\u000d\u000a2 BACKGROUND\u000d\u000a2.1 Related work\u000d\u000aIn PDF analysis, the uDGP is currently addressed through modelling. A PDF can be calculated from a structure model, and the\u000d\u000amodel is re\"ned until a good agreement between the experimental\u000d\u000aand calculated PDF is obtained. Multiple programs [9, 26, 34] provide a framework for assigning the distances through re\"nement of\u000d\u000aone or more structure models to the data. Minimizing the di!erence\u000d\u000abetween model and data during re\"nement can be a complex problem, and for nanomaterials [6, 23] in particular, identifying good\u000d\u000astarting models for the atomic structure is di$cult. To overcome\u000d\u000athis bottleneck, both model-based and data-driven approaches are\u000d\u000acurrently being developed. For model-based classi\"cation of PDF\u000d\u000adata, cluster-mining [2] and structure-mining [43] approaches have\u000d\u000abeen proposed to automate re\"nements of large numbers of structure models to the data. These methods can help scientists expand\u000d\u000atheir view of possible structure solutions, as thousands of structures\u000d\u000acan be tested. However, model-based algorithms are computationally heavy and are limited to already known starting structures that\u000d\u000acan be mined from databases.\u000d\u000aRecently, machine learning (ML) methods have started surfacing\u000d\u000ain the \"eld of structure characterisation with PDF. A Convolutional Neural Network (CNN) has been used to determine the space\u000d\u000agroup of a structure from an experimental PDF. [32] A few papers have also shown the use of dimensionality reduction tools\u000d\u000asuch as Principal Component Analysis (PCA) and Non-negative\u000d\u000aMatrix Factorization (NMF) on PDF data. Such methods are useful\u000d\u000ain particular when dealing with PDFs from multiphase systems,\u000d\u000aas the contributions from e.g. di!erent chemical species can be\u000d\u000aresolved. [10, 14, 20] Both the space-group classi\"cation and the dimensionality reduction methods do not directly characterise atomic\u000d\u000astructure, but help constrain the number of structural possibilities\u000d\u000afor further analysis.\u000d\u000aVariational Autoencoders (VAE) [29] have shown great promise\u000d\u000aas a method for generating new and/or targeted organic molecules.\u000d\u000aThese data-driven models use the SMILES [7, 22, 41] string representation of graphs to generate structures. VAEs have been used to\u000d\u000agenerate new and valid chemical structures for a diverse class of\u000d\u000amolecules. [22, 31] CVAEs can be trained to generate structures that\u000d\u000arespect targeted properties by conditioning the model on additional\u000d\u000adata. This allows organic molecules to be generated rapidly without\u000d\u000ahaving to explore all structural possibilities through theoretical computations. [7, 41] The contributions mentioned here are only a fraction of the work done applying ML to chemistry. [8, 15, 16, 19, 21]\u000d\u000aWithin inorganic chemistry, CNN have shown promise in characterising microstructures from scattering data [8]. Unsupervised ML\u000d\u000amethods have also been used to discover structural relationships\u000d\u000aand link them to microscopic properties on experimentally characterised structures from chemical databases. [36] However, work\u000d\u000afocusing on applying ML methods to characterise atomic structures\u000d\u000afrom x-ray data is still very sparse.\u000d\u000a2.2 X-ray total scattering and Pair Distribution\u000d\u000aFunction (PDF) for structural analysis\u000d\u000aA PDF can be obtained by Fourier transforming experimental xray total scattering data. In an x-ray scattering experiment, one\u000d\u000ameasures the angle-dependent interference pattern that arises when\u000d\u000amonochromatic x-rays scatter o! a sample, as this scattering pattern\u000d\u000acontains information on the sample structure. X-ray total scattering\u000d\u000aexperiments are best done at large scale synchrotron facilities where\u000d\u000ahigh energy x-rays are available, as scattering data must be collected\u000d\u000ato high values of the scattering vector \ud444:\u000d\u000a\ud444 = 4\ud70b\ud460\ud456\ud45b(\ud703)/\ud706 (1)\u000d\u000aHere, \ud706 is the radiation wavelength, and \ud703 is the scattering angle.\u000d\u000aFor PDF analysis, high quality scattering data are generally needed\u000d\u000ain the range ca. 0.5 \u2212 25 \u00c5\u22121.\u000d\u000aThe measured x-ray total scattering data is corrected and normalized to obtain the structure function \ud446 (\ud444). This function is then\u000d\u000aFourier transformed over the available \ud444-range yielding the PDF,\u000d\u000a\ud43a(\ud45f): [18]\u000d\u000a\ud43a(\ud45f) = 2\u000d\u000a\ud70b\u000d\u000a\u222b \ud444\ud45a\ud44e\ud465\u000d\u000a\ud444\ud45a\ud456\ud45b\u000d\u000a\ud444[\ud446 (\ud444) \u2212 1] sin(\ud444 \u00b7 \ud45f)\ud451\ud444, (2)\u000d\u000aThe Fourier transform of the scattering data in \ud444 thus yields a\u000d\u000afunction in \ud45f, which represents a histogram of interatomic distances\u000d\u000aor the uDM. [13, 24, 25]\u000d\u000aIn the current work, all scattering data and PDFs have been\u000d\u000asimulated from structure models. This has been done using the\u000d\u000aDi!py-CMI software. [26] Details are given in Section 4.3 and simulation parameters are shown in Appendix C, Table 3.\u000d\u000a3 METHODS\u000d\u000aWe are interested in reconstructing structures of mono-metallic\u000d\u000ananoparticles, s \u2208 S, from their corresponding PDFs, x \u2208 X. Each\u000d\u000astructure, s \u2208 R\ud441 \u00d7\ud439 , comprises \ud441 nodes with \ud439 node attributes. The\u000d\u000anode attributes are the atomic number, interatomic distances and\u000d\u000adistance to the satellites. The corresponding PDFs are represented\u000d\u000aas the sequence x \u2208 R\ud437 . The data used in this work are detailed in\u000d\u000aSection 4.3.\u000d\u000aReconstructing structures from PDFs or the task of going from\u000d\u000auDGP to aDGP can be formulated as the learning task: \ud453 (\u00b7) : X \u2192\u000d\u000aS. Seen from a density point of view, we are interested in estimating\u000d\u000a\ud45d(s|x). In this work, we take up a latent variable approach to capture\u000d\u000athe dependencies between s and x. This is done by introducing the\u000d\u000alatent variable z \u2208 R\ud43b , where \ud43b is the latent (hidden) dimension.\u000d\u000aSpeci\"cally, we use the CVAE framework to derive a model and to\u000d\u000aaccess a meaningful latent space that can be used for conditional\u000d\u000agenerative sampling. [29, 42]\u000d\u000aThe CVAE extends the VAE framework to incorporate conditioning inputs to the encoder-decoder structure. The CVAE objective\u000d\u000aminimizes the Kullback-Leibler (KL) divergence between the true","MLG 2020, Aug 24, 2020, San Deigo, CA Anker and Kj\u00e6r, et al.\u000d\u000aFigure 4: CVAE utilized to predict the aDM used to reconstruct the mono-metallic nanoparticle structure with conditional\u000d\u000ainput provided in the form of the PDF. The encoder gets the structure, s, and its PDF, x, as input. The PDF is also input to the\u000d\u000aprior network. The decoder network predicts the aDM, s\u02c6, from a latent sample, z, which is used to reconstruct the structure.\u000d\u000aposterior distribution \ud45d(z|s, x) and its variational approximation\u000d\u000a\ud45e\ud719 (z|s, x) resulting in an objective of the form: [42]\u000d\u000aLCVAE = L\ud45f\ud452\ud450 + L\ud45f\ud452\ud454 (3)\u000d\u000a= \u2212E\ud45e\ud719\u000d\u000a\"\u000d\u000alog \ud45d\ud703 (s|z, x)\u000d\u000a#\u000d\u000a+ KL\"\u000d\u000a\ud45e\ud719 (z|s, x)||\ud45d\ud713 (z|x)\u000d\u000a# (4)\u000d\u000aThis objective is derived in Appendix A.\u000d\u000aThe \"rst term in the objective in Eq. (4) can be interpreted as\u000d\u000athe reconstruction loss, L\ud45f\ud452\ud450 . More formally, it is the expected\u000d\u000aconditional log-likelihood under the approximate posterior density\u000d\u000apredicted by the encoder. The expectation is with respect to the\u000d\u000avariational density, \ud45e\ud719 (z|s, x) = N (z; \ud741\ud719, \ud7482\u000d\u000a\ud719 ), which is constrained\u000d\u000ato be a Gaussian with mean \ud741\ud719 and variance \ud7482\u000d\u000a\ud719 . The mean and\u000d\u000avariance of the posterior density are predicted by the encoder neural\u000d\u000anetwork parameterised by \ud719. The encoder maps the input structure,\u000d\u000as, and the corresponding PDF, x, to a latent random variable z \u2208 R\ud43b .\u000d\u000aThe term inside the expectation is the conditional log likelihood,\u000d\u000a\ud45d\ud703 (s|z, x), predicted by the decoder neural network parameterised\u000d\u000aby \ud703. The decoder predicts the aDM of interest, s\u02c6, from the latent\u000d\u000arepresentation, z, and the conditioning input, x. The reconstruction\u000d\u000aloss in practice is computed as the mean-squared error between\u000d\u000athe input aDM s and the predicted aDM s\u02c6, i.e.,\u000d\u000aL\ud45f\ud452\ud450 = ||s \u2212 s\u02c6||2. (5)\u000d\u000aThe second term in Eq. (4), L\ud45f\ud452\ud454, acts as the regularisation term\u000d\u000aforcing the posterior density to match the conditional prior density\u000d\u000a\ud45d\ud713 (z|x) = N (z; \ud741\ud713, \ud7482\u000d\u000a\ud713 ) which is also constrained to be Gaussian\u000d\u000awith mean \ud741\ud713 and variance \ud7482\u000d\u000a\ud713 . The prior network is parameterised\u000d\u000aby \ud713.\u000d\u000aThe CVAE model used in this work is illustrated in Figure 4\u000d\u000aand the network architecture details are described in Section 4.1.\u000d\u000aThe details of the training and generative (inference) processes are\u000d\u000adescribed below:\u000d\u000aTraining process: The encoder gets a structure in the form of\u000d\u000aan aDM and its corresponding PDF as input to predict the latent\u000d\u000adensity parameters. The prior network predicts the parameters of\u000d\u000athe prior density from the PDF. The KL divergence is computed\u000d\u000abetween the posterior and the prior densities, resulting in the L\ud45f\ud452\ud454\u000d\u000aterm. A sample from the latent posterior density is then decoded\u000d\u000ain the decoder network. The decoder\u2019s output is the aDM which is\u000d\u000acompared with the input structure to obtain the reconstruction loss\u000d\u000aL\ud45f\ud452\ud450 . The combined loss is backpropagated through the network,\u000d\u000atraining the model to reconstruct the aDM from the latent space.\u000d\u000aGenerative process: During the inference time, only the trained\u000d\u000aprior and decoder networks are used. The PDF is input to the prior\u000d\u000anetwork and a sample from the resulting prior density is decoded\u000d\u000aby the decoder, generating a new aDM. The \"nal location of atoms\u000d\u000aare determined by trilateration from the predicted distances, as\u000d\u000adescribed in Section 4.2.\u000d\u000a4 EXPERIMENTS\u000d\u000a4.1 Network Architecture\u000d\u000aThe CVAE model used in this work has three high level components\u000d\u000athat are implemented using neural networks.\u000d\u000aThe encoder, \ud45e\ud719 (\u00b7), consists of 4 fully connected layers with\u000d\u000arecti\"ed linear unit activation\u2019s except for the last layer. These\u000d\u000aencoder layers use [\ud439, 384, 256, 128, 2\ud43b] hidden units where \ud439 is\u000d\u000athe input feature dimension and \ud43b is the latent dimension. The\u000d\u000aPDF which is the conditional input, x, is concatenated to the input\u000d\u000astructure, s, by upsampling it to match the node feature dimensions.\u000d\u000aThe upsampling operation is performed with 3 layers of gated 1-\u000d\u000aD transpose convolution layers. The encoder predicts the mean,\u000d\u000a\ud741\ud719, \ud7482\u000d\u000a\ud719 \u2208 R\ud43b , of the encoder density. Latent vectors, z \u2208 R\ud43b , are\u000d\u000asampled from the predicted density: z \u223c N (\ud741\ud719, \ud7482\u000d\u000a\ud719 ).\u000d\u000aThe prior, \ud45d\ud713 (\u00b7), comprises 3 gated 1-D convolution layers with\u000d\u000arecti\"ed linear unit except for the last layer with [\ud437, 48, 24, 2\ud43b] hidden nodes where \ud437 is the input PDF dimension. The prior network\u000d\u000apredicts the mean, \ud741\ud713, \ud7482\u000d\u000a\ud713 \u2208 R\ud43b , of the prior density.\u000d\u000aThe decoder, \ud45d\ud703 (\u00b7), predicts s\u02c6 from the latent variable, z. The\u000d\u000adecoder is implemented with 5 fully connected layers with recti\"ed\u000d\u000alinear unit except for the last layer. These decoder layers have\u000d\u000a[\ud43b, 128, 256, 384, 512] hidden units and outputs s\u02c6 \u2208 R\ud441 \u00d7\ud439 which\u000d\u000aare the relative distance between the atoms and the atoms distances\u000d\u000ato the satellites.","Characterising the atomic structure of mono-metallic nanoparticles from x-ray sca!ering data using conditional generative models MLG 2020, Aug 24, 2020, San Deigo, CA\u000d\u000a4.2 Post-processing\u000d\u000aThe CVAE model reduces the uDGP to aDGP. The prediction from\u000d\u000athe CVAE model, s\u02c6, is a combination of the aDM and atom-satellite\u000d\u000adistances. From this prediction, one can reconstruct the structures\u000d\u000ausing trilateration. We start with the \"rst atom in the distance list\u000d\u000aas the origin and add subsequent atoms with respect to the previous\u000d\u000aones based on the relative distances between atoms and the satellites, which were introduced for the purpose of trilateration. The\u000d\u000areconstruction can be seen as solving the aDGP but with help from\u000d\u000athe atom-satellite distances, thereby minimizing the uncertainty\u000d\u000aon the placed atoms. If the predicted distances are accurate there\u000d\u000ais a unique solution. As the predicted distances are not exact, we\u000d\u000apose the trilateration task as an optimization problem and use an\u000d\u000aL-BFGS-B optimizer to solve for the reconstructed structure. [44]\u000d\u000a4.3 Data and model hyperparameters\u000d\u000aData: The nanoparticle structures were simulated using the Atomic\u000d\u000aSimulation Environment (ASE). [1] All nanoparticles were made\u000d\u000ato take the FCC structure, but multiple particles were obtained by\u000d\u000avarying the type of atoms, exposed surfaces, layers of atoms and\u000d\u000alattice constant. This simulation procedure yielded 3137 unique\u000d\u000ananoparticle structures consisting of fewer than 100 atoms. From\u000d\u000aeach of the 3137 structures, a PDF was simulated using Di!pyCMI. [26]. Details of the simulation procedure and the parameters\u000d\u000aused which re#ect typical values for synchrotron experiments are\u000d\u000adescribed in Appendix C, Table 3.\u000d\u000aBaseline model: There are no established baseline models to compare with for the tasks considered in this work. We compare the\u000d\u000aCVAE with a DAE model which has exactly the same architecture\u000d\u000aas the CVAE model except for the stochasticity. The DAE has been\u000d\u000achosen as a baseline as it can perform the regression task of going\u000d\u000afrom uDM to aDM and it also learns a (deterministic) latent representation of the input. The latent variable, z, in the DAE model\u000d\u000ais treated as a deterministic variable. And instead of the KL divergence based regularisation inherent to CVAE, L2 regularisation is\u000d\u000aintroduced between the latent vectors predicted by the encoder\u000d\u000anetwork and the prior networks during training.\u000d\u000aPerformance metric: Comparing generative models for the task\u000d\u000aunder consideration is not straightforward as there are no established metrics or measures to quantify the validity of these nanoparticle structures. One surrogate measure that could signify the quality of structures can be derived by obtaining the PDFs of the generated structures, x\u02c6, and comparing them to the input PDFs. Note\u000d\u000athat going from structure to PDF is easier than the converse task.\u000d\u000aThe di!erence in PDFs is quanti\"ed using a Mean-Squared Error\u000d\u000a(MSE) term, \ud445\ud45d = ||x \u2212 x\u02c6||2 and the Pearson correlation coe$cient.\u000d\u000aHyperparameters: The CVAE and DAE models were trained with\u000d\u000a2400, validated on 600 and tested on 137 structures. The models\u000d\u000awere implemented in PyTorch. [37] Training was performed with a\u000d\u000abatch size of 20. Adam optimizer with a learning rate of 5\ud452\u22125 was\u000d\u000aused to optimize the loss in Eq. (4) for all the experiments. [28] The\u000d\u000amodels were assumed to have converged if there was no improvement in validation loss for 50 epochs. The latent space dimension\u000d\u000awas set to 2 and the number of satellites 11. All the parameters\u000d\u000awere tuned based on experiments on the training set performance.\u000d\u000aTable 1: Mean-squared error \ud438\ud45d and average Pearson correlation between the input PDFs and the PDFs reconstructed\u000d\u000afrom the generated structures for the the DAE and CVAE\u000d\u000amodels. Signi\"cant di!erences (\ud45d < 0.001) based on twotailed paired sample t-tests are shown in bold.\u000d\u000aModel \ud445\ud45d Pearson\u000d\u000aDAE 0.4548 \u00b1 0.30 0.7482 \u00b1 0.17\u000d\u000aCVAE 0.4823 \u00b1 0.34 0.7321 \u00b1 0.19\u000d\u000aFigure 5: Visualisation of the PDF of the ground truth and\u000d\u000athe reconstructed PDF of the reconstructed mono-metallic\u000d\u000ananoparticle for two typical cases from the validation set.\u000d\u000aBoth cases show reasonable reconstructions, but the top example has broader peaks than the ground truth due to disorder in the structure. For the bottom example, the PDF is\u000d\u000ashifted due to incorrect distance between the atoms. The difference curve is scaled by 0.2 for better visualisation.\u000d\u000a5 RESULTS AND DISCUSSION\u000d\u000aThe two models were trained on the same training/validation/test\u000d\u000asplits and the performance metrics are reported in Table 1 where\u000d\u000awe observe that the DAE model is better at the reported metrics.\u000d\u000aThis is expected as there are no stochastic elements in the DAE\u000d\u000amodel. However, an important feature of the CVAE model is the\u000d\u000aaccess to a meaningful latent space which can be used to explore\u000d\u000aand obtain new structural motifs. This is clearly demonstrated in\u000d\u000aFigure 1. The CVAE model can not only reasonably reconstruct\u000d\u000astructure A and structure B from their PDFs, but it can also generate reasonable mono-metallic nanoparticle structures from the\u000d\u000ainterpolated latent vectors. As we move from left to right, we are\u000d\u000atraversing between the latent points for the two structures and we\u000d\u000asee a clear evolution of structures. This is not the case for the DAE\u000d\u000amodel, which produces invalid structures when decoding from the\u000d\u000ainterpolated latent vectors.\u000d\u000aOne common feature of both DAE and CVAE models is that they\u000d\u000acan generate structures comprising the right amount of atoms, as\u000d\u000aillustrated on the test set for the CVAE model in Figure 8. When\u000d\u000ainterpolating in latent space, the number of atoms take discrete\u000d\u000asteps.\u000d\u000aThis behaviour is probably due to the discrete nature of the\u000d\u000atraining set. A more diverse training set can be achieved by adding\u000d\u000aadditional structure geometries or facets of the nanoparticles.","MLG 2020, Aug 24, 2020, San Deigo, CA Anker and Kj\u00e6r, et al.\u000d\u000aIn Figure 5, two structures obtained from PDFs with the CVAE\u000d\u000amodel are visualised along with the ground truth structures. Apart\u000d\u000afrom the reconstructed structures, the PDFs reconstructed from\u000d\u000athe generated structure are shown. The reconstructed structure\u000d\u000ain the top row has some disorder which is also re#ected in the\u000d\u000areconstructed PDF. The reconstructed PDF in the top row has overlapping and broader peaks when compared to the ground truth.\u000d\u000aThe structure in the bottom row has longer distances between the\u000d\u000aatoms than the ground truth, which results in a slight o!set in the\u000d\u000areconstructed PDF. Currently, we do not explicitly optimize for the\u000d\u000aPDFs to be aligned. These inconsistencies in the PDFs can also be\u000d\u000aincorporated into the training process by introducing an additional\u000d\u000aloss term that is dependent on the PDFs which we expect could\u000d\u000aimprove the quality of generated structures.\u000d\u000aThe satellites were introduced to help with the trilateration. The\u000d\u000ae!ect of using [4, 11, 100] satellites is shown in Figure 6. It can be\u000d\u000aseen that the reconstructed structures improve with the number\u000d\u000aof satellites. There is a small improvement in the reconstructed\u000d\u000astructures when using 100 satellites compared to 11 but it comes at\u000d\u000aa large increase in computation time as shown in Table 2, where\u000d\u000athe computer time is calculated as how long time it takes to solve\u000d\u000athe trilateration problem. This might be a factor to consider if this\u000d\u000amethod is to be used to get instantaneous results. A reasonable\u000d\u000atrade-o! between performance and computation time was found\u000d\u000ato be with 11 satellites.\u000d\u000a6 CONCLUSION AND FUTURE WORK\u000d\u000aDeep learning based methods have proven to be extremely powerful\u000d\u000aat embedding complex data into low dimensions, while still learning\u000d\u000acomplex and useful features. In this work, we have demonstrated a\u000d\u000aproof of concept on embedding structures of mono-metallic nanoparticles while conditioning each structure on their corresponding simulated PDF. Both DAE and CVAE models show that valid structures\u000d\u000acan be obtained from the PDFs. Furthermore, the models show that\u000d\u000athey are not constrained to reconstruct a \"xed number of atoms.\u000d\u000aThese methods could allow for fast analysis of PDF data, which\u000d\u000acan prove extremely potent e.g. for real-time data analysis during\u000d\u000asynchrotron experiments.\u000d\u000aFurthermore, we have shown that the latent space learnt by the\u000d\u000aCVAE model is highly structured (Figure 7). This latent space can\u000d\u000abe used to obtain novel structural motifs as illustrated in Figure 1\u000d\u000awhere interpolated latent vectors resulted in valid structures when\u000d\u000acompared to the DAE model.\u000d\u000aBy normalizing the structures to reside inside a unit sphere we\u000d\u000ashow how trilateration can be used to reconstruct a chemical structure from an aDM. We investigated a range of \"xed satellite points\u000d\u000afor structure reconstruction showing that using 11 satellites was\u000d\u000aa reasonable trade-o! between structure reconstruction accuracy\u000d\u000aand computation time.\u000d\u000aThis work has shown great promise which calls for future investigations. The data used for this model only consist of a small\u000d\u000anumber of mono-metallic structures, all with the FCC structure.\u000d\u000aFor the model to be useful for chemists the chemical-space must\u000d\u000abe massively expanded, for instance by including structure types\u000d\u000aother than FCC. In addition, the model could bene\"t from having a\u000d\u000aPDF di!erence regularization term implemented in its loss function.\u000d\u000aTable 2: Average reconstruction time with varying number\u000d\u000aof satellites on a standard laptop with 8 CPU cores. The\u000d\u000amean and deviation are calculated based on 50 reconstructions.\u000d\u000a# of satellites Time (s)\u000d\u000a4 8.6 \u00b1 5.4\u000d\u000a11 9.6 \u00b1 5.9\u000d\u000a100 23.5 \u00b1 12.0\u000d\u000aFigure 6: In#uence of the number of satellites used for trilateration, on the quality of reconstructed structures for two\u000d\u000atypes of mono-metallic nanoparticles with: 19 atoms (top\u000d\u000arow) and 87 atoms (bottom row).\u000d\u000aCurrently the model only predicts the positions of the atoms for\u000d\u000amono-metallic nanoparticles as the composition is assumed to be\u000d\u000aknown. To expand the use cases of the model, the atom labels need\u000d\u000aalso be predicted. These are directions we look forward to pursuing.\u000d\u000aIn conclusion, the results reported in this work have been encouraging and the proposed model can be used as a generative\u000d\u000amodel for characterisation of mono-metallic FCC nanoparticles.\u000d\u000aAcknowledgments. This work is part of a project that has received\u000d\u000afunding from the European Research Council (ERC) under the European Union\u2019s Horizon 2020 research and innovation programme,\u000d\u000a(Grant agreement no. 804066).\u000d\u000aWork in the Billinge group was supported by the US National\u000d\u000aScience Foundation through grant DMR-1922234.\u000d\u000aREFERENCES\u000d\u000a[1] Jakob Blomqvist Ivano E. Castelli Rune Christensen Marcin Du\u0142ak Jesper Friis\u000d\u000aMichael N. Groves Bj\u00f8rk Hammer Cory Hargus Eric D. Hermes Paul C. Jennings\u000d\u000aPeter Bjerre Jensen James Kermode John R. Kitchin Esben Leonhard Kolsbjerg\u000d\u000aJoseph Kubal Kristen Kaasbjerg Steen Lysgaard J\u00f3n Bergmann Maronsson Tristan\u000d\u000aMaxson Thomas Olsen Lars Pastewka Andrew Peterson Carsten Rostgaard Jakob\u000d\u000aSchi\u00f8tz Ole Sch\u00fctt Mikkel Strange Kristian S. Thygesen Tejs Vegge Lasse Vilhelmsen Michael Walter Zhenhua Zeng Ask Hjorth Larsen, Jens J\u00f8rgen Mortensen\u000d\u000aand Karsten W Jacobsen. 2017. The atomic simulation environment\u2014a Python\u000d\u000alibrary for working with atoms. Journal of Physics: Condensed Matter 29, 27\u000d\u000a(2017), 273002.\u000d\u000a[2] Soham Banerjee, Chia-Hao Liu, Kirsten M. \u00d8. Jensen, Pavol Juh\u00e1s, Jennifer D.\u000d\u000aLee, Marcus Tofanelli, Christopher J. Ackerson, Christopher B. Murray, and\u000d\u000aSimon J. L. Billinge. 2020. Cluster-mining: an approach for determining core\u000d\u000astructures of metallic nanoparticles from atomic pair distribution function data.\u000d\u000aActa Crystallographica Section A 76, 1 (Jan 2020), 24\u201331.\u000d\u000a[3] Simon J. L. Billinge, Phillip M Duxbury, Douglas S. Gon\u00e7alves, Carlile Lavor,\u000d\u000aand Antonio Mucherino. 2016. Assigned and unassigned distance geometry:","Characterising the atomic structure of mono-metallic nanoparticles from x-ray sca!ering data using conditional generative models MLG 2020, Aug 24, 2020, San Deigo, CA\u000d\u000aapplications to biological molecules and nanostructures. 4OR 14, 4 (2016), 337\u2013\u000d\u000a376.\u000d\u000a[4] Simon J. L. Billinge, Phillip M. Duxbury, Douglas S. Gon\u00e7alves, Carlile Lavor, and\u000d\u000aAntonio Mucherino. 2018. Recent results on assigned and unassigned distance\u000d\u000ageometry with applications to protein molecules and nanostructures. Annals of\u000d\u000aOperations Research 271, 1 (December 2018), 161\u2013203.\u000d\u000a[5] Simon J. L. Billinge and Mercouri G. Kanatzidis. 2004. Beyond crystallography: the\u000d\u000astudy of disorder, nanocrystallinity and crystallographically challenged materials\u000d\u000awith pair distribution functions. Chemical communications 7 (2004), 749\u2013760.\u000d\u000a[6] Simon J. L. Billinge and Igor Levin. 2007. The Problem with Determining Atomic\u000d\u000aStructure at the Nanoscale. Science 316, 5824 (2007), 561\u2013565.\u000d\u000a[7] John Bradshaw, Brooks Paige, Matt J. Kusner, Marwin H. S. Segler, and Jos\u00e9 Miguel\u000d\u000aHern\u00e1ndez-Lobato. 2019. A Model to Search for Synthesizable Molecules. CoRR\u000d\u000aabs/1906.05221 (2019).\u000d\u000a[8] Keith T. Butler, Daniel W. Davies, Hugh Cartwright, Olexandr Isayev, and Aron\u000d\u000aWalsh. 2018. Machine learning for molecular and materials science. Nature 559,\u000d\u000a7715 (2018), 547\u2013555.\u000d\u000a[9] J. W. Liu D Bryndin E. S. Bo\u017ein J. Bloch Th Pro!en C L Farrow, P Juhas and\u000d\u000aSimon J. L. Billinge. 2007. PDF\"t2 and PDFgui: computer programs for studying\u000d\u000ananostructure in crystals. Journal of Physics: Condensed Matter 19, 33 (jul 2007),\u000d\u000a335219.\u000d\u000a[10] Karena W. Chapman, Saul H. Lapidus, and Peter J. Chupas. 2015. Applications\u000d\u000aof principal component analysis to pair distribution function data. Journal of\u000d\u000aApplied Crystallography 48, 6 (Dec 2015), 1619\u20131626.\u000d\u000a[11] Anthony K. Cheetham and Andrew L. Goodwin. 2014. Crystallography with\u000d\u000apowders. Nature materials 13, 8 (2014), 760\u2013762.\u000d\u000a[12] Troels Lindahl Christiansen, Susan R. Cooper, and Kirsten M. \u00d8. Jensen. 2020.\u000d\u000aThere\u2019s no place like real-space: elucidating size-dependent atomic structure of\u000d\u000ananomaterials using pair distribution function analysis. Nanoscale Adv. (2020).\u000d\u000a[13] Troels Lindahl Christiansen, Emil T. S. Kj\u00e6r, Anton Kovyakh, Morten L. R\u00f6deren,\u000d\u000aMartin H\u00f8j, Tom Vosch, and Kirsten M. \u00d8. Jensen. 2020. Structure analysis\u000d\u000aof supported disordered molybdenum oxides using pair distribution function\u000d\u000aanalysis and automated cluster modelling. Journal of Applied Crystallography 53,\u000d\u000a1 (Feb 2020), 148\u2013158.\u000d\u000a[14] Jacqueline M. Cole, Xie Cheng, and Michael C. Payne. 2016. Modeling Pair Distribution Functions of Rare-Earth Phosphate Glasses Using Principal Component\u000d\u000aAnalysis. Inorganic Chemistry 55, 21 (2016), 10870\u201310880.\u000d\u000a[15] T\u00e2nia F. G. G. Cova and Alberto A. C. C. Pais. 2019. Deep Learning for Deep\u000d\u000aChemistry: Optimizing the Prediction of Chemical Patterns. Frontiers in Chemistry\u000d\u000a7 (2019), 809.\u000d\u000a[16] David K. Duvenaud, Dougal Maclaurin, Jorge Iparraguirre, Rafael Bombarell,\u000d\u000aTimothy Hirzel, Al\u00e1n Aspuru-Guzik, and Ryan P Adams. 2015. Convolutional\u000d\u000anetworks on graphs for learning molecular \"ngerprints. In Advances in neural\u000d\u000ainformation processing systems. 2224\u20132232.\u000d\u000a[17] P.M. Duxbury, L. Granlund, S.R. Gujarathi, P. Juhas, and Simon J. L. Billinge. 2016.\u000d\u000aThe unassigned distance geometry problem. Discrete Applied Mathematics 204\u000d\u000a(2016), 117 \u2013 132.\u000d\u000a[18] Takeshi Egami and Simon J. L. Billinge. 2003. Underneath the Bragg peaks:\u000d\u000astructural analysis of complex materials. Elsevier.\u000d\u000a[19] Daniel C. Elton, Zois Boukouvalas, Mark D. Fuge, and Peter W. Chung. 2019.\u000d\u000aDeep learning for molecular design\u2014a review of the state of the art. Mol. Syst.\u000d\u000aDes. Eng. 4 (2019), 828\u2013849. Issue 4.\u000d\u000a[20] Harry S. Geddes, Helen Blade, James F. McCabe, Leslie P. Hughes, and Andrew L.\u000d\u000aGoodwin. 2019. Structural characterisation of amorphous solid dispersions via\u000d\u000ametropolis matrix factorisation of pair distribution function data. Chem. Commun.\u000d\u000a55 (2019), 13346\u201313349. Issue 89.\u000d\u000a[21] Garrett B. Goh, Nathan O. Hodas, and Abhinav Vishnu. 2017. Deep learning\u000d\u000afor computational chemistry. Journal of computational chemistry 38, 16 (2017),\u000d\u000a1291\u20131307.\u000d\u000a[22] Rafael G\u00f3mez-Bombarelli, Jennifer N. Wei, David Duvenaud, Jos\u00e9 Miguel\u000d\u000aHern\u00e1ndez-Lobato, Benjam\u00edn S\u00e1nchez-Lengeling, Dennis Sheberla, Jorge\u000d\u000aAguilera-Iparraguirre, Timothy D. Hirzel, Ryan P Adams, and Al\u00e1n Aspuru-Guzik.\u000d\u000a2018. Automatic chemical design using a data-driven continuous representation\u000d\u000aof molecules. ACS central science 4, 2 (2018), 268\u2013276.\u000d\u000a[23] Kirsten M. \u00d8. Jensen, Pavol Juhas, Marcus A. Tofanelli, Christine L. Heinecke,\u000d\u000aGavin Vaughan, Christopher J. Ackerson, and Simon J. L. Billinge. 2016. Polymorphism in magic-sized Au144(SR)60 clusters. Nature Communications 7 (2016).\u000d\u000a[24] Kirsten M. \u00d8. Jensen, Mogens Christensen, Pavol Juhas, Christo!er Tyrsted, Espen D. B\u00f8jesen, Nina Lock, Simon J. L. Billinge, and Bo B. Iversen. 2012. Revealing\u000d\u000athe Mechanisms behind SnO2 Nanoparticle Formation and Growth during Hydrothermal Synthesis: An In Situ Total Scattering Study. Journal of the American\u000d\u000aChemical Society 134, 15 (2012), 6785\u20136792.\u000d\u000a[25] Mikkel Juelsholt, Troels Lindahl Christiansen, and Kirsten M. \u00d8. Jensen. 2019.\u000d\u000aMechanisms for Tungsten Oxide Nanoparticle Formation in Solvothermal Synthesis: From Polyoxometalates to Crystalline Materials. The Journal of Physical\u000d\u000aChemistry C 123, 8 (2019), 5110\u20135119.\u000d\u000a[26] Pavol Juh\u00e1s, Christopher L. Farrow, Xiaohao Yang, Kevin R. Knox, and Simon J. L.\u000d\u000aBillinge. 2015. Complex modeling: a strategy and software program for combining\u000d\u000amultiple information sources to solve ill posed structure and nanostructure\u000d\u000ainverse problems. Acta Crystallographica Section A 71, 6 (Nov 2015), 562\u2013568.\u000d\u000a[27] Byung Hyo Kim, Michael J Hackett, Jongnam Park, and Taeghwan Hyeon. 2014.\u000d\u000aSynthesis, characterization, and application of ultrasmall nanoparticles. Chemistry of Materials 26, 1 (2014), 59\u201371.\u000d\u000a[28] Diederik P. Kingma and Jimmy Ba. 2014. Adam: A method for stochastic optimization. arXiv preprint arXiv:1412.6980 (2014).\u000d\u000a[29] Diederik P. Kingma and Max Welling. 2014. Auto-encoding variational bayes. In\u000d\u000aInternational Conference on Learning Representations.\u000d\u000a[30] Maksym V Kovalenko, Liberato Manna, Andreu Cabot, Zeger Hens, Dmitri V\u000d\u000aTalapin, Cherie R Kagan, Victor I Klimov, Andrey L Rogach, Peter Reiss, Delia J\u000d\u000aMilliron, et al. 2015. Prospects of nanoscience with nanocrystals.\u000d\u000a[31] Youngchun Kwon, Jiho Yoo, Youn-Suk Choi, Won-Joon Son, Dongseon Lee, and\u000d\u000aSeokho Kang. 2019. E$cient learning of non-autoregressive graph variational\u000d\u000aautoencoders for molecular graph generation. Journal of Cheminformatics 11, 1\u000d\u000a(2019), 70.\u000d\u000a[32] Chia-Hao Liu, Yunzhe Tao, Daniel Hsu, Qiang Du, and Simon J. L. Billinge. 2019.\u000d\u000aUsing a machine learning approach to determine the space group of a structure\u000d\u000afrom the atomic pair distribution function. Acta Crystallographica Section A 75, 4\u000d\u000a(2019), 633\u2013643.\u000d\u000a[33] Laurens van der Maaten and Geo!rey Hinton. 2008. Visualizing data using t-SNE.\u000d\u000aJournal of machine learning research 9, Nov (2008), 2579\u20132605.\u000d\u000a[34] Martin T. Dove Andrew L. Goodwin Matthew G. Tucker, David A. Keen and\u000d\u000aQun Hui. 2007. RMCPro\"le: reverse Monte Carlo for polycrystalline materials.\u000d\u000aJournal of Physics: Condensed Matter 19, 33 (jul 2007), 335218.\u000d\u000a[35] \u0141ukasz Mentel. [n.d.]. mendeleev \u2013 A Python resource for properties of chemical\u000d\u000aelements, ions and isotopes.\u000d\u000a[36] Thomas C. Nicholas, Andrew L. Goodwin, and Volker L. Deringer. 2020. Understanding the Geometric Diversity of Inorganic and Hybrid Frameworks through\u000d\u000aStructural Coarse-Graining. arXiv preprint arXiv:2005.09939 (2020).\u000d\u000a[37] Adam Paszke, Sam Gross, Francisco Massa, Adam Lerer, James Bradbury, Gregory\u000d\u000aChanan, Trevor Killeen, Zeming Lin, Natalia Gimelshein, Luca Antiga, et al.\u000d\u000a2019. PyTorch: An imperative style, high-performance deep learning library. In\u000d\u000aAdvances in Neural Information Processing Systems. 8024\u20138035.\u000d\u000a[38] Ekaterina Pomerantseva, Francesco Bonaccorso, Xinliang Feng, Yi Cui, and Yury\u000d\u000aGogotsi. 2019. Energy storage: The future enabled by nanomaterials. Science 366,\u000d\u000a6468 (2019).\u000d\u000a[39] Emil Roduner. 2006. Size matters: why nanomaterials are di!erent. Chem. Soc.\u000d\u000aRev. 35 (2006), 583\u2013592. Issue 7.\u000d\u000a[40] P. E. Schmid and J. J. Lynn. 1975. Results of the 3 November 1974 Applications\u000d\u000aTechnology Satellite-6 (ATS-6) trilateration test. NASA Technical Reports Server\u000d\u000a(1975).\u000d\u000a[41] Martin Simonovsky and Nikos Komodakis. 2018. GraphVAE: Towards Generation\u000d\u000aof Small Graphs Using Variational Autoencoders. CoRR abs/1802.03480 (2018).\u000d\u000a[42] Kihyuk Sohn, Honglak Lee, and Xinchen Yan. 2015. Learning structured output\u000d\u000arepresentation using deep conditional generative models. In Advances in neural\u000d\u000ainformation processing systems. 3483\u20133491.\u000d\u000a[43] Long Yang, Pavol Juh\u00e1s, Maxwell W. Terban, Matthew G. Tucker, and Simon\u000d\u000aJ. L. Billinge. 2020. Structure-mining: screening structure models by automated\u000d\u000a\"tting to the atomic pair distribution function over large numbers of models.\u000d\u000aActa Crystallographica Section A 76, 3 (May 2020), 395\u2013409.\u000d\u000a[44] Ciyou Zhu, Richard H. Byrd, Peihuang Lu, and Jorge Nocedal. 1997. Algorithm 778:\u000d\u000aL-BFGS-B: Fortran Subroutines for Large-Scale Bound-Constrained Optimization.\u000d\u000aACM Trans. Math. Softw. 23, 4 (1997), 550\u2013560.\u000d\u000aA DERIVING THE CVAE OBJECTIVE\u000d\u000aWe start with the motivation of approximating the latent posterior\u000d\u000adistribution with a variational density that minimizing the reverse\u000d\u000aKL divergence,\u000d\u000aKL$\u000d\u000a\ud45e(z|s, x)||\ud45d(z|s, x)\u000d\u000a%\u000d\u000a= E\ud45e(z|s,x)\u000d\u000a$\u000d\u000alog \ud45e(z|s, x)\u000d\u000a\ud45d(z|s, x)\u000d\u000a%\u000d\u000a= E\ud45e(z|s,x)\u000d\u000a$\u000d\u000alog & \ud45e(z|s, x)\u000d\u000a\ud45d(s|z, x)\u000d\u000a\ud45d(s|x)\u000d\u000a\ud45d(z|x)\u000d\u000a' % (6)\u000d\u000aThe last equality is due to Bayes\u2019 Rule. Some algebraic manipulations yield,\u000d\u000aKL$\u000d\u000a\ud45e(z|s, x)||\ud45d(z|s, x)\u000d\u000a%\u000d\u000a= E\ud45e(z|s,x)\u000d\u000a$\u000d\u000alog \ud45e(z|s, x)\u000d\u000a\ud45d(z|x)\u000d\u000a%\u000d\u000a(7)\u000d\u000a\u2212 E\ud45e(z|s,x)\u000d\u000a$\u000d\u000alog \ud45d(s|z, x)\u000d\u000a%\u000d\u000a+ log \ud45d(s|x)","MLG 2020, Aug 24, 2020, San Deigo, CA Anker and Kj\u00e6r, et al.\u000d\u000aFigure 7: Comparison of embedded latent space for latent dimensions \ud43b = 2 and \ud43b = 8. 600 PDFs are converted to the latent\u000d\u000avariables through the prior network. The points are colored with respect to the number of atoms in the structure in the same\u000d\u000acolor as the ground truth structures shown in the legend.\u000d\u000aTable 3: Simulation parameters for the PDF data.\u000d\u000aQ (\u00c5\u22121) r (\u00c5) Vibrations (\u00c52)\u000d\u000aQmin 0 rmin 0 ADP 1\u000d\u000aQmax 30 rmax 30 delta2 2\u000d\u000aQdamp 0.04 rstep 0.1 \u2013 \u2013\u000d\u000aFigure 8: Ten sample reconstructions from the test set predicted by the CVAE model with latent space dimension\ud43b = 2\u000d\u000aand 11 satellites.\u000d\u000aNote the \"rst term is KL$\u000d\u000a\ud45e(z|s, x)||\ud45d(z|x)\u000d\u000a%\u000d\u000aand the \"rst two terms\u000d\u000aform a lower bound on the conditional likelihood. Thus,\u000d\u000aKL$\u000d\u000a\ud45e(z|s, x)||\ud45d(z|s, x)\u000d\u000a%\u000d\u000a= \u2212L\ud44f\u000d\u000a\"\u000d\u000a\ud45d(s|x)\u000d\u000a#\u000d\u000a+ log \ud45d(s|x) (8)\u000d\u000awhere\u000d\u000aL\ud44f\u000d\u000a\"\u000d\u000a\ud45d(s|x)\u000d\u000a#\u000d\u000a= E\ud45e(z|s,x)\u000d\u000a$\u000d\u000alog \ud45d(s|z, x)\u000d\u000a%\u000d\u000a\u2212 KL$\u000d\u000a\ud45e(z|s, x)||\ud45d(z|x)\u000d\u000a%\u000d\u000a(9)\u000d\u000aThe bound L\ud44f\u000d\u000a(\u000d\u000a\ud45d(s|x)\u000d\u000a)\u000d\u000ais equal to the conditional likelihood when\u000d\u000athe KL divergence between the variational density and the true\u000d\u000aposterior density is zero. This is essentially the reason why we can\u000d\u000amaximize a surrogate objective such as the bound on the conditional\u000d\u000alikelihood to indirectly minimize the KL divergence we set to in\u000d\u000aEq. (6).\u000d\u000aThe bound in Eq. (9) is a maximization objective. We choose to\u000d\u000aminimize the negative of this bound in the CVAE objective. In (4),\u000d\u000aeach of the densities are parameterised by neural networks re#ected\u000d\u000ain the subscripts {\ud719, \ud703,\ud713}, resulting in the \"nal objective:\u000d\u000aLCVAE = \u2212E\ud45e\ud719\u000d\u000a\"\u000d\u000alog \ud45d\ud703 (s|z, x)\u000d\u000a#\u000d\u000a+ KL\"\u000d\u000a\ud45e\ud719 (z|s, x)||\ud45d\ud713 (z|x)\u000d\u000a#\u000d\u000a.\u000d\u000aB LATENT SPACE COMPARISON\u000d\u000aIn order to test the e!ect of the latent space dimension, two CVAE\u000d\u000amodels using latent space dimensions of 2 and 8 were trained. Figure 7 shows the latent space of the trained models with \ud43b = [2, 8].\u000d\u000aFor both latent space dimensions, 600 PDFs were input to the trained\u000d\u000aprior network to obtain 600 latent variables. The 8-dimensional\u000d\u000aspace was reduced to 2 dimensions by the use of t-SNE dimensionality reduction method. [33] The points were colored according to\u000d\u000athe number of atoms in the structure, to visually emphasize how\u000d\u000athe latent space clusters the structures. The reconstructions from\u000d\u000athe model using latent space dimension 8 is slightly better than of a\u000d\u000amodel with latent space dimension 2. Figure 7 shows that the latent\u000d\u000aspace is equally compact with latent space dimension 2 and 8, but\u000d\u000at-SNE makes it challenging to directly compare the compactness of\u000d\u000athe two latent spaces. Also, it is possible that the latent space with\u000d\u000adimension 8 captures some chemical details that is not present in\u000d\u000athe 2-dimensional space, however, we have not been able to identify\u000d\u000ait. In the end, we choose to use a smaller latent space dimension\u000d\u000aof 2 which could be bene\"cial when interpolating in the latent\u000d\u000aspace as demonstrated in Figure 1. However, if the goal is simply\u000d\u000ato reconstruct structures from the PDF a larger latent space or the\u000d\u000aDAE model can be utilized.\u000d\u000aC SIMULATION PARAMETERS FOR PDF\u000d\u000aMono-metallic nanoparticle structures for each of the atoms in\u000d\u000athe d-block of the periodic table (period 4-6) were generated. All\u000d\u000ananoparticles were built to have the FCC structure. The size and\u000d\u000ashape of each nanoparticle were controlled by parameters used\u000d\u000ain the ASE framework: [1] The number of atoms is given by the\u000d\u000anumber of layers, as counted from the center of the particles. The\u000d\u000ashape is de\"ned by specifying Miller indices for the exposed facets\u000d\u000aof the particles. Each structure was generated with layers ranging\u000d\u000afrom 2 to 8 and with exposed (100), (110) and (111) facets on the\u000d\u000asurface. For each con\"guration of metal, layers and surfaces, 10\u000d\u000astructures were obtained using lattice constants equally spaced in\u000d\u000athe range of 99% to 101% of the ideal lattice constant based on the\u000d\u000aatomic metallic radius\u2019s which were obtained from the mendeleev\u000d\u000apython package. [35] This process yielded 3137 unique structures\u000d\u000aconsisting of fewer than 100 atoms.","\u0009 |\u0009Generative\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a138\u000d\u000a4.1.2 Paper\u0009VIII\u000d\u000aOn\u0009 the\u0009 following\u0009 pages,\u0009 you\u0009 will\u0009 find\u0009 Paper\u0009 VIII\u0009 which\u0009 is\u0009 published\u0009 in\u0009 Digital\u0009 Discovery.\u0009 The\u0009\u000d\u000asupplementary\u0009information\u0009for\u0009Paper\u0009VIII\u0009can\u0009be\u0009found\u0009in\u0009APPENDICES\u0009\u2013\u0009PAPERS","Digital\u000d\u000a Discovery\u000d\u000arsc.li/digitaldiscovery\u000d\u000aVolume 2\u000d\u000aNumber 1\u000d\u000aFebruary 2023\u000d\u000aPages 1-250\u000d\u000aISSN 2635-098X\u000d\u000aPAPER\u000d\u000aKj\u00e6r, Anker, Jensen et al.\u000d\u000aDeepStruc: towards structure solution from pair distribution \u000d\u000afunction data using deep generative models","DeepStruc: towards structure solution from pair\u000d\u000adistribution function data using deep generative\u000d\u000amodels\u2020\u000d\u000aEmil T. S. Kj\u00e6r, \u2021a Andy S. Anker, \u2021a Marcus N. Weng,a Simon J. L. Billinge, *bc\u000d\u000aRaghavendra Selvan*de and Kirsten M. \u00d8. Jensen *a\u000d\u000aStructure solution of nanostructured materials that have limited long-range order remains a bottleneck in\u000d\u000amaterials development. We present a deep learning algorithm, DeepStruc, that can solve a simple\u000d\u000amonometallic nanoparticle structure directly from a Pair Distribution Function (PDF) obtained from total\u000d\u000ascattering data by using a conditional variational autoencoder. We first apply DeepStruc to PDFs from\u000d\u000aseven different structure types of monometallic nanoparticles, and show that structures can be solved\u000d\u000afrom both simulated and experimental PDFs, including PDFs from nanoparticles that are not present in\u000d\u000athe training distribution. We also apply DeepStruc to a system of hcp, fcc and stacking faulted\u000d\u000ananoparticles, where DeepStruc recognizes stacking faulted nanoparticles as an interpolation between\u000d\u000ahcp and fcc nanoparticles and is able to solve stacking faulted structures from PDFs. Our findings\u000d\u000asuggests that DeepStruc is a step towards a general approach for structure solution of nanomaterials.\u000d\u000aIntroduction\u000d\u000aCrystallographic methods, such as single crystal and powder\u000d\u000adiffraction, have been foundational in the development of\u000d\u000afunctional materials over the past century. They yield atomicscale structural models for crystalline materials and allow\u000d\u000aestablishing the links between material structure and properties that are at the heart of materials development.1,2 However,\u000d\u000aother approaches for structure determination are needed for\u000d\u000ananostructured materials that have limited long-range order,\u000d\u000aand total scattering methods such as atomic pair distribution\u000d\u000afunction (PDF) analysis have become increasingly important\u000d\u000atools.3\u20137 Currently, PDF analysis is mainly done by \ue103tting\u000d\u000aa known starting model to an experimental PDF, a process\u000d\u000aknown as structure re\ue103nement. Recent developments in automated modelling8\u201310 have made it possible to extend the\u000d\u000asearched structural space, but identifying a model or solving\u000d\u000aa structure de novo from a PDF is still an enormous challenge.\u000d\u000aSo far, only highly symmetrical nanostructures such as the C60\u000d\u000abuckyball have been solved ab initio from a PDF.11\u201315 Determining the structure of less symmetrical nanostructures is\u000d\u000alimited by the lost information caused by PDF peak overlap,\u000d\u000awhich challenges the use of PDF for structure solution of more\u000d\u000acomplicated nanomaterials.\u000d\u000aAn approach to handle the challenges due to the information\u000d\u000abarrier in PDFs is to employ supervised machine learning (ML)\u000d\u000amethods that can learn from well-known PDF-structure pairs. In\u000d\u000athis work, we use deep generative models (DGMs). DGMs are\u000d\u000aa class of ML models that can estimate the underlying data\u000d\u000adistribution from a reasonably small set of training examples.16\u000d\u000aA well-known use case of DGMs is in the generation of synthetic\u000d\u000a\u2018deep-fake\u2019 images17,18 based on large datasets of real images.\u000d\u000aWe here train our DGM to identify new structure models by\u000d\u000atraining on known chemical structures. The DGM learns the\u000d\u000arelation between PDF and atomic structure, which enables it to\u000d\u000asolve monometallic nanoparticle structures, based on PDFs it\u000d\u000ahas not seen before and its learned chemical knowledge. While\u000d\u000adetermining a unique structure from a PDF is not always\u000d\u000aa solvable problem, as several different structures may give rise\u000d\u000ato identical PDFs, ML methods can still learn to capture the\u000d\u000arelationship between PDF and structure and thereby push the\u000d\u000aboundaries of nanostructure solution from PDF. When there is\u000d\u000anot enough information in the PDF to provide a unique structure solution, ML methods may provide a distribution of starting models which can aid in further structure analysis.\u000d\u000aWe apply our DGM, which we refer to as \u2018DeepStruc\u2019, for\u000d\u000astructural analysis of a model system of monometallic\u000d\u000aa\u000d\u000aDepartment of Chemistry and Nano-Science Center, University of Copenhagen, 2100\u000d\u000aCopenhagen \u00d8, Denmark. E-mail: kirsten@chem.ku.dk\u000d\u000ab\u000d\u000aDepartment of Applied Physics and Applied Mathematics Science, Columbia\u000d\u000aUniversity, New York, NY 10027, USA. E-mail: sb2896@columbia.edu\u000d\u000ac\u000d\u000aCondensed Matter Physics and Materials Science Department, Brookhaven National\u000d\u000aLaboratory, Upton, NY 11973, USA\u000d\u000ad\u000d\u000aDepartment of Computer Science, University of Copenhagen, 2100 Copenhagen \u00d8,\u000d\u000aDenmark. E-mail: raghav@di.ku.dk\u000d\u000ae\u000d\u000aDepartment of Neuroscience, University of Copenhagen, 2200, Copenhagen N,\u000d\u000aDenmark\u000d\u000a\u2020 Electronic supplementary information (ESI) available. See DOI:\u000d\u000ahttps://doi.org/10.1039/d2dd00086e\u000d\u000a\u2021 Both authors contributed equally to this work.\u000d\u000aCite this: Digital Discovery, 2023, 2, 69\u000d\u000aReceived 16th August 2022\u000d\u000aAccepted 28th November 2022\u000d\u000aDOI: 10.1039/d2dd00086e\u000d\u000arsc.li/digitaldiscovery\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 69\u201380 | 69\u000d\u000aDigital\u000d\u000aDiscovery\u000d\u000aPAPER\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online\u000d\u000aView Journal | View Issue","nanoparticles (MMNPs) with seven different structure types\u000d\u000a(Fig. 1a) and demonstrate the method for both simulated and\u000d\u000aexperimental PDFs. DeepStruc is generative, which means that\u000d\u000ait can be used to construct structures that are not in the training\u000d\u000aset, i.e., solve a structure from a PDF. We demonstrate this\u000d\u000acapability on a dataset of face-centered cubic (fcc), hexagonal\u000d\u000aclosed packed (hcp) and stacking faulted structures, where\u000d\u000aDeepStruc can recognize the stacking faulted structures as an\u000d\u000ainterpolation between fcc and hcp and construct new structural\u000d\u000amodels based on a PDF.\u000d\u000aMethods\u000d\u000aIn the following sections, we brie\ue104y explain what a PDF is, how\u000d\u000awe obtained the simulated PDFs and their structures, and\u000d\u000a\ue103nally we elaborate on the CVAE method developed here to\u000d\u000aFig. 1 Training DeepStruc to determine the structure of MMNPs from PDFs. (a) DeepStruc predicts the xyz-coordinates of the MMNP structure\u000d\u000awith conditional input provided in the form of a PDF. The encoder uses the structure and its PDF as input while the prior only takes the PDF as\u000d\u000ainput. To obtain the structural output a latent space embedding is given as input to the decoder which produces the corresponding MMNP xyzcoordinates. During training of DeepStruc both the blue and green regions are used, while only the green region is used for structure prediction\u000d\u000aduring the inference process. (b) Examples of the seven different structure types which are used as input to DeepStruc together with their (c)\u000d\u000asimulated PDFs used as conditioning in DeepStruc. Each structure type has been included in the training set with varying sizes of 5 to 200 atoms\u000d\u000aand with varying lattice constants. The 3743 structures were split into training- (60%), validation- (20%), and testing sets (20%).\u000d\u000a70 | Digital Discovery, 2023, 2, 69\u201380 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","analyse PDFs. A more detailed description of the PDF is given\u000d\u000aelsewhere.19\u000d\u000aThe pair distribution function (PDF)\u000d\u000aThe PDF is the Fourier transform of total scattering data, which\u000d\u000acan be obtained through X-ray, neutron, or electron scattering.\u000d\u000aIn this work we focus on the usage of X-ray total scattering data.\u000d\u000aThe scattering vector Q is de\ue103ned as follows, where l is the\u000d\u000aradiation wavelength, and q is the scattering angle:\u000d\u000aQ \u00bc 4p sin\u00f0q\u00de\u000d\u000al\u000d\u000aThe measured scattering intensities are denoted I(Q), which\u000d\u000aare corrected for incoherent scattering, \ue104uorescence, etc. and\u000d\u000anormalized such that the total scattering structure function S(Q)\u000d\u000ais obtained.\u000d\u000aS\u00f0Q\u00de \u00bc\u000d\u000aI\u00f0Q\u00de $ D\u000d\u000af\u00f0Q\u00de\u000d\u000a2\u000d\u000aE\u000d\u000a\u00fe\u000d\u000aD\u000d\u000af\u00f0Q\u00de\u000d\u000aE2\u000d\u000aD\u000d\u000af\u00f0Q\u00de\u000d\u000aE2\u000d\u000aHere f is the atomic form factor. To obtain the structural realspace information, the total scattering structure function is\u000d\u000aFourier transformed over the truncated Q-range, hence yielding\u000d\u000athe reduced PDF also known as G(r):\u000d\u000aG\u00f0r\u00de \u00bc 2=p\u000d\u000a\u00f0Qmax\u000d\u000aQmin\u000d\u000aQ\u00bdS\u00f0Q\u00de $ 1' sin\u00f0Qr\u00dedQ\u000d\u000aG(r) can be interpreted as a histogram of real-space interatomic distances and the information is equivalent to that of an\u000d\u000aunassigned distance matrix (uDM). All PDF simulation parameters can be found in Section G in the ESI.\u2020 The PDFs used in\u000d\u000athis project are normalised to have max(G(r)) = 1 as illustrated\u000d\u000ain Section H in the ESI.\u2020\u000d\u000aSimulated and experimental data\u000d\u000aTo simulate the nanoparticles used in the training process of\u000d\u000aDeepStruc, the Python library atomic simulation environment\u000d\u000a(ASE) was used.20 The seven different structure types: fcc, bcc, sc,\u000d\u000ahcp, icosahedral, decahedral, and octahedral were constructed\u000d\u000awith the cluster module in ASE in the same manner as described\u000d\u000aby Banerjee et al.9 and Anker & Kj\u00e6r et al.21 All MMNPs were\u000d\u000agenerated in sizes ranging from 5 to 200 atoms. Each MMNP\u000d\u000awas then populated with different atoms hence changing the\u000d\u000alattice spacing/bond distances in the MMNP. To ensure that\u000d\u000athere were no duplicate MMNPs within the dataset, all MMNPs\u000d\u000awere decomposed into a distance list of all atom\u2013atom\u000d\u000adistances. The distance lists are a reduced format of the xyz\u000d\u000arepresentation as they are rotation- and translation-invariant in\u000d\u000aEuclidean space. All the distance-lists were sorted and duplicate\u000d\u000astructures with equivalent distance lists were removed. This\u000d\u000ayielded a total of 3742 unique MMNPs, see Section A in the ESI\u2020\u000d\u000afor the distribution of the seven structure types. The xyzcoordinates will be the label that DeepStruc must reconstruct.\u000d\u000aNanoparticles with each of the seven structure types can be seen\u000d\u000ain Fig. 1b along with their simulated PDF, Fig. 1a. All the\u000d\u000asimulation parameters used can been seen in Section G in the\u000d\u000aESI.\u2020\u000d\u000aTo further investigate the latent space behaviour of DeepStruc, a more chemically simple and intuitive dataset was made\u000d\u000aof fcc, hcp, and stacking faulted structures. Fcc and hcp can be\u000d\u000aconsidered layered structures that are only differentiated by the\u000d\u000arepetition of layers within the structure. Fcc consists of\u000d\u000aa repeated ABCABC layered structure where hcp is an ABABAB\u000d\u000alayered structure. A 5 layered stacking fault structure could then\u000d\u000abe described as ABCAC, as it does not satisfy either of the fcc or\u000d\u000ahcp stacking criteria. A total of 1620 stacking fault structures\u000d\u000awere generated.\u000d\u000aData representation\u000d\u000aIn this work, the structures from ASE are converted into a graphbased representation in order to capture the interatomic relationships, as the original representation generated with ASE are\u000d\u000anot optimal as input to DeepStruc. Graph representations have\u000d\u000aseen increasing success in machine learning applications\u000d\u000arelated to materials science as the interatomic relations in\u000d\u000agraphs are invariant to transformations of the structure such as\u000d\u000asolid translations and rotations.22,23 Each structure in graph\u000d\u000arepresentation can be described as G = (X,A), where X \u02db RN\u00d7F is\u000d\u000athe node feature matrix which contains F features that can\u000d\u000adescribe each of the N atoms in the structure. We use F = 3\u000d\u000acomprising only the Euclidean coordinates of the atom in a 3-\u000d\u000adimensional space. The interatomic relationships are captured\u000d\u000ausing the adjacency matrix A \u02db RN\u00d7N. In our case, the entries of\u000d\u000athe adjacency matrix are the Euclidean distance between each\u000d\u000apair of atoms, resulting in a so\ue09d adjacency matrix. However, to\u000d\u000amake the adjacency matrix sparse, when the distance between\u000d\u000aany pair of nodes is larger than the lattice constant the corresponding edge weight is set to zero. When the edge weight is\u000d\u000azero this corresponds to absence of an edge between the pair of\u000d\u000anodes, and in other cases the edges have a weight given by the\u000d\u000ainteratomic distance. Section I in the ESI\u2020 shows a decahedron\u000d\u000aconsisting of seven atoms alongside the components describing\u000d\u000ait in our chosen graph representation.\u000d\u000aThe conditional deep generative model (DGM)\u000d\u000aDGMs such as variational autoencoders (VAEs) are commonly\u000d\u000aused to synthesize novel, synthetic data by approximating the\u000d\u000aunderlying data-generating processes based on the training\u000d\u000adata.24 In this work, we are interested in generating structures\u000d\u000abased on properties such as the PDF resulting in the conditional\u000d\u000aDGM scenario. The speci\ue103c formulation of the conditional\u000d\u000aDGM used in this work is the CVAE, initially proposed for\u000d\u000acomputer vision tasks25 and more recently it has also been\u000d\u000aexplored for synthesizing novel drug molecules.26 The CVAE in\u000d\u000athis work is trained to solve the unassigned distance geometry\u000d\u000aproblem27 (uDGP) as it solves the task of converting the\u000d\u000adistances within a PDF to a chemical structure. In the uDGP the\u000d\u000aproblem of taking a starting point of a list of distances and\u000d\u000areconstructing it into a structure is broken down into two\u000d\u000adiscrete problems. First, is to discover the graph that connects\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 69\u201380 | 71\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","pairs of atoms, with the edges labelled by the distances from the\u000d\u000adistance list (the assignment problem). Second is to embed this\u000d\u000agraph into Euclidian space. An illustration of the CVAE can be\u000d\u000aseen in Fig. 1a. Here, the blue area is the training process, and\u000d\u000athe green area is the prediction/inference process. During\u000d\u000atraining of the CVAE, the encoder takes pairs of structures and\u000d\u000atheir corresponding PDFs as input. The encoder learns to map\u000d\u000athe structure-PDF pairs into a low-dimensional, latent Gaussian\u000d\u000adistribution, known as the encoder distribution. Each structurePDF pair is mapped to certain regions of the latent space. When\u000d\u000atrained with large amounts of diverse data, the latent space is\u000d\u000aable to capture relationships between different structures and\u000d\u000aPDF pairs so that similar structures are closer in this latent\u000d\u000aspace than very different structures. CVAEs are different from\u000d\u000aclassical autoencoders in that the latent space is probabilistic,\u000d\u000awhich makes it possible to sample structures from these latent\u000d\u000aencoder distributions. This is achieved during training by\u000d\u000aforcing the posterior and prior distributions to align. The prior\u000d\u000adistribution is generated with a much simpler network than\u000d\u000athat of the posterior and its only input is a PDF. The two\u000d\u000adistributions are matched by minimizing the Kullback\u2013Leibler\u000d\u000adivergence between the encoder and prior distributions and is\u000d\u000ainterpreted as the regularization term, Lreg.\u000d\u000aThe prior NN gets the PDF as input and maps it to the lowdimensional prior distribution. The low-dimensional latent\u000d\u000avector conditioned on the PDF is then input to the decoder,\u000d\u000awhich is tasked to predict the xyz-coordinates of the structural\u000d\u000ainput. During the training process, the mean squared error\u000d\u000a(MSE) between the xyz-coordinates of the input and output are\u000d\u000acomputed to force the decoder to predict xyx-coordinates from\u000d\u000athe latent representations. The MSE is de\ue103ned as the reconstruction loss, Lrec. The CVAE is trained by jointly optimizing\u000d\u000athese two loss components:\u000d\u000aLCVAE = Lrec + bLrecg\u000d\u000awhere b is a scaling factor that controls the relative in\ue104uence of\u000d\u000athe regularization- and reconstruction-terms. In our training\u000d\u000aprocess, at initialization b is set to 0 which allows the model to\u000d\u000afocus on minimizing Lrec. Each time Lrec gets below a certain\u000d\u000athreshold b is increased. This helps keep the model from falling\u000d\u000ainto a local minimum and the process is repeated until\u000d\u000aconvergence has been reached. Similar strategies for annealing\u000d\u000ab in VAEs have been attempted.28,29 At inference (test) time, the\u000d\u000aprior NN receives the PDF as input which is then mapped to the\u000d\u000alow-dimensional latent space which during training has been\u000d\u000atrained to match the encoder distribution. A sufficiently well\u000d\u000atrained CVAE is then able to predict structures from the latent\u000d\u000aspace based on the PDF input. A simpli\ue103ed version of the CVAE\u000d\u000aused for this work, DeepStruc, can be seen in Fig. 1a. The CVAE\u000d\u000ais presented more formally in our earlier work.21\u000d\u000aGraph conditional variational autoencoder (CVAE)\u000d\u000aIn this work, two types of CVAEs were utilized depending on the\u000d\u000atype of encoder. In the conventional CVAE, the encoder was\u000d\u000abased on multi-layered perceptrons which operate on a tabular\u000d\u000aformat of the node features, and the adjacency matrix populated\u000d\u000awith atom\u2013atom distances. For the second type of CVAE \u2013 that we\u000d\u000acall the graph CVAE \u2013 the encoder consists of a graph neural\u000d\u000anetwork (GNN)23,30 and is able to process graph structured data,\u000d\u000ataking the neighbourhood information into consideration. GNNs\u000d\u000aare generalized message passing methods that can aggregate\u000d\u000ainformation from the neighbourhood of a node by passing\u000d\u000amessages along the edges. These messages are learned during\u000d\u000atraining and can summarize the information present at the node\u000d\u000anecessary for the downstream tasks. Further, by making the\u000d\u000aencoder deep, i.e. adding additional GNN layers, nodes can get\u000d\u000aaccess to information from nodes that are farther from them. For\u000d\u000ainstance, in a k-layered GNN each node had access to information from nodes that are k-hops away. In our experiments, we\u000d\u000aobserved that the generative capabilities of the graph CVAE was\u000d\u000abetter than the conventional CVAE, part E in the ESI.\u2020 Further, we\u000d\u000awere able to obtain comparable reconstruction quality from the\u000d\u000agraph CVAE with only two latent dimensions compared to using\u000d\u000aeight dimensions for the conventional CVAE. This indicates that\u000d\u000athe graph encoder is able to better compress the information\u000d\u000apresent in the node and adjacency matrices. A minor technical\u000d\u000adetail in our CVAE models is that the predictions from the\u000d\u000adecoder do not exactly match the input features. That is, the\u000d\u000adecoder does not reconstruct the full input comprising node\u000d\u000afeatures and adjacency matrix but only the node features. The\u000d\u000aalgorithm we refer to as DeepStruc is a graph based CVAE.\u000d\u000aResults and discussion\u000d\u000aTraining DeepStruc to determine the structure of MMNPs\u000d\u000afrom PDF data\u000d\u000aDeepStruc, illustrated in Fig. 1a and discussed below, is a graphbased conditional variational autoencoder (graph CVAE).\u000d\u000aAutoencoders are a class of deep learning (DL) methods where\u000d\u000ahigh-dimensional inputs, such as chemical structures,21,26 are\u000d\u000areduced in dimensionality. The transformation into 2 or 3\u000d\u000adimensional vectors is achieved using an information bottleneck by an encoder neural network (NN),21,31,32 and the resulting\u000d\u000alower-dimensional, compressed feature space is known as the\u000d\u000alatent space. A decoder NN can reconstruct the input from these\u000d\u000alow-dimensional representations. When the latent space is\u000d\u000aregularized (smoothed) using normal distributions instead of\u000d\u000adiscrete points we obtain a variational autoencoder (VAE). We\u000d\u000ahave previously demonstrated that VAEs does a better job\u000d\u000ainterpolating in the latent space compared to deterministic\u000d\u000aAEs.21 The VAE can be made to be dependent (conditioned) on\u000d\u000aadditional information by the prior NN resulting in a CVAE.32\u000d\u000aWe here use MMNP structures (Fig. 1b) as input, and condition them on their simulated PDFs (Fig. 1c). The MMNP structures span seven different structure types computed using\u000d\u000aa variety of metals to emulate the variability in bond lengths in\u000d\u000areal metallic nanoparticle samples. The structure types are\u000d\u000asimple cubic (sc), body-centered cubic (bcc), face-centered cubic\u000d\u000a(fcc), hexagonal closed packed (hcp), decahedral, icosahedral, and\u000d\u000aoctahedral, and all structure types have been constructed in sizes\u000d\u000afrom 5 to 200 atoms. We used 3743 MMNP structures, which were\u000d\u000arandomly split into training- (60%), validation- (20%) and testingsets (20%). Note that the validation and test sets are derived from\u000d\u000a72 | Digital Discovery, 2023, 2, 69\u201380 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","the same underlying data distribution as the training set, and\u000d\u000aserve as intermediaries to the actual test set which is based on the\u000d\u000aexperimental PDF data. A histogram of the distribution of the\u000d\u000aseven structure types are provided in Section A in the ESI.\u2020\u000d\u000aDuring the training process (blue + green region Fig. 1a), DeepStruc learns to map the conditioning PDFs to their structures in\u000d\u000athe latent space. A\ue09der the training process is complete, DeepStruc can be used on data that have not been part of the training\u000d\u000aset, which is referred to as \u2018inference\u2019. Further details about the\u000d\u000aDeepStruc network can be found in the Method section.\u000d\u000aMapping of structures in a latent space\u000d\u000aWe \ue103rst evaluate DeepStruc's ability to map the MMNP structures in a low-dimensional latent space by investigating\u000d\u000astructural trends and clustering. Fig. 2 shows a visualization of\u000d\u000athe two-dimensional latent space with selected MMNP reconstructions indicated. The colour of the points indicates the\u000d\u000astructure type, and the relative point size indicates the size of\u000d\u000athe MMNP cluster. We observe that DeepStruc learns to map the\u000d\u000achemical structures in the latent space by size and symmetry. It\u000d\u000amaps the cubic structure types (sc, bcc, and fcc) together, and it\u000d\u000alearns that the octahedral MMNPs are closely related to the fcc\u000d\u000astructure type. Interestingly, DeepStruc also allocates the decahedral structures to be in between the fcc and hcp structures.\u000d\u000aThis can be rationalized by considering that decahedral structures are constructed from \ue103ve tetrahedrally shaped fcc crystals\u000d\u000awhich are separated by {111} twin boundaries that resemble\u000d\u000astacking faults.9,33,34 The twin boundaries will resemble stacking\u000d\u000aFig. 2 The two-dimensional latent space with structure reconstructions. The points in the latent space correspond to a structure and its\u000d\u000asimulated PDF. Data points from the test set are shown in solid colour and outlined. The points from the training and validation sets are shown as\u000d\u000asemi-transparent. The size of the points relates to the size of the embedded MMNP, and the orange background indicates the general size\u000d\u000aincrease throughout the latent space. The colour of each point resemblances its structure type, fcc (light blue), octahedral (dark grey), decahedral\u000d\u000a(orange), bcc (green), icosahedral (dark blue), hcp (pink), and sc (red). Note that the structures shown here are predicted by DeepStruc during\u000d\u000ainference on PDFs from the test set.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 69\u201380 | 73\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","faulted regions of fcc justifying that they exist in the latent space\u000d\u000abetween fcc and hcp.\u000d\u000aDeepStruc for structure determination from PDF\u000d\u000aWe now move on to identify structures directly from a PDF. The\u000d\u000aresults of using DeepStruc on seven simulated PDFs of MMNPs\u000d\u000anot used in the training process are illustrated in Fig. 3. Here,\u000d\u000awe show the structure that the input PDF was calculated from\u000d\u000a(le\ue09d), the reconstructed structure (right), and its agreement with\u000d\u000athe input PDF a\ue09der structure re\ue103nement (middle, discussed\u000d\u000abelow). In all seven cases, the structures are correctly reconstructed from the PDF input. Before structure re\ue103nement, the\u000d\u000amean absolute error (MAE) of the atom positions is 0.128 \u00b1\u000d\u000a0.073 A as described in Section B in the ESI. \u02da \u2020 However, the MAE\u000d\u000ais arti\ue103cially high due to a common aberration by DeepStruc,\u000d\u000awhere it predicts the right geometric atomic arrangement, but\u000d\u000aisotropically contracted or expanded compared to the original\u000d\u000astructure. We do not yet understand why DeepStruc has this\u000d\u000aaberration, but it is easily solvable by re\ue103ning an expansion/\u000d\u000acontraction variable as a post processing step to DeepStruc.\u000d\u000aA\ue09der re\ue103ning the structure to the PDF35 by \ue103tting a contraction/\u000d\u000aexpansion factor, a scale factor and an isotropic atomic\u000d\u000adisplacement parameter (ADP), as described in Section B in the\u000d\u000aESI,\u2020 the MAE of the atom positions is reduced to 0.093 \u00b1 0.058\u000d\u000aA. The inference is thus robust against moderate changes in \u02da\u000d\u000alattice parameter between a provided PDF and the structures\u000d\u000athat DeepStruc were trained on. The reconstructed structures\u000d\u000aexhibit some arti\ue103cial positional atomic disorder that broadens\u000d\u000athe PDF peaks. The \ue103tted ADP values (Section B in the ESI\u2020) are\u000d\u000athus lower than the ADP values of the conditioning PDFs.\u000d\u000aHaving established that DeepStruc works for structures\u000d\u000ahighly resembling those in the training set, we now consider\u000d\u000amore challenging cases and explore the capabilities of DeepStruc on an actual test set which is far from the training\u000d\u000adistribution. As described above, the largest structures in the\u000d\u000atraining set contained only 200 atoms.\u000d\u000aWe now evaluate it on a test set of simulated MMNPs with 5\u000d\u000ato 1000 atoms, i.e., containing much larger particles. The latent\u000d\u000aspace obtained from this new test set is plotted using diamond\u000d\u000amarkers in Fig. 4, where the latent space from the training\u000d\u000aprocess is shown with semi-transparent markers. We observe\u000d\u000athat the trends in the training area are comparable for the\u000d\u000atraining set and the test set of larger MMNPs. Notably, the\u000d\u000atrends of both the size and the structure types continue beyond\u000d\u000athe training area to structures containing about 400 atoms.\u000d\u000aBeyond 400 atoms, all structure types collapse onto a line,\u000d\u000ahowever, DeepStruc still provides a size estimate of the structure. Of course, DeepStruc could be retrained on a larger\u000d\u000atraining set if reconstructions are desired on clusters larger\u000d\u000athan 200 atoms. However, this experiment shows that DeepStruc can extrapolate signi\ue103cantly in the latent space. It can\u000d\u000athereby give useful information about PDFs from structures not\u000d\u000arepresented in the training set and is generative in a meaningful\u000d\u000away. This can be compared to, for example, a tree-based MLclassi\ue103er, which is limited to a prede\ue103ned structural database\u000d\u000aand cannot extrapolate. The capability of DeepStruc to extrapolate arises from each structure in the latent space being predicted as a normal distribution instead of a discrete point.\u000d\u000aIn practice, DeepStruc must be able to yield valid reconstructed structures from experimental data that contain noise\u000d\u000aand other aberrations. We therefore use DeepStruc to infer\u000d\u000astructures from previously published experimental PDFs from\u000d\u000aMMNPs. Fig. 5a shows the latent space with the predicted\u000d\u000alocation of structures from three experimental PDFs. Here, the\u000d\u000alocation in the latent space is represented as distributions\u000d\u000aFig. 3 Structure determination from PDFs. Simulated PDFs (grey) from\u000d\u000athe original structures of the seven different structure types (left) are\u000d\u000aused during inference for structure prediction (right). The middle\u000d\u000acolumn shows the fitted PDFs of the predicted structures to the\u000d\u000asimulated PDFs of the original structures. Only the scale-factor,\u000d\u000acontraction/expansion-factor, and ADP are refined, see Section B in\u000d\u000athe ESI.\u2020\u000d\u000a74 | Digital Discovery, 2023, 2, 69\u201380 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","rather than as discrete points, and multiple structures are\u000d\u000asampled from each distribution and compared to the experimental PDF to select the best candidate. The mean of the\u000d\u000aexperimental PDF distributions is represented as a black diamond with three ellipsoids indicating different con\ue103dence\u000d\u000aintervals with s: 3, 5 and 7, where s is the standard deviation of\u000d\u000athe normal distribution.\u000d\u000aThe \ue103rst experimental dataset that we evaluate was published by Jensen et al.,\u000d\u000a36 who identi\ue103ed a decahedral structure as\u000d\u000athe core motif of Au144(p-MBA)60 nanoparticles. DeepStruc\u000d\u000alocates the Au144(p-MBA)60 PDF (Fig. 5b) in a decahedral region\u000d\u000a(orange distributions in Fig. 5a) in the latent space. Given the\u000d\u000agenerative capabilities of DeepStruc, in theory, we can sample\u000d\u000aan unlimited number of structures for a given PDF. As described\u000d\u000ain Section D of the ESI,\u2020 we here sampled up to 1000 structures\u000d\u000afrom the three normal distributions (s: 3, 5, and 7), and\u000d\u000acompared their \ue103t to the experimental PDF. Fig. 5b shows the \ue103t\u000d\u000aof the best structural prediction, which was among the structures sampled from the s: 3 distributions. DeepStruc predicts\u000d\u000aa decahedral structure, which agrees well with the literature.36\u000d\u000aOther structures sampled from the three distributions are\u000d\u000ashown in Section E of the ESI,\u2020 where we also compare the\u000d\u000aDeepStruc analysis to baseline methods. We \ue103rst consider\u000d\u000aa brute-force structure-mining method inspired by Banerjee\u000d\u000aet al.,\u000d\u000a37 but also compare the DeepStruc results to two simpler\u000d\u000aML-algorithms, namely a tree-based ML classi\ue103er and a regular\u000d\u000aCVAE without a graph-based input.\u000d\u000aThe second dataset that we evaluate, published by Quinson\u000d\u000aet al.,\u000d\u000a38 are from 1.8 nm Pt nanoparticles with the fcc structure\u000d\u000a(described further in Section C in the ESI\u2020). This size\u000d\u000acorresponds to ca. 203 atoms, i.e. the number of atoms in the\u000d\u000aparticle goes slightly beyond the fcc structures in the training set\u000d\u000athat contain only 165 atoms.38 The location of the predicted\u000d\u000amean is again shown as a black diamond in Fig. 5a, enclosed by\u000d\u000athree blue ellipsoids illustrating different magnitudes of standard deviation. The mean of the predicted structure is placed\u000d\u000anear the largest sc structures. If DeepStruc only favoured\u000d\u000asymmetry it would be placed directly on the fcc structures.\u000d\u000aInterestingly, DeepStruc does not purely favour size either, as it\u000d\u000adoes not position the PDF near the largest structures which are\u000d\u000ahcp structures of 200 atoms. Instead, we observe that DeepStruc\u000d\u000atakes both symmetry and size into account by placing the mean\u000d\u000apredicted structure adjacent to the largest sc structures containing 185 atoms. To identify the structure from the experimental PDF, we again sample 1000 structures from the s: 3, 5\u000d\u000aand 7 distributions. When \ue103tting these sampled structures to\u000d\u000athe dataset, we obtain the best \ue103t from an fcc structure of 146\u000d\u000aatoms that is visualized in Fig. 5c and which agrees with the\u000d\u000abaseline models (Section E in the ESI\u2020). DeepStruc thus identi\ue103es an fcc structure even though the size of the MMNP is\u000d\u000aoutside the training set distribution.\u000d\u000aWe also attempted to input PDFs from even larger fcc\u000d\u000ananoparticles, estimated to have diameters of 2.2 and 3.4 nm,\u000d\u000acorresponding to 371 and 1368 atoms, respectively (Section C in\u000d\u000athe ESI\u2020).38 Their positions in the latent space are shown in\u000d\u000aFig. 4 along with the 1.8 nm fcc nanoparticles using cross\u000d\u000amarkers labelled 1, 2, and 3 for increasing size. We observe that\u000d\u000athey follow the trend of the simulated fcc structures discussed\u000d\u000aabove: while it is possible to estimate both size and symmetry\u000d\u000afor the 2.2 nm particles through extrapolation, DeepStruc can\u000d\u000aFig. 4 DeepStruc applied on PDFs of structures up to 1000 atoms. Each point is coloured after its structure type, i.e. fcc (light blue), octahedral\u000d\u000a(dark grey), decahedral (orange), bcc (green), icosahedral (dark blue), hcp (pink), and sc (red). Each point in the latent space corresponds to\u000d\u000aa structure based on its simulated PDF. Test PDFs from structures up to 1000 atoms are plotted as diamond markers on top of the training and\u000d\u000avalidation data which are made semi-transparent. Note that the training set latent space is identical to that plotted in Fig. 2. DeepStruc has only\u000d\u000abeen trained on structures up to 200 atoms. Three experimental PDFs (shown in Section C in the ESI\u2020) obtained from differently sized fcc\u000d\u000ananocrystals estimated to contain 203 (cross marker 1), 371 (cross marker 2), and 1368 (cross marker 3) atoms are illustrated as purple cross\u000d\u000amarkers in the latent space.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 69\u201380 | 75\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","only estimate size for the 3.4 nm particle. We note that the size\u000d\u000acan be read from a PDF directly without any modelling.\u000d\u000aHowever, the ability of DeepStruc to predict structures on\u000d\u000aexperimental data beyond those in the training set is promising\u000d\u000afor future structure solution from PDF.\u000d\u000aWhile DeepStruc only has been trained on simple MMNPs, we\u000d\u000a\ue103nally evaluate it on a PDF from Au144(PET)60 nanoparticles,\u000d\u000aconsisting of an icosahedral core of 54 atoms surrounded by\u000d\u000aa rhombicosidodecahedron shell of 60 atoms (Fig. 5d and e).36,39\u000d\u000aWe show the predicted mean position of the structure with\u000d\u000aa black diamond enclosed by pink ellipsoids. DeepStruc positions\u000d\u000athe PDF in the hcp region of the latent space, and when sampling\u000d\u000a1000 structures from the distribution with s: 7, the best \ue103tting\u000d\u000astructures is an hcp structure with 40 atoms for the Au144(PET)60\u000d\u000ananoparticle (Fig. 5d). Similar structures are found when\u000d\u000asampling from the s: 3 and s: 5 distributions. However, the PDF\u000d\u000a\ue103t reveals that the reconstructed structure does not capture all\u000d\u000apeaks in the experimental PDF. When considering further the\u000d\u000alatent space, icosahedral structures are strongly underrepresented in our dataset (Section A in the ESI\u2020) which results in an\u000d\u000ainconsistency when placing icosahedral structures in the latent\u000d\u000aspace. DeepStruc is thus challenged when solving the icosahedral\u000d\u000acore structure of the nanoparticle. However, we observe that one\u000d\u000aof the test icosahedral structures is placed near the experimental\u000d\u000aPDF in latent space within the s: 5 distribution. Therefore, we\u000d\u000aagain try to sample 1000 structures by moving the mean of the s:\u000d\u000a3 distribution to the nearest cluster of icosahedral structures in\u000d\u000athe latent space, which are located right outside the s: 7 distribution. The best \ue103tting structure (Fig. 5e) captures all main peaks\u000d\u000aof the experimental PDF. Strategies for sampling of underrepresented structures is discussed further in Section D in the ESI.\u2020\u000d\u000aStructure determination from PDF: fcc, hcp, and stacking\u000d\u000afaulted nanoparticles\u000d\u000aTo obtain a deeper understanding of the latent space's behaviour, we investigate a dataset only containing fcc, hcp, and\u000d\u000astacking faulted structures. Fcc and hcp structures are distinguished by the stacking sequence of closed packed layers in\u000d\u000atheir structures: while fcc structures can be described by\u000d\u000aABCABC stacking, hcp structures have ABABAB stacking.\u000d\u000aStructures with other sequences are stacking faulted structures.\u000d\u000aFig. 5 Fitting experimental PDFs with structures obtained by DeepStruc. (a) The DeepStruc latent space showing predicted latent space positions\u000d\u000afor structures from three experimental PDFs. The predicted means are shown as diamond markers, which are enclosed by three rings, indicating\u000d\u000athe sampling regions for s: 3, 5, and 7. (b) PDF fit of the reconstructed structure from the Au144(p-MBA)60 PDF36 (c) PDF fit of the reconstructed\u000d\u000astructure from the 1.8 nm Pt nanoparticle PDF from Quinson et al.,\u000d\u000a38 (d) PDF fit of the reconstructed structure from the Au144(PET)60 PDF36 using\u000d\u000aa hcp structure. (e) PDF fit of the reconstructed structure from the Au144(PET)60 PDF36 using an icosahedral structure. Note that the test set\u000d\u000astructures shown here are the predicted structures from DeepStruc obtained during inference on experimental PDFs.\u000d\u000a76 | Digital Discovery, 2023, 2, 69\u201380 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","We hypothesize that stacking faulted structures can be\u000d\u000aconsidered an \u2018interpolation\u2019 in the discrete space between the\u000d\u000afcc and hcp structure type.40\u000d\u000aExamples of reconstructed fcc (blue), hcp (pink), and different\u000d\u000astacking faulted structures (purple) and their position in the new\u000d\u000alatent space are illustrated in Fig. 6a. The MMNPs cluster in size,\u000d\u000aFig. 6 Latent space and reconstructions of stacking faulted nanoparticles. (a) The latent space and reconstructed structures shown with their\u000d\u000astacking sequence. The structures are shown in two dimensions, and the size (number of atoms) in the third dimension is given as \u2018depth\u2019. The\u000d\u000asemi-transparent dots in the latent space represent the training and validation data, and the solid dots represent the test data. Fcc structures are\u000d\u000aplotted in blue, hcp in pink, and the stacking faulted structures in purple. The marker size represents the size of the structures. (b) Fits from\u000d\u000areconstructed structures from the test PDF from a fcc (ABCABC stacking), a hcp (ABABAB stacking), and two stacking faulted structures. The\u000d\u000aoriginal conditioning PDFs are shown in grey, while the PDFs of the generated structures are coloured according to their structure type. The\u000d\u000adifference curves are shown in green. The latent space is two-dimensional, hence allowing it to be directly visualized. Note that the test set\u000d\u000astructures shown here are the predicted structures obtained from DeepStruc during inference.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 69\u201380 | 77\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","whilst we also observe that fcc and hcp structures separate in the\u000d\u000alatent space. It is evident that the stacking faulted structures are\u000d\u000alocated in between the fcc and hcp structures in the latent space\u000d\u000aas hypothesized. It is chemically reasonable that they are positioned in this exact order based on their similarity to fcc and hcp.\u000d\u000aFor example, the structure with ABCABA layers, shown in Fig. 6\u000d\u000awith a purple star is structurally close fcc. We see that it is also\u000d\u000alocated closer to the fcc structures in the latent space. On the\u000d\u000aother hand, the structure with ABCBCB layers (marked as\u000d\u000aa purple diamond in Fig. 6) can be considered structurally more\u000d\u000aclosely related to hcp than fcc. DeepStruc places this structure\u000d\u000aadjacent to hcp structures of the same size in the latent space.\u000d\u000aDeepStruc can thus insert stacking faulted structures between fcc\u000d\u000aand hcp into the latent space in a chemically meaningful way.\u000d\u000aFig. 6b illustrates the \ue103ts of the reconstructed structures to\u000d\u000athe PDF data. The difference curves indicate that the predicted\u000d\u000aand true structures are very close to being identical, which is\u000d\u000asupported by the MAE of the atomic positions on 0.030 \u00b1 0.019\u000d\u000aA (Section F in the ESI \u02da \u2020). While disorder causes a broadening of\u000d\u000athe peaks, the disorder in the generated structures is minor and\u000d\u000astructures with distinct difference between the layers and in the\u000d\u000acorrect sequence can be reconstructed to a satisfying degree.\u000d\u000aThis is a promising result, showing that a graph-based CVAE\u000d\u000acan be used as a tool to determine the structure of stacking\u000d\u000afaulted nanoparticles from PDFs,41,42 which is a topic of significant current interest.43\u201347\u000d\u000aConclusions\u000d\u000aWe have shown the potential of using a DGM for structure\u000d\u000adetermination from simulated and experimental PDFs. Our\u000d\u000agraph-based CVAE algorithm, DeepStruc, provides valuable\u000d\u000ainformation through its latent space, as the MMNP structures\u000d\u000acluster based on symmetry and size in agreement with their\u000d\u000astructural chemistry. Using experimental data, the Au144(p-MBA)60\u000d\u000ananoparticle was determined to be decahedral, Pt nanoparticles\u000d\u000awere determined to be fcc and the Au144(PET)60 was determined to\u000d\u000ahave an icosahedral core structure, all in agreement with previous\u000d\u000aliterature. While these systems are relatively simple MMNPs, we\u000d\u000arecognise that there are more complex materials where the\u000d\u000ameasured PDF would not contain sufficient information to solve\u000d\u000athe structure. DeepStruc would then provide a distribution of\u000d\u000astarting models which can aid in the further structure analysis.\u000d\u000aOur approach is only restricted by the distribution of the\u000d\u000astructural training set. When DeepStruc is trained on fcc, hcp,\u000d\u000aand stacking faulted structures, it will locate the stacking faulted structures in between the fcc and hcp structures. This\u000d\u000asuggests a strategy for training DeepStruc models on different\u000d\u000achemical systems that also \u2018interpolate\u2019 from one to another\u000d\u000awhen this can be identi\ue103ed. DeepStruc does not yet provide\u000d\u000aa completely general structure solution approach, but gives\u000d\u000acritical insight into how DGMs can interact with structural and\u000d\u000adiffraction information to yield candidate structures and ultimately structure solutions.\u000d\u000aWe plan to implement DeepStruc as part of PDF-in-the-cloud\u000d\u000a(PDFitc.org),48 where the training data can gradually be\u000d\u000aexpanded over time. So far, the structures investigated are fairly\u000d\u000aordered and contain some symmetry, but in the future, we plan\u000d\u000ato expand DeepStruc to chemical systems with more atoms and\u000d\u000ahigher complexity such as metal oxide nanoparticles and alloys.\u000d\u000aCombining the PDF conditioning with data from\u000d\u000acomplimentary techniques could prove important for structure\u000d\u000adetermination of more complex systems. Such studies would\u000d\u000aboth enable structure determination from a combined\u000d\u000amodelling perspective, but it would also reveal fundamental\u000d\u000aaspects of the information content of the different datasets for\u000d\u000asolving structure problems.\u000d\u000aData availability\u000d\u000aCode for the baseline models and DeepStruc are available at:\u000d\u000ahttps://github.com/EmilSkaaning/DeepStruc, https://github.com/\u000d\u000aAndySAnker/Brute-force-PDF-modelling, https://github.com/\u000d\u000aAndySAnker/MetalFinder, https://github.com/AndySAnker/CVAE.\u000d\u000aAuthor contributions\u000d\u000aETSK and ASA contributed to all aspects of the paper. MNW\u000d\u000awrote the code associated to the tree-based classi\ue103er. SJLB, RS\u000d\u000aand KM\u00d8J supervised the project. All authors contributed to the\u000d\u000awriting of the manuscript.\u000d\u000aConflicts of interest\u000d\u000aThe authors declare no competing interests.\u000d\u000aAcknowledgements\u000d\u000aThis work is part of a project that has received funding from the\u000d\u000aEuropean Research Council (ERC) under the European Union's\u000d\u000aHorizon 2020 Research and Innovation Programme (grant\u000d\u000aagreement No. 804066). We are grateful to the Villum Foundation for \ue103nancial support through a Villum Young Investigator\u000d\u000agrant (VKR00015416). Funding from the Danish Ministry of\u000d\u000aHigher Education and Science through the SMART Lighthouse\u000d\u000ais gratefully acknowledged. We acknowledge support from the\u000d\u000aDanish National Research Foundation Center for High Entropy\u000d\u000aAlloy Catalysis (DNRF 149). Work in the Billinge group was\u000d\u000asupported by the U.S. National Science Foundation through\u000d\u000agrant DMREF-1922234.\u000d\u000aReferences\u000d\u000a1 W. I. F. David and K. Shankland, Acta Crystallogr., Sect. A:\u000d\u000aFound. Crystallogr., 2008, 64, 52\u201364.\u000d\u000a2 A. K. Cheetham and A. L. Goodwin, Nat. Mater., 2014, 13,\u000d\u000a760\u2013762.\u000d\u000a3 S. J. L. Billinge and M. G. Kanatzidis, Chem. Commun., 2004,\u000d\u000a7, 749\u2013760.\u000d\u000a4 C. A. Young and A. L. Goodwin, J. Mater. Chem., 2011, 21,\u000d\u000a6464\u20136476.\u000d\u000a5 T. L. Christiansen, S. R. Cooper and K. M. \u00d8. Jensen,\u000d\u000aNanoscale Adv., 2020, 2, 2234\u20132254.\u000d\u000a78 | Digital Discovery, 2023, 2, 69\u201380 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","6 H. Zhu, Y. Huang, J. Ren, B. Zhang, Y. Ke, A. K.-Y. Jen,\u000d\u000aQ. Zhang, X.-L. Wang and Q. Liu, Adv. Sci., 2021, 8, 2003534.\u000d\u000a7 S. J. L. Billinge and I. Levin, Science, 2007, 316, 561\u2013565.\u000d\u000a8 L. Yang, P. Juhas, M. W. Terban, M. G. Tucker and\u000d\u000aS. J. L. Billinge, Acta Crystallogr., Sect. A: Found. Crystallogr.,\u000d\u000a2020, 76, 395\u2013409.\u000d\u000a9 S. Banerjee, C.-H. Liu, K. M. O. Jensen, P. Juhas, J. D. Lee,\u000d\u000aM. Tofanelli, C. J. Ackerson, C. B. Murray and\u000d\u000aS. J. L. Billinge, Acta Crystallogr., Sect. A: Found. Crystallogr.,\u000d\u000a2020, 76, 24\u201331.\u000d\u000a10 T. L. Christiansen, E. T. S. Kj\u00e6r, A. Kovyakh, M. L. R\u00f6deren,\u000d\u000aM. H\u00f8j, T. Vosch and K. M. \u00d8. Jensen, J. Appl. Crystallogr.,\u000d\u000a2020, 53, 148\u2013158.\u000d\u000a11 P. Juh\u00b4as, D. M. Cherba, P. M. Duxbury, W. F. Punch and\u000d\u000aS. J. L. Billinge, Nature, 2006, 440, 655\u2013658.\u000d\u000a12 P. Juh\u00b4as, L. Granlund, P. M. Duxbury, W. F. Punch and\u000d\u000aS. J. Billinge, Acta Crystallogr., Sect. A: Found. Crystallogr.,\u000d\u000a2008, 64, 631\u2013640.\u000d\u000a13 P. Juhas, L. Granlund, S. R. Gujarathi, P. M. Duxbury and\u000d\u000aS. J. Billinge, J. Appl. Crystallogr., 2010, 43, 623\u2013629.\u000d\u000a14 M. J. Cliffe, M. T. Dove, D. Drabold and A. L. Goodwin, Phys.\u000d\u000aRev. Lett., 2010, 104, 125501.\u000d\u000a15 M. J. Cliffe and A. L. Goodwin, J. Phys.: Condens. Matter, 2013,\u000d\u000a25, 454218.\u000d\u000a16 K. Choudhary, B. DeCost, C. Chen, A. Jain, F. Tavazza,\u000d\u000aR. Cohn, C. WooPark, A. Choudhary, A. Agrawal,\u000d\u000aS. J. L. Billinge, E. Holm, S. P. Ong and C. Wolverton, npj\u000d\u000aComput. Mater., 2022, 8, 1\u201326.\u000d\u000a17 A. Razavi, A. Van den Oord and O. Vinyals, NIPS\u201919:\u000d\u000aProceedings of the 33rd International Conference on Neural\u000d\u000aInformation Processing Systems, 2019, vol. 1331, pp. 14866\u2013\u000d\u000a14876.\u000d\u000a18 T. Karras, S. Laine, M. Aittala, J. Hellsten, J. Lehtinen and\u000d\u000aT. Aila, IEEE Comput. Soc. Conf. Comput. Vis. Pattern\u000d\u000aRecogn., 2020, 8110\u20138119.\u000d\u000a19 T. Egami and S. J. L. Billinge, Underneath the Bragg Peaks,\u000d\u000aPergamon, 2012.\u000d\u000a20 A. Hjorth Larsen, J. J\u00f8rgen Mortensen, J. Blomqvist,\u000d\u000aI. E. Castelli, R. Christensen, M. Du\u0142ak, J. Friis,\u000d\u000aM. N. Groves, B. Hammer, C. Hargus, E. D. Hermes,\u000d\u000aP. C. Jennings, P. Bjerre Jensen, J. Kermode, J. R. Kitchin,\u000d\u000aE. Leonhard Kolsbjerg, J. Kubal, K. Kaasbjerg, S. Lysgaard,\u000d\u000aJ. Bergmann Maronsson, T. Maxson, T. Olsen, L. Pastewka,\u000d\u000aA. Peterson, C. Rostgaard, J. Schi\u00f8tz, O. Sch\u00fctt, M. Strange,\u000d\u000aK. S. Thygesen, T. Vegge, L. Vilhelmsen, M. Walter, Z. Zeng\u000d\u000aand K. W. Jacobsen, J. Phys.: Condens. Matter, 2017, 29,\u000d\u000a273002.\u000d\u000a21 A. S. Anker, E. T. S. Kj\u00e6r, E. B. Dam, S. J. L. Billinge,\u000d\u000aK. M. \u00d8. Jensen and R. Selvan, Proceedings of the 16th\u000d\u000aInternational Workshop on Mining and Learning with Graphs,\u000d\u000aMLG, 2020, DOI: 10.26434/chemrxiv.12662222.v1.\u000d\u000a22 M. M. Bronstein, J. Bruna, Y. LeCun, A. Szlam and\u000d\u000aP. Vandergheynst, IEEE Signal Process. Mag., 2017, 34, 18\u201342.\u000d\u000a23 F. Scarselli, M. Gori, A. C. Tsoi, M. Hagenbuchner and\u000d\u000aG. Monfardini, IEEE Trans. Neural Network., 2008, 20, 61\u201380.\u000d\u000a24 D. P. Kingma and M. Welling, 2013, arXiv preprint\u000d\u000aarXiv:1312.6114.\u000d\u000a25 K. Sohn, H. Lee and X. Yan, Adv. Neural Inf. Process. Syst.,\u000d\u000a2015, 28, 3483\u20133491.\u000d\u000a26 J. Lim, S. Ryu, J. W. Kim and W. Y. Kim, J. Cheminf., 2018, 10,\u000d\u000a1\u20139.\u000d\u000a27 P. M. Duxbury, L. Granlund, S. Gujarathi, P. Juhas and\u000d\u000aS. J. Billinge, Discrete Appl. Math., 2016, 204, 117\u2013132.\u000d\u000a28 H. Shao, Z. Xiao, S. Yao, D. Sun, A. Zhang, S. Liu, T. Wang,\u000d\u000aJ. Li and T. Abdelzaher, IEEE Trans. Pattern Anal. Mach.\u000d\u000aIntell., 2021, 44, 12.\u000d\u000a29 K. Rydhmer and R. Selvan, 2021, arXiv preprint\u000d\u000aarXiv:2102.05526.\u000d\u000a30 T. N. Kipf and M. Welling, 2016, arXiv preprint\u000d\u000aarXiv:1609.02907.\u000d\u000a31 A. M. Samarakoon, K. Barros, Y. W. Li, M. Eisenbach,\u000d\u000aQ. Zhang, F. Ye, V. Sharma, Z. L. Dun, H. Zhou,\u000d\u000aS. A. Grigera, C. D. Batista and D. A. Tennant, Nat.\u000d\u000aCommun., 2020, 11, 892.\u000d\u000a32 R. G\u00b4omez-Bombarelli, J. N. Wei, D. Duvenaud,\u000d\u000aJ. M. Hern\u00b4andez-Lobato, B. S\u00b4anchez-Lengeling,\u000d\u000aD. Sheberla, J. Aguilera-Iparraguirre, T. D. Hirzel,\u000d\u000aR. P. Adams and A. Aspuru-Guzik, ACS Cent. Sci., 2018, 4,\u000d\u000a268\u2013276.\u000d\u000a33 L. D. Marks, Philos. Mag. A, 1984, 49, 81\u201393.\u000d\u000a34 S. Banerjee, C.-H. Liu, J. D. Lee, A. Kovyakh, V. Grasmik,\u000d\u000aO. Prymak, C. Koenigsmann, H. Liu, L. Wang,\u000d\u000aA. M. M. Abeykoon, S. S. Wong, M. Epple, C. B. Murray\u000d\u000aand S. J. L. Billinge, J. Phys. Chem. C, 2018, 122, 29498\u201329506.\u000d\u000a35 P. Juhas, C. L. Farrow, X. Yang, K. R. Knox and\u000d\u000aS. J. L. Billinge, Acta Crystallogr., Sect. A: Found.\u000d\u000aCrystallogr., 2015, 71, 562\u2013568.\u000d\u000a36 K. M. \u00d8. Jensen, P. Juhas, M. A. Tofanelli, C. L. Heinecke,\u000d\u000aG. Vaughan, C. J. Ackerson and S. J. L. Billinge, Nat.\u000d\u000aCommun., 2016, 7, 11859.\u000d\u000a37 S. Banerjee, C.-H. Liu, K. Jensen, P. Juh\u00b4as, J. D. Lee,\u000d\u000aM. Tofanelli, C. J. Ackerson, C. B. Murray and S. J. Billinge,\u000d\u000aActa Crystallogr., Sect. A: Found. Crystallogr., 2020, 76, 24\u201331.\u000d\u000a38 J. Quinson, L. Kacenauskaite, T. L. Christiansen, T. Vosch,\u000d\u000aM. Arenz and K. M. \u00d8. Jensen, ACS Omega, 2018, 3, 10351\u2013\u000d\u000a10356.\u000d\u000a39 N. Yan, N. Xia, L. Liao, M. Zhu, F. Jin, R. Jin and Z. Wu, Sci.\u000d\u000aAdv., 2018, 4, eaat7259.\u000d\u000a40 F. Bertolotti, D. Moscheni, A. Migliori, S. Zacchini,\u000d\u000aA. Cervellino, A. Guagliardi and N. Masciocchi, Acta\u000d\u000aCrystallogr., Sect. A: Found. Crystallogr., 2016, 72, 632\u2013644.\u000d\u000a41 A. S. Masadeh, E. S. Bozin, C. L. Farrow, G. Paglia, P. Juhas,\u000d\u000aS. J. L. Billinge, A. Karkamkar and M. G. Kanatzidis, Phys.\u000d\u000aRev. B: Condens. Matter Mater. Phys., 2007, 76, 115413.\u000d\u000a42 X. Yang, A. S. Masadeh, J. R. McBride, E. S. Bo\u02c7zin,\u000d\u000aS. J. Rosenthal and S. J. L. Billinge, Phys. Chem. Chem.\u000d\u000aPhys., 2013, 15, 8480\u20138486.\u000d\u000a43 J. Cenker, S. Sivakumar, K. Xie, A. Miller, P. Thijssen, Z. Liu,\u000d\u000aA. Dismukes, J. Fonseca, E. Anderson, X. Zhu, X. Roy,\u000d\u000aD. Xiao, J.-H. Chu, T. Cao and X. Xu, Nat. Nanotechnol.,\u000d\u000a2022, 17, 256\u2013261.\u000d\u000a44 X. Rong, J. Liu, E. Hu, Y. Liu, Y. Wang, J. Wu, X. Yu, K. Page,\u000d\u000aY.-S. Hu, W. Yang, H. Li, X.-Q. Yang, L. Chen and X. Huang,\u000d\u000aJoule, 2018, 2, 125\u2013140.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 69\u201380 | 79\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","45 D. S. Charles, M. Feygenson, K. Page, J. Neuefeind, W. Xu\u000d\u000aand X. Teng, Nat. Commun., 2017, 8, 15520.\u000d\u000a46 P. Gao, P. Metz, T. Hey, Y. Gong, D. Liu, D. D. Edwards,\u000d\u000aJ. Y. Howe, R. Huang and S. T. Misture, Nat. Commun.,\u000d\u000a2017, 8, 14559.\u000d\u000a47 P. C. Metz, R. Koch and S. T. Misture, J. Appl. Crystallogr.,\u000d\u000a2018, 51, 1437\u20131444.\u000d\u000a48 L. Yang, E. A. Culbertson, N. K. Thomas, H. T. Vuong,\u000d\u000aE. T. S. Kjaer, K. M. O. Jensen, M. G. Tucker and\u000d\u000aS. J. L. Billinge, Acta Crystallogr., Sect. A: Found.\u000d\u000aCrystallogr., 2021, 77, 2\u20136.\u000d\u000a80 | Digital Discovery, 2023, 2, 69\u201380 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 28 November 2022. Downloaded on 5/2/2023 1:39:18 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","\u0009 |\u0009Generative\u0009Machine\u0009Learning\u0009for\u0009Experimental\u0009Scattering\u0009Data\u0009Analysis\u000d\u000a152\u000d\u000a4.2 Matching\u0009Experimental\u0009and\u0009Simulated\u0009Data*\u000d\u000aIn\u0009materials\u0009chemistry,\u0009we\u0009often\u0009employ\u0009large\u0009datasets\u0009of\u0009physics-based\u0009simulations\u0009to\u0009train\u0009supervised\u0009\u000d\u000aML\u0009models.\u0009While\u0009we\u0009can\u0009easily\u0009gather\u0009vast\u0009amounts\u0009of\u0009simulated\u0009data,\u0009they\u0009may\u0009not\u0009entirely\u0009capture\u0009\u000d\u000athe\u0009 complex\u0009 nature\u0009 of\u0009 real-world\u0009 experimental\u0009 data.\u0009 As\u0009 a\u0009 result,\u0009the\u0009 supervised\u0009 ML\u0009 models can\u0009 be\u0009\u000d\u000aconfounded\u0009by\u0009factors like background\u0009noise,\u0009instrumental\u0009effects,\u0009or\u0009other\u0009physical\u0009phenomena\u0009absent\u0009\u000d\u000ain\u0009physics-based\u0009simulations.\u0009For\u0009example,\u0009none\u0009of\u0009these\u0009factors\u0009are\u0009included\u0009in\u0009the\u0009Debye\u0009scattering\u0009\u000d\u000aequation\u0009(Equation\u00091.1).\u0009This\u0009raises\u0009a\u0009critical question:\u0009How\u0009can we\u0009navigate\u0009these\u0009constraints\u0009to\u0009analyse\u0009\u000d\u000aexperimental\u0009data\u0009effectively?\u0009\u000d\u000aGANs might\u0009provide\u0009the\u0009answer as\u0009these\u0009models\u0009can\u0009estimate\u0009the\u0009underlying\u0009data\u0009distribution of\u0009both\u0009\u000d\u000aexperimental\u0009and\u0009 simulated\u0009data.\u0009GANs\u0009are\u0009 trained\u0009by\u0009 setting\u0009up\u0009 two\u0009competing\u0009neural\u0009networks:\u0009a\u0009\u000d\u000agenerator\u0009and\u0009a\u0009discriminator.\u0009During\u0009competition\u0009(training),\u0009the\u0009generator\u0009learns\u0009to\u0009create\u0009synthetic\u0009\u000d\u000adata that resemble the\u0009data distribution.\u0009The\u0009discriminator\u0009then\u0009proceeds\u0009to\u0009categorise the\u0009synthetic\u0009\u000d\u000adata as either\u0009within or\u0009outside of\u0009the data distribution.\u0009This\u0009lets\u0009the\u0009generator\u0009know\u0009if\u0009its\u0009synthetic\u0009data\u000d\u000awas\u0009good\u0009enough\u0009to\u0009fool the\u0009discriminator.\u0009Consequently, the\u0009generator\u0009learns\u0009to create\u0009synthetic\u0009data\u000d\u000athat\u0009truly\u0009match\u0009the\u0009target\u0009data\u0009distribution.\u000d\u000a102\u0009\u0009\u000d\u000aIn\u0009Paper\u0009IX,\u0009we\u0009introduce \"Exp2SimGAN\",\u0009an\u0009unsupervised\u0009image-to-image\u0009translation\u0009GAN\u0009designed\u0009to\u0009\u000d\u000aconvert\u0009 between\u0009 simulated\u0009 and\u0009 experimental\u0009 INS\u0009 data.103 This\u0009 type\u0009 of\u0009 GAN\u0009 has recently\u0009 attracted\u0009\u000d\u000aattention\u0009 due\u0009 to\u0009 its\u0009 capacity\u0009 to\u0009 translate\u0009 information\u0009 between\u0009 two\u0009 domains\u0009 without\u0009 the\u0009 need\u0009 for\u0009\u000d\u000amatched\u0009 domain\u0009 pairs.104-108 As illustrated\u0009 in\u0009 Figure\u0009 4.4A,\u0009 the\u0009 Exp2SimGAN\u0009 model uses\u0009 a\u0009 dual-GAN\u0009\u000d\u000asetting.\u0009One\u0009GAN,\u0009Exp2Simnetwork,\u0009translates the\u0009experimental\u0009data\u0009to\u0009simulated\u0009data,\u0009and\u0009another\u0009GAN,\u0009\u000d\u000aSim2Expnetwork,\u0009 performs\u0009 the\u0009 opposite\u0009 translation.\u0009 Furthermore,\u0009 Exp2SimGAN\u0009 employs\u0009 contrastive\u0009\u000d\u000alearning,\u0009a\u0009 technique\u0009 that\u0009ensures the\u0009synthetic\u0009data\u0009produced\u0009accurately\u0009reflects\u0009the\u0009 features\u0009of\u0009 the\u000d\u000aspecific\u0009 input\u0009 dataset\u0009 within the\u0009 data\u0009 distribution.\u0009 It\u0009 also\u0009 uses\u0009 an\u0009 auxiliary\u0009 loss\u0009 function,\u0009 adding\u0009\u000d\u000arobustness\u0009against\u0009noisy\u0009data. A\u0009more in-depth\u0009discussion\u0009of\u0009the\u0009Exp2SimGAN model can\u0009be\u0009found in\u0009\u000d\u000aPaper\u0009IX.\u0009\u000d\u000aWe\u0009 trained\u0009 Exp2SimGAN\u0009 on computationally\u0009 expensive\u0009 simulated\u0009 resolution\u0009 convolved\u0009 and\u0009\u000d\u000acomputationally\u0009 cheap\u0009 resolution\u0009 unconvolved\u0009 INS\u0009 spectra.\u0009INS\u0009is\u0009 a\u0009 spectroscopy\u0009 technique\u0009 used\u0009 to\u0009\u000d\u000ainvestigate the\u0009 vibrational\u0009 and\u0009magnetic\u0009 properties\u0009 of\u0009 atoms\u0009in\u0009materials.\u0009Here,\u0009 the\u0009 INS\u0009 data\u0009 serves\u0009\u000d\u000aprimarily\u0009to\u0009demonstrate\u0009that\u0009the\u0009modelling\u0009methodologies\u0009discussed\u0009in\u0009Chapters\u00092-4\u0009can\u0009be\u0009effectively\u0009\u000d\u000aextended\u0009 to\u0009 spectroscopy\u0009 data.\u0009 After\u0009 training,\u0009 Exp2SimGAN achieves visually\u0009 accurate results\u0009\u000d\u000aperforming\u0009both\u0009the\u0009convolution and\u0009deconvolution of the\u0009resolution\u0009function\u0009from\u0009both\u0009simulated\u00092D\u000d\u000a(as\u0009 illustrated\u0009 in\u0009 Figure\u0009 4.4B)\u0009 and\u0009 3D\u0009 INS\u0009 spectra.\u0009 Moreover,\u0009 the\u0009 use\u0009 of\u0009 Exp2SimGAN\u0009 to\u0009 transform\u0009\u000d\u000aexperimental\u0009data\u0009into\u0009a\u0009form\u0009that\u0009is\u0009resembling\u0009simulated\u0009data\u0009shows\u0009beneficial\u0009results.\u0009Specifically,\u0009a\u0009\u000d\u000aclassification\u0009ML\u0009model\u0009 trained\u0009on\u0009simulated\u0009data\u0009demonstrates\u0009improved\u0009accuracy\u0009when\u0009applied\u0009 to\u0009\u000d\u000athis\u0009transformed\u0009data,\u0009compared\u0009to\u0009its\u0009performance\u0009on\u0009the\u0009raw experimental\u0009data. We\u0009also present\u0009a\u0009\u000d\u000aunique\u0009 way\u0009 to\u0009 measure Exp2SimGAN's\u0009 applicability\u0009 domain,\u0009 which\u0009 is\u0009 a\u0009 vital precaution\u0009 to\u0009 prevent\u0009\u000d\u000amisuse\u0009and\u0009blind\u0009applications\u0009of\u0009the\u0009model.\u0009Our\u0009approach\u0009involves\u0009the\u0009computation\u0009of\u0009the\u0009Wasserstein\u0009\u000d\u000adistance,\u0009 a\u0009 concept\u0009 from\u0009 optimal\u0009 transport\u0009 theory\u0009 often\u0009 used\u0009 to\u0009 compare\u0009 different\u0009 probability\u0009\u000d\u000adistributions.109 In\u0009this\u0009case,\u0009we\u0009calculate\u0009the\u0009Wasserstein distance\u0009in\u0009the Exp2SimFeaturespace between\u0009the\u0009\u000d\u000adistribution\u0009of\u0009our\u0009training\u0009data\u0009and\u0009that\u0009of\u0009the\u0009data\u0009under\u0009investigation.\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009IX."," \u000d\u000a153\u000d\u000aIn\u0009 conclusion,\u0009 this\u0009 work\u0009 provides\u0009 a\u0009method\u0009 to\u0009make\u0009 physics-based\u0009 simulations\u0009 accurately\u0009 resemble\u0009\u000d\u000aexperimental\u0009data,\u0009which\u0009can\u0009then\u0009be\u0009used\u0009as\u0009training\u0009data\u0009for\u0009supervised\u0009ML\u0009models.\u0009Consequently,\u0009the\u0009\u000d\u000acomplex\u0009 nature\u0009 of\u0009 real-world\u0009 experimental\u0009 data can\u0009 be\u0009 captured\u0009into\u0009 the\u0009 training\u0009 set.\u0009 The\u0009 opposite\u0009\u000d\u000atransition\u0009of\u0009experimental\u0009data\u0009 to\u0009simulated\u0009data\u0009is\u0009also\u0009possible,\u0009enabling\u0009 the\u0009use\u0009of\u0009supervised\u0009ML\u0009\u000d\u000amodels,\u0009 trained\u0009 on\u0009 conventional\u0009 physics-based\u0009 simulations,\u0009 on\u0009 real-world\u0009 experimental\u0009 data.\u0009 Both\u0009\u000d\u000aapproaches open new\u0009 opportunities for\u0009 more\u0009 efficient\u0009 and\u0009 accurate\u0009 analysis\u0009 of\u0009 experimental\u0009 data.\u000d\u000aFurthermore,\u0009this\u0009Chapter\u0009demonstrate\u0009that\u0009the\u0009modelling\u0009methodologies\u0009discussed\u0009in\u0009Chapters\u00092-4\u0009can\u0009\u000d\u000abe\u0009effectively\u0009extended\u0009to\u0009spectroscopy\u0009data.\u0009\u000d\u000aFigure\u00094.4:\u0009Training\u0009Exp2SimGAN\u0009to\u0009translate\u0009between\u0009the\u0009domains\u0009of\u0009simulated\u0009and\u0009experimental\u0009data.\u0009(A)\u0009During\u0009\u000d\u000atraining,\u0009Exp2SimGAN\u0009learns\u0009to\u0009translate\u0009between\u0009experimental\u0009(left)\u0009and\u0009simulated\u0009(right)\u0009data\u0009by\u0009using\u0009a\u0009dual\u0009\u000d\u000asetting\u0009 setup\u0009 translating\u0009 from\u0009 experimental\u0009 to\u0009 simulated\u0009 data\u0009 (orange)\u0009 and vice\u0009 versa (blue)\u0009 in\u0009 two\u0009 different\u0009\u000d\u000anetworks, Exp2Simnetwork (orange)\u0009 and\u0009 Sim2Expnetwork (blue),\u0009 simultaneously.\u0009 Here\u0009 shown\u0009 for\u0009 INS\u0009 data.\u0009\u000d\u000aExp2SimGAN\u0009uses\u0009contrastive\u0009learning\u0009to\u0009ensure\u0009similarity\u0009between\u0009patches\u0009of\u0009white\u2013white\u0009pairs\u0009and\u0009dissimilarity\u0009\u000d\u000abetween\u0009 patches\u0009 of\u0009 white\u2013grey\u0009 pairs.\u0009 After\u0009 training,\u0009 Sim2Expnetwork and\u0009 Exp2Simnetwork can\u0009 be\u0009 used\u0009\u000d\u000aindependently\u0009to\u0009convert\u0009simulated\u0009data\u0009to\u0009experimental\u0009data\u0009or vice\u0009versa.\u0009Note\u0009that\u0009Exp2SimGAN\u0009does\u0009not\u0009need\u0009\u000d\u000athe\u0009corresponding\u0009image-to-image\u0009datasets,\u0009and\u0009it\u0009can\u0009be\u0009used\u0009on\u0009any\u0009shape\u0009of\u00092\u20133D\u0009data.\u0009 (B)\u0009An\u0009example\u0009of\u0009a\u0009\u000d\u000atranslation\u0009between\u0009simulated\u0009and\u0009experimental\u0009data\u0009using\u0009Exp2SimGAN.\u0009Figure\u0009and\u0009Figure\u0009caption\u0009are adapted\u0009\u000d\u000afrom\u0009Paper\u0009IX.103\u000d\u000a4.2.1 Paper\u0009IX\u000d\u000aOn\u0009 the\u0009 following\u0009 pages,\u0009 you\u0009 will\u0009 find\u0009 Paper\u0009 IX\u0009 which\u0009 is\u0009 published\u0009 in\u0009 Digital\u0009 Discovery.\u0009 The\u0009\u000d\u000asupplementary\u0009information\u0009for\u0009Paper\u0009IX\u0009can\u0009be\u0009found\u0009in\u0009APPENDICES\u0009\u2013\u0009PAPERS.","Digital\u000d\u000a Discovery\u000d\u000arsc.li/digitaldiscovery\u000d\u000aVolume 2\u000d\u000aNumber 3\u000d\u000aJune 2023\u000d\u000aPages 533-884\u000d\u000aISSN 2635-098X\u000d\u000aPAPER\u000d\u000aAndy S. Anker, Keith T. Butler et al.\u000d\u000aUsing generative adversarial networks to match \u000d\u000aexperimental and simulated inelastic neutron scattering data","Using generative adversarial networks to match\u000d\u000aexperimental and simulated inelastic neutron\u000d\u000ascattering data\u2020\u000d\u000aAndy S. Anker, *a Keith T. Butler, \u2021*b Manh Duc Le, c Toby G. Perringc\u000d\u000aand Jeyan Thiyagalingam b\u000d\u000aSupervised machine learning (ML) models are frequently trained on large datasets of physics-based\u000d\u000asimulations with the aim of being applied to experimental data. However, ML models trained on\u000d\u000asimulated data often struggle to perform on experimental data, because there is a shift in the data\u000d\u000acaused by experimental effects that might be challenging to simulate. We introduce Exp2SimGAN, an\u000d\u000aunsupervised image-to-image ML model to match simulated and experimental data. Ideally, training\u000d\u000aExp2SimGAN only requires a set of experimental data and a set of (not necessarily corresponding)\u000d\u000asimulated data. Once trained, it can convert a simulated dataset into one that resembles an experiment,\u000d\u000aand vice versa. We trained Exp2SimGAN on simulated resolution convolved and unconvolved INS\u000d\u000aspectra. Consequently, Exp2SimGAN can perform a resolution convolution and deconvolution of\u000d\u000asimulated two- and three-dimensional INS spectra. We demonstrate that this is sufficient for\u000d\u000aExp2SimGAN to match simulated and experimental INS data, enabling the analysis of experimental INS\u000d\u000adata using supervised ML, which was previously not possible. Finally, we provide a domain of application\u000d\u000ameasure for Exp2SimGAN, allowing us to assess the likelihood that Exp2SimGAN will be successful on\u000d\u000aa specific dataset. Exp2SimGAN is a step towards the analysis of experimental data using supervised ML\u000d\u000amodels trained on physics-based simulations.\u000d\u000aIntroduction\u000d\u000aDuring the past few decades, research in materials science has\u000d\u000abeen accelerated by the rapid development of synchrotron and\u000d\u000aneutron sources.1 Conventional data analysis approaches that\u000d\u000ainvolve signi\ue103cant human input and control cannot keep up\u000d\u000awith the growing size of measured datasets. Moreover,\u000d\u000aincreasingly sophisticated theory and simulations are being\u000d\u000aused to understand experimental data, the resource requirements for these simulations are considerably demanding, and\u000d\u000amay not be affordable for every possible case.1\u20134 As a result, data\u000d\u000aanalysis o\ue09den becomes a bottleneck in many areas of materials\u000d\u000ascience research.2,3 Therefore, it is crucial to advance the\u000d\u000acurrent state-of-the-art for materials science data analysis, and\u000d\u000aa particularly promising avenue is to exploit recent developments in arti\ue103cial intelligence and machine learning (ML).2\u20134\u000d\u000aAn outstanding issue for applying ML in many areas of\u000d\u000anatural science is the scarcity of labelled data. For example,\u000d\u000aadvances in computer vision and natural language processing\u000d\u000awere predicated on the existence of large, high-quality labelled\u000d\u000adatasets, such as ImageNet.5 For problems in natural science,\u000d\u000aobtaining data labels is o\ue09den much more challenging than for\u000d\u000asocio-economically focused datasets, such as street scenes. For\u000d\u000aproblems in natural science, however, one o\ue09den has access to\u000d\u000aphysics-based models to simulate the phenomena of interest.\u000d\u000aThese physics-based models allow (within computation\u000d\u000aconstraints) the generation of large, labelled datasets for\u000d\u000atraining models. Some recent examples include the Materials\u000d\u000aProject6 and JARVIS databases.7 However, models trained on\u000d\u000asimulated data o\ue09den struggle to work on the analysis of experimental data, because there is a shi\ue09d in the data, introduced by\u000d\u000aexperimental artefacts such as noise and instrument resolution.\u000d\u000aThis has been a particular problem for the analysis of inelastic\u000d\u000aneutron scattering (INS). INS is a powerful technique for\u000d\u000aprobing and understanding the dynamic behaviour of\u000d\u000acondensed matter, and has been important for understanding\u000d\u000adiverse properties such as charge and thermal transport8 and\u000d\u000amore exotic phenomena such as heavy fermions, high\u000d\u000aa\u000d\u000aNano-Science Center and Department of Chemistry, University of Copenhagen,\u000d\u000aDenmark. E-mail: andy@chem.ku.dk\u000d\u000ab\u000d\u000aScienti\ue103c Computing Department, Rutherford Appleton Laboratory, England, UK.\u000d\u000aE-mail: k.butler@qmul.ac.uk\u000d\u000ac\u000d\u000aISIS Neutron and Muon Source, Rutherford Appleton Laboratory, England, UK\u000d\u000a\u2020 Electronic supplementary information (ESI) available. See DOI:\u000d\u000ahttps://doi.org/10.1039/d2dd00147k\u000d\u000a\u2021 Current address: School of Engineering and Materials Science, Queen Mary\u000d\u000aUniversity of London, Mile End Rd, London E1 4NS, England, UK.\u000d\u000aCite this: Digital Discovery, 2023, 2, 578\u000d\u000aReceived 22nd December 2022\u000d\u000aAccepted 15th March 2023\u000d\u000aDOI: 10.1039/d2dd00147k\u000d\u000arsc.li/digitaldiscovery\u000d\u000a578 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital\u000d\u000aDiscovery\u000d\u000aPAPER\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online\u000d\u000aView Journal | View Issue","temperature superconductivity, topological insulators and spin\u000d\u000aliquids.9\u201312\u000d\u000aIt is o\ue09den the case that information present in the INS data\u000d\u000acannot be extracted directly, but relies on careful comparison to\u000d\u000athe predictions from physical models. The analysis of INS\u000d\u000adatasets frequently involves \ue103tting physics-based models, for\u000d\u000aexample, based on spin wave theory,13 to experimental spectra.\u000d\u000aO\ue09den one is interested in extracting the optimal parameters for\u000d\u000aa given magnetic Hamiltonian by comparison of simulated and\u000d\u000aexperimental INS spectra. Typically, this is achieved by\u000d\u000aa combination of good intuition about reasonable parameters\u000d\u000aalong with local optimization of the parameters to achieve the\u000d\u000abest \ue103t of the simulated to the experimental spectrum. This\u000d\u000aapproach has been successfully applied in numerous studies;\u000d\u000ahowever, it does suffer from a number of limitations.\u000d\u000aThe simulation of spectra with realistic instrument effects\u000d\u000acan be computationally demanding so \ue103tting may neglect\u000d\u000aexperimental artefacts that can originate from dead pixels,\u000d\u000aother detector artefacts, phenomena caused by multiple scattering from the sample and nearby instrument components\u000d\u000a(colloquially known as \u2018spurions\u2019), and signals originating from\u000d\u000athe sample alone but arising from phenomena not included in\u000d\u000athe theory underlying the simulation. Additionally, it is\u000d\u000acomputationally expensive to perform the convolution of the\u000d\u000asimulation and spectrometer resolution function to ensure that\u000d\u000athe simulated data correctly mimic the experimental signal and\u000d\u000abackground.14 In recent years, there has been a concerted effort\u000d\u000ain the INS community to develop methods to properly account\u000d\u000afor the effects of resolution broadening of experimental signals\u000d\u000aand experimental artefacts in order to better match simulated\u000d\u000ato measured data.15\u201324\u000d\u000aIn this work, we seek to improve this situation using ML to\u000d\u000ahelp us effectively analyze neutron datasets. Our approach is\u000d\u000ainspired by work in unpaired image-to-image translation, using\u000d\u000aML to match two domains of data.25\u201328 Ideally, training requires\u000d\u000aa set of experimental data and a set of (not necessarily corresponding) simulated data. As a proof of principle, we train on\u000d\u000acomputationally expensive simulated resolution convolved and\u000d\u000acomputationally cheap resolution unconvolved INS spectra.\u000d\u000aMore speci\ue103cally, we develop generative adversarial networks\u000d\u000a(GANs) that can perform a resolution convolution and deconvolution of simulated two- and three-dimensional INS spectra.\u000d\u000aWe show that our approach successfully matches and can\u000d\u000aconvert between simulated and experimental INS data. We\u000d\u000ademonstrate that a classi\ue103cation ML model trained on simulated data without resolution convolution performs badly on\u000d\u000aexperimental data. However, using our GAN to perform\u000d\u000aa \u201cresolution deconvolution\u201d of the experimental data (hence\u000d\u000amaking them similar to the simulated data), the classi\ue103cation\u000d\u000amodel can make accurate predictions on the experimental data.\u000d\u000aWe note that the results would possibly be further improved\u000d\u000ausing a set of experimental data and a set of simulated data for\u000d\u000atraining, which would allow the GAN to additionally account for\u000d\u000aphenomena not included in the theory underlying the simulation. Finally, we provide a domain of application measure for\u000d\u000aour GAN, demonstrating that our technique possesses not only\u000d\u000apredictive capacity but also a measure of the success probability\u000d\u000aof the model. Our approach is trained in a patch-wise manner\u000d\u000aon the input data, which means that the framework can \ue104exibly\u000d\u000aextend to different sizes of data. So, for example, the method\u000d\u000acould be used to perform inference on a much larger experimental spectrum than any included in the training set. This\u000d\u000a\ue104exibility is critical to ensure that the approach applies across\u000d\u000avaried data.\u000d\u000aExp2SimGAN and previous work\u000d\u000aThe Exp2SimGAN framework, illustrated in Fig. 1 and discussed\u000d\u000abelow, is a GAN. GANs are a class of unsupervised deep learning\u000d\u000a(DL) methods that can be used to create realistic synthetic\u000d\u000ainstances of a target dataset from either noise or a conditional\u000d\u000ainput. GANs are trained by setting up two competing neural\u000d\u000anetworks (NNs), the generator and discriminator. During\u000d\u000acompetition (training), the generator learns to create synthetic\u000d\u000ainstances similar to the target distribution, which the discriminator classi\ue103es to be either in or out of the target distribution.\u000d\u000aThus the generator gets better at creating synthetic instances\u000d\u000athat truly match the target dataset. The process is optimized via\u000d\u000aa minmax loss;\u000d\u000aL Exp2Sim\u000d\u000aGAN \u00f0G; D; X; Y\u00de \u00bc \u0395y$Y \u00bdlog\u00f0D\u00f0y\u00de\u00de& \u00fe \u0395x$X \u00bdlog\u00f01\u000d\u000a( D\u00f0G\u00f0x\u00de\u00de\u00de&;\u000d\u000awhere G is the generator (Exp2Simnetwork in Fig. 1), D is the\u000d\u000adiscriminator (DSim in Fig. 1), Ex is the expected value over all X\u000d\u000ainstances (experimental data in Fig. 1), and Ey is the expected\u000d\u000avalue over all Y instances (simulated data in Fig. 1). A similar\u000d\u000aL Sim2Exp\u000d\u000aGAN is used for the simulated to experimental data\u000d\u000aconversion. The GAN network is further explained elsewhere.29\u000d\u000aA class of GANs, CycleGANs, have, in recent years, attracted\u000d\u000asigni\ue103cant attention due to their ability to translate information\u000d\u000abetween two domains in an unsupervised setting, i.e. without\u000d\u000amatched domain pairs.27,28,30 A popular example is to translate\u000d\u000abetween 2D images of horses and zebras, where a GAN is used to\u000d\u000atranslate a horse to a zebra and back again.25,26,28 While GANs\u000d\u000ahave been applied a few times in materials science,31\u201336\u000d\u000aunpaired image-to-image translation, which is frequently done\u000d\u000awith CycleGANs, has only been applied in few instances.37,38\u000d\u000aWhile CycleGANs are somewhat restricted in their applicability;25,26,39 recent papers have employed contrastive learning to\u000d\u000aensure similarity by teaching the network to ensure a degree of\u000d\u000astructural similarity between the corresponding patches in the\u000d\u000ainput and output images (white\u2013white pairs, Fig. 1) but not\u000d\u000anecessarily between non-corresponding patches of the input\u000d\u000aand output images (white\u2013grey pairs, Fig. 1).25,26 The process is\u000d\u000aoptimized via a patch-wise contrastive loss;\u000d\u000aL Exp2Sim\u000d\u000aPatchNCE\u00f0G; H; X\u00de \u00bc \u0395x$X\u000d\u000aXL\u000d\u000al\u00bc1\u000d\u000aXSl\u000d\u000as\u00bc1\u000d\u000a\u2018\u000d\u000a!\u000d\u000az^\u000d\u000as\u000d\u000al ; zs\u000d\u000al ; z\u000d\u000aS=s\u000d\u000al\u000d\u000a\"\u000d\u000a;\u000d\u000awhere H is a multilayer perceptron-based NN which extracts\u000d\u000afeatures that embed L layers in S spatial locations to an\u000d\u000aembedding space where the cross entropy loss,40 \u2018, is calculated\u000d\u000aon the patches. The de\ue103nitions of z and ^z are given as\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 578\u2013590 | 579\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","zl = Hl\u000d\u000aSim(Sim2Expl\u000d\u000aEncoder(y)),\u000d\u000aand\u000d\u000az^l = Hl\u000d\u000aSim(Sim2Expl\u000d\u000aEncoder(G(x)))\u000d\u000aA similar L Exp2Sim\u000d\u000aPatchNCE is used for the simulated to experimental\u000d\u000adata conversion. This is further explained elsewhere.25,26\u000d\u000aOur work extends on a novel algorithm taking advantage of\u000d\u000acontrastive learning for unpaired image-to-image translation:\u000d\u000adual contrastive learning GAN (DCLGAN).25 Exp2SimGAN\u000d\u000aexploits a dual-GAN setting, with one GAN, Exp2Simnetwork\u000d\u000aFig. 1 Training Exp2SimGAN to translate between the domains of simulated and experimental data. (a) During training, Exp2SimGAN learns to\u000d\u000atranslate between experimental (left) and simulated (right) data by using a dual setting setup translating from experimental to simulated data\u000d\u000a(orange) and vice versa (blue) in two different networks, Exp2Simnetwork (orange) and Sim2Expnetwork (blue), simultaneously. Exp2SimGAN also\u000d\u000auses contrastive learning to ensure similarity between patches of white\u2013white pairs and dissimilarity between patches of white\u2013grey pairs. After\u000d\u000atraining, Sim2Expnetwork and Exp2Simnetwork can be used independently to convert simulated data to experimental data or vice versa. Note that\u000d\u000aExp2SimGAN does not need the corresponding image-to-image datasets, and it can be used on any shape of 2\u20133D data. (b) An example of\u000d\u000aa translation between simulated and experimental data using Exp2SimGAN.\u000d\u000a580 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","(orange), that translates the experimental data to simulated\u000d\u000adata and another GAN, Sim2Expnetwork (blue), for the opposite\u000d\u000atranslation. DCLGAN also uses contrastive learning using\u000d\u000asimilar (white\u2013white) and dissimilar (white\u2013grey) patches, as\u000d\u000ashown in Fig. 1. It has been demonstrated that this setting is\u000d\u000ahighly versatile taking any input shapes while reconstructing\u000d\u000astate-of-the-art performance synthetic data.25 While DCLGAN is\u000d\u000alimited to images in 2D, our extension makes it possible to do\u000d\u000aunpaired instance-to-instance translation of 2D and 3D\u000d\u000ainstances. We call the model Exp2SimGAN.\u000d\u000aAdditionally, we have included an auxiliary loss that was\u000d\u000afound to help stabilize the training of small datasets with a high\u000d\u000asignal-to-noise ratio, a situation which is frequently encountered in the \ue103eld of materials science. The auxiliary loss is\u000d\u000ade\ue103ned as the mean square error (MSE) between the real class, Y\u000d\u000a(simulated data in Fig. 1), and the class predicted by the\u000d\u000adiscriminator network, D(G(x)) used on Exp2Simnetwork applied\u000d\u000aon the experimental data. This loss function is inspired by41,42\u000d\u000aL Exp2Sim\u000d\u000aauxiliary \u00f0G; D; x; Y\u00de \u00bc MSE\u00f0Y; D\u00f0G\u00f0x\u00de\u00de\u00de\u000d\u000aA similar L Exp2Sim\u000d\u000aauxiliary is used for the simulated to experimental\u000d\u000adata conversion.\u000d\u000aAs can be observed from Fig. 1, Exp2SimGAN learns to\u000d\u000atranslate between experimental (le\ue09d) and simulated (right) data.\u000d\u000aExp2SimGAN does not need corresponding pairs of instances\u000d\u000a(2\u20133D data) and Exp2SimGAN can be applied to any 2\u20133D data\u000d\u000ashape inputs, even those that differ from the training data,\u000d\u000awhile retaining the predictive power. The four NNs in the\u000d\u000aarchitecture of Exp2SimGAN can make it memory intensive for\u000d\u000atraining. Exp2Simnetwork or Sim2Expnetwork can, however, be\u000d\u000aused independently a\ue09der training for inference, freeing up RAM\u000d\u000afor considerably bigger input shapes. The Exp2SimGAN architecture is further explained in the Methods section.\u000d\u000aResults and discussion\u000d\u000aRemoving instrumental resolution from 2D INS data with\u000d\u000aExp2SimGAN and vice versa\u000d\u000aHere we demonstrate that Exp2SimGAN can be used to translate\u000d\u000abetween the domains of experimental-like and simulated 2D\u000d\u000aINS data by the convolution or deconvolution of the resolution\u000d\u000afunction. As a metric of success, we use a spin wave model\u000d\u000aclassi\ue103er trained on INS spectra, which some of the authors of\u000d\u000athis work have previously shown to work better on simulated\u000d\u000adata than on experimental data.14 Exp2SimGAN is used to\u000d\u000abridge this gap between simulated and experimental data.\u000d\u000aWe previously demonstrated that the resolution function\u000d\u000asigni\ue103cantly in\ue104uences how INS data are interpreted in ML\u000d\u000amodels.14 To distinguish between two parameterized spin wave\u000d\u000amodels (denoted \u2018Dimer\u2019 and \u2018Goodenough\u2019) of INS data,\u000d\u000aFig. 2a, measured on the half-doped bilayer manganite\u000d\u000aPr(Ca,Sr)2Mn2O7 (abbreviated as PCSMO), we applied a deterministic uncertainty quanti\ue103cation (DUQ) classi\ue103er.14,43 The\u000d\u000adata were arranged into a 2D representation with incident\u000d\u000aneutron energy (Ei) and bins of energy transfer on the axes. The\u000d\u000atwo spin wave models produce comparable but distinguishable\u000d\u000aINS spectra for equivalent parameters in their different\u000d\u000aHamiltonians. Fig. 2b illustrates the input data including and\u000d\u000aexcluding instrumental resolution. In ref. 14, the instrument\u000d\u000aFig. 2 Experimental and simulated INS data of the half-doped bilayer manganite Pr(Ca,Sr)2Mn2O7 in its spin, charge, and orbital ordered phase\u000d\u000a(PCSMO). 2D representation of (a) the experimental data of PCSMO measured at 4 K using the MAPS spectrometer.43 The INS spectra are\u000d\u000aarranged in terms of incident neutron energy (Ei\u000d\u000a) and bins of energy transfer u = 0.10\u20130.16Ei\u000d\u000a, etc. 2D representation of (b) resolution convolved\u000d\u000aand unconvolved simulated INS spectra using the Dimer and the Goodenough spin wave models. Note that the INS spectra are the same as in\u000d\u000aFig. 1a. The calculations are described in the Methods section.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 578\u2013590 | 581\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","resolution convolutions were calculated using the Horace\u000d\u000apackage.15 See the Methods section for more information about\u000d\u000aboth the experimental INS spectrum, Fig. 2a, and the simulated\u000d\u000aINS spectra, Fig. 2b. In ref. 14 it was demonstrated that a classi\ue103er trained on simulated INS data that did not include accurate resolution functions cannot be used to predict the spin\u000d\u000awave model from experimental INS data because the experimental data differs too much from the training set (simulated\u000d\u000aINS data).14 We evaluated a number of computationally inexpensive methods for resolution functions but found that these\u000d\u000awere not accurate enough, and had to use a more accurate but\u000d\u000acomputationally expensive Monte Carlo integration method to\u000d\u000acalculate the resolution convolution for the training data, in\u000d\u000aorder to obtain con\ue103dent predictions on the experimental\u000d\u000adata.44\u000d\u000aHere, we execute a resolution deconvolution from the\u000d\u000aexperimental 2D INS spectra to enable classi\ue103cation with an ML\u000d\u000amodel trained on computationally cheap (unconvolved)\u000d\u000asimulations.\u000d\u000aUsing the Dimer and Goodenough spin wave models, we\u000d\u000asimulated resolution convolved and unconvolved 2D INS\u000d\u000aspectra and trained Exp2SimGAN to conduct a resolution\u000d\u000aconvolution (Sim2Expnetwork) or deconvolution (Exp2Simnetwork)\u000d\u000aoperation (Fig. 3a). We use 80% of the data (training set) to train\u000d\u000athe networks, whereas the last 20% of the data (test set) are used\u000d\u000ato evaluate their performance (Fig. 3b). Finally, we use\u000d\u000aExp2SimGAN on an experimental INS spectrum (Fig. 3c). See\u000d\u000aSection A in the ESI\u2020 for more details about the INS 2D data\u000d\u000adistribution.\u000d\u000aFig. 4 graphically demonstrates the performance of\u000d\u000aExp2Simnetwork for translation from the simulated convolved to\u000d\u000aunconvolved 2D INS data. The unconvolved INS data have\u000d\u000asigni\ue103cantly sharper features and for some incident energies\u000d\u000athe intensity is faint. Exp2SimGAN learns both the deconvolution (Exp2Simnetwork, running le\ue09d to right in Fig. 1a) and\u000d\u000aconvolution (Sim2Expnetwork, right to le\ue09d in Fig. 1a) operations\u000d\u000a(Section B in the ESI\u2020 shows the GAN resolution convolution).\u000d\u000aFig. 1b additionally shows a zoom-in view of the Exp2SimGAN\u000d\u000atranslation in the Ei = 35 meV and u = 0.22\u20130.28Ei range. Note\u000d\u000athat Exp2SimGAN did not have the corresponding datasets to\u000d\u000ado the translation during training.\u000d\u000aThe training and evaluating process of Exp2SimGAN is\u000d\u000aillustrated in Fig. 3. We trained the DUQ classi\ue103er45 on 80% of\u000d\u000athe resolution unconvolved data and tested it on the remaining\u000d\u000a20% (test set data), obtaining an accuracy of 98.9% (de\ue103ned as\u000d\u000athe fraction of correct predictions) which can be seen as the\u000d\u000aground truth performance. We then see that by deconvolving\u000d\u000athe convoluted data using Exp2Simnetwork the classi\ue103cation\u000d\u000anetwork performs noticeably better (96.0%) than when applied\u000d\u000ato convoluted data (93.7%) (Fig. 3). This demonstrates that\u000d\u000aExp2Simnetwork has successfully deconvolved the 2D INS spectra\u000d\u000abut, in the meantime, retained important information to\u000d\u000adistinguish between the spin wave models. The accuracies in\u000d\u000aFig. 4 are for the DUQ classi\ue103er trained on simulated data\u000d\u000awithout resolution convolution.\u000d\u000aWe can also apply our model in the reverse direction,\u000d\u000aapplying Sim2Expnetwork, which performs a resolution convolution of the simulated data. We can thereby train the DUQ\u000d\u000aclassi\ue103er on simulated resolution convolved data achieving\u000d\u000aaccuracies of 98.6% on simulated data with full resolution\u000d\u000aconvolution, but only 71.8% when applied to simulated data\u000d\u000awithout resolution convolution. However, when we use\u000d\u000aSim2Expnetwork to do a resolution convolution of the unconvolved data, we achieve an accuracy of 93.6% (we achieve 75.1%\u000d\u000awhen we GAN-deconvolve the data). This show that our network\u000d\u000aFig. 3 Flowchart illustrating the training procedure of Exp2SimGAN/Sim2ExpGAN and how we use it on simulated and experimental data. (a)\u000d\u000aExp2SimGAN and Sim2ExpGAN are trained simultaneously using a set of simulated INS spectra with and without convolved resolution functions.\u000d\u000aThe DUQ classifier is only trained on the computationally cheap INS spectra, which are resolution function unconvolved. (b) We use the DUQ\u000d\u000aclassifier on the computationally expensive simulated INS spectra that is resolution convolved. These data mimic experimental data. We achieve\u000d\u000aa low accuracy in classifying the Dimer or Goodenough spin wave model. We now use Exp2SimGAN to perform a resolution deconvolution of\u000d\u000athe data before the DUQ classifier classifies them. It now achieves a higher accuracy. (c) The DUQ classifier cannot identify the spin wave model\u000d\u000aof an experimental dataset with high certainty. However, Exp2SimGAN matches the experimental dataset to the simulated training set of the\u000d\u000aDUQ classifier enabling the classification of the spin wave model with high certainty.\u000d\u000a582 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","can be used in either direction: from experimental to simulated\u000d\u000adata or from simulated to experimental data.\u000d\u000aWe have used a DUQ classi\ue103er to assign the simulated\u000d\u000aspectra to their respective spin wave models, as in previous work\u000d\u000afor classifying these data.14 The DUQ classi\ue103er outputs a correlation value between 0 and 1 to indicate the distance between\u000d\u000athe output classes and the weight vector associated with the\u000d\u000ainput. Values close to the extremes (0 and 1) are associated with\u000d\u000aa prediction of high certainty; 1 implies no distance from the\u000d\u000aclass centroid, while 0 implies a very large distance from the\u000d\u000aclass centroid, and values that are not close to any of the class\u000d\u000acentroids implies that the example is far from the training\u000d\u000adistribution and there is high uncertainty about the\u000d\u000aclassi\ue103cation.\u000d\u000aMatching experimental and simulated INS data in 2D\u000d\u000aWe now proceed and use the Exp2SimGAN approach to perform\u000d\u000aresolution deconvolution on a real experimental INS dataset\u000d\u000a(Fig. 3c) and predict the magnetic model with the DUQ classi-\u000d\u000a\ue103er. Fig. 5a shows the raw experimental PCSMO INS spectrum\u000d\u000aand the GAN deconvolved INS spectrum (upper right), where the\u000d\u000afeatures are signi\ue103cantly sharper in the GAN deconvolved INS\u000d\u000aspectrum. The DUQ classi\ue103er (trained on unconvolved simulated data) predicts the raw experimental PCSMO INS spectrum\u000d\u000ato be [Dimer: 0.19jGoodenough: 0.92] showing that the network\u000d\u000acorrectly predicts the Goodenough spin wave model but with\u000d\u000aa signi\ue103cant uncertainty as indicated by the relatively high\u000d\u000acorrelation value for the Dimer class. This contrasts with the\u000d\u000aGAN deconvolved INS spectrum, which is predicted to be\u000d\u000a[Dimer: 0.07jGoodenough: 0.99] where it also correctly infers\u000d\u000athe Goodenough spin wave model but with a signi\ue103cantly\u000d\u000asmaller uncertainty. We also apply the DUQ classi\ue103er on the\u000d\u000asame experimentally based negative control as our previous\u000d\u000awork that combines experimental INS spectra measured with\u000d\u000athe same instrumental settings (and hence instrumental resolution) on various different materials.14 The negative control has\u000d\u000aequivalent instrumental settings to the PCSMO dataset but\u000d\u000asigni\ue103cantly different Hamiltonians (further details in the\u000d\u000aMethods section). Both the raw-[Dimer: 0.61jGoodenough: 0.55]\u000d\u000aand the GAN cleaned negative control [Dimer:\u000d\u000aFig. 4 Evaluating Exp2SimGAN on simulated 2D INS spectra from the test set. The INS data are split into an 80% training set and 20% test set.\u000d\u000aAfter the network has trained on the training set, we apply it on the test set data. Here is shown an example of performing resolution deconvolution on 2D INS spectra simulated with the Dimer spin wave model and an example using the Goodenough spin wave model. Note that the\u000d\u000aexperimental axis is the same as in Fig. 1a. The highlighted accuracies are the performance of the DUQ classifier, trained on simulated INS spectra\u000d\u000awithout resolution convolution, on the test set. Section B in the ESI\u2020 shows the results of conducting the GAN convolution. If the DUQ classifier is\u000d\u000atrained on simulated resolution convolved INS spectra the accuracies are 93.6% applied to GAN-convolved data and 71.8% applied to simulated\u000d\u000adata without resolution convolution.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 578\u2013590 | 583\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","0.31jGoodenough: 0.84] provide values far away from the\u000d\u000aextremes (0 and 1), which shows that the data are out of the\u000d\u000atraining set distribution and that Exp2SimGAN does not alter\u000d\u000athe data to describe the Dimer or Goodenough spin wave model.\u000d\u000aWe have demonstrated that Exp2SimGAN can successfully\u000d\u000aconvert between simulated and experimental 2D INS spectra, by\u000d\u000aadding or removing the effect of resolution broadening and\u000d\u000anoise associated with experiments in the case of INS spectra for\u000d\u000aPCSMO. Knowing when the experimental dataset is beyond the\u000d\u000atraining distribution and, consequently, the GAN's useable\u000d\u000adomain is important when thinking about real-world applications. Inspired by the Fr\u00b4echet inception distance (FID) score,46\u000d\u000awhich quanti\ue103es similarities of real and generated images by\u000d\u000ausing feature vectors from the inception v3 model, we can use\u000d\u000aExp2SimFeaturespace (shown in Fig. 1) to quantify whether a new\u000d\u000aexample comes from the same distribution as the training set.\u000d\u000aWe evaluate similarity by approximating the Wasserstein\u000d\u000adistance between the Exp2SimFeaturespace distribution of the new\u000d\u000ainstance and the Exp2SimFeaturespace distribution of the training\u000d\u000aset. Note that in Fig. 5, we only consider the Exp2SimGAN\u000d\u000anetwork, which is intended to perform a resolution\u000d\u000adeconvolution of the dataset. The same type of analysis is presented in Section D in the ESI,\u2020 where Sim2ExpGAN performs\u000d\u000aa resolution convolution.\u000d\u000aWe approximate the Wasserstein distance distribution\u000d\u000abetween a new data example and the training set using the\u000d\u000aSinkhorn distance approach.47 Here, we calculate the Wasserstein distance between a subset of the training data (20\u000d\u000arandomly sampled points) and the new data example. To obtain\u000d\u000aa distribution of Wasserstein distances, we repeat this process\u000d\u000a1000 times. Note that the Wasserstein distance applied in this\u000d\u000amanner only approximates the true distance between the relevant parameter distributions.\u000d\u000aAs seen in Fig. 5 and Section C in the ESI,\u2020 six of the datasets\u000d\u000aare INS spectra; the training set, the resolution convolved and\u000d\u000aunconvolved test set, the experimental PCSMO INS spectrum,\u000d\u000athe experimental negative control, and simulated 2D INS\u000d\u000aspectra of spin waves from a different atomic structure\u000d\u000a(Rb2MnF4).48 The last two datasets are the MNIST digit dataset\u000d\u000aand 15 random images of cute animals found on the internet.\u000d\u000aThese two datasets are hypothesized to be extremely far from\u000d\u000athe training set distribution (out-of-distribution examples).\u000d\u000aFig. 5 Applying Exp2SimGAN on a range of simulated and experimental 2D INS spectra. (a) After the network has been trained, it is used to\u000d\u000acompute a resolution deconvolution of the experimental INS spectra measured at 4 K on PCSMO (upper panels) and on a dataset that is used as\u000d\u000aa negative control (lower panels). The negative control dataset is composed of experimental INS spectra measured with the same instrumental\u000d\u000asettings (and hence instrumental resolution) on various different materials. Note that the experimental axis is the same as in Fig. 1a. The insets\u000d\u000ashow the DUQ classifications.45 (b) The Wasserstein distance of the Exp2SimFeaturespace position has been calculated between various datasets\u000d\u000a(target distributions) and the Exp2SimFeaturespace position of 20 randomly chosen points from the training set. This process was repeated 1000\u000d\u000atimes to sample distributions of Wasserstein distances from the target distributions to the training set distribution. Section C in the ESI\u2020 shows the\u000d\u000aresults of applying Exp2Simnetwork on the target distributions and Section D in the ESI\u2020 shows the same experiments conducted in this figure but\u000d\u000ausing Sim2Expnetwork.\u000d\u000a584 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","Fig. 5a demonstrates that both the training set and the\u000d\u000aresolution convolved test set have a mean Wasserstein distance\u000d\u000aof \u223c75 000, which can be taken as the baseline. These datasets\u000d\u000aare in the \u2018trustworthy\u2019 area, meaning that it is very likely that\u000d\u000aExp2SimGAN will be successful on the speci\ue103c datasets. The\u000d\u000aexperimental INS spectrum has a slightly higher mean Wasserstein distance (\u223c120 000) to the training set than the baseline, which we expect to be from experimental noise and\u000d\u000aartefacts as described in the Introduction section. The data of\u000d\u000athe digits (\u223c240 000) and animals (\u223c200 000) have signi\ue103cantly\u000d\u000alarger mean Wasserstein distances from the training data than\u000d\u000athe experimental INS spectrum does, demonstrating that the\u000d\u000aWasserstein metric can identify data that are very far from the\u000d\u000atraining set domain. These datasets are in the \u2018untrustworthy\u2019\u000d\u000aarea, meaning that it is very likely that Exp2SimGAN will not be\u000d\u000asuccessful on the speci\ue103c datasets. To look at data with more\u000d\u000asubtle differences, we turn to the unconvolved INS spectra\u000d\u000a(\u223c125 000), the negative control data (\u223c160 000) and the\u000d\u000aRb2MnF4 spectra (\u223c155 000) \u2013 in all cases, the mean\u000d\u000aWasserstein distance is greater than that of the experimental\u000d\u000aINS spectrum, showing that the Wasserstein metric can pick up\u000d\u000aon more nuanced differences in datasets that may mean that\u000d\u000aExp2Simnetwork is not applicable to a given data instance. These\u000d\u000adatasets are in the \u2018expert opinion\u2019 area, meaning that an expert\u000d\u000aopinion is needed to \ue103nally evaluate if Exp2SimGAN is\u000d\u000asuccessful on the speci\ue103c datasets. The measure of applicability\u000d\u000ais important for real-world applications of Exp2Simnetwork where\u000d\u000athe user must know how closely related the experimental data\u000d\u000aare to the training distribution before using Exp2Simnetwork\u000d\u000ablindly. As seen in Section D in the ESI,\u2020 the same conclusions\u000d\u000acan be made from the Sim2Expnetwork model.\u000d\u000aIn Section D in the ESI,\u2020 we present the same type of analysis\u000d\u000abut where Sim2ExpGAN performs a resolution convolution. The\u000d\u000aunconvolved INS spectra are located in the trustworthy area,\u000d\u000aand the convolved INS spectra are moved to the expert opinion\u000d\u000aarea, demonstrating that the trained models successfully bridge\u000d\u000athe two distributions for those samples. While the approach\u000d\u000athat we have demonstrated using the GAN to learn a mapping\u000d\u000aFig. 6 Evaluating Exp2SimGAN on simulated 3D INS data from the test set. The INS data are split into an 80% training set and 20% test set. After\u000d\u000athe network has trained on the training set, we apply it on the test set data. Here is shown an example of performing resolution deconvolution\u000d\u000aand convolution on 3D INS spectra simulated with the Dimer spin wave model. Section F in the ESI\u2020 shows an example with the Goodenough\u000d\u000aspin wave model.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 578\u2013590 | 585\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","from simulations with expensive instrument resolution added\u000d\u000aand pure simulated data is successful in this instance, the\u000d\u000amethod could be further improved by using experimental data\u000d\u000aas well for training. The advantage of using real experimental\u000d\u000adata would be that the method can learn to account for factors\u000d\u000awhich may have been missed, even in the high quality instrument resolution simulation, for example signals arising from\u000d\u000asources beyond the spin-wave simulations, such as phonons.\u000d\u000aThe \ue104exibility of the model means that including experimental\u000d\u000adata in the training, when it is available, will not be difficult.\u000d\u000aRemoving instrumental resolution from 3D INS data with\u000d\u000aExp2SimGAN, and vice versa\u000d\u000aUntil now, we have solely used Exp2SimGAN to convert between\u000d\u000asimulated and experimental 2D INS data. While organizing the\u000d\u000aINS spectra into 2D datasets helps conserve computer memory,\u000d\u000ait also compromises some of the information in the INS data,\u000d\u000awhich is frequently 3-dimensional.\u000d\u000aFig. 6 shows the results of converting between simulated and\u000d\u000aexperimental 3D INS spectra. Here, the INS spectra are similar\u000d\u000ato those in the previous section but without being arranged into\u000d\u000aa 2D representation. See Section E in the ESI\u2020 for more details\u000d\u000aabout the INS 3D data distribution. Some pixels are masked\u000d\u000a(shown as transparent in the \ue103gure) because they correspond to\u000d\u000agaps in the physical detector coverage so there are no data in\u000d\u000athose regions. Again, we can see that Exp2SimGAN performs\u000d\u000awell in convoluting or deconvoluting the resolution function\u000d\u000afrom the INS spectra, displaying a broadening of the dispersions with convolution very similar to those obtained with the\u000d\u000aaccurate Monte Carlo resolution in Horace, but at a fraction of\u000d\u000athe computational cost.\u000d\u000aConclusion\u000d\u000aOur GAN-based algorithm, Exp2SimGAN, shows promising\u000d\u000avisual results of convoluting and deconvoluting the resolution\u000d\u000afunction from both simulated 2D and 3D INS spectra. We have\u000d\u000afurther demonstrated the potential of using GANs for matching\u000d\u000asimulated and experimental data. Quantitively, we validated\u000d\u000aExp2SimGAN using a classi\ue103cation ML model trained on\u000d\u000asimulated data but used on experimental data. The accuracy\u000d\u000awas increased by \ue103rst using Exp2SimGAN to create simulatedlike data from the experimental data. To further demonstrate\u000d\u000athat Exp2SimGAN does not add arti\ue103cial features to the INS\u000d\u000adata, we used it on an experimental negative control that did\u000d\u000anot achieve high accuracy. In this study, we trained Exp2SimGAN on simulated INS spectra with and without applying\u000d\u000aresolution function convolution. More impactful would be to\u000d\u000atrain Exp2SimGAN on a set of simulated data and a set of\u000d\u000aexperimental data, enabling Exp2SimGAN to also remove or\u000d\u000aidentify effects from phenomena not yet described by the\u000d\u000aunderlying physics of the simulations.\u000d\u000aFinally, we have created a way to quantify Exp2SimGAN's\u000d\u000aapplicability domain. This metric can be used to determine\u000d\u000awhether the dataset being examined is closely connected to the\u000d\u000aExp2SimGAN training set distribution or whether it must be\u000d\u000aretrained using new data. Future ML models, in our opinion,\u000d\u000amust contain a domain of applicability metrics so that users do\u000d\u000anot employ them blindly. The approach demonstrated here\u000d\u000acould be an important step in the application of ML to more\u000d\u000aefficient analysis of large experimental datasets.\u000d\u000aMethods\u000d\u000aThe Exp2SimGAN network\u000d\u000aWe mostly follow the network architecture from DCLGAN to\u000d\u000atrain Exp2SimGAN,25 which has shown state-of-the-art results\u000d\u000aon various datasets to perform unpaired image-to-image\u000d\u000atranslation. Therefore, we hypothesize that it will also\u000d\u000aperform well on INS data.\u000d\u000aThis means that the generator, G, is ResNet-based49 with 9\u000d\u000aresidual blocks. We initialize the weights with Xavier initialization,50 and use instance normalization.51 We load all images\u000d\u000ain 286 \u00d7 286 and crop them to 256 \u00d7 256. We train for 200\u000d\u000aepochs with a learning rate of 0.0001, wherea\ue09der it decays\u000d\u000alinearly for 200 epochs more. The best model is evaluated using\u000d\u000aFig. 7 Simulating INS spectra of PCSMO. (Left) Diagram of the a\u2013b plane of Pr(Ca,Sr)2Mn2O7 showing just the Mn atoms, with Mn3+ sites in green\u000d\u000aand Mn4+ sites in red. Arrows denote the magnetic structure and black lines joining the atoms denote exchange interactions in the Goodenough\u000d\u000a(a) and Dimer (b) spin wave models. In addition to the Mn\u2013Mn interactions shown in the a\u2013b plane, there is an additional interaction, Jt,\u000d\u000aperpendicular to the a\u2013b plane coupling Mn ions in adjacent planes. (Right) Table of limits of exchange parameter values used to generate the\u000d\u000atraining data.\u000d\u000a586 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","the FID score46 using PyTorchs official implementation with\u000d\u000adefault settings (https://github.com/mseitzer/pytorch-\ue103d). We\u000d\u000ause the Adam optimizer52 with b1 = 0.5 and b2 = 0.999 and\u000d\u000aa batch size of 1. We use a similar discriminator to the\u000d\u000aPatchGAN discriminator architecture described for the\u000d\u000aoriginal CycleGAN paper28 and Pix2Pix53 but change the\u000d\u000aoutput prediction to two: an output for simulated/\u000d\u000aexperimental (auxiliary) and an output for real/fake (GAN).\u000d\u000aThe training was done on a Tesla V100-SXM3 32GB with GPU\u000d\u000adriver version 418.211.00 and CUDA version 10.1.\u000d\u000aWe weigh the loss functions 1 : 4 : 2 : 1 in the order GANloss :\u000d\u000aNCEloss : Idtloss : Auxiliaryloss, where the Idtloss is the identity loss\u000d\u000athat calculates the mean absolute error (MAE) of the generator\u000d\u000aoutput G(x) and its identity x:\u000d\u000aL Idt\u00f0G\u00de \u00bc MAE\u00f0G\u00f0x\u00de(\u00f0x\u00de\u00de\u000d\u000aExp2SimGAN can train on any shape of data; however,\u000d\u000aa limitation is GPU memory since Exp2SimGAN uses four large\u000d\u000aNNs with about 30 million parameters in total using default\u000d\u000aparameters. However, at inference (test) time, the generators,\u000d\u000aExp2Simnetwork or Sim2Expnetwork, can be used individually\u000d\u000asaving large amounts of memory. This means that Exp2SimGAN\u000d\u000acan be trained on small patches of the data to save memory and\u000d\u000aa\ue09derwards be applied on large datasets.\u000d\u000aExperimental INS data\u000d\u000aINS measurements, previously reported by Johnstone et al.,\u000d\u000a43\u000d\u000awere carried out using a MAPS spectrometer on a co-aligned\u000d\u000aarray of single crystals mounted such that the crystallographic\u000d\u000ac axis is parallel to the incident neutron beam. This ensured that\u000d\u000athe a\u2013b plane is imaged in the detector array. Measurements\u000d\u000awere made at this \ue103xed orientation at 4 K for a series of different\u000d\u000aincident neutron energies (Ei = 25, 35, 50, 70, 100, and 140 meV)\u000d\u000ato span the full bandwidth of the spin wave spectrum. For each\u000d\u000ameasurement, an estimate of the non-magnetic scattering and\u000d\u000abackground was made from regions in the data with no visible\u000d\u000amagnetic signal, and subtracted to leave an estimate of the\u000d\u000apurely magnetic scattering.\u000d\u000aThe negative control data, previously used in,14 are formed\u000d\u000afrom several different measurements, with each measurement\u000d\u000aforming a single row in the dataset, as follows (from the bottom\u000d\u000arow upwards).\u000d\u000aEi = 25 meV, 150 Hz: SrCuO2\u000d\u000aEi = 35 meV, 200 Hz: SrCo2As2\u000d\u000aEi = 50 meV, 200 Hz: La0.5Sr1.5MnO4\u000d\u000aEi = 70 meV, 250 Hz: SrCuO2\u000d\u000aEi = 100 meV, 250 Hz: La0.5Sr1.5MnO4\u000d\u000aEi = 140 meV, 400 Hz: La0.5Sr1.5MnO4\u000d\u000aAll measurements were done with the medium energy high\u000d\u000a\ue104ux (\u201csloppy\u201d) boron chopper at the CCR base temperature\u000d\u000a(around 5 K).\u000d\u000aSimulated INS data\u000d\u000aThe simulated INS images (both 2D and 3D) for PCSMO were\u000d\u000arecorded using linear spin wave theory (LSWT) as implemented\u000d\u000ain the SpinW54 code. The spin wave models are shown in Fig. 7,\u000d\u000atogether with a table of the ranges of the exchange parameters\u000d\u000afrom which values were randomly selected in the simulations\u000d\u000awhich generated the training data. We used the standard Heisenberg Hamiltonian with single-ion anisotropy:\u000d\u000aH \u00bc X\u000d\u000ai;j\u000d\u000aJijSi$Sj \u00fe D\u000d\u000aX\u000d\u000ai\u000d\u000a\u00f0Si\u000d\u000az\u000d\u000a\u00de\u000d\u000a2\u000d\u000awith 5 exchange parameters Jij in each spin wave model (Goodenough and Dimer) as shown in the \ue103gure. The calculated spin\u000d\u000awave spectrum was then convolved with the spectrometer resolution function using Horace.44 In the case of Rb2MnF4, a similar\u000d\u000aHamiltonian was used but without the anisotropy term proportional to D, and considering only nearest neighbour interactions\u000d\u000aJ1 in the range [0, 1] meV and next nearest neighbour interactions\u000d\u000aJ2 in the range [\u22120.2, 0.2] meV between Mn2+ ions on a square\u000d\u000alattice. Each pixel in the simulated image is a histogram bin,\u000d\u000awhich averages the measured intensity (neutron counts) of typically hundreds of individual detector-energy points, which\u000d\u000arepresents neutrons scattered through a particular angle and\u000d\u000aarriving at a particular time-of-\ue104ight (ToF). The nominal scattering angle and ToF de\ue103ne a nominal momentum and energy\u000d\u000atransfer. However, the neutron beam is neither perfectly collimated nor monochromatic, so the actual neutron momentum\u000d\u000aand energy have a spread which is the resolution function. The\u000d\u000adistributions de\ue103ning this spread are not Gaussian but are wellde\ue103ned by the geometry and other characteristics of the spectrometer. To perform the resolution convolution, we draw Nmc =\u000d\u000a10 samples from the distributions describing the resolution\u000d\u000afunction for each detector-energy point, calculate the INS intensity using LSWT and average the result. Since each image pixel\u000d\u000a(histogram bin) has several hundred detector-energy points, Nmc\u000d\u000adoes not need to be large to yield an accurate convolution.\u000d\u000aFor the resolution unconvolved calculations, the LSWT\u000d\u000amodel was only evaluated at the nominal centre of the pixel\u000d\u000a(histogram bin), which thus requires several thousand times\u000d\u000afewer evaluations of the LSWT model. In the case of PCSMO,\u000d\u000aeach evaluation requires the construction and diagonalization\u000d\u000aof a 32 \u00d7 32 element Hamiltonian matrix (8 \u00d7 8 in the case of\u000d\u000aRb2MnF4) and addition matrix\u2013matrix multiplications to\u000d\u000acompute the spin\u2013spin correlation function (which is proportional to the measured neutron intensity). As we need \u223c108 such\u000d\u000aevaluations per image for the resolution convolution calculation, this is prohibitive so we also used the Brille55 code to\u000d\u000aperform linear interpolation of the spin wave energy and spin\u2013\u000d\u000aspin correlation function, which provides \u223c5\u00d7 speedup\u000d\u000acompared to evaluating the diagonalization directly.\u000d\u000aEven with the linear interpolation speed up, the resolution\u000d\u000aconvolved calculation took \u223c720 CPU-minutes per image,\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 578\u2013590 | 587\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","whereas the resolution unconvolved calculation takes \u223c8 CPUminutes per image. The speedup is not larger because the linear\u000d\u000ainterpolation was not used for the unconvolved calculation\u000d\u000asince there are so few evaluation points that the overhead of\u000d\u000asetting up the interpolation grid would have made the overall\u000d\u000acalculation slower. In addition, the same script was used to run\u000d\u000aboth the resolution-convolved and unconvolved calculations,\u000d\u000awhich impose some overheads such as reading in the measured\u000d\u000adata \ue103les to obtain the coordinates of all detector-energy points\u000d\u000a(which is not needed for the resolution unconvolved\u000d\u000acalculation).\u000d\u000aThe measurements were carried out with a range of different\u000d\u000aincident neutron energies, in neutron energy loss mode (that is,\u000d\u000athe magnon energy corresponds to the energy lost by a scattered\u000d\u000aneutron). In this mode, a higher incident neutron energy will\u000d\u000agive a larger dynamic range, but coarser energy resolution.\u000d\u000aUsing several different neutron energies allowed us to both see\u000d\u000athe full dispersion of the spin waves and to resolve features\u000d\u000aaround 30 meV, which is critical to distinguishing between the\u000d\u000aDimer and Goodenough spin wave models.14 The 2D data\u000d\u000acontain data from all the different incident energies, where\u000d\u000aeach incident energy corresponds to a row of images, but with\u000d\u000aeach panel in a row integrating over a relatively large range in\u000d\u000amagnon energy. The 3D data, on the other hand, show only the\u000d\u000adata taken with Ei = 70 meV, but where the third dimension is\u000d\u000anow the energy transfer, allowing a more in-depth look at the\u000d\u000akey dataset for distinguishing between the spin wave models.\u000d\u000aBoth 2D and 3D datasets were generated using the same\u000d\u000awork\ue104ow of SpinW for the spin wave calculation and Horace\u000d\u000aand Brille for resolution convolution.\u000d\u000aData availability\u000d\u000aThe authors declare that the data supporting this study are\u000d\u000aavailable within the paper, its ESI\u2020 \ue103les and the associated\u000d\u000aGitHub and Zenodo to the paper: https://github.com/\u000d\u000aAndySAnker/Exp2SimGAN and https://zenodo.org/record/\u000d\u000a7308423#.Y2zgoOzML0o. Additional data that support the\u000d\u000a\ue103ndings of this study are available from the corresponding\u000d\u000aauthors upon request.\u000d\u000aCode availability\u000d\u000aThe authors declare that the code supporting this study is\u000d\u000aavailable on the associated GitHub to the paper: https://\u000d\u000agithub.com/AndySAnker/Exp2SimGAN. The additional code\u000d\u000athat supports the \ue103ndings of this study is available from the\u000d\u000acorresponding authors upon request.\u000d\u000aAuthor contributions\u000d\u000aA. S. A., K. T. B., and D. M. L. conceptualized the project and\u000d\u000adesigned the methodology. A. S. A. wrote the code. D. M. L.\u000d\u000asimulated the data. T. G. P. and J. T. procured funding. All\u000d\u000aauthors were involved in the writing of the paper.\u000d\u000aConflicts of interest\u000d\u000aThe authors declare no competing interests.\u000d\u000aAcknowledgements\u000d\u000aASA would like to thank the Augustinus Foundation, the Fabrikant Vilhelm Pedersen og hustrus Foundation, the Haynmann\u000d\u000aFoundation, the Henry og Mary Skovs Foundation, the Knud\u000d\u000aH\u00f8jgaard Foundation, the Thomas B. Thriges Foundation, and\u000d\u000athe Viet Jacobsen Foundation for \ue103nancial support to this\u000d\u000aresearch project. This work was partially supported by wave 1 of\u000d\u000athe UKRI Strategic Priorities Fund under the EPSRC (Grant No.\u000d\u000aEP/T001569/1), particularly the \u201cAI for Science\u201d theme within\u000d\u000athat grant and The Alan Turing Institute. The simulated datasets were generated using computing resources provided by\u000d\u000aSTFC Scienti\ue103c Computing Department's SCARF cluster.\u000d\u000aExp2SimGAN was trained using computing resources provided\u000d\u000aby STFC Scienti\ue103c Computing Department's PEARL cluster. TGP\u000d\u000athanks collaborators A. T. Boothroyd and D. Prabhakaran in ref.\u000d\u000a43 for their permission to use the PCSMO experimental\u000d\u000adatasets.\u000d\u000aReferences\u000d\u000a1 J. Armstrong, A. J. O'Malley, M. R. Ryder and K. T. Butler,\u000d\u000aUnderstanding dynamic properties of materials using\u000d\u000aneutron spectroscopy and atomistic simulation, J. Phys.\u000d\u000aCommun., 2020, 4(7), 072001.\u000d\u000a2 Z. Chen, N. Andrejevic, N. C. Drucker, T. Nguyen, R. P. Xian,\u000d\u000aT. Smidt, Y. Wang, R. Ernstorfer, D. A. Tennant, M. Chan and\u000d\u000aM. Li, Machine learning on neutron and X-ray scattering and\u000d\u000aspectroscopies, Chem. Phys. Rev., 2021, 2(3), 031301.\u000d\u000a3 S. V. Kalinin, C. Ophus, P. M. Voyles, R. Erni,\u000d\u000aD. Kepaptsoglou, V. Grillo, A. R. Lupini, M. P. Oxley,\u000d\u000aE. Schwenker, M. K. Y. Chan, J. Etheridge, X. Li,\u000d\u000aG. G. D. Han, M. Ziatdinov, N. Shibata and S. J. Pennycook,\u000d\u000aMachine learning in scanning transmission electron\u000d\u000amicroscopy, Nat. Rev. Methods Primers, 2022, 2(1), 11.\u000d\u000a4 E. O. Pyzer-Knapp, J. W. Pitera, P. W. J. Staar, S. Takeda,\u000d\u000aT. Laino, D. P. Sanders, J. Sexton, J. R. Smith and\u000d\u000aA. Curioni, Accelerating materials discovery using arti\ue103cial\u000d\u000aintelligence, high performance computing and robotics,\u000d\u000anpj Comput. Mater., 2022, 8(1), 84.\u000d\u000a5 J. Deng, W. Dong, R. Socher, L. J. Li, L. Kai and F.-F. Li,\u000d\u000aImageNet: a large-scale hierarchical image database, in\u000d\u000a2009 IEEE Conference on Computer Vision and Pattern\u000d\u000aRecognition, 20\u201325 June 2009, pp. 248\u2013255.\u000d\u000a6 A. Jain, S. P. Ong, G. Hautier, W. Chen, W. D. Richards,\u000d\u000aS. Dacek, S. Cholia, D. Gunter, D. Skinner, G. Ceder and\u000d\u000aK. A. Persson, Commentary: the Materials Project:\u000d\u000aa materials genome approach to accelerating materials\u000d\u000ainnovation, APL Mater., 2013, 1(1), 011002.\u000d\u000a7 K. Choudhary, K. F. Garrity, A. C. E. Reid, B. DeCost,\u000d\u000aA. J. Biacchi, A. R. Hight Walker, Z. Trautt, J. HattrickSimpers, A. G. Kusne, A. Centrone, A. Davydov, J. Jiang,\u000d\u000aR. Pachter, G. Cheon, E. Reed, A. Agrawal, X. Qian,\u000d\u000a588 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","V. Sharma, H. Zhuang, S. V. Kalinin, B. G. Sumpter,\u000d\u000aG. Pilania, P. Acar, S. Mandal, K. Haule, D. Vanderbilt,\u000d\u000aK. Rabe and F. Tavazza, The joint automated repository for\u000d\u000avarious integrated simulations (JARVIS) for data-driven\u000d\u000amaterials design, npj Comput. Mater., 2020, 6(1), 173.\u000d\u000a8 X. Li, P.-F. Liu, E. Zhao, Z. Zhang, T. Guidi, M. D. Le,\u000d\u000aM. Avdeev, K. Ikeda, T. Otomo, M. Kofu, K. Nakajima,\u000d\u000aJ. Chen, L. He, Y. Ren, X.-L. Wang, B.-T. Wang, Z. Ren,\u000d\u000aH. Zhao and F. Wang, Ultralow thermal conductivity from\u000d\u000atransverse acoustic phonon suppression in distorted\u000d\u000acrystalline a-MgAgSb, Nat. Commun., 2020, 11(1), 942.\u000d\u000a9 E. A. Goremychkin, H. Park, R. Osborn, S. Rosenkranz,\u000d\u000aJ.-P. Castellan, V. R. Fanelli, A. D. Christianson,\u000d\u000aM. B. Stone, E. D. Bauer, K. J. McClellan, D. D. Byler and\u000d\u000aJ. M. Lawrence, Coherent band excitations in CePd3:\u000d\u000aa comparison of neutron scattering and ab initio theory,\u000d\u000aScience, 2018, 359(6372), 186\u2013191.\u000d\u000a10 T. Chen, Y. Chen, A. Kreisel, X. Lu, A. Schneidewind, Y. Qiu,\u000d\u000aJ. T. Park, T. G. Perring, J. R. Stewart, H. Cao, R. Zhang, Y. Li,\u000d\u000aY. Rong, Y. Wei, B. M. Andersen, P. J. Hirschfeld, C. Broholm\u000d\u000aand P. Dai, Anisotropic spin \ue104uctuations in detwinned FeSe,\u000d\u000aNat. Mater., 2019, 18(7), 709\u2013716.\u000d\u000a11 P. A. McClarty, F. Kr\u00fcger, T. Guidi, S. F. Parker, K. Refson,\u000d\u000aA. W. Parker, D. Prabhakaran and R. Coldea, Topological\u000d\u000atriplon modes and bound states in a Shastry\u2013Sutherland\u000d\u000amagnet, Nat. Phys., 2017, 13(8), 736\u2013741.\u000d\u000a12 S.-H. Do, S.-Y. Park, J. Yoshitake, J. Nasu, Y. Motome,\u000d\u000aY. S. Kwon, D. T. Adroja, D. J. Voneshen, K. Kim,\u000d\u000aT. H. Jang, J. H. Park, K.-Y. Choi and S. Ji, Majorana\u000d\u000afermions in the Kitaev quantum spin system a-RuCl3, Nat.\u000d\u000aPhys., 2017, 13(11), 1079\u20131084.\u000d\u000a13 T. Holstein and H. Primakoff, Field Dependence of the\u000d\u000aIntrinsic Domain Magnetization of a Ferromagnet, Phys.\u000d\u000aRev., 1940, 58(12), 1098\u20131113.\u000d\u000a14 K. T. Butler, M. D. Le, J. Thiyagalingam and T. G. Perring,\u000d\u000aInterpretable, calibrated neural networks for analysis and\u000d\u000aunderstanding of inelastic neutron scattering data, J. Phys.:\u000d\u000aCondens. Matter, 2021, 33(19), 194006.\u000d\u000a15 R. A. Ewings, A. Buts, M. D. Le, J. van Duijn, I. Bustinduy and\u000d\u000aT. G. Perring, Horace: so\ue09dware for the analysis of data from\u000d\u000asingle crystal spectroscopy experiments at time-of-\ue104ight\u000d\u000aneutron instruments, Nucl. Instrum. Methods Phys. Res.,\u000d\u000aSect. A, 2016, 834, 132\u2013142.\u000d\u000a16 T. G. Perring, High energy magnetic excitations in hexagonal\u000d\u000acobalt, University of Cambridge, 1991.\u000d\u000a17 J. Y. Y. Lin, H. L. Smith, G. E. Granroth, D. L. Abernathy,\u000d\u000aM. D. Lumsden, B. Winn, A. A. Aczel, M. Aivazis and\u000d\u000aB. Fultz, MCViNE \u2013 an object oriented Monte Carlo\u000d\u000aneutron ray tracing simulation package, Nucl. Instrum.\u000d\u000aMethods Phys. Res., Sect. A, 2016, 810, 86\u201399.\u000d\u000a18 K. Lefmann and K. Nielsen, McStas, a general so\ue09dware\u000d\u000apackage for neutron ray-tracing simulations, Neutron News,\u000d\u000a1999, 10(3), 20\u201323.\u000d\u000a19 P. Willendrup, E. Farhi, E. Knudsen, U. Filges and\u000d\u000aK. Lefmann, McStas: past, present and future, J. Neutron\u000d\u000aRes., 2014, 17, 35\u201343.\u000d\u000a20 P. Willendrup, E. Farhi and K. Lefmann, McStas 1.7-a new\u000d\u000aversion of the \ue104exible Monte Carlo neutron scattering\u000d\u000apackage, Phys. B, 2004, 350(1\u20133), E735\u2013E737.\u000d\u000a21 P. K. Willendrup and K. Lefmann, McStas (ii): an overview of\u000d\u000acomponents, their use, and advice for user contributions, J.\u000d\u000aNeutron Res., 2021, 23, 7\u201327.\u000d\u000a22 P. K. Willendrup and K. Lefmann, McStas (i): introduction,\u000d\u000ause, and basic principles for ray-tracing simulations, J.\u000d\u000aNeutron Res., 2020, 22, 1\u201316.\u000d\u000a23 E. B. Knudsen, P. K. Willendrup, J. Garde and M. Bertelsen.\u000d\u000aMcXtrace anno 2020-complex sample geometries and GPU\u000d\u000aacceleration, in Advances in Computational Methods for XRay Optics V, SPIE, 2020, pp. 46\u201352.\u000d\u000a24 J. Y. Y. Lin, F. Islam, G. Sala, I. Lumsden, H. Smith,\u000d\u000aM. Doucet, M. B. Stone, D. L. Abernathy, G. Ehlers,\u000d\u000aJ. F. Ankner and G. E. Granroth, Recent developments of\u000d\u000aMCViNE and its applications at SNS, J. Phys. Commun.,\u000d\u000a2019, 3(8), 085005.\u000d\u000a25 J. Han, M. Shoeiby, L. Petersson and M. A. Armin, Dual\u000d\u000aContrastive Learning for Unsupervised Image-to-Image\u000d\u000aTranslation, in Proceedings of the IEEE/CVF Conference on\u000d\u000aComputer Vision and Pattern Recognition, 2021, pp. 746\u2013755.\u000d\u000a26 T. Park, A. A. Efros, R. Zhang and J.-Y. Zhu, Contrastive\u000d\u000alearning for unpaired image-to-image translation, in\u000d\u000aEuropean Conference on Computer Vision, Springer, 2020,\u000d\u000app. 319\u2013345.\u000d\u000a27 Z. Yi, H. Zhang, P. Tan and M. Gong, Dualgan: unsupervised\u000d\u000adual learning for image-to-image translation, in Proceedings\u000d\u000aof the IEEE international conference on computer vision, 2017,\u000d\u000app. 2849\u20132857.\u000d\u000a28 J.-Y. Zhu, T. Park, P. Isola, and A. A. Efros, Unpaired imageto-image translation using cycle-consistent adversarial\u000d\u000anetworks, in Proceedings of the IEEE international conference\u000d\u000aon computer vision, 2017, pp. 2223\u20132232.\u000d\u000a29 I. Goodfellow, J. Pouget-Abadie, M. Mirza, B. Xu, D. WardeFarley, S. Ozair, A. Courville and Y. Bengio, Generative\u000d\u000aadversarial nets, Advances in Neural Information Processing\u000d\u000aSystems, 2014, 27, https://papers.nips.cc/paper_\ue103les/paper/\u000d\u000a2014/hash/5ca3e9b122f61f8f06494c97b1afccf3-\u000d\u000aAbstract.html.\u000d\u000a30 T. Kim, M. Cha, H. Kim, J. K. Lee and J. Kim, Learning to\u000d\u000adiscover cross-domain relations with generative adversarial\u000d\u000anetworks, in International conference on machine learning,\u000d\u000aPMLR, 2017, pp. 1857\u20131865.\u000d\u000a31 S. Kench and S. J. Cooper, Generating three-dimensional\u000d\u000astructures from a two-dimensional slice with generative\u000d\u000aadversarial network-based dimensionality expansion,\u000d\u000aNature Machine Intelligence, 2021, 3(4), 299\u2013305.\u000d\u000a32 A. Nouira, N. Sokolovska and J.-C. Crivello, Crystalgan:\u000d\u000alearning to discover crystallographic structures with\u000d\u000agenerative adversarial networks, arXiv preprint\u000d\u000aarXiv:1810.11203, 2018.\u000d\u000a33 S. Kim, J. Noh, G. H. Gu, A. Aspuru-Guzik and Y. Jung,\u000d\u000aGenerative Adversarial Networks for Crystal Structure\u000d\u000aPrediction, ACS Cent. Sci., 2020, 6(8), 1412\u20131420.\u000d\u000a34 Y. Hong, B. Hou, H. Jiang and J. Zhang, Machine learning\u000d\u000aand arti\ue103cial neural network accelerated computational\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Digital Discovery, 2023, 2, 578\u2013590 | 589\u000d\u000aPaper Digital Discovery\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online","discoveries in materials science, Wiley Interdiscip. Rev.:\u000d\u000aComput. Mol. Sci., 2020, 10(3), e1450.\u000d\u000a35 S. Papadopoulos, A. Drosou and D. Tzovaras, Modelling of\u000d\u000aMaterial Ageing with Generative Adversarial Networks, in\u000d\u000a2018 IEEE 13th Image, Video, and Multidimensional Signal\u000d\u000aProcessing Workshop (IVMSP), 10\u201312 June 2018, pp. 1\u20135.\u000d\u000a36 Y. Mao, Q. He and X. Zhao, Designing complex architectured\u000d\u000amaterials with generative adversarial networks, Sci. Adv.,\u000d\u000a2020, 6(17), eaaz4169.\u000d\u000a37 Q. Ai, A. J. Norquist and J. Schrier, Predicting compositional\u000d\u000achanges of organic\u2013inorganic hybrid materials with\u000d\u000aaugmented CycleGAN, Digital Discovery, 2022, 1(3), 255\u2013265.\u000d\u000a38 A. Khan, C.-H. Lee, P. Huang and B. Clark, Using CycleGANs\u000d\u000ato Generate Realistic STEM Images for Machine Learning, in\u000d\u000aMachine Learning and the Physical Sciences Workshop at the\u000d\u000a36th Conference on Neural Information Processing Systems\u000d\u000a(NeurIPS), 2022.\u000d\u000a39 T. Chen, S. Kornblith, M. Norouzi and G. Hinton, A simple\u000d\u000aframework for contrastive learning of visual\u000d\u000arepresentations, in International conference on machine\u000d\u000alearning, PMLR, 2020, pp. 1597\u20131607.\u000d\u000a40 M. Gutmann and A. Hyv\u00a8arinen, Noise-contrastive\u000d\u000aestimation: a new estimation principle for unnormalized\u000d\u000astatistical models, in Proceedings of the Thirteenth\u000d\u000aInternational Conference on Arti\ue103cial Intelligence and\u000d\u000aStatistics, JMLR Workshop and Conference Proceedings, 2010,\u000d\u000app. 297\u2013304.\u000d\u000a41 A. Odena, C. Olah and J. Shlens, Conditional image synthesis\u000d\u000awith auxiliary classi\ue103er gans, in International conference on\u000d\u000amachine learning, PMLR, 2017, pp. 2642\u20132651.\u000d\u000a42 K. Khan, G. Sahu, V. Balasubramanian, L. Mou and\u000d\u000aO. Vechtomova, Adversarial learning on the latent space\u000d\u000afor diverse dialog generation, arXiv preprint\u000d\u000aarXiv:1911.03817, 2019.\u000d\u000a43 G. E. Johnstone, T. G. Perring, O. Sikora, D. Prabhakaran and\u000d\u000aA. T. Boothroyd, Ground State in a Half-Doped Manganite\u000d\u000aDistinguished by Neutron Spectroscopy, Phys. Rev. Lett.,\u000d\u000a2012, 109(23), 237202.\u000d\u000a44 T. Perring, TOBYFIT version 2.0. Least-squares \ue103tting to single\u000d\u000acrystal data, 2004.\u000d\u000a45 J. Van Amersfoort, L. Smith, Y. W. Teh and Y. Gal,\u000d\u000aUncertainty estimation using a single deep deterministic\u000d\u000aneural network, in International conference on machine\u000d\u000alearning, PMLR, 2020, pp. 9690\u20139700.\u000d\u000a46 M. Heusel, H. Ramsauer, T. Unterthiner, B. Nessler and\u000d\u000aS. Hochreiter, Gans trained by a two time-scale update rule\u000d\u000aconverge to a local nash equilibrium, Advances in Neural\u000d\u000aInformation Processing Systems, 2017, 30, https://\u000d\u000aproceedings.neurips.cc/paper_\ue103les/paper/2017/hash/\u000d\u000a8a1d694707eb0fefe65871369074926d-Abstract.html.\u000d\u000a47 M. Cuturi, Sinkhorn distances: lightspeed computation of\u000d\u000aoptimal transport, Advances in Neural Information\u000d\u000aProcessing Systems, 2013, 26, https://\u000d\u000aproceedings.neurips.cc/paper/2013/hash/\u000d\u000aaf21d0c97db2e27e13572cbf59eb343d-Abstract.html.\u000d\u000a48 T. Huberman, R. Coldea, R. A. Cowley, D. A. Tennant,\u000d\u000aR. L. Leheny, R. J. Christianson and C. D. Frost, Twomagnon excitations observed by neutron scattering in the\u000d\u000atwo-dimensional spin-5/2 Heisenberg antiferromagnet\u000d\u000aRb2MnF4, Phys. Rev., 2005, 72(1), 014413.\u000d\u000a49 K. He, X. Zhang, S. Ren and J. Sun, Deep residual learning for\u000d\u000aimage recognition, in Proceedings of the IEEE conference on\u000d\u000acomputer vision and pattern recognition, 2016, pp. 770\u2013778.\u000d\u000a50 X. Glorot and Y. Bengio, Understanding the difficulty of\u000d\u000atraining deep feedforward neural networks, in Proceedings\u000d\u000aof the Thirteenth International Conference on Arti\ue103cial\u000d\u000aIntelligence and Statistics, JMLR Workshop and Conference\u000d\u000aProceedings, 2010, pp. 249\u2013256.\u000d\u000a51 D. Ulyanov, A. Vedaldi and V. Lempitsky, Instance\u000d\u000anormalization: the missing ingredient for fast stylization,\u000d\u000aarXiv preprint arXiv:1607.08022, 2016.\u000d\u000a52 D. P. Kingma and J. Ba, Adam: a method for stochastic\u000d\u000aoptimization, arXiv preprint arXiv:1412.6980, 2014.\u000d\u000a53 P. Isola, J.-Y. Zhu, T. Zhou and A. A. Efros, Image-to-image\u000d\u000atranslation with conditional adversarial networks, in\u000d\u000aProceedings of the IEEE conference on computer vision and\u000d\u000apattern recognition, 2017, pp. 1125\u20131134.\u000d\u000a54 S. Toth and B. Lake, Linear spin wave theory for single-Q\u000d\u000aincommensurate magnetic structures, J. Phys.: Condens.\u000d\u000aMatter, 2015, 27(16), 166002.\u000d\u000a55 G. S. Tucker, https://brille.github.io/stable/index.html.\u000d\u000a590 | Digital Discovery, 2023, 2, 578\u2013590 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aDigital Discovery Paper\u000d\u000aOpen Access Article. Published on 15 March 2023. Downloaded on 6/12/2023 3:48:39 PM. This article is licensed under a Creative Commons Attribution 3.0 Unported Licence.\u000d\u000aView Article Online",""," \u000d\u000a169\u000d\u000a5 Concluding\u0009Remarks and\u0009Outlook*\u000d\u000aIn\u0009 this\u0009PhD\u0009dissertation,\u0009we\u0009 have\u0009 delved\u0009into\u0009 the\u0009field of automated scattering\u0009 data\u0009 analysis,\u0009with\u0009 a\u0009\u000d\u000aparticular\u0009focus\u0009on\u0009nanostructure\u0009characterisation.\u0009Traditional\u0009crystallographic\u0009methods,\u0009while vital\u0009to\u0009\u000d\u000aour\u0009research in materials\u0009chemistry,\u0009face\u0009limitations\u0009when\u0009applied\u0009to\u0009nanomaterials\u0009due\u0009to\u0009their\u0009limited\u0009\u000d\u000along-range\u0009order.\u0009Consequently,\u0009we\u0009have\u0009turned\u0009to\u0009alternative\u0009methods\u0009such\u0009as\u0009PDF\u0009and\u0009SAXS\u0009for\u0009the\u0009\u000d\u000acharacterisation\u0009of\u0009nanostructures.\u0009Here,\u0009the Debye\u0009scattering\u0009equation plays\u0009a\u0009central role\u0009in\u0009modelling\u0009\u000d\u000aboth\u0009PDF\u0009and\u0009SAXS,\u0009as\u0009it\u0009is\u0009used\u0009to\u0009calculate\u0009the\u0009scattering\u0009intensities\u0009considering\u0009the\u0009position\u0009of\u0009each\u0009\u000d\u000aatom\u0009in\u0009the\u0009structure.\u0009\u000d\u000aThe\u0009conventional\u0009approach\u0009for\u0009scattering\u0009data\u0009analysis\u0009involves\u0009the\u0009use\u0009of\u0009minimisation\u0009techniques\u0009to\u0009\u000d\u000afit\u0009 the\u0009 data.\u0009 These\u0009 techniques,\u0009 such\u0009 as\u0009 least-squares\u0009 fitting\u0009 algorithms,\u0009 are\u0009 effective\u0009 at\u0009 minimising\u0009\u000d\u000adiscrepancies\u0009 between\u0009 experimental\u0009 and\u0009 simulated\u0009 data\u0009 derived\u0009 from\u0009 a\u0009 structural\u0009model,\u0009 providing\u0009\u000d\u000ainvaluable\u0009structural\u0009information\u0009like\u0009atomic\u0009vibrations,\u0009particle\u0009shape,\u0009and\u0009particle\u0009size.\u0009However,\u0009they\u000d\u000arely\u0009on\u0009a\u0009structural\u0009model\u0009as\u0009input,\u0009which\u0009can\u0009require\u0009 time-consuming\u0009literature\u0009reviews\u0009 to\u0009identify,\u0009\u000d\u000afollowed\u0009by\u0009locating\u0009the\u0009structural\u0009model\u0009in\u0009structural\u0009databases.\u0009Once\u0009the\u0009structural\u0009model\u0009is\u0009found,\u0009it\u0009\u000d\u000amust\u0009 be\u0009 fitted\u0009 to\u0009 the\u0009 dataset\u0009 using\u0009 specialised\u0009 software.\u0009 This\u0009 process\u0009 can\u0009 be\u0009 complicated and\u0009 often\u0009\u000d\u000ademands\u0009expertise\u0009in\u0009the\u0009field.\u0009\u000d\u000aThis\u0009dissertation\u0009demonstrates the\u0009potential of\u0009using\u0009ML\u0009to\u0009automate\u0009and\u0009accelerate\u0009the\u0009most\u0009labourintensive\u0009aspects\u0009of\u0009scattering\u0009data\u0009analysis.\u0009For\u0009instance,\u0009it\u0009is demonstrated\u0009how supervised\u0009ML\u0009can\u0009\u000d\u000ainstantaneously\u0009identify the\u0009structural\u0009model\u0009 from\u0009a\u0009scattering\u0009dataset.\u0009Once\u0009 the\u0009structural\u0009model\u0009is\u0009\u000d\u000adetermined,\u0009it\u0009can\u0009be\u0009used in\u0009 the\u0009conventional\u0009modelling\u0009scheme.\u0009Fitting\u0009 the\u0009structural\u0009model\u0009 to\u0009 the\u0009\u000d\u000aexperimental\u0009 data allows\u0009 for\u0009 the\u0009 extraction\u0009 of structural\u0009 parameters.\u0009 Supervised\u0009 ML\u0009 also\u0009 has\u0009 the\u0009\u000d\u000acapacity\u0009 to\u0009 directly\u0009 predict\u0009 structural\u0009 parameters.\u0009However,\u0009 I\u0009 recognise\u0009 that\u0009 structure\u0009 refinement\u0009is\u0009\u000d\u000ahighly\u0009effective\u0009in\u0009providing\u0009invaluable\u0009structural\u0009information and\u0009is widely\u0009used\u0009and\u0009endorsed\u0009by\u0009the\u0009\u000d\u000acommunity.\u0009Therefore,\u0009I\u0009believe\u0009that\u0009the\u0009use\u0009of\u0009supervised\u0009ML\u0009for\u0009predicting\u0009the\u0009structural\u0009model,\u0009while\u0009\u000d\u000apreserving\u0009the\u0009fitting\u0009process,\u0009presents\u0009a\u0009significant\u0009strength.\u0009In\u0009addition,\u0009structure\u0009refinement\u0009validates\u0009\u000d\u000athe\u0009ML\u0009models'\u0009ability\u0009to\u0009identify\u0009the\u0009structural\u0009model.\u0009\u000d\u000aThroughout\u0009this\u0009dissertation,\u0009I\u0009have\u0009identified\u0009several\u0009challenges\u0009inherent\u0009to\u0009the\u0009use\u0009of\u0009supervised\u0009ML.\u0009\u000d\u000aOne\u0009key\u0009challenge\u0009is\u0009curating\u0009an\u0009accurate\u0009training\u0009set,\u0009which\u0009becomes\u0009particularly\u0009complex\u0009when\u0009dealing\u0009\u000d\u000awith\u0009data\u0009that\u0009involves\u0009multiple\u0009chemical\u0009components,\u0009or\u0009when\u0009the\u0009data\u0009represents\u0009structural\u0009models\u0009\u000d\u000aabsent\u0009from\u0009existing\u0009structural\u0009databases.\u0009Furthermore,\u0009experimental\u0009data\u0009often\u0009contain\u0009signals\u0009that\u0009are\u0009\u000d\u000anot\u0009 present\u0009 in\u0009 simulated\u0009 data,\u0009 adding\u0009 an\u0009 additional\u0009 layer\u0009 of\u0009 complexity.\u0009 To\u0009 address\u0009 these\u0009 issues,\u0009\u000d\u000aspecifically\u0009 the\u0009absence\u0009of\u0009certain\u0009structures\u0009in\u0009databases\u0009and\u0009variances\u0009in\u0009experimental\u0009data,\u0009 I\u0009have\u0009\u000d\u000aexplored\u0009the\u0009potential\u0009of\u0009generative ML,\u0009a\u0009hybrid\u0009approach\u0009between\u0009supervised-\u0009and\u0009unsupervised\u0009ML.\u0009\u000d\u000aUnsupervised\u0009 ML operates\u0009 independently\u0009 of\u0009 paired\u0009 labels\u0009 and\u0009 input\u0009 features,\u0009 offering\u0009 a\u0009 promising\u0009\u000d\u000aalternative to\u0009supervised\u0009ML.\u0009While\u0009these challenges\u0009are\u0009not\u0009completely\u0009overcome\u0009yet,\u0009we\u0009have\u0009achieved\u0009\u000d\u000aencouraging\u0009results\u0009for\u0009a\u0009subset\u0009of\u0009chemical\u0009systems.\u000d\u000aMost\u0009 of\u0009 the\u0009ML\u0009analysis\u0009methodologies\u0009used\u0009 for\u0009scattering\u0009data\u0009analysis in\u0009 this\u0009dissertation\u0009are also\u000d\u000aapplicable\u0009on\u0009spectroscopy\u0009data as\u0009demonstrated in\u0009Chapter\u00094.1 and\u0009Paper\u0009 II\u0009+\u0009IV.\u0009However,\u0009to\u0009 fully\u0009\u000d\u000arealise\u0009the\u0009potential\u0009of\u0009ML\u0009for\u0009scattering\u0009and\u0009spectroscopy\u0009data\u0009analysis,\u0009we\u0009need\u0009a\u0009shift\u0009in\u0009the\u0009way\u0009we\u0009\u000d\u000apublish\u0009research.\u0009I\u0009suggest that\u0009it\u0009should\u0009become standard\u0009practice\u0009to\u0009publish\u0009data,\u0009code,\u0009and\u0009software\u0009\u000d\u000a* This\u0009Chapter\u0009is\u0009based\u0009on Paper\u0009II.","|\u0009Concluding\u0009Remarks\u0009and\u0009Outlook\u000d\u000a170\u000d\u000arequirements\u0009 alongside\u0009 research\u0009 papers.\u0009 This\u0009 would\u0009 make\u0009 it\u0009 easier\u0009 for\u0009 other\u0009 researchers\u0009 to\u0009 apply\u0009\u000d\u000atrained\u0009ML\u0009models\u0009to\u0009their\u0009experimental\u0009data.\u0009Furthermore,\u0009it\u0009is\u0009important\u0009that\u0009the\u0009ML\u0009model\u0009provides\u0009\u000d\u000aan indication\u0009 of\u0009 its\u0009 risk of failing on\u0009 a\u0009 specific\u0009 dataset.\u0009 Therefore,\u0009 I\u0009 have\u0009 developed\u0009 a\u0009 domain\u0009 of\u0009\u000d\u000aapplication\u0009measure,\u0009which\u0009allows us\u0009to\u0009assess\u0009the\u0009likelihood\u0009that\u0009the\u0009ML\u0009model\u0009will\u0009be\u0009successful\u0009on\u0009a\u0009\u000d\u000agiven\u0009dataset.\u0009Another\u0009similar\u0009tool\u0009is\u0009interpretable\u0009and\u0009explainable\u0009ML.\u0009Making\u0009the\u0009models\u0009interpretable\u000d\u000aenables researchers\u0009 to\u0009 understand\u0009 the\u0009 underlying\u0009 prediction\u0009 mechanisms,\u0009 build\u0009 trust\u0009 in\u0009 ML\u0009 model\u0009\u000d\u000aoutcomes,\u0009and\u0009ultimately\u0009uncover\u0009unexpected\u0009correlations\u0009that\u0009may\u0009lead\u0009to\u0009scientific\u0009insights.\u000d\u000aLooking\u0009ahead, there\u0009are\u0009still\u0009several\u0009challenges\u0009and\u0009opportunities\u0009in\u0009the\u0009field\u0009of\u0009ML-assisted\u0009scattering\u0009\u000d\u000aand\u0009spectroscopy\u0009data\u0009analysis.\u0009The\u0009limited\u0009size\u0009of\u0009datasets\u0009in\u0009materials\u0009chemistry\u0009is\u0009a\u0009challenge\u0009for\u0009ML.\u0009\u000d\u000aOne\u0009potential\u0009solution\u0009is\u0009to\u0009constrain\u0009the\u0009ML\u0009model\u0009with chemical\u0009knowledge.\u0009Equivariant\u0009graph-based\u0009\u000d\u000aneural\u0009 networks,\u0009 which\u0009 leverage\u0009 group\u0009 representation\u0009 theory\u0009 to\u0009 design\u0009 architectures\u0009 that\u0009 are\u0009\u000d\u000aequivariant\u0009to\u0009specified\u0009symmetry\u0009groups,\u0009are\u0009particularly\u0009promising.110 Another\u0009potential\u0009solution is\u0009to\u0009\u000d\u000aincorporate\u0009quantum\u0009mechanical\u0009information\u0009into\u0009the\u0009ML\u0009models for\u0009scattering\u0009and\u0009spectroscopy\u0009data\u0009\u000d\u000aanalysis.\u0009Here,\u0009the\u0009added\u0009information\u0009will\u0009assist the\u0009ML\u0009models\u0009to\u0009identify\u0009energy-stable\u0009structures\u0009from\u0009\u000d\u000ascattering\u0009and\u0009spectroscopy\u0009data.\u0009\u0009\u000d\u000aIn\u0009the\u0009next\u0009decade,\u0009I\u0009envision\u0009the\u0009integration\u0009of\u0009ML\u0009into\u0009self-driving\u0009laboratories,\u0009where\u0009the\u0009synthesis\u0009of\u0009\u000d\u000aa\u0009material,\u0009the\u0009performance\u0009of\u0009a\u0009scattering\u0009or\u0009spectroscopy\u0009experiment,\u0009and\u0009the\u0009automated\u0009data analysis\u0009\u000d\u000awith\u0009ML\u0009are\u0009all\u0009interconnected.\u0009The\u0009findings\u0009would\u0009then\u0009be\u0009fed\u0009into\u0009an\u0009active\u0009learning\u0009framework\u0009that\u0009\u000d\u000asuggests\u0009the\u0009next\u0009experiment\u0009based\u0009on\u0009structural\u0009insight.\u0009"," \u000d\u000a171\u000d\u000a6 A\u0009Word\u0009About\u0009Automation\u000d\u000aIf\u0009we\u0009were\u0009on\u0009a\u0009journey\u0009to\u0009define\u0009automation,\u0009we would quickly\u0009realise\u0009that\u0009it\u0009has\u0009various\u0009identities.\u0009For\u0009\u000d\u000asome,\u0009automation\u0009is\u0009a\u0009simple\u0009script\u0009that\u0009automatically\u0009renders\u0009a\u0009plot\u0009from\u0009raw\u0009data.\u0009For\u0009others,\u0009it\u0009takes\u0009\u000d\u000acare\u0009of\u0009the data\u0009analysis pipeline.\u0009For\u0009an\u0009increasing\u0009number\u0009of\u0009people,\u0009it\u0009is\u0009an\u0009advanced\u0009technology\u0009that\u0009\u000d\u000aautomates\u0009the\u0009entire\u0009experimental\u0009cycle. In\u0009my\u0009academic\u0009journey,\u0009my\u0009obsession\u0009with\u0009automation\u0009has\u0009\u000d\u000abeen tied\u0009 to\u0009data\u0009analysis,\u0009specifically\u0009 the\u0009analysis\u0009of\u0009PDF\u0009data.\u0009The\u0009complexity\u0009and\u0009difficulty\u0009of\u0009PDF\u0009\u000d\u000aanalysis,\u0009I\u0009witnessed\u0009during\u0009my\u0009BSc\u0009and\u0009MSc,\u0009have been driving\u0009me\u0009to\u0009streamline\u0009the\u0009process.\u0009I\u0009vision a\u0009\u000d\u000aworld\u0009where\u0009scientists\u0009can analyse\u0009their\u0009data\u0009while\u0009performing\u0009the\u0009experiment\u0009\u2013\u0009a\u0009world\u0009where\u0009realtime\u0009adjustments\u0009are\u0009made\u0009to\u0009the synthesis\u0009during\u0009experiments.\u0009\u0009\u000d\u000aMy\u0009motivation,\u0009however,\u0009has not\u0009solely\u0009been\u0009rooted\u0009in\u0009accelerating\u0009the\u0009data\u0009analysis\u0009process.\u0009I\u0009have\u0009later\u0009\u000d\u000abeen drawn\u0009to\u0009the\u0009larger\u0009impact\u0009\u2013\u0009a\u0009paradigm\u0009shift,\u0009if\u0009you\u0009will.\u0009By\u0009automating\u0009data\u0009analysis,\u0009a\u0009closed\u0009loop\u0009\u000d\u000asystem\u0009can be\u0009established,\u0009which\u0009enables\u0009new\u0009types\u0009of\u0009science\u0009 that\u0009were\u0009previously\u0009unimaginable.\u0009I\u0009\u000d\u000awould\u0009like\u0009to\u0009highlight\u0009three\u0009examples.\u0009First,\u0009we\u0009look\u0009at\u0009the\u0009work\u0009of\u0009Prof.\u0009Lee\u0009Cronin's\u0009lab,\u0009where\u0009the\u0009\u000d\u000adevelopment\u0009of\u0009the\u0009Chemputer\u0009has\u0009automated\u0009the\u0009synthesis\u0009of\u0009molecules.\u0009This\u0009automation\u0009takes\u0009away\u0009\u000d\u000athe\u0009laborious\u0009nature of\u0009manual\u0009synthesis,\u0009opening\u0009the\u0009door\u0009to\u0009complex\u0009molecular\u0009structures\u0009that\u0009could\u0009\u000d\u000arevolutionise\u0009 various\u0009 fields,\u0009 from\u0009 pharmaceuticals\u0009 to\u0009 materials\u0009 science.111-114 Second,\u0009 Prof.\u0009 Sergei\u0009\u000d\u000aKalinin\u0009 and\u0009 his\u0009 collaborators\u0009 have\u0009 illustrated\u0009 the\u0009 transformative\u0009 power\u0009 of\u0009 automation\u0009 in\u0009 electron\u0009\u000d\u000amicroscopy\u0009 for\u0009 structure\u0009 characterisation.\u0009 Their\u0009 work\u0009 has\u0009 pushed the\u0009 boundaries\u0009 of\u0009 what\u0009 we\u0009 can\u0009\u000d\u000avisualise\u0009and\u0009understand\u0009at\u0009 the\u0009atomic\u0009scale.115-117 Lastly,\u0009there\u0009are\u0009remarkable\u0009examples\u0009 from\u0009other\u0009\u000d\u000ateams,\u0009such as\u0009Prof.\u0009Andrew\u0009Cooper\u2019s\u0009group and\u0009the\u0009team\u0009at\u0009the\u0009Materials\u0009Innovation\u0009Factory\u0009and\u0009Prof.\u0009\u000d\u000aAlan\u0009 Aspuru-Guzik\u2019s\u0009 group and\u0009 the\u0009 Acceleration\u0009 Consortium,\u0009 that showcase how\u0009 automation\u0009 can\u0009\u000d\u000aexpedite\u0009 the\u0009 process\u0009 of discovering functional\u0009 materials.\u0009 For\u0009 example,\u0009 Prof.\u0009 Cooper\u2019s\u0009 team\u0009 has\u0009\u000d\u000aeffectively\u0009 used automation\u0009 to\u0009 discover\u0009 new\u0009 photocatalysts118 and\u0009 Prof.\u0009 Aspuru-Guzik\u2019s\u0009 team has\u0009\u000d\u000aaccelerated\u0009the\u0009discovery of\u0009thin\u0009film\u0009materials.\u000d\u000a119\u0009\u0009These\u0009examples\u0009are\u0009testaments\u0009to\u0009how\u0009automation\u0009\u000d\u000aenables\u0009 us\u0009 to\u0009 dive into\u0009 scientific\u0009 territories\u0009 previously\u0009 thought\u0009 unattainable.\u0009 The\u0009 tools\u0009 and\u0009\u000d\u000amethodologies\u0009 will evolve\u0009 with\u0009 time and one\u0009 thing\u0009 is clear\u0009 -\u0009 automation\u0009 is\u0009 a\u0009 catalyst\u0009 for\u0009 scientific\u0009\u000d\u000arevolution. This\u0009new\u0009degree\u0009of\u0009automation,\u0009self-driving\u0009laboratories, not\u0009only\u0009complements\u0009my\u0009initial\u0009\u000d\u000ainspiration for\u0009automation\u0009but\u0009has\u0009sparked\u0009a\u0009new\u0009inspiration in me.\u0009\u000d\u000aThere\u0009was\u0009one\u0009aspect\u0009I\u0009once\u0009thought\u0009impossible\u0009to\u0009automate\u0009-\u0009the\u0009communication\u0009of\u0009science,\u0009especially\u0009\u000d\u000aas\u0009learning\u0009English\u0009was\u0009a\u0009significant\u0009challenge\u0009 for\u0009me.\u0009During\u0009college,\u0009 I\u0009took two\u0009years\u0009extra\u0009English\u0009\u000d\u000alessons.\u0009Though\u0009 I\u0009have\u0009greatly\u0009improved my\u0009language\u0009skills\u0009since\u0009then,\u0009 I have discovered\u0009that\u0009recent\u0009\u000d\u000aadvances\u0009in\u0009generative\u0009modelling\u0009have\u0009significantly\u0009eased\u0009the\u0009process\u0009of\u0009scientific\u0009writing.\u0009Several\u0009AI\u0009\u000d\u000atools\u0009greatly\u0009aided\u0009the\u0009editing\u0009of\u0009this\u0009dissertation.\u0009For\u0009instance,\u0009OpenAI's\u0009language\u0009model,\u0009ChatGPT\u0009v4,120\u000d\u000acontributed\u0009to\u0009refining\u0009and\u0009giving feedback\u0009on small\u0009pieces\u0009of text that\u0009I\u0009provided and Quillbot121 and\u0009\u000d\u000aGrammarly122 played\u0009a\u0009critical\u0009role\u0009in\u0009providing\u0009feedback\u0009on grammar\u0009and\u0009sentence\u0009constructions.\u0009Of\u0009\u000d\u000acourse,\u0009 the\u0009 outputs\u0009 from\u0009 these\u0009 tools\u0009 were\u0009 rigorously\u0009 reviewed\u0009 and\u0009 extensively\u0009 modified\u0009 by\u0009 me\u0009 to\u0009\u000d\u000apreserve\u0009scientific\u0009accuracy\u0009and\u0009appropriateness. In\u0009parallel,\u0009the\u0009automation\u0009of\u0009image\u0009generation\u0009has\u0009\u000d\u000ashown\u0009immense\u0009potential.\u0009The\u0009front\u0009covers\u0009in\u0009this\u0009dissertation\u0009have\u0009been\u0009drawn by\u0009AI\u0009tools and\u0009many\u0009\u000d\u000aother\u0009 figures\u0009 have\u0009 elements\u0009 made\u0009 by\u0009 AI.\u0009 Midjourney123 and\u0009 Real-ESRGAN124 were\u0009 instrumental\u0009 in\u0009\u000d\u000acreating\u0009and\u0009adapting\u0009figures\u0009and\u0009enhancing\u0009image\u0009resolution.\u0009Again,\u0009as\u0009with\u0009text,\u0009the\u0009output\u0009from\u0009these\u0009\u000d\u000atools\u0009underwent\u0009careful review\u0009and\u0009editing\u0009to\u0009ensure\u0009they\u0009accurately\u0009portrayed\u0009the\u0009discussed\u0009concepts.\u0009\u0009\u0009\u000d\u000aThus,\u0009 to\u0009 conclude,\u0009automation\u0009 has\u0009 been\u0009 the\u0009invisible\u0009 hand\u0009guiding\u0009my\u0009 PhD\u0009journey,\u0009 from\u0009aiding\u0009my\u0009\u000d\u000aresearch\u0009and\u0009data\u0009analysis\u0009to\u0009enhancing\u0009my\u0009scientific\u0009communication.\u0009It\u0009embodies\u0009the\u0009spirit\u0009of\u0009progress,\u0009\u000d\u000acontinually\u0009pushing\u0009boundaries\u0009to\u0009shape\u0009a\u0009future\u0009where we\u0009work\u0009more\u0009intelligently\u0009and\u0009effectively.\u0009",""," \u000d\u000a173\u000d\u000aReferences\u0009\u000d\u000a1 Anker, A. S. Pushing the boundaries of Nanocluster Modelling from Pair Distribution Function and Small-Angle \u000d\u000aX-ray Scattering Analysis Master\u2019s Thesis, Department of Chemistry, University of Copenhagen (2021).\u000d\u000a2 David, W. I. F. & Shankland, K. Structure determination from powder diffraction data. Acta Crystallogr. A 64, \u000d\u000a52-64, doi:10.1107/S0108767307064252 (2008).\u000d\u000a3 Cheetham, A. K. & Goodwin, A. L. Crystallography with powders. Nat. Mater. 13, 760-762, \u000d\u000adoi:10.1038/nmat4044 (2014).\u000d\u000a4 Billinge, S. J. L. & Levin, I. The Problem with Determining Atomic Structure at the Nanoscale. Science 316, \u000d\u000a561-565, doi:10.1126/science.1135080 (2007).\u000d\u000a5 Billinge, S. J. L. & Kanatzidis, M. G. Beyond crystallography: the study of disorder, nanocrystallinity and \u000d\u000acrystallographically challenged materials with pair distribution functions. Chem. Commun. 7, 749-760, \u000d\u000adoi:10.1039/b309577k (2004).\u000d\u000a6 Keen, D. A. & Goodwin, A. L. The crystallography of correlated disorder. Nature 521, 303-309, \u000d\u000adoi:10.1038/nature14453 (2015).\u000d\u000a7 Christiansen, T. L., Cooper, S. R. & Jensen, K. M. \u00d8. There's no place like real-space: elucidating size-dependent \u000d\u000aatomic structure of nanomaterials using pair distribution function analysis. Nanoscale Adv. 2, 2234-2254, \u000d\u000adoi:10.1039/D0NA00120A (2020).\u000d\u000a8 Juelsholt, M. et al. Size-induced amorphous structure in tungsten oxide nanoparticles. Nanoscale 13, 20144-\u000d\u000a20156, doi:10.1039/D1NR05991B (2021).\u000d\u000a9 Yang, X. et al. Confirmation of disordered structure of ultrasmall CdSe nanoparticles from X-ray atomic pair \u000d\u000adistribution function analysis. Phys. Chem. Chem. Phys. 15, 8480-8486, doi:10.1039/C3CP00111C (2013).\u000d\u000a10 Christiansen, T. L. et al. Structure analysis of supported disordered molybdenum oxides using pair distribution \u000d\u000afunction analysis and automated cluster modelling. J. Appl. Cryst. 53, 148-158, \u000d\u000adoi:10.1107/S1600576719016832 (2020).\u000d\u000a11 Bennett, T. D. & Cheetham, A. K. Amorphous Metal\u2013Organic Frameworks. Acc. Chem. Res. 47, 1555-1562, \u000d\u000adoi:10.1021/ar5000314 (2014).\u000d\u000a12 Farrow, C., Shi, C., Juh\u00e1s, P., Peng, X. & Billinge, S. J. L. Robust structure and morphology parameters for CdS \u000d\u000ananoparticles by combining small-angle X-ray scattering and atomic pair distribution function data in a complex \u000d\u000amodeling framework. J. Appl. Cryst. 47, 561-565, doi:10.1107/S1600576713034055 (2014).\u000d\u000a13 Juh\u00e1s, P., Farrow, C. L., Yang, X., Knox, K. R. & Billinge, S. J. L. Complex modeling: a strategy and software \u000d\u000aprogram for combining multiple information sources to solve ill posed structure and nanostructure inverse \u000d\u000aproblems. Acta Crystallogr. A 71, 562-568, doi:10.1107/S2053273315014473 (2015).\u000d\u000a14 Farrow, C. L. & Billinge, S. J. L. Relationship between the atomic pair distribution function and small-angle \u000d\u000ascattering: implications for modeling of nanoparticles. Acta Crystallogr. A 65, 232-239, \u000d\u000adoi:10.1107/S0108767309009714 (2009).\u000d\u000a15 Scardi, P., Billinge, S. J. L., Neder, R. & Cervellino, A. Celebrating 100 years of the Debye scattering equation. \u000d\u000aActa Crystallogr. A 72, 589-590, doi:10.1107/S2053273316015680 (2016).\u000d\u000a16 Debye, P. Zerstreuung von R\u00f6ntgenstrahlen. Annalen der Physik 351, 809-823, doi:10.1002/andp.19153510606 \u000d\u000a(1915).\u000d\u000a17 Woolfson, M. M. An Introduction to X-ray Crystallography. 2 edn, (Cambridge University Press, 1997).\u000d\u000a18 Dinnebier, R. E. et al. Powder Diffraction : Theory and Practice. (Royal Society of Chemistry, 2008).\u000d\u000a19 Prince, E. (Third edition. Dordrecht ; Boston ; London : Published for the International Union of \u000d\u000aCrystallography by Kluwer Academic Publishers., 2004).\u000d\u000a20 Egami, T. & Billinge, S. J. L. Underneath the Bragg Peaks, Pergamon (2012).","|\u0009References\u000d\u000a174\u000d\u000a21 Als-Nielsen, J. & McMorrow, D. Elements of Modern X-Ray Physics. (John Wiley & Sons, Incorporated, 2011).\u000d\u000a22 Scardi, P. & Gelisio, L. Vibrational Properties of Nanocrystals from the Debye Scattering Equation. Sci. Rep. 6, \u000d\u000a22221, doi:10.1038/srep22221 (2016).\u000d\u000a23 Juh\u00e1s, P., Farrow, C. L., Yang, X., Knox, K. R. & Billinge, S. J. L. Complex modeling: a strategy and software \u000d\u000aprogram for combining multiple information sources to solve ill posed structure and nanostructure inverse \u000d\u000aproblems. Acta Crystallographica A 71, 562-568, doi:10.1107/S2053273315014473 (2015).\u000d\u000a24 Anker, A. S. et al. Structural Changes during the Growth of Atomically Precise Metal Oxido Nanoclusters from \u000d\u000aCombined Pair Distribution Function and Small-Angle X-ray Scattering Analysis. Angew. Chem. Int. Ed. 60, 2-\u000d\u000a12, doi:10.1002/anie.202103641 (2021).\u000d\u000a25 Juhas, P., Davis, T., Farrow, C. L. & Billinge, S. J. L. PDFgetX3: a rapid and highly automatable program for \u000d\u000aprocessing powder diffraction data into total scattering pair distribution functions. J. Appl. Cryst. 46, 560-566, \u000d\u000adoi:10.1107/S0021889813005190 (2013).\u000d\u000a26 Chupas, P. J. et al. Rapid-acquisition pair distribution function (RA-PDF) analysis. J. Appl. Cryst. 36, 1342-\u000d\u000a1347, doi:10.1107/S0021889803017564 (2003).\u000d\u000a27 Mehring, M., Mansfeld, D., Paalasmaa, S. & Sch\u00fcrmann, M. Polynuclear Bismuth\u2010Oxo Clusters: Insight into the \u000d\u000aFormation Process of a Metal Oxide. Chem. Eur. J. 12, 1767-1781, doi:10.1002/chem.200500857 (2006).\u000d\u000a28 Mansfeld, D., Mehring, M. & Sch\u00fcrmann, M. From a Monomeric Bismuth Silanolate to a Molecular Bismuth \u000d\u000aOxo Cluster: [Bi22O26(OSiMe2tBu)14]. Angew. Chem. Int. Ed. 44, 245-249, doi:10.1002/anie.200461476 (2005).\u000d\u000a29 Wang, C., Steiner, U. & Sepe, A. Synchrotron Big Data Science. Small 14, e1802291, \u000d\u000adoi:10.1002/smll.201802291 (2018).\u000d\u000a30 Dong, H. et al. A deep convolutional neural network for real-time full profile analysis of big powder diffraction \u000d\u000adata. npj Comput. Mater. 7, 74, doi:10.1038/s41524-021-00542-4 (2021).\u000d\u000a31 Xia, M. et al. Lab-Scale In Situ X-Ray Diffraction Technique for Different Battery Systems: Designs, \u000d\u000aApplications, and Perspectives. Small Methods 3, 1900119, doi:10.1002/smtd.201900119 (2019).\u000d\u000a32 Ebensperger, T., Stahlhut, P., Nachtrab, F., Zabler, S. & Hanke, R. Comparison of different sources for laboratory \u000d\u000aX-ray microscopy. J. Instrum. 7, C10008, doi:10.1088/1748-0221/7/10/c10008 (2012).\u000d\u000a33 Hornberger, B., Kasahara, J., Gifford, M., Ruth, R. & Loewen, R. in SPIE Optical Engineering + Applications\u000d\u000aVol. 11110 (SPIE, 2019).\u000d\u000a34 Nelson, J. et al. In Operando X-ray Diffraction and Transmission X-ray Microscopy of Lithium Sulfur Batteries. \u000d\u000aJ. Am. Chem. Soc. 134, 6337-6343, doi:10.1021/ja2121926 (2012).\u000d\u000a35 Elia, G. A. et al. Simultaneous X-Ray Diffraction and Tomography Operando Investigation of \u000d\u000aAluminum/Graphite Batteries. Adv. Funct. Mater. 30, 2003913, doi:10.1002/adfm.202003913 (2020).\u000d\u000a36 Kj\u00e6r, E. T. S. et al. In Situ Studies of the Formation of Tungsten and Niobium Oxide Nanoparticles: Towards \u000d\u000aAutomated Analysis of Reaction Pathways from PDF Analysis using the Pearson Correlation Coefficient. Chem. \u000d\u000aMethods 2, e202200034, doi:10.1002/cmtd.202200034 (2022).\u000d\u000a37 Bergerhoff, G., Brown, I. & Allen, F. Crystallographic databases. International Union of Crystallography, \u000d\u000aChester 360, 77-95 (1987).\u000d\u000a38 Gra\u017eulis, S., Merkys, A. & Vaitkus, A. in Handbook of Materials Modeling : Methods: Theory and Modeling \u000d\u000a(eds Wanda Andreoni & Sidney Yip) 1-19 (Springer International Publishing, 2018).\u000d\u000a39 Mathiesen, J. K. et al. Simple Setup Miniaturization with Multiple Benefits for Green Chemistry in Nanoparticle \u000d\u000aSynthesis. ACS Omega 7, 4714-4721, doi:10.1021/acsomega.2c00030 (2022).\u000d\u000a40 Mathiesen, J. K. et al. Breaking with the Principles of Coreduction to Form Stoichiometric Intermetallic PdCu \u000d\u000aNanoparticles. Small Methods 6, 2200420, doi:10.1002/smtd.202200420 (2022).\u000d\u000a41 Pauw, B. R. et al. The human factor: results of a small-angle scattering data analysis Round Robin. arXiv preprint \u000d\u000aarXiv:2303.03772, doi:10.48550/arXiv.2303.03772 (2023)."," \u000d\u000a175\u000d\u000a42 Lindh\u00f8j, M. B. et al. Iron Oxide Nanoparticles as a Contrast Agent for Synchrotron Imaging of Sperm. submitted \u000d\u000ato Heliyon (2023).\u000d\u000a43 Barto\u0161, D. et al. Synthesis of fluorescent polystyrene nanoparticles: a reproducible and scalable method. PeerJ \u000d\u000aMaterials Science 4, e22, doi:10.7717/peerj-matsci.22 (2022).\u000d\u000a44 Aalling-Frederiksen, O., Juelsholt, M., Anker, A. S. & Jensen, K. M. \u00d8. Formation and growth mechanism for \u000d\u000aniobium oxide nanoparticles: atomistic insight from in situ X-ray total scattering. Nanoscale 13, 8087-8097, \u000d\u000adoi:10.1039/D0NR08299F (2021).\u000d\u000a45 Xu, H., Sommer, S., Broge, N. L. N., Gao, J. & Iversen, B. B. The Chemistry of Nucleation: In Situ Pair \u000d\u000aDistribution Function Analysis of Secondary Building Units During UiO-66 MOF Formation. Chem. Eur. J. 25, \u000d\u000a2051-2058, doi:10.1002/chem.201805024 (2019).\u000d\u000a46 Anker, A. S., Butler, K. T., Selvan, R. & Jensen, K. M. \u00d8. Machine Learning for Analysis of Experimental \u000d\u000aScattering and Spectroscopy Data in Materials Chemistry. Preprint at \u000d\u000ahttps://chemrxiv.org/engage/chemrxiv/article-details/64ed9d6079853bbd78a73fb0. doi:10.26434/chemrxiv2023-27t53 (2023).\u000d\u000a47 Yang, L., Juh\u00e1s, P., Terban, M. W., Tucker, M. G. & Billinge, S. J. L. Structure-mining: screening structure \u000d\u000amodels by automated fitting to the atomic pair distribution function over large numbers of models. Acta \u000d\u000aCrystallogr. A 76, 395-409, doi:10.1107/S2053273320002028 (2020).\u000d\u000a48 Banerjee, S. et al. Cluster-mining: an approach for determining core structures of metallic nanoparticles from \u000d\u000aatomic pair distribution function data. Acta Crystallogr. A 76, 24-31, doi:10.1107/S2053273319013214 (2020).\u000d\u000a49 Lindahl Christiansen, T. et al. Structure analysis of supported disordered molybdenum oxides using pair \u000d\u000adistribution function analysis and automated cluster modelling. J. Appl. Cryst 53, 148-158, \u000d\u000adoi:10.1107/S1600576719016832 (2020).\u000d\u000a50 Lazarini, F. The crystal structure of a bismuth basic nitrate, [Bi6O5(OH)3](NO3)5\u00b73H2O. Acta Crystallogr. B 34, \u000d\u000a3169-3173, doi:10.1107/S0567740878010419 (1978).\u000d\u000a51 Krayzman, V., Levin, I. & Tucker, M. G. Simultaneous reverse Monte Carlo refinements of local structures in \u000d\u000aperovskite solid solutions using EXAFS and the total scattering pair-distribution function. J. Appl. Cryst., \u000d\u000adoi:10.1107/S0021889808013277 (2008).\u000d\u000a52 Terban, M. W. & Billinge, S. J. L. Structural Analysis of Molecular Materials Using the Pair Distribution \u000d\u000aFunction. Chem. Rev. 122, 1208-1272, doi:10.1021/acs.chemrev.1c00237 (2022).\u000d\u000a53 Agrawal, A. & Choudhary, A. Perspective: Materials informatics and big data: Realization of the \u201cfourth \u000d\u000aparadigm\u201d of science in materials science. APL Mater. 4, 053208, doi:10.1063/1.4946894 (2016).\u000d\u000a54 Butler, K. T., Davies, D. W., Cartwright, H., Isayev, O. & Walsh, A. Machine learning for molecular and \u000d\u000amaterials science. Nature 559, 547-555, doi:10.1038/s41586-018-0337-2 (2018).\u000d\u000a55 Pacchioni, G. An upgrade to a bright future. Nat. Rev. Phys. 1, 100-101, doi:10.1038/s42254-019-0019-5 (2019).\u000d\u000a56 Chapman, H. Fourth-generation light sources. IUCrJ 10, 246-247, doi:10.1107/S2052252523003585 (2023).\u000d\u000a57 Sullivan, B. et al. BraggNet: integrating Bragg peaks using neural networks. J. Appl. Cryst. 52, 854-863, \u000d\u000adoi:10.1107/S1600576719008665 (2019).\u000d\u000a58 Ke, T. W. et al. A convolutional neural network-based screening tool for X-ray serial crystallography. J \u000d\u000aSynchrotron Radiat 25, 655-670, doi:10.1107/s1600577518004873 (2018).\u000d\u000a59 Doucet, M. et al. Machine learning for neutron scattering at ORNL*. Mach. learn.: sci. technol. 2, 023001, \u000d\u000adoi:10.1088/2632-2153/abcf88 (2021).\u000d\u000a60 Aguiar, J. A., Gong, M. L., Unocic, R. R., Tasdizen, T. & Miller, B. D. Decoding crystallography from highresolution electron imaging and diffraction datasets with deep learning. Sci. Adv. 5, eaaw1949, \u000d\u000adoi:10.1126/sciadv.aaw1949 (2019).","|\u0009References\u000d\u000a176\u000d\u000a61 Muto, S. & Shiga, M. Application of machine learning techniques to electron microscopic/spectroscopic image \u000d\u000adata analysis. Microscopy 69, 110-122, doi:10.1093/jmicro/dfz036 (2019).\u000d\u000a62 Wang, H. et al. Rapid Identification of X-ray Diffraction Patterns Based on Very Limited Data by Interpretable \u000d\u000aConvolutional Neural Networks. J. Chem. Inf. Model 60, 2004-2011, doi:10.1021/acs.jcim.0c00020 (2020).\u000d\u000a63 Tatlier, M. Artificial neural network methods for the prediction of framework crystal structures of zeolites from \u000d\u000aXRD data. Neural. Comput. Appl. 20, 365-371, doi:10.1007/s00521-010-0386-4 (2011).\u000d\u000a64 Bunn, J. K., Hu, J. & Hattrick-Simpers, J. R. Semi-Supervised Approach to Phase Identification from \u000d\u000aCombinatorial Sample Diffraction Patterns. JOM 68, 2116-2125, doi:10.1007/s11837-016-2033-8 (2016).\u000d\u000a65 Oviedo, F. et al. Fast and interpretable classification of small X-ray diffraction datasets using data augmentation \u000d\u000aand deep neural networks. npj Comput. Mater. 5, 60, doi:10.1038/s41524-019-0196-x (2019).\u000d\u000a66 Butler, K. T., Le, M. D., Thiyagalingam, J. & Perring, T. G. Interpretable, calibrated neural networks for analysis \u000d\u000aand understanding of inelastic neutron scattering data. J. Phys.: Condens. Matter 33, 194006, doi:10.1088/1361-\u000d\u000a648x/abea1c (2021).\u000d\u000a67 Anker, A. S., Kj\u00e6r, E. T., Juelsholt, M. & Jensen, K. M. \u00d8. POMFinder: Identifying polyoxometalate cluster \u000d\u000astructures from pair distribution function data using explainable machine learning. Preprint at \u000d\u000ahttps://chemrxiv.org/engage/chemrxiv/article-details/64e5fef7dd1a73847f5951b9. doi:10.26434/chemrxiv2023-91xz7 (2023).\u000d\u000a68 Gumerova, N. I. & Rompel, A. Synthesis, structures and applications of electron-rich polyoxometalates. Nat. \u000d\u000aRev. Chem. 2, 0112, doi:10.1038/s41570-018-0112 (2018).\u000d\u000a69 Gumerova, N. I. & Rompel, A. Polyoxometalates in solution: speciation under spotlight. Chem. Soc. Rev., \u000d\u000adoi:10.1039/D0CS00392A (2020).\u000d\u000a70 Miras, H. N., Yan, J., Long, D.-L. & Cronin, L. Engineering polyoxometalates with emergent properties. Chem. \u000d\u000aSoc. Rev. 41, 7403-7430, doi:10.1039/C2CS35190K (2012).\u000d\u000a71 Long, D.-L., Tsunashima, R. & Cronin, L. Polyoxometalates: Building Blocks for Functional Nanoscale Systems. \u000d\u000aAngew. Chem. Int. Ed. 49, 1736-1758, doi:10.1002/anie.200902483 (2010).\u000d\u000a72 Juelsholt, M., Lindahl Christiansen, T. & Jensen, K. M. \u00d8. Mechanisms for Tungsten Oxide Nanoparticle \u000d\u000aFormation in Solvothermal Synthesis: From Polyoxometalates to Crystalline Materials. J. Phys. Chem. C 123, \u000d\u000a5110-5119, doi:10.1021/acs.jpcc.8b12395 (2019).\u000d\u000a73 Grinsztajn, L., Oyallon, E. & Varoquaux, G. Why do tree-based models still outperform deep learning on typical \u000d\u000atabular data? Advances in Neural Information Processing Systems 35, 507-520, doi:10.48550/arXiv.2207.08815 \u000d\u000a(2022).\u000d\u000a74 Saini, M. K. et al. Dimeric complexes of rare-earth substituted Keggin-type silicotungstates: syntheses, crystal \u000d\u000astructure and solid state properties. RSC Adv. 4, 25357-25364, doi:10.1039/C4RA02751E (2014).\u000d\u000a75 Glass, E. N. et al. Extending Metal-to-Polyoxometalate Charge Transfer Lifetimes: The Effect of Heterometal \u000d\u000aLocation. Chem. Eur. J. 20, 4297-4307, doi:10.1002/chem.201304119 (2014).\u000d\u000a76 Son, J. H., Kwon, Y.-U. & Han, O. H. New Ionic Crystals of Oppositely Charged Cluster Ions and Their \u000d\u000aCharacterization. Inorg. Chem. 42, 4153-4159, doi:10.1021/ic0340377 (2003).\u000d\u000a77 Han, X.-B. et al. A 3D purely inorganic porous framework based on Keggin polyoxoanions. Inorg. Chem. \u000d\u000aCommun. 18, 47-49, doi:10.1016/j.inoche.2012.01.006 (2012).\u000d\u000a78 Uchida, S., Kawamoto, R. & Mizuno, N. Recognition of Small Polar Molecules with an Ionic Crystal of \u03b1Keggin-Type Polyoxometalate with a Macrocation. Inorg. Chem. 45, 5136-5144, doi:10.1021/ic060684x (2006).\u000d\u000a79 Juh\u00e1s, P., Cherba, D. M., Duxbury, P. M., Punch, W. F. & Billinge, S. J. L. Ab initio determination of solid-state \u000d\u000ananostructure. Nature 440, 655-658, doi:10.1038/nature04556 (2006).\u000d\u000a80 Juh\u00e1s, P., Granlund, L., Duxbury, P. M., Punch, W. F. & Billinge, S. J. L. The Liga algorithm for ab initio \u000d\u000adetermination of nanostructure. Acta Crystallogr. A 64, 631-640, doi:10.1107/s0108767308027591 (2008)."," \u000d\u000a177\u000d\u000a81 Juhas, P., Granlund, L., Gujarathi, S. R., Duxbury, P. M. & Billinge, S. J. L. Crystal structure solution from \u000d\u000aexperimentally determined atomic pair distribution functions. J. Appl. Cryst. 43, 623-629, \u000d\u000adoi:10.1107/S002188981000988X (2010).\u000d\u000a82 Cliffe, M. J., Dove, M. T., Drabold, D. & Goodwin, A. L. Structure determination of disordered materials from \u000d\u000adiffraction data. Phys. Rev. Lett. 104, 125501, doi:10.1103/PhysRevLett.104.125501 (2010).\u000d\u000a83 Cliffe, M. J. & Goodwin, A. L. Nanostructure determination from the pair distribution function: a parametric \u000d\u000astudy of the INVERT approach. J. Phys.: Condens. Matter 25, 454218, doi:10.1088/0953-8984/25/45/454218 \u000d\u000a(2013).\u000d\u000a84 Anker, A. S. et al. Extracting structural motifs from pair distribution function data of nanostructures using \u000d\u000aexplainable machine learning. npj Comput. Mater. 8, 213, doi:10.1038/s41524-022-00896-3 (2022).\u000d\u000a85 Chen, T. & Guestrin, C. XGBoost: A Scalable Tree Boosting System. Proceedings of the 22nd ACM SIGKDD \u000d\u000aInternational Conference on Knowledge Discovery and Data Mining, 785\u2013794, doi:10.1145/2939672.2939785 \u000d\u000a(2016).\u000d\u000a86 Lundberg, S. M. & Lee, S.-I. A Unified Approach to Interpreting Model Predictions. Proceedings of the 31st\u000d\u000aInternational Conference on Neural Information Processing Systems, 4765-4774, \u000d\u000adoi:10.48550/arXiv.1705.07874 (2017).\u000d\u000a87 Lundberg, S. M. et al. From local explanations to global understanding with explainable AI for trees. Nat. Mach. \u000d\u000aIntell. 2, 56-67, doi:10.1038/s42256-019-0138-9 (2020).\u000d\u000a88 Magnard, N. P. L., Anker, A. S., Aalling-Frederiksen, O., Kirsch, A. & Jensen, K. M. \u00d8. Characterisation of \u000d\u000aintergrowth in metal oxide materials using structure-mining: the case of \u03b3-MnO2. Dalton Trans. 51, 17150-17161, \u000d\u000adoi:10.1039/D2DT02153F (2022).\u000d\u000a89 Skj\u00e6rv\u00f8, S. L. et al. Atomic structural changes in the formation of transition metal tungstates: The role of \u000d\u000apolyoxometalate structures in material crystallization. Chem. Sci. 14, 4806-4816, doi:10.1039/d3sc00426k \u000d\u000a(2023).\u000d\u000a90 Kingma, D. P. & Welling, M. Auto-encoding variational bayes. arXiv preprint arXiv:1312.6114, \u000d\u000adoi:10.48550/arXiv.1312.6114 (2013).\u000d\u000a91 Schmidt, J., Marques, M. R. G., Botti, S. & Marques, M. A. L. Recent advances and applications of machine \u000d\u000alearning in solid-state materials science. npj Comput. Mater. 5, 83, doi:10.1038/s41524-019-0221-0 (2019).\u000d\u000a92 Anker, A. S. et al. Characterising the Atomic Structure of Mono-Metallic Nanoparticles from X-Ray Scattering \u000d\u000aData Using Conditional Generative Models in Proceedings of the 16th International Workshop on Mining and \u000d\u000aLearning with Graphs (MLG). doi:10.26434/chemrxiv.12662222.v1 (2020).\u000d\u000a93 Kj\u00e6r, E. T. S. et al. DeepStruc: towards structure solution from pair distribution function data using deep \u000d\u000agenerative models. Digital Discovery 2, 69-80, doi:10.1039/D2DD00086E (2023).\u000d\u000a94 Bronstein, M. M., Bruna, J., LeCun, Y., Szlam, A. & Vandergheynst, P. Geometric deep learning: going beyond \u000d\u000aeuclidean data. IEEE Signal Processing Magazine 34, 18-42, doi:10.1109/MSP.2017.2693418 (2017).\u000d\u000a95 Scarselli, F., Gori, M., Tsoi, A. C., Hagenbuchner, M. & Monfardini, G. The graph neural network model. IEEE \u000d\u000atransactions on neural networks 20, 61-80, doi:10.1109/TNN.2008.2005605 (2008).\u000d\u000a96 Reiser, P. et al. Graph neural networks for materials science and chemistry. Commun. Mater. 3, 93, \u000d\u000adoi:10.1038/s43246-022-00315-6 (2022).\u000d\u000a97 Schmid, P. & Lynn, J. Results of the 3 November 1974 applications technology satellite-6 (ATS-6) trilateration \u000d\u000atest. (https://ntrs.nasa.gov/citations/19750015584, 1975).\u000d\u000a98 Quinson, J. et al. Spatially Localized Synthesis and Structural Characterization of Platinum Nanocrystals \u000d\u000aObtained Using UV Light. ACS Omega 3, 10351-10356, doi:10.1021/acsomega.8b01613 (2018).\u000d\u000a99 Jensen, K. M. \u00d8. et al. Polymorphism in magic-sized Au144(SR)60 clusters. Nat. Commun. 7, 11859, \u000d\u000adoi:10.1038/ncomms11859 (2016).","|\u0009References\u000d\u000a178\u000d\u000a100 Marks, L. D. Surface structure and energetics of multiply twinned particles. Philos. Mag. A 49, 81-93, \u000d\u000adoi:10.1080/01418618408233431 (1984).\u000d\u000a101 Banerjee, S. et al. Improved Models for Metallic Nanoparticle Cores from Atomic Pair Distribution Function \u000d\u000a(PDF) Analysis. J. Phys. Chem. C 122, 29498-29506, doi:10.1021/acs.jpcc.8b05897 (2018).\u000d\u000a102 Goodfellow, I. et al. Generative adversarial nets. Advances in neural information processing systems 27, \u000d\u000adoi:10.48550/arXiv.1406.2661 (2014).\u000d\u000a103 Anker, A. S., Butler, K. T., Le, M. D., Perring, T. G. & Thiyagalingam, J. Using generative adversarial networks \u000d\u000ato match experimental and simulated inelastic neutron scattering data. Digital Discovery 2, 578-590, \u000d\u000adoi:10.1039/D2DD00147K (2023).\u000d\u000a104 Yi, Z., Zhang, H., Tan, P. & Gong, M. Dualgan: Unsupervised dual learning for image-to-image translation in \u000d\u000aProceedings of the IEEE international conference on computer vision. 2849-2857, \u000d\u000adoi:10.48550/arXiv.1704.02510 (2017).\u000d\u000a105 Zhu, J.-Y., Park, T., Isola, P. & Efros, A. A. Unpaired image-to-image translation using cycle-consistent \u000d\u000aadversarial networks in Proceedings of the IEEE international conference on computer vision. 2223-2232, \u000d\u000adoi:10.48550/arXiv.1703.10593 (2017).\u000d\u000a106 Kim, T., Cha, M., Kim, H., Lee, J. K. & Kim, J. Learning to discover cross-domain relations with generative \u000d\u000aadversarial networks in International conference on machine learning. 1857-1865, \u000d\u000adoi:10.48550/arXiv.1703.05192 (2017).\u000d\u000a107 Han, J., Shoeiby, M., Petersson, L. & Armin, M. A. Dual Contrastive Learning for Unsupervised Image-to-Image \u000d\u000aTranslation in Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition. 746-755, \u000d\u000adoi:10.48550/arXiv.2104.07689 (2021).\u000d\u000a108 Park, T., Efros, A. A., Zhang, R. & Zhu, J.-Y. Contrastive learning for unpaired image-to-image translation in \u000d\u000aEuropean Conference on Computer Vision. 319-345, doi:10.1007/978-3-030-58545-7_19 (2020).\u000d\u000a109 Cuturi, M. Sinkhorn distances: Lightspeed computation of optimal transport. Advances in neural information \u000d\u000aprocessing systems 26, doi:10.48550/arXiv.1306.0895 (2013).\u000d\u000a110 Kaba, S.-O. & Ravanbakhsh, S. Equivariant Networks for Crystal Structures in Advances in Neural Information \u000d\u000aProcessing Systems. doi:10.48550/arXiv.2211.15420 (2022).\u000d\u000a111 Mehr, S. H. M., Craven, M., Leonov, A. I., Keenan, G. & Cronin, L. A universal system for digitization and \u000d\u000aautomatic execution of the chemical synthesis literature. Science 370, 101-108, doi:10.1126/science.abc2986 \u000d\u000a(2020).\u000d\u000a112 Manzano, J. S. et al. An autonomous portable platform for universal chemical synthesis. Nat. Chem. 14, 1311-\u000d\u000a1318, doi:10.1038/s41557-022-01016-w (2022).\u000d\u000a113 Jiang, Y. et al. An artificial intelligence enabled chemical synthesis robot for exploration and optimization of \u000d\u000ananomaterials. Sci. Adv. 8, eabo2626, doi:10.1126/sciadv.abo2626 (2022).\u000d\u000a114 Rohrbach, S. et al. Digitization and validation of a chemical synthesis literature database in the ChemPU. Science\u000d\u000a377, 172-180, doi:10.1126/science.abo0058 (2022).\u000d\u000a115 Ziatdinov, M., Ghosh, A., Wong, C. Y. & Kalinin, S. V. AtomAI framework for deep learning analysis of image \u000d\u000aand spectroscopy data in electron and scanning probe microscopy. Nat. Mach. Intell. 4, 1101-1112, \u000d\u000adoi:10.1038/s42256-022-00555-8 (2022).\u000d\u000a116 Liu, Y. et al. Automated Experiments of Local Non-Linear Behavior in Ferroelectric Materials. Small 18, \u000d\u000a2204130, doi:10.1002/smll.202204130 (2022).\u000d\u000a117 Roccapriore, K. M., Kalinin, S. V. & Ziatdinov, M. Physics Discovery in Nanoplasmonic Systems via \u000d\u000aAutonomous Experiments in Scanning Transmission Electron Microscopy. Adv. Sci. 9, 2203422, \u000d\u000adoi:10.1002/advs.202203422 (2022).\u000d\u000a118 Burger, B. et al. A mobile robotic chemist. Nature 583, 237-241, doi:10.1038/s41586-020-2442-2 (2020)."," \u000d\u000a179\u000d\u000a119 MacLeod, B. P. et al. Self-driving laboratory for accelerated discovery of thin-film materials. Sci. Adv. 6, \u000d\u000aeaaz8867, doi:10.1126/sciadv.aaz8867 (2020).\u000d\u000a120 https://openai.com/gpt-4. \u000d\u000a121 https://quillbot.com/. \u000d\u000a122 https://app.grammarly.com/. \u000d\u000a123 https://docs.midjourney.com/. \u000d\u000a124 Wang, X., Xie, L., Dong, C. & Shan, Y. Real-esrgan: Training real-world blind super-resolution with pure \u000d\u000asynthetic data in Proceedings of the IEEE/CVF international conference on computer vision. 1905-1914, \u000d\u000adoi:10.48550/arXiv.2107.10833 (2021).","","181\u000d\u000aUNIVERSIT Y\u0009OF\u0009COPENHAGEN\u000d\u000aFACULTY\u0009OF\u0009 SCIENCE\u000d\u000aDEPARTMENT\u0009OF\u0009CHEMISTRY\u0009&\u0009NANO - SCIENCE\u0009CENTER\u000d\u000aAPPENDICES\u000d\u000aAppendix\u0009I\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00091\u000d\u000aSimulation details\u0009of\u0009Q-space\u0009and\u0009r-space\u0009data\u0009of\u0009the [Bi38O45]\u0009cluster\u000d\u000aThe\u0009Q-space\u0009data\u0009has\u0009been\u0009simulated\u0009in\u0009CrystalDiffract1 using\u0009a\u0009wavelength\u0009of\u00090.20717\u0009\u00c5\u0009and\u0009a\u00091.2\u0009nm\u0009\u000d\u000asized\u0009 [(Bi38O45(NO3)24((CH3)2SO)26)((CH3)2SO)4)((Bi38O45(NO3)24((CH3)2SO)24)((CH3)2SO)4]\u0009 crystal,\u0009\u000d\u000amimicking\u0009 the\u0009size\u0009of\u0009a\u0009single\u0009cluster.2 Unfortunately,\u0009CrystalDiffract1 cannot\u0009simulate\u0009data\u0009 to\u0009higher\u0009\u000d\u000aQrange than\u0009shown\u0009in\u0009the\u0009plot.\u0009The\u0009r-space\u0009data\u0009has\u0009been\u0009simulated\u0009in\u0009DiffPy-CMI3 using\u0009a\u0009[Bi38O45]\u0009cluster\u0009\u000d\u000aand\u0009the\u0009following\u0009parameters:\u0009Qmin =\u00090\u0009\u00c5-1,\u0009Qmax =\u00099999\u0009\u00c5-1,\u0009Qdamp =\u00090.01\u0009\u00c5-1,\u0009ADP\u0009=\u00090.03\u0009\u00c52.\u0009Qdamp is\u0009a\u0009\u000d\u000aGaussian\u0009dampened\u0009function\u0009which\u0009models the\u0009instrumental\u0009dampening.3\u000d\u000aDetails\u0009of\u0009transformation\u0009from\u0009Q-space\u0009to\u0009r-space\u0009data\u0009of\u0009the [Bi38O45]\u0009cluster\u000d\u000aThe\u0009data\u0009was\u0009 transformed\u0009 to\u0009r-space\u0009 from\u0009Q-space\u0009using\u0009PDFgetX34 using\u0009 the\u0009 following\u0009parameters:\u0009\u000d\u000aRpoly =\u00090.9\u0009\u00c5,\u0009Qmax,\u0009instrument =\u000917.5\u0009\u00c5-1,\u0009Qmax =\u000917\u0009\u00c5-1,\u0009Qmin =\u00090.7\u0009\u00c5-1 and\u0009a\u0009chemical\u0009composition\u0009of\u0009Bi38O45.\u0009\u000d\u000aQmax,\u0009instrument is\u0009the\u0009highest\u0009Q\u0009value\u0009used\u0009in\u0009the\u0009data\u0009to\u0009calculate\u0009S(Q)\u0009and\u0009Rpoly is\u0009the\u0009threshold\u0009for\u0009what\u0009\u000d\u000adata\u0009is\u0009used\u0009to\u0009estimate\u0009the\u0009incoherent\u0009scattering\u0009in\u0009the\u0009data\u0009with\u0009a\u0009polynomial\u0009function.4\u000d\u000aSimulation\u0009details\u0009of\u0009total\u0009scattering,\u0009PDF\u0009and\u0009SAXS\u0009data\u0009of\u0009the\u0009[Bi18O36]\u0009cluster and\u0009the\u0009[Bi22O38]\u0009\u000d\u000acluster\u000d\u000aThe\u0009total\u0009scattering\u0009data\u0009has\u0009been\u0009simulated\u0009in\u0009CrystalDiffract1 using\u0009a\u0009wavelength\u0009of\u00090.20717\u0009\u00c5\u0009and\u0009a\u0009\u000d\u000a1.2\u0009 nm\u0009 sized\u0009 [Bi18O18(OSiMe3)18]\u22c52C7H8 crystal\u0009 and\u0009 a\u0009 1.2\u0009 nm\u0009 sized\u0009 [Bi22O26(OSiMe2tBu)14]\u0009 crystal,\u0009\u000d\u000amimicking\u0009the\u0009size\u0009of\u0009single\u0009clusters. Unfortunately,\u0009CrystalDiffract1 cannot\u0009simulate\u0009data\u0009to\u0009higher\u0009Qrange\u000d\u000athan\u0009shown\u0009in\u0009the\u0009plot.\u0009The\u0009PDF\u0009data\u0009was\u0009simulated\u0009using\u0009Diffpy-CMI3 with\u0009the\u0009following\u0009parameters:\u0009\u000d\u000aQmin =\u00090.7\u0009\u00c5-1,\u0009Qmax =\u000920\u0009\u00c5-1,\u0009Qdamp =\u00090.03\u0009\u00c5-1,\u0009ADP\u0009=\u00090.3\u0009\u00c52.\u0009The\u0009SAXS\u0009data\u0009were simulated\u0009with DiffpyCMI3 using\u0009a\u0009constant\u0009background\u0009to\u0009describe\u0009the\u0009Compton\u0009scattering\u0009of\u00091.5.\u0009Both\u0009the\u0009PDF\u0009and\u0009SAXS\u0009\u000d\u000adata\u0009 have been\u0009 calculated\u0009 with\u0009 the\u0009 Debye\u0009 equation\u0009 using\u0009 simulation\u0009 parameters\u0009 mimicking\u0009\u000d\u000aexperimental\u0009data.\u000d\u000aAppendix\u0009II\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00092\u000d\u000aConventional\u0009data\u0009modelling\u0009approaches,\u0009such\u0009as\u0009least-squares\u0009fitting\u0009algorithms,\u0009excel\u0009at\u0009minimizing\u0009\u000d\u000adiscrepancies between\u0009 experimental\u0009 and\u0009 simulated\u0009 data\u0009 derived\u0009 from\u0009 a\u0009 structural\u0009 model.\u0009 The\u0009\u000d\u000adiscrepancies\u0009are\u0009typically\u0009quantified\u0009using\u0009the\u0009weighted\u0009profile\u0009agreement\u0009factor.\u0009\u000d\u000a\"!\" = $\u000d\u000a\u2211 ['#$%()&) \u2212 ''()'()&, -)] + * &,-\u000d\u000a\u2211 '#$%()&) + * &,-\u000d\u000a\u2219 100 %,\u000d\u000awhere,\u0009Gobs and\u0009Gcalc are\u0009the\u0009observed\u0009and\u0009calculated\u0009PDFs,\u0009and\u0009P\u0009is\u0009the\u0009refinement\u0009parameters\u0009in\u0009the\u0009\u000d\u000amodel.","|\u0009APPENDICES\u000d\u000a182\u000d\u000aAppendix\u0009III\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00093\u000d\u000aRefined\u0009 parameters\u0009 for\u0009 the\u0009 five\u0009 best\u0009 candidate\u0009 POMs\u0009 obtained\u0009 from\u0009 POMFinder\u0009 for\u0009 PDF\u0009 data\u0009 analysis.\u0009 The\u0009\u000d\u000astructures\u0009are\u0009shown\u0009in\u0009Figure\u00093.3 in\u0009the\u0009main\u0009paper.\u0009The\u0009Biso value\u0009for\u0009oxygen\u0009is\u0009set\u0009to\u00092\u0009\u00c5-2.\u0009\u000d\u000aRank Probability\u0009\u000d\u000a[%] Structure Scale\u000d\u000aExpansion/\u0009\u000d\u000acontraction\u0009\u000d\u000afactor\u000d\u000aBiso\u000d\u000a(metal)\u000d\u000a[\u00c5-2]\u0009\u000d\u000a1st 82.8\u000d\u000aA\u0009W11O35\u0009Kegginbased\u0009fragment\u0009\u000d\u000afrom\u0009the\u0009dimeric\u0009\u000d\u000aK5.5Na7Nd(SiW11O39\u000d\u000a(H2O))2(CH3COO)2(\u000d\u000aH2O)10 complex5\u000d\u000a1.53 0.989 0.660\u000d\u000a2nd\u0009\u0009 14.4\u000d\u000aA\u0009W12O36 fragment\u0009\u000d\u000afrom\u0009the\u0009\u000d\u000aK5H(CoW12O40)\u0009\u000d\u000a(H2O)15 crystal6\u000d\u000a1.601 1.000 0.738\u000d\u000a3rd 1.5\u000d\u000aA\u0009W12O40\u0009fragment\u0009\u000d\u000afrom\u0009an\u0009ionic\u0009\u000d\u000acrystal\u0009structure7 of\u0009\u000d\u000a(Al13O4(OH)24(H2O)\u000d\u000a12)(H2W12O40)(OH)(\u000d\u000aH2O)23.12\u000d\u000a1.505 1.003 0.788\u000d\u000a4th\u0009\u0009 1.0\u000d\u000aA\u0009W12O36\u0009fragment\u0009\u000d\u000afrom\u0009the\u0009porous\u0009\u000d\u000ainorganic\u0009of\u0009the\u0009\u000d\u000aform\u0009\u000d\u000aK2NaH2(BW12O40)(\u000d\u000aH2O)128\u000d\u000a1.414 1.004 0.822\u000d\u000a5th\u0009\u0009 0.2\u000d\u000aA\u0009W12Rb4BO43\u0009\u000d\u000afragment\u0009from\u0009\u000d\u000aanother\u0009ionic\u0009\u000d\u000acrystal9\u000d\u000aRb4(Cr3O(OOCH)6(\u000d\u000aH2O)3(BW12O40)(H2\u000d\u000aO)16\u000d\u000a0.514 1.000 0.000","183\u000d\u000aAppendix IV\u0009\u2013\u0009Supporting\u0009Information\u0009for\u0009Chapter\u00094\u000d\u000aRefined\u0009parameters\u0009for\u0009mono-metallic\u0009nanoparticles obtained\u0009from\u0009the\u0009DeepStruc\u0009algorithm\u0009for\u0009analysis\u0009of\u0009seven\u0009\u000d\u000asimulated\u0009PDFs.\u0009The\u0009mean\u0009absolute\u0009errors\u0009for\u0009the\u0009predicted\u0009xyz-coordinates\u0009are\u0009given\u0009along\u0009with\u0009the\u0009Rwp-values\u0009\u000d\u000aobtained\u0009in\u0009the\u0009PDF\u0009fit.\u0009\u000d\u000aName #Atoms Scale\u000d\u000aExpansion/\u0009\u000d\u000acontraction\u0009\u000d\u000afactor\u000d\u000aBiso\u000d\u000aMAE\u0009\u000d\u000a[\u00c5]\u000d\u000aMAE\u0009fit\u0009\u000d\u000a[\u00c5]\u000d\u000aRwp\u0009\u000d\u000a[%]\u000d\u000abcc 89 0.146 0.959 0.428 0.132\u0009\u00b1\u0009\u000d\u000a0.086\u000d\u000a0.083\u0009\u00b1\u0009\u000d\u000a0.068 21.2\u000d\u000aDecahe\u000d\u000adral 105 0.107 1.030 0.238 0.140\u0009\u00b1\u0009\u000d\u000a0.095\u000d\u000a0.116\u0009\u00b1\u0009\u000d\u000a0.081 39.0\u000d\u000afcc 171 0.117 0.957 0.284 0.182\u0009\u00b1\u0009\u000d\u000a0.073\u000d\u000a0.116\u0009\u00b1\u0009\u000d\u000a0.053 54.2\u000d\u000ahcp 128 0.082 0.974 0.081 0.093\u0009\u00b1\u0009\u000d\u000a0.041\u000d\u000a0.043\u0009\u00b1\u0009\u000d\u000a0.025 10.8\u000d\u000aIcosah\u000d\u000aedral 55 0.119 0.962 0.390 0.120\u0009\u00b1\u0009\u000d\u000a0.058\u000d\u000a0.092\u0009\u00b1\u0009\u000d\u000a0.044 37.8\u000d\u000aOctahe\u000d\u000adral 146 0.111 0.975 0.306 0.132\u0009\u00b1\u0009\u000d\u000a0.056\u000d\u000a0.106\u0009\u00b1\u0009\u000d\u000a0.051 48.2\u000d\u000asc 177 0.158 0.984 0.210 0.091\u0009\u00b1\u0009\u000d\u000a0.045\u000d\u000a0.091\u0009\u00b1\u0009\u000d\u000a0.043 46.4\u000d\u000aReferences\u000d\u000a1 . Generated using CrystalDiffract\u00ae: a powder diffraction program for Mac and Windows. CrystalMaker \u000d\u000aSoftware Ltd, Oxford, England (www.crystalmaker.com). \u000d\u000a2 Miersch, L., R\u00fcffer, T., Schlesinger, M., Lang, H. & Mehring, M. Hydrolysis Studies on Bismuth Nitrate: \u000d\u000aSynthesis and Crystallization of Four Novel Polynuclear Basic Bismuth Nitrates. Inorg. Chem. 51, 9376-9384, \u000d\u000adoi:10.1021/ic301148p (2012).\u000d\u000a3 Juh\u00e1s, P., Farrow, C. L., Yang, X., Knox, K. R. & Billinge, S. J. L. Complex modeling: a strategy and software \u000d\u000aprogram for combining multiple information sources to solve ill posed structure and nanostructure inverse \u000d\u000aproblems. Acta Crystallogr. A 71, 562-568, doi:10.1107/S2053273315014473 (2015).\u000d\u000a4 Juhas, P., Davis, T., Farrow, C. L. & Billinge, S. J. L. PDFgetX3: a rapid and highly automatable program for \u000d\u000aprocessing powder diffraction data into total scattering pair distribution functions. J. Appl. Cryst. 46, 560-566, \u000d\u000adoi:10.1107/S0021889813005190 (2013).\u000d\u000a5 Saini, M. K. et al. Dimeric complexes of rare-earth substituted Keggin-type silicotungstates: syntheses, crystal \u000d\u000astructure and solid state properties. RSC Adv. 4, 25357-25364, doi:10.1039/C4RA02751E (2014).\u000d\u000a6 Glass, E. N. et al. Extending Metal-to-Polyoxometalate Charge Transfer Lifetimes: The Effect of Heterometal \u000d\u000aLocation. Chem. Eur. J. 20, 4297-4307, doi:10.1002/chem.201304119 (2014).\u000d\u000a7 Son, J. H., Kwon, Y.-U. & Han, O. H. New Ionic Crystals of Oppositely Charged Cluster Ions and Their \u000d\u000aCharacterization. Inorg. Chem. 42, 4153-4159, doi:10.1021/ic0340377 (2003).\u000d\u000a8 Han, X.-B. et al. A 3D purely inorganic porous framework based on Keggin polyoxoanions. Inorg. Chem. \u000d\u000aCommun. 18, 47-49, doi:10.1016/j.inoche.2012.01.006 (2012).\u000d\u000a9 Uchida, S., Kawamoto, R. & Mizuno, N. Recognition of Small Polar Molecules with an Ionic Crystal of \u03b1Keggin-Type Polyoxometalate with a Macrocation. Inorg. Chem. 45, 5136-5144, doi:10.1021/ic060684x (2006).","","185\u000d\u000aUNI VERSITY\u0009OF\u0009COPENHAGEN\u000d\u000aFACULTY\u0009OF\u0009SCIENCE\u000d\u000aDEPARTMENT\u0009OF\u0009CHEMISTRY\u0009&\u0009NANO - SCIENCE\u0009CENTER\u000d\u000aAPPENDICES\u0009\u2013\u0009PAPERS\u000d\u000aPaper\u0009I\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00091\u0009of\u0009the\u0009Dissertation with its\u0009Supporting\u0009Information\u0009(SI).\u0009\u000d\u000aPaper\u0009II\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00092\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI is\u0009located\u0009here:\u000d\u000ahttps://chemrxiv.org/engage/chemrxiv/article-details/64ed9d6079853bbd78a73fb0\u000d\u000aPaper\u0009III\u0009\u000d\u000aCan\u0009 be\u0009 found\u0009 in\u0009 Chapter\u0009 2\u0009 of\u0009 the\u0009 Dissertation,\u0009 while\u0009 its\u0009 SI\u0009 is\u0009 located\u0009 here:\u0009\u000d\u000ahttps://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fanie.202103641&file\u000d\u000a=anie202103641-sup-0001-misc_information.pdf\u000d\u000aPaper\u0009IV\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00093\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI\u0009is\u0009located\u0009here:\u0009\u0009\u000d\u000ahttps://chemrxiv.org/engage/chemrxiv/article-details/64e5fef7dd1a73847f5951b9\u000d\u000aPaper\u0009V\u0009\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00093\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI\u0009is\u0009located\u0009here:\u0009\u0009\u000d\u000ahttps://static-content.springer.com/esm/art%3A10.1038%2Fs41524-022-00896-\u000d\u000a3/MediaObjects/41524_2022_896_MOESM1_ESM.pdf\u000d\u000aPaper\u0009VI\u0009\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00093\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI\u0009is\u0009located\u0009here:\u0009\u0009","Supplementary information for: \u000d\u000aClusterFinder: A fast tool to find cluster structures from pair distribution function data\u000d\u000aAndy S. Anker*1\u000d\u000a, Frederik L. Johansen\u20201,2\u000d\u000a, Ulrik Friis-Jensen\u20201,2, Simon J. L. Billinge*3,4, Kirsten M. \u00d8. \u000d\u000aJensen*1\u000d\u000a*Correspondence to andy@chem.ku.dk (ASA), sb2896@columbia.edu (SJLB), kirsten@chem.ku.dk (KM\u00d8J) \u000d\u000a\u2020Both authors contributed equally to this work \u000d\u000a1: Department of Chemistry and Nano-Science Center, University of Copenhagen, 2100 Copenhagen \u00d8, \u000d\u000aDenmark \u000d\u000a2: Department of Computer Science, University of Copenhagen, 2100 Copenhagen \u00d8, Denmark \u000d\u000a3: Department of Applied Physics and Applied Mathematics Science, Columbia University, New York, NY \u000d\u000a10027, USA \u000d\u000a4: Condensed Matter Physics and Materials Science Department, Brookhaven National Laboratory, Upton, NY \u000d\u000a11973, USA ","Table of Contents\u000d\u000aA: PDF calculation parameters for all the examples provided in the paper ................................................... 3\u000d\u000aB: Meaning of the atom colour coding after the ClusterFinder process. ........................................................ 3\u000d\u000aC: Visualisation of the ClusterFinder analysis on the simulated PDFs from Figure 3 using a continuous \u000d\u000acolorbar 4\u000d\u000aD: Fitting the 5 best candidates from the ClusterFinder process to simulated PDF obtained from a !-keggin \u000d\u000acluster cut out of a spinel crystal .................................................................................................................. 5\u000d\u000aE: Fitting the 5 best candidates from the ClusterFinder process to simulated PDF obtained from a !-keggin \u000d\u000acluster cut out of an !-keggin crystal ............................................................................................................ 6\u000d\u000aF: Using ClusterFinder to screen the ICSD for the right starting model of an experimental PDF obtained from \u000d\u000aa [Bi38O45] cluster in solution........................................................................................................................ 6\u000d\u000aG: Using ClusterFinder to screen the ICSD for the right starting model of an experimental PDF obtained \u000d\u000afrom ceria nanoparticles .............................................................................................................................. 8","A: PDF calculation parameters for all the examples provided in the paper \u000d\u000aFigure 1: \u000d\u000aSimulated PDFs \u000d\u000aFigure 2: \u000d\u000a A single C60\u000d\u000abuckyball \u000d\u000aFigure 2: \u000d\u000aSimulated \u000d\u000aparatungstate and \u000d\u000adecatungstate \u000d\u000aclusters \u000d\u000aFigure 3+4: A \u000d\u000aW12O40 unit\u000d\u000a(data collected at \u000d\u000aDanMax) \u000d\u000ar-range (\u00c5) 0 \u2013 30.1 3 \u2013 12 0 \u2013 30.1 1.6 \u2013 10 \u000d\u000ar-step 0.01 0.01 0.01 0.01 \u000d\u000aQmin (\u00c5-1) 0.7 0.5 0.7 0.7 \u000d\u000aQmax (\u00c5-1) 25 24 25 20 \u000d\u000aQdamp (\u00c5-1) 0.04 0.04 0.04 0.05 \u000d\u000aADP (\u00c52\u000d\u000a) 0.3 0.3 0.3 0.3 \u000d\u000aTable 1 | PDF calculation parameters for all the examples provided in the paper.\u000d\u000aB: Meaning of the atom colour coding after the ClusterFinder process. \u000d\u000aEffect on the Rwp when the \u000d\u000aatom is present in the structure \u000d\u000aAtom contribution \u000d\u000avalue \u000d\u000aFitting when the \u000d\u000aatom is present \u000d\u000aDenomination of \u000d\u000athe atom \u000d\u000aColour \u000d\u000acode \u000d\u000aNegative (-) More negative (-) Better Good atom Yellow \u000d\u000aPositive (-) More positive (+) Worse Bad atom Blue\u000d\u000aTable 2. Meaning of the atom colour coding after the ClusterFinder process. ","C: Visualisation of the ClusterFinder analysis on the simulated PDFs from Figure 3 using a continuous \u000d\u000acolour bar \u000d\u000aFig. S1 | Analysis of simulated PDFs from well-known cluster structures. A) Simulated PDFs of blue) a \u000d\u000adecatungstate polyoxometalate cluster cut out the Na5(H7W12O42)(H2O)20 crystal (Redrup & Weller, 2009), \u000d\u000agreen) a C60 buckyball cut out from a single unit cell of a crystal structure of C60 (Chen & Yamanaka, 2002), and \u000d\u000ared) a paratungstate polyoxometalate cluster cut out of the \u000d\u000a(Ba(H2O)2(H(N(CH3)2)CO)3)2(W10O32)(H(N(CH3)2)CO)2 crystal (Poimanova et al., 2015). The simulation \u000d\u000aparameters mimic typical values of a PDF dataset and can be seen in Table S1. B-D) Results of using \u000d\u000aClusterFinder on the three simulated PDFs colouring the atom contribution values with a continuous colour bar.","D: Fitting the five best candidates from the ClusterFinder process to simulated PDF obtained from a !-\u000d\u000akeggin cluster cut out of a spinel crystal \u000d\u000aFig. S2 | PDF fits of the top five structures from the ClusterFinder process used to screen ICSD for the \u000d\u000acorrect starting model of a simulated PDF obtained from a !-keggin cluster cut out of a spinel crystal. ","E: Fitting the five best candidates from the ClusterFinder process to simulated PDF obtained from a !-\u000d\u000akeggin cluster cut out of an !-keggin crystal \u000d\u000aFig. S3 | PDF fits of the top five structures from the ClusterFinder process used to screen ICSD for the \u000d\u000acorrect starting model of a simulated PDF obtained from a !-keggin cluster cut out of an !-keggin crystal. \u000d\u000aF: Using ClusterFinder to screen the ICSD for the correct starting model of an experimental PDF \u000d\u000aobtained from a [Bi38O45] cluster in solution \u000d\u000aWe use ClusterFinder to scan a database of the best-fitting structure models from the experimental PDF of \u000d\u000a[Bi38O45] clusters in solution (Anker et al., 2021). The [Bi38O45] type clusters are previously seen to be highly \u000d\u000asimilar to cutouts of the \"-Bi2O3 and the #-Bi2O3 crystal structures (Weber et al., 2017). \u000d\u000aAgain, we use ClusterFinder iteratively on each structure (188,631 structures) in the ICSD as the starting model. \u000d\u000aTo accelerate ClusterFinder, we only refine the scale factor, and we remove all crystals not containing Bi atoms \u000d\u000a(181,215 structures) and supercells with more than 1000 atoms (704 structures), leaving us with 5550 structures. \u000d\u000aAfterwards, we rank the structures in ICSD by the average Rwp value during the ClusterFinder process. The entire \u000d\u000aprocedure takes 8 min (468 seconds) on an AMD Ryzen Threadripper 3990X with 64-core 2.9/4.3GHz and 95 ","min (5727 s) on an Intel(R) Core\u2122 i7-8665U CPU @ 1.9/2.11 GHz. Figure S4 demonstrates that the crystal \u000d\u000astructures in the top five are mainly fluorite #-Bi2O3 type structures or similar: \u000d\u000a1) Cd3.573Bi19.427V4O42,(Labidi et al., 2008) 2) Ca4.207Bi18.793V4O42,(Labidi et al., 2008) 3) Bi4.66Ca1.09VO10.5\u000d\u000a(Radosavljevic Evans et al., 2002), and 5) Na1.533Bi21.467V4O42 (Labidi et al., 2008), but the 4th ranked structure \u000d\u000ais a Bi2SeO5 crystal instead (Rademacher et al., 2001). \u000d\u000aFig. S4 | Illustration of how ClusterFinder is used to screen ICSD for the correct starting model of an \u000d\u000aexperimental PDF obtained from a [Bi38O45] cluster in solution. For each structure in the ICSD, the \u000d\u000aClusterFinder procedure is performed, and the atoms are coloured according to their impact on the fit quality \u000d\u000ausing a continuous colour bar. The structures in ICSD are afterwards sorted after their average Rwp value during \u000d\u000athe ClusterFinder procedure. The five candidates with the lowest Rwp value are highlighted along with their fits \u000d\u000ato the dataset, only fitting the scaling factor. More extensive PDF fits, including Rwp values, can be seen in Figure \u000d\u000aS5. ","Fig. S5 | PDF fits of the top five structures from the ClusterFinder process used to screen ICSD for the \u000d\u000acorrect starting model of a simulated PDF obtained from a [Bi38O45] cluster in solution. \u000d\u000aG: Using ClusterFinder to screen the ICSD for the correct starting model of an experimental PDF \u000d\u000aobtained from ceria nanoparticles \u000d\u000aWe use ClusterFinder to scan a database of the best-fitting structure models from the experimental PDF of a \u000d\u000aceria nanoparticle. Iteratively, we use ClusterFinder on each structure (188,631 structures) in the ICSD as the \u000d\u000astarting model. To accelerate ClusterFinder, we only refine the scale factor, and we remove crystals not \u000d\u000acontaining Ce atoms (181,311 structures) and supercells with more than 1000 atoms (704 structures), leaving us \u000d\u000awith 5454 structures. Afterwards, we rank the structures in ICSD by the average Rwp value during the \u000d\u000aClusterFinder process. The entire procedure takes 3 min (183 s) on an AMD Ryzen Threadripper 3990X with \u000d\u000a64-core 2.9/4.3GHz and 56 min (3342 s) on an Intel(R) Core\u2122 i7-8665U CPU @ 1.9/2.11 GHz. Figure S6\u000d\u000ademonstrates that all crystal structures in the top 4 are fluorite structures:","1) Ce0.6875Y0.3125O2 (Coduri et al., 2013), 2) Ce0.65625YO2 (Coduri et al., 2013), 3) Ce0.9Gd0.1O1.95 (Artini et al., \u000d\u000a2014), and 4) Ce0.5Nd0.5O1.75 (Chakraborty et al., 2006), whereas the 5th ranked structure is a 5) Ce2Zr2O7\u000d\u000apyrochlore structure (Sasaki et al., 2004). \u000d\u000aFig. S6 | Illustration of how ClusterFinder is used to screen ICSD for the correct starting model of an \u000d\u000aexperimental PDF obtained from ceria nanoparticles. For each structure in the ICSD, the ClusterFinder \u000d\u000aprocedure is performed, and the atoms are coloured according to their impact on the fit quality using a continuous \u000d\u000acolour bar. The structures in ICSD are afterwards sorted after their average Rwp value during the ClusterFinder \u000d\u000aprocedure. The five candidates with the lowest Rwp value are highlighted along with their fits to the dataset, only \u000d\u000afitting the scaling factor. More extensive PDF fits, including Rwp values, can be seen in Figure S7.","Fig. S7 | PDF fits of the top five structures from the ClusterFinder process used to screen ICSD for the \u000d\u000acorrect starting model of a simulated PDF obtained from ceria nanoparticles. \u000d\u000aReferences \u000d\u000aAnker, A. S., Christiansen, T. L., Weber, M., Schmiele, M., Brok, E., Kj\u00e6r, E. T. S., Juh\u00e1s, P., Thomas, R., \u000d\u000aMehring, M. & Jensen, K. M. \u00d8. (2021). Angew. Chem. Int. Ed. 60, 2-12. \u000d\u000aArtini, C., Pani, M., Lausi, A., Masini, R. & Costa, G. A. (2014). Inorg. Chem. 53, 10140-10149. \u000d\u000aChakraborty, K. R., Krishna, P. S. R., Chavan, S. V. & Tyagi, A. K. (2006). Powder Diffr. 21, 36-39. \u000d\u000aChen, X. & Yamanaka, S. (2002). Chem. Phys. Lett. 360, 501-508. \u000d\u000aCoduri, M., Scavini, M., Allieta, M., Brunelli, M. & Ferrero, C. (2013). Chem. Mater. 25, 4278-4289. \u000d\u000aLabidi, O., Drache, M., Roussel, P. & Wignacourt, J.-P. (2008). Solid State Sciences 10, 1074-1082. \u000d\u000aPoimanova, O. Y., Radio, S. V., Bilousova, K. Y., Baumer, V. N. & Rozantsev, G. M. (2015). J. Coord. Chem.\u000d\u000a68, 1-17. \u000d\u000aRademacher, O., G\u00f6bel, H., Ruck, \u039c. & Oppermann, \u0397. (2001). Zeitschrift f\u00fcr Kristallographie - New Crystal \u000d\u000aStructures 216, 29-30. \u000d\u000aRadosavljevic Evans, I., Tao, S., Irvine, J. T. S. & Howard, J. A. K. (2002). Chem. Mater. 14, 3700-3704. \u000d\u000aRedrup, K. V. & Weller, M. T. (2009). Dalton Trans., 4468-4472. \u000d\u000aSasaki, T., Ukyo, Y., Kuroda, K., Arai, S., Muto, S. & Saka, H. (2004). J. Ceram. Soc. Jpn. 112, 440-444. \u000d\u000aWeber, M., Schlesinger, M., Walther, M., Zahn, D., Schalley, C. A. & Mehring, M. (2017). Zeitschrift f\u00fcr \u000d\u000aKristallographie-Crystalline Materials 232, 185-207. ","","197\u000d\u000aPaper\u0009VII\u0009\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00094\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI\u0009is\u0009located\u0009here:\u0009\u0009\u000d\u000ahttps://par.nsf.gov/servlets/purl/10300745\u0009\u0009\u000d\u000aPaper\u0009VIII\u0009\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00094\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI\u0009is\u0009located\u0009here:\u0009\u0009\u000d\u000ahttps://www.rsc.org/suppdata/d2/dd/d2dd00086e/d2dd00086e1.pdf\u000d\u000aPaper\u0009IX\u0009\u0009\u000d\u000aCan\u0009be\u0009found\u0009in\u0009Chapter\u00094\u0009of\u0009the\u0009Dissertation,\u0009while\u0009its\u0009SI\u0009is\u0009located\u0009here:\u0009\u0009\u000d\u000ahttps://www.rsc.org/suppdata/d2/dd/d2dd00147k/d2dd00147k1.pdf\u000d\u000aCIF-Finder\u0009paper\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009the\u0009CIF-Finder\u0009paper which\u0009is\u0009in\u0009preparation.\u0009","","221\u000d\u000aIntergrowth paper\u0009\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009the\u0009Intergrowth\u0009paper which\u0009is\u0009published\u0009in\u0009Dalton\u0009Transactions.\u0009","Dalton\u000d\u000a Transactions\u000d\u000aAn international journal of inorganic chemistry\u000d\u000arsc.li/dalton\u000d\u000aVolume 51\u000d\u000aNumber 45\u000d\u000a7 December 2022\u000d\u000aPages 17101-17512\u000d\u000aISSN 1477-9226 \u000d\u000a PAPER \u000d\u000aKirsten M. \u00d8. Jensen et al. \u000d\u000a Characterisation of intergrowth in metal oxide materials \u000d\u000ausing structure-mining: the case of \u03b3-MnO 2 ","Dalton\u000d\u000aTransactions\u000d\u000aPAPER\u000d\u000aCite this: Dalton Trans., 2022, 51,\u000d\u000a17150\u000d\u000aReceived 5th July 2022,\u000d\u000aAccepted 29th August 2022\u000d\u000aDOI: 10.1039/d2dt02153f\u000d\u000arsc.li/dalton\u000d\u000aCharacterisation of intergrowth in metal oxide\u000d\u000amaterials using structure-mining: the case of\u000d\u000a\u03b3-MnO2\u2020\u000d\u000aNicolas P. L. Magnard, Andy S. Anker, Olivia Aalling-Frederiksen, Andrea Kirsch\u000d\u000aand Kirsten M. \u00d8. Jensen *\u000d\u000aManganese dioxide compounds are widely used in electrochemical applications e.g. as electrode\u000d\u000amaterials or photocatalysts. One of the most used polymorphs is \u03b3-MnO2 which is a disordered intergrowth of pyrolusite \u03b2-MnO2 and ramsdellite R-MnO2. The presence of intergrowth defects alters the\u000d\u000amaterial properties, however, they are difficult to characterise using standard X-ray diffraction due to anisotropic broadening of Bragg reflections. We here propose a characterisation method for intergrown\u000d\u000astructures by modelling of X-ray diffraction patterns and pair distribution functions (PDF) using \u03b3-MnO2 as\u000d\u000aan example. Firstly, we present a fast peak-fitting analysis approach, where features in experimental diffraction patterns and PDFs are matched to simulated patterns from intergrowth structures, allowing quick\u000d\u000acharacterisation of defect densities. Secondly, we present a structure-mining-based analysis using simulated \u03b3-MnO2 superstructures which are compared to our experimental data to extract trends on defect\u000d\u000adensities with synthesis conditions. We applied the methodology to a series of \u03b3-MnO2 samples synthesised by a hydrothermal route. Our results show that with synthesis time, the intergrowth structure\u000d\u000areorders from a R-like to a \u03b2-like structure, with the \u03b2-MnO2 fraction ranging from ca. 27 to 82% in the\u000d\u000asamples investigated here. Further analysis of the structure-mining results using machine learning can\u000d\u000aenable extraction of more nanostructural information such as the distribution and size of intergrown\u000d\u000adomains in the structure. Using this analysis, we observe segregation of R- and \u03b2-MnO2 domains in the\u000d\u000amanganese oxide nanoparticles. While R-MnO2 domains keep a constant size of ca. 1\u20132 nm, the \u03b2-MnO2\u000d\u000adomains grow with synthesis time.\u000d\u000aIntroduction\u000d\u000aDue to the increasing need for sustainable technologies, a\u000d\u000ahigh interest has grown for functional materials based on\u000d\u000aearth abundant elements. Manganese dioxide MnO2 is an\u000d\u000ainteresting compound in that perspective. Manganese oxide\u000d\u000acrystallises in several polymorphs including layered structures,\u000d\u000atunnel structures, as well as spinel, rutile, and hollandite\u000d\u000astructures. These show a range of different electrochemical\u000d\u000aproperties, allowing for targeted materials design for e.g.,\u000d\u000aenergy technologies.1\u20136 In this study, we focus on the \u03b3-MnO2\u000d\u000apolymorph in the MnO2 family. \u03b3-MnO2 is of great interest as\u000d\u000ait is frequently used in commercial batteries as an electrode\u000d\u000amaterial and is investigated for secondary battery systems.7\u201310\u000d\u000aHowever, the structural complexity of \u03b3-MnO2 has made it\u000d\u000adifficult to fully map its relations between synthesis, structure\u000d\u000aand properties. The structure can be represented as an intergrowth of \u03b2-MnO2 (P42/mnm) and R-MnO2 (Pnam) slabs\u000d\u000a(Fig. 1). \u03b2-MnO2 crystallises in the rutile structure and is built\u000d\u000afrom single-chains of edge-sharing [MnO6] octahedra, which\u000d\u000aare connected through corner-sharing octahedra along the aFig. 1 \u03b2-MnO2 (a) and R-MnO2 (b) can randomly be stacked along their\u000d\u000acommon a-axis to form an intergrowth \u03b3-MnO2 structure (c).\u000d\u000a\u2020Electronic supplementary information (ESI) available. See DOI: https://doi.org/\u000d\u000a10.1039/d2dt02153f\u000d\u000aDepartment of Chemistry and Nano-Science Center, University of Copenhagen, 2100\u000d\u000aCopenhagen \u00d8, Denmark. E-mail: kirsten@chem.ku.dk\u000d\u000a17150 | Dalton Trans., 2022, 51, 17150\u201317161 This journal is \u00a9 The Royal Society of Chemistry 2022\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online\u000d\u000aView Journal | View Issue","and b-axis. R-MnO2 is closely related but contains doublechains of edge-sharing [MnO6] octahedra along the a-axis. The\u000d\u000amixing, or intergrowth, of these two parent structures within\u000d\u000athe same O2\u2212 hcp sublattice results in the \u03b3-MnO2 structure as\u000d\u000aillustrated in Fig. 1.11\u201314 The intergrowth structure can be considered a sub-ordered sequence of faults along a common crystallographic direction, here the a-axis, within the O2\u2212 hcp\u000d\u000asublattice.12,14 The structure is also referred to as containing\u000d\u000a\u201cDe Wolff defects\u201d, named after P. M. De Wolff who was one of\u000d\u000athe first to work on the characterisation of this phase.15 A consequence of intergrowth is a huge variability in the structure of\u000d\u000adifferent \u03b3-MnO2 samples, which heavily influences the\u000d\u000amaterial properties.3,16\u201318\u000d\u000aHowever, the defects make the characterisation of \u03b3-MnO2\u000d\u000achallenging. The structural disorder gives rise to anisotropic\u000d\u000abroadening of Bragg reflections in powder diffraction patterns\u000d\u000afrom \u03b3-MnO2, and analysis beyond e.g., standard Rietveld\u000d\u000arefinement is required. In the past decades, several approaches\u000d\u000ausing X-ray diffraction (XRD) have been developed to characterise the \u03b3-MnO2 structure. A first approach was proposed by De\u000d\u000aWolff,\u000d\u000a15 who related the broadening of specific Bragg peaks in\u000d\u000aR-MnO2-like \u03b3-MnO2 to the amount of \u03b2-MnO2 domains. Later,\u000d\u000aChabre and Pannetier12 interpreted De Wolff defects using a\u000d\u000astatistical model of the intergrowth structure. By defining a De\u000d\u000aWolff defect distribution to describe the intergrowth rate, they\u000d\u000amodelled a library of \u03b3-MnO2 XRD patterns. From this library,\u000d\u000athey were able to find patterns in the line position and broadening and determined empirical rules to estimate the intergrowth rate of \u03b3-MnO2 for real samples from their diffraction\u000d\u000apatterns. Another approach has been used by Simon et al.,\u000d\u000awho performed multi-phase Rietveld refinements on patterns\u000d\u000afrom \u03b3-MnO2 samples using the R-MnO2 and \u03b2-MnO2 structures, as well as a highly disordered \u03b5-MnO2 model, where\u000d\u000aMn4+ ions partially occupy all octahedral sites in the O2\u2212 hcp\u000d\u000asublattice.19 This provided good agreement between the model\u000d\u000aand experimental data and allowed quantification of the\u000d\u000adifferent phases. The intergrowth in \u03b3-MnO2 have also been\u000d\u000acharacterised using Pair Distribution Function (PDF) analysis.\u000d\u000aThe PDF is obtained by Fourier transforming the normalised\u000d\u000aand corrected total scattering data, and can provide information on the local and medium range structure in materials.\u000d\u000aUsing PDF, Galliez et al.20 characterised \u03b3-MnO2 samples by\u000d\u000afitting their PDFs with the \u03b2-MnO2 and R-MnO2 parent structures. This approach gave information about the fractions of\u000d\u000aparent structures in \u03b3-MnO2 samples, but was not able to\u000d\u000aaccount for the De Wolff defect distribution in the structure.\u000d\u000aIn a broader context, several tools have been developed for\u000d\u000acharacterisation of planar faults in various crystal structures.\u000d\u000aIn DIFFaX,21 faulted crystal structures are modelled by defining the unit layers that the system consists of along with a\u000d\u000astacking fault probability for each layer, which allows calculation of scattering intensities. The scattered intensity of the\u000d\u000afaulted structure is calculated analytically using the structure\u000d\u000afactor of individual unit layers and an inter-layer interference\u000d\u000aterm that account for the stacking probability of the different\u000d\u000alayers. This method was used by Chabre and Pannetier in their\u000d\u000aanalysis of the effect of De Wolff defects in \u03b3-MnO2 in XRD patterns.12 DIFFaX+, developed by Leoni et al.22 allows to directly\u000d\u000arefine stacking probabilities in order to quantitatively estimate\u000d\u000adefect densities. A similar methodology has been\u000d\u000aimplemented in the FullProf suite software package through\u000d\u000athe FAULTS program, developed by Casas-Cabanas et al.23 A\u000d\u000atool for simulating XRD patterns from structures with stacking\u000d\u000afaults has also been implemented in TOPAS by Coelho\u000d\u000aet al.24,25 This approach is based on the generation of supercells, from which individual scattering signals can be calculated. Simulated XRD patterns are then obtained by averaging\u000d\u000aover these signals. Other approaches in the literature are also\u000d\u000abased on modelling supercells for calculation of scattering\u000d\u000apatterns.26,27 Planar defects in nanocrystalline materials have\u000d\u000afurthermore been studied with PDF analysis, where the unit\u000d\u000alayers can be accurately described by the local range of the\u000d\u000aPDF and medium range by modelling faulted supercells.28\u201333\u000d\u000aIn this paper, we develop a new method for intergrowth\u000d\u000acharacterization. We combine analysis of \u03b3-MnO2 XRD patterns\u000d\u000aand PDFs, which allows to investigate which of the two\u000d\u000amethods is better suited for characterization of intergrown\u000d\u000asystems such as \u03b3-MnO2. First, a large database of intergrowth\u000d\u000asuperstructures with varying De Wolff defect distributions is\u000d\u000aconstructed, using the statistical model developed by Chabre\u000d\u000aand Pannetier,12 and XRD patterns and PDFs from these structures are simulated. We then use the modelled patterns to\u000d\u000aestablish trends in XRD/PDF peak intensity and shape with\u000d\u000aintergrowth parameters, which allows development of a peakfitting method for estimation of defect densities. Secondly, we\u000d\u000ause the structures in the database for \u2018structure-mining\u2019\u000d\u000athrough Rietveld analysis against experimental data from\u000d\u000ahydrothermally synthesized manganese oxide nanoparticles.\u000d\u000aThis analysis allows identification of the best fitting structure\u000d\u000afor a given dataset, so that trends on defect densities in a\u000d\u000aseries of samples can be extracted. Thirdly, the structuremining results are given to a machine learning algorithm\u000d\u000awhich makes it possible to extract information on the size of\u000d\u000adomains in the intergrowth samples. This gives new insight\u000d\u000ainto the formation and growth of \u03b3-MnO2 intergrowths.\u000d\u000aMethods\u000d\u000aSample synthesis\u000d\u000aAll \u03b3-MnO2 samples have been synthesised using a soft chemistry route. The synthesis method is adapted from the protocol\u000d\u000aused by Wang et al.34 MnSO4 and (NH4)2S2O8 are dissolved in\u000d\u000adeionised (DI) water. The synthesis was conducted hydrothermally at 150 \u00b0C for 2 h, 3.5 h and 5 h with a starting concentration of Mn2+ ions of 0.7 M and a Mn2+ : S2O8\u000d\u000a2\u2212 ratio of 5 : 3\u000d\u000ausing a Teflon-lined steel autoclave. The autoclaves were then\u000d\u000aleft to cool down at room temperature after heating. The assynthesised black solid was filtered and washed 3 times each\u000d\u000awith DI water and ethanol before being left to dry at room\u000d\u000atemperature. The samples were labelled sample 1, 2 and 3,\u000d\u000acorresponding to heating time of 2, 3.5 and 5 h, respectively.\u000d\u000aDalton Transactions Paper\u000d\u000aThis journal is \u00a9 The Royal Society of Chemistry 2022 Dalton Trans., 2022, 51, 17150\u201317161 | 17151\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","XRD and X-ray total scattering measurements\u000d\u000aThe samples were put in Kapton tubes and XRD as well as\u000d\u000aX-ray total scattering patterns were measured at the I15-1\u000d\u000abeamline at Diamond Light Source, United Kingdom.\u000d\u000aScattering patterns from an empty Kapton tube and from a Si\u000d\u000astandard sample were measured as well. The Kapton tube data\u000d\u000awere used for background subtraction and the Si standard was\u000d\u000aused to account for instrumental broadening of the Q-space\u000d\u000apeaks. The total scattering data were acquired on a Perkin\u000d\u000aElmer XRD 1611 CP3 2D detector with a pixel size of 100 \u00d7\u000d\u000a100 \u03bcm2\u000d\u000a. The XRD data were collected on a Perkin Elmer XRD\u000d\u000a4343 CT 2D detector with a pixel size of 150 \u00d7 150 \u03bcm2\u000d\u000a. The\u000d\u000asample to detector distances were 283 mm (total scattering)\u000d\u000aand 670 mm (XRD) using a wavelength of 0.1618 \u00c5.\u000d\u000aThe 2D images were integrated using Dawn.35 PDF data\u000d\u000awere obtained by Fourier transformation of the normalised\u000d\u000aand corrected total scattering patterns using xPDFSuite36 with\u000d\u000aa Qmin value of 1.0 \u00c5\u22121 and a Qmax value of 23 \u00c5\u22121 after subtracting the scattering contribution of the Kapton tube.\u000d\u000aTransmission electron microscopy (TEM)\u000d\u000aTransmission electron microscopy images have been recorded\u000d\u000awith a Tecnai T20 G2 S-TEM with HT 200 keV and a LaB6 filament. The samples were prepared by dropcasting a suspension\u000d\u000aof MnO2 nanoparticles in ethanol on lacey carbon coated\u000d\u000acopper TEM grids. The ImageJ37 software was used for image\u000d\u000aprocessing and analysis.\u000d\u000aResults and discussion\u000d\u000aThree \u03b3-MnO2 samples with different amounts of De Wolff\u000d\u000adefects were synthesised. To achieve different defect densities,\u000d\u000athe samples were hydrothermally treated for 2 h (sample 1),\u000d\u000a3.5 h (sample 2) and 5 h at 150 \u00b0C (sample 3), respectively.\u000d\u000aFig. 2a and b shows experimental XRD patterns and PDFs\u000d\u000aobtained for sample 1, 2 and 3 along with calculated patterns\u000d\u000aof \u03b2-MnO2 and R-MnO2. The preparation of the samples differs\u000d\u000aonly by synthesis time, and we can observe that the longer the\u000d\u000asynthesis (sample 3), the structure of the samples becomes\u000d\u000acloser related to \u03b2-MnO2. The samples prepared with shorter\u000d\u000asyntheses times, sample 1 and 2, show reflections from both\u000d\u000aphases, indicating a structure containing features from both\u000d\u000a\u03b2-MnO2 and R-MnO2. For all three samples, some reflections\u000d\u000ain the XRD patterns are broader than others, such as the 110\u000d\u000apeak of R- and \u03b2-MnO2 at 1.5 and 2.0 \u00c5\u22121 in comparison to the\u000d\u000a011 peak of \u03b2- and R-MnO2 around 2.5 and 2.6 \u00c5\u22121\u000d\u000a, highlighted by the dotted line in Fig. 2a. This anisotropic broadening is coupled with a small shift of the peak position of the\u000d\u000a110 peaks of the parent structures from one sample to the\u000d\u000aother. According to DIFFaX simulations from Chabre and\u000d\u000aPannetier,12 this peak shift is related to changes in the De\u000d\u000aWolff defect distribution, confirming the presence of intergrown structures in our \u03b3-MnO2 samples. Micro-twinning\u000d\u000aalong the 201 and 601 planes of the R-MnO2 parent structure\u000d\u000acan also happen in \u03b3-MnO2 samples, but only affects the peak\u000d\u000aposition of these particular reflections, according to DIFFaX\u000d\u000acalculations of Chabre and Pannetier.12 We do not observe any\u000d\u000asuch significant effect here. In addition to \u03b3-MnO2, we observe\u000d\u000aimpurities of \u03b1-MnO2, indicated by stars in Fig. 2a. No evidence of a significant fraction of remaining Mn2+ ions from\u000d\u000athe precursors could be observed. Mn2+ containing compounds would likely give rise to a Mn\u2013O peak around 2.2 \u00c5 in\u000d\u000athe PDF,38 which is not observed in our experimental data.\u000d\u000aFurthermore, synthesis of manganese oxides often leads to the\u000d\u000aformation of manganese dioxide hydrate MnO(OH)2 that give\u000d\u000arise to a peak on the XRD data around 0.5 \u00c5\u22121\u000d\u000a.\u000d\u000a39 Such a peak\u000d\u000ais not observed in our experimental data.\u000d\u000aThe PDFs of the three samples (Fig. 2b) show similarities in\u000d\u000apeak positions, but differ in relative intensities. They also\u000d\u000aclearly reflect different sizes of the crystallites, or at least their\u000d\u000acoherent scattering domains, which increases from sample 1\u000d\u000ato 3, as less damping of the PDF signal is seen at high r\u000d\u000avalues. TEM images of the synthesised samples (Fig. 2d\u2013f and\u000d\u000aFig. S1\u2020) do indeed show that samples 1, 2 and 3 show needleshaped particles whose diameter increase with synthesis time,\u000d\u000aand we estimate them to be ca. 7 nm, 15 nm, and 54 nm,\u000d\u000arespectively. According to previous work, the long dimension\u000d\u000aof the needles can be assigned to the [001] direction of the \u03b2and R-MnO2 structures,7,40\u201343 such that the intergrowth direction is along the diameter of the needles. This anisotropic\u000d\u000ashape can also explain partially the anisotropic broadening of\u000d\u000aFig. 2 Experimental XRD patterns (a) and PDFs (b) of samples 1, 2 and 3\u000d\u000ashowing diverse intergrowth rates as relative intensity of reflections of\u000d\u000aXRD and peaks of PDF varies. The presence of a secondary phase\u000d\u000a\u03b1-MnO2 is indicated by (*) in the XRD patterns. Theoretical XRD and PDF\u000d\u000apatterns of R- (red) and \u03b2-MnO2 (black) parent structures are shown as\u000d\u000awell. Yellow, green and blue lines in (b) correspond to the first Mn\u2013O\u000d\u000aand Mn\u2013Mn distances found in these structures as emphasised in (c) in\u000d\u000athe R-MnO2 structure. TEM images of samples 1 (d), 2 (e) and 3 (f).\u000d\u000aPaper Dalton Transactions\u000d\u000a17152 | Dalton Trans., 2022, 51, 17150\u201317161 This journal is \u00a9 The Royal Society of Chemistry 2022\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","the peaks on XRD data. However, the effect of De Wolff defects\u000d\u000ais prominent as it gives rise to even smaller intergrown\u000d\u000adomains within the sample.\u000d\u000aCreation of \u03b3-MnO2 supercells with various De Wolff defect\u000d\u000adistributions\u000d\u000aTo characterise the defect chemistry of the three samples, we first\u000d\u000ainvestigate how different De Wolff defect distributions generally\u000d\u000aaffect the XRD patterns and PDFs. The first step for the development of the methodology was therefore to generate a database of\u000d\u000a\u03b3-MnO2 supercells containing various De Wolff defect distributions using the approach sketched in Fig. 3. To do so, rules\u000d\u000abased on the study from Chabre and Pannetier12 were applied.\u000d\u000aFirst, a pair of asymmetric units A and B (Fig. 3a) were identified\u000d\u000aand described in a primitive cell. Both consist of a pair of [MnO6]\u000d\u000aoctahedra sharing one corner. However, for A, one of the Mn\u000d\u000aatoms is placed at the origin of the cell and the other one at (1/2,\u000d\u000a1/2, 1/2), whereas for B the Mn atoms are placed at (0, 1/2, 1/2)\u000d\u000aand (1/2, 1/2, 0). Stacking of 2 similar units A\u2013A or B\u2013B with a\u000d\u000astacking vector (a\u03b2, 0, 0), (a\u03b2 being the a cell parameter in the\u000d\u000astructure) yields a regular \u03b2-MnO2 structure. In contrast, stacking\u000d\u000aof two different units A\u2013B or B\u2013A yields a regular R-MnO2 structure. Here, the stacking vector is (aR/2, \u00b10.026bR, 0), with aR and\u000d\u000abR being the a and b cell parameters of R-MnO2. The slight shift\u000d\u000a(\u00b10.026) along the b-axis needs to be added to adjust for cornersharing [MnO6] octahedra in the R-MnO2 structure. To keep a\u000d\u000astraight structure, the shift along the b-axis alternates from one\u000d\u000asequence to the other. As such, the unit B will be shifted by\u000d\u000a+0.026bR along the b-axis with respect to unit A in a stacking\u000d\u000asequence A\u2013B, whereas an opposite shift of \u22120.026bR along the\u000d\u000ab-axis is applied in a B\u2013A sequence.\u000d\u000aThe statistical approach of Chabre and Pannetier12 involves\u000d\u000athe definition of De Wolff defect distribution probabilities Pb\u000d\u000aand Pr. Pb corresponds to the probability of having a regular\u000d\u000a\u03b2-MnO2 stacking, i.e. an A\u2013A or a B\u2013B sequence and Pr corresponds to the probability of having a regular R-MnO2 stacking,\u000d\u000ai.e. an A\u2013B or a B\u2013A sequence. A combination of these two\u000d\u000aprobabilities gives the respective phase fractions of \u03b2-MnO2\u000d\u000adomains (%\u03b2-MnO2) and R-MnO2 domains (%R-MnO2) by eqn\u000d\u000a(1) and (2):\u000d\u000a%\u03b2 ! MnO2 \u00bc 1 ! Pr\u000d\u000a2 ! Pr ! Pb\u000d\u000a# 100% \u00f01\u00de\u000d\u000a%R ! MnO2 \u00bc 1 ! %\u03b2 ! MnO2 \u00bc 1 ! Pb\u000d\u000a2 ! Pr ! Pb\u000d\u000a# 100% \u00f02\u00de\u000d\u000aWe generate the supercells with varying Pr and Pb using a\u000d\u000asimple method based on iterative stacking of asymmetrical\u000d\u000aunits A and B. The process is based on setting the value of Pb,\u000d\u000aPr and of a random number x (between 0 and 1), which is generated in each stacking step as shown in Fig. 2b. The following\u000d\u000arules were used: if the i\u000d\u000ath and (i \u2212 1)th units are the same, i.e. A\u000d\u000aand A or B and B, the type of the (i + 1)th unit to stack depends\u000d\u000aon Pb and x. If x < Pb, the stacking remains regular (A\u2013A\u2013A or\u000d\u000aB\u2013B\u2013B) representing \u03b2-MnO2. If x > Pb, the opposite unit is\u000d\u000astacked (resulting in A\u2013B\u2013A or B\u2013A\u2013B), thus generating stacking faults in the \u03b2-MnO2 structure. Conversely, if the i-th and\u000d\u000a(i \u2212 1)th units are different, i.e. A\u2013B or B\u2013A, the nature of the\u000d\u000a(i + 1)th unit depends on Pr. If x < Pr, the same unit present in\u000d\u000athe (i \u2212 1)th step is added, resulting in A\u2013B\u2013A or B\u2013A\u2013B structures, corresponding to a regular R-MnO2 sequence. If x > Pr,\u000d\u000athe same unit is stacked (resulting in A\u2013B\u2013B or B\u2013A\u2013A) creating\u000d\u000aa stacking fault in the R-MnO2 structure.\u000d\u000aWe generated such superstructures in the whole (Pb, Pr)\u000d\u000aparameter space, varying both probabilities between 0.01 and\u000d\u000a0.99 with an increment of 0.01, yielding 9801 different supercells. The starting point was sequence A\u2013A. Each of them consists of 250 stacked sequences. The a cell parameter of the\u000d\u000asupercell is calculated by summing up all individual a-components of the stacking vectors for the whole supercell,\u000d\u000aranging from 900 \u00c5 to 950 \u00c5.\u000d\u000aPattern simulation and peak-fitting analysis\u000d\u000aUsing the database of 9801 supercells generated above, we first\u000d\u000ause a simple, peak-fitting analysis to estimate the defect distribution in our experimental samples from the appearance of\u000d\u000athe scattering patterns and PDFs. XRD and PDF patterns were\u000d\u000acalculated from each of the supercells using TOPAS\u000d\u000aAcademic.44 In both cases, the isotropic atomic displacement\u000d\u000aparameters (ADP) of manganese and oxygen Biso were set to\u000d\u000a0.2 \u00c52\u000d\u000a. A Gaussian profile with a peak width based on a\u000d\u000aCaglioti polynomial, U tan(\u03b8)\u000d\u000a2 + V tan(\u03b8) + W45 with U = 0.42, V\u000d\u000a= \u22120.36 and W = 0.70 was used.\u000d\u000aPDFs from the superstructures were calculated in an\u000d\u000ar-range up to 20 \u00c5 with a Qdamp of 0.04 \u00c5\u22121 and a Qmax of\u000d\u000a23 \u00c5\u22121 to model instrumental parameters close to those of a\u000d\u000areal experiment.\u000d\u000aFig. 4a and d show contour plots of the simulated XRD patterns and PDFs for supercells for which eqn (1) satisfies the\u000d\u000acondition %\u03b2-MnO2 = Pb (and conversely eqn (2) satisfies the\u000d\u000aFig. 3 Creation of supercells. The asymmetric units A and B (a) stacked\u000d\u000ain a regular fashion yield either the \u03b2- or R-MnO2 structure. When\u000d\u000astacked iteratively based on De Wolff defect distribution probabilities Pb\u000d\u000aand Pr (b), superstructures with different amounts of De Wolff defects\u000d\u000aare created. (b) gives an example of a stacking sequence where the (i \u2212\u000d\u000a1)th unit A is assumed to be part of a \u03b2-MnO2 block, and therefore the\u000d\u000astacking of the i\u000d\u000ath unit relies on Pb.\u000d\u000aDalton Transactions Paper\u000d\u000aThis journal is \u00a9 The Royal Society of Chemistry 2022 Dalton Trans., 2022, 51, 17150\u201317161 | 17153\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","condition %R-MnO2 = Pr) in the range 0% \u2264 %\u03b2-MnO2 \u2264\u000d\u000a100%. The modelled XRD patterns show good agreement with\u000d\u000athose previously reported.12,46 When %\u03b2-MnO2 is close to 0%,\u000d\u000athe PDF and XRD pattern show features and Bragg peaks of\u000d\u000apure R-MnO2 (Fig. 4b). Similarly, when %\u03b2-MnO2 tends\u000d\u000atowards 100%, XRD peaks sharpen, i.e. the structure orders\u000d\u000aback again. At %\u03b2-MnO2 = 100%, the XRD patterns and PDFs\u000d\u000ashow similar features to the ones calculated from \u03b2-MnO2 as\u000d\u000ashown in Fig. 4c and f, respectively.\u000d\u000aWhen the \u03b2-MnO2 fraction deviates from the two end\u000d\u000amembers (0% < %\u03b2-MnO2 < 100%), reflections from both\u000d\u000aparent structures can be observed. Some reflections strongly\u000d\u000abroadened, indicating a loss of long-range order in the \u03b3-MnO2\u000d\u000astructure. These are for instance the 110 reflections of both \u03b2and R-MnO2 reflecting disorder along the a-axis. On the contrary, 011 reflections are not affected by the intergrowth.\u000d\u000aThe simulated data show that an estimate of the \u03b2-MnO2\u000d\u000afraction can be obtained from the intensity ratio between the\u000d\u000a011 reflections of R- and \u03b2-MnO2. For the simulated XRD patterns we extract the intensities and FWHMs by fitting PseudoVoigt functions to these peaks. An example of such a fit is\u000d\u000ashown in Fig. S2a.\u2020 With increasing \u03b2-MnO2 fraction, a\u000d\u000adecrease of the intensity ratio (I011R\u000d\u000a/I011\u03b2\u000d\u000a) can be observed. By\u000d\u000afitting an empirical quadratic function (eqn (3)) to the calculated data points, the intensities of the 011 reflection of the \u03b2and R-phases can be linked to their structure factors F(hkl)\u000d\u000aand to %-\u03b2-MnO2 via eqn (3):\u000d\u000aI011R\u000d\u000aI011\u03b2\u000d\u000a\u00bc F011R j j2\u000d\u000aF011\u03b2\u000d\u000a!\u000d\u000a! !\u000d\u000a!\u000d\u000a2\u000d\u000a1 ! %\u03b2 ! MnO2\u000d\u000a%\u03b2 ! MnO2\u000d\u000a\u00f03\u00de\u000d\u000aThis procedure can be compared to a simple phase quantification in a binary mixture system, where the intensity ratio of\u000d\u000atwo peaks is proportional to the ratio of the phase fractions,\u000d\u000aweighted by the ratio of their squared structure factors. The\u000d\u000afitted (I011R\u000d\u000a/I011\u03b2\u000d\u000a) ratio is plotted as function of %\u03b2-MnO2 in\u000d\u000aFig. 5a. We note that some data points (%\u03b2-MnO2 < 20%,\u000d\u000aFig. 5a) do not follow the same trend. This can be attributed to\u000d\u000athe fact that the 011 reflections of \u03b2- and R-MnO2 have very\u000d\u000asimilar d-spacings. In the case where the structure of a \u03b3-MnO2\u000d\u000asample is closely related to those of one of the parents, its\u000d\u000areflections will dominate the XRD pattern. In that situation it\u000d\u000abecomes difficult to fit two peaks as the XRD would show a\u000d\u000astrong reflection next to a weak one and some lack of convergence may thus occur.\u000d\u000aWhile this intensity ratio can inform on %\u03b2-MnO2 from\u000d\u000aXRD patterns, a second relation is needed to fully quantify the\u000d\u000astacking disorder within \u03b3-MnO2 intergrowth. Once %-\u03b2-MnO2\u000d\u000ais determined, we still need to find Pb or Pr as they are all\u000d\u000arelated to each other through eqn (1). Determining these three\u000d\u000avalues would allow not only to get an estimate of the fraction\u000d\u000aof both parent structures, but also how their domains are distributed within the intergrowth. For this purpose, the ratio\u000d\u000abetween the FWHM of 011R and 110R is calculated for all\u000d\u000asimulated XRD patterns of the \u03b3-MnO2 superstructures. While\u000d\u000athe first reflection is not impacted by the intergrowth (its structure factor does not have any contribution from the position of\u000d\u000aatoms along the a-axis) the second one is. Therefore, a given\u000d\u000afraction of R-MnO2 in the intergrowth will impact the relative\u000d\u000abroadening of these two reflections which will depend on the\u000d\u000adomain size of R-MnO2. The calculated ratio of XRD patterns\u000d\u000aof all supercells having the same \u03b2-MnO2 fraction are then\u000d\u000aplotted against Pr and are shown in Fig. S3.\u2020 When values for\u000d\u000aPr and %\u03b2-MnO2 have been established, Pb can be calculated\u000d\u000ausing eqn (1).\u000d\u000aWe can now calculate the \u03b2-MnO2 fraction, Pb and Pr for our\u000d\u000athree experimental XRD datasets of sample 1, 2 and 3. The\u000d\u000aBragg peak intensity ratio was determined for each of the three\u000d\u000asamples by fitting Pseudo-Voigt functions to the 110R, 110\u03b2,\u000d\u000a011\u03b2 and 011R reflections of the experimental data as shown in\u000d\u000aFig. S4a, d and g.\u2020 The corresponding \u03b2-MnO2 fraction can be\u000d\u000aestimated from the ratio of intensities as presented in eqn (3).\u000d\u000aOnce this value is determined, Pr is estimated using the fitted\u000d\u000aFWHM of 011R and 110R and, Pb is calculated using eqn (1).\u000d\u000aThe resulting %\u03b2-MnO2, Pb and Pr for the three samples are\u000d\u000alisted in Table 1. The \u03b2-MnO2 fraction gradually increases from\u000d\u000aFig. 4 Modelled XRD patterns (a) and PDFs (d) from supercells for\u000d\u000awhich eqn (1) satisfies the condition %\u03b2-MnO2 = Pb, 110 (1.5 \u00c5\u22121 and\u000d\u000a2.0 \u00c5\u22121\u000d\u000a) and 011 (ca. 2.6 \u00c5\u22121\u000d\u000a) reflections of R-MnO2 and \u03b2-MnO2 are\u000d\u000ashown by (\u00b0) and (*), respectively. Modelled XRD patterns (b) and PDFs\u000d\u000a(e) for %\u03b2-MnO2 \u2192 0% (red curve) compared to parent R-MnO2 (blue\u000d\u000acurve). Modelled XRD patterns (c) and PDFs (f) for %\u03b2-MnO2 \u2192 100%\u000d\u000a(red curve) compared to parent \u03b2-MnO2 (blue curve).\u000d\u000aFig. 5 (a) Intensity ratio between 011R and 011\u03b2 lines from the modelled\u000d\u000aXRD patterns plotted against the \u03b2-MnO2 fraction (%\u03b2-MnO2, blue dots)\u000d\u000aand fitted using eqn (3) (red solid line). (b) Intensity ratio between PDF\u000d\u000apeaks at 2.8 \u00c5 and 3.4 \u00c5 from the modelled PDFs plotted against\u000d\u000a%\u03b2-MnO2 (blue dots) and fitted using a linear function (red solid line).\u000d\u000aPaper Dalton Transactions\u000d\u000a17154 | Dalton Trans., 2022, 51, 17150\u201317161 This journal is \u00a9 The Royal Society of Chemistry 2022\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","sample 1 to 3. This can be related to the chemistry of the\u000d\u000asystem. The chosen synthesis pathway leads initially to a metastable R-MnO2 phase that reorders into \u03b2-MnO2 as the synthesis progresses. Conversely, the structure becomes more and\u000d\u000amore ordered, as Pb and Pr both increase with synthesis time.\u000d\u000aBesides having more and more \u03b2-MnO2 blocks in the structure,\u000d\u000athe \u03b2- and R-MnO2 blocks tend to segregate with time.\u000d\u000aHaving established a simple method for intergrowth characterisation from XRD data, we can use a similar approach for\u000d\u000aPDF analysis. We first identify peaks arising from specific\u000d\u000amotifs in the \u03b2-MnO2 and R-MnO2 structure. The PDF peak at\u000d\u000a2.8 \u00c5 can be attributed to the Mn\u2013Mn distance between two\u000d\u000a[MnO6] edge-sharing octahedra, while the peak at 3.4 \u00c5 arises\u000d\u000afrom the Mn\u2013Mn distance between two [MnO6] corner-sharing\u000d\u000aoctahedra as shown in Fig. 2c. R-MnO2 contains more edgesharing octahedra in its structure (4) than \u03b2-MnO2 (2).\u000d\u000aConversely, a manganese atom has less 2nd neighbours at\u000d\u000a3.4 \u00c5 in R-MnO2 (where it has 4) than \u03b2-MnO2 (with 8 neighbours). The intensity ratio between the height of the two peaks\u000d\u000acan thus be used as a measure for %\u03b2-MnO2. We calculated\u000d\u000athe ratio for each of the simulated PDFs by fitting Gaussian\u000d\u000afunctions to the two peaks, an example of the fitting procedure\u000d\u000ais shown in Fig. S2b.\u2020 The results are plotted as a function of\u000d\u000a%\u03b2-MnO2 in Fig. 5b. In contrast to the XRD analysis, a linear\u000d\u000atrend is observed for the PDF analysis. This can be explained\u000d\u000aby the fact that the amount of edge- and corner-sharing octahedra changes linearly when switching from blocks of \u03b2-MnO2\u000d\u000ato blocks of R-MnO2. The obtained data can be fitted using a\u000d\u000alinear regression (red line, Fig. 5b). The %\u03b2-MnO2 fractions of\u000d\u000asample 1, 2 and 3 were calculated by this approach and are\u000d\u000areported in Table 1.\u000d\u000aAs in the XRD analysis, a second trend is used to quantify Pb\u000d\u000aand Pr once %\u03b2-MnO2 is determined. The intensity ratio\u000d\u000abetween the peaks at 5.3 \u00c5 and 5.7 \u00c5 is calculated for \u03b3-MnO2\u000d\u000asuperstructures and plotted as a function of Pb for superstructures with a given %\u03b2-MnO2 in Fig. S5.\u2020 Examples of fits\u000d\u000aare shown in Fig. S2c.\u2020 The two peaks used here correspond to\u000d\u000asecond and third nearest neighbour in a (2 \u00d7 1) channel and\u000d\u000awhile the former is present in both \u03b2- and R-MnO2 structures,\u000d\u000athe latter can only be found in R-MnO2. However, because in an\u000d\u000aintergrown structure a \u03b2-MnO2 block can be next to a R-MnO2\u000d\u000ablock, these two peaks may arise as well. Therefore, the ratio of\u000d\u000athese two peaks contain information about how many \u03b2-MnO2\u000d\u000ablocks are connected to R-MnO2 blocks, i.e. the stacking probabilities Pr and Pb. The calculated ratio can be fitted using a\u000d\u000alinear function. Again, we can use these correlations to determine the defect density in our three samples. For each experimental PDF, the %\u03b2-MnO2 is first calculated and used to extract\u000d\u000athe corresponding Pb value shown in Table 1. Knowing both\u000d\u000a%\u03b2-MnO2 and Pb, Pr can be determined using eqn (1). Using the\u000d\u000adescribed procedure, we can now estimate %\u03b2-MnO2, Pb and Pr\u000d\u000afor the experimental samples 1, 2 and 3 and the fitting of the\u000d\u000aPDF peaks is shown in Fig. S4b, c, e, f, h, and i.\u2020 The results are\u000d\u000asummarised in Table 1. We are thus able to get a full picture of\u000d\u000athe intergrowth in \u03b3-MnO2 using PDF analysis.\u000d\u000aAs seen in Table 1, the values extracted from the XRD and\u000d\u000aPDF analysis are of the same order of magnitude. However,\u000d\u000asome variations can be observed, especially regarding the estimation of Pb and Pr. In general, the peak analysis from PDF\u000d\u000agives higher values than those obtained from XRD. This could\u000d\u000aimply that some other effects influence the intensity of PDF\u000d\u000apeaks in the local range. Indeed, other types of defects, such\u000d\u000aas micro-twinning and proton-induced defects may be seen in\u000d\u000aR-MnO2-like intergrowth samples (as in sample 1).12,13,47,48\u000d\u000aThose may alter the short-range order in the crystal, and thus\u000d\u000athe intensity of the first peaks of the PDF. The proton-induced\u000d\u000adefects consist of substitution of Mn4+ by Mn3+ along with the\u000d\u000aformation of hydroxyl groups on the neighbouring O2\u2212 site, or\u000d\u000aby compensation of a manganese vacancy by four hydroxyl\u000d\u000agroups. Such defects may have a large impact on the properties\u000d\u000aof the material,49 however, characterization of these is outside\u000d\u000athe scope of this paper, and will be addressed in a future\u000d\u000astudy. Finally, some stacking faults in the MnO2 O2\u2212 hcp sublattice has been predicted50 but no experimental evidence of\u000d\u000asuch stacking events has been reported.\u000d\u000aStructure refinement and structure-mining\u000d\u000aThe peak-fitting-approach presented above gives a good first estimate of the defect density in the materials. However, to further\u000d\u000aextend our analysis and to identify the best model to describe\u000d\u000athe data, we use \u2018structure-mining\u2019. Here, the structures generated above are fitted to the experimental data, and the trends in\u000d\u000athe goodness of fits of the different models are investigated. In\u000d\u000aorder to optimise calculation time, Pb and Pr were limited to (0.0\u000d\u000a< Pb < 1.0, 0.5 < Pr < 1.0) for sample 1, (0.5 < Pb < 1.0, 0.5 < Pr <\u000d\u000a1.0) for sample 2 and (0.5 < Pb < 1.0, 0.0 < Pr < 1.0) for sample 3\u000d\u000aaccording to the results of the peak-fitting analysis.\u000d\u000aAll the structure refinements were performed using\u000d\u000aTOPAS.44 For the XRD analysis, the supercell cell parameters a,\u000d\u000ab and c and the isotropic atomic displacement parameters\u000d\u000a(ADP) for both manganese and oxygen were refined along with\u000d\u000abackground and peak shape parameters, as described further\u000d\u000ain Table S1 in the ESI.\u2020 To minimise the number of refinement parameters, all atomic coordinates in the supercell are\u000d\u000aexpressed as a function of the ones in the asymmetric unit. To\u000d\u000aachieve that, a translational operation in accordance with the\u000d\u000astacking sequence in the supercell is applied to the atoms in\u000d\u000aeach layer. In addition, the hollandite \u03b1-MnO2 structure was\u000d\u000aadded as a secondary phase. A similar approach was used for\u000d\u000athe PDF analysis.\u000d\u000aTable 1 Calculated %\u03b2-MnO2, Pb and Pr values of samples 1, 2, 3 based\u000d\u000aon XRD and PDF peak-fitting analysis\u000d\u000aSample\u000d\u000a%\u03b2-MnO2\u000d\u000aXRD\u000d\u000aPb\u000d\u000aXRD\u000d\u000aPr\u000d\u000aXRD\u000d\u000a%\u03b2-MnO2\u000d\u000aPDF\u000d\u000aPb\u000d\u000aPDF\u000d\u000aPr\u000d\u000aPDF\u000d\u000aSample\u000d\u000a1\u000d\u000a27 0.53 0.83 21 0.4 0.84\u000d\u000aSample\u000d\u000a2\u000d\u000a40 0.72 0.81 48 0.69 0.72\u000d\u000aSample\u000d\u000a3\u000d\u000a82 0.97 0.9 87 0.98 0.87\u000d\u000aDalton Transactions Paper\u000d\u000aThis journal is \u00a9 The Royal Society of Chemistry 2022 Dalton Trans., 2022, 51, 17150\u201317161 | 17155\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","Fig. 6 shows the Rwp values obtained for the fitting of\u000d\u000amined structures for samples 1, 2 and 3. The Rwp values are\u000d\u000arepresented in a colour map, where the Pr and Pb values used\u000d\u000afor the superstructure generation are plotted on the x- and\u000d\u000ay-axes. The results are also represented in one dimension as a\u000d\u000afunction of \u03b2-MnO2 fraction of the refined superstructures\u000d\u000a(Fig. 6b\u2013q). The Rwp value corresponding to a simple twophase refinement using the two parent structures (\u03b2-MnO2 and\u000d\u000aR-MnO2) on the same experimental dataset is shown as a red\u000d\u000avertical line (Fig. 6b\u2013q).\u000d\u000aWhen performing refinements of the mined superstructures on the three samples, we see that the lowest Rwp\u000d\u000avalues are systematically clustered in a region of the (Pb, Pr)\u000d\u000aspace (dark blue regions). This is the case for both the XRD\u000d\u000aand PDF analysis, which means we can use this region as a\u000d\u000a\u03b2-MnO2 fraction estimate: for sample 1, we estimate the\u000d\u000a\u03b2-MnO2 fraction to be in the range 20\u201340%, sample 2 in the\u000d\u000arange 40\u201360% and for sample 3 in the range 80\u2013100% (see\u000d\u000aFig. 6b\u2013q respectively). The fact that a whole range of supercells yielded similar fits, i.e. similar Rwp values, implies that in\u000d\u000athe case of such intergrown system, a single model is not\u000d\u000asufficient to describe the material structure, as also seen from\u000d\u000athe poor fit quality for especially sample 1 which is further discussed below. This can be related to the random nature of the\u000d\u000astacking of R- and \u03b2-MnO2 blocks in the supercell. Indeed,\u000d\u000asome key structural motifs in different quantities may be\u000d\u000apresent in all the superstructures that yielded a good fit in the\u000d\u000ablue area on Fig. 6a\u2013g and j\u2013p. The idea of structural motifs\u000d\u000awill be further investigated in the next section.\u000d\u000aWhen comparing the results of the fitting of mined structures on sample 1, 2 and 3 for XRD and PDF (Fig. 6b\u2013q respectively) as a function of %\u03b2-MnO2, we observe that the dip in the\u000d\u000aRwp around the best fits is deeper for the XRD superstructure\u000d\u000arefinements than for PDF. This indicates that XRD is more sensitive to disorder in the intergrowth sequence than PDF. XRD\u000d\u000aprobes the average structure of a material while PDF probes\u000d\u000athe local structure. Defects such as intergrowth affect the longrange structure, which is better characterised in XRD.\u000d\u000aFig. 6 Results from fitting of mined structures of sample 1, 2 and 3. Contour plots show Rwp values from Rietveld (a, d and g) and PDF refinements\u000d\u000a( j, m and p) as a function of (Pb, Pr) for each superstructure for samples 1, 2 and 3, respectively. The same Rwp values as a function of the \u03b2-MnO2\u000d\u000afraction are shown in (b, e and h) and (k, n and q), respectively. The red solid line represents the Rwp value of the refinement of a two-phase mixture\u000d\u000aof parent structure \u03b2- and R-MnO2. The Rietveld and PDF refinement of the superstructure with (Pb, Pr) values determined from the peak-fitting analysis are shown in (c, f and h) and (l, o and r), respectively.\u000d\u000aPaper Dalton Transactions\u000d\u000a17156 | Dalton Trans., 2022, 51, 17150\u201317161 This journal is \u00a9 The Royal Society of Chemistry 2022\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","The results of the peak-fitting analysis were also compared\u000d\u000ato the ones from the structure-mining approach. The fit of the\u000d\u000asuperstructures corresponding to the (Pb, Pr) values obtained\u000d\u000afrom the peak-fitting analysis of samples 1, 2 and 3 are shown\u000d\u000ain Fig. 6c\u2013i (XRD) and l\u2013r (PDF). The Pr and Pb values identified from the peak-fitting analysis fall into the region where\u000d\u000asuperstructures fitted better to the experimental data than a\u000d\u000atwo-phase mixture of the parent structures. However, some discrepancies are still observed, especially for sample 1 and 2.\u000d\u000aThis is more clearly seen in Fig. S6,\u2020 showing a zoom of the\u000d\u000alow Q region of the Rietveld refinements along with the low r\u000d\u000aregion or the PDF refinements presented in Fig. 6. For the\u000d\u000aRietveld refinement of the XRD data, the intensity of some\u000d\u000areflections is not fully described, including the 110R and 110\u03b2\u000d\u000areflections. Furthermore, the 201R reflection at 2.4 \u00c5\u22121 is not\u000d\u000amatched in position or in intensity. In r-space, the intensity of\u000d\u000athe first peaks, at 1.90 \u00c5, 2.88 \u00c5 and 3.40 \u00c5 are not well\u000d\u000adescribed. The discrepancies in the fits can arise from\u000d\u000adifferent effects. Firstly, the limited size of the supercells (250\u000d\u000aasymmetric units) may not provide a sufficient statistical\u000d\u000aaverage of the De Wolff defects distribution in the sample,\u000d\u000awhich affects the description of the peak intensities in the\u000d\u000aXRD patterns. A second effect, as observed from TEM (Fig. 2\u000d\u000aand Fig. S1\u2020), is the particle shape, which affects the Bragg\u000d\u000apeak width and r-dependent PDF peak intensity. Finally, as\u000d\u000adiscussed above, the \u03b3-MnO2 structures can contain several\u000d\u000atypes of defects, i.e. De Wollf or intergrowth defects, microtwinning and H-related defects. All of them impact the XRD\u000d\u000apatterns and PDFs. The aim of the present work is to develop a\u000d\u000asimple and fast method, building on structure mining, which\u000d\u000acan reveal trends in De Wolff defect densities. Therefore, our\u000d\u000astructure refinements may miss some of the information\u000d\u000ahidden in the scattering signal of the samples as we focus here\u000d\u000aon only one aspect of the problem.\u000d\u000aThe Rwp values for the corresponding superstructures and\u000d\u000afor the two-phase refinement of the parent structures are compared and summarised in Table 2. The superstructure refinements on experimental data generally yield a better fit than\u000d\u000athe refinement of the simple mix of the parent structures, with\u000d\u000aRwp values being ca. 3% systematically lower. An exception\u000d\u000aarises for the PDF refinements of sample 1 and 3. For sample\u000d\u000a1, the Rwp values obtained for analysis in both Q- and r-space\u000d\u000aare higher than those of the two other samples. Broader reflections in Q-space as well as a rapid damping of the PDF in\u000d\u000adirect space imply that sample 1 contains smaller crystallites,\u000d\u000aas also seen from our TEM analysis.\u000d\u000aNanostructure analysis\u000d\u000aIn order to extract more information from the structuremining process, we use our recently published approach for\u000d\u000astructure analysis, namely the Machine Learning-based Motif\u000d\u000aExtractor (ML-MotEx).51 The idea of ML-MotEx is to give the\u000d\u000aresults from a large set of structure refinements to an explainable ML algorithm, which learns to predict the goodness-of-fit\u000d\u000avalue for a structural refinement to the experimental data.\u000d\u000aMost importantly, the use of an explainable ML algorithm\u000d\u000amakes it possible to obtain information on the effect that each\u000d\u000afeature of the model has on the fit quality. ML-MotEx uses a\u000d\u000aGradient Bosting Decision Tree (GBDT)52 to predict the Rwp\u000d\u000avalue from structural information. SHAP (SHapley Additive\u000d\u000aexPlanation)53,54 values are used to explain each feature\u2019s\u000d\u000aeffect on the Rwp value by taking into consideration the full\u000d\u000adataset given as input. While the magnitude of the SHAP value\u000d\u000aquantifies the importance of a feature, its sign signifies\u000d\u000awhether the presence of the feature in the model affects the\u000d\u000aRwp value positively or negatively, as described further below.\u000d\u000aThe method is also described in detail in our recent\u000d\u000apublication.51\u000d\u000aWe here want to use the ML-MotEx method to extract information on the nanostructure of the samples, i.e. determine\u000d\u000athe size of the R-MnO2 and \u03b2-MnO2 domains present in the\u000d\u000aintergrowth structures. Results from each of the 2500 superstructure refinements, performed on both XRD and PDF data,\u000d\u000awere used as input parameters for ML-MotEx. The structural\u000d\u000afeatures investigated using explainable ML were the \u03b2-MnO2\u000d\u000afraction, Pb and Pr, and the intergrowth sequence of each\u000d\u000asupercell. This input allows to investigate what structural features are important for the fit quality.42 The intergrowth\u000d\u000asequence of a superstructure is generated from the asymmetric\u000d\u000aunit sequence used to build it (presented in Fig. 3). By taking\u000d\u000athe asymmetric unit at the i\u000d\u000ath and (i \u2212 1)th position we determine whether a \u201cblock\u201d constituting the intergrowth sequence\u000d\u000acorresponds to \u03b2- or R-MnO2.\u000d\u000aFig. 7a shows the results of using ML-MotEx on the PDF\u000d\u000ameasured on sample 2. ML-MotEx on the XRD and PDF\u000d\u000aobtained for samples 1 and 3 are shown in Fig. S7 in the ESI.\u2020\u000d\u000aFor each fit, a SHAP value is obtained for all of the features.\u000d\u000aWe first consider the %\u03b2-MnO2 fraction, which we also investigated from the XRD and PDF analysis above. In the figure, the\u000d\u000afeature value (i.e. here %\u03b2-MnO2) is shown on the colour axis,\u000d\u000awhere red represents a low %\u03b2-MnO2 fraction, while blue represents a high %\u03b2-MnO2 fraction. The SHAP values obtained\u000d\u000afor this feature show that when %\u03b2-MnO2 is either high (red/\u000d\u000a\u223c100%) or low (blue/\u223c0%), the SHAP values are positive. This\u000d\u000ameans that the Rwp values are increased in these fits.\u000d\u000aConversely, medium %\u03b2-MnO2 values (purple, ca. 50%) show\u000d\u000anegative SHAP values and thus lower the Rwp value. This\u000d\u000aagrees well with the results from the analysis in Fig. 6e, where\u000d\u000awe determined %\u03b2-MnO2 to be 48% for sample 2. The SHAP\u000d\u000aTable 2 Rwp values of Rietveld and PDF refinements of the superstructures corresponding to results from peak-fitting analysis and from\u000d\u000arefinements of parent structures \u03b2- and R-MnO2 on samples 1, 2 and 3.\u000d\u000aFits can be found in Fig. 6c, f and Fig. S6(a\u2013c)\u2020\u000d\u000aSample Refinement\u000d\u000aRwp, superstructures (%)\u000d\u000aRwp, \u03b2-MnO2 +\u000d\u000aR-MnO2 (%)\u000d\u000aSample 1 Rietveld 13.7 16.7\u000d\u000aPDF 52.3 47.9\u000d\u000aSample 2 Rietveld 11.2 14.5\u000d\u000aPDF 40.0 44.4\u000d\u000aSample 3 Rietveld 9.5 13.4\u000d\u000aPDF 36.9 33.6\u000d\u000aDalton Transactions Paper\u000d\u000aThis journal is \u00a9 The Royal Society of Chemistry 2022 Dalton Trans., 2022, 51, 17150\u201317161 | 17157\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","analysis also suggests that the Rwp value of the refinements\u000d\u000acan be pushed down by having a low Pb value. This is new\u000d\u000ainformation as refinements of the mined structures shown in\u000d\u000aFig. 6m did not indicate a region of better fits for particularly\u000d\u000alow Pb values. Finally, the SHAP analysis shows that the Pr\u000d\u000avalue does not seem to have a strong effect on the fit quality.\u000d\u000aTo get deeper understanding of the influence of De Wolff\u000d\u000adefect distribution on the fit, we next investigate the effect of\u000d\u000athe intergrowth sequence itself on the refinements through the\u000d\u000aSHAP analysis. We correlate here the nature of individual intergrowth block within the intergrowth sequence to the Rwp values\u000d\u000ausing the SHAP analysis. As each block in the sequence is\u000d\u000adirectly related to the structure refined to the experimental data,\u000d\u000ait is possible to establish a correlation between each intergrowth\u000d\u000ablock and the Rwp value when investigating the whole range of\u000d\u000asupercells refined to one single experimental dataset. The influence of the intergrowth sequence on the fit quality is illustrated\u000d\u000ain Fig. 7b, where \u201cintergrowth block X\u201d parameters refer to a\u000d\u000aspecific block in the 250-block long intergrowth sequence.\u000d\u000aHere, a positive feature value (red colour in Fig. 7a) refers to a\u000d\u000a\u03b2-MnO2 block. Conversely, a negative feature value (blue colour\u000d\u000ain Fig. 7a) refers to a block of R-MnO2. As an example, the SHAP\u000d\u000avalue for intergrowth block 79 in Fig. 7a is red in the negative\u000d\u000aregion which means that the Rwp value of the refinement is\u000d\u000adecreased if the unit is a \u03b2-MnO2 block. Conversely, the SHAP\u000d\u000avalue is blue in the positive region so a R-MnO2 block will\u000d\u000aincrease the Rwp of the refinement. The SHAP analysis performed on XRD results of sample 2, along with PDF and XRD\u000d\u000aresults of samples 1 and 3 are shown in Fig. S7.\u2020\u000d\u000aBased on the interpretation of SHAP values given above, we\u000d\u000asearch for any continuous R- or \u03b2-MnO2 domains. First, we\u000d\u000asort the intergrowth sequence blocks by whether it is more\u000d\u000abeneficial for the refinement to have them as a \u03b2- or a R-MnO2\u000d\u000ablock, i.e. when the SHAP value of a given block should rather\u000d\u000abe negative or positive to push down the Rwp value of the\u000d\u000arefinement. Then, we screen the complete intergrowth\u000d\u000asequence to search for sequences of consecutive \u201cpreferentially\u000d\u000a\u03b2-MnO2 block\u201d or \u201cpreferentially R-MnO2 block\u201d and count the\u000d\u000anumber of similar blocks in these sub-sequences. The result of\u000d\u000athis analysis is shown in Fig. 7b where blocks of R- and\u000d\u000a\u03b2-MnO2 are shown in blue and red, respectively. One can\u000d\u000anotice that these blocks are not randomly distributed in the\u000d\u000aintergrowth sequence but rather segregated into some small\u000d\u000adomains of a few consecutive R- or \u03b2-MnO2 blocks. We extract\u000d\u000athe size of these continuous domains of R-MnO2 and \u03b2-MnO2\u000d\u000aby multiplying this number to the unit cell parameter a of Rand \u03b2-MnO2. The number of domains of a given size is\u000d\u000acounted and normalised by the total number of domains. The\u000d\u000aresulting domain size distributions are plotted as a function of\u000d\u000asize to build a Domain Size Distribution (DSD) function. This\u000d\u000aprocess is carried out for the results from structural refinements performed on both XRD and PDF data for sample 1, 2\u000d\u000aand 3 and shown in Fig. 8. The results show small domain\u000d\u000aFig. 7 Summary of the ML-MotEx analysis of the PDF data measured at\u000d\u000asample 2. Violin plot of the SHAP values obtained in the analysis of the\u000d\u000aPDF data measured on sample 2, showing preferable % \u03b2-MnO2, Pb and\u000d\u000aPr values and if intergrowth blocks in the starting model are favorable\u000d\u000arather as a \u03b2- or R-MnO2 block for the fit quality. (a) Each block in the\u000d\u000asupercell is colored with respect to the results from the ML-MotEx analysis. If ML-MotEx prefers the block to be R-MnO2, it is colored blue, if\u000d\u000aML-MotEx does not differentiate between the blocks, it is colored black\u000d\u000aand if ML-MotEx prefers \u03b2-MnO2 it is colored red. (b) Representation of\u000d\u000aa \u03b2-MnO2 and of a R-MnO2 block (c).\u000d\u000aFig. 8 Domain Size Distributions (DSD) determined by Whole Powder\u000d\u000aPattern Modelling (WPPM), ML-MotEx analysis of XRD (ML XRD) and PDF\u000d\u000a(ML PDF) refinement results of R- and \u03b2-MnO2 domains in samples 1 (a\u000d\u000aand b), 2 (c and d) and 3 (e and f).\u000d\u000aPaper Dalton Transactions\u000d\u000a17158 | Dalton Trans., 2022, 51, 17150\u201317161 This journal is \u00a9 The Royal Society of Chemistry 2022\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","sizes of 1\u20132 nm depending on the sample and the parent structure (either R- or \u03b2-MnO2), as discussed below.\u000d\u000aAs a reference for the approach presented above, we also\u000d\u000aanalysed the line profile of individual reflections for both Rand \u03b2-MnO2 in the experimental data using Whole Powder\u000d\u000aPattern Modelling (WPPM). We carry out this analysis independently of the structure mining approach. This method,\u000d\u000adetailed in ESI Notes 1,\u2020 was used in order to determine the\u000d\u000adomain size distribution in the intergrowth direction, i.e. the\u000d\u000a[100] direction for both \u03b2-MnO2 and R-MnO2 domains. The\u000d\u000asoftware PM2K developed by Leoni et al.55 was used to estimate the DSD of \u03b2-MnO2 and R-MnO2 domains based on the\u000d\u000a110R and 110\u03b2 reflections of the XRD patterns of the three\u000d\u000asamples. The coefficients of the Caglioti-based instrumental\u000d\u000aprofile were determined by analyzing the XRD pattern from\u000d\u000athe Si standard beforehand and a Chebyshev polynomial function was used to model the background. A lognormal function\u000d\u000awas used to model the DSD of \u03b2-MnO2 and R-MnO2 domains\u000d\u000aand the average domain size and standard deviation in the\u000d\u000acorresponding directions were determined. The fitting of the\u000d\u000apeaks is shown in Fig. S8.\u2020 From the lognormal distribution\u000d\u000acenter \u03bc and variance \u03c32\u000d\u000a, the average crystallite size (Daverage)\u000d\u000aand standard deviation of the crystallite size distribution (s.d.)\u000d\u000acan be derived as shown in eqn (4) and (5) below:\u000d\u000aDaverage \u00bc e \u03bc\u00fe\u03c32\u000d\u000a\u00f0 \u00de2 \u00f04\u00de\u000d\u000as:d: \u00bc \u00f0e 2\u03bc\u00fe\u03c3 2\u000d\u000a\u00f0e \u03c3 2\u000d\u000a! 1\u00de\u00de \u00f05\u00de\u000d\u000aFinally, as we aim to determine the DSD of R- and \u03b2-MnO2\u000d\u000adomains along the [100] direction, we need to make some\u000d\u000aassumptions since we use the 110 reflection which has a contribution in the [100] and [010] directions. We assume that the size\u000d\u000aof the domains is large along the [010] direction (no intergrowth\u000d\u000ain this direction) and we correct the size by using the relation (a2\u000d\u000a+ b2\u000d\u000a)\u000d\u000a1/2/a, i.e., 21/2/2 for \u03b2-MnO2 domains, and ca. 1.11 for R-MnO2\u000d\u000adomains. By doing that, we can consider the same dimension for\u000d\u000athe results from DSD and ML-MotEx as the latter is already\u000d\u000alooking at number of intergrowth blocks along the [100] direction.\u000d\u000aThe DSDs extracted from the WPPM analyses are plotted in Fig. 8\u000d\u000aand compared to the ones obtained by ML-MotEx.\u000d\u000aThe DSD analysis show some differences between sample 1, 2\u000d\u000aand 3. The R-MnO2 DSDs, which is centred around 1.0 nm,\u000d\u000abecome narrower with increasing synthesis time. For instance,\u000d\u000athe average crystallite size estimated from WPPM is 2.5 \u00b1 1.4 nm\u000d\u000afor sample 1 and 1.3 \u00b1 0.9 nm for sample 3. This corresponds to\u000d\u000aonly one to two unit cells of regular R-MnO2 domains within the\u000d\u000aintergrowth structure. Meanwhile, the \u03b2-MnO2 DSD shows a\u000d\u000anarrow distribution of small domains of 1.4 \u00b1 1.0 nm in sample\u000d\u000a1, which broadens and shifts towards larger domain size in\u000d\u000asample 2 (2.5 \u00b1 1.7 nm) and 3 (3.7 \u00b1 2.2 nm). Similar trends in\u000d\u000asize are seen from the ML-Motex results. From this observation,\u000d\u000ait seems that in \u03b3-MnO2, the intergrowth structure is stabilised by\u000d\u000asmall R-MnO2 domains, around which \u03b2-MnO2 domains may\u000d\u000agrow. This is in line with theoretical studies from Sun et al.\u000d\u000awhere they showed that the R-MnO2 structure is the most\u000d\u000athermodynamically stable MnO2 polymorph at small crystallite\u000d\u000asizes up to 5 nm.56 Beyond that size, \u03b2-MnO2 becomes the most\u000d\u000athermodynamically stable MnO2 polymorph. The calculated DSD\u000d\u000aof the supercells determined by the peak-fitting method are\u000d\u000ashown in Fig. S9\u2020 and show similar trends as those that we\u000d\u000aobserve using WPPM and ML. While the R-MnO2 DSD remains\u000d\u000athe same from sample 1 to 3, the \u03b2-MnO2 domains in the supercell become larger, although shifting towards larger values than\u000d\u000aexpected for sample 3, with domain sizes ranging up to 12 nm.\u000d\u000aBecause we investigate here only one single supercell, there can\u000d\u000aonly exist a limited number of large domains, in opposition to\u000d\u000athe ML results where these large domains are mitigated by\u000d\u000ahaving some intergrowth blocks that are estimated are as neither\u000d\u000a\u201cpreferentially \u03b2-MnO2 block\u201d or \u201cpreferentially R-MnO2 block\u201d.\u000d\u000aThe similarity between the WPPM and our ML-MotEx-based\u000d\u000amethod illustrates the crystal intergrowth observed can be\u000d\u000acompared to a stack of ultra-small nano-sized domains of each\u000d\u000aparent crystal structure, giving rise to the anisotropic line\u000d\u000abroadening in XRD patterns. Although the model did not\u000d\u000aaccount for all the effects impacting the XRD pattern of an\u000d\u000aintergrown crystal (peak asymmetry, microstrain), this\u000d\u000aapproach gives a good description of the nanostructure.\u000d\u000aFor the 3 samples that we characterised, the size of the \u03b2and R-MnO2 domains are ca. 1\u20132 nm, which corresponds also\u000d\u000ato the range in which the PDF data are refined. This observation can explain why only a few superstructures yielded a\u000d\u000abetter refinement than that done with the two separate parent\u000d\u000astructures. Because PDF probes short-range order in a\u000d\u000amaterial, intergrowth systems can be treated as a binary\u000d\u000amixture of the parent structures. While PDF analysis can thus\u000d\u000abe used to estimate the \u03b2-MnO2 and R-MnO2 fractions, XRD is\u000d\u000amore sensitive with respect to the way the two crystal structures are intergrown, i.e. the effect of Pb and Pr. However, in a\u000d\u000acase of an intergrowth structure that shows a high amount of\u000d\u000adisorder and where Bragg peaks can no longer be observed,\u000d\u000aPDF can still provide the missing information.\u000d\u000aConclusion\u000d\u000aWe have developed a method to characterise crystal intergrowth in \u03b3-MnO2 using a combination of XRD and PDF analysis which is potentially applicable to many other systems.\u000d\u000aOur samples were hydrothermally synthesised \u03b3-MnO2, where\u000d\u000adifferent synthesis times led to different defect densities. Our\u000d\u000aaim was to characterize the intergrowth density, and analyse\u000d\u000athe size of the domains present in the structure.\u000d\u000aWe first simulated XRD and PDF patterns from a large database of \u03b3-MnO2 structure models generated with different\u000d\u000aintergrowth densities. By considering trends in the simulated\u000d\u000apatterns, we developed a simple peak-fitting approach to estimate the defect density. This peak-fitting approach allowed us\u000d\u000ato obtain an estimate for the three parameters characterising\u000d\u000athe intergrowth, namely the \u03b2-MnO2 fraction, Pb and Pr.\u000d\u000aTo further characterise the materials, we used a structuremining approach. Again using our database of \u03b3-MnO2 strucDalton Transactions Paper\u000d\u000aThis journal is \u00a9 The Royal Society of Chemistry 2022 Dalton Trans., 2022, 51, 17150\u201317161 | 17159\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","ture models, we refined large subsets of these structures\u000d\u000aagainst our experimental data. Using the fit qualities obtained,\u000d\u000awe picked the best fitting structure for each sample and its\u000d\u000acorresponding intergrowth parameters. As the synthesis time\u000d\u000aof \u03b3-MnO2 is increased from 2 to 5 hours, the intergrowth\u000d\u000astructure reorders from a R-like to a \u03b2-like structure, with segregation of R- and \u03b2-MnO2 domains. Our structure-mining\u000d\u000aapproach also shows that structures with a fairly broad distribution of Pb and Pr lead to similar fit qualities. This suggests\u000d\u000athat it is in general difficult to assign one specific model to\u000d\u000adescribe the structure of such disordered samples.\u000d\u000aTo address this issue, and to analyse the nanostructure in\u000d\u000athe samples, we characterised the size of the intergrowth\u000d\u000adomains in the three samples. Here, we applied an explainable\u000d\u000amachine learning algorithm, ML-MotEx, to extract information\u000d\u000afrom the 2500 fits done for the structure-mining approach.\u000d\u000aML-MotEx extracts the size of domains of both parent structures that help to yield a better fit of the structure, thus comparing crystal intergrowth structures to a mere stack of ultrasmall nano-sized domains of two parent structures. The\u000d\u000aresults showed that while R-MnO2 domains keep a constant\u000d\u000asize, \u03b2-MnO2 domains tend to grow as synthesis is carried out\u000d\u000afor longer time. The study furthermore showed that XRD is\u000d\u000amore suitable than PDF to characterise crystal intergrowths.\u000d\u000aThe intergrowth affects especially the long-range order, which\u000d\u000ais better investigated with XRD.\u000d\u000aConflicts of interest\u000d\u000aThere are no conflicts to declare.\u000d\u000aAcknowledgements\u000d\u000aThis work is part of a project that has received funding from\u000d\u000athe European Research Council (ERC) under the European\u000d\u000aUnion\u2019s Horizon 2020 Research and Innovation Programme\u000d\u000a(grant agreement no. 804066). We are grateful to the Villum\u000d\u000aFoundation for financial support through a Villum Young\u000d\u000aInvestigator grant (VKR00015416). Funding from the Danish\u000d\u000aMinistry of Higher Education and Science through the SMART\u000d\u000aLighthouse is gratefully acknowledged. We acknowledge\u000d\u000aDiamond Light Source for time on Beamline I15-1 under\u000d\u000aProposal CY21791-4. We thank Phillip Chapter for collecting\u000d\u000adata during remote beamtime. TEM images were collected at\u000d\u000athe DTU Nanolab, the National Centre for Nano Fabrication\u000d\u000aand Characterization at the Technical University of Denmark.\u000d\u000aReferences\u000d\u000a1 L. I. Hill, A. Verbaere and D. Guyomard, J. Power Sources,\u000d\u000a2003, 119\u2013121, 226\u2013231.\u000d\u000a2 S. Devaraj and N. Munichandraiah, J. Phys. Chem. C, 2008,\u000d\u000a112, 4406\u20134417.\u000d\u000a3 K. Galliez, P. Deniard, D. Lambertin, S. Jobic and F. Bart,\u000d\u000aJ. Nucl. Mater., 2013, 438, 261\u2013267.\u000d\u000a4 D. M. Robinson, Y. B. Go, M. Mui, G. Gardner, Z. Zhang,\u000d\u000aD. Mastrogiovanni, E. Garfunkel, J. Li, M. Greenblatt and\u000d\u000aG. C. Dismukes, J. Am. Chem. Soc., 2013, 135, 3494\u20133501.\u000d\u000a5 Y. Meng, W. Song, H. Huang, Z. Ren, S. Y. Chen and\u000d\u000aS. L. Suib, J. Am. Chem. Soc., 2014, 136, 11452\u201311464.\u000d\u000a6 T. R. Juran, J. Young and M. Smeu, J. Phys. Chem. C, 2018,\u000d\u000a122, 8788\u20138795.\u000d\u000a7 F. Y. Cheng, J. Chen, X. L. Gou and P. W. Shen, Adv. Mater.,\u000d\u000a2005, 17, 2753\u20132756.\u000d\u000a8 M. Chamoun, W. R. Brant, C.-W. Tai, G. Karlsson and\u000d\u000aD. Nor\u00e9us, Energy Storage Mater., 2018, 15, 351\u2013360.\u000d\u000a9 I. Vasiliev, B. A. Magar, J. Duay, T. N. Lambert and\u000d\u000aB. Chalamala, J. Electrochem. Soc., 2018, 165, A3517\u2013A3524.\u000d\u000a10 M. M. Thackeray, M. H. Rossouw, A. de Kock, A. P. de la\u000d\u000aHarpe, R. J. Gummow, K. Pearce and D. C. Liles, J. Power\u000d\u000aSources, 1993, 43, 289\u2013300.\u000d\u000a11 C. Godart, M. Latroche, C. Freticny and C. Levy-Clement,\u000d\u000aPhys. Status Solidi A, 1992, 132, 253\u2013268.\u000d\u000a12 Y. Chabre and J. Pannetier, Prog. Solid State Chem., 1995,\u000d\u000a23, 1\u2013130.\u000d\u000a13 A. Le Gal La Salle, S. Sarciaux, A. Verbaere, Y. Piffard and\u000d\u000aD. Guyomard, J. Electrochem. Soc., 2000, 147, 945.\u000d\u000a14 L. I. Hill and A. Verbaere, J. Solid State Chem., 2004, 177,\u000d\u000a4706\u20134723.\u000d\u000a15 P. M. De Wolff, Acta Crystallogr., 1959, 12, 341\u2013345.\u000d\u000a16 C. Poinsignon, M. Amarilla and F. Tedjar, J. Mater. Chem.,\u000d\u000a1993, 3, 1227\u20131229.\u000d\u000a17 S. Jouanneau, S. Sarciaux, A. Le Gal La Salle and\u000d\u000aD. Guyomard, Solid State Ionics, 2001, 140, 223\u2013232.\u000d\u000a18 K. Galliez, P. Deniard, C. Payen, D. Lambertin, F. Bart,\u000d\u000aH.-J. Koo, M.-H. Whangbo and S. Jobic, Inorg. Chem., 2015,\u000d\u000a54, 1194\u20131196.\u000d\u000a19 D. Simon, R. Morton and J. Gislason, J. Adv. X-ray Anal.,\u000d\u000a2004, 47, 267\u2013280.\u000d\u000a20 K. Galliez, P. Deniard, P.-E. Petit, D. Lambertin, F. Bart and\u000d\u000aS. Jobic, J. Appl. Crystallogr., 2014, 47, 552\u2013560.\u000d\u000a21 M. M. J. Treacy, J. M. Newsam and M. W. Deem,\u000d\u000aProc. R. Soc. London, Ser. A, 1991, 433, 499\u2013520.\u000d\u000a22 M. Leoni, A. F. Gualtieri and N. Roveri, J. Appl. Crystallogr.,\u000d\u000a2004, 37, 166\u2013173.\u000d\u000a23 M. Casas-Cabanas, M. Reynaud, J. Rikarte, P. Horbach and\u000d\u000aJ. Rodr\u00edguez-Carvajal, J. Appl. Crystallogr., 2016, 49, 2259\u20132269.\u000d\u000a24 A. A. Coelho, J. S. O. Evans and J. W. Lewis, J. Appl.\u000d\u000aCrystallogr., 2016, 49, 1740\u20131749.\u000d\u000a25 C. M. Ainsworth, J. W. Lewis, C.-H. Wang, A. A. Coelho,\u000d\u000aH. E. Johnston, H. E. A. Brand and J. S. O. Evans, Chem.\u000d\u000aMater., 2016, 28, 3184\u20133195.\u000d\u000a26 W. A. S\u0142awi\u0144ski, D. S. Wragg, D. Akporiaye and H. Fjellv\u00e5g,\u000d\u000aMicroporous Mesoporous Mater., 2014, 195, 311\u2013318.\u000d\u000a27 S. Bette, T. Takayama, V. Duppel, A. Poulain, H. Takagi and\u000d\u000aR. E. Dinnebier, Dalton Trans., 2019, 48, 9250\u20139259.\u000d\u000a28 A. S. Masadeh, E. S. Bozin, C. L. Farrow, G. Paglia, P. Juhas,\u000d\u000aS. J. L. Billinge, A. Karkamkar and M. G. Kanatzidis, Phys.\u000d\u000aRev. B: Condens. Matter Mater. Phys., 2007, 76, 115413.\u000d\u000aPaper Dalton Transactions\u000d\u000a17160 | Dalton Trans., 2022, 51, 17150\u201317161 This journal is \u00a9 The Royal Society of Chemistry 2022\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","29 X. Yang, A. S. Masadeh, J. R. McBride, E. S. Bo\u017ein,\u000d\u000aS. J. Rosenthal and S. J. L. Billinge, Phys. Chem. Chem.\u000d\u000aPhys., 2013, 15, 8480\u20138486.\u000d\u000a30 A. Kudielka, S. Bette, R. E. Dinnebier, M. Abeykoon,\u000d\u000aC. Pietzonka and B. Harbrecht, J. Mater. Chem. C, 2017, 5,\u000d\u000a2899\u20132909.\u000d\u000a31 S. Bette, T. Takayama, K. Kitagawa, R. Takano, H. Takagi\u000d\u000aand R. E. Dinnebier, Dalton Trans., 2017, 46, 15216\u201315227.\u000d\u000a32 A. M. P\u00fctz, M. W. Terban, S. Bette, F. Haase,\u000d\u000aR. E. Dinnebier and B. V. Lotsch, Chem. Sci., 2020, 11,\u000d\u000a12647\u201312654.\u000d\u000a33 S. Mangelsen, B. R. Srinivasan, U. Sch\u00fcrmann, L. Kienle,\u000d\u000aC. N\u00e4ther and W. Bensch, Dalton Trans., 2019, 48, 1184\u20131201.\u000d\u000a34 X. Wang and Y. Li, J. Am. Chem. Soc., 2002, 124, 2880\u20132881.\u000d\u000a35 J. Filik, A. Ashton, P. Chang, P. Chater, S. Day,\u000d\u000aM. Drakopoulos, M. Gerring, M. Hart, O. Magdysyuk and\u000d\u000aS. Michalik, J. Appl. Crystallogr., 2017, 50, 959\u2013966.\u000d\u000a36 X. Yang, P. Juhas, C. L. Farrow and S. J. L. Billinge, 2014,\u000d\u000aeprint, DOI: 10.48550/arXiv.1402.3163.\u000d\u000a37 M. D. Abr\u00e0moff, P. J. Magalh\u00e3es and S. J. Ram,\u000d\u000aJ. Biophotonics Int., 2004, 11, 36\u201342.\u000d\u000a38 K. Pussi, J. Gallo, K. Ohara, E. Carbo-Argibay,\u000d\u000aY. V. Kolen\u2019ko, B. Barbiellini, A. Bansil and S. Kamali,\u000d\u000aCondens. Matter, 2020, 5, 19.\u000d\u000a39 K. S. Abou-El-Sherbini, M. Askar and R. Sch\u00f6llhorn, Solid\u000d\u000aState Ionics, 2002, 150, 407\u2013415.\u000d\u000a40 F. Cheng, J. Zhao, W. Song, C. Li, H. Ma, J. Chen and\u000d\u000aP. Shen, Inorg. Chem., 2006, 45, 2038\u20132044.\u000d\u000a41 C. Wang, Y. Zeng, X. Xiao, S. Wu, G. Zhong, K. Xu, Z. Wei,\u000d\u000aW. Su and X. Lu, J. Energy Chem., 2020, 43, 182\u2013187.\u000d\u000a42 X. Zhang, P. Yu, D. Zhang, H. Zhang, X. Sun and Y. Ma,\u000d\u000aMater. Lett., 2013, 92, 401\u2013404.\u000d\u000a43 D. Su, H.-J. Ahn and G. Wang, J. Mater. Chem. A, 2013, 1,\u000d\u000a4845\u20134850.\u000d\u000a44 A. Coelho, J. Appl. Crystallogr., 2018, 51, 210\u2013218.\u000d\u000a45 G. Caglioti, A. Paoletti and F. P. Ricci, Nucl. Instrum.\u000d\u000aMethods, 1958, 3, 223\u2013228.\u000d\u000a46 O. Schilling and J. Dahn, J. Appl. Crystallogr., 1998, 31, 396\u2013\u000d\u000a406.\u000d\u000a47 J.-R. Hill, C. M. Freeman and M. H. Rossouw, J. Solid State\u000d\u000aChem., 2004, 177, 165\u2013175.\u000d\u000a48 D. Balachandran, D. Morgan, G. Ceder and A. van de Walle,\u000d\u000aJ. Solid State Chem., 2003, 173, 462\u2013475.\u000d\u000a49 P. Ruetschi, J. Electrochem. Soc., 1984, 131, 2737.\u000d\u000a50 T. X. Sayle, C. R. A. Catlow, R. R. Maphanga,\u000d\u000aP. E. Ngoepe and D. C. Sayle, J. Cryst. Growth, 2006, 294,\u000d\u000a118\u2013129.\u000d\u000a51 A. S. Anker, M. Juelsholt, S. L. Skj\u00e6rv\u00f8, I. Kantor,\u000d\u000aD. R. S\u00f8rensen, S. J. L. Billinge and R. Selvan, ChemRxiv,\u000d\u000a2022, DOI: 10.26434/chemrxiv-22022-v26431vfx.\u000d\u000a52 T. Chen and C. Guestrin, Presented in part at the\u000d\u000aProceedings of the 22nd ACM SIGKDD International\u000d\u000aConference on Knowledge Discovery and Data Mining,\u000d\u000aSan Francisco, California, USA, 2016.\u000d\u000a53 S. M. Lundberg, G. Erion, H. Chen, A. DeGrave,\u000d\u000aJ. M. Prutkin, B. Nair, R. Katz, J. Himmelfarb, N. Bansal\u000d\u000aand S.-I. Lee, Nat. Mach. Intell., 2020, 2, 56\u201367.\u000d\u000a54 S. M. Lundberg and S.-I. Lee, Proceedings of the 31st\u000d\u000aInternational Conference on Neural Information\u000d\u000aProcessing Systems, 2017, pp. 4765\u20134774.\u000d\u000a55 M. Leoni, T. Confente and P. Scardi, Z. Kristallogr., 2006,\u000d\u000a249\u2013254.\u000d\u000a56 W. Sun, D. A. Kitchaev, D. Kramer and G. Ceder, Nat.\u000d\u000aCommun., 2019, 10, 573.\u000d\u000aDalton Transactions Paper\u000d\u000aThis journal is \u00a9 The Royal Society of Chemistry 2022 Dalton Trans., 2022, 51, 17150\u201317161 | 17161\u000d\u000aOpen Access Article. Published on 20 September 2022. Downloaded on 5/19/2023 3:59:09 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","235\u000d\u000aTransition\u0009metal\u0009tungstate\u0009paper\u000d\u000aOn\u0009the\u0009following\u0009pages,\u0009you\u0009will\u0009find\u0009the\u0009Transition\u0009metal\u0009tungstate paper which\u0009is\u0009published\u0009in\u0009Chemical\u0009\u000d\u000aScience.\u0009","Atomic structural changes in the formation of\u000d\u000atransition metal tungstates: the role of\u000d\u000apolyoxometalate structures in material\u000d\u000acrystallization\u2020\u000d\u000aSusanne Linn Skj\u00e6rv\u00f8, Andy S. Anker, Magnus C. Wied, Emil T. S. Kj\u00e6r,\u000d\u000aMikkel Juelsholt, Troels Lindahl Christiansen and Kirsten M. \u00d8. Jensen *\u000d\u000aMaterial nucleation processes are poorly understood; nevertheless, an atomistic understanding of material\u000d\u000aformation would aid in the design of material synthesis methods. Here, we apply in situ X-ray total\u000d\u000ascattering experiments with pair distribution function (PDF) analysis to study the hydrothermal synthesis of\u000d\u000awolframite-type MWO4 (M : Mn, Fe, Co, Ni). The data obtained allow the mapping of the material formation\u000d\u000apathway in detail. We first show that upon mixing of the aqueous precursors, a crystalline precursor\u000d\u000acontaining [W8O27]\u000d\u000a6- clusters forms for the MnWO4 synthesis, while amorphous pastes form for the\u000d\u000aFeWO4, CoWO4 and NiWO4 syntheses. The structure of the amorphous precursors was studied in detail\u000d\u000awith PDF analysis. Using database structure mining and an automated modelling strategy by applying\u000d\u000amachine learning, we show that the amorphous precursor structure can be described through\u000d\u000apolyoxometalate chemistry. A skewed sandwich cluster containing Keggin fragments describes the PDF of\u000d\u000athe precursor structure well, and the analysis shows that the precursor for FeWO4 is more ordered than\u000d\u000athat of CoWO4 and NiWO4. Upon heating, the crystalline MnWO4 precursor quickly converts directly to\u000d\u000acrystalline MnWO4, while the amorphous precursors transform into a disordered intermediate phase before\u000d\u000athe crystalline tungstates appear. Our data show that the more disordered the precursor is, the longer the\u000d\u000areaction time required to form crystalline products, and disorder in the precursor phase appears to be\u000d\u000aa barrier for crystallization. More generally, we see that polyoxometalate chemistry is useful when\u000d\u000adescribing the initial wet-chemical formation of mixed metal oxides.\u000d\u000aIntroduction\u000d\u000aFormation pathways during synthesis can have a huge impact on\u000d\u000aa material's structure and properties, and yet, our knowledge\u000d\u000aregarding the mechanisms involved in material formation during\u000d\u000awet-chemical synthesis is poor. Traditionally, nucleation has\u000d\u000abeen described in terms of classical nucleation theory, which\u000d\u000auses simple thermodynamic functions to describe the formation\u000d\u000aof the nuclei that lead to particles. Classical nuclei are considered\u000d\u000ato be hard spheres with a sharp interface between the solution\u000d\u000aand solid, with the same atomic structure and properties of the\u000d\u000a\ue103nal crystalline material. This is highly approximate, and classical nucleation theory does not re\ue104ect the complex chemical\u000d\u000areactions that take place during synthesis.1 More recently, \u2018nonclassical nucleation\u2019 theories have therefore been introduced,\u000d\u000aincluding the existence of \u2018pre-nucleation clusters\u2019, which are\u000d\u000aconsidered to be assemblies of atoms existing in the early stages\u000d\u000aof the synthesis.2 These species may meet in solution and\u000d\u000aaggregate, and crystallization within the aggregate can lead to the\u000d\u000aformation of nuclei from which nanoparticles can grow.3,4 While\u000d\u000athe development of non-classical nucleation theory has widened\u000d\u000aour understanding of nucleation processes, many aspects of\u000d\u000anucleation are still le\ue09d in the dark. Speci\ue103cally, we are missing\u000d\u000aatomistic structural information on the clusters and nuclei as\u000d\u000athey form and grow into particles. For example, the clusters\u000d\u000ainvolved in the nucleation process are generally described as\u000d\u000aamorphous assemblies of atoms, even though they are key to\u000d\u000aunderstanding material formation.\u000d\u000aIn order to unravel the cluster chemistry at play in nucleation\u000d\u000aprocesses, we here investigate the nucleation of mixed metal\u000d\u000aoxides such as wolframite-type tungstates (Fig. 1a). Wolframitetype MWO4 are complex tungsten(+VI)-based oxides containing\u000d\u000abivalent 3d-metals such as Mn, Fe, Co, Ni, Cu and Zn.5 These\u000d\u000amaterials have interesting electrochemical properties, which are\u000d\u000auseful in electro- and photocatalysis,5 and changes in their\u000d\u000acatalytic behaviour have been observed as the structure goes from\u000d\u000acrystalline to amorphous.6 Hydro- and solvothermal syntheses of\u000d\u000aDepartment of Chemistry and Nano-Science Center, University of Copenhagen, 2100\u000d\u000aCopenhagen \u00d8, Denmark. E-mail: kirsten@chem.ku.dk\u000d\u000a\u2020 Electronic supplementary information (ESI) available. See DOI:\u000d\u000ahttps://doi.org/10.1039/d3sc00426k\u000d\u000aCite this: Chem. Sci., 2023, 14, 4806\u000d\u000aAll publication charges for this article\u000d\u000ahave been paid for by the Royal Society\u000d\u000aof Chemistry\u000d\u000aReceived 25th January 2023\u000d\u000aAccepted 5th April 2023\u000d\u000aDOI: 10.1039/d3sc00426k\u000d\u000arsc.li/chemical-science\u000d\u000a4806 | Chem. Sci., 2023, 14, 4806\u20134816 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aChemical\u000d\u000aScience\u000d\u000aEDGE ARTICLE\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online\u000d\u000aView Journal | View Issue","wolframite-type tungstates have been studied extensively in the\u000d\u000aliterature,5,7\u201310 and usually involves simple mixing of metal cation\u000d\u000aaqueous solutions, with optional pH adjustment followed by\u000d\u000ahydro- or solvothermal treatment. The formation mechanism of\u000d\u000aMWO4 has not yet been given much attention, but recent insights\u000d\u000aon the hydrothermal formation of WO3 by Juelsholt et al.11 and\u000d\u000aZnWO4 by B\u00f8jesen et al.12 have shown that clusters with structures similar to those well-known from classical polyoxometalate\u000d\u000a(POM) chemistry13\u201319 might play an important role.12 From in situ\u000d\u000aX-ray total scattering studies, it was shown that ZnWO4 forms via\u000d\u000aa sandwich-type POM ion. This sandwich ion, with the general\u000d\u000aformula [M4(H2O)2(XW9O34)2]\u000d\u000an\u2212 with metal M and heteroatom X\u000d\u000aas described by Limanski et al.,\u000d\u000a20 can be seen in Fig. 1b. The\u000d\u000asandwich structure is simply described as two Keggin fragments\u000d\u000a(Fig. 1c) sandwiching a bridging layer of 3d metal atoms.21\u000d\u000aAnalogous ions have been found for samples containing M =\u000d\u000aMn2+, Fe2+, Fe3+ or Cu2+,\u000d\u000a20,22,23 and for a mix of several different\u000d\u000ametal cations in the mineral ophirite.23\u000d\u000aFig. 1 (a) Visualization of the wolframite-type MWO4 structure. The 3d-metal atoms (M) are orange, tungsten atoms are grey and oxygen atoms\u000d\u000aare red. The structure consists of chains of edge-sharing metal\u2013oxygen octahedra, and the chains are linked with each other by corner-sharing.\u000d\u000a(b) Illustration of a sandwich-type cluster, consisting of two Keggin fragments and a 3d metal (M) bridging layer. Each Keggin fragment is missing\u000d\u000aa triad (i.e. a group of three edge-sharing [WO6]\u000d\u000a10\u2212 octahedra) which is replaced by the bridging layer, and is cradling a heteroatom (X), which in\u000d\u000asome cases is the same element as in the bridging layer. The bridging layer connects to the fragments with a rotational angle typical for (c) bKeggins, 60\u00b0 compared to the a-Keggin. Structures are drawn with VESTA.24\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Chem. Sci., 2023, 14, 4806\u20134816 | 4807\u000d\u000aEdge Article Chemical Science\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","To shed light on the formation process of complex oxides and\u000d\u000athe role of POM clusters, we present an in situ time-resolved X-ray\u000d\u000atotal scattering study of MWO4 formation for M = Mn, Fe, Co and\u000d\u000aNi. We analyse the total scattering data through pair distribution\u000d\u000afunction (PDF) analysis, which allows for extraction of structural\u000d\u000ainformation from all stages of the synthesis, i.e. from ions in\u000d\u000asolution, amorphous intermediates and the crystalline\u000d\u000aproducts.1,11,25\u201331 Our data show that POM-like structures play\u000d\u000aa role in crystallization in all the tungstates we have investigated,\u000d\u000aalthough their structure and the process differ depending on the\u000d\u000atransition metal in the structure. None of the compounds studied\u000d\u000ahere were formed from the Tourn\u00b4e sandwich cluster previously\u000d\u000aobserved during the formation of ZnWO4.\u000d\u000a12 Thus, to identify new,\u000d\u000arelevant cluster structures from PDF, we have applied a new\u000d\u000amodelling strategy involving machine learning (ML). We use our\u000d\u000anewly developed ML method, ML-MotEx,32 which allows us to\u000d\u000aextract important structural motifs through SHAP (SHapley\u000d\u000aAdditive exPlanation) values.33,34 This method allowed mapping\u000d\u000aof structural changes in the entire reaction process, following\u000d\u000ahow different structural motifs appear and disappear in the\u000d\u000amaterial as they form. We show that the formation of MnWO4\u000d\u000aproceeds directly from a crystalline precursor structure, while\u000d\u000athat of FeWO4, CoWO4 and NiWO4 proceeds from an amorphous\u000d\u000aprecursor over an intermediate. The amorphous precursor\u000d\u000astructure can be described through a skewed sandwich cluster\u000d\u000awith Keggin-like fragments, previously seen in crystalline\u000d\u000a(Ni6(H2O)9(OH)3(HSiW9O34))2(H2O)12.\u000d\u000aThe precursor clusters observed in the case of CoWO4 and\u000d\u000aNiWO4 are more disordered than that for FeWO4. This disorder\u000d\u000aappears to affect the reaction process and kinetics, as we only\u000d\u000aobserve the formation of crystalline FeWO4 in the time scale of\u000d\u000athe in situ experiments, while CoWO4 and NiWO4 retain their\u000d\u000aintermediate structure. The experiments and PDF analysis thus\u000d\u000aprovide an understanding of how mixed metal oxides form in\u000d\u000asolution, and the role polyoxometalates play in this regard.\u000d\u000aResults and discussion\u000d\u000aOverview of the phase evolution\u000d\u000aWe performed in situ X-ray total scattering with a time-resolution\u000d\u000aof 5 seconds/frame of the formation of four different tungstates,\u000d\u000aMnWO4, FeWO4, CoWO4 and NiWO4. In all cases, the synthesis\u000d\u000awas done hydrothermally. As described in detail in the Experimental Methods section, an aqueous solution of Na2WO4$2H2O\u000d\u000awas mixed with an aqueous solution of (1) MnCl2$4H2O (MnWO4\u000d\u000asynthesis), (2) FeCl2$4H2O (FeWO4 synthesis), (3) CoCl2$6H2O\u000d\u000a(CoWO4 synthesis) and (4) NiCl2$6H2O (NiWO4 synthesis). All the\u000d\u000amixtures became paste-like (i.e. opaque with high viscosity)\u000d\u000aimmediately upon mixing, but with lower viscosity when moving\u000d\u000aright along the 4th period in the periodic table, from Mn to Ni.\u000d\u000aThe pastes had different colours; beige, medium brown, bright\u000d\u000apurple and pale green for the MnWO4, FeWO4, CoWO4 and\u000d\u000aNiWO4 synthesis, respectively, as can be seen from Fig. S1 in the\u000d\u000aESI.\u2020 The hydrothermal syntheses were performed at 160 \u00b0C and\u000d\u000aa static pressure of 100 bar.\u000d\u000aWe \ue103rst consider the reaction leading to FeWO4. Fig. 2a\u000d\u000apresents the time-resolved PDFs calculated from in situ total\u000d\u000ascattering data for the FeWO4 synthesis. We observe three\u000d\u000adifferent phases in the process: a precursor, an intermediate\u000d\u000aphase and a product. The precursor phase has only short range\u000d\u000aorder, with signi\ue103cant PDF peaks up to r z 5 \u00c5 and weak\u000d\u000aoscillations up to r z 10 \u00c5. These oscillations indicate that the\u000d\u000aprecursor is amorphous and most likely does not consist of\u000d\u000aFig. 2 (a) Time-resolved PDFs calculated from in situ total scattering\u000d\u000adata showing the evolution from the FeWO4 precursor structure, via an\u000d\u000aintermediate phase to a product at 160 \u00b0C and 100 bar. Dashed lines\u000d\u000ahave been added to highlight the region of the intermediate phase. (b)\u000d\u000aPDFs and (c) F(Q) from in situ total scattering data of intermediate and\u000d\u000aproduct phases present during the MWO4 (M = Mn, Fe, Co, Ni)\u000d\u000asyntheses at 160 \u00b0C and 100 bar. The grey areas highlight crucial\u000d\u000afeatures which are discussed in the text.\u000d\u000a4808 | Chem. Sci., 2023, 14, 4806\u20134816 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aChemical Science Edge Article\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","isolated clusters in solution, as also expected from the pasty\u000d\u000aappearance of the mixtures. The intensity of the strongest peaks\u000d\u000aof the precursor phase (around 3\u20134 \u00c5) decreases upon heating,\u000d\u000aand the small peaks above 5 \u00c5 seem to shi\ue09d slightly upon\u000d\u000aapplying heat. The precursor phase remains for approx. 1 min\u000d\u000aof heating, at which point the transient intermediate phase with\u000d\u000alimited long-range order appears, persisting for ca. 10 s. The\u000d\u000aintermediate phase transforms into a \ue103nal crystalline product\u000d\u000aphase. Contour plots of the data obtained for the MnWO4,\u000d\u000aCoWO4 and NiWO4 syntheses are shown in Fig. S2 in the ESI.\u2020\u000d\u000aFig. 2b and c show PDFs and reduced structure functions\u000d\u000aF(Q) for all phases observed. The same data with wider r and Q\u000d\u000aranges plotted are included in Fig. S3 in the ESI.\u2020 The MnWO4\u000d\u000aand FeWO4 reactions produce very similar phases with the wellknown tungstate structure as discussed further below. However,\u000d\u000awe do not observe crystalline product phases for the CoWO4 and\u000d\u000aNiWO4 experiments in the reaction time and under the conditions of our in situ experiment. Instead, the phases observed at\u000d\u000athe end of the experiment have a structure similar to the FeWO4\u000d\u000aintermediate phase. The similarity between these three intermediate phases is highlighted in Fig. S4 in the ESI,\u2020 where ex\u000d\u000asitu data of this phase are also shown to con\ue103rm stability under\u000d\u000aambient conditions. Note that the samples described in Fig. S4\u2020\u000d\u000aare hydrothermally synthesized in a stainless-steel autoclave\u000d\u000a(described in detail in the Experimental methods section), i.e. at\u000d\u000aa temperature and pressure very different from that of the in situ\u000d\u000astudy. The reaction may thus proceed differently; however, the\u000d\u000astructure of the ex situ samples and the intermediate seen in the\u000d\u000ain situ study appear similar. The literature suggests that a higher\u000d\u000areaction temperature or subsequent annealing is needed to\u000d\u000aform crystalline product phases for the CoWO4 and NiWO4\u000d\u000areactions.35\u201338\u000d\u000aOverview of local structural changes\u000d\u000aIn Fig. 2b, we have highlighted the range 3\u20134 \u00c5 of the PDFs\u000d\u000a(labelled \u201c#1\u201d). In this region, we \ue103nd peaks originating from\u000d\u000anearest-neighbour metal\u2013metal polyhedra, and speci\ue103cally, the\u000d\u000atwo peaks at r z 3.3 \u00c5 and r z 3.7 \u00c5 originate from metal\u2013metal\u000d\u000adistances between edge-sharing (ES) and corner-sharing (CS)\u000d\u000ametal\u2013oxygen octahedra, respectively. As highlighted in Fig. 3a\u2013\u000d\u000ad, the data show a signi\ue103cantly higher degree of edge-sharing in\u000d\u000athe precursor phases compared to in the crystalline product\u000d\u000aphases. To study this transition from precursor to product phases, we \ue103rst analysed these two peaks with a simple, model-free\u000d\u000aapproach by \ue103tting the peaks with Gaussian functions, as\u000d\u000adescribed in more detail in the Experimental methods section. In\u000d\u000aFig. 3e, we plot the calculated ratio between the intensity of the\u000d\u000aedge- and corner-sharing peaks (ES/CS) along the reaction for\u000d\u000aeach synthesis. When heating starts, the ES/CS intensity ratio\u000d\u000adecreases for all the reactions a\ue09der a delay. The MnWO4 reaction\u000d\u000ahas the lowest ES/CS intensity ratio decrease, and the ratio stabilises at the same time as the crystalline product phase forms.\u000d\u000aThe ES/CS intensity ratios for the FeWO4 and CoWO4 precursor\u000d\u000aphases are similar. A\ue09der a slightly slower onset for the FeWO4\u000d\u000areaction compared to for the CoWO4 reaction, they have the same\u000d\u000aEC/CS intensity ratio at 1.1 min, where they form the same\u000d\u000aintermediate phase. While CoWO4 remains at this EC/CS intensity ratio, FeWO4 proceeds to form a crystalline product phase,\u000d\u000awith a slightly higher ES/CS intensity ratio compared to that of\u000d\u000athe intermediate phase. The NiWO4 reaction, which just like\u000d\u000aCoWO4 did not produce a crystalline product phase, ends up at\u000d\u000athe same ES/CS intensity ratio as the CoWO4 reaction. In\u000d\u000aconclusion, the lowest ES/CS intensity ratio throughout the\u000d\u000areactions can be observed when transitioning through the\u000d\u000aintermediate phase, i.e. the intermediate phase has the lowest\u000d\u000adegree of edge-sharing (or highest degree of corner-sharing).\u000d\u000aThe structure of the FeWO4 and MnWO4 product phases\u000d\u000aHaving established the phase evolution, we now look further to\u000d\u000athe product phases. Despite having different reaction schemes,\u000d\u000athe MnWO4 and FeWO4 syntheses both produced a similar\u000d\u000aproduct phase, which could be successfully \ue103tted (Fig. 4a and\u000d\u000ab and S5\u2020) to the wolframite-type MnWO4 and FeWO4 structures.39,40 Sequential \ue103ts of the time-resolved data were performed\u000d\u000ato follow structural changes taking place a\ue09der the formation of\u000d\u000athe product phase. The scale factor, crystallite size, isotropic\u000d\u000aatomic displacement parameters (ADPs) for W and Mn/Fe, and\u000d\u000aFig. 3 Closer view of the peaks in the PDFs indicating edge- (ca. 3.3 \u00c5)\u000d\u000aand corner-sharing (ca. 3.6\u20133.7 \u00c5) for the (a) MnWO4, (b) FeWO4, (c)\u000d\u000aCoWO4 and (d) NiWO4 syntheses. (e) The ratio between the integrated\u000d\u000aintensities of these peaks with heating time. The vertical lines show the\u000d\u000atime when the MnWO4 and FeWO4 products are formed. The grey area\u000d\u000ashows the ES/CS range where the intermediate phase is observed.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Chem. Sci., 2023, 14, 4806\u20134816 | 4809\u000d\u000aEdge Article Chemical Science\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","the atomic position deviation from the centre of the W\u2013O octahedra along the y-axis for W were re\ue103ned. The results are presented in Fig. 4c and S6.\u2020 Generally, the model provides a good\u000d\u000adescription of the data as seen from the Rw factor (Fig. S6\u2020) which\u000d\u000ais below 0.2 a\ue09der the formation of the (Mn/Fe)WO4 phase and\u000d\u000adisappearance of intermediate/precursor phases. The scale\u000d\u000afactors show that the onset of the formation is later for FeWO4\u000d\u000acompared to MnWO4. While the scale factor of MnWO4 increases\u000d\u000aabruptly when transitioning directly from the crystalline\u000d\u000aprecursor phase to MnWO4, the scale factor of FeWO4 increases\u000d\u000aslightly slower as the product phase grows from the intermediate\u000d\u000aphase. The crystallite size of both MnWO4 and FeWO4 continues\u000d\u000ato increase a\ue09der the scale factor has reached its maximum for\u000d\u000aboth reactions, indicating that the crystallites may continue to\u000d\u000agrow through Ostwald ripening as the reaction proceeds. Note\u000d\u000athat the crystallite size of especially the MnWO4 particles grows\u000d\u000abeyond what can be resolved from the rapid acquisition PDF (RAPDF)41 measurements, and we therefore only rely on trends rather\u000d\u000athan absolute sizes.\u000d\u000aThe isotropic ADPs of Mn/Fe and W decrease during crystallization, indicating that the structural order is increasing. This\u000d\u000atrend is more apparent for FeWO4 compared to MnWO4, which\u000d\u000aindicates that a less ordered product will initially form when it\u000d\u000agrows from a less crystalline phase, i.e. the FeWO4 intermediate\u000d\u000ain contrast to the crystalline MnWO4 precursor phase. The same\u000d\u000aconclusion can also be drawn from the deviation of the W atomic\u000d\u000aposition from the octahedral center (DWy), where a larger deviation is seen in the re\ue103ned crystal structure from the early stages of\u000d\u000acrystallization. The \ue103nal deviation of the W atom from the octahedral center re\ue103nes to be slightly larger for MnWO4 than for\u000d\u000aFeWO4. However, the difference observed here may be within the\u000d\u000auncertainties of the re\ue103nements, and we cannot relate this\u000d\u000adifference to the formation mechanism.\u000d\u000aMWO4 (M = Fe, Co, Ni) intermediate: relation to the product\u000d\u000aphase\u000d\u000aThe literature suggests that the intermediate phase appearing\u000d\u000aduring the MWO4 reaction is a nanostructured or \u2018amorphous\u2019\u000d\u000awolframite-type MWO4,\u000d\u000a35,38,42\u201344 i.e. having a similar structure to\u000d\u000athe product phase discussed above. When comparing their PDFs\u000d\u000a(Fig. 2b) we see a difference in the ratio of the PDF peak intensity\u000d\u000afrom edge- and corner-sharing metal\u2013oxygen octahedra, and the\u000d\u000asame two peaks separate more in the product compared to in the\u000d\u000aintermediates. Also, the PDF peak at r z 4.6 \u00c5 is splitting up\u000d\u000awhen transitioning from the intermediate to the product, highlighted in grey area #2. Still, the intermediate and product phase\u000d\u000aPDFs have some similarities, and so we hypothesize that the\u000d\u000asame building blocks, i.e. the local structure, might be present in\u000d\u000athe two phases. To test this hypothesis, we \ue103tted the wolframitetype CoWO4 (ref. 45) structure to the CoWO4 intermediate phase\u000d\u000aup to just 20 \u00c5, and the result is presented in Fig. 5. The atomic\u000d\u000apositions of Co and W are re\ue103ned in the y-direction, and this\u000d\u000are\ue103nement results in a centering of metal atoms inside the\u000d\u000ametal\u2013oxygen octahedra, removing the splitting of the peak at 4.6\u000d\u000a\u00c5, highlighted in the grey area in Fig. 5. The local structure of the\u000d\u000aintermediate phase can thus be described fairly well with\u000d\u000aa disordered wolframite-type structure.\u000d\u000aThe fact that the local structure of the intermediate phase can\u000d\u000abe described by a disordered wolframite-type MWO4 structure\u000d\u000aleads to the assumption that the similarly positioned peaks in the\u000d\u000aF(Q) data of the intermediate and product phases could originate\u000d\u000afrom the same lattice planes in the MWO4 structure. By assuming\u000d\u000athis, the signi\ue103cantly narrow Bragg peak seen from the intermediate phases at 2.5 \u00c5\u22121 (Fig. 2 and S4\u2020) could originate from\u000d\u000athe (002) or (021) re\ue104ections, as suggested by both He et al.35 and\u000d\u000aWang et al.38 This points to the intermediate phase crystallites\u000d\u000abeing anisotropic, with signi\ue103cant growth along the b- or cFig. 4 Fits to the (a) MnWO4 and (b) FeWO4 product phases, using\u000d\u000awolframite-MWO4 structures from the ICSD (code 67 906 (ref. 42) and\u000d\u000a26 843,43 respectively). Details of the fits are found in tables of Fig. S6 in\u000d\u000athe ESI.\u2020 (c) Results from reverse sequential real space Rietveld\u000d\u000arefinements of MnWO4 and FeWO4 structures to the PDFs in the range\u000d\u000aof 1.3\u201360 \u00c5. Scale factor, crystallite size (cr. size) in nm and isotropic\u000d\u000aADPs (uiso in \u00c52\u000d\u000a) for W and Mn/Fe are shown in addition to DWy, which\u000d\u000ais the atomic position deviation of W from the octahedral center\u000d\u000a(defined as 0.13), along the y-axis. The time range where the FeWO4\u000d\u000aintermediate phase is present is highlighted with a grey colour.\u000d\u000a4810 | Chem. Sci., 2023, 14, 4806\u20134816 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aChemical Science Edge Article\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","direction. By connecting this observation to the low degree of\u000d\u000aedge-sharing calculated for the intermediate phase (Fig. 3) we\u000d\u000asuggest that the anisotropic growth occurs mainly in the bdirection, as this direction is where chains of equal metal type\u000d\u000aconnect through corner-sharing.\u000d\u000aAnalysis of precursor structure motifs using structure mining\u000d\u000aWe now turn to analysis of the precursor structure before\u000d\u000aestablishing the reaction mechanism. The PDFs from the four\u000d\u000aprecursor compounds are plotted in Fig. 6. In the \ue103gure, we also\u000d\u000acompare the PDFs from the four mixed precursor pastes with\u000d\u000athose of their respective unmixed aqueous solutions (MCl2-\u000d\u000a$xH2O, M = Mn, Fe, Co, Ni and Na2WO4$2H2O).\u000d\u000aThe PDFs of the precursor pastes for the FeWO4, CoWO4 and\u000d\u000aNiWO4 syntheses are almost identical. As marked in the \ue103gure,\u000d\u000athe PDFs have signi\ue103cant peaks at r z 1.8 \u00c5 and r z 2.2 \u00c5, which\u000d\u000aare expected to arise from metal\u2013oxygen distances in [MOx] units.\u000d\u000aAdditionally, nearest-neighbour metal\u2013metal distances between\u000d\u000aedge- and corner-sharing [MOx] octahedra are indicated by the\u000d\u000apeaks at r z 3.3 \u00c5 and r z 3.6 \u00c5, as discussed above. The PDFs of\u000d\u000athe individual precursors dissolved in water show no such signs\u000d\u000aof signi\ue103cant edge- and corner-sharing W/M\u2013O octahedra,\u000d\u000ashowing that mixing of the aqueous precursors in water leads to\u000d\u000athe formation of new precursor phases.\u000d\u000aThe MnWO4 precursor is crystalline. The F(Q) for the\u000d\u000aMnWO4 precursor shows clear Bragg peaks and the PDF shows\u000d\u000athat the structural range goes beyond 40 \u00c5 (Fig. 2). No structure\u000d\u000ain the COD or ICSD databases satisfyingly described the entire\u000d\u000astructure, but the local range (1\u20138 \u00c5) was successfully described\u000d\u000aby the structure Na4Mn[Mo8O27]$20H2O46 where Mo was\u000d\u000areplaced with W (see the \ue103t and structure in Fig. S7\u2020). This\u000d\u000astructure can be described as [W8O27]\u000d\u000a6\u2212, units linked together\u000d\u000ain chains by Mn2+, Na+ and H2O. Each [W8O27]\u000d\u000a6\u2212 unit is approx.\u000d\u000a8 \u00c5 in size, which corresponds to the successful \ue103tting range.\u000d\u000aTherefore, it is likely that the [W8O27]\u000d\u000a6\u2212 unit is the dominating\u000d\u000amotif of the structure, and that the long-range deviation from\u000d\u000athe model arises from a different linking of these units.\u000d\u000aThe FeWO4, CoWO4 and NiWO4 precursors are all amorphous. In Fig. 6a, we compare their PDFs with calculated PDFs\u000d\u000afrom known clusters, which could be expected to form the zinc\u000d\u000atungstate sandwich cluster observed in the synthesis of ZnWO4\u000d\u000a(ref. 12) and the equivalent iron tungstate sandwich cluster reported by Barats et al.22 (Fig. 1). When comparing these PDFs, it is\u000d\u000aclear that the structures of the FeWO4, CoWO4 and NiWO4\u000d\u000aprecursors observed in this study have a local structure similar to\u000d\u000athe sandwich clusters, however, with less order. A \ue103t with this\u000d\u000astructure shows signi\ue103cant features in the difference curve, as\u000d\u000aseen in Fig. S8 in the ESI.\u2020 We therefore also wanted to investigate if other, well-known cluster structures would \ue103t the PDF.\u000d\u000aMany relevant compounds and structures are reported in the\u000d\u000aliterature and databases. All chemically relevant structures (i.e.\u000d\u000aFig. 5 Fit of the wolframite-CoWO4 structure41 to the CoWO4 intermediate phase (a) without and (b) with refined Co and W atomic\u000d\u000apositions. Scale factor unit cell parameters, isotropic ADPs, d2 and the\u000d\u000aatomic positions of Co and W along the y-direction were refined. The\u000d\u000arefinement results are given in Table S1 in the ESI.\u2020\u000d\u000aFig. 6 (a) PDFs of the precursors forming on mixing each of the metal\u000d\u000achloride precursors with Na2WO4 in water, compared to that of the\u000d\u000aindividual precursors in water. The PDFs of the Zn\u2013W and Fe\u2013W\u000d\u000asandwich-type clusters8,20 described by B\u00f8jesen et al.12 and Barats\u000d\u000aet al.22 are plotted for comparison. (b) The ten best suggestions for\u000d\u000adescribing the FeWO4 precursor mixture, found by sifting through\u000d\u000adatabases. Pearson correlation fitting was performed between calculated PDFs of all relevant clusters extracted from structures in the COD\u000d\u000a+ ICSD databases and the obtained PDF in the range of 1\u20135 \u00c5.\u000d\u000aCorrelation coefficients are placed to the left of each structure. The\u000d\u000astructure visualizations were performed with VESTA.24\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Chem. Sci., 2023, 14, 4806\u20134816 | 4811\u000d\u000aEdge Article Chemical Science\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","structures containing elements present in the synthesis) were\u000d\u000a\ue103ltered out from the Crystallography Open Database (COD) and\u000d\u000aInorganic Crystal Structure Database (ICSD). Using the CIFs obtained from this \ue103ltering, POM cluster structures were isolated,\u000d\u000aresulting in ca. 800 discrete cluster models. Their PDFs were\u000d\u000acalculated using the same Qmin, Qmax and Qdamp values as for the\u000d\u000aexperimental PDF. The calculated PDFs were automatically\u000d\u000acompared to the experimental PDFs using the Pearson correlation, where the PDF range from 1\u20135 \u00c5 was included. This is\u000d\u000aa simple but fast way of quantifying the similarities between two\u000d\u000acurves using the Pearson coefficient.47\u000d\u000aThe ten cluster structures giving the highest Pearson correlation value for the FeWO4 precursor phases are shown in\u000d\u000aFig. 6b. The equivalent for the CoWO4 and NiWO4 precursors\u000d\u000aare found in Fig. S9.\u2020 All the suggested structures for the FeWO4\u000d\u000aprecursor contain Keggin fragments, [W9O34].14 The CoWO4\u000d\u000aresults were almost identical to those of the FeWO4 precursor,\u000d\u000abut for the NiWO4 precursor, some of the suggested structures\u000d\u000ahad more edge-sharing octahedra, for instance the heptatungstate cluster [W7O24]\u000d\u000a6\u2212.\u000d\u000a48 While this method has allowed us to\u000d\u000aidentify the main structural motifs, the clusters found from the\u000d\u000aPearson analysis cannot give a satisfactory \ue103t of the PDF,\u000d\u000aespecially in the range above 4 \u00c5. We therefore move on to\u000d\u000afurther investigate which structural features and connection of\u000d\u000aKeggin fragments can describe the precursor PDFs.\u000d\u000aPrecursor structure analysis using ML-MotEx\u000d\u000aIt is clear from the suggested clusters presented in Fig. 6b that\u000d\u000athe (Fe/Co/Ni)WO4 precursor phases contain tungsten-based\u000d\u000aKeggin fragments. If we assume a formation mechanism\u000d\u000ainvolving some version of a Tourn\u00b4e-type sandwich cluster as\u000d\u000aseen in Fig. 1b, the most chemically relevant cluster containing\u000d\u000asuch fragments is the Fe\u2013W sandwich cluster reported by Barats\u000d\u000aet al.22 We already know that this cluster in itself cannot\u000d\u000adescribe the experimental PDF, but we can investigate whether\u000d\u000athere are parts of this structure which agree with our data and\u000d\u000afrom that, further analyse the precursor structure. For this\u000d\u000apurpose, we use a recently developed automated modelling\u000d\u000amethod, ML-MotEx32 in which thousands of all fragments based\u000d\u000aon a starting cluster model are built and tested against an\u000d\u000aexperimental PDF.29 Using these \ue103ts and an explainable ML\u000d\u000aalgorithm, it is possible to obtain information on the importance of every single atom in the starting model for the \ue103t\u000d\u000aquality. The method is described in detail by Anker et al.32 and\u000d\u000aan overview is given in the ESI.\u2020\u000d\u000aWe use ML-MotEx here for analysis of the PDFs obtained from\u000d\u000athe FeWO4 synthesis, with the Fe\u2013W sandwich cluster23 used as\u000d\u000athe starting model. We \ue103rst created a structure catalogue with 10\u000d\u000a000 structure motifs, which were all fragments built of the Fe\u2013W\u000d\u000asandwich cluster with a varying number of atoms. As described in\u000d\u000adetail in the ESI,\u2020 each generated fragment was \ue103tted to each PDF\u000d\u000aobtained from the reaction, yielding a Rwp value for each \ue103t.\u000d\u000aThese \ue103ts and their corresponding Rwp values were then fed to the\u000d\u000aexplainable ML algorithm, which was trained to predict the Rwp\u000d\u000avalue from a structure motif. A\ue09derwards, the importance and\u000d\u000aeffect of each atom on the \ue103ts could be explained using SHapley\u000d\u000aAdditive explanation (SHAP) values, as described in detail in the\u000d\u000aESI and illustrated in Fig. S10.\u2020 Brie\ue104y, the SHAP amplitude for\u000d\u000aa speci\ue103c atom tells us how much it affects the Rwp value of the \ue103t,\u000d\u000aand the sign of the SHAP values tells us whether the ML model\u000d\u000awants to remove or keep that atom in the structure. A negative\u000d\u000avalue suggests that the atom should be kept in the model to\u000d\u000aobtain a good \ue103t, and a positive value suggests that it should be\u000d\u000aremoved. A large absolute mean SHAP value means that the effect\u000d\u000aon the Rwp value is signi\ue103cant.\u000d\u000aFig. 7a shows the evolution of the average SHAP value obtained for atoms in the cluster structure as a function of synthesis\u000d\u000atime. The standard deviation of the 10 000 points for each metal\u000d\u000aatom was also calculated and is presented as bars around each\u000d\u000acalculated average. A small SHAP standard deviation suggests\u000d\u000athat ML-MotEx is certain about its decision, while the opposite is\u000d\u000atrue for a large standard deviation. To avoid crowdedness, the\u000d\u000aatoms of the cluster have been divided into three parts: the (a)\u000d\u000aperipheral and (b) centre W atoms, and the (c) Fe atoms. Each\u000d\u000aatom pair in this symmetrical cluster has been given a unique\u000d\u000acolour, which corresponds to the colour of the plotted mean\u000d\u000aSHAP values.\u000d\u000aThe SHAP analysis shows several interesting trends, which\u000d\u000aare visualized at representative times for the precursor, intermediate, and product phases in Fig. 7b, where we use colour to\u000d\u000aindicate which polyhedron ML-MotEx wants to keep (green) or\u000d\u000aremove (red). We \ue103rst consider the results for the precursor\u000d\u000astructure, i.e. before heating is initiated. Firstly, ML-MotEx\u000d\u000aseems to prefer W-triad units to be present (groups of three\u000d\u000aedge-sharing [WO6]\u000d\u000a10\u2212 octahedra). Secondly, the triad it prefers\u000d\u000athe most is located at the far edges of the cluster structure,\u000d\u000askewed relative to one another along the bridging layer.\u000d\u000aSpeci\ue103cally, the mean SHAP values tend to be lower for the W\u000d\u000aatoms being the most skewed from the center of the cluster (W1\u2013\u000d\u000a2, W3\u20134, W9\u201310, and W13\u201314). When examining the results from\u000d\u000athe intermediate and product stages of the reaction, this\u000d\u000afavouring of skewed W atoms is not present. Additionally, no\u000d\u000atriad units are preferred by ML-MotEx a\ue09der the formation of the\u000d\u000aintermediate phase. As for Fe atoms, the tetrahedrally coordinated Fe atoms are strongly favoured in the precursor phase.\u000d\u000aThe preference for tetrahedrally coordinated Fe atoms\u000d\u000adecreases when the intermediate phase forms, and in the\u000d\u000aproduct phase, ML-MotEx is indecisive about them. On the\u000d\u000aother hand, a full edge-sharing layer of octahedrally coordinated Fe atoms is preferred in the product. The presence of the\u000d\u000afull edge-sharing bridging layer corresponds well with the\u000d\u000achains of edge-sharing metals present in the wolframite-type\u000d\u000aphase. Interestingly, the presence of both an edge-sharing\u000d\u000abridging layer and tetrahedrally coordinated Fe atoms in the\u000d\u000aintermediate phase shows that this phase has characteristics\u000d\u000afrom both the precursor and the product phase.\u000d\u000aThe favoured skewness of the precursor cluster initially\u000d\u000aseemed rather puzzling, which led us to search for other POMs\u000d\u000awith a higher degree of skewness. Such a structure was found in\u000d\u000athe ICSD database; a recently published structure with the\u000d\u000aformula (Ni6(H2O)9(OH)3(HSiW9O34))2(H2O)12 (ICSD Code 3561\u000d\u000a(ref. 49)). The extracted cluster, a\ue09der replacing Si and Ni with Fe\u000d\u000aand removing crystal water, has the formula (Fe6(FeW9O34))2.\u000d\u000a4812 | Chem. Sci., 2023, 14, 4806\u20134816 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aChemical Science Edge Article\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","The cluster can be described as two Keggin fragments equal to\u000d\u000athe ones in the Fe\u2013W sandwich cluster, but instead of being\u000d\u000aoppositely connected by the same bridging layer, this new POM\u000d\u000ahas the two Keggin fragments connected to their own edgesharing transition metal layer. These two layers are connected\u000d\u000athrough corner-sharing. A visualization of this cluster, along\u000d\u000awith its \ue103t to the FeWO4 precursor PDF, can be seen in Fig. 8.\u000d\u000aThe PDF can be well described by this skewed cluster, showing\u000d\u000athat the less de\ue103ned peaks over r z 5 \u00c5 in the PDF are due to the\u000d\u000aless symmetrical structure of this cluster compared to the\u000d\u000astandard Tourn\u00b4e sandwich-type. It is also possible that the\u000d\u000amixture consists of a broad range of clusters with structural\u000d\u000asimilarities, where a single Keggin fragment [W9O34]\u000d\u000a14\u2212 is the\u000d\u000aonly stable unit present. The CoWO4 and NiWO4 precursor\u000d\u000aphases can also be described by the skewed sandwich cluster, as\u000d\u000aseen in Fig. S11.\u2020 The Rwp value for these \ue103ts is slightly larger,\u000d\u000adue to the less de\ue103ned PDFs of these phases.\u000d\u000aThe skewed cluster has a more equal amount between Fe/Co/\u000d\u000aNi and W (nine W to every seven Fe/Co/Ni) compared to the\u000d\u000aTourn\u00b4e-type sandwich cluster (nine W to every three Fe/Co/Ni).\u000d\u000aThis cluster might therefore be promoted over the Tourn\u00b4e-type\u000d\u000asandwich cluster in our mixtures, as the metal ratio in the\u000d\u000aprecursor is equal to 1. The remaining free [FeO6]\u000d\u000a10\u2212 in the\u000d\u000amixture, which are not incorporated in the model, can explain\u000d\u000athe peak at r z 2.2 \u00c5, which is not fully described in the \ue103t. The\u000d\u000apaste-like appearance of the precursor mixtures points toward\u000d\u000athat these clusters are binding a lot of the water present, and\u000d\u000athat they are weakly bound to each other, forming amorphous\u000d\u000aparticles.\u000d\u000aReaction mechanism\u000d\u000aFig. 9 summarizes the proposed reaction mechanism from\u000d\u000aamorphous precursors to crystalline MWO4. The precursor\u000d\u000astructure can be described as an amorphous mixture of weakly\u000d\u000aFig. 7 (a) The effect of a metal atom's presence in the structure, given\u000d\u000aas the mean and standard deviation (given as vertical lines for each\u000d\u000amean) for each metal atom's SHAP values. The SHAP values are the\u000d\u000aoutput of the ML-MotEx algorithm, trained with fits to 10 000 random\u000d\u000afragments of the [Fe4(FeW9O34)2]\u000d\u000a16\u2212 cluster,20 for each PDF in the in\u000d\u000asitu series. The colour of the metal\u2013oxygen polyhedra in the investigated structure is reflected in the colour of the plots (split into three\u000d\u000asegments for clarification). (b) The resulting cluster structures with fits\u000d\u000afor the precursor, intermediate and product phases after \u22120.17, 1.08\u000d\u000aand 1.42 min of heating. Green and red polyhedra are deemed\u000d\u000asignificant contributors to the fit by the ML algorithm, having a trending\u000d\u000amean SHAP value below \u22120.001 or above +0.001, respectively. Grey\u000d\u000apolyhedra are deemed insignificant by the ML algorithm, with\u000d\u000aa trending mean SHAP value in the range of \u22120.001 to 0.001. Below\u000d\u000aeach of the structures, fits to the data are shown, which includes only\u000d\u000asignificant metal atoms contributing positively to the Rwp value (i.e.\u000d\u000agreen atoms in the structures). The structure visualizations were performed with VESTA.24\u000d\u000aFig. 8 Fit of the skewed (Fe6(FeW9O34))2 cluster, stemming from the\u000d\u000a(Ni6(H2O)9(OH)3(HSiW9O34))2(H2O)12 structure to the FeWO4\u000d\u000aprecursor PDF. Zoom factor for the cluster related to the crystallographic directions of the parent crystal structure, scale factor, isotropic\u000d\u000aatomic displacement factors and d2 were refined.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Chem. Sci., 2023, 14, 4806\u20134816 | 4813\u000d\u000aEdge Article Chemical Science\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","linked, skewed sandwich clusters. The intermediate phase has\u000d\u000asome similar characteristics to the precursor phase, including\u000d\u000atetrahedrally coordinated Fe atoms (not shown in the \ue103gure).\u000d\u000aThe degree of edge-sharing in the structure is the lowest in the\u000d\u000aintermediate phase, despite this being the transitioning state\u000d\u000abetween a highly edge-sharing precursor with stable Fe bridging\u000d\u000alayers and a product with edge-sharing chains. This connects\u000d\u000awell to the breaking up of the W-triad units (groups of three\u000d\u000aedge-sharing [WO6]\u000d\u000a10\u2212 octahedra) in the Keggin fragments,\u000d\u000aproposed by the ML-MotEx algorithm (Fig. 7). Such a break-up is\u000d\u000anecessary to open for growth of the MWO4 structure if the edgesharing Fe bridging layer is assumed to be stable. The intermediate phase can be described as disordered fragments with\u000d\u000athe MWO4 structure, with dominating growth in the b-direction.\u000d\u000aA\ue09der a brief transition through this intermediate phase, growth\u000d\u000awill occur in all directions and an ordered MWO4 structure will\u000d\u000aform, where the metal atoms relax towards the edge of the\u000d\u000aoctahedra, in the b-direction.\u000d\u000aThe lower degree of ordering observed for the CoWO4 and\u000d\u000aNiWO4 precursor phases compared to the FeWO4 precursor\u000d\u000aphase gives us a clue regarding the reason these reactions do\u000d\u000anot form crystalline products like the FeWO4 reaction. A high\u000d\u000adegree of disorder in the precursor phases would naturally lead\u000d\u000ato a high degree of disorder in the intermediate phase, which\u000d\u000aultimately leads to a larger crystallization barrier for the product\u000d\u000aphase.\u000d\u000aConclusions\u000d\u000aHydrothermal synthesis of wolframite-type MWO4 (M : Mn, Fe,\u000d\u000aCo, Ni) from soluble aqueous precursors has been investigated.\u000d\u000aUpon mixing of the aqueous precursors, a crystalline precursor\u000d\u000aformed for the MnWO4 synthesis, while amorphous pastes\u000d\u000aformed for the FeWO4, CoWO4 and NiWO4 syntheses, whose\u000d\u000astructures are distinct from those of the ions present in the\u000d\u000aindividual precursor salt solutions. Upon heating, the crystalline MnWO4 precursor converted directly to a crystalline\u000d\u000awolframite-type MnWO4 phase, while the remaining syntheses\u000d\u000aproduced an intermediate phase. The FeWO4 intermediate\u000d\u000aphase eventually turned into a crystalline wolframite-type\u000d\u000aFeWO4 phase, while the CoWO4 and NiWO4 reactions halted\u000d\u000aa\ue09der the formation of the intermediate phase.\u000d\u000aUsing database mining, it was concluded that the structure\u000d\u000aof the amorphous precursor pastes should include a Keggin\u000d\u000afragment [W9O34]\u000d\u000a14\u2212. The precursor material had some similarity to a Tourn\u00b4e sandwich-type, and by using this cluster as\u000d\u000aa starting structure, we analyzed its \ue103t to the PDF using our\u000d\u000aalgorithm ML-MotEx,32 which can be applied to determine the\u000d\u000asigni\ue103cance of individual structural features in a structure in\u000d\u000aa PDF \ue103t. For PDFs of the amorphous precursor phase, MLMotEx preferred W atoms giving a skewness of the cluster,\u000d\u000aand a signi\ue103cant presence of tetrahedrally coordinated Fe\u000d\u000aatoms. This led us to identify a skewed transition metal/\u000d\u000atungsten oxide cluster which described the precursor PDF\u000d\u000awell. This skewed cluster had a more equal ratio between Fe\u000d\u000aand W atoms compared to the Tourn\u00b4e sandwich cluster, which\u000d\u000ais chemically reasonable considering the stoichiometry in the\u000d\u000aactual mixture.\u000d\u000aThe intermediate phase appearing a\ue09der the amorphous\u000d\u000aprecursors had many structural similarities with both the\u000d\u000aprecursor and product phases. Our analysis showed a signi\ue103-\u000d\u000acant presence of tetrahedrally coordinated Fe atoms in the\u000d\u000aintermediate structure along with a strong presence of an edgesharing bridging layer, despite the low edge-to corner-sharing\u000d\u000aintensity ratio observed at this point in the reaction. This led\u000d\u000aus to assume that there is a breaking up of W triad units in the\u000d\u000aKeggin fragments, which opens for growth of the structure. We\u000d\u000afound that the more disordered the precursor phase is, the\u000d\u000alonger the reaction time required to form crystalline products.\u000d\u000aIn summary, our work establishes the importance of polyoxometalates in the formation mechanisms of mixed metal\u000d\u000aoxides, and we provide an atomistic understanding of the\u000d\u000aprocess. By understanding how materials initially assemble in\u000d\u000aa solution, we are signi\ue103cantly closer to fully understanding\u000d\u000ahow mixed metal oxides form.\u000d\u000aExperimental methods\u000d\u000aChemical procedures\u000d\u000aFour different precursor mixtures were prepared; 0.01 mol\u000d\u000aNa2WO4$2H2O dissolved in 5 mL deionized water was mixed\u000d\u000awith 0.01 mol metal chloride hydrate (MnCl2$4H2O, FeCl2-\u000d\u000a$4H2O, CoCl2$6H2O, and NiCl2$6H2O) dissolved separately in\u000d\u000a5 mL deionized water. This resulted in 1 M tungsten\u000d\u000aFig. 9 Proposed reaction mechanism for the hydrothermal synthesis\u000d\u000aof MWO4. Growth directions are indicated by brown arrows. Skewed\u000d\u000asandwich-clusters reorganize into chains of M\u2013O and W\u2013O octahedra, equivalent to the c-directions in the final MWO4 structure.\u000d\u000aEventually, the structure will start growing in the other directions as\u000d\u000awell. The structure visualizations were performed with VESTA.24\u000d\u000a4814 | Chem. Sci., 2023, 14, 4806\u20134816 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aChemical Science Edge Article\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","concentrations. The mixing happened under vigorous stirring\u000d\u000aand the resulting mixtures had a pH of 6 as measured with\u000d\u000aa HANNA HI 2020-02 pH meter. All the chemicals used were\u000d\u000afrom Sigma-Aldrich and of $99% purity.\u000d\u000aIn situ X-ray total scattering experiments\u000d\u000aIn situ total scattering experiments were performed on beamline\u000d\u000aP02.1 at PETRAIII, DESY in Hamburg, using an X-ray wavelength of 0.2072 \u00c5 (59.82 keV). The scattered X-rays were\u000d\u000acollected with a PerkinElmer 2D detector, positioned 216 mm\u000d\u000aaway from the sample, typical for rapid acquisition PDF (RAPDF) experiments.41 In situ experiments were performed by\u000d\u000ainjecting the prepared precursor mixtures into a fused silica\u000d\u000acapillary (inner and outer diameter of 0.70 and 0.88 mm,\u000d\u000arespectively), connected to Swagelok\u00ae tubing. The fused silica\u000d\u000acapillary was stabilized by a custom-built steel frame and the\u000d\u000asurrounding tubing was then pressurized to 100 bar and kept\u000d\u000astatic using a standard high-performance liquid chromatography pump, similar to the setup of Becker et al.50 Rapid heating\u000d\u000ato 160 \u00b0C was initiated with a hot-air blower. It took ca. \u223c90 s for\u000d\u000athe sample inside the capillary to reach the reaction temperature. Frames were collected with a time-resolution of 5 seconds/\u000d\u000aframe.\u000d\u000aData treatment and PDF modelling\u000d\u000aThe collected total scattering data were integrated using\u000d\u000aPyFAI.51 Background subtraction, correction, normalization and\u000d\u000aFourier transformation to PDFs31 were performed using\u000d\u000aPDFgetX3.52 The Fourier transformation was executed using the\u000d\u000aparameters Qmin = 1.2 \u00c5\u22121\u000d\u000a, Qmax = 18 \u00c5\u22121\u000d\u000a, Qmaxinst = 20 \u00c5\u22121 and\u000d\u000ar-poly = 0.9 \u00c5. The instrumental parameters were obtained from\u000d\u000aPDF analysis of a NIST CeO2 standard, giving a Qdamp of 0.043\u000d\u000a\u00c5\u22121 and a Qbroad of 0.021 \u00c5\u22121\u000d\u000a. PDF modelling of precursor\u000d\u000astructures was performed in DiffPy-CMI,53 by \ue103tting structures\u000d\u000aobtained from the Inorganic Crystal Structure Database (ICSD).\u000d\u000aSequential \ue103ts of PDFs from the crystalline products were performed in reverse (starting with the last acquired frame) in\u000d\u000aPDFgui.54 Additional information about the modelling parameters and further analysis can be found in the ESI.\u2020\u000d\u000aEx situ syntheses and PXRD acquisition\u000d\u000aThe nanocrystalline intermediate phases of the CoWO4 and\u000d\u000aFeWO4 syntheses were synthesized using autoclaves and characterized ex situ. An identical precursor mixture as for the in situ\u000d\u000aexperiments, but with a total volume of 10 mL, was prepared for\u000d\u000aboth syntheses, before being transferred to 25 mL Te\ue104on-lined\u000d\u000astainless-steel autoclaves (40% \ue103lled autoclave). The autoclaves\u000d\u000awere then placed in an oven at 110 \u00b0C for 24 hours (corresponding to 1.3 bar autogenous pressure55,56). The heating rate\u000d\u000ain the autoclave was not measured and is unknown. A\ue09der the\u000d\u000aautoclaves had cooled, the suspensions were washed twice with\u000d\u000a40 mL of demineralized water and once with 40 mL of ethanol.\u000d\u000aA\ue09der drying at room-temperature for two days in a crystallization dish, the powders were gently mortared. Ex situ PXRD data\u000d\u000awere collected with a Bruker D8 Discover using a Bragg\u2013\u000d\u000aBrentano setup with a Cu Ka source (1.54 \u00c5) and a LYNXEYE\u000d\u000adetector.\u000d\u000aData availability\u000d\u000aThe data that support the \ue103ndings of this study are available\u000d\u000afrom the corresponding author upon reasonable request.\u000d\u000aAuthor contributions\u000d\u000aS. L. S. and K. M. \u00d8. J. conceptualized the project. M. C. W., M. J.,\u000d\u000aand T. L. C. performed the in situ experiments. S. L. S. performed the ex situ experiments. S. L. S. analyzed the data with\u000d\u000ainput from A. S. A. and E. T. S. K. S. L. S., A. S. A., and K. M. \u00d8. J.\u000d\u000awrote the original dra\ue09d. All authors contributed to the review\u000d\u000aand editing of the paper. K. M. \u00d8. J. supervised the project.\u000d\u000aConflicts of interest\u000d\u000aThere are no con\ue104icts to declare.\u000d\u000aAcknowledgements\u000d\u000aThis work is part of a project that has received funding from the\u000d\u000aEuropean Research Council (ERC) under the European Union's\u000d\u000aHorizon 2020 Research and Innovation Programme (grant\u000d\u000aagreement No. 804066). We are grateful to the Villum Foundation for \ue103nancial support through a Villum Young Investigator\u000d\u000agrant. We furthermore thank DANSCATT (supported by the\u000d\u000aDanish Agency for Science and Higher Education) for support.\u000d\u000aFunding from the Danish Ministry of Higher Education and\u000d\u000aScience through the SMART Lighthouse is gratefully acknowledged. We acknowledge DESY (Hamburg, Germany), a member\u000d\u000aof the Helmholtz Association HGF, for the provision of experimental facilities. Parts of this research were carried out at\u000d\u000abeamline P02.1 at Petra III and we would like to thank Martin\u000d\u000aEtter for assistance in using the beamline.\u000d\u000aReferences\u000d\u000a1 E. D. Bojesen and B. B. Iversen, CrystEngComm, 2016, 18,\u000d\u000a8332\u20138353.\u000d\u000a2 D. Gebauer, A. V\u00f6lkel and H. C\u00f6lfen, Science, 2008, 322,\u000d\u000a1819\u20131822.\u000d\u000a3 R. Demichelis, P. Raiteri, J. D. Gale, D. Quigley and\u000d\u000aD. Gebauer, Nat. Commun., 2011, 2, 1\u20138.\u000d\u000a4 D. Gebauer and H. C\u00f6lfen, Nano Today, 2011, 6, 564\u2013584.\u000d\u000a5 J. Ke, M. A. Younis, Y. Kong, H. Zhou, J. Liu, L. Lei and\u000d\u000aY. Hou, Nano-Micro Lett., 2018, 10, 1\u201327.\u000d\u000a6 H. Jia, J. Stark, L. Qin Zhou, C. Ling, T. Sekito and Z. Markin,\u000d\u000aRSC Adv., 2012, 2, 10874\u201310881.\u000d\u000a7 M. G. Joaqu\u00b4\u0131n-Morales, A. F. Fuentes, S. M. Montemayor,\u000d\u000aM. J. Mel\u00b4endez-Zaragoza, J. M. S. Guti\u00b4errez, A. L. Ortiz and\u000d\u000aV. Collins-Mart\u00b4\u0131nez, Int. J. Hydrogen Energy, 2019, 44,\u000d\u000a12390\u201312398.\u000d\u000a8 C. Liu, H. L\u00fc, C. Yu, X. Wu and P. Wang, Mater. Lett., 2019,\u000d\u000a257, 126707.\u000d\u000a\u00a9 2023 The Author(s). Published by the Royal Society of Chemistry Chem. Sci., 2023, 14, 4806\u20134816 | 4815\u000d\u000aEdge Article Chemical Science\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","9 S. Rajagopal, D. Nataraj, O. Y. Khyzhun, Y. Djaoued,\u000d\u000aJ. Robichaud and D. Mangalaraj, J. Alloys Compd., 2010,\u000d\u000a493, 340\u2013345.\u000d\u000a10 J. Tian, Y. Xue, X. Yu, Y. Pei, H. Zhang and J. Wang, RSC Adv.,\u000d\u000a2018, 8, 417470.\u000d\u000a11 M. Juelsholt, T. L. Christiansen and K. M. \u00d8. Jensen, J. Phys.\u000d\u000aChem. C, 2019, 123, 5110\u20135119.\u000d\u000a12 E. D. B\u00f8jesen, K. M. \u00d8. Jensen, C. Tyrsted, A. H. Mamakhel,\u000d\u000aH. L. Andersen, H. Reardon, J. Chevalier, A.-C. Dippel and\u000d\u000aB. B. Iversen, Chem. Sci., 2016, 7, 6394\u20136406.\u000d\u000a13 M. Ammam, J. Mater. Chem. A, 2013, 1, 6291\u20136312.\u000d\u000a14 N. I. Gumerova, A. Rompel and U. Wien, Chem. Soc. Rev.,\u000d\u000a2020, 49, 7568\u20137601.\u000d\u000a15 M. Hutin, M. H. Rosnes, D. Long and L. Cronin, Compr.\u000d\u000aInorg. Chem. II, 2013, 2, 241\u2013269.\u000d\u000a16 D. L. Long, R. Tsunashima and L. Cronin, Angew. Chem., Int.\u000d\u000aEd., 2010, 49, 1736\u20131758.\u000d\u000a17 X. L\u00b4opez, J. J. Carb\u00b4o, C. Bo and J. M. Poblet, Chem. Soc. Rev.,\u000d\u000a2012, 41, 7537\u20137571.\u000d\u000a18 M. Nyman and P. C. Burns, Chem. Soc. Rev., 2012, 41, 7354\u2013\u000d\u000a7367.\u000d\u000a19 O. Sadeghi, L. N. Zakharov and M. Nyman, Science, 2015,\u000d\u000a347, 1359\u20131362.\u000d\u000a20 E. M. Limanski, M. Piepenbrink, E. Droste, K. Burgemeister\u000d\u000aand B. Krebs, J. Cluster Sci., 2002, 13, 369\u2013379.\u000d\u000a21 C. M. Tourn\u00b4e, G. F. Tourn\u00b4e and F. Zonnevijlle, J. Chem. Soc.,\u000d\u000aDalton Trans., 1991, 143\u2013155.\u000d\u000a22 D. Barats, G. Leitus, R. Popovitz-Biro, L. J. W. Shimon and\u000d\u000aR. Neumann, Angew. Chem., Int. Ed., 2008, 47, 9908\u20139912.\u000d\u000a23 A. R. Kampf, J. M. Hughes, B. P. Nash, S. E. Wright,\u000d\u000aG. R. Rossman and J. Marty, Am. Mineral., 2014, 99, 1045\u2013\u000d\u000a1051.\u000d\u000a24 K. Momma and F. Izumi, J. Appl. Crystallogr., 2011, 44, 1272\u2013\u000d\u000a1276.\u000d\u000a25 P. J. Chupas, K. W. Chapman, G. Jennings, P. L. Lee and\u000d\u000aC. P. Grey, J. Am. Chem. Soc., 2007, 129, 13822\u201313824.\u000d\u000a26 D. S. Cook, Y. Wu, K. Lienau, R. Mor\u00b4e, R. J. Kashtiban,\u000d\u000aO. V. Magdysyuk, G. R. Patzke and R. I. Walton, Chem.\u000d\u000aMater., 2017, 29, 5053\u20135057.\u000d\u000a27 O. G. Grendal, I. E. Nylund, A. B. Blichfeld, S. Tominaka,\u000d\u000aK. Ohara, S. M. Selbach, T. Grande and M. A. Einarsrud,\u000d\u000aChem. \u2013 Eur. J., 2020, 26, 9348\u20139355.\u000d\u000a28 K. M. \u00d8. Jensen, M. Christensen, P. Juhas, C. Tyrsted,\u000d\u000aE. D. B\u00f8jesen, N. Lock, S. J. L. Billinge and B. B. Iversen, J.\u000d\u000aAm. Chem. Soc., 2012, 134, 6785\u20136792.\u000d\u000a29 T. Lindahl Christiansen, E. T. S. Kj\u00e6r, A. Kovyakh,\u000d\u000aM. L. R\u00f6deren, M. H\u00f8j, T. Vosch and K. M. \u00d8. Jensen, J.\u000d\u000aAppl. Crystallogr., 2020, 53, 148\u2013158.\u000d\u000a30 C. Tyrsted, K. M. \u00d8rnsbjerg Jensen, E. D. B\u00f8jesen, N. Lock,\u000d\u000aM. Christensen, S. J. L. Billinge and B. Brummerstedt\u000d\u000aIversen, Angew. Chem., Int. Ed., 2012, 51, 9030\u20139033.\u000d\u000a31 T. Lindahl Christiansen, S. R. Cooper and K. M. \u00d8. Jensen,\u000d\u000aNanoscale Adv., 2020, 13, 8725\u20138735.\u000d\u000a32 A. S. Anker, E. T. S. Kj\u00e6r, M. Juelsholt, T. L. Christiansen,\u000d\u000aS. L. Skj\u00e6rv\u00f8, M. R. V. J\u00f8rgensen, I. Kantor, D. R. S\u00f8rensen,\u000d\u000aS. J. L. Billinge, R. Selvan and K. M. \u00d8. Jensen, npj Comput.\u000d\u000aMater., 2022, 8, 213.\u000d\u000a33 S. M. Lundberg, G. Erion, H. Chen, A. DeGrave, J. M. Prutkin,\u000d\u000aB. Nair, R. Katz, J. Himmelfarb, N. Bansal and S.-I. Lee, Nat.\u000d\u000aMach. Intell., 2020, 2, 56\u201367.\u000d\u000a34 S. M. Lundberg and S.-I. Lee, Proceedings of the 31st\u000d\u000aInternational Conference on Neural Information Processing\u000d\u000aSystems, 2017, pp. 4765\u20134774.\u000d\u000a35 G. He, J. Li, W. Li, B. Li, N. Noor, K. Xu, J. Hu and I. P. Parkin,\u000d\u000aJ. Mater. Chem. A, 2015, 3, 14272\u201314278.\u000d\u000a36 S. Jha, S. Mehta, E. Chen, S. S. Sankar, S. Kundu and\u000d\u000aH. Liang, Mater. Adv., 2020, 1, 2124\u20132135.\u000d\u000a37 N. L. M. Tri, D. S. Duc, D. Van Thuan, T. A. Tahtamouni,\u000d\u000aT. D. Pham, D. T. Tran, N. Thi Phuong Le Chi and\u000d\u000aV. N. Nguyen, Chem. Phys., 2019, 525, 1\u20137.\u000d\u000a38 X. X. Wang, Y. Li, M. C. Liu and L. B. Kong, Ionics, 2018, 24,\u000d\u000a363\u2013372.\u000d\u000a39 J. Macavei and H. Schulz, Z. Kristallogr. \u2013 New Cryst. Struct.,\u000d\u000a1993, 207, 193\u2013208.\u000d\u000a40 D. Ulk\u00fc, \u00a8 Z. Kristallogr. \u2013 New Cryst. Struct., 1967, 124, 192\u2013\u000d\u000a219.\u000d\u000a41 P. J. Chupas, X. Qiu, J. C. Hanson, P. L. Lee, C. P. Grey and\u000d\u000aS. J. L. Billinge, J. Appl. Crystallogr., 2003, 36, 1342\u20131347.\u000d\u000a42 L. Niu, Z. Li, Y. Xu, J. Sun, W. Hong, X. Liu, J. Wang and\u000d\u000aS. Yang, ACS Appl. Mater. Interfaces, 2013, 5, 8044\u20138052.\u000d\u000a43 M. I. Ahmed, A. Adam, A. Khan, A. U. Rehman,\u000d\u000aM. Qamaruddin, M. N. Siddiqui and M. Qamar, Mater.\u000d\u000aLett., 2016, 183, 281\u2013284.\u000d\u000a44 H. Feng, Y. Xi and Q. Huang, Dalton Trans., 2020, 49, 12242\u2013\u000d\u000a12248.\u000d\u000a45 H. Weitzel, Z. Kristallogr. \u2013 Cryst. Mater., 1976, 144, 238\u2013258.\u000d\u000a46 S. J. Angus-Dunne, J. A. Irwin, R. C. Burns, G. A. Lawrance\u000d\u000aand D. C. Craig, J. Chem. Soc., Dalton Trans., 1993, 1993,\u000d\u000a2717\u20132726.\u000d\u000a47 E. T. S. Kj\u00e6r, O. Aalling-Frederiksen, L. Yang, N. K. Thomas,\u000d\u000aS. J. L. Billinge and K. M. \u00d8. Jensen, Chem. Methods, 2022, 2,\u000d\u000ae202200034.\u000d\u000a48 K. G. Burtseva, T. S. Chernaya and M. I. Sirota, Akad. Nauk\u000d\u000aSSSR, 1978, 243, 104\u2013107.\u000d\u000a49 X.-M. Luo, L. Chen, Y.-Y. Dong, J. Li, C.-H. Cui, J.-P. Cao and\u000d\u000aY. Xu, Dalton Trans., 2018, 47, 9504\u20139511.\u000d\u000a50 J. Becker, M. Bremholm, C. Tyrsted, B. R. Pauw,\u000d\u000aK. M. \u00d8. Jensen, J. Eltzholtz, M. Christensen and\u000d\u000aB. B. Iversen, J. Appl. Crystallogr., 2010, 43, 729\u2013736.\u000d\u000a51 G. Ashiotis, A. Deschildre, Z. Nawaz, J. P. Wright,\u000d\u000aD. Karkoulis, F. E. Picca and J. Kieffer, J. Appl. Crystallogr.,\u000d\u000a2015, 48, 510\u2013519.\u000d\u000a52 P. Juh\u00b4as, T. Davis, C. L. Farrow and S. J. L. Billinge, J. Appl.\u000d\u000aCrystallogr., 2012, 46, 560\u2013566.\u000d\u000a53 P. Juh\u00b4as, C. L. Farrow, X. Yang, K. R. Knox and\u000d\u000aS. J. L. Billinge, Acta Crystallogr., Sect. A: Found.\u000d\u000aCrystallogr., 2015, 71, 562\u2013568.\u000d\u000a54 C. L. Farrow, P. Juhas, J. W. Liu, D. Bryndin, E. S. Bo\u02c7zin,\u000d\u000aJ. Bloch, P. Th and S. J. L. Billinge, J. Phys.: Condens.\u000d\u000aMatter, 2007, 19, 335219.\u000d\u000a55 A. Rabenau, Angew. Chem., Int. Ed., 1985, 24, 1026\u20131040.\u000d\u000a56 R. I. Walton, Chem. Soc. Rev., 2002, 31, 230\u2013238.\u000d\u000a4816 | Chem. Sci., 2023, 14, 4806\u20134816 \u00a9 2023 The Author(s). Published by the Royal Society of Chemistry\u000d\u000aChemical Science Edge Article\u000d\u000aOpen Access Article. Published on 06 April 2023. Downloaded on 5/19/2023 3:58:57 PM. This article is licensed under a Creative Commons Attribution-NonCommercial 3.0 Unported Licence.\u000d\u000aView Article Online","","In\u0009 the\u0009world\u0009of\u0009crime\u0009dramas,\u0009a\u0009 fingerprint\u0009left\u0009at\u0009a\u0009crime\u0009scene\u0009can\u0009be\u0009 the\u0009crucial\u0009evidence\u0009 that\u0009identifies\u0009 the\u0009\u000d\u000acriminal.\u0009Similarly,\u0009in\u0009the\u0009realm\u0009of\u0009materials\u0009chemistry,\u0009X-ray\u0009scattering\u0009experiments\u0009give\u0009us\u0009a\u0009distinct\u0009'fingerprint'\u0009\u000d\u000aof\u0009every\u0009material.\u0009This\u0009fingerprint\u0009reveals\u0009the\u0009intricate\u0009details\u0009of\u0009its\u0009atomic\u0009structure.\u0009The\u0009challenge?\u0009Much\u0009like\u0009\u000d\u000awaiting\u0009 for\u0009 the\u0009 season\u0009 finale\u0009 to\u0009 expose\u0009 the\u0009 criminal,\u0009 researchers\u0009 often\u0009 spend\u0009 a\u0009 significant\u0009 amount\u0009 of\u0009 time\u0009\u000d\u000ainterpreting\u0009these\u0009X-ray\u0009patterns\u0009to\u0009identify\u0009the\u0009material.\u000d\u000aMy\u0009PhD\u0009work\u0009dives\u0009right\u0009into\u0009this\u0009detective\u0009challenge.\u0009Traditional\u0009methods\u0009are\u0009somewhat\u0009slow,\u0009akin\u0009to\u0009a\u0009detective\u0009\u000d\u000amanually\u0009 going\u0009 through\u0009 files,\u0009 looking\u0009 for\u0009 that\u0009 one\u0009 matching\u0009 fingerprint.\u0009 But\u0009 what\u0009 if\u0009 we\u0009 could\u0009 speed\u0009 this\u0009 up,\u0009\u000d\u000aidentifying\u0009the\u0009matching\u0009structure\u0009as\u0009rapidly\u0009as\u0009modern\u0009crime\u0009databases\u0009can\u0009match\u0009a\u0009suspect's\u0009prints?\u000d\u000aTo\u0009 do\u0009 that,\u0009 we\u0009 use\u0009 machine learning\u0009 \u2013\u0009 our\u0009 high-speed\u0009 detective\u0009 assistant.\u0009 This\u0009 advanced\u0009 computational\u0009 tool,\u0009\u000d\u000atrained\u0009 on\u0009 vast\u0009amounts\u0009 of\u0009 data,\u0009 has\u0009 the\u0009ability\u0009 to\u0009 recognise\u0009 patterns\u0009almost\u0009instantly.\u0009With\u0009it,\u0009we\u0009 can\u0009almost\u0009\u000d\u000ainstantly\u0009match\u0009a\u0009given\u0009X-ray\u0009fingerprint\u0009to\u0009its\u0009corresponding\u0009material\u0009structure.\u0009It\u0009is\u0009like\u0009having\u0009a\u0009top\u0009detective\u0009\u000d\u000awith\u0009a\u0009brilliant\u0009memory\u0009who\u0009recalls\u0009every\u0009suspect's\u0009fingerprint.\u000d\u000aHowever,\u0009even\u0009the\u0009best\u0009detectives\u0009can\u0009occasionally\u0009be\u0009challenged\u0009by\u0009unfamiliar\u0009or\u0009complex\u0009fingerprints.\u0009Curious\u0009\u000d\u000aabout\u0009these\u0009thrilling\u0009challenges?\u0009Well,\u0009for\u0009that,\u0009you\u0009will\u0009have\u0009to\u0009delve\u0009deeper\u0009into\u0009my\u0009PhD\u0009dissertation.\u0009\u000d\u000aFront\u0009cover\u0009illustration:\u0009A\u0009machine\u0009learning\u0009model,\u0009DeepStruc,\u0009is\u0009analysing\u0009pair distribution\u0009function (red)\u0009and\u0009\u000d\u000asmall-angle\u0009X-ray\u0009scattering (blue)\u0009data\u0009obtained\u0009from\u0009the\u0009growth\u0009of\u0009atomically\u0009precise\u0009nanoclusters in\u0009solution.\u0009\u000d\u000aBack\u0009cover\u0009illustration:\u0009Illustration\u0009of\u0009how\u0009beautiful,\u0009clean\u0009and\u0009simple\u0009simulated\u0009data\u0009(blue\u0009side)\u0009must\u0009be\u0009bridged\u0009\u000d\u000awith\u0009messy,\u0009chaotic\u0009experimental\u0009data\u0009(red\u0009side)\u0009to\u0009truly\u0009open\u0009the\u0009machine\u0009learning toolbox\u0009for\u0009experimental\u0009data."]